<script>// Pandoc 2.9 adds attributes on both header and div. We remove the former (to
  // be compatible with the behavior of Pandoc < 2.8).
  document.addEventListener('DOMContentLoaded', function (e) {
    var hs = document.querySelectorAll("div.section[class*='level'] > :first-child");
    var i, h, a;
    for (i = 0; i < hs.length; i++) {
      h = hs[i];
      if (!/^h[1-6]$/i.test(h.tagName)) continue;  // it should be a header h1-h6
      a = h.attributes;
      while (a.length > 0) h.removeAttribute(a[0].name);
    }
  });
</script>
<script>
* @preserve HTML5 Shiv 3.7.2 | @afarkas @jdalton @jon_neal @rem | MIT/GPL2 Licensed
*/
  // Only run this code in IE 8
  if (!!window.navigator.userAgent.match("MSIE 8")) {
    !function (a, b) { function c(a, b) { var c = a.createElement("p"), d = a.getElementsByTagName("head")[0] || a.documentElement; return c.innerHTML = "x<style>" + b + "</style>", d.insertBefore(c.lastChild, d.firstChild) } function d() { var a = t.elements; return "string" == typeof a ? a.split(" ") : a } function e(a, b) { var c = t.elements; "string" != typeof c && (c = c.join(" ")), "string" != typeof a && (a = a.join(" ")), t.elements = c + " " + a, j(b) } function f(a) { var b = s[a[q]]; return b || (b = {}, r++, a[q] = r, s[r] = b), b } function g(a, c, d) { if (c || (c = b), l) return c.createElement(a); d || (d = f(c)); var e; return e = d.cache[a] ? d.cache[a].cloneNode() : p.test(a) ? (d.cache[a] = d.createElem(a)).cloneNode() : d.createElem(a), !e.canHaveChildren || o.test(a) || e.tagUrn ? e : d.frag.appendChild(e) } function h(a, c) { if (a || (a = b), l) return a.createDocumentFragment(); c = c || f(a); for (var e = c.frag.cloneNode(), g = 0, h = d(), i = h.length; i > g; g++)e.createElement(h[g]); return e } function i(a, b) { b.cache || (b.cache = {}, b.createElem = a.createElement, b.createFrag = a.createDocumentFragment, b.frag = b.createFrag()), a.createElement = function (c) { return t.shivMethods ? g(c, a, b) : b.createElem(c) }, a.createDocumentFragment = Function("h,f", "return function(){var n=f.cloneNode(),c=n.createElement;h.shivMethods&&(" + d().join().replace(/[\w\-:]+/g, function (a) { return b.createElem(a), b.frag.createElement(a), 'c("' + a + '")' }) + ");return n}")(t, b.frag) } function j(a) { a || (a = b); var d = f(a); return !t.shivCSS || k || d.hasCSS || (d.hasCSS = !!c(a, "article,aside,dialog,figcaption,figure,footer,header,hgroup,main,nav,section{display:block}mark{background:#FF0;color:#000}template{display:none}")), l || i(a, d), a } var k, l, m = "3.7.2", n = a.html5 || {}, o = /^<|^(?:button|map|select|textarea|object|iframe|option|optgroup)$/i, p = /^(?:a|b|code|div|fieldset|h1|h2|h3|h4|h5|h6|i|label|li|ol|p|q|span|strong|style|table|tbody|td|th|tr|ul)$/i, q = "_html5shiv", r = 0, s = {}; !function () { try { var a = b.createElement("a"); a.innerHTML = "<xyz></xyz>", k = "hidden" in a, l = 1 == a.childNodes.length || function () { b.createElement("a"); var a = b.createDocumentFragment(); return "undefined" == typeof a.cloneNode || "undefined" == typeof a.createDocumentFragment || "undefined" == typeof a.createElement }() } catch (c) { k = !0, l = !0 } }(); var t = { elements: n.elements || "abbr article aside audio bdi canvas data datalist details dialog figcaption figure footer header hgroup main mark meter nav output picture progress section summary template time video", version: m, shivCSS: n.shivCSS !== !1, supportsUnknownElements: l, shivMethods: n.shivMethods !== !1, type: "default", shivDocument: j, createElement: g, createDocumentFragment: h, addElements: e }; a.html5 = t, j(b) }(this, document);
  };
</script>
<script>/*! Respond.js v1.4.2: min/max-width media query polyfill * Copyright 2013 Scott Jehl
* Licensed under https://github.com/scottjehl/Respond/blob/master/LICENSE-MIT
*  */

  // Only run this code in IE 8
  if (!!window.navigator.userAgent.match("MSIE 8")) {
    !function (a) { "use strict"; a.matchMedia = a.matchMedia || function (a) { var b, c = a.documentElement, d = c.firstElementChild || c.firstChild, e = a.createElement("body"), f = a.createElement("div"); return f.id = "mq-test-1", f.style.cssText = "position:absolute;top:-100em", e.style.background = "none", e.appendChild(f), function (a) { return f.innerHTML = '&shy;<style media="' + a + '"> #mq-test-1 { width: 42px; }</style>', c.insertBefore(e, d), b = 42 === f.offsetWidth, c.removeChild(e), { matches: b, media: a } } }(a.document) }(this), function (a) { "use strict"; function b() { u(!0) } var c = {}; a.respond = c, c.update = function () { }; var d = [], e = function () { var b = !1; try { b = new a.XMLHttpRequest } catch (c) { b = new a.ActiveXObject("Microsoft.XMLHTTP") } return function () { return b } }(), f = function (a, b) { var c = e(); c && (c.open("GET", a, !0), c.onreadystatechange = function () { 4 !== c.readyState || 200 !== c.status && 304 !== c.status || b(c.responseText) }, 4 !== c.readyState && c.send(null)) }; if (c.ajax = f, c.queue = d, c.regex = { media: /@media[^\{]+\{([^\{\}]*\{[^\}\{]*\})+/gi, keyframes: /@(?:\-(?:o|moz|webkit)\-)?keyframes[^\{]+\{(?:[^\{\}]*\{[^\}\{]*\})+[^\}]*\}/gi, urls: /(url\()['"]?([^\/\)'"][^:\)'"]+)['"]?(\))/g, findStyles: /@media *([^\{]+)\{([\S\s]+?)$/, only: /(only\s+)?([a-zA-Z]+)\s?/, minw: /\([\s]*min\-width\s*:[\s]*([\s]*[0-9\.]+)(px|em)[\s]*\)/, maxw: /\([\s]*max\-width\s*:[\s]*([\s]*[0-9\.]+)(px|em)[\s]*\)/ }, c.mediaQueriesSupported = a.matchMedia && null !== a.matchMedia("only all") && a.matchMedia("only all").matches, !c.mediaQueriesSupported) { var g, h, i, j = a.document, k = j.documentElement, l = [], m = [], n = [], o = {}, p = 30, q = j.getElementsByTagName("head")[0] || k, r = j.getElementsByTagName("base")[0], s = q.getElementsByTagName("link"), t = function () { var a, b = j.createElement("div"), c = j.body, d = k.style.fontSize, e = c && c.style.fontSize, f = !1; return b.style.cssText = "position:absolute;font-size:1em;width:1em", c || (c = f = j.createElement("body"), c.style.background = "none"), k.style.fontSize = "100%", c.style.fontSize = "100%", c.appendChild(b), f && k.insertBefore(c, k.firstChild), a = b.offsetWidth, f ? k.removeChild(c) : c.removeChild(b), k.style.fontSize = d, e && (c.style.fontSize = e), a = i = parseFloat(a) }, u = function (b) { var c = "clientWidth", d = k[c], e = "CSS1Compat" === j.compatMode && d || j.body[c] || d, f = {}, o = s[s.length - 1], r = (new Date).getTime(); if (b && g && p > r - g) return a.clearTimeout(h), h = a.setTimeout(u, p), void 0; g = r; for (var v in l) if (l.hasOwnProperty(v)) { var w = l[v], x = w.minw, y = w.maxw, z = null === x, A = null === y, B = "em"; x && (x = parseFloat(x) * (x.indexOf(B) > -1 ? i || t() : 1)), y && (y = parseFloat(y) * (y.indexOf(B) > -1 ? i || t() : 1)), w.hasquery && (z && A || !(z || e >= x) || !(A || y >= e)) || (f[w.media] || (f[w.media] = []), f[w.media].push(m[w.rules])) } for (var C in n) n.hasOwnProperty(C) && n[C] && n[C].parentNode === q && q.removeChild(n[C]); n.length = 0; for (var D in f) if (f.hasOwnProperty(D)) { var E = j.createElement("style"), F = f[D].join("\n"); E.type = "text/css", E.media = D, q.insertBefore(E, o.nextSibling), E.styleSheet ? E.styleSheet.cssText = F : E.appendChild(j.createTextNode(F)), n.push(E) } }, v = function (a, b, d) { var e = a.replace(c.regex.keyframes, "").match(c.regex.media), f = e && e.length || 0; b = b.substring(0, b.lastIndexOf("/")); var g = function (a) { return a.replace(c.regex.urls, "$1" + b + "$2$3") }, h = !f && d; b.length && (b += "/"), h && (f = 1); for (var i = 0; f > i; i++) { var j, k, n, o; h ? (j = d, m.push(g(a))) : (j = e[i].match(c.regex.findStyles) && RegExp.$1, m.push(RegExp.$2 && g(RegExp.$2))), n = j.split(","), o = n.length; for (var p = 0; o > p; p++)k = n[p], l.push({ media: k.split("(")[0].match(c.regex.only) && RegExp.$2 || "all", rules: m.length - 1, hasquery: k.indexOf("(") > -1, minw: k.match(c.regex.minw) && parseFloat(RegExp.$1) + (RegExp.$2 || ""), maxw: k.match(c.regex.maxw) && parseFloat(RegExp.$1) + (RegExp.$2 || "") }) } u() }, w = function () { if (d.length) { var b = d.shift(); f(b.href, function (c) { v(c, b.href, b.media), o[b.href] = !0, a.setTimeout(function () { w() }, 0) }) } }, x = function () { for (var b = 0; b < s.length; b++) { var c = s[b], e = c.href, f = c.media, g = c.rel && "stylesheet" === c.rel.toLowerCase(); e && g && !o[e] && (c.styleSheet && c.styleSheet.rawCssText ? (v(c.styleSheet.rawCssText, e, f), o[e] = !0) : (!/^([a-zA-Z:]*\/\/)/.test(e) && !r || e.replace(RegExp.$1, "").split("/")[0] === a.location.host) && ("//" === e.substring(0, 2) && (e = a.location.protocol + e), d.push({ href: e, media: f }))) } w() }; x(), c.update = x, c.getEmValue = t, a.addEventListener ? a.addEventListener("resize", b, !1) : a.attachEvent && a.attachEvent("onresize", b) } }(this);
  };
</script>

<style type="text/css">
  .lightable-minimal {
    border-collapse: separate;
    border-spacing: 16px 1px;
    width: 100%;
    margin-bottom: 10px;
  }

  .lightable-minimal td {
    margin-left: 5px;
    margin-right: 5px;
  }

  .lightable-minimal th {
    margin-left: 5px;
    margin-right: 5px;
  }

  .lightable-minimal thead tr:last-child th {
    border-bottom: 2px solid #00000050;
    empty-cells: hide;
  }

  .lightable-minimal tbody tr:first-child td {
    padding-top: 0.5em;
  }

  .lightable-minimal.lightable-hover tbody tr:hover {
    background-color: #f5f5f5;
  }

  .lightable-minimal.lightable-striped tbody tr:nth-child(even) {
    background-color: #f5f5f5;
  }

  .lightable-classic {
    border-top: 0.16em solid #111111;
    border-bottom: 0.16em solid #111111;
    width: 100%;
    margin-bottom: 10px;
    margin: 10px 5px;
  }

  .lightable-classic tfoot tr td {
    border: 0;
  }

  .lightable-classic tfoot tr:first-child td {
    border-top: 0.14em solid #111111;
  }

  .lightable-classic caption {
    color: #222222;
  }

  .lightable-classic td {
    padding-left: 5px;
    padding-right: 5px;
    color: #222222;
  }

  .lightable-classic th {
    padding-left: 5px;
    padding-right: 5px;
    font-weight: normal;
    color: #222222;
  }

  .lightable-classic thead tr:last-child th {
    border-bottom: 0.10em solid #111111;
  }

  .lightable-classic.lightable-hover tbody tr:hover {
    background-color: #F9EEC1;
  }

  .lightable-classic.lightable-striped tbody tr:nth-child(even) {
    background-color: #f5f5f5;
  }

  .lightable-classic-2 {
    border-top: 3px double #111111;
    border-bottom: 3px double #111111;
    width: 100%;
    margin-bottom: 10px;
  }

  .lightable-classic-2 tfoot tr td {
    border: 0;
  }

  .lightable-classic-2 tfoot tr:first-child td {
    border-top: 3px double #111111;
  }

  .lightable-classic-2 caption {
    color: #222222;
  }

  .lightable-classic-2 td {
    padding-left: 5px;
    padding-right: 5px;
    color: #222222;
  }

  .lightable-classic-2 th {
    padding-left: 5px;
    padding-right: 5px;
    font-weight: normal;
    color: #222222;
  }

  .lightable-classic-2 tbody tr:last-child td {
    border-bottom: 3px double #111111;
  }

  .lightable-classic-2 thead tr:last-child th {
    border-bottom: 1px solid #111111;
  }

  .lightable-classic-2.lightable-hover tbody tr:hover {
    background-color: #F9EEC1;
  }

  .lightable-classic-2.lightable-striped tbody tr:nth-child(even) {
    background-color: #f5f5f5;
  }

  .lightable-material {
    min-width: 100%;
    white-space: nowrap;
    table-layout: fixed;
    font-family: Roboto, sans-serif;
    border: 1px solid #EEE;
    border-collapse: collapse;
    margin-bottom: 10px;
  }

  .lightable-material tfoot tr td {
    border: 0;
  }

  .lightable-material tfoot tr:first-child td {
    border-top: 1px solid #EEE;
  }

  .lightable-material th {
    height: 56px;
    padding-left: 16px;
    padding-right: 16px;
  }

  .lightable-material td {
    height: 52px;
    padding-left: 16px;
    padding-right: 16px;
    border-top: 1px solid #eeeeee;
  }

  .lightable-material.lightable-hover tbody tr:hover {
    background-color: #f5f5f5;
  }

  .lightable-material.lightable-striped tbody tr:nth-child(even) {
    background-color: #f5f5f5;
  }

  .lightable-material.lightable-striped tbody td {
    border: 0;
  }

  .lightable-material.lightable-striped thead tr:last-child th {
    border-bottom: 1px solid #ddd;
  }

  .lightable-material-dark {
    min-width: 100%;
    white-space: nowrap;
    table-layout: fixed;
    font-family: Roboto, sans-serif;
    border: 1px solid #FFFFFF12;
    border-collapse: collapse;
    margin-bottom: 10px;
    background-color: #363640;
  }

  .lightable-material-dark tfoot tr td {
    border: 0;
  }

  .lightable-material-dark tfoot tr:first-child td {
    border-top: 1px solid #FFFFFF12;
  }

  .lightable-material-dark th {
    height: 56px;
    padding-left: 16px;
    padding-right: 16px;
    color: #FFFFFF60;
  }

  .lightable-material-dark td {
    height: 52px;
    padding-left: 16px;
    padding-right: 16px;
    color: #FFFFFF;
    border-top: 1px solid #FFFFFF12;
  }

  .lightable-material-dark.lightable-hover tbody tr:hover {
    background-color: #FFFFFF12;
  }

  .lightable-material-dark.lightable-striped tbody tr:nth-child(even) {
    background-color: #FFFFFF12;
  }

  .lightable-material-dark.lightable-striped tbody td {
    border: 0;
  }

  .lightable-material-dark.lightable-striped thead tr:last-child th {
    border-bottom: 1px solid #FFFFFF12;
  }

  .lightable-paper {
    width: 100%;
    margin-bottom: 10px;
    color: #444;
  }

  .lightable-paper tfoot tr td {
    border: 0;
  }

  .lightable-paper tfoot tr:first-child td {
    border-top: 1px solid #00000020;
  }

  .lightable-paper thead tr:last-child th {
    color: #666;
    vertical-align: bottom;
    border-bottom: 1px solid #00000020;
    line-height: 1.15em;
    padding: 10px 5px;
  }

  .lightable-paper td {
    vertical-align: middle;
    border-bottom: 1px solid #00000010;
    line-height: 1.15em;
    padding: 7px 5px;
  }

  .lightable-paper.lightable-hover tbody tr:hover {
    background-color: #F9EEC1;
  }

  .lightable-paper.lightable-striped tbody tr:nth-child(even) {
    background-color: #00000008;
  }

  .lightable-paper.lightable-striped tbody td {
    border: 0;
  }

</style>
<script>(function () {
    // If window.HTMLWidgets is already defined, then use it; otherwise create a
    // new object. This allows preceding code to set options that affect the
    // initialization process (though none currently exist).
    window.HTMLWidgets = window.HTMLWidgets || {};

    // See if we're running in a viewer pane. If not, we're in a web browser.
    var viewerMode = window.HTMLWidgets.viewerMode =
      /\bviewer_pane=1\b/.test(window.location);

    // See if we're running in Shiny mode. If not, it's a static document.
    // Note that static widgets can appear in both Shiny and static modes, but
    // obviously, Shiny widgets can only appear in Shiny apps/documents.
    var shinyMode = window.HTMLWidgets.shinyMode =
      typeof (window.Shiny) !== "undefined" && !!window.Shiny.outputBindings;

    // We can't count on jQuery being available, so we implement our own
    // version if necessary.
    function querySelectorAll(scope, selector) {
      if (typeof (jQuery) !== "undefined" && scope instanceof jQuery) {
        return scope.find(selector);
      }
      if (scope.querySelectorAll) {
        return scope.querySelectorAll(selector);
      }
    }

    function asArray(value) {
      if (value === null)
        return [];
      if ($.isArray(value))
        return value;
      return [value];
    }

    // Implement jQuery's extend
    function extend(target /*, ... */) {
      if (arguments.length == 1) {
        return target;
      }
      for (var i = 1; i < arguments.length; i++) {
        var source = arguments[i];
        for (var prop in source) {
          if (source.hasOwnProperty(prop)) {
            target[prop] = source[prop];
          }
        }
      }
      return target;
    }

    // IE8 doesn't support Array.forEach.
    function forEach(values, callback, thisArg) {
      if (values.forEach) {
        values.forEach(callback, thisArg);
      } else {
        for (var i = 0; i < values.length; i++) {
          callback.call(thisArg, values[i], i, values);
        }
      }
    }

    // Replaces the specified method with the return value of funcSource.
    //
    // Note that funcSource should not BE the new method, it should be a function
    // that RETURNS the new method. funcSource receives a single argument that is
    // the overridden method, it can be called from the new method. The overridden
    // method can be called like a regular function, it has the target permanently
    // bound to it so "this" will work correctly.
    function overrideMethod(target, methodName, funcSource) {
      var superFunc = target[methodName] || function () { };
      var superFuncBound = function () {
        return superFunc.apply(target, arguments);
      };
      target[methodName] = funcSource(superFuncBound);
    }

    // Add a method to delegator that, when invoked, calls
    // delegatee.methodName. If there is no such method on
    // the delegatee, but there was one on delegator before
    // delegateMethod was called, then the original version
    // is invoked instead.
    // For example:
    //
    // var a = {
    //   method1: function() { console.log('a1'); }
    //   method2: function() { console.log('a2'); }
    // };
    // var b = {
    //   method1: function() { console.log('b1'); }
    // };
    // delegateMethod(a, b, "method1");
    // delegateMethod(a, b, "method2");
    // a.method1();
    // a.method2();
    //
    // The output would be "b1", "a2".
    function delegateMethod(delegator, delegatee, methodName) {
      var inherited = delegator[methodName];
      delegator[methodName] = function () {
        var target = delegatee;
        var method = delegatee[methodName];

        // The method doesn't exist on the delegatee. Instead,
        // call the method on the delegator, if it exists.
        if (!method) {
          target = delegator;
          method = inherited;
        }

        if (method) {
          return method.apply(target, arguments);
        }
      };
    }

    // Implement a vague facsimilie of jQuery's data method
    function elementData(el, name, value) {
      if (arguments.length == 2) {
        return el["htmlwidget_data_" + name];
      } else if (arguments.length == 3) {
        el["htmlwidget_data_" + name] = value;
        return el;
      } else {
        throw new Error("Wrong number of arguments for elementData: " +
          arguments.length);
      }
    }

    // http://stackoverflow.com/questions/3446170/escape-string-for-use-in-javascript-regex
    function escapeRegExp(str) {
      return str.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, "\\$&");
    }

    function hasClass(el, className) {
      var re = new RegExp("\\b" + escapeRegExp(className) + "\\b");
      return re.test(el.className);
    }

    // elements - array (or array-like object) of HTML elements
    // className - class name to test for
    // include - if true, only return elements with given className;
    //   if false, only return elements *without* given className
    function filterByClass(elements, className, include) {
      var results = [];
      for (var i = 0; i < elements.length; i++) {
        if (hasClass(elements[i], className) == include)
          results.push(elements[i]);
      }
      return results;
    }

    function on(obj, eventName, func) {
      if (obj.addEventListener) {
        obj.addEventListener(eventName, func, false);
      } else if (obj.attachEvent) {
        obj.attachEvent(eventName, func);
      }
    }

    function off(obj, eventName, func) {
      if (obj.removeEventListener)
        obj.removeEventListener(eventName, func, false);
      else if (obj.detachEvent) {
        obj.detachEvent(eventName, func);
      }
    }

    // Translate array of values to top/right/bottom/left, as usual with
    // the "padding" CSS property
    // https://developer.mozilla.org/en-US/docs/Web/CSS/padding
    function unpackPadding(value) {
      if (typeof (value) === "number")
        value = [value];
      if (value.length === 1) {
        return { top: value[0], right: value[0], bottom: value[0], left: value[0] };
      }
      if (value.length === 2) {
        return { top: value[0], right: value[1], bottom: value[0], left: value[1] };
      }
      if (value.length === 3) {
        return { top: value[0], right: value[1], bottom: value[2], left: value[1] };
      }
      if (value.length === 4) {
        return { top: value[0], right: value[1], bottom: value[2], left: value[3] };
      }
    }

    // Convert an unpacked padding object to a CSS value
    function paddingToCss(paddingObj) {
      return paddingObj.top + "px " + paddingObj.right + "px " + paddingObj.bottom + "px " + paddingObj.left + "px";
    }

    // Makes a number suitable for CSS
    function px(x) {
      if (typeof (x) === "number")
        return x + "px";
      else
        return x;
    }

    // Retrieves runtime widget sizing information for an element.
    // The return value is either null, or an object with fill, padding,
    // defaultWidth, defaultHeight fields.
    function sizingPolicy(el) {
      var sizingEl = document.querySelector("script[data-for='" + el.id + "'][type='application/htmlwidget-sizing']");
      if (!sizingEl)
        return null;
      var sp = JSON.parse(sizingEl.textContent || sizingEl.text || "{}");
      if (viewerMode) {
        return sp.viewer;
      } else {
        return sp.browser;
      }
    }

    // @param tasks Array of strings (or falsy value, in which case no-op).
    //   Each element must be a valid JavaScript expression that yields a
    //   function. Or, can be an array of objects with "code" and "data"
    //   properties; in this case, the "code" property should be a string
    //   of JS that's an expr that yields a function, and "data" should be
    //   an object that will be added as an additional argument when that
    //   function is called.
    // @param target The object that will be "this" for each function
    //   execution.
    // @param args Array of arguments to be passed to the functions. (The
    //   same arguments will be passed to all functions.)
    function evalAndRun(tasks, target, args) {
      if (tasks) {
        forEach(tasks, function (task) {
          var theseArgs = args;
          if (typeof (task) === "object") {
            theseArgs = theseArgs.concat([task.data]);
            task = task.code;
          }
          var taskFunc = tryEval(task);
          if (typeof (taskFunc) !== "function") {
            throw new Error("Task must be a function! Source:\n" + task);
          }
          taskFunc.apply(target, theseArgs);
        });
      }
    }

    // Attempt eval() both with and without enclosing in parentheses.
    // Note that enclosing coerces a function declaration into
    // an expression that eval() can parse
    // (otherwise, a SyntaxError is thrown)
    function tryEval(code) {
      var result = null;
      try {
        result = eval("(" + code + ")");
      } catch (error) {
        if (!(error instanceof SyntaxError)) {
          throw error;
        }
        try {
          result = eval(code);
        } catch (e) {
          if (e instanceof SyntaxError) {
            throw error;
          } else {
            throw e;
          }
        }
      }
      return result;
    }

    function initSizing(el) {
      var sizing = sizingPolicy(el);
      if (!sizing)
        return;

      var cel = document.getElementById("htmlwidget_container");
      if (!cel)
        return;

      if (typeof (sizing.padding) !== "undefined") {
        document.body.style.margin = "0";
        document.body.style.padding = paddingToCss(unpackPadding(sizing.padding));
      }

      if (sizing.fill) {
        document.body.style.overflow = "hidden";
        document.body.style.width = "100%";
        document.body.style.height = "100%";
        document.documentElement.style.width = "100%";
        document.documentElement.style.height = "100%";
        cel.style.position = "absolute";
        var pad = unpackPadding(sizing.padding);
        cel.style.top = pad.top + "px";
        cel.style.right = pad.right + "px";
        cel.style.bottom = pad.bottom + "px";
        cel.style.left = pad.left + "px";
        el.style.width = "100%";
        el.style.height = "100%";

        return {
          getWidth: function () { return cel.getBoundingClientRect().width; },
          getHeight: function () { return cel.getBoundingClientRect().height; }
        };

      } else {
        el.style.width = px(sizing.width);
        el.style.height = px(sizing.height);

        return {
          getWidth: function () { return cel.getBoundingClientRect().width; },
          getHeight: function () { return cel.getBoundingClientRect().height; }
        };
      }
    }

    // Default implementations for methods
    var defaults = {
      find: function (scope) {
        return querySelectorAll(scope, "." + this.name);
      },
      renderError: function (el, err) {
        var $el = $(el);

        this.clearError(el);

        // Add all these error classes, as Shiny does
        var errClass = "shiny-output-error";
        if (err.type !== null) {
          // use the classes of the error condition as CSS class names
          errClass = errClass + " " + $.map(asArray(err.type), function (type) {
            return errClass + "-" + type;
          }).join(" ");
        }
        errClass = errClass + " htmlwidgets-error";

        // Is el inline or block? If inline or inline-block, just display:none it
        // and add an inline error.
        var display = $el.css("display");
        $el.data("restore-display-mode", display);

        if (display === "inline" || display === "inline-block") {
          $el.hide();
          if (err.message !== "") {
            var errorSpan = $("<span>").addClass(errClass);
            errorSpan.text(err.message);
            $el.after(errorSpan);
          }
        } else if (display === "block") {
          // If block, add an error just after the el, set visibility:none on the
          // el, and position the error to be on top of the el.
          // Mark it with a unique ID and CSS class so we can remove it later.
          $el.css("visibility", "hidden");
          if (err.message !== "") {
            var errorDiv = $("<div>").addClass(errClass).css("position", "absolute")
              .css("top", el.offsetTop)
              .css("left", el.offsetLeft)
              // setting width can push out the page size, forcing otherwise
              // unnecessary scrollbars to appear and making it impossible for
              // the element to shrink; so use max-width instead
              .css("maxWidth", el.offsetWidth)
              .css("height", el.offsetHeight);
            errorDiv.text(err.message);
            $el.after(errorDiv);

            // Really dumb way to keep the size/position of the error in sync with
            // the parent element as the window is resized or whatever.
            var intId = setInterval(function () {
              if (!errorDiv[0].parentElement) {
                clearInterval(intId);
                return;
              }
              errorDiv
                .css("top", el.offsetTop)
                .css("left", el.offsetLeft)
                .css("maxWidth", el.offsetWidth)
                .css("height", el.offsetHeight);
            }, 500);
          }
        }
      },
      clearError: function (el) {
        var $el = $(el);
        var display = $el.data("restore-display-mode");
        $el.data("restore-display-mode", null);

        if (display === "inline" || display === "inline-block") {
          if (display)
            $el.css("display", display);
          $(el.nextSibling).filter(".htmlwidgets-error").remove();
        } else if (display === "block") {
          $el.css("visibility", "inherit");
          $(el.nextSibling).filter(".htmlwidgets-error").remove();
        }
      },
      sizing: {}
    };

    // Called by widget bindings to register a new type of widget. The definition
    // object can contain the following properties:
    // - name (required) - A string indicating the binding name, which will be
    //   used by default as the CSS classname to look for.
    // - initialize (optional) - A function(el) that will be called once per
    //   widget element; if a value is returned, it will be passed as the third
    //   value to renderValue.
    // - renderValue (required) - A function(el, data, initValue) that will be
    //   called with data. Static contexts will cause this to be called once per
    //   element; Shiny apps will cause this to be called multiple times per
    //   element, as the data changes.
    window.HTMLWidgets.widget = function (definition) {
      if (!definition.name) {
        throw new Error("Widget must have a name");
      }
      if (!definition.type) {
        throw new Error("Widget must have a type");
      }
      // Currently we only support output widgets
      if (definition.type !== "output") {
        throw new Error("Unrecognized widget type '" + definition.type + "'");
      }
      // TODO: Verify that .name is a valid CSS classname

      // Support new-style instance-bound definitions. Old-style class-bound
      // definitions have one widget "object" per widget per type/class of
      // widget; the renderValue and resize methods on such widget objects
      // take el and instance arguments, because the widget object can't
      // store them. New-style instance-bound definitions have one widget
      // object per widget instance; the definition that's passed in doesn't
      // provide renderValue or resize methods at all, just the single method
      //   factory(el, width, height)
      // which returns an object that has renderValue(x) and resize(w, h).
      // This enables a far more natural programming style for the widget
      // author, who can store per-instance state using either OO-style
      // instance fields or functional-style closure variables (I guess this
      // is in contrast to what can only be called C-style pseudo-OO which is
      // what we required before).
      if (definition.factory) {
        definition = createLegacyDefinitionAdapter(definition);
      }

      if (!definition.renderValue) {
        throw new Error("Widget must have a renderValue function");
      }

      // For static rendering (non-Shiny), use a simple widget registration
      // scheme. We also use this scheme for Shiny apps/documents that also
      // contain static widgets.
      window.HTMLWidgets.widgets = window.HTMLWidgets.widgets || [];
      // Merge defaults into the definition; don't mutate the original definition.
      var staticBinding = extend({}, defaults, definition);
      overrideMethod(staticBinding, "find", function (superfunc) {
        return function (scope) {
          var results = superfunc(scope);
          // Filter out Shiny outputs, we only want the static kind
          return filterByClass(results, "html-widget-output", false);
        };
      });
      window.HTMLWidgets.widgets.push(staticBinding);

      if (shinyMode) {
        // Shiny is running. Register the definition with an output binding.
        // The definition itself will not be the output binding, instead
        // we will make an output binding object that delegates to the
        // definition. This is because we foolishly used the same method
        // name (renderValue) for htmlwidgets definition and Shiny bindings
        // but they actually have quite different semantics (the Shiny
        // bindings receive data that includes lots of metadata that it
        // strips off before calling htmlwidgets renderValue). We can't
        // just ignore the difference because in some widgets it's helpful
        // to call this.renderValue() from inside of resize(), and if
        // we're not delegating, then that call will go to the Shiny
        // version instead of the htmlwidgets version.

        // Merge defaults with definition, without mutating either.
        var bindingDef = extend({}, defaults, definition);

        // This object will be our actual Shiny binding.
        var shinyBinding = new Shiny.OutputBinding();

        // With a few exceptions, we'll want to simply use the bindingDef's
        // version of methods if they are available, otherwise fall back to
        // Shiny's defaults. NOTE: If Shiny's output bindings gain additional
        // methods in the future, and we want them to be overrideable by
        // HTMLWidget binding definitions, then we'll need to add them to this
        // list.
        delegateMethod(shinyBinding, bindingDef, "getId");
        delegateMethod(shinyBinding, bindingDef, "onValueChange");
        delegateMethod(shinyBinding, bindingDef, "onValueError");
        delegateMethod(shinyBinding, bindingDef, "renderError");
        delegateMethod(shinyBinding, bindingDef, "clearError");
        delegateMethod(shinyBinding, bindingDef, "showProgress");

        // The find, renderValue, and resize are handled differently, because we
        // want to actually decorate the behavior of the bindingDef methods.

        shinyBinding.find = function (scope) {
          var results = bindingDef.find(scope);

          // Only return elements that are Shiny outputs, not static ones
          var dynamicResults = results.filter(".html-widget-output");

          // It's possible that whatever caused Shiny to think there might be
          // new dynamic outputs, also caused there to be new static outputs.
          // Since there might be lots of different htmlwidgets bindings, we
          // schedule execution for later--no need to staticRender multiple
          // times.
          if (results.length !== dynamicResults.length)
            scheduleStaticRender();

          return dynamicResults;
        };

        // Wrap renderValue to handle initialization, which unfortunately isn't
        // supported natively by Shiny at the time of this writing.

        shinyBinding.renderValue = function (el, data) {
          Shiny.renderDependencies(data.deps);
          // Resolve strings marked as javascript literals to objects
          if (!(data.evals instanceof Array)) data.evals = [data.evals];
          for (var i = 0; data.evals && i < data.evals.length; i++) {
            window.HTMLWidgets.evaluateStringMember(data.x, data.evals[i]);
          }
          if (!bindingDef.renderOnNullValue) {
            if (data.x === null) {
              el.style.visibility = "hidden";
              return;
            } else {
              el.style.visibility = "inherit";
            }
          }
          if (!elementData(el, "initialized")) {
            initSizing(el);

            elementData(el, "initialized", true);
            if (bindingDef.initialize) {
              var rect = el.getBoundingClientRect();
              var result = bindingDef.initialize(el, rect.width, rect.height);
              elementData(el, "init_result", result);
            }
          }
          bindingDef.renderValue(el, data.x, elementData(el, "init_result"));
          evalAndRun(data.jsHooks.render, elementData(el, "init_result"), [el, data.x]);
        };

        // Only override resize if bindingDef implements it
        if (bindingDef.resize) {
          shinyBinding.resize = function (el, width, height) {
            // Shiny can call resize before initialize/renderValue have been
            // called, which doesn't make sense for widgets.
            if (elementData(el, "initialized")) {
              bindingDef.resize(el, width, height, elementData(el, "init_result"));
            }
          };
        }

        Shiny.outputBindings.register(shinyBinding, bindingDef.name);
      }
    };

    var scheduleStaticRenderTimerId = null;
    function scheduleStaticRender() {
      if (!scheduleStaticRenderTimerId) {
        scheduleStaticRenderTimerId = setTimeout(function () {
          scheduleStaticRenderTimerId = null;
          window.HTMLWidgets.staticRender();
        }, 1);
      }
    }

    // Render static widgets after the document finishes loading
    // Statically render all elements that are of this widget's class
    window.HTMLWidgets.staticRender = function () {
      var bindings = window.HTMLWidgets.widgets || [];
      forEach(bindings, function (binding) {
        var matches = binding.find(document.documentElement);
        forEach(matches, function (el) {
          var sizeObj = initSizing(el, binding);

          var getSize = function (el) {
            if (sizeObj) {
              return { w: sizeObj.getWidth(), h: sizeObj.getHeight() }
            } else {
              var rect = el.getBoundingClientRect();
              return { w: rect.width, h: rect.height }
            }
          };

          if (hasClass(el, "html-widget-static-bound"))
            return;
          el.className = el.className + " html-widget-static-bound";

          var initResult;
          if (binding.initialize) {
            var size = getSize(el);
            initResult = binding.initialize(el, size.w, size.h);
            elementData(el, "init_result", initResult);
          }

          if (binding.resize) {
            var lastSize = getSize(el);
            var resizeHandler = function (e) {
              var size = getSize(el);
              if (size.w === 0 && size.h === 0)
                return;
              if (size.w === lastSize.w && size.h === lastSize.h)
                return;
              lastSize = size;
              binding.resize(el, size.w, size.h, initResult);
            };

            on(window, "resize", resizeHandler);

            // This is needed for cases where we're running in a Shiny
            // app, but the widget itself is not a Shiny output, but
            // rather a simple static widget. One example of this is
            // an rmarkdown document that has runtime:shiny and widget
            // that isn't in a render function. Shiny only knows to
            // call resize handlers for Shiny outputs, not for static
            // widgets, so we do it ourselves.
            if (window.jQuery) {
              window.jQuery(document).on(
                "shown.htmlwidgets shown.bs.tab.htmlwidgets shown.bs.collapse.htmlwidgets",
                resizeHandler
              );
              window.jQuery(document).on(
                "hidden.htmlwidgets hidden.bs.tab.htmlwidgets hidden.bs.collapse.htmlwidgets",
                resizeHandler
              );
            }

            // This is needed for the specific case of ioslides, which
            // flips slides between display:none and display:block.
            // Ideally we would not have to have ioslide-specific code
            // here, but rather have ioslides raise a generic event,
            // but the rmarkdown package just went to CRAN so the
            // window to getting that fixed may be long.
            if (window.addEventListener) {
              // It's OK to limit this to window.addEventListener
              // browsers because ioslides itself only supports
              // such browsers.
              on(document, "slideenter", resizeHandler);
              on(document, "slideleave", resizeHandler);
            }
          }

          var scriptData = document.querySelector("script[data-for='" + el.id + "'][type='application/json']");
          if (scriptData) {
            var data = JSON.parse(scriptData.textContent || scriptData.text);
            // Resolve strings marked as javascript literals to objects
            if (!(data.evals instanceof Array)) data.evals = [data.evals];
            for (var k = 0; data.evals && k < data.evals.length; k++) {
              window.HTMLWidgets.evaluateStringMember(data.x, data.evals[k]);
            }
            binding.renderValue(el, data.x, initResult);
            evalAndRun(data.jsHooks.render, initResult, [el, data.x]);
          }
        });
      });

      invokePostRenderHandlers();
    }


    function has_jQuery3() {
      if (!window.jQuery) {
        return false;
      }
      var $version = window.jQuery.fn.jquery;
      var $major_version = parseInt($version.split(".")[0]);
      return $major_version >= 3;
    }

    /*
    / Shiny 1.4 bumped jQuery from 1.x to 3.x which means jQuery's
    / on-ready handler (i.e., $(fn)) is now asyncronous (i.e., it now
    / really means $(setTimeout(fn)).
    / https://jquery.com/upgrade-guide/3.0/#breaking-change-document-ready-handlers-are-now-asynchronous
    /
    / Since Shiny uses $() to schedule initShiny, shiny>=1.4 calls initShiny
    / one tick later than it did before, which means staticRender() is
    / called renderValue() earlier than (advanced) widget authors might be expecting.
    / https://github.com/rstudio/shiny/issues/2630
    /
    / For a concrete example, leaflet has some methods (e.g., updateBounds)
    / which reference Shiny methods registered in initShiny (e.g., setInputValue).
    / Since leaflet is privy to this life-cycle, it knows to use setTimeout() to
    / delay execution of those methods (until Shiny methods are ready)
    / https://github.com/rstudio/leaflet/blob/18ec981/javascript/src/index.js#L266-L268
    /
    / Ideally widget authors wouldn't need to use this setTimeout() hack that
    / leaflet uses to call Shiny methods on a staticRender(). In the long run,
    / the logic initShiny should be broken up so that method registration happens
    / right away, but binding happens later.
    */
    function maybeStaticRenderLater() {
      if (shinyMode && has_jQuery3()) {
        window.jQuery(window.HTMLWidgets.staticRender);
      } else {
        window.HTMLWidgets.staticRender();
      }
    }

    if (document.addEventListener) {
      document.addEventListener("DOMContentLoaded", function () {
        document.removeEventListener("DOMContentLoaded", arguments.callee, false);
        maybeStaticRenderLater();
      }, false);
    } else if (document.attachEvent) {
      document.attachEvent("onreadystatechange", function () {
        if (document.readyState === "complete") {
          document.detachEvent("onreadystatechange", arguments.callee);
          maybeStaticRenderLater();
        }
      });
    }


    window.HTMLWidgets.getAttachmentUrl = function (depname, key) {
      // If no key, default to the first item
      if (typeof (key) === "undefined")
        key = 1;

      var link = document.getElementById(depname + "-" + key + "-attachment");
      if (!link) {
        throw new Error("Attachment " + depname + "/" + key + " not found in document");
      }
      return link.getAttribute("href");
    };

    window.HTMLWidgets.dataframeToD3 = function (df) {
      var names = [];
      var length;
      for (var name in df) {
        if (df.hasOwnProperty(name))
          names.push(name);
        if (typeof (df[name]) !== "object" || typeof (df[name].length) === "undefined") {
          throw new Error("All fields must be arrays");
        } else if (typeof (length) !== "undefined" && length !== df[name].length) {
          throw new Error("All fields must be arrays of the same length");
        }
        length = df[name].length;
      }
      var results = [];
      var item;
      for (var row = 0; row < length; row++) {
        item = {};
        for (var col = 0; col < names.length; col++) {
          item[names[col]] = df[names[col]][row];
        }
        results.push(item);
      }
      return results;
    };

    window.HTMLWidgets.transposeArray2D = function (array) {
      if (array.length === 0) return array;
      var newArray = array[0].map(function (col, i) {
        return array.map(function (row) {
          return row[i]
        })
      });
      return newArray;
    };
    // Split value at splitChar, but allow splitChar to be escaped
    // using escapeChar. Any other characters escaped by escapeChar
    // will be included as usual (including escapeChar itself).
    function splitWithEscape(value, splitChar, escapeChar) {
      var results = [];
      var escapeMode = false;
      var currentResult = "";
      for (var pos = 0; pos < value.length; pos++) {
        if (!escapeMode) {
          if (value[pos] === splitChar) {
            results.push(currentResult);
            currentResult = "";
          } else if (value[pos] === escapeChar) {
            escapeMode = true;
          } else {
            currentResult += value[pos];
          }
        } else {
          currentResult += value[pos];
          escapeMode = false;
        }
      }
      if (currentResult !== "") {
        results.push(currentResult);
      }
      return results;
    }
    // Function authored by Yihui/JJ Allaire
    window.HTMLWidgets.evaluateStringMember = function (o, member) {
      var parts = splitWithEscape(member, '.', '\\');
      for (var i = 0, l = parts.length; i < l; i++) {
        var part = parts[i];
        // part may be a character or 'numeric' member name
        if (o !== null && typeof o === "object" && part in o) {
          if (i == (l - 1)) { // if we are at the end of the line then evalulate
            if (typeof o[part] === "string")
              o[part] = tryEval(o[part]);
          } else { // otherwise continue to next embedded object
            o = o[part];
          }
        }
      }
    };

    // Retrieve the HTMLWidget instance (i.e. the return value of an
    // HTMLWidget binding's initialize() or factory() function)
    // associated with an element, or null if none.
    window.HTMLWidgets.getInstance = function (el) {
      return elementData(el, "init_result");
    };

    // Finds the first element in the scope that matches the selector,
    // and returns the HTMLWidget instance (i.e. the return value of
    // an HTMLWidget binding's initialize() or factory() function)
    // associated with that element, if any. If no element matches the
    // selector, or the first matching element has no HTMLWidget
    // instance associated with it, then null is returned.
    //
    // The scope argument is optional, and defaults to window.document.
    window.HTMLWidgets.find = function (scope, selector) {
      if (arguments.length == 1) {
        selector = scope;
        scope = document;
      }

      var el = scope.querySelector(selector);
      if (el === null) {
        return null;
      } else {
        return window.HTMLWidgets.getInstance(el);
      }
    };

    // Finds all elements in the scope that match the selector, and
    // returns the HTMLWidget instances (i.e. the return values of
    // an HTMLWidget binding's initialize() or factory() function)
    // associated with the elements, in an array. If elements that
    // match the selector don't have an associated HTMLWidget
    // instance, the returned array will contain nulls.
    //
    // The scope argument is optional, and defaults to window.document.
    window.HTMLWidgets.findAll = function (scope, selector) {
      if (arguments.length == 1) {
        selector = scope;
        scope = document;
      }

      var nodes = scope.querySelectorAll(selector);
      var results = [];
      for (var i = 0; i < nodes.length; i++) {
        results.push(window.HTMLWidgets.getInstance(nodes[i]));
      }
      return results;
    };

    var postRenderHandlers = [];
    function invokePostRenderHandlers() {
      while (postRenderHandlers.length) {
        var handler = postRenderHandlers.shift();
        if (handler) {
          handler();
        }
      }
    }

    // Register the given callback function to be invoked after the
    // next time static widgets are rendered.
    window.HTMLWidgets.addPostRenderHandler = function (callback) {
      postRenderHandlers.push(callback);
    };

    // Takes a new-style instance-bound definition, and returns an
    // old-style class-bound definition. This saves us from having
    // to rewrite all the logic in this file to accomodate both
    // types of definitions.
    function createLegacyDefinitionAdapter(defn) {
      var result = {
        name: defn.name,
        type: defn.type,
        initialize: function (el, width, height) {
          return defn.factory(el, width, height);
        },
        renderValue: function (el, x, instance) {
          return instance.renderValue(x);
        },
        resize: function (el, width, height, instance) {
          return instance.resize(width, height);
        }
      };

      if (defn.find)
        result.find = defn.find;
      if (defn.renderError)
        result.renderError = defn.renderError;
      if (defn.clearError)
        result.clearError = defn.clearError;

      return result;
    }
  })();
</script>
<style type="text/css">
  .dt-crosstalk-fade {
    opacity: 0.2;
  }

  html body div.DTS div.dataTables_scrollBody {
    background: none;
  }

  table.dataTable {
    display: table;
  }

  .html-fill-container>.html-fill-item.datatables {
    flex-basis: 400px;
  }

</style>
<script>(function () {

    // some helper functions: using a global object DTWidget so that it can be used
    // in JS() code, e.g. datatable(options = list(foo = JS('code'))); unlike R's
    // dynamic scoping, when 'code' is eval()'ed, JavaScript does not know objects
    // from the "parent frame", e.g. JS('DTWidget') will not work unless it was made
    // a global object
    var DTWidget = {};

    // 123456666.7890 -> 123,456,666.7890
    var markInterval = function (d, digits, interval, mark, decMark, precision) {
      x = precision ? d.toPrecision(digits) : d.toFixed(digits);
      if (!/^-?[\d.]+$/.test(x)) return x;
      var xv = x.split('.');
      if (xv.length > 2) return x;  // should have at most one decimal point
      xv[0] = xv[0].replace(new RegExp('\\B(?=(\\d{' + interval + '})+(?!\\d))', 'g'), mark);
      return xv.join(decMark);
    };

    DTWidget.formatCurrency = function (data, currency, digits, interval, mark, decMark, before, zeroPrint) {
      var d = parseFloat(data);
      if (isNaN(d)) return '';
      if (zeroPrint !== null && d === 0.0) return zeroPrint;
      var res = markInterval(d, digits, interval, mark, decMark);
      res = before ? (/^-/.test(res) ? '-' + currency + res.replace(/^-/, '') : currency + res) :
        res + currency;
      return res;
    };

    DTWidget.formatString = function (data, prefix, suffix) {
      var d = data;
      if (d === null) return '';
      return prefix + d + suffix;
    };

    DTWidget.formatPercentage = function (data, digits, interval, mark, decMark, zeroPrint) {
      var d = parseFloat(data);
      if (isNaN(d)) return '';
      if (zeroPrint !== null && d === 0.0) return zeroPrint;
      return markInterval(d * 100, digits, interval, mark, decMark) + '%';
    };

    DTWidget.formatRound = function (data, digits, interval, mark, decMark, zeroPrint) {
      var d = parseFloat(data);
      if (isNaN(d)) return '';
      if (zeroPrint !== null && d === 0.0) return zeroPrint;
      return markInterval(d, digits, interval, mark, decMark);
    };

    DTWidget.formatSignif = function (data, digits, interval, mark, decMark, zeroPrint) {
      var d = parseFloat(data);
      if (isNaN(d)) return '';
      if (zeroPrint !== null && d === 0.0) return zeroPrint;
      return markInterval(d, digits, interval, mark, decMark, true);
    };

    DTWidget.formatDate = function (data, method, params) {
      var d = data;
      if (d === null) return '';
      // (new Date('2015-10-28')).toDateString() may return 2015-10-27 because the
      // actual time created could be like 'Tue Oct 27 2015 19:00:00 GMT-0500 (CDT)',
      // i.e. the date-only string is treated as UTC time instead of local time
      if ((method === 'toDateString' || method === 'toLocaleDateString') && /^\d{4,}\D\d{2}\D\d{2}$/.test(d)) {
        d = d.split(/\D/);
        d = new Date(d[0], d[1] - 1, d[2]);
      } else {
        d = new Date(d);
      }
      return d[method].apply(d, params);
    };

    window.DTWidget = DTWidget;

    // A helper function to update the properties of existing filters
    var setFilterProps = function (td, props) {
      // Update enabled/disabled state
      var $input = $(td).find('input').first();
      var searchable = $input.data('searchable');
      $input.prop('disabled', !searchable || props.disabled);

      // Based on the filter type, set its new values
      var type = td.getAttribute('data-type');
      if (['factor', 'logical'].includes(type)) {
        // Reformat the new dropdown options for use with selectize
        var new_vals = props.params.options.map(function (item) {
          return { text: item, value: item };
        });

        // Find the selectize object
        var dropdown = $(td).find('.selectized').eq(0)[0].selectize;

        // Note the current values
        var old_vals = dropdown.getValue();

        // Remove the existing values
        dropdown.clearOptions();

        // Add the new options
        dropdown.addOption(new_vals);

        // Preserve the existing values
        dropdown.setValue(old_vals);

      } else if (['number', 'integer', 'date', 'time'].includes(type)) {
        // Apply internal scaling to new limits. Updating scale not yet implemented.
        var slider = $(td).find('.noUi-target').eq(0);
        var scale = Math.pow(10, Math.max(0, +slider.data('scale') || 0));
        var new_vals = [props.params.min * scale, props.params.max * scale];

        // Note what the new limits will be just for this filter
        var new_lims = new_vals.slice();

        // Determine the current values and limits
        var old_vals = slider.val().map(Number);
        var old_lims = slider.noUiSlider('options').range;
        old_lims = [old_lims.min, old_lims.max];

        // Preserve the current values if filters have been applied; otherwise, apply no filtering
        if (old_vals[0] != old_lims[0]) {
          new_vals[0] = Math.max(old_vals[0], new_vals[0]);
        }

        if (old_vals[1] != old_lims[1]) {
          new_vals[1] = Math.min(old_vals[1], new_vals[1]);
        }

        // Update the endpoints of the slider
        slider.noUiSlider({
          start: new_vals,
          range: { 'min': new_lims[0], 'max': new_lims[1] }
        }, true);
      }
    };

    var transposeArray2D = function (a) {
      return a.length === 0 ? a : HTMLWidgets.transposeArray2D(a);
    };

    var crosstalkPluginsInstalled = false;

    function maybeInstallCrosstalkPlugins() {
      if (crosstalkPluginsInstalled)
        return;
      crosstalkPluginsInstalled = true;

      $.fn.dataTable.ext.afnFiltering.push(
        function (oSettings, aData, iDataIndex) {
          var ctfilter = oSettings.nTable.ctfilter;
          if (ctfilter && !ctfilter[iDataIndex])
            return false;

          var ctselect = oSettings.nTable.ctselect;
          if (ctselect && !ctselect[iDataIndex])
            return false;

          return true;
        }
      );
    }

    HTMLWidgets.widget({
      name: "datatables",
      type: "output",
      renderOnNullValue: true,
      initialize: function (el, width, height) {
        // in order that the type=number inputs return a number
        $.valHooks.number = {
          get: function (el) {
            var value = parseFloat(el.value);
            return isNaN(value) ? "" : value;
          }
        };
        $(el).html('&nbsp;');
        return {
          data: null,
          ctfilterHandle: new crosstalk.FilterHandle(),
          ctfilterSubscription: null,
          ctselectHandle: new crosstalk.SelectionHandle(),
          ctselectSubscription: null
        };
      },
      renderValue: function (el, data, instance) {
        if (el.offsetWidth === 0 || el.offsetHeight === 0) {
          instance.data = data;
          return;
        }
        instance.data = null;
        var $el = $(el);
        $el.empty();

        if (data === null) {
          $el.append('&nbsp;');
          // clear previous Shiny inputs (if any)
          for (var i in instance.clearInputs) instance.clearInputs[i]();
          instance.clearInputs = {};
          return;
        }

        var crosstalkOptions = data.crosstalkOptions;
        if (!crosstalkOptions) crosstalkOptions = {
          'key': null, 'group': null
        };
        if (crosstalkOptions.group) {
          maybeInstallCrosstalkPlugins();
          instance.ctfilterHandle.setGroup(crosstalkOptions.group);
          instance.ctselectHandle.setGroup(crosstalkOptions.group);
        }

        // if we are in the viewer then we always want to fillContainer and
        // and autoHideNavigation (unless the user has explicitly set these)
        if (window.HTMLWidgets.viewerMode) {
          if (!data.hasOwnProperty("fillContainer"))
            data.fillContainer = true;
          if (!data.hasOwnProperty("autoHideNavigation"))
            data.autoHideNavigation = true;
        }

        // propagate fillContainer to instance (so we have it in resize)
        instance.fillContainer = data.fillContainer;

        var cells = data.data;

        if (cells instanceof Array) cells = transposeArray2D(cells);

        $el.append(data.container);
        var $table = $el.find('table');
        if (data.class) $table.addClass(data.class);
        if (data.caption) $table.prepend(data.caption);

        if (!data.selection) data.selection = {
          mode: 'none', selected: null, target: 'row', selectable: null
        };
        if (HTMLWidgets.shinyMode && data.selection.mode !== 'none' &&
          data.selection.target === 'row+column') {
          if ($table.children('tfoot').length === 0) {
            $table.append($('<tfoot>'));
            $table.find('thead tr').clone().appendTo($table.find('tfoot'));
          }
        }

        // column filters
        var filterRow;
        switch (data.filter) {
          case 'top':
            $table.children('thead').append(data.filterHTML);
            filterRow = $table.find('thead tr:last td');
            break;
          case 'bottom':
            if ($table.children('tfoot').length === 0) {
              $table.append($('<tfoot>'));
            }
            $table.children('tfoot').prepend(data.filterHTML);
            filterRow = $table.find('tfoot tr:first td');
            break;
        }

        var options = { searchDelay: 1000 };
        if (cells !== null) $.extend(options, {
          data: cells
        });

        // options for fillContainer
        var bootstrapActive = typeof ($.fn.popover) != 'undefined';
        if (instance.fillContainer) {

          // force scrollX/scrollY and turn off autoWidth
          options.scrollX = true;
          options.scrollY = "100px"; // can be any value, we'll adjust below

          // if we aren't paginating then move around the info/filter controls
          // to save space at the bottom and rephrase the info callback
          if (data.options.paging === false) {

            // we know how to do this cleanly for bootstrap, not so much
            // for other themes/layouts
            if (bootstrapActive) {
              options.dom = "<'row'<'col-sm-4'i><'col-sm-8'f>>" +
                "<'row'<'col-sm-12'tr>>";
            }

            options.fnInfoCallback = function (oSettings, iStart, iEnd,
              iMax, iTotal, sPre) {
              return Number(iTotal).toLocaleString() + " records";
            };
          }
        }

        // auto hide navigation if requested
        // Note, this only works on client-side processing mode as on server-side,
        // cells (data.data) is null; In addition, we require the pageLength option
        // being provided explicitly to enable this. Despite we may be able to deduce
        // the default value of pageLength, it may complicate things so we'd rather
        // put this responsiblity to users and warn them on the R side.
        if (data.autoHideNavigation === true && data.options.paging !== false) {
          // strip all nav if length >= cells
          if ((cells instanceof Array) && data.options.pageLength >= cells.length)
            options.dom = bootstrapActive ? "<'row'<'col-sm-12'tr>>" : "t";
          // alternatively lean things out for flexdashboard mobile portrait
          else if (bootstrapActive && window.FlexDashboard && window.FlexDashboard.isMobilePhone())
            options.dom = "<'row'<'col-sm-12'f>>" +
              "<'row'<'col-sm-12'tr>>" +
              "<'row'<'col-sm-12'p>>";
        }

        $.extend(true, options, data.options || {});

        var searchCols = options.searchCols;
        if (searchCols) {
          searchCols = searchCols.map(function (x) {
            return x === null ? '' : x.search;
          });
          // FIXME: this means I don't respect the escapeRegex setting
          delete options.searchCols;
        }

        // server-side processing?
        var server = options.serverSide === true;

        // use the dataSrc function to pre-process JSON data returned from R
        var DT_rows_all = [], DT_rows_current = [];
        if (server && HTMLWidgets.shinyMode && typeof options.ajax === 'object' &&
          /^session\/[\da-z]+\/dataobj/.test(options.ajax.url) && !options.ajax.dataSrc) {
          options.ajax.dataSrc = function (json) {
            DT_rows_all = $.makeArray(json.DT_rows_all);
            DT_rows_current = $.makeArray(json.DT_rows_current);
            var data = json.data;
            if (!colReorderEnabled()) return data;
            var table = $table.DataTable(), order = table.colReorder.order(), flag = true, i, j, row;
            for (i = 0; i < order.length; ++i) if (order[i] !== i) flag = false;
            if (flag) return data;
            for (i = 0; i < data.length; ++i) {
              row = data[i].slice();
              for (j = 0; j < order.length; ++j) data[i][j] = row[order[j]];
            }
            return data;
          };
        }

        var thiz = this;
        if (instance.fillContainer) $table.on('init.dt', function (e) {
          thiz.fillAvailableHeight(el, $(el).innerHeight());
        });
        // If the page contains serveral datatables and one of which enables colReorder,
        // the table.colReorder.order() function will exist but throws error when called.
        // So it seems like the only way to know if colReorder is enabled or not is to
        // check the options.
        var colReorderEnabled = function () { return "colReorder" in options; };
        var table = $table.DataTable(options);
        $el.data('datatable', table);

        // Unregister previous Crosstalk event subscriptions, if they exist
        if (instance.ctfilterSubscription) {
          instance.ctfilterHandle.off("change", instance.ctfilterSubscription);
          instance.ctfilterSubscription = null;
        }
        if (instance.ctselectSubscription) {
          instance.ctselectHandle.off("change", instance.ctselectSubscription);
          instance.ctselectSubscription = null;
        }

        if (!crosstalkOptions.group) {
          $table[0].ctfilter = null;
          $table[0].ctselect = null;
        } else {
          var key = crosstalkOptions.key;
          function keysToMatches(keys) {
            if (!keys) {
              return null;
            } else {
              var selectedKeys = {};
              for (var i = 0; i < keys.length; i++) {
                selectedKeys[keys[i]] = true;
              }
              var matches = {};
              for (var j = 0; j < key.length; j++) {
                if (selectedKeys[key[j]])
                  matches[j] = true;
              }
              return matches;
            }
          }

          function applyCrosstalkFilter(e) {
            $table[0].ctfilter = keysToMatches(e.value);
            table.draw();
          }
          instance.ctfilterSubscription = instance.ctfilterHandle.on("change", applyCrosstalkFilter);
          applyCrosstalkFilter({ value: instance.ctfilterHandle.filteredKeys });

          function applyCrosstalkSelection(e) {
            if (e.sender !== instance.ctselectHandle) {
              table
                .rows('.' + selClass, { search: 'applied' })
                .nodes()
                .to$()
                .removeClass(selClass);
              if (selectedRows)
                changeInput('rows_selected', selectedRows(), void 0, true);
            }

            if (e.sender !== instance.ctselectHandle && e.value && e.value.length) {
              var matches = keysToMatches(e.value);

              // persistent selection with plotly (& leaflet)
              var ctOpts = crosstalk.var("plotlyCrosstalkOpts").get() || {};
              if (ctOpts.persistent === true) {
                var matches = $.extend(matches, $table[0].ctselect);
              }

              $table[0].ctselect = matches;
              table.draw();
            } else {
              if ($table[0].ctselect) {
                $table[0].ctselect = null;
                table.draw();
              }
            }
          }
          instance.ctselectSubscription = instance.ctselectHandle.on("change", applyCrosstalkSelection);
          // TODO: This next line doesn't seem to work when renderDataTable is used
          applyCrosstalkSelection({ value: instance.ctselectHandle.value });
        }

        var inArray = function (val, array) {
          return $.inArray(val, $.makeArray(array)) > -1;
        };

        // search the i-th column
        var searchColumn = function (i, value) {
          var regex = false, ci = true;
          if (options.search) {
            regex = options.search.regex,
              ci = options.search.caseInsensitive !== false;
          }
          return table.column(i).search(value, regex, !regex, ci);
        };

        if (data.filter !== 'none') {

          filterRow.each(function (i, td) {

            var $td = $(td), type = $td.data('type'), filter;
            var $input = $td.children('div').first().children('input');
            var disabled = $input.prop('disabled');
            var searchable = table.settings()[0].aoColumns[i].bSearchable;
            $input.prop('disabled', !searchable || disabled);
            $input.data('searchable', searchable); // for updating later
            $input.on('input blur', function () {
              $input.next('span').toggle(Boolean($input.val()));
            });
            // Bootstrap sets pointer-events to none and we won't be able to click
            // the clear button
            $input.next('span').css('pointer-events', 'auto').hide().click(function () {
              $(this).hide().prev('input').val('').trigger('input').focus();
            });
            var searchCol;  // search string for this column
            if (searchCols && searchCols[i]) {
              searchCol = searchCols[i];
              $input.val(searchCol).trigger('input');
            }
            var $x = $td.children('div').last();

            // remove the overflow: hidden attribute of the scrollHead
            // (otherwise the scrolling table body obscures the filters)
            // The workaround and the discussion from
            // https://github.com/rstudio/DT/issues/554#issuecomment-518007347
            // Otherwise the filter selection will not be anchored to the values
            // when the columns number is many and scrollX is enabled.
            var scrollHead = $(el).find('.dataTables_scrollHead,.dataTables_scrollFoot');
            var cssOverflowHead = scrollHead.css('overflow');
            var scrollBody = $(el).find('.dataTables_scrollBody');
            var cssOverflowBody = scrollBody.css('overflow');
            var scrollTable = $(el).find('.dataTables_scroll');
            var cssOverflowTable = scrollTable.css('overflow');
            if (cssOverflowHead === 'hidden') {
              $x.on('show hide', function (e) {
                if (e.type === 'show') {
                  scrollHead.css('overflow', 'visible');
                  scrollBody.css('overflow', 'visible');
                  scrollTable.css('overflow-x', 'scroll');
                } else {
                  scrollHead.css('overflow', cssOverflowHead);
                  scrollBody.css('overflow', cssOverflowBody);
                  scrollTable.css('overflow-x', cssOverflowTable);
                }
              });
              $x.css('z-index', 25);
            }

            if (inArray(type, ['factor', 'logical'])) {
              $input.on({
                click: function () {
                  $input.parent().hide(); $x.show().trigger('show'); filter[0].selectize.focus();
                },
                input: function () {
                  if ($input.val() === '') filter[0].selectize.setValue([]);
                }
              });
              var $input2 = $x.children('select');
              filter = $input2.selectize({
                options: $input2.data('options').map(function (v, i) {
                  return ({ text: v, value: v });
                }),
                plugins: ['remove_button'],
                hideSelected: true,
                onChange: function (value) {
                  if (value === null) value = []; // compatibility with jQuery 3.0
                  $input.val(value.length ? JSON.stringify(value) : '');
                  if (value.length) $input.trigger('input');
                  $input.attr('title', $input.val());
                  if (server) {
                    table.column(i).search(value.length ? JSON.stringify(value) : '').draw();
                    return;
                  }
                  // turn off filter if nothing selected
                  $td.data('filter', value.length > 0);
                  table.draw();  // redraw table, and filters will be applied
                }
              });
              if (searchCol) filter[0].selectize.setValue(JSON.parse(searchCol));
              filter[0].selectize.on('blur', function () {
                $x.hide().trigger('hide'); $input.parent().show(); $input.trigger('blur');
              });
              filter.next('div').css('margin-bottom', 'auto');
            } else if (type === 'character') {
              var fun = function () {
                searchColumn(i, $input.val()).draw();
              };
              if (server) {
                fun = $.fn.dataTable.util.throttle(fun, options.searchDelay);
              }
              $input.on('input', fun);
            } else if (inArray(type, ['number', 'integer', 'date', 'time'])) {
              var $x0 = $x;
              $x = $x0.children('div').first();
              $x0.css({
                'background-color': '#fff',
                'border': '1px #ddd solid',
                'border-radius': '4px',
                'padding': data.vertical ? '35px 20px' : '20px 20px 10px 20px'
              });
              var $spans = $x0.children('span').css({
                'margin-top': data.vertical ? '0' : '10px',
                'white-space': 'nowrap'
              });
              var $span1 = $spans.first(), $span2 = $spans.last();
              var r1 = +$x.data('min'), r2 = +$x.data('max');
              // when the numbers are too small or have many decimal places, the
              // slider may have numeric precision problems (#150)
              var scale = Math.pow(10, Math.max(0, +$x.data('scale') || 0));
              r1 = Math.round(r1 * scale); r2 = Math.round(r2 * scale);
              var scaleBack = function (x, scale) {
                if (scale === 1) return x;
                var d = Math.round(Math.log(scale) / Math.log(10));
                // to avoid problems like 3.423/100 -> 0.034230000000000003
                return (x / scale).toFixed(d);
              };
              var slider_min = function () {
                return filter.noUiSlider('options').range.min;
              };
              var slider_max = function () {
                return filter.noUiSlider('options').range.max;
              };
              $input.on({
                focus: function () {
                  $x0.show().trigger('show');
                  // first, make sure the slider div leaves at least 20px between
                  // the two (slider value) span's
                  $x0.width(Math.max(160, $span1.outerWidth() + $span2.outerWidth() + 20));
                  // then, if the input is really wide or slider is vertical,
                  // make the slider the same width as the input
                  if ($x0.outerWidth() < $input.outerWidth() || data.vertical) {
                    $x0.outerWidth($input.outerWidth());
                  }
                  // make sure the slider div does not reach beyond the right margin
                  if ($(window).width() < $x0.offset().left + $x0.width()) {
                    $x0.offset({
                      'left': $input.offset().left + $input.outerWidth() - $x0.outerWidth()
                    });
                  }
                },
                blur: function () {
                  $x0.hide().trigger('hide');
                },
                input: function () {
                  if ($input.val() === '') filter.val([slider_min(), slider_max()]);
                },
                change: function () {
                  var v = $input.val().replace(/\s/g, '');
                  if (v === '') return;
                  v = v.split('...');
                  if (v.length !== 2) {
                    $input.parent().addClass('has-error');
                    return;
                  }
                  if (v[0] === '') v[0] = slider_min();
                  if (v[1] === '') v[1] = slider_max();
                  $input.parent().removeClass('has-error');
                  // treat date as UTC time at midnight
                  var strTime = function (x) {
                    var s = type === 'date' ? 'T00:00:00Z' : '';
                    var t = new Date(x + s).getTime();
                    // add 10 minutes to date since it does not hurt the date, and
                    // it helps avoid the tricky floating point arithmetic problems,
                    // e.g. sometimes the date may be a few milliseconds earlier
                    // than the midnight due to precision problems in noUiSlider
                    return type === 'date' ? t + 3600000 : t;
                  };
                  if (inArray(type, ['date', 'time'])) {
                    v[0] = strTime(v[0]);
                    v[1] = strTime(v[1]);
                  }
                  if (v[0] != slider_min()) v[0] *= scale;
                  if (v[1] != slider_max()) v[1] *= scale;
                  filter.val(v);
                }
              });
              var formatDate = function (d, isoFmt) {
                d = scaleBack(d, scale);
                if (type === 'number') return d;
                if (type === 'integer') return parseInt(d);
                var x = new Date(+d);
                var fmt = ('filterDateFmt' in data) ? data.filterDateFmt[i] : undefined;
                if (fmt !== undefined && isoFmt === false) return x[fmt.method].apply(x, fmt.params);
                if (type === 'date') {
                  var pad0 = function (x) {
                    return ('0' + x).substr(-2, 2);
                  };
                  return x.getUTCFullYear() + '-' + pad0(1 + x.getUTCMonth())
                    + '-' + pad0(x.getUTCDate());
                } else {
                  return x.toISOString();
                }
              };
              var opts = type === 'date' ? { step: 60 * 60 * 1000 } :
                type === 'integer' ? { step: 1 } : {};

              opts.orientation = data.vertical ? 'vertical' : 'horizontal';
              opts.direction = data.vertical ? 'rtl' : 'ltr';

              filter = $x.noUiSlider($.extend({
                start: [r1, r2],
                range: { min: r1, max: r2 },
                connect: true
              }, opts));
              if (scale > 1) (function () {
                var t1 = r1, t2 = r2;
                var val = filter.val();
                while (val[0] > r1 || val[1] < r2) {
                  if (val[0] > r1) {
                    t1 -= val[0] - r1;
                  }
                  if (val[1] < r2) {
                    t2 += r2 - val[1];
                  }
                  filter = $x.noUiSlider($.extend({
                    start: [t1, t2],
                    range: { min: t1, max: t2 },
                    connect: true
                  }, opts), true);
                  val = filter.val();
                }
                r1 = t1; r2 = t2;
              })();
              var updateSliderText = function (v1, v2) {
                $span1.text(formatDate(v1, false)); $span2.text(formatDate(v2, false));
              };
              updateSliderText(r1, r2);
              var updateSlider = function (e) {
                var val = filter.val();
                // turn off filter if in full range
                $td.data('filter', val[0] > slider_min() || val[1] < slider_max());
                var v1 = formatDate(val[0]), v2 = formatDate(val[1]), ival;
                if ($td.data('filter')) {
                  ival = v1 + ' ... ' + v2;
                  $input.attr('title', ival).val(ival).trigger('input');
                } else {
                  $input.attr('title', '').val('');
                }
                updateSliderText(val[0], val[1]);
                if (e.type === 'slide') return;  // no searching when sliding only
                if (server) {
                  table.column(i).search($td.data('filter') ? ival : '').draw();
                  return;
                }
                table.draw();
              };
              filter.on({
                set: updateSlider,
                slide: updateSlider
              });
            }

            // server-side processing will be handled by R (or whatever server
            // language you use); the following code is only needed for client-side
            // processing
            if (server) {
              // if a search string has been pre-set, search now
              if (searchCol) searchColumn(i, searchCol).draw();
              return;
            }

            var customFilter = function (settings, data, dataIndex) {
              // there is no way to attach a search function to a specific table,
              // and we need to make sure a global search function is not applied to
              // all tables (i.e. a range filter in a previous table should not be
              // applied to the current table); we use the settings object to
              // determine if we want to perform searching on the current table,
              // since settings.sTableId will be different to different tables
              if (table.settings()[0] !== settings) return true;
              // no filter on this column or no need to filter this column
              if (typeof filter === 'undefined' || !$td.data('filter')) return true;

              var r = filter.val(), v, r0, r1;
              var i_data = function (i) {
                if (!colReorderEnabled()) return i;
                var order = table.colReorder.order(), k;
                for (k = 0; k < order.length; ++k) if (order[k] === i) return k;
                return i; // in theory it will never be here...
              }
              v = data[i_data(i)];
              if (type === 'number' || type === 'integer') {
                v = parseFloat(v);
                // how to handle NaN? currently exclude these rows
                if (isNaN(v)) return (false);
                r0 = parseFloat(scaleBack(r[0], scale))
                r1 = parseFloat(scaleBack(r[1], scale));
                if (v >= r0 && v <= r1) return true;
              } else if (type === 'date' || type === 'time') {
                v = new Date(v);
                r0 = new Date(r[0] / scale); r1 = new Date(r[1] / scale);
                if (v >= r0 && v <= r1) return true;
              } else if (type === 'factor') {
                if (r.length === 0 || inArray(v, r)) return true;
              } else if (type === 'logical') {
                if (r.length === 0) return true;
                if (inArray(v === '' ? 'na' : v, r)) return true;
              }
              return false;
            };

            $.fn.dataTable.ext.search.push(customFilter);

            // search for the preset search strings if it is non-empty
            if (searchCol) {
              if (inArray(type, ['factor', 'logical'])) {
                filter[0].selectize.setValue(JSON.parse(searchCol));
              } else if (type === 'character') {
                $input.trigger('input');
              } else if (inArray(type, ['number', 'integer', 'date', 'time'])) {
                $input.trigger('change');
              }
            }

          });

        }

        // highlight search keywords
        var highlight = function () {
          var body = $(table.table().body());
          // removing the old highlighting first
          body.unhighlight();

          // don't highlight the "not found" row, so we get the rows using the api
          if (table.rows({ filter: 'applied' }).data().length === 0) return;
          // highlight global search keywords
          body.highlight($.trim(table.search()).split(/\s+/));
          // then highlight keywords from individual column filters
          if (filterRow) filterRow.each(function (i, td) {
            var $td = $(td), type = $td.data('type');
            if (type !== 'character') return;
            var $input = $td.children('div').first().children('input');
            var column = table.column(i).nodes().to$(),
              val = $.trim($input.val());
            if (type !== 'character' || val === '') return;
            column.highlight(val.split(/\s+/));
          });
        };

        if (options.searchHighlight) {
          table
            .on('draw.dt.dth column-visibility.dt.dth column-reorder.dt.dth', highlight)
            .on('destroy', function () {
              // remove event handler
              table.off('draw.dt.dth column-visibility.dt.dth column-reorder.dt.dth');
            });

          // Set the option for escaping regex characters in our search string.  This will be used
          // for all future matching.
          jQuery.fn.highlight.options.escapeRegex = (!options.search || !options.search.regex);

          // initial highlight for state saved conditions and initial states
          highlight();
        }

        // run the callback function on the table instance
        if (typeof data.callback === 'function') data.callback(table);

        // double click to edit the cell, row, column, or all cells
        if (data.editable) table.on('dblclick.dt', 'tbody td', function (e) {
          // only bring up the editor when the cell itself is dbclicked, and ignore
          // other dbclick events bubbled up (e.g. from the <input>)
          if (e.target !== this) return;
          var target = [], immediate = false;
          switch (data.editable.target) {
            case 'cell':
              target = [this];
              immediate = true;  // edit will take effect immediately
              break;
            case 'row':
              target = table.cells(table.cell(this).index().row, '*').nodes();
              break;
            case 'column':
              target = table.cells('*', table.cell(this).index().column).nodes();
              break;
            case 'all':
              target = table.cells().nodes();
              break;
            default:
              throw 'The editable parameter must be "cell", "row", "column", or "all"';
          }
          var disableCols = data.editable.disable ? data.editable.disable.columns : null;
          var numericCols = data.editable.numeric;
          var areaCols = data.editable.area;
          for (var i = 0; i < target.length; i++) {
            (function (cell, current) {
              var $cell = $(cell), html = $cell.html();
              var _cell = table.cell(cell), value = _cell.data(), index = _cell.index().column;
              var $input;
              if (inArray(index, numericCols)) {
                $input = $('<input type="number">');
              } else if (inArray(index, areaCols)) {
                $input = $('<textarea></textarea>');
              } else {
                $input = $('<input type="text">');
              }
              if (!immediate) {
                $cell.data('input', $input).data('html', html);
                $input.attr('title', 'Hit Ctrl+Enter to finish editing, or Esc to cancel');
              }
              $input.val(value);
              if (inArray(index, disableCols)) {
                $input.attr('readonly', '').css('filter', 'invert(25%)');
              }
              $cell.empty().append($input);
              if (cell === current) $input.focus();
              $input.css('width', '100%');

              if (immediate) $input.on('blur', function (e) {
                var valueNew = $input.val();
                if (valueNew !== value) {
                  _cell.data(valueNew);
                  if (HTMLWidgets.shinyMode) {
                    changeInput('cell_edit', [cellInfo(cell)], 'DT.cellInfo', null, { priority: 'event' });
                  }
                  // for server-side processing, users have to call replaceData() to update the table
                  if (!server) table.draw(false);
                } else {
                  $cell.html(html);
                }
              }).on('keyup', function (e) {
                // hit Escape to cancel editing
                if (e.keyCode === 27) $input.trigger('blur');
              });

              // bulk edit (row, column, or all)
              if (!immediate) $input.on('keyup', function (e) {
                var removeInput = function ($cell, restore) {
                  $cell.data('input').remove();
                  if (restore) $cell.html($cell.data('html'));
                }
                if (e.keyCode === 27) {
                  for (var i = 0; i < target.length; i++) {
                    removeInput($(target[i]), true);
                  }
                } else if (e.keyCode === 13 && e.ctrlKey) {
                  // Ctrl + Enter
                  var cell, $cell, _cell, cellData = [];
                  for (var i = 0; i < target.length; i++) {
                    cell = target[i]; $cell = $(cell); _cell = table.cell(cell);
                    _cell.data($cell.data('input').val());
                    HTMLWidgets.shinyMode && cellData.push(cellInfo(cell));
                    removeInput($cell, false);
                  }
                  if (HTMLWidgets.shinyMode) {
                    changeInput('cell_edit', cellData, 'DT.cellInfo', null, { priority: "event" });
                  }
                  if (!server) table.draw(false);
                }
              });
            })(target[i], this);
          }
        });

        // interaction with shiny
        if (!HTMLWidgets.shinyMode && !crosstalkOptions.group) return;

        var methods = {};
        var shinyData = {};

        methods.updateCaption = function (caption) {
          if (!caption) return;
          $table.children('caption').replaceWith(caption);
        }

        // register clear functions to remove input values when the table is removed
        instance.clearInputs = {};

        var changeInput = function (id, value, type, noCrosstalk, opts) {
          var event = id;
          id = el.id + '_' + id;
          if (type) id = id + ':' + type;
          // do not update if the new value is the same as old value
          if (event !== 'cell_edit' && !/_clicked$/.test(event) && shinyData.hasOwnProperty(id) && shinyData[id] === JSON.stringify(value))
            return;
          shinyData[id] = JSON.stringify(value);
          if (HTMLWidgets.shinyMode && Shiny.setInputValue) {
            Shiny.setInputValue(id, value, opts);
            if (!instance.clearInputs[id]) instance.clearInputs[id] = function () {
              Shiny.setInputValue(id, null);
            }
          }

          // HACK
          if (event === "rows_selected" && !noCrosstalk) {
            if (crosstalkOptions.group) {
              var keys = crosstalkOptions.key;
              var selectedKeys = null;
              if (value) {
                selectedKeys = [];
                for (var i = 0; i < value.length; i++) {
                  // The value array's contents use 1-based row numbers, so we must
                  // convert to 0-based before indexing into the keys array.
                  selectedKeys.push(keys[value[i] - 1]);
                }
              }
              instance.ctselectHandle.set(selectedKeys);
            }
          }
        };

        var addOne = function (x) {
          return x.map(function (i) { return 1 + i; });
        };

        var unique = function (x) {
          var ux = [];
          $.each(x, function (i, el) {
            if ($.inArray(el, ux) === -1) ux.push(el);
          });
          return ux;
        }

        // change the row index of a cell
        var tweakCellIndex = function (cell) {
          var info = cell.index();
          // some cell may not be valid. e.g, #759
          // when using the RowGroup extension, datatables will
          // generate the row label and the cells are not part of
          // the data thus contain no row/col info
          if (info === undefined)
            return { row: null, col: null };
          if (server) {
            info.row = DT_rows_current[info.row];
          } else {
            info.row += 1;
          }
          return { row: info.row, col: info.column };
        }

        var cleanSelectedValues = function () {
          changeInput('rows_selected', []);
          changeInput('columns_selected', []);
          changeInput('cells_selected', transposeArray2D([]), 'shiny.matrix');
        }
        // #828 we should clean the selection on the server-side when the table reloads
        cleanSelectedValues();

        // a flag to indicates if select extension is initialized or not
        var flagSelectExt = table.settings()[0]._select !== undefined;
        // the Select extension should only be used in the client mode and
        // when the selection.mode is set to none
        if (data.selection.mode === 'none' && !server && flagSelectExt) {
          var updateRowsSelected = function () {
            var rows = table.rows({ selected: true });
            var selected = [];
            $.each(rows.indexes().toArray(), function (i, v) {
              selected.push(v + 1);
            });
            changeInput('rows_selected', selected);
          }
          var updateColsSelected = function () {
            var columns = table.columns({ selected: true });
            changeInput('columns_selected', columns.indexes().toArray());
          }
          var updateCellsSelected = function () {
            var cells = table.cells({ selected: true });
            var selected = [];
            cells.every(function () {
              var row = this.index().row;
              var col = this.index().column;
              selected = selected.concat([[row + 1, col]]);
            });
            changeInput('cells_selected', transposeArray2D(selected), 'shiny.matrix');
          }
          table.on('select deselect', function (e, dt, type, indexes) {
            updateRowsSelected();
            updateColsSelected();
            updateCellsSelected();
          })
        }

        var selMode = data.selection.mode, selTarget = data.selection.target;
        var selDisable = data.selection.selectable === false;
        if (inArray(selMode, ['single', 'multiple'])) {
          var selClass = inArray(data.style, ['bootstrap', 'bootstrap4']) ? 'active' : 'selected';
          // selected1: row indices; selected2: column indices
          var initSel = function (x) {
            if (x === null || typeof x === 'boolean' || selTarget === 'cell') {
              return { rows: [], cols: [] };
            } else if (selTarget === 'row') {
              return { rows: $.makeArray(x), cols: [] };
            } else if (selTarget === 'column') {
              return { rows: [], cols: $.makeArray(x) };
            } else if (selTarget === 'row+column') {
              return { rows: $.makeArray(x.rows), cols: $.makeArray(x.cols) };
            }
          }
          var selected = data.selection.selected;
          var selected1 = initSel(selected).rows, selected2 = initSel(selected).cols;
          // selectable should contain either all positive or all non-positive values, not both
          // positive values indicate "selectable" while non-positive values means "nonselectable"
          // the assertion is performed on R side. (only column indicides could be zero which indicates
          // the row name)
          var selectable = data.selection.selectable;
          var selectable1 = initSel(selectable).rows, selectable2 = initSel(selectable).cols;

          // After users reorder the rows or filter the table, we cannot use the table index
          // directly. Instead, we need this function to find out the rows between the two clicks.
          // If user filter the table again between the start click and the end click, the behavior
          // would be undefined, but it should not be a problem.
          var shiftSelRowsIndex = function (start, end) {
            var indexes = server ? DT_rows_all : table.rows({ search: 'applied' }).indexes().toArray();
            start = indexes.indexOf(start); end = indexes.indexOf(end);
            // if start is larger than end, we need to swap
            if (start > end) {
              var tmp = end; end = start; start = tmp;
            }
            return indexes.slice(start, end + 1);
          }

          var serverRowIndex = function (clientRowIndex) {
            return server ? DT_rows_current[clientRowIndex] : clientRowIndex + 1;
          }

          // row, column, or cell selection
          var lastClickedRow;
          if (inArray(selTarget, ['row', 'row+column'])) {
            // Get the current selected rows. It will also
            // update the selected1's value based on the current row selection state
            // Note we can't put this function inside selectRows() directly,
            // the reason is method.selectRows() will override selected1's value but this
            // function will add rows to selected1 (keep the existing selection), which is
            // inconsistent with column and cell selection.
            var selectedRows = function () {
              var rows = table.rows('.' + selClass);
              var idx = rows.indexes().toArray();
              if (!server) {
                selected1 = addOne(idx);
                return selected1;
              }
              idx = idx.map(function (i) {
                return DT_rows_current[i];
              });
              selected1 = selMode === 'multiple' ? unique(selected1.concat(idx)) : idx;
              return selected1;
            }
            // Change selected1's value based on selectable1, then refresh the row state
            var onlyKeepSelectableRows = function () {
              if (selDisable) { // users can't select; useful when only want backend select
                selected1 = [];
                return;
              }
              if (selectable1.length === 0) return;
              var nonselectable = selectable1[0] <= 0;
              if (nonselectable) {
                // should make selectable1 positive
                selected1 = $(selected1).not(selectable1.map(function (i) { return -i; })).get();
              } else {
                selected1 = $(selected1).filter(selectable1).get();
              }
            }
            // Change selected1's value based on selectable1, then
            // refresh the row selection state according to values in selected1
            var selectRows = function (ignoreSelectable) {
              if (!ignoreSelectable) onlyKeepSelectableRows();
              table.$('tr.' + selClass).removeClass(selClass);
              if (selected1.length === 0) return;
              if (server) {
                table.rows({ page: 'current' }).every(function () {
                  if (inArray(DT_rows_current[this.index()], selected1)) {
                    $(this.node()).addClass(selClass);
                  }
                });
              } else {
                var selected0 = selected1.map(function (i) { return i - 1; });
                $(table.rows(selected0).nodes()).addClass(selClass);
              }
            }
            table.on('mousedown.dt', 'tbody tr', function (e) {
              var $this = $(this), thisRow = table.row(this);
              if (selMode === 'multiple') {
                if (e.shiftKey && lastClickedRow !== undefined) {
                  // select or de-select depends on the last clicked row's status
                  var flagSel = !$this.hasClass(selClass);
                  var crtClickedRow = serverRowIndex(thisRow.index());
                  if (server) {
                    var rowsIndex = shiftSelRowsIndex(lastClickedRow, crtClickedRow);
                    // update current page's selClass
                    rowsIndex.map(function (i) {
                      var rowIndex = DT_rows_current.indexOf(i);
                      if (rowIndex >= 0) {
                        var row = table.row(rowIndex).nodes().to$();
                        var flagRowSel = !row.hasClass(selClass);
                        if (flagSel === flagRowSel) row.toggleClass(selClass);
                      }
                    });
                    // update selected1
                    if (flagSel) {
                      selected1 = unique(selected1.concat(rowsIndex));
                    } else {
                      selected1 = selected1.filter(function (index) {
                        return !inArray(index, rowsIndex);
                      });
                    }
                  } else {
                    // js starts from 0
                    shiftSelRowsIndex(lastClickedRow - 1, crtClickedRow - 1).map(function (value) {
                      var row = table.row(value).nodes().to$();
                      var flagRowSel = !row.hasClass(selClass);
                      if (flagSel === flagRowSel) row.toggleClass(selClass);
                    });
                  }
                  e.preventDefault();
                } else {
                  $this.toggleClass(selClass);
                }
              } else {
                if ($this.hasClass(selClass)) {
                  $this.removeClass(selClass);
                } else {
                  table.$('tr.' + selClass).removeClass(selClass);
                  $this.addClass(selClass);
                }
              }
              if (server && !$this.hasClass(selClass)) {
                var id = DT_rows_current[thisRow.index()];
                // remove id from selected1 since its class .selected has been removed
                if (inArray(id, selected1)) selected1.splice($.inArray(id, selected1), 1);
              }
              selectedRows(); // update selected1's value based on selClass
              selectRows(false); // only keep the selectable rows
              changeInput('rows_selected', selected1);
              changeInput('row_last_clicked', serverRowIndex(thisRow.index()), null, null, { priority: 'event' });
              lastClickedRow = serverRowIndex(thisRow.index());
            });
            selectRows(false);  // in case users have specified pre-selected rows
            // restore selected rows after the table is redrawn (e.g. sort/search/page);
            // client-side tables will preserve the selections automatically; for
            // server-side tables, we have to *real* row indices are in `selected1`
            changeInput('rows_selected', selected1);
            if (server) table.on('draw.dt', function (e) { selectRows(false); });
            methods.selectRows = function (selected, ignoreSelectable) {
              selected1 = $.makeArray(selected);
              selectRows(ignoreSelectable);
              changeInput('rows_selected', selected1);
            }
          }

          if (inArray(selTarget, ['column', 'row+column'])) {
            if (selTarget === 'row+column') {
              $(table.columns().footer()).css('cursor', 'pointer');
            }
            // update selected2's value based on selectable2
            var onlyKeepSelectableCols = function () {
              if (selDisable) { // users can't select; useful when only want backend select
                selected2 = [];
                return;
              }
              if (selectable2.length === 0) return;
              var nonselectable = selectable2[0] <= 0;
              if (nonselectable) {
                // need to make selectable2 positive
                selected2 = $(selected2).not(selectable2.map(function (i) { return -i; })).get();
              } else {
                selected2 = $(selected2).filter(selectable2).get();
              }
            }
            // update selected2 and then
            // refresh the col selection state according to values in selected2
            var selectCols = function (ignoreSelectable) {
              if (!ignoreSelectable) onlyKeepSelectableCols();
              // if selected2 is not a valide index (e.g., larger than the column number)
              // table.columns(selected2) will fail and result in a blank table
              // this is different from the table.rows(), where the out-of-range indexes
              // doesn't affect at all
              selected2 = $(selected2).filter(table.columns().indexes()).get();
              table.columns().nodes().flatten().to$().removeClass(selClass);
              if (selected2.length > 0)
                table.columns(selected2).nodes().flatten().to$().addClass(selClass);
            }
            var callback = function () {
              var colIdx = selTarget === 'column' ? table.cell(this).index().column :
                $.inArray(this, table.columns().footer()),
                thisCol = $(table.column(colIdx).nodes());
              if (colIdx === -1) return;
              if (thisCol.hasClass(selClass)) {
                thisCol.removeClass(selClass);
                selected2.splice($.inArray(colIdx, selected2), 1);
              } else {
                if (selMode === 'single') $(table.cells().nodes()).removeClass(selClass);
                thisCol.addClass(selClass);
                selected2 = selMode === 'single' ? [colIdx] : unique(selected2.concat([colIdx]));
              }
              selectCols(false); // update selected2 based on selectable
              changeInput('columns_selected', selected2);
            }
            if (selTarget === 'column') {
              $(table.table().body()).on('click.dt', 'td', callback);
            } else {
              $(table.table().footer()).on('click.dt', 'tr th', callback);
            }
            selectCols(false);  // in case users have specified pre-selected columns
            changeInput('columns_selected', selected2);
            if (server) table.on('draw.dt', function (e) { selectCols(false); });
            methods.selectColumns = function (selected, ignoreSelectable) {
              selected2 = $.makeArray(selected);
              selectCols(ignoreSelectable);
              changeInput('columns_selected', selected2);
            }
          }

          if (selTarget === 'cell') {
            var selected3 = [], selectable3 = [];
            if (selected !== null) selected3 = selected;
            if (selectable !== null && typeof selectable !== 'boolean') selectable3 = selectable;
            var findIndex = function (ij, sel) {
              for (var i = 0; i < sel.length; i++) {
                if (ij[0] === sel[i][0] && ij[1] === sel[i][1]) return i;
              }
              return -1;
            }
            // Change selected3's value based on selectable3, then refresh the cell state
            var onlyKeepSelectableCells = function () {
              if (selDisable) { // users can't select; useful when only want backend select
                selected3 = [];
                return;
              }
              if (selectable3.length === 0) return;
              var nonselectable = selectable3[0][0] <= 0;
              var out = [];
              if (nonselectable) {
                selected3.map(function (ij) {
                  // should make selectable3 positive
                  if (findIndex([-ij[0], -ij[1]], selectable3) === -1) { out.push(ij); }
                });
              } else {
                selected3.map(function (ij) {
                  if (findIndex(ij, selectable3) > -1) { out.push(ij); }
                });
              }
              selected3 = out;
            }
            // Change selected3's value based on selectable3, then
            // refresh the cell selection state according to values in selected3
            var selectCells = function (ignoreSelectable) {
              if (!ignoreSelectable) onlyKeepSelectableCells();
              table.$('td.' + selClass).removeClass(selClass);
              if (selected3.length === 0) return;
              if (server) {
                table.cells({ page: 'current' }).every(function () {
                  var info = tweakCellIndex(this);
                  if (findIndex([info.row, info.col], selected3) > -1)
                    $(this.node()).addClass(selClass);
                });
              } else {
                selected3.map(function (ij) {
                  $(table.cell(ij[0] - 1, ij[1]).node()).addClass(selClass);
                });
              }
            };
            table.on('click.dt', 'tbody td', function () {
              var $this = $(this), info = tweakCellIndex(table.cell(this));
              if ($this.hasClass(selClass)) {
                $this.removeClass(selClass);
                selected3.splice(findIndex([info.row, info.col], selected3), 1);
              } else {
                if (selMode === 'single') $(table.cells().nodes()).removeClass(selClass);
                $this.addClass(selClass);
                selected3 = selMode === 'single' ? [[info.row, info.col]] :
                  unique(selected3.concat([[info.row, info.col]]));
              }
              selectCells(false); // must call this to update selected3 based on selectable3
              changeInput('cells_selected', transposeArray2D(selected3), 'shiny.matrix');
            });
            selectCells(false);  // in case users have specified pre-selected columns
            changeInput('cells_selected', transposeArray2D(selected3), 'shiny.matrix');

            if (server) table.on('draw.dt', function (e) { selectCells(false); });
            methods.selectCells = function (selected, ignoreSelectable) {
              selected3 = selected ? selected : [];
              selectCells(ignoreSelectable);
              changeInput('cells_selected', transposeArray2D(selected3), 'shiny.matrix');
            }
          }
        }

        // expose some table info to Shiny
        var updateTableInfo = function (e, settings) {
          // TODO: is anyone interested in the page info?
          // changeInput('page_info', table.page.info());
          var updateRowInfo = function (id, modifier) {
            var idx;
            if (server) {
              idx = modifier.page === 'current' ? DT_rows_current : DT_rows_all;
            } else {
              var rows = table.rows($.extend({
                search: 'applied',
                page: 'all'
              }, modifier));
              idx = addOne(rows.indexes().toArray());
            }
            changeInput('rows' + '_' + id, idx);
          };
          updateRowInfo('current', { page: 'current' });
          updateRowInfo('all', {});
        }
        table.on('draw.dt', updateTableInfo);
        updateTableInfo();

        // state info
        table.on('draw.dt column-visibility.dt', function () {
          changeInput('state', table.state());
        });
        changeInput('state', table.state());

        // search info
        var updateSearchInfo = function () {
          changeInput('search', table.search());
          if (filterRow) changeInput('search_columns', filterRow.toArray().map(function (td) {
            return $(td).find('input').first().val();
          }));
        }
        table.on('draw.dt', updateSearchInfo);
        updateSearchInfo();

        var cellInfo = function (thiz) {
          var info = tweakCellIndex(table.cell(thiz));
          info.value = table.cell(thiz).data();
          return info;
        }
        // the current cell clicked on
        table.on('click.dt', 'tbody td', function () {
          changeInput('cell_clicked', cellInfo(this), null, null, { priority: 'event' });
        })
        changeInput('cell_clicked', {});

        // do not trigger table selection when clicking on links unless they have classes
        table.on('click.dt', 'tbody td a', function (e) {
          if (this.className === '') e.stopPropagation();
        });

        methods.addRow = function (data, rowname, resetPaging) {
          var n = table.columns().indexes().length, d = n - data.length;
          if (d === 1) {
            data = rowname.concat(data)
          } else if (d !== 0) {
            console.log(data);
            console.log(table.columns().indexes());
            throw 'New data must be of the same length as current data (' + n + ')';
          };
          table.row.add(data).draw(resetPaging);
        }

        methods.updateSearch = function (keywords) {
          if (keywords.global !== null)
            $(table.table().container()).find('input[type=search]').first()
              .val(keywords.global).trigger('input');
          var columns = keywords.columns;
          if (!filterRow || columns === null) return;
          filterRow.toArray().map(function (td, i) {
            var v = typeof columns === 'string' ? columns : columns[i];
            if (typeof v === 'undefined') {
              console.log('The search keyword for column ' + i + ' is undefined')
              return;
            }
            $(td).find('input').first().val(v);
            searchColumn(i, v);
          });
          table.draw();
        }

        methods.hideCols = function (hide, reset) {
          if (reset) table.columns().visible(true, false);
          table.columns(hide).visible(false);
        }

        methods.showCols = function (show, reset) {
          if (reset) table.columns().visible(false, false);
          table.columns(show).visible(true);
        }

        methods.colReorder = function (order, origOrder) {
          table.colReorder.order(order, origOrder);
        }

        methods.selectPage = function (page) {
          if (table.page.info().pages < page || page < 1) {
            throw 'Selected page is out of range';
          };
          table.page(page - 1).draw(false);
        }

        methods.reloadData = function (resetPaging, clearSelection) {
          // empty selections first if necessary
          if (methods.selectRows && inArray('row', clearSelection)) methods.selectRows([]);
          if (methods.selectColumns && inArray('column', clearSelection)) methods.selectColumns([]);
          if (methods.selectCells && inArray('cell', clearSelection)) methods.selectCells([]);
          table.ajax.reload(null, resetPaging);
        }

        // update table filters (set new limits of sliders)
        methods.updateFilters = function (newProps) {
          // loop through each filter in the filter row
          filterRow.each(function (i, td) {
            var k = i;
            if (filterRow.length > newProps.length) {
              if (i === 0) return;  // first column is row names
              k = i - 1;
            }
            // Update the filters to reflect the updated data.
            // Allow "falsy" (e.g. NULL) to signify a no-op.
            if (newProps[k]) {
              setFilterProps(td, newProps[k]);
            }
          });
        };

        table.shinyMethods = methods;
      },
      resize: function (el, width, height, instance) {
        if (instance.data) this.renderValue(el, instance.data, instance);

        // dynamically adjust height if fillContainer = TRUE
        if (instance.fillContainer)
          this.fillAvailableHeight(el, height);

        this.adjustWidth(el);
      },

      // dynamically set the scroll body to fill available height
      // (used with fillContainer = TRUE)
      fillAvailableHeight: function (el, availableHeight) {

        // see how much of the table is occupied by header/footer elements
        // and use that to compute a target scroll body height
        var dtWrapper = $(el).find('div.dataTables_wrapper');
        var dtScrollBody = $(el).find($('div.dataTables_scrollBody'));
        var framingHeight = dtWrapper.innerHeight() - dtScrollBody.innerHeight();
        var scrollBodyHeight = availableHeight - framingHeight;

        // we need to set `max-height` to none as datatables library now sets this
        // to a fixed height, disabling the ability to resize to fill the window,
        // as it will be set to a fixed 100px under such circumstances, e.g., RStudio IDE,
        // or FlexDashboard
        // see https://github.com/rstudio/DT/issues/951#issuecomment-1026464509
        dtScrollBody.css('max-height', 'none');
        // set the height
        dtScrollBody.height(scrollBodyHeight + 'px');
      },

      // adjust the width of columns; remove the hard-coded widths on table and the
      // scroll header when scrollX/Y are enabled
      adjustWidth: function (el) {
        var $el = $(el), table = $el.data('datatable');
        if (table) table.columns.adjust();
        $el.find('.dataTables_scrollHeadInner').css('width', '')
          .children('table').css('margin-left', '');
      }
    });

    if (!HTMLWidgets.shinyMode) return;

    Shiny.addCustomMessageHandler('datatable-calls', function (data) {
      var id = data.id;
      var el = document.getElementById(id);
      var table = el ? $(el).data('datatable') : null;
      if (!table) {
        console.log("Couldn't find table with id " + id);
        return;
      }

      var methods = table.shinyMethods, call = data.call;
      if (methods[call.method]) {
        methods[call.method].apply(table, call.args);
      } else {
        console.log("Unknown method " + call.method);
      }
    });

  })();
</script>

<script>/*!
SpryMedia Ltd.

This source file is free software, available under the following license:
MIT license - http://datatables.net/license

This source file is distributed in the hope that it will be useful, but
WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
or FITNESS FOR A PARTICULAR PURPOSE. See the license files for details.

For details please refer to: http://www.datatables.net
DataTables 1.12.1
©2008-2022 SpryMedia Ltd - datatables.net/license
*/
  var $jscomp = $jscomp || {}; $jscomp.scope = {}; $jscomp.findInternal = function (l, y, A) { l instanceof String && (l = String(l)); for (var q = l.length, E = 0; E < q; E++) { var P = l[E]; if (y.call(A, P, E, l)) return { i: E, v: P } } return { i: -1, v: void 0 } }; $jscomp.ASSUME_ES5 = !1; $jscomp.ASSUME_NO_NATIVE_MAP = !1; $jscomp.ASSUME_NO_NATIVE_SET = !1; $jscomp.SIMPLE_FROUND_POLYFILL = !1; $jscomp.ISOLATE_POLYFILLS = !1;
  $jscomp.defineProperty = $jscomp.ASSUME_ES5 || "function" == typeof Object.defineProperties ? Object.defineProperty : function (l, y, A) { if (l == Array.prototype || l == Object.prototype) return l; l[y] = A.value; return l }; $jscomp.getGlobal = function (l) { l = ["object" == typeof globalThis && globalThis, l, "object" == typeof window && window, "object" == typeof self && self, "object" == typeof global && global]; for (var y = 0; y < l.length; ++y) { var A = l[y]; if (A && A.Math == Math) return A } throw Error("Cannot find global object"); }; $jscomp.global = $jscomp.getGlobal(this);
  $jscomp.IS_SYMBOL_NATIVE = "function" === typeof Symbol && "symbol" === typeof Symbol("x"); $jscomp.TRUST_ES6_POLYFILLS = !$jscomp.ISOLATE_POLYFILLS || $jscomp.IS_SYMBOL_NATIVE; $jscomp.polyfills = {}; $jscomp.propertyToPolyfillSymbol = {}; $jscomp.POLYFILL_PREFIX = "$jscp$"; var $jscomp$lookupPolyfilledValue = function (l, y) { var A = $jscomp.propertyToPolyfillSymbol[y]; if (null == A) return l[y]; A = l[A]; return void 0 !== A ? A : l[y] };
  $jscomp.polyfill = function (l, y, A, q) { y && ($jscomp.ISOLATE_POLYFILLS ? $jscomp.polyfillIsolated(l, y, A, q) : $jscomp.polyfillUnisolated(l, y, A, q)) }; $jscomp.polyfillUnisolated = function (l, y, A, q) { A = $jscomp.global; l = l.split("."); for (q = 0; q < l.length - 1; q++) { var E = l[q]; if (!(E in A)) return; A = A[E] } l = l[l.length - 1]; q = A[l]; y = y(q); y != q && null != y && $jscomp.defineProperty(A, l, { configurable: !0, writable: !0, value: y }) };
  $jscomp.polyfillIsolated = function (l, y, A, q) {
    var E = l.split("."); l = 1 === E.length; q = E[0]; q = !l && q in $jscomp.polyfills ? $jscomp.polyfills : $jscomp.global; for (var P = 0; P < E.length - 1; P++) { var la = E[P]; if (!(la in q)) return; q = q[la] } E = E[E.length - 1]; A = $jscomp.IS_SYMBOL_NATIVE && "es6" === A ? q[E] : null; y = y(A); null != y && (l ? $jscomp.defineProperty($jscomp.polyfills, E, { configurable: !0, writable: !0, value: y }) : y !== A && ($jscomp.propertyToPolyfillSymbol[E] = $jscomp.IS_SYMBOL_NATIVE ? $jscomp.global.Symbol(E) : $jscomp.POLYFILL_PREFIX + E,
      E = $jscomp.propertyToPolyfillSymbol[E], $jscomp.defineProperty(q, E, { configurable: !0, writable: !0, value: y })))
  }; $jscomp.polyfill("Array.prototype.find", function (l) { return l ? l : function (y, A) { return $jscomp.findInternal(this, y, A).v } }, "es6", "es3");
  (function (l) { "function" === typeof define && define.amd ? define(["jquery"], function (y) { return l(y, window, document) }) : "object" === typeof exports ? module.exports = function (y, A) { y || (y = window); A || (A = "undefined" !== typeof window ? require("jquery") : require("jquery")(y)); return l(A, y, y.document) } : window.DataTable = l(jQuery, window, document) })(function (l, y, A, q) {
    function E(a) {
      var b, c, d = {}; l.each(a, function (e, h) {
        (b = e.match(/^([^A-Z]+?)([A-Z])/)) && -1 !== "a aa ai ao as b fn i m o s ".indexOf(b[1] + " ") && (c = e.replace(b[0],
          b[2].toLowerCase()), d[c] = e, "o" === b[1] && E(a[e]))
      }); a._hungarianMap = d
    } function P(a, b, c) { a._hungarianMap || E(a); var d; l.each(b, function (e, h) { d = a._hungarianMap[e]; d === q || !c && b[d] !== q || ("o" === d.charAt(0) ? (b[d] || (b[d] = {}), l.extend(!0, b[d], b[e]), P(a[d], b[d], c)) : b[d] = b[e]) }) } function la(a) {
      var b = u.defaults.oLanguage, c = b.sDecimal; c && bb(c); if (a) {
        var d = a.sZeroRecords; !a.sEmptyTable && d && "No data available in table" === b.sEmptyTable && Y(a, a, "sZeroRecords", "sEmptyTable"); !a.sLoadingRecords && d && "Loading..." === b.sLoadingRecords &&
          Y(a, a, "sZeroRecords", "sLoadingRecords"); a.sInfoThousands && (a.sThousands = a.sInfoThousands); (a = a.sDecimal) && c !== a && bb(a)
      }
    } function Db(a) {
      S(a, "ordering", "bSort"); S(a, "orderMulti", "bSortMulti"); S(a, "orderClasses", "bSortClasses"); S(a, "orderCellsTop", "bSortCellsTop"); S(a, "order", "aaSorting"); S(a, "orderFixed", "aaSortingFixed"); S(a, "paging", "bPaginate"); S(a, "pagingType", "sPaginationType"); S(a, "pageLength", "iDisplayLength"); S(a, "searching", "bFilter"); "boolean" === typeof a.sScrollX && (a.sScrollX = a.sScrollX ? "100%" :
        ""); "boolean" === typeof a.scrollX && (a.scrollX = a.scrollX ? "100%" : ""); if (a = a.aoSearchCols) for (var b = 0, c = a.length; b < c; b++)a[b] && P(u.models.oSearch, a[b])
    } function Eb(a) { S(a, "orderable", "bSortable"); S(a, "orderData", "aDataSort"); S(a, "orderSequence", "asSorting"); S(a, "orderDataType", "sortDataType"); var b = a.aDataSort; "number" !== typeof b || Array.isArray(b) || (a.aDataSort = [b]) } function Fb(a) {
      if (!u.__browser) {
        var b = {}; u.__browser = b; var c = l("<div/>").css({
          position: "fixed", top: 0, left: -1 * l(y).scrollLeft(), height: 1,
          width: 1, overflow: "hidden"
        }).append(l("<div/>").css({ position: "absolute", top: 1, left: 1, width: 100, overflow: "scroll" }).append(l("<div/>").css({ width: "100%", height: 10 }))).appendTo("body"), d = c.children(), e = d.children(); b.barWidth = d[0].offsetWidth - d[0].clientWidth; b.bScrollOversize = 100 === e[0].offsetWidth && 100 !== d[0].clientWidth; b.bScrollbarLeft = 1 !== Math.round(e.offset().left); b.bBounding = c[0].getBoundingClientRect().width ? !0 : !1; c.remove()
      } l.extend(a.oBrowser, u.__browser); a.oScroll.iBarWidth = u.__browser.barWidth
    }
    function Gb(a, b, c, d, e, h) { var f = !1; if (c !== q) { var g = c; f = !0 } for (; d !== e;)a.hasOwnProperty(d) && (g = f ? b(g, a[d], d, a) : a[d], f = !0, d += h); return g } function cb(a, b) { var c = u.defaults.column, d = a.aoColumns.length; c = l.extend({}, u.models.oColumn, c, { nTh: b ? b : A.createElement("th"), sTitle: c.sTitle ? c.sTitle : b ? b.innerHTML : "", aDataSort: c.aDataSort ? c.aDataSort : [d], mData: c.mData ? c.mData : d, idx: d }); a.aoColumns.push(c); c = a.aoPreSearchCols; c[d] = l.extend({}, u.models.oSearch, c[d]); Ia(a, d, l(b).data()) } function Ia(a, b, c) {
      b = a.aoColumns[b];
      var d = a.oClasses, e = l(b.nTh); if (!b.sWidthOrig) { b.sWidthOrig = e.attr("width") || null; var h = (e.attr("style") || "").match(/width:\s*(\d+[pxem%]+)/); h && (b.sWidthOrig = h[1]) } c !== q && null !== c && (Eb(c), P(u.defaults.column, c, !0), c.mDataProp === q || c.mData || (c.mData = c.mDataProp), c.sType && (b._sManualType = c.sType), c.className && !c.sClass && (c.sClass = c.className), c.sClass && e.addClass(c.sClass), h = b.sClass, l.extend(b, c), Y(b, c, "sWidth", "sWidthOrig"), h !== b.sClass && (b.sClass = h + " " + b.sClass), c.iDataSort !== q && (b.aDataSort = [c.iDataSort]),
        Y(b, c, "aDataSort")); var f = b.mData, g = ma(f), k = b.mRender ? ma(b.mRender) : null; c = function (m) { return "string" === typeof m && -1 !== m.indexOf("@") }; b._bAttrSrc = l.isPlainObject(f) && (c(f.sort) || c(f.type) || c(f.filter)); b._setter = null; b.fnGetData = function (m, n, p) { var t = g(m, n, q, p); return k && n ? k(t, n, m, p) : t }; b.fnSetData = function (m, n, p) { return ha(f)(m, n, p) }; "number" !== typeof f && (a._rowReadObject = !0); a.oFeatures.bSort || (b.bSortable = !1, e.addClass(d.sSortableNone)); a = -1 !== l.inArray("asc", b.asSorting); c = -1 !== l.inArray("desc",
          b.asSorting); b.bSortable && (a || c) ? a && !c ? (b.sSortingClass = d.sSortableAsc, b.sSortingClassJUI = d.sSortJUIAscAllowed) : !a && c ? (b.sSortingClass = d.sSortableDesc, b.sSortingClassJUI = d.sSortJUIDescAllowed) : (b.sSortingClass = d.sSortable, b.sSortingClassJUI = d.sSortJUI) : (b.sSortingClass = d.sSortableNone, b.sSortingClassJUI = "")
    } function sa(a) {
      if (!1 !== a.oFeatures.bAutoWidth) { var b = a.aoColumns; db(a); for (var c = 0, d = b.length; c < d; c++)b[c].nTh.style.width = b[c].sWidth } b = a.oScroll; "" === b.sY && "" === b.sX || Ja(a); F(a, null, "column-sizing",
        [a])
    } function ta(a, b) { a = Ka(a, "bVisible"); return "number" === typeof a[b] ? a[b] : null } function ua(a, b) { a = Ka(a, "bVisible"); b = l.inArray(b, a); return -1 !== b ? b : null } function na(a) { var b = 0; l.each(a.aoColumns, function (c, d) { d.bVisible && "none" !== l(d.nTh).css("display") && b++ }); return b } function Ka(a, b) { var c = []; l.map(a.aoColumns, function (d, e) { d[b] && c.push(e) }); return c } function eb(a) {
      var b = a.aoColumns, c = a.aoData, d = u.ext.type.detect, e, h, f; var g = 0; for (e = b.length; g < e; g++) {
        var k = b[g]; var m = []; if (!k.sType && k._sManualType) k.sType =
          k._sManualType; else if (!k.sType) { var n = 0; for (h = d.length; n < h; n++) { var p = 0; for (f = c.length; p < f; p++) { m[p] === q && (m[p] = T(a, p, g, "type")); var t = d[n](m[p], a); if (!t && n !== d.length - 1) break; if ("html" === t && !aa(m[p])) break } if (t) { k.sType = t; break } } k.sType || (k.sType = "string") }
      }
    } function Hb(a, b, c, d) {
      var e, h, f, g = a.aoColumns; if (b) for (e = b.length - 1; 0 <= e; e--) {
        var k = b[e]; var m = k.target !== q ? k.target : k.targets !== q ? k.targets : k.aTargets; Array.isArray(m) || (m = [m]); var n = 0; for (h = m.length; n < h; n++)if ("number" === typeof m[n] && 0 <= m[n]) {
          for (; g.length <=
            m[n];)cb(a); d(m[n], k)
        } else if ("number" === typeof m[n] && 0 > m[n]) d(g.length + m[n], k); else if ("string" === typeof m[n]) { var p = 0; for (f = g.length; p < f; p++)("_all" == m[n] || l(g[p].nTh).hasClass(m[n])) && d(p, k) }
      } if (c) for (e = 0, a = c.length; e < a; e++)d(e, c[e])
    } function ia(a, b, c, d) {
      var e = a.aoData.length, h = l.extend(!0, {}, u.models.oRow, { src: c ? "dom" : "data", idx: e }); h._aData = b; a.aoData.push(h); for (var f = a.aoColumns, g = 0, k = f.length; g < k; g++)f[g].sType = null; a.aiDisplayMaster.push(e); b = a.rowIdFn(b); b !== q && (a.aIds[b] = h); !c && a.oFeatures.bDeferRender ||
        fb(a, e, c, d); return e
    } function La(a, b) { var c; b instanceof l || (b = l(b)); return b.map(function (d, e) { c = gb(a, e); return ia(a, c.data, e, c.cells) }) } function T(a, b, c, d) {
      "search" === d ? d = "filter" : "order" === d && (d = "sort"); var e = a.iDraw, h = a.aoColumns[c], f = a.aoData[b]._aData, g = h.sDefaultContent, k = h.fnGetData(f, d, { settings: a, row: b, col: c }); if (k === q) return a.iDrawError != e && null === g && (ea(a, 0, "Requested unknown parameter " + ("function" == typeof h.mData ? "{function}" : "'" + h.mData + "'") + " for row " + b + ", column " + c, 4), a.iDrawError =
        e), g; if ((k === f || null === k) && null !== g && d !== q) k = g; else if ("function" === typeof k) return k.call(f); if (null === k && "display" === d) return ""; "filter" === d && (a = u.ext.type.search, a[h.sType] && (k = a[h.sType](k))); return k
    } function Ib(a, b, c, d) { a.aoColumns[c].fnSetData(a.aoData[b]._aData, d, { settings: a, row: b, col: c }) } function hb(a) { return l.map(a.match(/(\\.|[^\.])+/g) || [""], function (b) { return b.replace(/\\\./g, ".") }) } function ib(a) { return U(a.aoData, "_aData") } function Ma(a) {
      a.aoData.length = 0; a.aiDisplayMaster.length =
        0; a.aiDisplay.length = 0; a.aIds = {}
    } function Na(a, b, c) { for (var d = -1, e = 0, h = a.length; e < h; e++)a[e] == b ? d = e : a[e] > b && a[e]--; -1 != d && c === q && a.splice(d, 1) } function va(a, b, c, d) {
      var e = a.aoData[b], h, f = function (k, m) { for (; k.childNodes.length;)k.removeChild(k.firstChild); k.innerHTML = T(a, b, m, "display") }; if ("dom" !== c && (c && "auto" !== c || "dom" !== e.src)) { var g = e.anCells; if (g) if (d !== q) f(g[d], d); else for (c = 0, h = g.length; c < h; c++)f(g[c], c) } else e._aData = gb(a, e, d, d === q ? q : e._aData).data; e._aSortData = null; e._aFilterData = null; f =
        a.aoColumns; if (d !== q) f[d].sType = null; else { c = 0; for (h = f.length; c < h; c++)f[c].sType = null; jb(a, e) }
    } function gb(a, b, c, d) {
      var e = [], h = b.firstChild, f, g = 0, k, m = a.aoColumns, n = a._rowReadObject; d = d !== q ? d : n ? {} : []; var p = function (x, w) { if ("string" === typeof x) { var r = x.indexOf("@"); -1 !== r && (r = x.substring(r + 1), ha(x)(d, w.getAttribute(r))) } }, t = function (x) {
        if (c === q || c === g) f = m[g], k = x.innerHTML.trim(), f && f._bAttrSrc ? (ha(f.mData._)(d, k), p(f.mData.sort, x), p(f.mData.type, x), p(f.mData.filter, x)) : n ? (f._setter || (f._setter = ha(f.mData)),
          f._setter(d, k)) : d[g] = k; g++
      }; if (h) for (; h;) { var v = h.nodeName.toUpperCase(); if ("TD" == v || "TH" == v) t(h), e.push(h); h = h.nextSibling } else for (e = b.anCells, h = 0, v = e.length; h < v; h++)t(e[h]); (b = b.firstChild ? b : b.nTr) && (b = b.getAttribute("id")) && ha(a.rowId)(d, b); return { data: d, cells: e }
    } function fb(a, b, c, d) {
      var e = a.aoData[b], h = e._aData, f = [], g, k; if (null === e.nTr) {
        var m = c || A.createElement("tr"); e.nTr = m; e.anCells = f; m._DT_RowIndex = b; jb(a, e); var n = 0; for (g = a.aoColumns.length; n < g; n++) {
          var p = a.aoColumns[n]; e = (k = c ? !1 : !0) ? A.createElement(p.sCellType) :
            d[n]; e._DT_CellIndex = { row: b, column: n }; f.push(e); if (k || !(!p.mRender && p.mData === n || l.isPlainObject(p.mData) && p.mData._ === n + ".display")) e.innerHTML = T(a, b, n, "display"); p.sClass && (e.className += " " + p.sClass); p.bVisible && !c ? m.appendChild(e) : !p.bVisible && c && e.parentNode.removeChild(e); p.fnCreatedCell && p.fnCreatedCell.call(a.oInstance, e, T(a, b, n), h, b, n)
        } F(a, "aoRowCreatedCallback", null, [m, h, b, f])
      }
    } function jb(a, b) {
      var c = b.nTr, d = b._aData; if (c) {
        if (a = a.rowIdFn(d)) c.id = a; d.DT_RowClass && (a = d.DT_RowClass.split(" "),
          b.__rowc = b.__rowc ? Oa(b.__rowc.concat(a)) : a, l(c).removeClass(b.__rowc.join(" ")).addClass(d.DT_RowClass)); d.DT_RowAttr && l(c).attr(d.DT_RowAttr); d.DT_RowData && l(c).data(d.DT_RowData)
      }
    } function Jb(a) {
      var b, c, d = a.nTHead, e = a.nTFoot, h = 0 === l("th, td", d).length, f = a.oClasses, g = a.aoColumns; h && (c = l("<tr/>").appendTo(d)); var k = 0; for (b = g.length; k < b; k++) {
        var m = g[k]; var n = l(m.nTh).addClass(m.sClass); h && n.appendTo(c); a.oFeatures.bSort && (n.addClass(m.sSortingClass), !1 !== m.bSortable && (n.attr("tabindex", a.iTabIndex).attr("aria-controls",
          a.sTableId), kb(a, m.nTh, k))); m.sTitle != n[0].innerHTML && n.html(m.sTitle); lb(a, "header")(a, n, m, f)
      } h && wa(a.aoHeader, d); l(d).children("tr").children("th, td").addClass(f.sHeaderTH); l(e).children("tr").children("th, td").addClass(f.sFooterTH); if (null !== e) for (a = a.aoFooter[0], k = 0, b = a.length; k < b; k++)m = g[k], m.nTf = a[k].cell, m.sClass && l(m.nTf).addClass(m.sClass)
    } function xa(a, b, c) {
      var d, e, h = [], f = [], g = a.aoColumns.length; if (b) {
        c === q && (c = !1); var k = 0; for (d = b.length; k < d; k++) {
          h[k] = b[k].slice(); h[k].nTr = b[k].nTr; for (e =
            g - 1; 0 <= e; e--)a.aoColumns[e].bVisible || c || h[k].splice(e, 1); f.push([])
        } k = 0; for (d = h.length; k < d; k++) { if (a = h[k].nTr) for (; e = a.firstChild;)a.removeChild(e); e = 0; for (b = h[k].length; e < b; e++) { var m = g = 1; if (f[k][e] === q) { a.appendChild(h[k][e].cell); for (f[k][e] = 1; h[k + g] !== q && h[k][e].cell == h[k + g][e].cell;)f[k + g][e] = 1, g++; for (; h[k][e + m] !== q && h[k][e].cell == h[k][e + m].cell;) { for (c = 0; c < g; c++)f[k + c][e + m] = 1; m++ } l(h[k][e].cell).attr("rowspan", g).attr("colspan", m) } } }
      }
    } function ja(a, b) {
      var c = "ssp" == Q(a), d = a.iInitDisplayStart;
      d !== q && -1 !== d && (a._iDisplayStart = c ? d : d >= a.fnRecordsDisplay() ? 0 : d, a.iInitDisplayStart = -1); c = F(a, "aoPreDrawCallback", "preDraw", [a]); if (-1 !== l.inArray(!1, c)) V(a, !1); else {
        c = []; var e = 0; d = a.asStripeClasses; var h = d.length, f = a.oLanguage, g = "ssp" == Q(a), k = a.aiDisplay, m = a._iDisplayStart, n = a.fnDisplayEnd(); a.bDrawing = !0; if (a.bDeferLoading) a.bDeferLoading = !1, a.iDraw++, V(a, !1); else if (!g) a.iDraw++; else if (!a.bDestroying && !b) { Kb(a); return } if (0 !== k.length) for (b = g ? a.aoData.length : n, f = g ? 0 : m; f < b; f++) {
          g = k[f]; var p = a.aoData[g];
          null === p.nTr && fb(a, g); var t = p.nTr; if (0 !== h) { var v = d[e % h]; p._sRowStripe != v && (l(t).removeClass(p._sRowStripe).addClass(v), p._sRowStripe = v) } F(a, "aoRowCallback", null, [t, p._aData, e, f, g]); c.push(t); e++
        } else e = f.sZeroRecords, 1 == a.iDraw && "ajax" == Q(a) ? e = f.sLoadingRecords : f.sEmptyTable && 0 === a.fnRecordsTotal() && (e = f.sEmptyTable), c[0] = l("<tr/>", { "class": h ? d[0] : "" }).append(l("<td />", { valign: "top", colSpan: na(a), "class": a.oClasses.sRowEmpty }).html(e))[0]; F(a, "aoHeaderCallback", "header", [l(a.nTHead).children("tr")[0],
        ib(a), m, n, k]); F(a, "aoFooterCallback", "footer", [l(a.nTFoot).children("tr")[0], ib(a), m, n, k]); d = l(a.nTBody); d.children().detach(); d.append(l(c)); F(a, "aoDrawCallback", "draw", [a]); a.bSorted = !1; a.bFiltered = !1; a.bDrawing = !1
      }
    } function ka(a, b) { var c = a.oFeatures, d = c.bFilter; c.bSort && Lb(a); d ? ya(a, a.oPreviousSearch) : a.aiDisplay = a.aiDisplayMaster.slice(); !0 !== b && (a._iDisplayStart = 0); a._drawHold = b; ja(a); a._drawHold = !1 } function Mb(a) {
      var b = a.oClasses, c = l(a.nTable); c = l("<div/>").insertBefore(c); var d = a.oFeatures,
        e = l("<div/>", { id: a.sTableId + "_wrapper", "class": b.sWrapper + (a.nTFoot ? "" : " " + b.sNoFooter) }); a.nHolding = c[0]; a.nTableWrapper = e[0]; a.nTableReinsertBefore = a.nTable.nextSibling; for (var h = a.sDom.split(""), f, g, k, m, n, p, t = 0; t < h.length; t++) {
          f = null; g = h[t]; if ("<" == g) {
            k = l("<div/>")[0]; m = h[t + 1]; if ("'" == m || '"' == m) {
              n = ""; for (p = 2; h[t + p] != m;)n += h[t + p], p++; "H" == n ? n = b.sJUIHeader : "F" == n && (n = b.sJUIFooter); -1 != n.indexOf(".") ? (m = n.split("."), k.id = m[0].substr(1, m[0].length - 1), k.className = m[1]) : "#" == n.charAt(0) ? k.id = n.substr(1,
                n.length - 1) : k.className = n; t += p
            } e.append(k); e = l(k)
          } else if (">" == g) e = e.parent(); else if ("l" == g && d.bPaginate && d.bLengthChange) f = Nb(a); else if ("f" == g && d.bFilter) f = Ob(a); else if ("r" == g && d.bProcessing) f = Pb(a); else if ("t" == g) f = Qb(a); else if ("i" == g && d.bInfo) f = Rb(a); else if ("p" == g && d.bPaginate) f = Sb(a); else if (0 !== u.ext.feature.length) for (k = u.ext.feature, p = 0, m = k.length; p < m; p++)if (g == k[p].cFeature) { f = k[p].fnInit(a); break } f && (k = a.aanFeatures, k[g] || (k[g] = []), k[g].push(f), e.append(f))
        } c.replaceWith(e); a.nHolding =
          null
    } function wa(a, b) { b = l(b).children("tr"); var c, d, e; a.splice(0, a.length); var h = 0; for (e = b.length; h < e; h++)a.push([]); h = 0; for (e = b.length; h < e; h++) { var f = b[h]; for (c = f.firstChild; c;) { if ("TD" == c.nodeName.toUpperCase() || "TH" == c.nodeName.toUpperCase()) { var g = 1 * c.getAttribute("colspan"); var k = 1 * c.getAttribute("rowspan"); g = g && 0 !== g && 1 !== g ? g : 1; k = k && 0 !== k && 1 !== k ? k : 1; var m = 0; for (d = a[h]; d[m];)m++; var n = m; var p = 1 === g ? !0 : !1; for (d = 0; d < g; d++)for (m = 0; m < k; m++)a[h + m][n + d] = { cell: c, unique: p }, a[h + m].nTr = f } c = c.nextSibling } } }
    function Pa(a, b, c) { var d = []; c || (c = a.aoHeader, b && (c = [], wa(c, b))); b = 0; for (var e = c.length; b < e; b++)for (var h = 0, f = c[b].length; h < f; h++)!c[b][h].unique || d[h] && a.bSortCellsTop || (d[h] = c[b][h].cell); return d } function Qa(a, b, c) {
      F(a, "aoServerParams", "serverParams", [b]); if (b && Array.isArray(b)) { var d = {}, e = /(.*?)\[\]$/; l.each(b, function (n, p) { (n = p.name.match(e)) ? (n = n[0], d[n] || (d[n] = []), d[n].push(p.value)) : d[p.name] = p.value }); b = d } var h = a.ajax, f = a.oInstance, g = function (n) {
        var p = a.jqXHR ? a.jqXHR.status : null; if (null ===
          n || "number" === typeof p && 204 == p) n = {}, za(a, n, []); (p = n.error || n.sError) && ea(a, 0, p); a.json = n; F(a, null, "xhr", [a, n, a.jqXHR]); c(n)
      }; if (l.isPlainObject(h) && h.data) { var k = h.data; var m = "function" === typeof k ? k(b, a) : k; b = "function" === typeof k && m ? m : l.extend(!0, b, m); delete h.data } m = {
        data: b, success: g, dataType: "json", cache: !1, type: a.sServerMethod, error: function (n, p, t) {
          t = F(a, null, "xhr", [a, null, a.jqXHR]); -1 === l.inArray(!0, t) && ("parsererror" == p ? ea(a, 0, "Invalid JSON response", 1) : 4 === n.readyState && ea(a, 0, "Ajax error",
            7)); V(a, !1)
        }
      }; a.oAjaxData = b; F(a, null, "preXhr", [a, b]); a.fnServerData ? a.fnServerData.call(f, a.sAjaxSource, l.map(b, function (n, p) { return { name: p, value: n } }), g, a) : a.sAjaxSource || "string" === typeof h ? a.jqXHR = l.ajax(l.extend(m, { url: h || a.sAjaxSource })) : "function" === typeof h ? a.jqXHR = h.call(f, b, g, a) : (a.jqXHR = l.ajax(l.extend(m, h)), h.data = k)
    } function Kb(a) { a.iDraw++; V(a, !0); Qa(a, Tb(a), function (b) { Ub(a, b) }) } function Tb(a) {
      var b = a.aoColumns, c = b.length, d = a.oFeatures, e = a.oPreviousSearch, h = a.aoPreSearchCols, f = [], g = oa(a);
      var k = a._iDisplayStart; var m = !1 !== d.bPaginate ? a._iDisplayLength : -1; var n = function (x, w) { f.push({ name: x, value: w }) }; n("sEcho", a.iDraw); n("iColumns", c); n("sColumns", U(b, "sName").join(",")); n("iDisplayStart", k); n("iDisplayLength", m); var p = { draw: a.iDraw, columns: [], order: [], start: k, length: m, search: { value: e.sSearch, regex: e.bRegex } }; for (k = 0; k < c; k++) {
        var t = b[k]; var v = h[k]; m = "function" == typeof t.mData ? "function" : t.mData; p.columns.push({
          data: m, name: t.sName, searchable: t.bSearchable, orderable: t.bSortable, search: {
            value: v.sSearch,
            regex: v.bRegex
          }
        }); n("mDataProp_" + k, m); d.bFilter && (n("sSearch_" + k, v.sSearch), n("bRegex_" + k, v.bRegex), n("bSearchable_" + k, t.bSearchable)); d.bSort && n("bSortable_" + k, t.bSortable)
      } d.bFilter && (n("sSearch", e.sSearch), n("bRegex", e.bRegex)); d.bSort && (l.each(g, function (x, w) { p.order.push({ column: w.col, dir: w.dir }); n("iSortCol_" + x, w.col); n("sSortDir_" + x, w.dir) }), n("iSortingCols", g.length)); b = u.ext.legacy.ajax; return null === b ? a.sAjaxSource ? f : p : b ? f : p
    } function Ub(a, b) {
      var c = function (f, g) { return b[f] !== q ? b[f] : b[g] },
      d = za(a, b), e = c("sEcho", "draw"), h = c("iTotalRecords", "recordsTotal"); c = c("iTotalDisplayRecords", "recordsFiltered"); if (e !== q) { if (1 * e < a.iDraw) return; a.iDraw = 1 * e } d || (d = []); Ma(a); a._iRecordsTotal = parseInt(h, 10); a._iRecordsDisplay = parseInt(c, 10); e = 0; for (h = d.length; e < h; e++)ia(a, d[e]); a.aiDisplay = a.aiDisplayMaster.slice(); ja(a, !0); a._bInitComplete || Ra(a, b); V(a, !1)
    } function za(a, b, c) {
      a = l.isPlainObject(a.ajax) && a.ajax.dataSrc !== q ? a.ajax.dataSrc : a.sAjaxDataProp; if (!c) return "data" === a ? b.aaData || b[a] : "" !== a ? ma(a)(b) :
        b; ha(a)(b, c)
    } function Ob(a) {
      var b = a.oClasses, c = a.sTableId, d = a.oLanguage, e = a.oPreviousSearch, h = a.aanFeatures, f = '<input type="search" class="' + b.sFilterInput + '"/>', g = d.sSearch; g = g.match(/_INPUT_/) ? g.replace("_INPUT_", f) : g + f; b = l("<div/>", { id: h.f ? null : c + "_filter", "class": b.sFilter }).append(l("<label/>").append(g)); var k = function (n) {
        var p = this.value ? this.value : ""; e.return && "Enter" !== n.key || p == e.sSearch || (ya(a, { sSearch: p, bRegex: e.bRegex, bSmart: e.bSmart, bCaseInsensitive: e.bCaseInsensitive, "return": e.return }),
          a._iDisplayStart = 0, ja(a))
      }; h = null !== a.searchDelay ? a.searchDelay : "ssp" === Q(a) ? 400 : 0; var m = l("input", b).val(e.sSearch).attr("placeholder", d.sSearchPlaceholder).on("keyup.DT search.DT input.DT paste.DT cut.DT", h ? mb(k, h) : k).on("mouseup", function (n) { setTimeout(function () { k.call(m[0], n) }, 10) }).on("keypress.DT", function (n) { if (13 == n.keyCode) return !1 }).attr("aria-controls", c); l(a.nTable).on("search.dt.DT", function (n, p) { if (a === p) try { m[0] !== A.activeElement && m.val(e.sSearch) } catch (t) { } }); return b[0]
    } function ya(a,
      b, c) { var d = a.oPreviousSearch, e = a.aoPreSearchCols, h = function (g) { d.sSearch = g.sSearch; d.bRegex = g.bRegex; d.bSmart = g.bSmart; d.bCaseInsensitive = g.bCaseInsensitive; d.return = g.return }, f = function (g) { return g.bEscapeRegex !== q ? !g.bEscapeRegex : g.bRegex }; eb(a); if ("ssp" != Q(a)) { Vb(a, b.sSearch, c, f(b), b.bSmart, b.bCaseInsensitive, b.return); h(b); for (b = 0; b < e.length; b++)Wb(a, e[b].sSearch, b, f(e[b]), e[b].bSmart, e[b].bCaseInsensitive); Xb(a) } else h(b); a.bFiltered = !0; F(a, null, "search", [a]) } function Xb(a) {
        for (var b = u.ext.search,
          c = a.aiDisplay, d, e, h = 0, f = b.length; h < f; h++) { for (var g = [], k = 0, m = c.length; k < m; k++)e = c[k], d = a.aoData[e], b[h](a, d._aFilterData, e, d._aData, k) && g.push(e); c.length = 0; l.merge(c, g) }
      } function Wb(a, b, c, d, e, h) { if ("" !== b) { var f = [], g = a.aiDisplay; d = nb(b, d, e, h); for (e = 0; e < g.length; e++)b = a.aoData[g[e]]._aFilterData[c], d.test(b) && f.push(g[e]); a.aiDisplay = f } } function Vb(a, b, c, d, e, h) {
        e = nb(b, d, e, h); var f = a.oPreviousSearch.sSearch, g = a.aiDisplayMaster; h = []; 0 !== u.ext.search.length && (c = !0); var k = Yb(a); if (0 >= b.length) a.aiDisplay =
          g.slice(); else { if (k || c || d || f.length > b.length || 0 !== b.indexOf(f) || a.bSorted) a.aiDisplay = g.slice(); b = a.aiDisplay; for (c = 0; c < b.length; c++)e.test(a.aoData[b[c]]._sFilterRow) && h.push(b[c]); a.aiDisplay = h }
      } function nb(a, b, c, d) { a = b ? a : ob(a); c && (a = "^(?=.*?" + l.map(a.match(/"[^"]+"|[^ ]+/g) || [""], function (e) { if ('"' === e.charAt(0)) { var h = e.match(/^"(.*)"$/); e = h ? h[1] : e } return e.replace('"', "") }).join(")(?=.*?") + ").*$"); return new RegExp(a, d ? "i" : "") } function Yb(a) {
        var b = a.aoColumns, c, d; var e = !1; var h = 0; for (c = a.aoData.length; h <
          c; h++) { var f = a.aoData[h]; if (!f._aFilterData) { var g = []; e = 0; for (d = b.length; e < d; e++) { var k = b[e]; k.bSearchable ? (k = T(a, h, e, "filter"), null === k && (k = ""), "string" !== typeof k && k.toString && (k = k.toString())) : k = ""; k.indexOf && -1 !== k.indexOf("&") && (Sa.innerHTML = k, k = Bc ? Sa.textContent : Sa.innerText); k.replace && (k = k.replace(/[\r\n\u2028]/g, "")); g.push(k) } f._aFilterData = g; f._sFilterRow = g.join("  "); e = !0 } } return e
      } function Zb(a) { return { search: a.sSearch, smart: a.bSmart, regex: a.bRegex, caseInsensitive: a.bCaseInsensitive } }
    function $b(a) { return { sSearch: a.search, bSmart: a.smart, bRegex: a.regex, bCaseInsensitive: a.caseInsensitive } } function Rb(a) { var b = a.sTableId, c = a.aanFeatures.i, d = l("<div/>", { "class": a.oClasses.sInfo, id: c ? null : b + "_info" }); c || (a.aoDrawCallback.push({ fn: ac, sName: "information" }), d.attr("role", "status").attr("aria-live", "polite"), l(a.nTable).attr("aria-describedby", b + "_info")); return d[0] } function ac(a) {
      var b = a.aanFeatures.i; if (0 !== b.length) {
        var c = a.oLanguage, d = a._iDisplayStart + 1, e = a.fnDisplayEnd(), h = a.fnRecordsTotal(),
        f = a.fnRecordsDisplay(), g = f ? c.sInfo : c.sInfoEmpty; f !== h && (g += " " + c.sInfoFiltered); g += c.sInfoPostFix; g = bc(a, g); c = c.fnInfoCallback; null !== c && (g = c.call(a.oInstance, a, d, e, h, f, g)); l(b).html(g)
      }
    } function bc(a, b) {
      var c = a.fnFormatNumber, d = a._iDisplayStart + 1, e = a._iDisplayLength, h = a.fnRecordsDisplay(), f = -1 === e; return b.replace(/_START_/g, c.call(a, d)).replace(/_END_/g, c.call(a, a.fnDisplayEnd())).replace(/_MAX_/g, c.call(a, a.fnRecordsTotal())).replace(/_TOTAL_/g, c.call(a, h)).replace(/_PAGE_/g, c.call(a, f ? 1 : Math.ceil(d /
        e))).replace(/_PAGES_/g, c.call(a, f ? 1 : Math.ceil(h / e)))
    } function Aa(a) {
      var b = a.iInitDisplayStart, c = a.aoColumns; var d = a.oFeatures; var e = a.bDeferLoading; if (a.bInitialised) {
        Mb(a); Jb(a); xa(a, a.aoHeader); xa(a, a.aoFooter); V(a, !0); d.bAutoWidth && db(a); var h = 0; for (d = c.length; h < d; h++) { var f = c[h]; f.sWidth && (f.nTh.style.width = K(f.sWidth)) } F(a, null, "preInit", [a]); ka(a); c = Q(a); if ("ssp" != c || e) "ajax" == c ? Qa(a, [], function (g) { var k = za(a, g); for (h = 0; h < k.length; h++)ia(a, k[h]); a.iInitDisplayStart = b; ka(a); V(a, !1); Ra(a, g) },
          a) : (V(a, !1), Ra(a))
      } else setTimeout(function () { Aa(a) }, 200)
    } function Ra(a, b) { a._bInitComplete = !0; (b || a.oInit.aaData) && sa(a); F(a, null, "plugin-init", [a, b]); F(a, "aoInitComplete", "init", [a, b]) } function pb(a, b) { b = parseInt(b, 10); a._iDisplayLength = b; qb(a); F(a, null, "length", [a, b]) } function Nb(a) {
      var b = a.oClasses, c = a.sTableId, d = a.aLengthMenu, e = Array.isArray(d[0]), h = e ? d[0] : d; d = e ? d[1] : d; e = l("<select/>", { name: c + "_length", "aria-controls": c, "class": b.sLengthSelect }); for (var f = 0, g = h.length; f < g; f++)e[0][f] = new Option("number" ===
        typeof d[f] ? a.fnFormatNumber(d[f]) : d[f], h[f]); var k = l("<div><label/></div>").addClass(b.sLength); a.aanFeatures.l || (k[0].id = c + "_length"); k.children().append(a.oLanguage.sLengthMenu.replace("_MENU_", e[0].outerHTML)); l("select", k).val(a._iDisplayLength).on("change.DT", function (m) { pb(a, l(this).val()); ja(a) }); l(a.nTable).on("length.dt.DT", function (m, n, p) { a === n && l("select", k).val(p) }); return k[0]
    } function Sb(a) {
      var b = a.sPaginationType, c = u.ext.pager[b], d = "function" === typeof c, e = function (f) { ja(f) }; b = l("<div/>").addClass(a.oClasses.sPaging +
        b)[0]; var h = a.aanFeatures; d || c.fnInit(a, b, e); h.p || (b.id = a.sTableId + "_paginate", a.aoDrawCallback.push({ fn: function (f) { if (d) { var g = f._iDisplayStart, k = f._iDisplayLength, m = f.fnRecordsDisplay(), n = -1 === k; g = n ? 0 : Math.ceil(g / k); k = n ? 1 : Math.ceil(m / k); m = c(g, k); var p; n = 0; for (p = h.p.length; n < p; n++)lb(f, "pageButton")(f, h.p[n], n, m, g, k) } else c.fnUpdate(f, e) }, sName: "pagination" })); return b
    } function Ta(a, b, c) {
      var d = a._iDisplayStart, e = a._iDisplayLength, h = a.fnRecordsDisplay(); 0 === h || -1 === e ? d = 0 : "number" === typeof b ? (d = b *
        e, d > h && (d = 0)) : "first" == b ? d = 0 : "previous" == b ? (d = 0 <= e ? d - e : 0, 0 > d && (d = 0)) : "next" == b ? d + e < h && (d += e) : "last" == b ? d = Math.floor((h - 1) / e) * e : ea(a, 0, "Unknown paging action: " + b, 5); b = a._iDisplayStart !== d; a._iDisplayStart = d; b && (F(a, null, "page", [a]), c && ja(a)); return b
    } function Pb(a) { return l("<div/>", { id: a.aanFeatures.r ? null : a.sTableId + "_processing", "class": a.oClasses.sProcessing }).html(a.oLanguage.sProcessing).append("<div><div></div><div></div><div></div><div></div></div>").insertBefore(a.nTable)[0] } function V(a,
      b) { a.oFeatures.bProcessing && l(a.aanFeatures.r).css("display", b ? "block" : "none"); F(a, null, "processing", [a, b]) } function Qb(a) {
        var b = l(a.nTable), c = a.oScroll; if ("" === c.sX && "" === c.sY) return a.nTable; var d = c.sX, e = c.sY, h = a.oClasses, f = b.children("caption"), g = f.length ? f[0]._captionSide : null, k = l(b[0].cloneNode(!1)), m = l(b[0].cloneNode(!1)), n = b.children("tfoot"); n.length || (n = null); k = l("<div/>", { "class": h.sScrollWrapper }).append(l("<div/>", { "class": h.sScrollHead }).css({
          overflow: "hidden", position: "relative", border: 0,
          width: d ? d ? K(d) : null : "100%"
        }).append(l("<div/>", { "class": h.sScrollHeadInner }).css({ "box-sizing": "content-box", width: c.sXInner || "100%" }).append(k.removeAttr("id").css("margin-left", 0).append("top" === g ? f : null).append(b.children("thead"))))).append(l("<div/>", { "class": h.sScrollBody }).css({ position: "relative", overflow: "auto", width: d ? K(d) : null }).append(b)); n && k.append(l("<div/>", { "class": h.sScrollFoot }).css({ overflow: "hidden", border: 0, width: d ? d ? K(d) : null : "100%" }).append(l("<div/>", { "class": h.sScrollFootInner }).append(m.removeAttr("id").css("margin-left",
          0).append("bottom" === g ? f : null).append(b.children("tfoot"))))); b = k.children(); var p = b[0]; h = b[1]; var t = n ? b[2] : null; if (d) l(h).on("scroll.DT", function (v) { v = this.scrollLeft; p.scrollLeft = v; n && (t.scrollLeft = v) }); l(h).css("max-height", e); c.bCollapse || l(h).css("height", e); a.nScrollHead = p; a.nScrollBody = h; a.nScrollFoot = t; a.aoDrawCallback.push({ fn: Ja, sName: "scrolling" }); return k[0]
      } function Ja(a) {
        var b = a.oScroll, c = b.sX, d = b.sXInner, e = b.sY; b = b.iBarWidth; var h = l(a.nScrollHead), f = h[0].style, g = h.children("div"), k =
          g[0].style, m = g.children("table"); g = a.nScrollBody; var n = l(g), p = g.style, t = l(a.nScrollFoot).children("div"), v = t.children("table"), x = l(a.nTHead), w = l(a.nTable), r = w[0], C = r.style, G = a.nTFoot ? l(a.nTFoot) : null, ba = a.oBrowser, L = ba.bScrollOversize; U(a.aoColumns, "nTh"); var O = [], I = [], H = [], fa = [], Z, Ba = function (D) { D = D.style; D.paddingTop = "0"; D.paddingBottom = "0"; D.borderTopWidth = "0"; D.borderBottomWidth = "0"; D.height = 0 }; var X = g.scrollHeight > g.clientHeight; if (a.scrollBarVis !== X && a.scrollBarVis !== q) a.scrollBarVis = X, sa(a);
        else {
          a.scrollBarVis = X; w.children("thead, tfoot").remove(); if (G) { X = G.clone().prependTo(w); var ca = G.find("tr"); var Ca = X.find("tr"); X.find("[id]").removeAttr("id") } var Ua = x.clone().prependTo(w); x = x.find("tr"); X = Ua.find("tr"); Ua.find("th, td").removeAttr("tabindex"); Ua.find("[id]").removeAttr("id"); c || (p.width = "100%", h[0].style.width = "100%"); l.each(Pa(a, Ua), function (D, W) { Z = ta(a, D); W.style.width = a.aoColumns[Z].sWidth }); G && da(function (D) { D.style.width = "" }, Ca); h = w.outerWidth(); "" === c ? (C.width = "100%", L &&
            (w.find("tbody").height() > g.offsetHeight || "scroll" == n.css("overflow-y")) && (C.width = K(w.outerWidth() - b)), h = w.outerWidth()) : "" !== d && (C.width = K(d), h = w.outerWidth()); da(Ba, X); da(function (D) { var W = y.getComputedStyle ? y.getComputedStyle(D).width : K(l(D).width()); H.push(D.innerHTML); O.push(W) }, X); da(function (D, W) { D.style.width = O[W] }, x); l(X).css("height", 0); G && (da(Ba, Ca), da(function (D) { fa.push(D.innerHTML); I.push(K(l(D).css("width"))) }, Ca), da(function (D, W) { D.style.width = I[W] }, ca), l(Ca).height(0)); da(function (D,
              W) { D.innerHTML = '<div class="dataTables_sizing">' + H[W] + "</div>"; D.childNodes[0].style.height = "0"; D.childNodes[0].style.overflow = "hidden"; D.style.width = O[W] }, X); G && da(function (D, W) { D.innerHTML = '<div class="dataTables_sizing">' + fa[W] + "</div>"; D.childNodes[0].style.height = "0"; D.childNodes[0].style.overflow = "hidden"; D.style.width = I[W] }, Ca); Math.round(w.outerWidth()) < Math.round(h) ? (ca = g.scrollHeight > g.offsetHeight || "scroll" == n.css("overflow-y") ? h + b : h, L && (g.scrollHeight > g.offsetHeight || "scroll" == n.css("overflow-y")) &&
                (C.width = K(ca - b)), "" !== c && "" === d || ea(a, 1, "Possible column misalignment", 6)) : ca = "100%"; p.width = K(ca); f.width = K(ca); G && (a.nScrollFoot.style.width = K(ca)); !e && L && (p.height = K(r.offsetHeight + b)); c = w.outerWidth(); m[0].style.width = K(c); k.width = K(c); d = w.height() > g.clientHeight || "scroll" == n.css("overflow-y"); e = "padding" + (ba.bScrollbarLeft ? "Left" : "Right"); k[e] = d ? b + "px" : "0px"; G && (v[0].style.width = K(c), t[0].style.width = K(c), t[0].style[e] = d ? b + "px" : "0px"); w.children("colgroup").insertBefore(w.children("thead"));
          n.trigger("scroll"); !a.bSorted && !a.bFiltered || a._drawHold || (g.scrollTop = 0)
        }
      } function da(a, b, c) { for (var d = 0, e = 0, h = b.length, f, g; e < h;) { f = b[e].firstChild; for (g = c ? c[e].firstChild : null; f;)1 === f.nodeType && (c ? a(f, g, d) : a(f, d), d++), f = f.nextSibling, g = c ? g.nextSibling : null; e++ } } function db(a) {
        var b = a.nTable, c = a.aoColumns, d = a.oScroll, e = d.sY, h = d.sX, f = d.sXInner, g = c.length, k = Ka(a, "bVisible"), m = l("th", a.nTHead), n = b.getAttribute("width"), p = b.parentNode, t = !1, v, x = a.oBrowser; d = x.bScrollOversize; (v = b.style.width) && -1 !==
          v.indexOf("%") && (n = v); for (v = 0; v < k.length; v++) { var w = c[k[v]]; null !== w.sWidth && (w.sWidth = cc(w.sWidthOrig, p), t = !0) } if (d || !t && !h && !e && g == na(a) && g == m.length) for (v = 0; v < g; v++)k = ta(a, v), null !== k && (c[k].sWidth = K(m.eq(v).width())); else {
            g = l(b).clone().css("visibility", "hidden").removeAttr("id"); g.find("tbody tr").remove(); var r = l("<tr/>").appendTo(g.find("tbody")); g.find("thead, tfoot").remove(); g.append(l(a.nTHead).clone()).append(l(a.nTFoot).clone()); g.find("tfoot th, tfoot td").css("width", ""); m = Pa(a, g.find("thead")[0]);
            for (v = 0; v < k.length; v++)w = c[k[v]], m[v].style.width = null !== w.sWidthOrig && "" !== w.sWidthOrig ? K(w.sWidthOrig) : "", w.sWidthOrig && h && l(m[v]).append(l("<div/>").css({ width: w.sWidthOrig, margin: 0, padding: 0, border: 0, height: 1 })); if (a.aoData.length) for (v = 0; v < k.length; v++)t = k[v], w = c[t], l(dc(a, t)).clone(!1).append(w.sContentPadding).appendTo(r); l("[name]", g).removeAttr("name"); w = l("<div/>").css(h || e ? { position: "absolute", top: 0, left: 0, height: 1, right: 0, overflow: "hidden" } : {}).append(g).appendTo(p); h && f ? g.width(f) : h ?
              (g.css("width", "auto"), g.removeAttr("width"), g.width() < p.clientWidth && n && g.width(p.clientWidth)) : e ? g.width(p.clientWidth) : n && g.width(n); for (v = e = 0; v < k.length; v++)p = l(m[v]), f = p.outerWidth() - p.width(), p = x.bBounding ? Math.ceil(m[v].getBoundingClientRect().width) : p.outerWidth(), e += p, c[k[v]].sWidth = K(p - f); b.style.width = K(e); w.remove()
          } n && (b.style.width = K(n)); !n && !h || a._reszEvt || (b = function () { l(y).on("resize.DT-" + a.sInstance, mb(function () { sa(a) })) }, d ? setTimeout(b, 1E3) : b(), a._reszEvt = !0)
      } function cc(a, b) {
        if (!a) return 0;
        a = l("<div/>").css("width", K(a)).appendTo(b || A.body); b = a[0].offsetWidth; a.remove(); return b
      } function dc(a, b) { var c = ec(a, b); if (0 > c) return null; var d = a.aoData[c]; return d.nTr ? d.anCells[b] : l("<td/>").html(T(a, c, b, "display"))[0] } function ec(a, b) { for (var c, d = -1, e = -1, h = 0, f = a.aoData.length; h < f; h++)c = T(a, h, b, "display") + "", c = c.replace(Cc, ""), c = c.replace(/&nbsp;/g, " "), c.length > d && (d = c.length, e = h); return e } function K(a) { return null === a ? "0px" : "number" == typeof a ? 0 > a ? "0px" : a + "px" : a.match(/\d$/) ? a + "px" : a } function oa(a) {
        var b =
          [], c = a.aoColumns; var d = a.aaSortingFixed; var e = l.isPlainObject(d); var h = []; var f = function (n) { n.length && !Array.isArray(n[0]) ? h.push(n) : l.merge(h, n) }; Array.isArray(d) && f(d); e && d.pre && f(d.pre); f(a.aaSorting); e && d.post && f(d.post); for (a = 0; a < h.length; a++) { var g = h[a][0]; f = c[g].aDataSort; d = 0; for (e = f.length; d < e; d++) { var k = f[d]; var m = c[k].sType || "string"; h[a]._idx === q && (h[a]._idx = l.inArray(h[a][1], c[k].asSorting)); b.push({ src: g, col: k, dir: h[a][1], index: h[a]._idx, type: m, formatter: u.ext.type.order[m + "-pre"] }) } } return b
      }
    function Lb(a) {
      var b, c = [], d = u.ext.type.order, e = a.aoData, h = 0, f = a.aiDisplayMaster; eb(a); var g = oa(a); var k = 0; for (b = g.length; k < b; k++) { var m = g[k]; m.formatter && h++; fc(a, m.col) } if ("ssp" != Q(a) && 0 !== g.length) {
        k = 0; for (b = f.length; k < b; k++)c[f[k]] = k; h === g.length ? f.sort(function (n, p) { var t, v = g.length, x = e[n]._aSortData, w = e[p]._aSortData; for (t = 0; t < v; t++) { var r = g[t]; var C = x[r.col]; var G = w[r.col]; C = C < G ? -1 : C > G ? 1 : 0; if (0 !== C) return "asc" === r.dir ? C : -C } C = c[n]; G = c[p]; return C < G ? -1 : C > G ? 1 : 0 }) : f.sort(function (n, p) {
          var t, v = g.length,
          x = e[n]._aSortData, w = e[p]._aSortData; for (t = 0; t < v; t++) { var r = g[t]; var C = x[r.col]; var G = w[r.col]; r = d[r.type + "-" + r.dir] || d["string-" + r.dir]; C = r(C, G); if (0 !== C) return C } C = c[n]; G = c[p]; return C < G ? -1 : C > G ? 1 : 0
        })
      } a.bSorted = !0
    } function gc(a) {
      var b = a.aoColumns, c = oa(a); a = a.oLanguage.oAria; for (var d = 0, e = b.length; d < e; d++) {
        var h = b[d]; var f = h.asSorting; var g = h.ariaTitle || h.sTitle.replace(/<.*?>/g, ""); var k = h.nTh; k.removeAttribute("aria-sort"); h.bSortable && (0 < c.length && c[0].col == d ? (k.setAttribute("aria-sort", "asc" ==
          c[0].dir ? "ascending" : "descending"), h = f[c[0].index + 1] || f[0]) : h = f[0], g += "asc" === h ? a.sSortAscending : a.sSortDescending); k.setAttribute("aria-label", g)
      }
    } function rb(a, b, c, d) {
      var e = a.aaSorting, h = a.aoColumns[b].asSorting, f = function (g, k) { var m = g._idx; m === q && (m = l.inArray(g[1], h)); return m + 1 < h.length ? m + 1 : k ? null : 0 }; "number" === typeof e[0] && (e = a.aaSorting = [e]); c && a.oFeatures.bSortMulti ? (c = l.inArray(b, U(e, "0")), -1 !== c ? (b = f(e[c], !0), null === b && 1 === e.length && (b = 0), null === b ? e.splice(c, 1) : (e[c][1] = h[b], e[c]._idx = b)) :
        (e.push([b, h[0], 0]), e[e.length - 1]._idx = 0)) : e.length && e[0][0] == b ? (b = f(e[0]), e.length = 1, e[0][1] = h[b], e[0]._idx = b) : (e.length = 0, e.push([b, h[0]]), e[0]._idx = 0); ka(a); "function" == typeof d && d(a)
    } function kb(a, b, c, d) { var e = a.aoColumns[c]; sb(b, {}, function (h) { !1 !== e.bSortable && (a.oFeatures.bProcessing ? (V(a, !0), setTimeout(function () { rb(a, c, h.shiftKey, d); "ssp" !== Q(a) && V(a, !1) }, 0)) : rb(a, c, h.shiftKey, d)) }) } function Va(a) {
      var b = a.aLastSort, c = a.oClasses.sSortColumn, d = oa(a), e = a.oFeatures, h; if (e.bSort && e.bSortClasses) {
        e =
        0; for (h = b.length; e < h; e++) { var f = b[e].src; l(U(a.aoData, "anCells", f)).removeClass(c + (2 > e ? e + 1 : 3)) } e = 0; for (h = d.length; e < h; e++)f = d[e].src, l(U(a.aoData, "anCells", f)).addClass(c + (2 > e ? e + 1 : 3))
      } a.aLastSort = d
    } function fc(a, b) { var c = a.aoColumns[b], d = u.ext.order[c.sSortDataType], e; d && (e = d.call(a.oInstance, a, b, ua(a, b))); for (var h, f = u.ext.type.order[c.sType + "-pre"], g = 0, k = a.aoData.length; g < k; g++)if (c = a.aoData[g], c._aSortData || (c._aSortData = []), !c._aSortData[b] || d) h = d ? e[g] : T(a, g, b, "sort"), c._aSortData[b] = f ? f(h) : h }
    function Da(a) { if (!a._bLoadingState) { var b = { time: +new Date, start: a._iDisplayStart, length: a._iDisplayLength, order: l.extend(!0, [], a.aaSorting), search: Zb(a.oPreviousSearch), columns: l.map(a.aoColumns, function (c, d) { return { visible: c.bVisible, search: Zb(a.aoPreSearchCols[d]) } }) }; a.oSavedState = b; F(a, "aoStateSaveParams", "stateSaveParams", [a, b]); a.oFeatures.bStateSave && !a.bDestroying && a.fnStateSaveCallback.call(a.oInstance, a, b) } } function hc(a, b, c) {
      if (a.oFeatures.bStateSave) return b = a.fnStateLoadCallback.call(a.oInstance,
        a, function (d) { tb(a, d, c) }), b !== q && tb(a, b, c), !0; c()
    } function tb(a, b, c) {
      var d, e = a.aoColumns; a._bLoadingState = !0; var h = a._bInitComplete ? new u.Api(a) : null; if (b && b.time) {
        var f = F(a, "aoStateLoadParams", "stateLoadParams", [a, b]); if (-1 !== l.inArray(!1, f)) a._bLoadingState = !1; else if (f = a.iStateDuration, 0 < f && b.time < +new Date - 1E3 * f) a._bLoadingState = !1; else if (b.columns && e.length !== b.columns.length) a._bLoadingState = !1; else {
          a.oLoadedState = l.extend(!0, {}, b); b.length !== q && (h ? h.page.len(b.length) : a._iDisplayLength = b.length);
          b.start !== q && (null === h ? (a._iDisplayStart = b.start, a.iInitDisplayStart = b.start) : Ta(a, b.start / a._iDisplayLength)); b.order !== q && (a.aaSorting = [], l.each(b.order, function (k, m) { a.aaSorting.push(m[0] >= e.length ? [0, m[1]] : m) })); b.search !== q && l.extend(a.oPreviousSearch, $b(b.search)); if (b.columns) { f = 0; for (d = b.columns.length; f < d; f++) { var g = b.columns[f]; g.visible !== q && (h ? h.column(f).visible(g.visible, !1) : e[f].bVisible = g.visible); g.search !== q && l.extend(a.aoPreSearchCols[f], $b(g.search)) } h && h.columns.adjust() } a._bLoadingState =
            !1; F(a, "aoStateLoaded", "stateLoaded", [a, b])
        }
      } else a._bLoadingState = !1; c()
    } function Wa(a) { var b = u.settings; a = l.inArray(a, U(b, "nTable")); return -1 !== a ? b[a] : null } function ea(a, b, c, d) {
      c = "DataTables warning: " + (a ? "table id=" + a.sTableId + " - " : "") + c; d && (c += ". For more information about this error, please see http://datatables.net/tn/" + d); if (b) y.console && console.log && console.log(c); else if (b = u.ext, b = b.sErrMode || b.errMode, a && F(a, null, "error", [a, d, c]), "alert" == b) alert(c); else {
        if ("throw" == b) throw Error(c); "function" ==
          typeof b && b(a, d, c)
      }
    } function Y(a, b, c, d) { Array.isArray(c) ? l.each(c, function (e, h) { Array.isArray(h) ? Y(a, b, h[0], h[1]) : Y(a, b, h) }) : (d === q && (d = c), b[c] !== q && (a[d] = b[c])) } function ub(a, b, c) { var d; for (d in b) if (b.hasOwnProperty(d)) { var e = b[d]; l.isPlainObject(e) ? (l.isPlainObject(a[d]) || (a[d] = {}), l.extend(!0, a[d], e)) : c && "data" !== d && "aaData" !== d && Array.isArray(e) ? a[d] = e.slice() : a[d] = e } return a } function sb(a, b, c) {
      l(a).on("click.DT", b, function (d) { l(a).trigger("blur"); c(d) }).on("keypress.DT", b, function (d) {
        13 === d.which &&
        (d.preventDefault(), c(d))
      }).on("selectstart.DT", function () { return !1 })
    } function R(a, b, c, d) { c && a[b].push({ fn: c, sName: d }) } function F(a, b, c, d) { var e = []; b && (e = l.map(a[b].slice().reverse(), function (h, f) { return h.fn.apply(a.oInstance, d) })); null !== c && (b = l.Event(c + ".dt"), l(a.nTable).trigger(b, d), e.push(b.result)); return e } function qb(a) { var b = a._iDisplayStart, c = a.fnDisplayEnd(), d = a._iDisplayLength; b >= c && (b = c - d); b -= b % d; if (-1 === d || 0 > b) b = 0; a._iDisplayStart = b } function lb(a, b) {
      a = a.renderer; var c = u.ext.renderer[b];
      return l.isPlainObject(a) && a[b] ? c[a[b]] || c._ : "string" === typeof a ? c[a] || c._ : c._
    } function Q(a) { return a.oFeatures.bServerSide ? "ssp" : a.ajax || a.sAjaxSource ? "ajax" : "dom" } function Ea(a, b) { var c = ic.numbers_length, d = Math.floor(c / 2); b <= c ? a = pa(0, b) : a <= d ? (a = pa(0, c - 2), a.push("ellipsis"), a.push(b - 1)) : (a >= b - 1 - d ? a = pa(b - (c - 2), b) : (a = pa(a - d + 2, a + d - 1), a.push("ellipsis"), a.push(b - 1)), a.splice(0, 0, "ellipsis"), a.splice(0, 0, 0)); a.DT_el = "span"; return a } function bb(a) {
      l.each({
        num: function (b) { return Xa(b, a) }, "num-fmt": function (b) {
          return Xa(b,
            a, vb)
        }, "html-num": function (b) { return Xa(b, a, Ya) }, "html-num-fmt": function (b) { return Xa(b, a, Ya, vb) }
      }, function (b, c) { M.type.order[b + a + "-pre"] = c; b.match(/^html\-/) && (M.type.search[b + a] = M.type.search.html) })
    } function jc(a, b, c, d, e) { return y.moment ? a[b](e) : y.luxon ? a[c](e) : d ? a[d](e) : a } function Za(a, b, c) {
      if (y.moment) { var d = y.moment.utc(a, b, c, !0); if (!d.isValid()) return null } else if (y.luxon) { d = b ? y.luxon.DateTime.fromFormat(a, b) : y.luxon.DateTime.fromISO(a); if (!d.isValid) return null; d.setLocale(c) } else b ? (kc ||
        alert("DataTables warning: Formatted date without Moment.js or Luxon - https://datatables.net/tn/17"), kc = !0) : d = new Date(a); return d
    } function wb(a) {
      return function (b, c, d, e) {
        0 === arguments.length ? (d = "en", b = c = null) : 1 === arguments.length ? (d = "en", c = b, b = null) : 2 === arguments.length && (d = c, c = b, b = null); var h = "datetime-" + c; u.ext.type.order[h] || (u.ext.type.detect.unshift(function (f) { return f === h ? h : !1 }), u.ext.type.order[h + "-asc"] = function (f, g) { f = f.valueOf(); g = g.valueOf(); return f === g ? 0 : f < g ? -1 : 1 }, u.ext.type.order[h +
          "-desc"] = function (f, g) { f = f.valueOf(); g = g.valueOf(); return f === g ? 0 : f > g ? -1 : 1 }); return function (f, g) {
            if (null === f || f === q) "--now" === e ? (f = new Date, f = new Date(Date.UTC(f.getFullYear(), f.getMonth(), f.getDate(), f.getHours(), f.getMinutes(), f.getSeconds()))) : f = ""; if ("type" === g) return h; if ("" === f) return "sort" !== g ? "" : Za("0000-01-01 00:00:00", null, d); if (null !== c && b === c && "sort" !== g && "type" !== g && !(f instanceof Date)) return f; var k = Za(f, b, d); if (null === k) return f; if ("sort" === g) return k; f = null === c ? jc(k, "toDate", "toJSDate",
              "")[a]() : jc(k, "format", "toFormat", "toISOString", c); return "display" === g ? $a(f) : f
          }
      }
    } function lc(a) { return function () { var b = [Wa(this[u.ext.iApiIndex])].concat(Array.prototype.slice.call(arguments)); return u.ext.internal[a].apply(this, b) } } var u = function (a, b) {
      if (this instanceof u) return l(a).DataTable(b); b = a; this.$ = function (f, g) { return this.api(!0).$(f, g) }; this._ = function (f, g) { return this.api(!0).rows(f, g).data() }; this.api = function (f) { return f ? new B(Wa(this[M.iApiIndex])) : new B(this) }; this.fnAddData = function (f,
        g) { var k = this.api(!0); f = Array.isArray(f) && (Array.isArray(f[0]) || l.isPlainObject(f[0])) ? k.rows.add(f) : k.row.add(f); (g === q || g) && k.draw(); return f.flatten().toArray() }; this.fnAdjustColumnSizing = function (f) { var g = this.api(!0).columns.adjust(), k = g.settings()[0], m = k.oScroll; f === q || f ? g.draw(!1) : ("" !== m.sX || "" !== m.sY) && Ja(k) }; this.fnClearTable = function (f) { var g = this.api(!0).clear(); (f === q || f) && g.draw() }; this.fnClose = function (f) { this.api(!0).row(f).child.hide() }; this.fnDeleteRow = function (f, g, k) {
          var m = this.api(!0);
          f = m.rows(f); var n = f.settings()[0], p = n.aoData[f[0][0]]; f.remove(); g && g.call(this, n, p); (k === q || k) && m.draw(); return p
        }; this.fnDestroy = function (f) { this.api(!0).destroy(f) }; this.fnDraw = function (f) { this.api(!0).draw(f) }; this.fnFilter = function (f, g, k, m, n, p) { n = this.api(!0); null === g || g === q ? n.search(f, k, m, p) : n.column(g).search(f, k, m, p); n.draw() }; this.fnGetData = function (f, g) {
          var k = this.api(!0); if (f !== q) {
            var m = f.nodeName ? f.nodeName.toLowerCase() : ""; return g !== q || "td" == m || "th" == m ? k.cell(f, g).data() : k.row(f).data() ||
              null
          } return k.data().toArray()
        }; this.fnGetNodes = function (f) { var g = this.api(!0); return f !== q ? g.row(f).node() : g.rows().nodes().flatten().toArray() }; this.fnGetPosition = function (f) { var g = this.api(!0), k = f.nodeName.toUpperCase(); return "TR" == k ? g.row(f).index() : "TD" == k || "TH" == k ? (f = g.cell(f).index(), [f.row, f.columnVisible, f.column]) : null }; this.fnIsOpen = function (f) { return this.api(!0).row(f).child.isShown() }; this.fnOpen = function (f, g, k) { return this.api(!0).row(f).child(g, k).show().child()[0] }; this.fnPageChange =
          function (f, g) { f = this.api(!0).page(f); (g === q || g) && f.draw(!1) }; this.fnSetColumnVis = function (f, g, k) { f = this.api(!0).column(f).visible(g); (k === q || k) && f.columns.adjust().draw() }; this.fnSettings = function () { return Wa(this[M.iApiIndex]) }; this.fnSort = function (f) { this.api(!0).order(f).draw() }; this.fnSortListener = function (f, g, k) { this.api(!0).order.listener(f, g, k) }; this.fnUpdate = function (f, g, k, m, n) {
            var p = this.api(!0); k === q || null === k ? p.row(g).data(f) : p.cell(g, k).data(f); (n === q || n) && p.columns.adjust(); (m === q || m) &&
              p.draw(); return 0
          }; this.fnVersionCheck = M.fnVersionCheck; var c = this, d = b === q, e = this.length; d && (b = {}); this.oApi = this.internal = M.internal; for (var h in u.ext.internal) h && (this[h] = lc(h)); this.each(function () {
            var f = {}, g = 1 < e ? ub(f, b, !0) : b, k = 0, m; f = this.getAttribute("id"); var n = !1, p = u.defaults, t = l(this); if ("table" != this.nodeName.toLowerCase()) ea(null, 0, "Non-table node initialisation (" + this.nodeName + ")", 2); else {
              Db(p); Eb(p.column); P(p, p, !0); P(p.column, p.column, !0); P(p, l.extend(g, t.data()), !0); var v = u.settings;
              k = 0; for (m = v.length; k < m; k++) { var x = v[k]; if (x.nTable == this || x.nTHead && x.nTHead.parentNode == this || x.nTFoot && x.nTFoot.parentNode == this) { var w = g.bRetrieve !== q ? g.bRetrieve : p.bRetrieve; if (d || w) return x.oInstance; if (g.bDestroy !== q ? g.bDestroy : p.bDestroy) { x.oInstance.fnDestroy(); break } else { ea(x, 0, "Cannot reinitialise DataTable", 3); return } } if (x.sTableId == this.id) { v.splice(k, 1); break } } if (null === f || "" === f) this.id = f = "DataTables_Table_" + u.ext._unique++; var r = l.extend(!0, {}, u.models.oSettings, {
                sDestroyWidth: t[0].style.width,
                sInstance: f, sTableId: f
              }); r.nTable = this; r.oApi = c.internal; r.oInit = g; v.push(r); r.oInstance = 1 === c.length ? c : t.dataTable(); Db(g); la(g.oLanguage); g.aLengthMenu && !g.iDisplayLength && (g.iDisplayLength = Array.isArray(g.aLengthMenu[0]) ? g.aLengthMenu[0][0] : g.aLengthMenu[0]); g = ub(l.extend(!0, {}, p), g); Y(r.oFeatures, g, "bPaginate bLengthChange bFilter bSort bSortMulti bInfo bProcessing bAutoWidth bSortClasses bServerSide bDeferRender".split(" ")); Y(r, g, ["asStripeClasses", "ajax", "fnServerData", "fnFormatNumber", "sServerMethod",
                "aaSorting", "aaSortingFixed", "aLengthMenu", "sPaginationType", "sAjaxSource", "sAjaxDataProp", "iStateDuration", "sDom", "bSortCellsTop", "iTabIndex", "fnStateLoadCallback", "fnStateSaveCallback", "renderer", "searchDelay", "rowId", ["iCookieDuration", "iStateDuration"], ["oSearch", "oPreviousSearch"], ["aoSearchCols", "aoPreSearchCols"], ["iDisplayLength", "_iDisplayLength"]]); Y(r.oScroll, g, [["sScrollX", "sX"], ["sScrollXInner", "sXInner"], ["sScrollY", "sY"], ["bScrollCollapse", "bCollapse"]]); Y(r.oLanguage, g, "fnInfoCallback");
              R(r, "aoDrawCallback", g.fnDrawCallback, "user"); R(r, "aoServerParams", g.fnServerParams, "user"); R(r, "aoStateSaveParams", g.fnStateSaveParams, "user"); R(r, "aoStateLoadParams", g.fnStateLoadParams, "user"); R(r, "aoStateLoaded", g.fnStateLoaded, "user"); R(r, "aoRowCallback", g.fnRowCallback, "user"); R(r, "aoRowCreatedCallback", g.fnCreatedRow, "user"); R(r, "aoHeaderCallback", g.fnHeaderCallback, "user"); R(r, "aoFooterCallback", g.fnFooterCallback, "user"); R(r, "aoInitComplete", g.fnInitComplete, "user"); R(r, "aoPreDrawCallback",
                g.fnPreDrawCallback, "user"); r.rowIdFn = ma(g.rowId); Fb(r); var C = r.oClasses; l.extend(C, u.ext.classes, g.oClasses); t.addClass(C.sTable); r.iInitDisplayStart === q && (r.iInitDisplayStart = g.iDisplayStart, r._iDisplayStart = g.iDisplayStart); null !== g.iDeferLoading && (r.bDeferLoading = !0, f = Array.isArray(g.iDeferLoading), r._iRecordsDisplay = f ? g.iDeferLoading[0] : g.iDeferLoading, r._iRecordsTotal = f ? g.iDeferLoading[1] : g.iDeferLoading); var G = r.oLanguage; l.extend(!0, G, g.oLanguage); G.sUrl ? (l.ajax({
                  dataType: "json", url: G.sUrl,
                  success: function (I) { P(p.oLanguage, I); la(I); l.extend(!0, G, I, r.oInit.oLanguage); F(r, null, "i18n", [r]); Aa(r) }, error: function () { Aa(r) }
                }), n = !0) : F(r, null, "i18n", [r]); null === g.asStripeClasses && (r.asStripeClasses = [C.sStripeOdd, C.sStripeEven]); f = r.asStripeClasses; var ba = t.children("tbody").find("tr").eq(0); -1 !== l.inArray(!0, l.map(f, function (I, H) { return ba.hasClass(I) })) && (l("tbody tr", this).removeClass(f.join(" ")), r.asDestroyStripes = f.slice()); f = []; v = this.getElementsByTagName("thead"); 0 !== v.length && (wa(r.aoHeader,
                  v[0]), f = Pa(r)); if (null === g.aoColumns) for (v = [], k = 0, m = f.length; k < m; k++)v.push(null); else v = g.aoColumns; k = 0; for (m = v.length; k < m; k++)cb(r, f ? f[k] : null); Hb(r, g.aoColumnDefs, v, function (I, H) { Ia(r, I, H) }); if (ba.length) {
                    var L = function (I, H) { return null !== I.getAttribute("data-" + H) ? H : null }; l(ba[0]).children("th, td").each(function (I, H) {
                      var fa = r.aoColumns[I]; if (fa.mData === I) {
                        var Z = L(H, "sort") || L(H, "order"); H = L(H, "filter") || L(H, "search"); if (null !== Z || null !== H) fa.mData = {
                          _: I + ".display", sort: null !== Z ? I + ".@data-" + Z : q,
                          type: null !== Z ? I + ".@data-" + Z : q, filter: null !== H ? I + ".@data-" + H : q
                        }, Ia(r, I)
                      }
                    })
                  } var O = r.oFeatures; f = function () {
                    if (g.aaSorting === q) { var I = r.aaSorting; k = 0; for (m = I.length; k < m; k++)I[k][1] = r.aoColumns[k].asSorting[0] } Va(r); O.bSort && R(r, "aoDrawCallback", function () { if (r.bSorted) { var Z = oa(r), Ba = {}; l.each(Z, function (X, ca) { Ba[ca.src] = ca.dir }); F(r, null, "order", [r, Z, Ba]); gc(r) } }); R(r, "aoDrawCallback", function () { (r.bSorted || "ssp" === Q(r) || O.bDeferRender) && Va(r) }, "sc"); I = t.children("caption").each(function () {
                      this._captionSide =
                      l(this).css("caption-side")
                    }); var H = t.children("thead"); 0 === H.length && (H = l("<thead/>").appendTo(t)); r.nTHead = H[0]; var fa = t.children("tbody"); 0 === fa.length && (fa = l("<tbody/>").insertAfter(H)); r.nTBody = fa[0]; H = t.children("tfoot"); 0 === H.length && 0 < I.length && ("" !== r.oScroll.sX || "" !== r.oScroll.sY) && (H = l("<tfoot/>").appendTo(t)); 0 === H.length || 0 === H.children().length ? t.addClass(C.sNoFooter) : 0 < H.length && (r.nTFoot = H[0], wa(r.aoFooter, r.nTFoot)); if (g.aaData) for (k = 0; k < g.aaData.length; k++)ia(r, g.aaData[k]); else (r.bDeferLoading ||
                      "dom" == Q(r)) && La(r, l(r.nTBody).children("tr")); r.aiDisplay = r.aiDisplayMaster.slice(); r.bInitialised = !0; !1 === n && Aa(r)
                  }; R(r, "aoDrawCallback", Da, "state_save"); g.bStateSave ? (O.bStateSave = !0, hc(r, g, f)) : f()
            }
          }); c = null; return this
    }, M, z, J, xb = {}, mc = /[\r\n\u2028]/g, Ya = /<.*?>/g, Dc = /^\d{2,4}[\.\/\-]\d{1,2}[\.\/\-]\d{1,2}([T ]{1}\d{1,2}[:\.]\d{2}([\.:]\d{2})?)?$/, Ec = /(\/|\.|\*|\+|\?|\||\(|\)|\[|\]|\{|\}|\\|\$|\^|\-)/g, vb = /['\u00A0,$£€¥%\u2009\u202F\u20BD\u20a9\u20BArfkɃΞ]/gi, aa = function (a) {
      return a && !0 !== a && "-" !==
        a ? !1 : !0
    }, nc = function (a) { var b = parseInt(a, 10); return !isNaN(b) && isFinite(a) ? b : null }, oc = function (a, b) { xb[b] || (xb[b] = new RegExp(ob(b), "g")); return "string" === typeof a && "." !== b ? a.replace(/\./g, "").replace(xb[b], ".") : a }, yb = function (a, b, c) { var d = "string" === typeof a; if (aa(a)) return !0; b && d && (a = oc(a, b)); c && d && (a = a.replace(vb, "")); return !isNaN(parseFloat(a)) && isFinite(a) }, pc = function (a, b, c) { return aa(a) ? !0 : aa(a) || "string" === typeof a ? yb(a.replace(Ya, ""), b, c) ? !0 : null : null }, U = function (a, b, c) {
      var d = [], e = 0, h = a.length;
      if (c !== q) for (; e < h; e++)a[e] && a[e][b] && d.push(a[e][b][c]); else for (; e < h; e++)a[e] && d.push(a[e][b]); return d
    }, Fa = function (a, b, c, d) { var e = [], h = 0, f = b.length; if (d !== q) for (; h < f; h++)a[b[h]][c] && e.push(a[b[h]][c][d]); else for (; h < f; h++)e.push(a[b[h]][c]); return e }, pa = function (a, b) { var c = []; if (b === q) { b = 0; var d = a } else d = b, b = a; for (a = b; a < d; a++)c.push(a); return c }, qc = function (a) { for (var b = [], c = 0, d = a.length; c < d; c++)a[c] && b.push(a[c]); return b }, Oa = function (a) {
      a: {
        if (!(2 > a.length)) {
          var b = a.slice().sort(); for (var c = b[0],
            d = 1, e = b.length; d < e; d++) { if (b[d] === c) { b = !1; break a } c = b[d] }
        } b = !0
      } if (b) return a.slice(); b = []; e = a.length; var h, f = 0; d = 0; a: for (; d < e; d++) { c = a[d]; for (h = 0; h < f; h++)if (b[h] === c) continue a; b.push(c); f++ } return b
    }, rc = function (a, b) { if (Array.isArray(b)) for (var c = 0; c < b.length; c++)rc(a, b[c]); else a.push(b); return a }, sc = function (a, b) { b === q && (b = 0); return -1 !== this.indexOf(a, b) }; Array.isArray || (Array.isArray = function (a) { return "[object Array]" === Object.prototype.toString.call(a) }); Array.prototype.includes || (Array.prototype.includes =
      sc); String.prototype.trim || (String.prototype.trim = function () { return this.replace(/^[\s\uFEFF\xA0]+|[\s\uFEFF\xA0]+$/g, "") }); String.prototype.includes || (String.prototype.includes = sc); u.util = {
        throttle: function (a, b) { var c = b !== q ? b : 200, d, e; return function () { var h = this, f = +new Date, g = arguments; d && f < d + c ? (clearTimeout(e), e = setTimeout(function () { d = q; a.apply(h, g) }, c)) : (d = f, a.apply(h, g)) } }, escapeRegex: function (a) { return a.replace(Ec, "\\$1") }, set: function (a) {
          if (l.isPlainObject(a)) return u.util.set(a._); if (null ===
            a) return function () { }; if ("function" === typeof a) return function (c, d, e) { a(c, "set", d, e) }; if ("string" !== typeof a || -1 === a.indexOf(".") && -1 === a.indexOf("[") && -1 === a.indexOf("(")) return function (c, d) { c[a] = d }; var b = function (c, d, e) {
              e = hb(e); var h = e[e.length - 1]; for (var f, g, k = 0, m = e.length - 1; k < m; k++) {
                if ("__proto__" === e[k] || "constructor" === e[k]) throw Error("Cannot set prototype values"); f = e[k].match(Ga); g = e[k].match(qa); if (f) {
                  e[k] = e[k].replace(Ga, ""); c[e[k]] = []; h = e.slice(); h.splice(0, k + 1); f = h.join("."); if (Array.isArray(d)) for (g =
                    0, m = d.length; g < m; g++)h = {}, b(h, d[g], f), c[e[k]].push(h); else c[e[k]] = d; return
                } g && (e[k] = e[k].replace(qa, ""), c = c[e[k]](d)); if (null === c[e[k]] || c[e[k]] === q) c[e[k]] = {}; c = c[e[k]]
              } if (h.match(qa)) c[h.replace(qa, "")](d); else c[h.replace(Ga, "")] = d
            }; return function (c, d) { return b(c, d, a) }
        }, get: function (a) {
          if (l.isPlainObject(a)) { var b = {}; l.each(a, function (d, e) { e && (b[d] = u.util.get(e)) }); return function (d, e, h, f) { var g = b[e] || b._; return g !== q ? g(d, e, h, f) : d } } if (null === a) return function (d) { return d }; if ("function" === typeof a) return function (d,
            e, h, f) { return a(d, e, h, f) }; if ("string" !== typeof a || -1 === a.indexOf(".") && -1 === a.indexOf("[") && -1 === a.indexOf("(")) return function (d, e) { return d[a] }; var c = function (d, e, h) {
              if ("" !== h) {
                var f = hb(h); for (var g = 0, k = f.length; g < k; g++) {
                  h = f[g].match(Ga); var m = f[g].match(qa); if (h) { f[g] = f[g].replace(Ga, ""); "" !== f[g] && (d = d[f[g]]); m = []; f.splice(0, g + 1); f = f.join("."); if (Array.isArray(d)) for (g = 0, k = d.length; g < k; g++)m.push(c(d[g], e, f)); d = h[0].substring(1, h[0].length - 1); d = "" === d ? m : m.join(d); break } else if (m) {
                    f[g] = f[g].replace(qa,
                      ""); d = d[f[g]](); continue
                  } if (null === d || d[f[g]] === q) return q; d = d[f[g]]
                }
              } return d
            }; return function (d, e) { return c(d, e, a) }
        }
      }; var S = function (a, b, c) { a[b] !== q && (a[c] = a[b]) }, Ga = /\[.*?\]$/, qa = /\(\)$/, ma = u.util.get, ha = u.util.set, ob = u.util.escapeRegex, Sa = l("<div>")[0], Bc = Sa.textContent !== q, Cc = /<.*?>/g, mb = u.util.throttle, tc = [], N = Array.prototype, Fc = function (a) {
        var b, c = u.settings, d = l.map(c, function (h, f) { return h.nTable }); if (a) {
          if (a.nTable && a.oApi) return [a]; if (a.nodeName && "table" === a.nodeName.toLowerCase()) {
            var e =
              l.inArray(a, d); return -1 !== e ? [c[e]] : null
          } if (a && "function" === typeof a.settings) return a.settings().toArray(); "string" === typeof a ? b = l(a) : a instanceof l && (b = a)
        } else return []; if (b) return b.map(function (h) { e = l.inArray(this, d); return -1 !== e ? c[e] : null }).toArray()
      }; var B = function (a, b) {
        if (!(this instanceof B)) return new B(a, b); var c = [], d = function (f) { (f = Fc(f)) && c.push.apply(c, f) }; if (Array.isArray(a)) for (var e = 0, h = a.length; e < h; e++)d(a[e]); else d(a); this.context = Oa(c); b && l.merge(this, b); this.selector = {
          rows: null,
          cols: null, opts: null
        }; B.extend(this, this, tc)
      }; u.Api = B; l.extend(B.prototype, {
        any: function () { return 0 !== this.count() }, concat: N.concat, context: [], count: function () { return this.flatten().length }, each: function (a) { for (var b = 0, c = this.length; b < c; b++)a.call(this, this[b], b, this); return this }, eq: function (a) { var b = this.context; return b.length > a ? new B(b[a], this[a]) : null }, filter: function (a) {
          var b = []; if (N.filter) b = N.filter.call(this, a, this); else for (var c = 0, d = this.length; c < d; c++)a.call(this, this[c], c, this) && b.push(this[c]);
          return new B(this.context, b)
        }, flatten: function () { var a = []; return new B(this.context, a.concat.apply(a, this.toArray())) }, join: N.join, indexOf: N.indexOf || function (a, b) { b = b || 0; for (var c = this.length; b < c; b++)if (this[b] === a) return b; return -1 }, iterator: function (a, b, c, d) {
          var e = [], h, f, g = this.context, k, m = this.selector; "string" === typeof a && (d = c, c = b, b = a, a = !1); var n = 0; for (h = g.length; n < h; n++) {
            var p = new B(g[n]); if ("table" === b) { var t = c.call(p, g[n], n); t !== q && e.push(t) } else if ("columns" === b || "rows" === b) t = c.call(p, g[n],
              this[n], n), t !== q && e.push(t); else if ("column" === b || "column-rows" === b || "row" === b || "cell" === b) { var v = this[n]; "column-rows" === b && (k = ab(g[n], m.opts)); var x = 0; for (f = v.length; x < f; x++)t = v[x], t = "cell" === b ? c.call(p, g[n], t.row, t.column, n, x) : c.call(p, g[n], t, n, x, k), t !== q && e.push(t) }
          } return e.length || d ? (a = new B(g, a ? e.concat.apply([], e) : e), b = a.selector, b.rows = m.rows, b.cols = m.cols, b.opts = m.opts, a) : this
        }, lastIndexOf: N.lastIndexOf || function (a, b) { return this.indexOf.apply(this.toArray.reverse(), arguments) }, length: 0,
        map: function (a) { var b = []; if (N.map) b = N.map.call(this, a, this); else for (var c = 0, d = this.length; c < d; c++)b.push(a.call(this, this[c], c)); return new B(this.context, b) }, pluck: function (a) { var b = u.util.get(a); return this.map(function (c) { return b(c) }) }, pop: N.pop, push: N.push, reduce: N.reduce || function (a, b) { return Gb(this, a, b, 0, this.length, 1) }, reduceRight: N.reduceRight || function (a, b) { return Gb(this, a, b, this.length - 1, -1, -1) }, reverse: N.reverse, selector: null, shift: N.shift, slice: function () {
          return new B(this.context,
            this)
        }, sort: N.sort, splice: N.splice, toArray: function () { return N.slice.call(this) }, to$: function () { return l(this) }, toJQuery: function () { return l(this) }, unique: function () { return new B(this.context, Oa(this)) }, unshift: N.unshift
      }); B.extend = function (a, b, c) {
        if (c.length && b && (b instanceof B || b.__dt_wrapper)) {
          var d, e = function (g, k, m) { return function () { var n = k.apply(g, arguments); B.extend(n, n, m.methodExt); return n } }; var h = 0; for (d = c.length; h < d; h++) {
            var f = c[h]; b[f.name] = "function" === f.type ? e(a, f.val, f) : "object" ===
              f.type ? {} : f.val; b[f.name].__dt_wrapper = !0; B.extend(a, b[f.name], f.propExt)
          }
        }
      }; B.register = z = function (a, b) {
        if (Array.isArray(a)) for (var c = 0, d = a.length; c < d; c++)B.register(a[c], b); else {
          d = a.split("."); var e = tc, h; a = 0; for (c = d.length; a < c; a++) {
            var f = (h = -1 !== d[a].indexOf("()")) ? d[a].replace("()", "") : d[a]; a: { var g = 0; for (var k = e.length; g < k; g++)if (e[g].name === f) { g = e[g]; break a } g = null } g || (g = { name: f, val: {}, methodExt: [], propExt: [], type: "object" }, e.push(g)); a === c - 1 ? (g.val = b, g.type = "function" === typeof b ? "function" : l.isPlainObject(b) ?
              "object" : "other") : e = h ? g.methodExt : g.propExt
          }
        }
      }; B.registerPlural = J = function (a, b, c) { B.register(a, c); B.register(b, function () { var d = c.apply(this, arguments); return d === this ? this : d instanceof B ? d.length ? Array.isArray(d[0]) ? new B(d.context, d[0]) : d[0] : q : d }) }; var uc = function (a, b) { if (Array.isArray(a)) return l.map(a, function (d) { return uc(d, b) }); if ("number" === typeof a) return [b[a]]; var c = l.map(b, function (d, e) { return d.nTable }); return l(c).filter(a).map(function (d) { d = l.inArray(this, c); return b[d] }).toArray() };
    z("tables()", function (a) { return a !== q && null !== a ? new B(uc(a, this.context)) : this }); z("table()", function (a) { a = this.tables(a); var b = a.context; return b.length ? new B(b[0]) : a }); J("tables().nodes()", "table().node()", function () { return this.iterator("table", function (a) { return a.nTable }, 1) }); J("tables().body()", "table().body()", function () { return this.iterator("table", function (a) { return a.nTBody }, 1) }); J("tables().header()", "table().header()", function () {
      return this.iterator("table", function (a) { return a.nTHead },
        1)
    }); J("tables().footer()", "table().footer()", function () { return this.iterator("table", function (a) { return a.nTFoot }, 1) }); J("tables().containers()", "table().container()", function () { return this.iterator("table", function (a) { return a.nTableWrapper }, 1) }); z("draw()", function (a) { return this.iterator("table", function (b) { "page" === a ? ja(b) : ("string" === typeof a && (a = "full-hold" === a ? !1 : !0), ka(b, !1 === a)) }) }); z("page()", function (a) { return a === q ? this.page.info().page : this.iterator("table", function (b) { Ta(b, a) }) }); z("page.info()",
      function (a) { if (0 === this.context.length) return q; a = this.context[0]; var b = a._iDisplayStart, c = a.oFeatures.bPaginate ? a._iDisplayLength : -1, d = a.fnRecordsDisplay(), e = -1 === c; return { page: e ? 0 : Math.floor(b / c), pages: e ? 1 : Math.ceil(d / c), start: b, end: a.fnDisplayEnd(), length: c, recordsTotal: a.fnRecordsTotal(), recordsDisplay: d, serverSide: "ssp" === Q(a) } }); z("page.len()", function (a) { return a === q ? 0 !== this.context.length ? this.context[0]._iDisplayLength : q : this.iterator("table", function (b) { pb(b, a) }) }); var vc = function (a, b,
        c) { if (c) { var d = new B(a); d.one("draw", function () { c(d.ajax.json()) }) } if ("ssp" == Q(a)) ka(a, b); else { V(a, !0); var e = a.jqXHR; e && 4 !== e.readyState && e.abort(); Qa(a, [], function (h) { Ma(a); h = za(a, h); for (var f = 0, g = h.length; f < g; f++)ia(a, h[f]); ka(a, b); V(a, !1) }) } }; z("ajax.json()", function () { var a = this.context; if (0 < a.length) return a[0].json }); z("ajax.params()", function () { var a = this.context; if (0 < a.length) return a[0].oAjaxData }); z("ajax.reload()", function (a, b) { return this.iterator("table", function (c) { vc(c, !1 === b, a) }) });
    z("ajax.url()", function (a) { var b = this.context; if (a === q) { if (0 === b.length) return q; b = b[0]; return b.ajax ? l.isPlainObject(b.ajax) ? b.ajax.url : b.ajax : b.sAjaxSource } return this.iterator("table", function (c) { l.isPlainObject(c.ajax) ? c.ajax.url = a : c.ajax = a }) }); z("ajax.url().load()", function (a, b) { return this.iterator("table", function (c) { vc(c, !1 === b, a) }) }); var zb = function (a, b, c, d, e) {
      var h = [], f, g, k; var m = typeof b; b && "string" !== m && "function" !== m && b.length !== q || (b = [b]); m = 0; for (g = b.length; m < g; m++) {
        var n = b[m] && b[m].split &&
          !b[m].match(/[\[\(:]/) ? b[m].split(",") : [b[m]]; var p = 0; for (k = n.length; p < k; p++)(f = c("string" === typeof n[p] ? n[p].trim() : n[p])) && f.length && (h = h.concat(f))
      } a = M.selector[a]; if (a.length) for (m = 0, g = a.length; m < g; m++)h = a[m](d, e, h); return Oa(h)
    }, Ab = function (a) { a || (a = {}); a.filter && a.search === q && (a.search = a.filter); return l.extend({ search: "none", order: "current", page: "all" }, a) }, Bb = function (a) {
      for (var b = 0, c = a.length; b < c; b++)if (0 < a[b].length) return a[0] = a[b], a[0].length = 1, a.length = 1, a.context = [a.context[b]], a; a.length =
        0; return a
    }, ab = function (a, b) {
      var c = [], d = a.aiDisplay; var e = a.aiDisplayMaster; var h = b.search; var f = b.order; b = b.page; if ("ssp" == Q(a)) return "removed" === h ? [] : pa(0, e.length); if ("current" == b) for (f = a._iDisplayStart, a = a.fnDisplayEnd(); f < a; f++)c.push(d[f]); else if ("current" == f || "applied" == f) if ("none" == h) c = e.slice(); else if ("applied" == h) c = d.slice(); else { if ("removed" == h) { var g = {}; f = 0; for (a = d.length; f < a; f++)g[d[f]] = null; c = l.map(e, function (k) { return g.hasOwnProperty(k) ? null : k }) } } else if ("index" == f || "original" ==
        f) for (f = 0, a = a.aoData.length; f < a; f++)"none" == h ? c.push(f) : (e = l.inArray(f, d), (-1 === e && "removed" == h || 0 <= e && "applied" == h) && c.push(f)); return c
    }, Gc = function (a, b, c) {
      var d; return zb("row", b, function (e) {
        var h = nc(e), f = a.aoData; if (null !== h && !c) return [h]; d || (d = ab(a, c)); if (null !== h && -1 !== l.inArray(h, d)) return [h]; if (null === e || e === q || "" === e) return d; if ("function" === typeof e) return l.map(d, function (k) { var m = f[k]; return e(k, m._aData, m.nTr) ? k : null }); if (e.nodeName) {
          h = e._DT_RowIndex; var g = e._DT_CellIndex; if (h !== q) return f[h] &&
            f[h].nTr === e ? [h] : []; if (g) return f[g.row] && f[g.row].nTr === e.parentNode ? [g.row] : []; h = l(e).closest("*[data-dt-row]"); return h.length ? [h.data("dt-row")] : []
        } if ("string" === typeof e && "#" === e.charAt(0) && (h = a.aIds[e.replace(/^#/, "")], h !== q)) return [h.idx]; h = qc(Fa(a.aoData, d, "nTr")); return l(h).filter(e).map(function () { return this._DT_RowIndex }).toArray()
      }, a, c)
    }; z("rows()", function (a, b) {
      a === q ? a = "" : l.isPlainObject(a) && (b = a, a = ""); b = Ab(b); var c = this.iterator("table", function (d) { return Gc(d, a, b) }, 1); c.selector.rows =
        a; c.selector.opts = b; return c
    }); z("rows().nodes()", function () { return this.iterator("row", function (a, b) { return a.aoData[b].nTr || q }, 1) }); z("rows().data()", function () { return this.iterator(!0, "rows", function (a, b) { return Fa(a.aoData, b, "_aData") }, 1) }); J("rows().cache()", "row().cache()", function (a) { return this.iterator("row", function (b, c) { b = b.aoData[c]; return "search" === a ? b._aFilterData : b._aSortData }, 1) }); J("rows().invalidate()", "row().invalidate()", function (a) {
      return this.iterator("row", function (b, c) {
        va(b,
          c, a)
      })
    }); J("rows().indexes()", "row().index()", function () { return this.iterator("row", function (a, b) { return b }, 1) }); J("rows().ids()", "row().id()", function (a) { for (var b = [], c = this.context, d = 0, e = c.length; d < e; d++)for (var h = 0, f = this[d].length; h < f; h++) { var g = c[d].rowIdFn(c[d].aoData[this[d][h]]._aData); b.push((!0 === a ? "#" : "") + g) } return new B(c, b) }); J("rows().remove()", "row().remove()", function () {
      var a = this; this.iterator("row", function (b, c, d) {
        var e = b.aoData, h = e[c], f, g; e.splice(c, 1); var k = 0; for (f = e.length; k <
          f; k++) { var m = e[k]; var n = m.anCells; null !== m.nTr && (m.nTr._DT_RowIndex = k); if (null !== n) for (m = 0, g = n.length; m < g; m++)n[m]._DT_CellIndex.row = k } Na(b.aiDisplayMaster, c); Na(b.aiDisplay, c); Na(a[d], c, !1); 0 < b._iRecordsDisplay && b._iRecordsDisplay--; qb(b); c = b.rowIdFn(h._aData); c !== q && delete b.aIds[c]
      }); this.iterator("table", function (b) { for (var c = 0, d = b.aoData.length; c < d; c++)b.aoData[c].idx = c }); return this
    }); z("rows.add()", function (a) {
      var b = this.iterator("table", function (d) {
        var e, h = []; var f = 0; for (e = a.length; f < e; f++) {
          var g =
            a[f]; g.nodeName && "TR" === g.nodeName.toUpperCase() ? h.push(La(d, g)[0]) : h.push(ia(d, g))
        } return h
      }, 1), c = this.rows(-1); c.pop(); l.merge(c, b); return c
    }); z("row()", function (a, b) { return Bb(this.rows(a, b)) }); z("row().data()", function (a) { var b = this.context; if (a === q) return b.length && this.length ? b[0].aoData[this[0]]._aData : q; var c = b[0].aoData[this[0]]; c._aData = a; Array.isArray(a) && c.nTr && c.nTr.id && ha(b[0].rowId)(a, c.nTr.id); va(b[0], this[0], "data"); return this }); z("row().node()", function () {
      var a = this.context; return a.length &&
        this.length ? a[0].aoData[this[0]].nTr || null : null
    }); z("row.add()", function (a) { a instanceof l && a.length && (a = a[0]); var b = this.iterator("table", function (c) { return a.nodeName && "TR" === a.nodeName.toUpperCase() ? La(c, a)[0] : ia(c, a) }); return this.row(b[0]) }); l(A).on("plugin-init.dt", function (a, b) {
      a = new B(b); a.on("stateSaveParams", function (d, e, h) { d = e.rowIdFn; e = e.aoData; for (var f = [], g = 0; g < e.length; g++)e[g]._detailsShow && f.push("#" + d(e[g]._aData)); h.childRows = f }); var c = a.state.loaded(); c && c.childRows && a.rows(l.map(c.childRows,
        function (d) { return d.replace(/:/g, "\\:") })).every(function () { F(b, null, "requestChild", [this]) })
    }); var Hc = function (a, b, c, d) { var e = [], h = function (f, g) { if (Array.isArray(f) || f instanceof l) for (var k = 0, m = f.length; k < m; k++)h(f[k], g); else f.nodeName && "tr" === f.nodeName.toLowerCase() ? e.push(f) : (k = l("<tr><td></td></tr>").addClass(g), l("td", k).addClass(g).html(f)[0].colSpan = na(a), e.push(k[0])) }; h(c, d); b._details && b._details.detach(); b._details = l(e); b._detailsShow && b._details.insertAfter(b.nTr) }, wc = u.util.throttle(function (a) { Da(a[0]) },
      500), Cb = function (a, b) { var c = a.context; c.length && (a = c[0].aoData[b !== q ? b : a[0]]) && a._details && (a._details.remove(), a._detailsShow = q, a._details = q, l(a.nTr).removeClass("dt-hasChild"), wc(c)) }, xc = function (a, b) { var c = a.context; if (c.length && a.length) { var d = c[0].aoData[a[0]]; d._details && ((d._detailsShow = b) ? (d._details.insertAfter(d.nTr), l(d.nTr).addClass("dt-hasChild")) : (d._details.detach(), l(d.nTr).removeClass("dt-hasChild")), F(c[0], null, "childRow", [b, a.row(a[0])]), Ic(c[0]), wc(c)) } }, Ic = function (a) {
        var b = new B(a),
        c = a.aoData; b.off("draw.dt.DT_details column-sizing.dt.DT_details destroy.dt.DT_details"); 0 < U(c, "_details").length && (b.on("draw.dt.DT_details", function (d, e) { a === e && b.rows({ page: "current" }).eq(0).each(function (h) { h = c[h]; h._detailsShow && h._details.insertAfter(h.nTr) }) }), b.on("column-sizing.dt.DT_details", function (d, e, h, f) { if (a === e) for (e = na(e), h = 0, f = c.length; h < f; h++)d = c[h], d._details && d._details.children("td[colspan]").attr("colspan", e) }), b.on("destroy.dt.DT_details", function (d, e) {
          if (a === e) for (d = 0, e =
            c.length; d < e; d++)c[d]._details && Cb(b, d)
        }))
      }; z("row().child()", function (a, b) { var c = this.context; if (a === q) return c.length && this.length ? c[0].aoData[this[0]]._details : q; !0 === a ? this.child.show() : !1 === a ? Cb(this) : c.length && this.length && Hc(c[0], c[0].aoData[this[0]], a, b); return this }); z(["row().child.show()", "row().child().show()"], function (a) { xc(this, !0); return this }); z(["row().child.hide()", "row().child().hide()"], function () { xc(this, !1); return this }); z(["row().child.remove()", "row().child().remove()"], function () {
        Cb(this);
        return this
      }); z("row().child.isShown()", function () { var a = this.context; return a.length && this.length ? a[0].aoData[this[0]]._detailsShow || !1 : !1 }); var Jc = /^([^:]+):(name|visIdx|visible)$/, yc = function (a, b, c, d, e) { c = []; d = 0; for (var h = e.length; d < h; d++)c.push(T(a, e[d], b)); return c }, Kc = function (a, b, c) {
        var d = a.aoColumns, e = U(d, "sName"), h = U(d, "nTh"); return zb("column", b, function (f) {
          var g = nc(f); if ("" === f) return pa(d.length); if (null !== g) return [0 <= g ? g : d.length + g]; if ("function" === typeof f) {
            var k = ab(a, c); return l.map(d,
              function (p, t) { return f(t, yc(a, t, 0, 0, k), h[t]) ? t : null })
          } var m = "string" === typeof f ? f.match(Jc) : ""; if (m) switch (m[2]) { case "visIdx": case "visible": g = parseInt(m[1], 10); if (0 > g) { var n = l.map(d, function (p, t) { return p.bVisible ? t : null }); return [n[n.length + g]] } return [ta(a, g)]; case "name": return l.map(e, function (p, t) { return p === m[1] ? t : null }); default: return [] }if (f.nodeName && f._DT_CellIndex) return [f._DT_CellIndex.column]; g = l(h).filter(f).map(function () { return l.inArray(this, h) }).toArray(); if (g.length || !f.nodeName) return g;
          g = l(f).closest("*[data-dt-column]"); return g.length ? [g.data("dt-column")] : []
        }, a, c)
      }; z("columns()", function (a, b) { a === q ? a = "" : l.isPlainObject(a) && (b = a, a = ""); b = Ab(b); var c = this.iterator("table", function (d) { return Kc(d, a, b) }, 1); c.selector.cols = a; c.selector.opts = b; return c }); J("columns().header()", "column().header()", function (a, b) { return this.iterator("column", function (c, d) { return c.aoColumns[d].nTh }, 1) }); J("columns().footer()", "column().footer()", function (a, b) {
        return this.iterator("column", function (c,
          d) { return c.aoColumns[d].nTf }, 1)
      }); J("columns().data()", "column().data()", function () { return this.iterator("column-rows", yc, 1) }); J("columns().dataSrc()", "column().dataSrc()", function () { return this.iterator("column", function (a, b) { return a.aoColumns[b].mData }, 1) }); J("columns().cache()", "column().cache()", function (a) { return this.iterator("column-rows", function (b, c, d, e, h) { return Fa(b.aoData, h, "search" === a ? "_aFilterData" : "_aSortData", c) }, 1) }); J("columns().nodes()", "column().nodes()", function () {
        return this.iterator("column-rows",
          function (a, b, c, d, e) { return Fa(a.aoData, e, "anCells", b) }, 1)
      }); J("columns().visible()", "column().visible()", function (a, b) {
        var c = this, d = this.iterator("column", function (e, h) { if (a === q) return e.aoColumns[h].bVisible; var f = e.aoColumns, g = f[h], k = e.aoData, m; if (a !== q && g.bVisible !== a) { if (a) { var n = l.inArray(!0, U(f, "bVisible"), h + 1); f = 0; for (m = k.length; f < m; f++) { var p = k[f].nTr; e = k[f].anCells; p && p.insertBefore(e[h], e[n] || null) } } else l(U(e.aoData, "anCells", h)).detach(); g.bVisible = a } }); a !== q && this.iterator("table", function (e) {
          xa(e,
            e.aoHeader); xa(e, e.aoFooter); e.aiDisplay.length || l(e.nTBody).find("td[colspan]").attr("colspan", na(e)); Da(e); c.iterator("column", function (h, f) { F(h, null, "column-visibility", [h, f, a, b]) }); (b === q || b) && c.columns.adjust()
        }); return d
      }); J("columns().indexes()", "column().index()", function (a) { return this.iterator("column", function (b, c) { return "visible" === a ? ua(b, c) : c }, 1) }); z("columns.adjust()", function () { return this.iterator("table", function (a) { sa(a) }, 1) }); z("column.index()", function (a, b) {
        if (0 !== this.context.length) {
          var c =
            this.context[0]; if ("fromVisible" === a || "toData" === a) return ta(c, b); if ("fromData" === a || "toVisible" === a) return ua(c, b)
        }
      }); z("column()", function (a, b) { return Bb(this.columns(a, b)) }); var Lc = function (a, b, c) {
        var d = a.aoData, e = ab(a, c), h = qc(Fa(d, e, "anCells")), f = l(rc([], h)), g, k = a.aoColumns.length, m, n, p, t, v, x; return zb("cell", b, function (w) {
          var r = "function" === typeof w; if (null === w || w === q || r) {
            m = []; n = 0; for (p = e.length; n < p; n++)for (g = e[n], t = 0; t < k; t++)v = { row: g, column: t }, r ? (x = d[g], w(v, T(a, g, t), x.anCells ? x.anCells[t] : null) &&
              m.push(v)) : m.push(v); return m
          } if (l.isPlainObject(w)) return w.column !== q && w.row !== q && -1 !== l.inArray(w.row, e) ? [w] : []; r = f.filter(w).map(function (C, G) { return { row: G._DT_CellIndex.row, column: G._DT_CellIndex.column } }).toArray(); if (r.length || !w.nodeName) return r; x = l(w).closest("*[data-dt-row]"); return x.length ? [{ row: x.data("dt-row"), column: x.data("dt-column") }] : []
        }, a, c)
      }; z("cells()", function (a, b, c) {
        l.isPlainObject(a) && (a.row === q ? (c = a, a = null) : (c = b, b = null)); l.isPlainObject(b) && (c = b, b = null); if (null === b || b ===
          q) return this.iterator("table", function (n) { return Lc(n, a, Ab(c)) }); var d = c ? { page: c.page, order: c.order, search: c.search } : {}, e = this.columns(b, d), h = this.rows(a, d), f, g, k, m; d = this.iterator("table", function (n, p) { n = []; f = 0; for (g = h[p].length; f < g; f++)for (k = 0, m = e[p].length; k < m; k++)n.push({ row: h[p][f], column: e[p][k] }); return n }, 1); d = c && c.selected ? this.cells(d, c) : d; l.extend(d.selector, { cols: b, rows: a, opts: c }); return d
      }); J("cells().nodes()", "cell().node()", function () {
        return this.iterator("cell", function (a, b, c) {
          return (a =
            a.aoData[b]) && a.anCells ? a.anCells[c] : q
        }, 1)
      }); z("cells().data()", function () { return this.iterator("cell", function (a, b, c) { return T(a, b, c) }, 1) }); J("cells().cache()", "cell().cache()", function (a) { a = "search" === a ? "_aFilterData" : "_aSortData"; return this.iterator("cell", function (b, c, d) { return b.aoData[c][a][d] }, 1) }); J("cells().render()", "cell().render()", function (a) { return this.iterator("cell", function (b, c, d) { return T(b, c, d, a) }, 1) }); J("cells().indexes()", "cell().index()", function () {
        return this.iterator("cell",
          function (a, b, c) { return { row: b, column: c, columnVisible: ua(a, c) } }, 1)
      }); J("cells().invalidate()", "cell().invalidate()", function (a) { return this.iterator("cell", function (b, c, d) { va(b, c, a, d) }) }); z("cell()", function (a, b, c) { return Bb(this.cells(a, b, c)) }); z("cell().data()", function (a) { var b = this.context, c = this[0]; if (a === q) return b.length && c.length ? T(b[0], c[0].row, c[0].column) : q; Ib(b[0], c[0].row, c[0].column, a); va(b[0], c[0].row, "data", c[0].column); return this }); z("order()", function (a, b) {
        var c = this.context; if (a ===
          q) return 0 !== c.length ? c[0].aaSorting : q; "number" === typeof a ? a = [[a, b]] : a.length && !Array.isArray(a[0]) && (a = Array.prototype.slice.call(arguments)); return this.iterator("table", function (d) { d.aaSorting = a.slice() })
      }); z("order.listener()", function (a, b, c) { return this.iterator("table", function (d) { kb(d, a, b, c) }) }); z("order.fixed()", function (a) { if (!a) { var b = this.context; b = b.length ? b[0].aaSortingFixed : q; return Array.isArray(b) ? { pre: b } : b } return this.iterator("table", function (c) { c.aaSortingFixed = l.extend(!0, {}, a) }) });
    z(["columns().order()", "column().order()"], function (a) { var b = this; return this.iterator("table", function (c, d) { var e = []; l.each(b[d], function (h, f) { e.push([f, a]) }); c.aaSorting = e }) }); z("search()", function (a, b, c, d) { var e = this.context; return a === q ? 0 !== e.length ? e[0].oPreviousSearch.sSearch : q : this.iterator("table", function (h) { h.oFeatures.bFilter && ya(h, l.extend({}, h.oPreviousSearch, { sSearch: a + "", bRegex: null === b ? !1 : b, bSmart: null === c ? !0 : c, bCaseInsensitive: null === d ? !0 : d }), 1) }) }); J("columns().search()", "column().search()",
      function (a, b, c, d) { return this.iterator("column", function (e, h) { var f = e.aoPreSearchCols; if (a === q) return f[h].sSearch; e.oFeatures.bFilter && (l.extend(f[h], { sSearch: a + "", bRegex: null === b ? !1 : b, bSmart: null === c ? !0 : c, bCaseInsensitive: null === d ? !0 : d }), ya(e, e.oPreviousSearch, 1)) }) }); z("state()", function () { return this.context.length ? this.context[0].oSavedState : null }); z("state.clear()", function () { return this.iterator("table", function (a) { a.fnStateSaveCallback.call(a.oInstance, a, {}) }) }); z("state.loaded()", function () {
        return this.context.length ?
          this.context[0].oLoadedState : null
      }); z("state.save()", function () { return this.iterator("table", function (a) { Da(a) }) }); u.versionCheck = u.fnVersionCheck = function (a) { var b = u.version.split("."); a = a.split("."); for (var c, d, e = 0, h = a.length; e < h; e++)if (c = parseInt(b[e], 10) || 0, d = parseInt(a[e], 10) || 0, c !== d) return c > d; return !0 }; u.isDataTable = u.fnIsDataTable = function (a) {
        var b = l(a).get(0), c = !1; if (a instanceof u.Api) return !0; l.each(u.settings, function (d, e) {
          d = e.nScrollHead ? l("table", e.nScrollHead)[0] : null; var h = e.nScrollFoot ?
            l("table", e.nScrollFoot)[0] : null; if (e.nTable === b || d === b || h === b) c = !0
        }); return c
      }; u.tables = u.fnTables = function (a) { var b = !1; l.isPlainObject(a) && (b = a.api, a = a.visible); var c = l.map(u.settings, function (d) { if (!a || a && l(d.nTable).is(":visible")) return d.nTable }); return b ? new B(c) : c }; u.camelToHungarian = P; z("$()", function (a, b) { b = this.rows(b).nodes(); b = l(b); return l([].concat(b.filter(a).toArray(), b.find(a).toArray())) }); l.each(["on", "one", "off"], function (a, b) {
        z(b + "()", function () {
          var c = Array.prototype.slice.call(arguments);
          c[0] = l.map(c[0].split(/\s/), function (e) { return e.match(/\.dt\b/) ? e : e + ".dt" }).join(" "); var d = l(this.tables().nodes()); d[b].apply(d, c); return this
        })
      }); z("clear()", function () { return this.iterator("table", function (a) { Ma(a) }) }); z("settings()", function () { return new B(this.context, this.context) }); z("init()", function () { var a = this.context; return a.length ? a[0].oInit : null }); z("data()", function () { return this.iterator("table", function (a) { return U(a.aoData, "_aData") }).flatten() }); z("destroy()", function (a) {
        a = a ||
        !1; return this.iterator("table", function (b) {
          var c = b.oClasses, d = b.nTable, e = b.nTBody, h = b.nTHead, f = b.nTFoot, g = l(d); e = l(e); var k = l(b.nTableWrapper), m = l.map(b.aoData, function (p) { return p.nTr }), n; b.bDestroying = !0; F(b, "aoDestroyCallback", "destroy", [b]); a || (new B(b)).columns().visible(!0); k.off(".DT").find(":not(tbody *)").off(".DT"); l(y).off(".DT-" + b.sInstance); d != h.parentNode && (g.children("thead").detach(), g.append(h)); f && d != f.parentNode && (g.children("tfoot").detach(), g.append(f)); b.aaSorting = []; b.aaSortingFixed =
            []; Va(b); l(m).removeClass(b.asStripeClasses.join(" ")); l("th, td", h).removeClass(c.sSortable + " " + c.sSortableAsc + " " + c.sSortableDesc + " " + c.sSortableNone); e.children().detach(); e.append(m); h = b.nTableWrapper.parentNode; f = a ? "remove" : "detach"; g[f](); k[f](); !a && h && (h.insertBefore(d, b.nTableReinsertBefore), g.css("width", b.sDestroyWidth).removeClass(c.sTable), (n = b.asDestroyStripes.length) && e.children().each(function (p) { l(this).addClass(b.asDestroyStripes[p % n]) })); c = l.inArray(b, u.settings); -1 !== c && u.settings.splice(c,
              1)
        })
      }); l.each(["column", "row", "cell"], function (a, b) { z(b + "s().every()", function (c) { var d = this.selector.opts, e = this; return this.iterator(b, function (h, f, g, k, m) { c.call(e[b](f, "cell" === b ? g : d, "cell" === b ? d : q), f, g, k, m) }) }) }); z("i18n()", function (a, b, c) { var d = this.context[0]; a = ma(a)(d.oLanguage); a === q && (a = b); c !== q && l.isPlainObject(a) && (a = a[c] !== q ? a[c] : a._); return a.replace("%d", c) }); u.version = "1.12.1"; u.settings = []; u.models = {}; u.models.oSearch = { bCaseInsensitive: !0, sSearch: "", bRegex: !1, bSmart: !0, "return": !1 };
    u.models.oRow = { nTr: null, anCells: null, _aData: [], _aSortData: null, _aFilterData: null, _sFilterRow: null, _sRowStripe: "", src: null, idx: -1 }; u.models.oColumn = {
      idx: null, aDataSort: null, asSorting: null, bSearchable: null, bSortable: null, bVisible: null, _sManualType: null, _bAttrSrc: !1, fnCreatedCell: null, fnGetData: null, fnSetData: null, mData: null, mRender: null, nTh: null, nTf: null, sClass: null, sContentPadding: null, sDefaultContent: null, sName: null, sSortDataType: "std", sSortingClass: null, sSortingClassJUI: null, sTitle: null, sType: null,
      sWidth: null, sWidthOrig: null
    }; u.defaults = {
      aaData: null, aaSorting: [[0, "asc"]], aaSortingFixed: [], ajax: null, aLengthMenu: [10, 25, 50, 100], aoColumns: null, aoColumnDefs: null, aoSearchCols: [], asStripeClasses: null, bAutoWidth: !0, bDeferRender: !1, bDestroy: !1, bFilter: !0, bInfo: !0, bLengthChange: !0, bPaginate: !0, bProcessing: !1, bRetrieve: !1, bScrollCollapse: !1, bServerSide: !1, bSort: !0, bSortMulti: !0, bSortCellsTop: !1, bSortClasses: !0, bStateSave: !1, fnCreatedRow: null, fnDrawCallback: null, fnFooterCallback: null, fnFormatNumber: function (a) {
        return a.toString().replace(/\B(?=(\d{3})+(?!\d))/g,
          this.oLanguage.sThousands)
      }, fnHeaderCallback: null, fnInfoCallback: null, fnInitComplete: null, fnPreDrawCallback: null, fnRowCallback: null, fnServerData: null, fnServerParams: null, fnStateLoadCallback: function (a) { try { return JSON.parse((-1 === a.iStateDuration ? sessionStorage : localStorage).getItem("DataTables_" + a.sInstance + "_" + location.pathname)) } catch (b) { return {} } }, fnStateLoadParams: null, fnStateLoaded: null, fnStateSaveCallback: function (a, b) {
        try {
          (-1 === a.iStateDuration ? sessionStorage : localStorage).setItem("DataTables_" +
            a.sInstance + "_" + location.pathname, JSON.stringify(b))
        } catch (c) { }
      }, fnStateSaveParams: null, iStateDuration: 7200, iDeferLoading: null, iDisplayLength: 10, iDisplayStart: 0, iTabIndex: 0, oClasses: {}, oLanguage: {
        oAria: { sSortAscending: ": activate to sort column ascending", sSortDescending: ": activate to sort column descending" }, oPaginate: { sFirst: "First", sLast: "Last", sNext: "Next", sPrevious: "Previous" }, sEmptyTable: "No data available in table", sInfo: "Showing _START_ to _END_ of _TOTAL_ entries", sInfoEmpty: "Showing 0 to 0 of 0 entries",
        sInfoFiltered: "(filtered from _MAX_ total entries)", sInfoPostFix: "", sDecimal: "", sThousands: ",", sLengthMenu: "Show _MENU_ entries", sLoadingRecords: "Loading...", sProcessing: "", sSearch: "Search:", sSearchPlaceholder: "", sUrl: "", sZeroRecords: "No matching records found"
      }, oSearch: l.extend({}, u.models.oSearch), sAjaxDataProp: "data", sAjaxSource: null, sDom: "lfrtip", searchDelay: null, sPaginationType: "simple_numbers", sScrollX: "", sScrollXInner: "", sScrollY: "", sServerMethod: "GET", renderer: null, rowId: "DT_RowId"
    }; E(u.defaults);
    u.defaults.column = { aDataSort: null, iDataSort: -1, asSorting: ["asc", "desc"], bSearchable: !0, bSortable: !0, bVisible: !0, fnCreatedCell: null, mData: null, mRender: null, sCellType: "td", sClass: "", sContentPadding: "", sDefaultContent: null, sName: "", sSortDataType: "std", sTitle: null, sType: null, sWidth: null }; E(u.defaults.column); u.models.oSettings = {
      oFeatures: {
        bAutoWidth: null, bDeferRender: null, bFilter: null, bInfo: null, bLengthChange: null, bPaginate: null, bProcessing: null, bServerSide: null, bSort: null, bSortMulti: null, bSortClasses: null,
        bStateSave: null
      }, oScroll: { bCollapse: null, iBarWidth: 0, sX: null, sXInner: null, sY: null }, oLanguage: { fnInfoCallback: null }, oBrowser: { bScrollOversize: !1, bScrollbarLeft: !1, bBounding: !1, barWidth: 0 }, ajax: null, aanFeatures: [], aoData: [], aiDisplay: [], aiDisplayMaster: [], aIds: {}, aoColumns: [], aoHeader: [], aoFooter: [], oPreviousSearch: {}, aoPreSearchCols: [], aaSorting: null, aaSortingFixed: [], asStripeClasses: null, asDestroyStripes: [], sDestroyWidth: 0, aoRowCallback: [], aoHeaderCallback: [], aoFooterCallback: [], aoDrawCallback: [],
      aoRowCreatedCallback: [], aoPreDrawCallback: [], aoInitComplete: [], aoStateSaveParams: [], aoStateLoadParams: [], aoStateLoaded: [], sTableId: "", nTable: null, nTHead: null, nTFoot: null, nTBody: null, nTableWrapper: null, bDeferLoading: !1, bInitialised: !1, aoOpenRows: [], sDom: null, searchDelay: null, sPaginationType: "two_button", iStateDuration: 0, aoStateSave: [], aoStateLoad: [], oSavedState: null, oLoadedState: null, sAjaxSource: null, sAjaxDataProp: null, jqXHR: null, json: q, oAjaxData: q, fnServerData: null, aoServerParams: [], sServerMethod: null,
      fnFormatNumber: null, aLengthMenu: null, iDraw: 0, bDrawing: !1, iDrawError: -1, _iDisplayLength: 10, _iDisplayStart: 0, _iRecordsTotal: 0, _iRecordsDisplay: 0, oClasses: {}, bFiltered: !1, bSorted: !1, bSortCellsTop: null, oInit: null, aoDestroyCallback: [], fnRecordsTotal: function () { return "ssp" == Q(this) ? 1 * this._iRecordsTotal : this.aiDisplayMaster.length }, fnRecordsDisplay: function () { return "ssp" == Q(this) ? 1 * this._iRecordsDisplay : this.aiDisplay.length }, fnDisplayEnd: function () {
        var a = this._iDisplayLength, b = this._iDisplayStart, c = b +
          a, d = this.aiDisplay.length, e = this.oFeatures, h = e.bPaginate; return e.bServerSide ? !1 === h || -1 === a ? b + d : Math.min(b + a, this._iRecordsDisplay) : !h || c > d || -1 === a ? d : c
      }, oInstance: null, sInstance: null, iTabIndex: 0, nScrollHead: null, nScrollFoot: null, aLastSort: [], oPlugins: {}, rowIdFn: null, rowId: null
    }; u.ext = M = {
      buttons: {}, classes: {}, builder: "-source-", errMode: "alert", feature: [], search: [], selector: { cell: [], column: [], row: [] }, internal: {}, legacy: { ajax: null }, pager: {}, renderer: { pageButton: {}, header: {} }, order: {}, type: {
        detect: [],
        search: {}, order: {}
      }, _unique: 0, fnVersionCheck: u.fnVersionCheck, iApiIndex: 0, oJUIClasses: {}, sVersion: u.version
    }; l.extend(M, { afnFiltering: M.search, aTypes: M.type.detect, ofnSearch: M.type.search, oSort: M.type.order, afnSortData: M.order, aoFeatures: M.feature, oApi: M.internal, oStdClasses: M.classes, oPagination: M.pager }); l.extend(u.ext.classes, {
      sTable: "dataTable", sNoFooter: "no-footer", sPageButton: "paginate_button", sPageButtonActive: "current", sPageButtonDisabled: "disabled", sStripeOdd: "odd", sStripeEven: "even", sRowEmpty: "dataTables_empty",
      sWrapper: "dataTables_wrapper", sFilter: "dataTables_filter", sInfo: "dataTables_info", sPaging: "dataTables_paginate paging_", sLength: "dataTables_length", sProcessing: "dataTables_processing", sSortAsc: "sorting_asc", sSortDesc: "sorting_desc", sSortable: "sorting", sSortableAsc: "sorting_desc_disabled", sSortableDesc: "sorting_asc_disabled", sSortableNone: "sorting_disabled", sSortColumn: "sorting_", sFilterInput: "", sLengthSelect: "", sScrollWrapper: "dataTables_scroll", sScrollHead: "dataTables_scrollHead", sScrollHeadInner: "dataTables_scrollHeadInner",
      sScrollBody: "dataTables_scrollBody", sScrollFoot: "dataTables_scrollFoot", sScrollFootInner: "dataTables_scrollFootInner", sHeaderTH: "", sFooterTH: "", sSortJUIAsc: "", sSortJUIDesc: "", sSortJUI: "", sSortJUIAscAllowed: "", sSortJUIDescAllowed: "", sSortJUIWrapper: "", sSortIcon: "", sJUIHeader: "", sJUIFooter: ""
    }); var ic = u.ext.pager; l.extend(ic, {
      simple: function (a, b) { return ["previous", "next"] }, full: function (a, b) { return ["first", "previous", "next", "last"] }, numbers: function (a, b) { return [Ea(a, b)] }, simple_numbers: function (a, b) {
        return ["previous",
          Ea(a, b), "next"]
      }, full_numbers: function (a, b) { return ["first", "previous", Ea(a, b), "next", "last"] }, first_last_numbers: function (a, b) { return ["first", Ea(a, b), "last"] }, _numbers: Ea, numbers_length: 7
    }); l.extend(!0, u.ext.renderer, {
      pageButton: {
        _: function (a, b, c, d, e, h) {
          var f = a.oClasses, g = a.oLanguage.oPaginate, k = a.oLanguage.oAria.paginate || {}, m, n, p = 0, t = function (x, w) {
            var r, C = f.sPageButtonDisabled, G = function (I) { Ta(a, I.data.action, !0) }; var ba = 0; for (r = w.length; ba < r; ba++) {
              var L = w[ba]; if (Array.isArray(L)) {
                var O = l("<" + (L.DT_el ||
                  "div") + "/>").appendTo(x); t(O, L)
              } else {
                m = null; n = L; O = a.iTabIndex; switch (L) { case "ellipsis": x.append('<span class="ellipsis">&#x2026;</span>'); break; case "first": m = g.sFirst; 0 === e && (O = -1, n += " " + C); break; case "previous": m = g.sPrevious; 0 === e && (O = -1, n += " " + C); break; case "next": m = g.sNext; if (0 === h || e === h - 1) O = -1, n += " " + C; break; case "last": m = g.sLast; if (0 === h || e === h - 1) O = -1, n += " " + C; break; default: m = a.fnFormatNumber(L + 1), n = e === L ? f.sPageButtonActive : "" }null !== m && (O = l("<a>", {
                  "class": f.sPageButton + " " + n, "aria-controls": a.sTableId,
                  "aria-label": k[L], "data-dt-idx": p, tabindex: O, id: 0 === c && "string" === typeof L ? a.sTableId + "_" + L : null
                }).html(m).appendTo(x), sb(O, { action: L }, G), p++)
              }
            }
          }; try { var v = l(b).find(A.activeElement).data("dt-idx") } catch (x) { } t(l(b).empty(), d); v !== q && l(b).find("[data-dt-idx=" + v + "]").trigger("focus")
        }
      }
    }); l.extend(u.ext.type.detect, [function (a, b) { b = b.oLanguage.sDecimal; return yb(a, b) ? "num" + b : null }, function (a, b) {
      if (a && !(a instanceof Date) && !Dc.test(a)) return null; b = Date.parse(a); return null !== b && !isNaN(b) || aa(a) ? "date" :
        null
    }, function (a, b) { b = b.oLanguage.sDecimal; return yb(a, b, !0) ? "num-fmt" + b : null }, function (a, b) { b = b.oLanguage.sDecimal; return pc(a, b) ? "html-num" + b : null }, function (a, b) { b = b.oLanguage.sDecimal; return pc(a, b, !0) ? "html-num-fmt" + b : null }, function (a, b) { return aa(a) || "string" === typeof a && -1 !== a.indexOf("<") ? "html" : null }]); l.extend(u.ext.type.search, {
      html: function (a) { return aa(a) ? a : "string" === typeof a ? a.replace(mc, " ").replace(Ya, "") : "" }, string: function (a) {
        return aa(a) ? a : "string" === typeof a ? a.replace(mc, " ") :
          a
      }
    }); var Xa = function (a, b, c, d) { if (0 !== a && (!a || "-" === a)) return -Infinity; b && (a = oc(a, b)); a.replace && (c && (a = a.replace(c, "")), d && (a = a.replace(d, ""))); return 1 * a }; l.extend(M.type.order, {
      "date-pre": function (a) { a = Date.parse(a); return isNaN(a) ? -Infinity : a }, "html-pre": function (a) { return aa(a) ? "" : a.replace ? a.replace(/<.*?>/g, "").toLowerCase() : a + "" }, "string-pre": function (a) { return aa(a) ? "" : "string" === typeof a ? a.toLowerCase() : a.toString ? a.toString() : "" }, "string-asc": function (a, b) { return a < b ? -1 : a > b ? 1 : 0 }, "string-desc": function (a,
        b) { return a < b ? 1 : a > b ? -1 : 0 }
    }); bb(""); l.extend(!0, u.ext.renderer, {
      header: {
        _: function (a, b, c, d) { l(a.nTable).on("order.dt.DT", function (e, h, f, g) { a === h && (e = c.idx, b.removeClass(d.sSortAsc + " " + d.sSortDesc).addClass("asc" == g[e] ? d.sSortAsc : "desc" == g[e] ? d.sSortDesc : c.sSortingClass)) }) }, jqueryui: function (a, b, c, d) {
          l("<div/>").addClass(d.sSortJUIWrapper).append(b.contents()).append(l("<span/>").addClass(d.sSortIcon + " " + c.sSortingClassJUI)).appendTo(b); l(a.nTable).on("order.dt.DT", function (e, h, f, g) {
            a === h && (e = c.idx,
              b.removeClass(d.sSortAsc + " " + d.sSortDesc).addClass("asc" == g[e] ? d.sSortAsc : "desc" == g[e] ? d.sSortDesc : c.sSortingClass), b.find("span." + d.sSortIcon).removeClass(d.sSortJUIAsc + " " + d.sSortJUIDesc + " " + d.sSortJUI + " " + d.sSortJUIAscAllowed + " " + d.sSortJUIDescAllowed).addClass("asc" == g[e] ? d.sSortJUIAsc : "desc" == g[e] ? d.sSortJUIDesc : c.sSortingClassJUI))
          })
        }
      }
    }); var $a = function (a) {
      Array.isArray(a) && (a = a.join(",")); return "string" === typeof a ? a.replace(/&/g, "&amp;").replace(/</g, "&lt;").replace(/>/g, "&gt;").replace(/"/g,
        "&quot;") : a
    }, kc = !1, zc = ",", Ac = "."; if (Intl) try { for (var Ha = (new Intl.NumberFormat).formatToParts(100000.1), ra = 0; ra < Ha.length; ra++)"group" === Ha[ra].type ? zc = Ha[ra].value : "decimal" === Ha[ra].type && (Ac = Ha[ra].value) } catch (a) { } u.datetime = function (a, b) { var c = "datetime-detect-" + a; b || (b = "en"); u.ext.type.order[c] || (u.ext.type.detect.unshift(function (d) { var e = Za(d, a, b); return "" === d || e ? c : !1 }), u.ext.type.order[c + "-pre"] = function (d) { return Za(d, a, b) || 0 }) }; u.render = {
      date: wb("toLocaleDateString"), datetime: wb("toLocaleString"),
      time: wb("toLocaleTimeString"), number: function (a, b, c, d, e) { if (null === a || a === q) a = zc; if (null === b || b === q) b = Ac; return { display: function (h) { if ("number" !== typeof h && "string" !== typeof h || "" === h || null === h) return h; var f = 0 > h ? "-" : "", g = parseFloat(h); if (isNaN(g)) return $a(h); g = g.toFixed(c); h = Math.abs(g); g = parseInt(h, 10); h = c ? b + (h - g).toFixed(c).substring(2) : ""; 0 === g && 0 === parseFloat(h) && (f = ""); return f + (d || "") + g.toString().replace(/\B(?=(\d{3})+(?!\d))/g, a) + h + (e || "") } } }, text: function () { return { display: $a, filter: $a } }
    };
    l.extend(u.ext.internal, {
      _fnExternApiFunc: lc, _fnBuildAjax: Qa, _fnAjaxUpdate: Kb, _fnAjaxParameters: Tb, _fnAjaxUpdateDraw: Ub, _fnAjaxDataSrc: za, _fnAddColumn: cb, _fnColumnOptions: Ia, _fnAdjustColumnSizing: sa, _fnVisibleToColumnIndex: ta, _fnColumnIndexToVisible: ua, _fnVisbleColumns: na, _fnGetColumns: Ka, _fnColumnTypes: eb, _fnApplyColumnDefs: Hb, _fnHungarianMap: E, _fnCamelToHungarian: P, _fnLanguageCompat: la, _fnBrowserDetect: Fb, _fnAddData: ia, _fnAddTr: La, _fnNodeToDataIndex: function (a, b) {
        return b._DT_RowIndex !== q ? b._DT_RowIndex :
          null
      }, _fnNodeToColumnIndex: function (a, b, c) { return l.inArray(c, a.aoData[b].anCells) }, _fnGetCellData: T, _fnSetCellData: Ib, _fnSplitObjNotation: hb, _fnGetObjectDataFn: ma, _fnSetObjectDataFn: ha, _fnGetDataMaster: ib, _fnClearTable: Ma, _fnDeleteIndex: Na, _fnInvalidate: va, _fnGetRowElements: gb, _fnCreateTr: fb, _fnBuildHead: Jb, _fnDrawHead: xa, _fnDraw: ja, _fnReDraw: ka, _fnAddOptionsHtml: Mb, _fnDetectHeader: wa, _fnGetUniqueThs: Pa, _fnFeatureHtmlFilter: Ob, _fnFilterComplete: ya, _fnFilterCustom: Xb, _fnFilterColumn: Wb, _fnFilter: Vb,
      _fnFilterCreateSearch: nb, _fnEscapeRegex: ob, _fnFilterData: Yb, _fnFeatureHtmlInfo: Rb, _fnUpdateInfo: ac, _fnInfoMacros: bc, _fnInitialise: Aa, _fnInitComplete: Ra, _fnLengthChange: pb, _fnFeatureHtmlLength: Nb, _fnFeatureHtmlPaginate: Sb, _fnPageChange: Ta, _fnFeatureHtmlProcessing: Pb, _fnProcessingDisplay: V, _fnFeatureHtmlTable: Qb, _fnScrollDraw: Ja, _fnApplyToChildren: da, _fnCalculateColumnWidths: db, _fnThrottle: mb, _fnConvertToWidth: cc, _fnGetWidestNode: dc, _fnGetMaxLenString: ec, _fnStringToCss: K, _fnSortFlatten: oa, _fnSort: Lb,
      _fnSortAria: gc, _fnSortListener: rb, _fnSortAttachListener: kb, _fnSortingClasses: Va, _fnSortData: fc, _fnSaveState: Da, _fnLoadState: hc, _fnImplementState: tb, _fnSettingsFromNode: Wa, _fnLog: ea, _fnMap: Y, _fnBindAction: sb, _fnCallbackReg: R, _fnCallbackFire: F, _fnLengthOverflow: qb, _fnRenderer: lb, _fnDataSource: Q, _fnRowAttributes: jb, _fnExtend: ub, _fnCalculateEnd: function () { }
    }); l.fn.dataTable = u; u.$ = l; l.fn.dataTableSettings = u.settings; l.fn.dataTableExt = u.ext; l.fn.DataTable = function (a) { return l(this).dataTable(a).api() };
    l.each(u, function (a, b) { l.fn.DataTable[a] = b }); return u
  });
</script>
<style type="text/css">
  @keyframes dtb-spinner {
    100% {
      transform: rotate(360deg)
    }
  }

  @-o-keyframes dtb-spinner {
    100% {
      -o-transform: rotate(360deg);
      transform: rotate(360deg)
    }
  }

  @-ms-keyframes dtb-spinner {
    100% {
      -ms-transform: rotate(360deg);
      transform: rotate(360deg)
    }
  }

  @-webkit-keyframes dtb-spinner {
    100% {
      -webkit-transform: rotate(360deg);
      transform: rotate(360deg)
    }
  }

  @-moz-keyframes dtb-spinner {
    100% {
      -moz-transform: rotate(360deg);
      transform: rotate(360deg)
    }
  }

  div.dataTables_wrapper {
    position: relative
  }

  div.dt-buttons {
    position: initial
  }

  div.dt-button-info {
    position: fixed;
    top: 50%;
    left: 50%;
    width: 400px;
    margin-top: -100px;
    margin-left: -200px;
    background-color: white;
    border: 2px solid #111;
    box-shadow: 3px 4px 10px 1px rgba(0, 0, 0, 0.3);
    border-radius: 3px;
    text-align: center;
    z-index: 21
  }

  div.dt-button-info h2 {
    padding: .5em;
    margin: 0;
    font-weight: normal;
    border-bottom: 1px solid #ddd;
    background-color: #f3f3f3
  }

  div.dt-button-info>div {
    padding: 1em
  }

  div.dtb-popover-close {
    position: absolute;
    top: 10px;
    right: 10px;
    width: 22px;
    height: 22px;
    border: 1px solid #eaeaea;
    background-color: #f9f9f9;
    text-align: center;
    border-radius: 3px;
    cursor: pointer;
    z-index: 2003
  }

  button.dtb-hide-drop {
    display: none !important
  }

  div.dt-button-collection-title {
    text-align: center;
    padding: .3em 0 .5em;
    margin-left: .5em;
    margin-right: .5em;
    font-size: .9em
  }

  div.dt-button-collection-title:empty {
    display: none
  }

  span.dt-button-spacer {
    display: inline-block;
    margin: .5em;
    white-space: nowrap
  }

  span.dt-button-spacer.bar {
    border-left: 1px solid rgba(0, 0, 0, 0.3);
    vertical-align: middle;
    padding-left: .5em
  }

  span.dt-button-spacer.bar:empty {
    height: 1em;
    width: 1px;
    padding-left: 0
  }

  div.dt-button-collection span.dt-button-spacer {
    width: 100%;
    font-size: .9em;
    text-align: center;
    margin: .5em 0
  }

  div.dt-button-collection span.dt-button-spacer:empty {
    height: 0;
    width: 100%
  }

  div.dt-button-collection span.dt-button-spacer.bar {
    border-left: none;
    border-bottom: 1px solid rgba(0, 0, 0, 0.3);
    padding-left: 0
  }

  button.dt-button,
  div.dt-button,
  a.dt-button,
  input.dt-button {
    position: relative;
    display: inline-block;
    box-sizing: border-box;
    margin-left: .167em;
    margin-right: .167em;
    margin-bottom: .333em;
    padding: .5em 1em;
    border: 1px solid rgba(0, 0, 0, 0.3);
    border-radius: 2px;
    cursor: pointer;
    font-size: .88em;
    line-height: 1.6em;
    color: black;
    white-space: nowrap;
    overflow: hidden;
    background-color: rgba(0, 0, 0, 0.1);
    background: linear-gradient(to bottom, rgba(230, 230, 230, 0.1) 0%, rgba(0, 0, 0, 0.1) 100%);
    filter: progid:DXImageTransform.Microsoft.gradient(GradientType=0, StartColorStr="rgba(230, 230, 230, 0.1)", EndColorStr="rgba(0, 0, 0, 0.1)");
    -webkit-user-select: none;
    -moz-user-select: none;
    -ms-user-select: none;
    user-select: none;
    text-decoration: none;
    outline: none;
    text-overflow: ellipsis
  }

  button.dt-button:first-child,
  div.dt-button:first-child,
  a.dt-button:first-child,
  input.dt-button:first-child {
    margin-left: 0
  }

  button.dt-button.disabled,
  div.dt-button.disabled,
  a.dt-button.disabled,
  input.dt-button.disabled {
    cursor: default;
    opacity: .4
  }

  button.dt-button:active:not(.disabled),
  div.dt-button:active:not(.disabled),
  a.dt-button:active:not(.disabled),
  input.dt-button:active:not(.disabled) {
    background-color: rgba(0, 0, 0, 0.1);
    background: linear-gradient(to bottom, rgba(179, 179, 179, 0.1) 0%, rgba(0, 0, 0, 0.1) 100%);
    filter: progid:DXImageTransform.Microsoft.gradient(GradientType=0, StartColorStr="rgba(179, 179, 179, 0.1)", EndColorStr="rgba(0, 0, 0, 0.1)");
    box-shadow: inset 1px 1px 3px #999
  }

  button.dt-button:active:not(.disabled):hover:not(.disabled),
  div.dt-button:active:not(.disabled):hover:not(.disabled),
  a.dt-button:active:not(.disabled):hover:not(.disabled),
  input.dt-button:active:not(.disabled):hover:not(.disabled) {
    box-shadow: inset 1px 1px 3px #999;
    background-color: rgba(0, 0, 0, 0.1);
    background: linear-gradient(to bottom, rgba(128, 128, 128, 0.1) 0%, rgba(0, 0, 0, 0.1) 100%);
    filter: progid:DXImageTransform.Microsoft.gradient(GradientType=0, StartColorStr="rgba(128, 128, 128, 0.1)", EndColorStr="rgba(0, 0, 0, 0.1)")
  }

  button.dt-button:hover,
  div.dt-button:hover,
  a.dt-button:hover,
  input.dt-button:hover {
    text-decoration: none
  }

  button.dt-button:hover:not(.disabled),
  div.dt-button:hover:not(.disabled),
  a.dt-button:hover:not(.disabled),
  input.dt-button:hover:not(.disabled) {
    border: 1px solid #666;
    background-color: rgba(0, 0, 0, 0.1);
    background: linear-gradient(to bottom, rgba(153, 153, 153, 0.1) 0%, rgba(0, 0, 0, 0.1) 100%);
    filter: progid:DXImageTransform.Microsoft.gradient(GradientType=0, StartColorStr="rgba(153, 153, 153, 0.1)", EndColorStr="rgba(0, 0, 0, 0.1)")
  }

  button.dt-button:focus:not(.disabled),
  div.dt-button:focus:not(.disabled),
  a.dt-button:focus:not(.disabled),
  input.dt-button:focus:not(.disabled) {
    border: 1px solid #426c9e;
    text-shadow: 0 1px 0 #c4def1;
    outline: none;
    background-color: #79ace9;
    background: linear-gradient(to bottom, #d1e2f7 0%, #79ace9 100%);
    filter: progid:DXImageTransform.Microsoft.gradient(GradientType=0, StartColorStr="#d1e2f7", EndColorStr="#79ace9")
  }

  button.dt-button.active:focus:not(.disabled),
  div.dt-button.active:focus:not(.disabled),
  a.dt-button.active:focus:not(.disabled),
  input.dt-button.active:focus:not(.disabled) {
    background: linear-gradient(to bottom, #d1e2f7 0%, #79ace9 100%) !important
  }

  button.dt-button span.dt-down-arrow,
  div.dt-button span.dt-down-arrow,
  a.dt-button span.dt-down-arrow,
  input.dt-button span.dt-down-arrow {
    position: relative;
    top: -2px;
    color: rgba(70, 70, 70, 0.75);
    font-size: 8px;
    padding-left: 10px;
    line-height: 1em
  }

  .dt-button embed {
    outline: none
  }

  div.dt-buttons {
    float: left
  }

  div.dt-buttons.buttons-right {
    float: right
  }

  div.dataTables_layout_cell div.dt-buttons {
    float: none
  }

  div.dataTables_layout_cell div.dt-buttons.buttons-right {
    float: none
  }

  div.dt-btn-split-wrapper {
    display: inline-block
  }

  div.dt-button-collection {
    position: absolute;
    top: 0;
    left: 0;
    width: 200px;
    margin-top: 3px;
    margin-bottom: 3px;
    padding: 4px 4px 2px 4px;
    border: 1px solid #ccc;
    border: 1px solid rgba(0, 0, 0, 0.4);
    background-color: white;
    overflow: hidden;
    z-index: 2002;
    border-radius: 5px;
    box-shadow: 3px 4px 10px 1px rgba(0, 0, 0, 0.3);
    box-sizing: border-box
  }

  div.dt-button-collection button.dt-button,
  div.dt-button-collection div.dt-button,
  div.dt-button-collection a.dt-button {
    position: relative;
    left: 0;
    right: 0;
    width: 100%;
    display: block;
    float: none;
    margin: 4px 0 2px 0
  }

  div.dt-button-collection button.dt-button:active:not(.disabled),
  div.dt-button-collection button.dt-button.active:not(.disabled),
  div.dt-button-collection div.dt-button:active:not(.disabled),
  div.dt-button-collection div.dt-button.active:not(.disabled),
  div.dt-button-collection a.dt-button:active:not(.disabled),
  div.dt-button-collection a.dt-button.active:not(.disabled) {
    background-color: #dadada;
    background: linear-gradient(to bottom, #f0f0f0 0%, #dadada 100%);
    filter: progid:DXImageTransform.Microsoft.gradient(GradientType=0, StartColorStr="#f0f0f0", EndColorStr="#dadada");
    box-shadow: inset 1px 1px 3px #666
  }

  div.dt-button-collection button.dt-button:first-child,
  div.dt-button-collection div.dt-button:first-child,
  div.dt-button-collection a.dt-button:first-child {
    margin-top: 0;
    border-top-left-radius: 3px;
    border-top-right-radius: 3px
  }

  div.dt-button-collection button.dt-button:last-child,
  div.dt-button-collection div.dt-button:last-child,
  div.dt-button-collection a.dt-button:last-child {
    border-bottom-left-radius: 3px;
    border-bottom-right-radius: 3px
  }

  div.dt-button-collection div.dt-btn-split-wrapper {
    display: flex;
    flex-direction: row;
    flex-wrap: wrap;
    justify-content: flex-start;
    align-content: flex-start;
    align-items: stretch;
    margin: 4px 0 2px 0
  }

  div.dt-button-collection div.dt-btn-split-wrapper button.dt-button {
    margin: 0;
    display: inline-block;
    width: 0;
    flex-grow: 1;
    flex-shrink: 0;
    flex-basis: 50px;
    border-radius: 0
  }

  div.dt-button-collection div.dt-btn-split-wrapper button.dt-btn-split-drop {
    min-width: 20px;
    flex-grow: 0;
    flex-shrink: 0;
    flex-basis: 0
  }

  div.dt-button-collection div.dt-btn-split-wrapper:first-child {
    margin-top: 0
  }

  div.dt-button-collection div.dt-btn-split-wrapper:first-child button.dt-button {
    border-top-left-radius: 3px
  }

  div.dt-button-collection div.dt-btn-split-wrapper:first-child button.dt-btn-split-drop {
    border-top-right-radius: 3px
  }

  div.dt-button-collection div.dt-btn-split-wrapper:last-child button.dt-button {
    border-bottom-left-radius: 3px
  }

  div.dt-button-collection div.dt-btn-split-wrapper:last-child button.dt-btn-split-drop {
    border-bottom-right-radius: 3px
  }

  div.dt-button-collection div.dt-btn-split-wrapper:active:not(.disabled) button.dt-button,
  div.dt-button-collection div.dt-btn-split-wrapper.active:not(.disabled) button.dt-button {
    background-color: #dadada;
    background: linear-gradient(to bottom, #f0f0f0 0%, #dadada 100%);
    filter: progid:DXImageTransform.Microsoft.gradient(GradientType=0, StartColorStr="#f0f0f0", EndColorStr="#dadada");
    box-shadow: inset 0px 0px 4px #666
  }

  div.dt-button-collection div.dt-btn-split-wrapper:active:not(.disabled) button.dt-btn-split-drop,
  div.dt-button-collection div.dt-btn-split-wrapper.active:not(.disabled) button.dt-btn-split-drop {
    box-shadow: none
  }

  div.dt-button-collection.fixed .dt-button:first-child {
    margin-top: 0;
    border-top-left-radius: 0;
    border-top-right-radius: 0
  }

  div.dt-button-collection.fixed .dt-button:last-child {
    border-bottom-left-radius: 0;
    border-bottom-right-radius: 0
  }

  div.dt-button-collection.fixed {
    position: fixed;
    display: block;
    top: 50%;
    left: 50%;
    margin-left: -75px;
    border-radius: 5px;
    background-color: white
  }

  div.dt-button-collection.fixed.two-column {
    margin-left: -200px
  }

  div.dt-button-collection.fixed.three-column {
    margin-left: -225px
  }

  div.dt-button-collection.fixed.four-column {
    margin-left: -300px
  }

  div.dt-button-collection.fixed.columns {
    margin-left: -409px
  }

  @media screen and (max-width: 1024px) {
    div.dt-button-collection.fixed.columns {
      margin-left: -308px
    }
  }

  @media screen and (max-width: 640px) {
    div.dt-button-collection.fixed.columns {
      margin-left: -203px
    }
  }

  @media screen and (max-width: 460px) {
    div.dt-button-collection.fixed.columns {
      margin-left: -100px
    }
  }

  div.dt-button-collection.fixed>:last-child {
    max-height: 100vh;
    overflow: auto
  }

  div.dt-button-collection.two-column>:last-child,
  div.dt-button-collection.three-column>:last-child,
  div.dt-button-collection.four-column>:last-child {
    display: block !important;
    -webkit-column-gap: 8px;
    -moz-column-gap: 8px;
    -ms-column-gap: 8px;
    -o-column-gap: 8px;
    column-gap: 8px
  }

  div.dt-button-collection.two-column>:last-child>*,
  div.dt-button-collection.three-column>:last-child>*,
  div.dt-button-collection.four-column>:last-child>* {
    -webkit-column-break-inside: avoid;
    break-inside: avoid
  }

  div.dt-button-collection.two-column {
    width: 400px
  }

  div.dt-button-collection.two-column>:last-child {
    padding-bottom: 1px;
    column-count: 2
  }

  div.dt-button-collection.three-column {
    width: 450px
  }

  div.dt-button-collection.three-column>:last-child {
    padding-bottom: 1px;
    column-count: 3
  }

  div.dt-button-collection.four-column {
    width: 600px
  }

  div.dt-button-collection.four-column>:last-child {
    padding-bottom: 1px;
    column-count: 4
  }

  div.dt-button-collection .dt-button {
    border-radius: 0
  }

  div.dt-button-collection.columns {
    width: auto
  }

  div.dt-button-collection.columns>:last-child {
    display: flex;
    flex-wrap: wrap;
    justify-content: flex-start;
    align-items: center;
    gap: 6px;
    width: 818px;
    padding-bottom: 1px
  }

  div.dt-button-collection.columns>:last-child .dt-button {
    min-width: 200px;
    flex: 0 1;
    margin: 0
  }

  div.dt-button-collection.columns.dtb-b3>:last-child,
  div.dt-button-collection.columns.dtb-b2>:last-child,
  div.dt-button-collection.columns.dtb-b1>:last-child {
    justify-content: space-between
  }

  div.dt-button-collection.columns.dtb-b3 .dt-button {
    flex: 1 1 32%
  }

  div.dt-button-collection.columns.dtb-b2 .dt-button {
    flex: 1 1 48%
  }

  div.dt-button-collection.columns.dtb-b1 .dt-button {
    flex: 1 1 100%
  }

  @media screen and (max-width: 1024px) {
    div.dt-button-collection.columns>:last-child {
      width: 612px
    }
  }

  @media screen and (max-width: 640px) {
    div.dt-button-collection.columns>:last-child {
      width: 406px
    }

    div.dt-button-collection.columns.dtb-b3 .dt-button {
      flex: 0 1 32%
    }
  }

  @media screen and (max-width: 460px) {
    div.dt-button-collection.columns>:last-child {
      width: 200px
    }
  }

  div.dt-button-background {
    position: fixed;
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    background: rgba(0, 0, 0, 0.7);
    background: radial-gradient(ellipse farthest-corner at center, rgba(0, 0, 0, 0.3) 0%, rgba(0, 0, 0, 0.7) 100%);
    z-index: 2001
  }

  @media screen and (max-width: 640px) {
    div.dt-buttons {
      float: none !important;
      text-align: center
    }
  }

  button.dt-button.processing,
  div.dt-button.processing,
  a.dt-button.processing {
    color: rgba(0, 0, 0, 0.2)
  }

  button.dt-button.processing:after,
  div.dt-button.processing:after,
  a.dt-button.processing:after {
    position: absolute;
    top: 50%;
    left: 50%;
    width: 16px;
    height: 16px;
    margin: -8px 0 0 -8px;
    box-sizing: border-box;
    display: block;
    content: " ";
    border: 2px solid #282828;
    border-radius: 50%;
    border-left-color: transparent;
    border-right-color: transparent;
    animation: dtb-spinner 1500ms infinite linear;
    -o-animation: dtb-spinner 1500ms infinite linear;
    -ms-animation: dtb-spinner 1500ms infinite linear;
    -webkit-animation: dtb-spinner 1500ms infinite linear;
    -moz-animation: dtb-spinner 1500ms infinite linear
  }

  button.dt-btn-split-drop {
    margin-left: calc(-1px - .333em);
    padding-bottom: calc(.5em - 1px);
    border-radius: 0px 1px 1px 0px;
    color: rgba(70, 70, 70, 0.9);
    border-left: none
  }

  button.dt-btn-split-drop span.dt-btn-split-drop-arrow {
    position: relative;
    top: -1px;
    left: -2px;
    font-size: 8px
  }

  button.dt-btn-split-drop:hover {
    z-index: 2
  }

  button.buttons-split {
    border-right: 1px solid rgba(70, 70, 70, 0);
    border-radius: 1px 0px 0px 1px
  }

  button.dt-btn-split-drop-button {
    background-color: white
  }

  button.dt-btn-split-drop-button:hover {
    background-color: white
  }

</style>
<script>/*!
Buttons for DataTables 2.2.3
©2016-2022 SpryMedia Ltd - datatables.net/license
*/
  (function (d) { "function" === typeof define && define.amd ? define(["jquery", "datatables.net"], function (A) { return d(A, window, document) }) : "object" === typeof exports ? module.exports = function (A, C) { A || (A = window); C && C.fn.dataTable || (C = require("datatables.net")(A, C).$); return d(C, A, A.document) } : d(jQuery, window, document) })(function (d, A, C, p) {
    function I(a, b, c) { d.fn.animate ? a.stop().fadeIn(b, c) : (a.css("display", "block"), c && c.call(a)) } function J(a, b, c) { d.fn.animate ? a.stop().fadeOut(b, c) : (a.css("display", "none"), c && c.call(a)) }
    function L(a, b) { a = new u.Api(a); b = b ? b : a.init().buttons || u.defaults.buttons; return (new x(a, b)).container() } var u = d.fn.dataTable, O = 0, P = 0, D = u.ext.buttons, x = function (a, b) {
      if (!(this instanceof x)) return function (c) { return (new x(c, a)).container() }; "undefined" === typeof b && (b = {}); !0 === b && (b = {}); Array.isArray(b) && (b = { buttons: b }); this.c = d.extend(!0, {}, x.defaults, b); b.buttons && (this.c.buttons = b.buttons); this.s = { dt: new u.Api(a), buttons: [], listenKeys: "", namespace: "dtb" + O++ }; this.dom = {
        container: d("<" + this.c.dom.container.tag +
          "/>").addClass(this.c.dom.container.className)
      }; this._constructor()
    }; d.extend(x.prototype, {
      action: function (a, b) { a = this._nodeToButton(a); if (b === p) return a.conf.action; a.conf.action = b; return this }, active: function (a, b) { var c = this._nodeToButton(a); a = this.c.dom.button.active; c = d(c.node); if (b === p) return c.hasClass(a); c.toggleClass(a, b === p ? !0 : b); return this }, add: function (a, b, c) {
        var e = this.s.buttons; if ("string" === typeof b) {
          b = b.split("-"); var h = this.s; e = 0; for (var f = b.length - 1; e < f; e++)h = h.buttons[1 * b[e]]; e =
            h.buttons; b = 1 * b[b.length - 1]
        } this._expandButton(e, a, a !== p ? a.split : p, (a === p || a.split === p || 0 === a.split.length) && h !== p, !1, b); c !== p && !0 !== c || this._draw(); return this
      }, collectionRebuild: function (a, b) { a = this._nodeToButton(a); if (b !== p) { var c; for (c = a.buttons.length - 1; 0 <= c; c--)this.remove(a.buttons[c].node); for (c = 0; c < b.length; c++) { var e = b[c]; this._expandButton(a.buttons, e, e !== p && e.config !== p && e.config.split !== p, !0, e.parentConf !== p && e.parentConf.split !== p, c, e.parentConf) } } this._draw(a.collection, a.buttons) },
      container: function () { return this.dom.container }, disable: function (a) { a = this._nodeToButton(a); d(a.node).addClass(this.c.dom.button.disabled).attr("disabled", !0); return this }, destroy: function () { d("body").off("keyup." + this.s.namespace); var a = this.s.buttons.slice(), b; var c = 0; for (b = a.length; c < b; c++)this.remove(a[c].node); this.dom.container.remove(); a = this.s.dt.settings()[0]; c = 0; for (b = a.length; c < b; c++)if (a.inst === this) { a.splice(c, 1); break } return this }, enable: function (a, b) {
        if (!1 === b) return this.disable(a);
        a = this._nodeToButton(a); d(a.node).removeClass(this.c.dom.button.disabled).removeAttr("disabled"); return this
      }, index: function (a, b, c) { b || (b = "", c = this.s.buttons); for (var e = 0, h = c.length; e < h; e++) { var f = c[e].buttons; if (c[e].node === a) return b + e; if (f && f.length && (f = this.index(a, e + "-", f), null !== f)) return f } return null }, name: function () { return this.c.name }, node: function (a) { if (!a) return this.dom.container; a = this._nodeToButton(a); return d(a.node) }, processing: function (a, b) {
        var c = this.s.dt, e = this._nodeToButton(a);
        if (b === p) return d(e.node).hasClass("processing"); d(e.node).toggleClass("processing", b); d(c.table().node()).triggerHandler("buttons-processing.dt", [b, c.button(a), c, d(a), e.conf]); return this
      }, remove: function (a) {
        var b = this._nodeToButton(a), c = this._nodeToHost(a), e = this.s.dt; if (b.buttons.length) for (var h = b.buttons.length - 1; 0 <= h; h--)this.remove(b.buttons[h].node); b.conf.destroying = !0; b.conf.destroy && b.conf.destroy.call(e.button(a), e, d(a), b.conf); this._removeKey(b.conf); d(b.node).remove(); a = d.inArray(b,
          c); c.splice(a, 1); return this
      }, text: function (a, b) { var c = this._nodeToButton(a); a = this.c.dom.collection.buttonLiner; a = c.inCollection && a && a.tag ? a.tag : this.c.dom.buttonLiner.tag; var e = this.s.dt, h = d(c.node), f = function (g) { return "function" === typeof g ? g(e, h, c.conf) : g }; if (b === p) return f(c.conf.text); c.conf.text = b; a ? h.children(a).eq(0).filter(":not(.dt-down-arrow)").html(f(b)) : h.html(f(b)); return this }, _constructor: function () {
        var a = this, b = this.s.dt, c = b.settings()[0], e = this.c.buttons; c._buttons || (c._buttons =
          []); c._buttons.push({ inst: this, name: this.c.name }); for (var h = 0, f = e.length; h < f; h++)this.add(e[h]); b.on("destroy", function (g, l) { l === c && a.destroy() }); d("body").on("keyup." + this.s.namespace, function (g) { if (!C.activeElement || C.activeElement === C.body) { var l = String.fromCharCode(g.keyCode).toLowerCase(); -1 !== a.s.listenKeys.toLowerCase().indexOf(l) && a._keypress(l, g) } })
      }, _addKey: function (a) { a.key && (this.s.listenKeys += d.isPlainObject(a.key) ? a.key.key : a.key) }, _draw: function (a, b) {
        a || (a = this.dom.container, b = this.s.buttons);
        a.children().detach(); for (var c = 0, e = b.length; c < e; c++)a.append(b[c].inserter), a.append(" "), b[c].buttons && b[c].buttons.length && this._draw(b[c].collection, b[c].buttons)
      }, _expandButton: function (a, b, c, e, h, f, g) {
        var l = this.s.dt, m = 0, r = Array.isArray(b) ? b : [b]; b === p && (r = Array.isArray(c) ? c : [c]); c = 0; for (var q = r.length; c < q; c++) {
          var n = this._resolveExtends(r[c]); if (n) if (b = n.config !== p && n.config.split ? !0 : !1, Array.isArray(n)) this._expandButton(a, n, k !== p && k.conf !== p ? k.conf.split : p, e, g !== p && g.split !== p, f, g); else {
            var k =
              this._buildButton(n, e, n.split !== p || n.config !== p && n.config.split !== p, h); if (k) {
                f !== p && null !== f ? (a.splice(f, 0, k), f++) : a.push(k); if (k.conf.buttons || k.conf.split) {
                  k.collection = d("<" + (b ? this.c.dom.splitCollection.tag : this.c.dom.collection.tag) + "/>"); k.conf._collection = k.collection; if (k.conf.split) for (var t = 0; t < k.conf.split.length; t++)"object" === typeof k.conf.split[t] && (k.conf.split[t].parent = g, k.conf.split[t].collectionLayout === p && (k.conf.split[t].collectionLayout = k.conf.collectionLayout), k.conf.split[t].dropup ===
                    p && (k.conf.split[t].dropup = k.conf.dropup), k.conf.split[t].fade === p && (k.conf.split[t].fade = k.conf.fade)); else d(k.node).append(d('<span class="dt-down-arrow">' + this.c.dom.splitDropdown.text + "</span>")); this._expandButton(k.buttons, k.conf.buttons, k.conf.split, !b, b, f, k.conf)
                } k.conf.parent = g; n.init && n.init.call(l.button(k.node), l, d(k.node), n); m++
              }
          }
        }
      }, _buildButton: function (a, b, c, e) {
        var h = this.c.dom.button, f = this.c.dom.buttonLiner, g = this.c.dom.collection, l = this.c.dom.splitCollection, m = this.c.dom.splitDropdownButton,
        r = this.s.dt, q = function (v) { return "function" === typeof v ? v(r, k, a) : v }; if (a.spacer) { var n = d("<span></span>").addClass("dt-button-spacer " + a.style + " " + h.spacerClass).html(q(a.text)); return { conf: a, node: n, inserter: n, buttons: [], inCollection: b, isSplit: c, inSplit: e, collection: null } } !c && e && l ? h = m : !c && b && g.button && (h = g.button); !c && e && l.buttonLiner ? f = l.buttonLiner : !c && b && g.buttonLiner && (f = g.buttonLiner); if (a.available && !a.available(r, a) && !a.hasOwnProperty("html")) return !1; if (a.hasOwnProperty("html")) var k = d(a.html);
        else {
          var t = function (v, E, F, G) { G.action.call(E.button(F), v, E, F, G); d(E.table().node()).triggerHandler("buttons-action.dt", [E.button(F), E, F, G]) }; g = a.tag || h.tag; var y = a.clickBlurs === p ? !0 : a.clickBlurs; k = d("<" + g + "/>").addClass(h.className).addClass(e ? this.c.dom.splitDropdownButton.className : "").attr("tabindex", this.s.dt.settings()[0].iTabIndex).attr("aria-controls", this.s.dt.table().node().id).on("click.dtb", function (v) { v.preventDefault(); !k.hasClass(h.disabled) && a.action && t(v, r, k, a); y && k.trigger("blur") }).on("keypress.dtb",
            function (v) { 13 === v.keyCode && (v.preventDefault(), !k.hasClass(h.disabled) && a.action && t(v, r, k, a)) }); "a" === g.toLowerCase() && k.attr("href", "#"); "button" === g.toLowerCase() && k.attr("type", "button"); f.tag ? (g = d("<" + f.tag + "/>").html(q(a.text)).addClass(f.className), "a" === f.tag.toLowerCase() && g.attr("href", "#"), k.append(g)) : k.html(q(a.text)); !1 === a.enabled && k.addClass(h.disabled); a.className && k.addClass(a.className); a.titleAttr && k.attr("title", q(a.titleAttr)); a.attr && k.attr(a.attr); a.namespace || (a.namespace =
              ".dt-button-" + P++); a.config !== p && a.config.split && (a.split = a.config.split)
        } f = (f = this.c.dom.buttonContainer) && f.tag ? d("<" + f.tag + "/>").addClass(f.className).append(k) : k; this._addKey(a); this.c.buttonCreated && (f = this.c.buttonCreated(a, f)); if (c) {
          n = d("<div/>").addClass(this.c.dom.splitWrapper.className); n.append(k); var w = d.extend(a, {
            text: this.c.dom.splitDropdown.text, className: this.c.dom.splitDropdown.className, closeButton: !1, attr: { "aria-haspopup": "dialog", "aria-expanded": !1 }, align: this.c.dom.splitDropdown.align,
            splitAlignClass: this.c.dom.splitDropdown.splitAlignClass
          }); this._addKey(w); var B = function (v, E, F, G) { D.split.action.call(E.button(d("div.dt-btn-split-wrapper")[0]), v, E, F, G); d(E.table().node()).triggerHandler("buttons-action.dt", [E.button(F), E, F, G]); F.attr("aria-expanded", !0) }, z = d('<button class="' + this.c.dom.splitDropdown.className + ' dt-button"><span class="dt-btn-split-drop-arrow">' + this.c.dom.splitDropdown.text + "</span></button>").on("click.dtb", function (v) {
            v.preventDefault(); v.stopPropagation();
            z.hasClass(h.disabled) || B(v, r, z, w); y && z.trigger("blur")
          }).on("keypress.dtb", function (v) { 13 === v.keyCode && (v.preventDefault(), z.hasClass(h.disabled) || B(v, r, z, w)) }); 0 === a.split.length && z.addClass("dtb-hide-drop"); n.append(z).attr(w.attr)
        } return { conf: a, node: c ? n.get(0) : k.get(0), inserter: c ? n : f, buttons: [], inCollection: b, isSplit: c, inSplit: e, collection: null }
      }, _nodeToButton: function (a, b) {
        b || (b = this.s.buttons); for (var c = 0, e = b.length; c < e; c++) {
          if (b[c].node === a) return b[c]; if (b[c].buttons.length) {
            var h = this._nodeToButton(a,
              b[c].buttons); if (h) return h
          }
        }
      }, _nodeToHost: function (a, b) { b || (b = this.s.buttons); for (var c = 0, e = b.length; c < e; c++) { if (b[c].node === a) return b; if (b[c].buttons.length) { var h = this._nodeToHost(a, b[c].buttons); if (h) return h } } }, _keypress: function (a, b) {
        if (!b._buttonsHandled) {
          var c = function (e) {
            for (var h = 0, f = e.length; h < f; h++) {
              var g = e[h].conf, l = e[h].node; g.key && (g.key === a ? (b._buttonsHandled = !0, d(l).click()) : !d.isPlainObject(g.key) || g.key.key !== a || g.key.shiftKey && !b.shiftKey || g.key.altKey && !b.altKey || g.key.ctrlKey &&
                !b.ctrlKey || g.key.metaKey && !b.metaKey || (b._buttonsHandled = !0, d(l).click())); e[h].buttons.length && c(e[h].buttons)
            }
          }; c(this.s.buttons)
        }
      }, _removeKey: function (a) { if (a.key) { var b = d.isPlainObject(a.key) ? a.key.key : a.key; a = this.s.listenKeys.split(""); b = d.inArray(b, a); a.splice(b, 1); this.s.listenKeys = a.join("") } }, _resolveExtends: function (a) {
        var b = this, c = this.s.dt, e, h = function (m) {
          for (var r = 0; !d.isPlainObject(m) && !Array.isArray(m);) {
            if (m === p) return; if ("function" === typeof m) { if (m = m.call(b, c, a), !m) return !1 } else if ("string" ===
              typeof m) { if (!D[m]) return { html: m }; m = D[m] } r++; if (30 < r) throw "Buttons: Too many iterations";
          } return Array.isArray(m) ? m : d.extend({}, m)
        }; for (a = h(a); a && a.extend;) {
          if (!D[a.extend]) throw "Cannot extend unknown button type: " + a.extend; var f = h(D[a.extend]); if (Array.isArray(f)) return f; if (!f) return !1; var g = f.className; a.config !== p && f.config !== p && (a.config = d.extend({}, f.config, a.config)); a = d.extend({}, f, a); g && a.className !== g && (a.className = g + " " + a.className); var l = a.postfixButtons; if (l) {
            a.buttons || (a.buttons = []);
            g = 0; for (e = l.length; g < e; g++)a.buttons.push(l[g]); a.postfixButtons = null
          } if (l = a.prefixButtons) { a.buttons || (a.buttons = []); g = 0; for (e = l.length; g < e; g++)a.buttons.splice(g, 0, l[g]); a.prefixButtons = null } a.extend = f.extend
        } return a
      }, _popover: function (a, b, c, e) {
        e = this.c; var h = !1, f = d.extend({
          align: "button-left", autoClose: !1, background: !0, backgroundClassName: "dt-button-background", closeButton: !0, contentClassName: e.dom.collection.className, collectionLayout: "", collectionTitle: "", dropup: !1, fade: 400, popoverTitle: "",
          rightAlignClassName: "dt-button-right", tag: e.dom.collection.tag
        }, c), g = b.node(), l = function () { h = !0; J(d(".dt-button-collection"), f.fade, function () { d(this).detach() }); d(b.buttons('[aria-haspopup="dialog"][aria-expanded="true"]').nodes()).attr("aria-expanded", "false"); d("div.dt-button-background").off("click.dtb-collection"); x.background(!1, f.backgroundClassName, f.fade, g); d(A).off("resize.resize.dtb-collection"); d("body").off(".dtb-collection"); b.off("buttons-action.b-internal"); b.off("destroy") }; if (!1 ===
          a) l(); else {
            c = d(b.buttons('[aria-haspopup="dialog"][aria-expanded="true"]').nodes()); c.length && (g.closest("div.dt-button-collection").length && (g = c.eq(0)), l()); c = d(".dt-button", a).length; e = ""; 3 === c ? e = "dtb-b3" : 2 === c ? e = "dtb-b2" : 1 === c && (e = "dtb-b1"); var m = d("<div/>").addClass("dt-button-collection").addClass(f.collectionLayout).addClass(f.splitAlignClass).addClass(e).css("display", "none").attr({ "aria-modal": !0, role: "dialog" }); a = d(a).addClass(f.contentClassName).attr("role", "menu").appendTo(m); g.attr("aria-expanded",
              "true"); g.parents("body")[0] !== C.body && (g = C.body.lastChild); f.popoverTitle ? m.prepend('<div class="dt-button-collection-title">' + f.popoverTitle + "</div>") : f.collectionTitle && m.prepend('<div class="dt-button-collection-title">' + f.collectionTitle + "</div>"); f.closeButton && m.prepend('<div class="dtb-popover-close">x</div>').addClass("dtb-collection-closeable"); I(m.insertAfter(g), f.fade); c = d(b.table().container()); var r = m.css("position"); if ("container" === f.span || "dt-container" === f.align) g = g.parent(), m.css("width",
                c.width()); if ("absolute" === r) {
                  var q = d(g[0].offsetParent); c = g.position(); e = g.offset(); var n = q.offset(), k = q.position(), t = A.getComputedStyle(q[0]); n.height = q.outerHeight(); n.width = q.width() + parseFloat(t.paddingLeft); n.right = n.left + n.width; n.bottom = n.top + n.height; q = c.top + g.outerHeight(); var y = c.left; m.css({ top: q, left: y }); t = A.getComputedStyle(m[0]); var w = m.offset(); w.height = m.outerHeight(); w.width = m.outerWidth(); w.right = w.left + w.width; w.bottom = w.top + w.height; w.marginTop = parseFloat(t.marginTop); w.marginBottom =
                    parseFloat(t.marginBottom); f.dropup && (q = c.top - w.height - w.marginTop - w.marginBottom); if ("button-right" === f.align || m.hasClass(f.rightAlignClassName)) y = c.left - w.width + g.outerWidth(); if ("dt-container" === f.align || "container" === f.align) y < c.left && (y = -c.left), y + w.width > n.width && (y = n.width - w.width); k.left + y + w.width > d(A).width() && (y = d(A).width() - w.width - k.left); 0 > e.left + y && (y = -e.left); k.top + q + w.height > d(A).height() + d(A).scrollTop() && (q = c.top - w.height - w.marginTop - w.marginBottom); k.top + q < d(A).scrollTop() && (q =
                      c.top + g.outerHeight()); m.css({ top: q, left: y })
                } else r = function () { var B = d(A).height() / 2, z = m.height() / 2; z > B && (z = B); m.css("marginTop", -1 * z) }, r(), d(A).on("resize.dtb-collection", function () { r() }); f.background && x.background(!0, f.backgroundClassName, f.fade, f.backgroundHost || g); d("div.dt-button-background").on("click.dtb-collection", function () { }); f.autoClose && setTimeout(function () { b.on("buttons-action.b-internal", function (B, z, v, E) { E[0] !== g[0] && l() }) }, 0); d(m).trigger("buttons-popover.dt"); b.on("destroy", l);
          setTimeout(function () {
            h = !1; d("body").on("click.dtb-collection", function (B) { if (!h) { var z = d.fn.addBack ? "addBack" : "andSelf", v = d(B.target).parent()[0]; (!d(B.target).parents()[z]().filter(a).length && !d(v).hasClass("dt-buttons") || d(B.target).hasClass("dt-button-background")) && l() } }).on("keyup.dtb-collection", function (B) { 27 === B.keyCode && l() }).on("keydown.dtb-collection", function (B) {
              var z = d("a, button", a), v = C.activeElement; 9 === B.keyCode && (-1 === z.index(v) ? (z.first().focus(), B.preventDefault()) : B.shiftKey ?
                v === z[0] && (z.last().focus(), B.preventDefault()) : v === z.last()[0] && (z.first().focus(), B.preventDefault()))
            })
          }, 0)
        }
      }
    }); x.background = function (a, b, c, e) { c === p && (c = 400); e || (e = C.body); a ? I(d("<div/>").addClass(b).css("display", "none").insertAfter(e), c) : J(d("div." + b), c, function () { d(this).removeClass(b).remove() }) }; x.instanceSelector = function (a, b) {
      if (a === p || null === a) return d.map(b, function (f) { return f.inst }); var c = [], e = d.map(b, function (f) { return f.name }), h = function (f) {
        if (Array.isArray(f)) for (var g = 0, l = f.length; g <
          l; g++)h(f[g]); else "string" === typeof f ? -1 !== f.indexOf(",") ? h(f.split(",")) : (f = d.inArray(f.trim(), e), -1 !== f && c.push(b[f].inst)) : "number" === typeof f ? c.push(b[f].inst) : "object" === typeof f && c.push(f)
      }; h(a); return c
    }; x.buttonSelector = function (a, b) {
      for (var c = [], e = function (l, m, r) { for (var q, n, k = 0, t = m.length; k < t; k++)if (q = m[k]) n = r !== p ? r + k : k + "", l.push({ node: q.node, name: q.conf.name, idx: n }), q.buttons && e(l, q.buttons, n + "-") }, h = function (l, m) {
        var r, q = []; e(q, m.s.buttons); var n = d.map(q, function (k) { return k.node }); if (Array.isArray(l) ||
          l instanceof d) for (n = 0, r = l.length; n < r; n++)h(l[n], m); else if (null === l || l === p || "*" === l) for (n = 0, r = q.length; n < r; n++)c.push({ inst: m, node: q[n].node }); else if ("number" === typeof l) m.s.buttons[l] && c.push({ inst: m, node: m.s.buttons[l].node }); else if ("string" === typeof l) if (-1 !== l.indexOf(",")) for (q = l.split(","), n = 0, r = q.length; n < r; n++)h(q[n].trim(), m); else if (l.match(/^\d+(\-\d+)*$/)) n = d.map(q, function (k) { return k.idx }), c.push({ inst: m, node: q[d.inArray(l, n)].node }); else if (-1 !== l.indexOf(":name")) for (l = l.replace(":name",
            ""), n = 0, r = q.length; n < r; n++)q[n].name === l && c.push({ inst: m, node: q[n].node }); else d(n).filter(l).each(function () { c.push({ inst: m, node: this }) }); else "object" === typeof l && l.nodeName && (q = d.inArray(l, n), -1 !== q && c.push({ inst: m, node: n[q] }))
      }, f = 0, g = a.length; f < g; f++)h(b, a[f]); return c
    }; x.stripData = function (a, b) {
      if ("string" !== typeof a) return a; a = a.replace(/<script\b[^<]*(?:(?!<\/script>)<[^<]*)*<\/script>/gi, ""); a = a.replace(/<!\-\-.*?\-\->/g, ""); if (!b || b.stripHtml) a = a.replace(/<[^>]*>/g, ""); if (!b || b.trim) a = a.replace(/^\s+|\s+$/g,
        ""); if (!b || b.stripNewlines) a = a.replace(/\n/g, " "); if (!b || b.decodeEntities) M.innerHTML = a, a = M.value; return a
    }; x.defaults = {
      buttons: ["copy", "excel", "csv", "pdf", "print"], name: "main", tabIndex: 0, dom: {
        container: { tag: "div", className: "dt-buttons" }, collection: { tag: "div", className: "" }, button: { tag: "button", className: "dt-button", active: "active", disabled: "disabled", spacerClass: "" }, buttonLiner: { tag: "span", className: "" }, split: { tag: "div", className: "dt-button-split" }, splitWrapper: { tag: "div", className: "dt-btn-split-wrapper" },
        splitDropdown: { tag: "button", text: "&#x25BC;", className: "dt-btn-split-drop", align: "split-right", splitAlignClass: "dt-button-split-left" }, splitDropdownButton: { tag: "button", className: "dt-btn-split-drop-button dt-button" }, splitCollection: { tag: "div", className: "dt-button-split-collection" }
      }
    }; x.version = "2.2.3"; d.extend(D, {
      collection: {
        text: function (a) { return a.i18n("buttons.collection", "Collection") }, className: "buttons-collection", closeButton: !1, init: function (a, b, c) { b.attr("aria-expanded", !1) }, action: function (a,
          b, c, e) { e._collection.parents("body").length ? this.popover(!1, e) : this.popover(e._collection, e); "keypress" === a.type && d("a, button", e._collection).eq(0).focus() }, attr: { "aria-haspopup": "dialog" }
      }, split: { text: function (a) { return a.i18n("buttons.split", "Split") }, className: "buttons-split", closeButton: !1, init: function (a, b, c) { return b.attr("aria-expanded", !1) }, action: function (a, b, c, e) { this.popover(e._collection, e) }, attr: { "aria-haspopup": "dialog" } }, copy: function (a, b) { if (D.copyHtml5) return "copyHtml5" }, csv: function (a,
        b) { if (D.csvHtml5 && D.csvHtml5.available(a, b)) return "csvHtml5" }, excel: function (a, b) { if (D.excelHtml5 && D.excelHtml5.available(a, b)) return "excelHtml5" }, pdf: function (a, b) { if (D.pdfHtml5 && D.pdfHtml5.available(a, b)) return "pdfHtml5" }, pageLength: function (a) {
          a = a.settings()[0].aLengthMenu; var b = [], c = []; if (Array.isArray(a[0])) b = a[0], c = a[1]; else for (var e = 0; e < a.length; e++) { var h = a[e]; d.isPlainObject(h) ? (b.push(h.value), c.push(h.label)) : (b.push(h), c.push(h)) } return {
            extend: "collection", text: function (f) {
              return f.i18n("buttons.pageLength",
                { "-1": "Show all rows", _: "Show %d rows" }, f.page.len())
            }, className: "buttons-page-length", autoClose: !0, buttons: d.map(b, function (f, g) { return { text: c[g], className: "button-page-length", action: function (l, m) { m.page.len(f).draw() }, init: function (l, m, r) { var q = this; m = function () { q.active(l.page.len() === f) }; l.on("length.dt" + r.namespace, m); m() }, destroy: function (l, m, r) { l.off("length.dt" + r.namespace) } } }), init: function (f, g, l) { var m = this; f.on("length.dt" + l.namespace, function () { m.text(l.text) }) }, destroy: function (f, g,
              l) { f.off("length.dt" + l.namespace) }
          }
        }, spacer: { style: "empty", spacer: !0, text: function (a) { return a.i18n("buttons.spacer", "") } }
    }); u.Api.register("buttons()", function (a, b) { b === p && (b = a, a = p); this.selector.buttonGroup = a; var c = this.iterator(!0, "table", function (e) { if (e._buttons) return x.buttonSelector(x.instanceSelector(a, e._buttons), b) }, !0); c._groupSelector = a; return c }); u.Api.register("button()", function (a, b) { a = this.buttons(a, b); 1 < a.length && a.splice(1, a.length); return a }); u.Api.registerPlural("buttons().active()",
      "button().active()", function (a) { return a === p ? this.map(function (b) { return b.inst.active(b.node) }) : this.each(function (b) { b.inst.active(b.node, a) }) }); u.Api.registerPlural("buttons().action()", "button().action()", function (a) { return a === p ? this.map(function (b) { return b.inst.action(b.node) }) : this.each(function (b) { b.inst.action(b.node, a) }) }); u.Api.registerPlural("buttons().collectionRebuild()", "button().collectionRebuild()", function (a) {
        return this.each(function (b) {
          for (var c = 0; c < a.length; c++)"object" === typeof a[c] &&
            (a[c].parentConf = b); b.inst.collectionRebuild(b.node, a)
        })
      }); u.Api.register(["buttons().enable()", "button().enable()"], function (a) { return this.each(function (b) { b.inst.enable(b.node, a) }) }); u.Api.register(["buttons().disable()", "button().disable()"], function () { return this.each(function (a) { a.inst.disable(a.node) }) }); u.Api.register("button().index()", function () { var a = null; this.each(function (b) { b = b.inst.index(b.node); null !== b && (a = b) }); return a }); u.Api.registerPlural("buttons().nodes()", "button().node()",
        function () { var a = d(); d(this.each(function (b) { a = a.add(b.inst.node(b.node)) })); return a }); u.Api.registerPlural("buttons().processing()", "button().processing()", function (a) { return a === p ? this.map(function (b) { return b.inst.processing(b.node) }) : this.each(function (b) { b.inst.processing(b.node, a) }) }); u.Api.registerPlural("buttons().text()", "button().text()", function (a) { return a === p ? this.map(function (b) { return b.inst.text(b.node) }) : this.each(function (b) { b.inst.text(b.node, a) }) }); u.Api.registerPlural("buttons().trigger()",
          "button().trigger()", function () { return this.each(function (a) { a.inst.node(a.node).trigger("click") }) }); u.Api.register("button().popover()", function (a, b) { return this.map(function (c) { return c.inst._popover(a, this.button(this[0].node), b) }) }); u.Api.register("buttons().containers()", function () { var a = d(), b = this._groupSelector; this.iterator(!0, "table", function (c) { if (c._buttons) { c = x.instanceSelector(b, c._buttons); for (var e = 0, h = c.length; e < h; e++)a = a.add(c[e].container()) } }); return a }); u.Api.register("buttons().container()",
            function () { return this.containers().eq(0) }); u.Api.register("button().add()", function (a, b, c) { var e = this.context; e.length && (e = x.instanceSelector(this._groupSelector, e[0]._buttons), e.length && e[0].add(b, a, c)); return this.button(this._groupSelector, a) }); u.Api.register("buttons().destroy()", function () { this.pluck("inst").unique().each(function (a) { a.destroy() }); return this }); u.Api.registerPlural("buttons().remove()", "buttons().remove()", function () { this.each(function (a) { a.inst.remove(a.node) }); return this });
    var H; u.Api.register("buttons.info()", function (a, b, c) {
      var e = this; if (!1 === a) return this.off("destroy.btn-info"), J(d("#datatables_buttons_info"), 400, function () { d(this).remove() }), clearTimeout(H), H = null, this; H && clearTimeout(H); d("#datatables_buttons_info").length && d("#datatables_buttons_info").remove(); a = a ? "<h2>" + a + "</h2>" : ""; I(d('<div id="datatables_buttons_info" class="dt-button-info"/>').html(a).append(d("<div/>")["string" === typeof b ? "html" : "append"](b)).css("display", "none").appendTo("body")); c !==
        p && 0 !== c && (H = setTimeout(function () { e.buttons.info(!1) }, c)); this.on("destroy.btn-info", function () { e.buttons.info(!1) }); return this
    }); u.Api.register("buttons.exportData()", function (a) { if (this.context.length) return Q(new u.Api(this.context[0]), a) }); u.Api.register("buttons.exportInfo()", function (a) {
      a || (a = {}); var b = a; var c = "*" === b.filename && "*" !== b.title && b.title !== p && null !== b.title && "" !== b.title ? b.title : b.filename; "function" === typeof c && (c = c()); c === p || null === c ? c = null : (-1 !== c.indexOf("*") && (c = c.replace("*",
        d("head > title").text()).trim()), c = c.replace(/[^a-zA-Z0-9_\u00A1-\uFFFF\.,\-_ !\(\)]/g, ""), (b = K(b.extension)) || (b = ""), c += b); b = K(a.title); b = null === b ? null : -1 !== b.indexOf("*") ? b.replace("*", d("head > title").text() || "Exported data") : b; return { filename: c, title: b, messageTop: N(this, a.message || a.messageTop, "top"), messageBottom: N(this, a.messageBottom, "bottom") }
    }); var K = function (a) { return null === a || a === p ? null : "function" === typeof a ? a() : a }, N = function (a, b, c) {
      b = K(b); if (null === b) return null; a = d("caption", a.table().container()).eq(0);
      return "*" === b ? a.css("caption-side") !== c ? null : a.length ? a.text() : "" : b
    }, M = d("<textarea/>")[0], Q = function (a, b) {
      var c = d.extend(!0, {}, { rows: null, columns: "", modifier: { search: "applied", order: "applied" }, orthogonal: "display", stripHtml: !0, stripNewlines: !0, decodeEntities: !0, trim: !0, format: { header: function (t) { return x.stripData(t, c) }, footer: function (t) { return x.stripData(t, c) }, body: function (t) { return x.stripData(t, c) } }, customizeData: null }, b); b = a.columns(c.columns).indexes().map(function (t) {
        var y = a.column(t).header();
        return c.format.header(y.innerHTML, t, y)
      }).toArray(); var e = a.table().footer() ? a.columns(c.columns).indexes().map(function (t) { var y = a.column(t).footer(); return c.format.footer(y ? y.innerHTML : "", t, y) }).toArray() : null, h = d.extend({}, c.modifier); a.select && "function" === typeof a.select.info && h.selected === p && a.rows(c.rows, d.extend({ selected: !0 }, h)).any() && d.extend(h, { selected: !0 }); h = a.rows(c.rows, h).indexes().toArray(); var f = a.cells(h, c.columns); h = f.render(c.orthogonal).toArray(); f = f.nodes().toArray(); for (var g =
        b.length, l = [], m = 0, r = 0, q = 0 < g ? h.length / g : 0; r < q; r++) { for (var n = [g], k = 0; k < g; k++)n[k] = c.format.body(h[m], r, k, f[m]), m++; l[r] = n } b = { header: b, footer: e, body: l }; c.customizeData && c.customizeData(b); return b
    }; d.fn.dataTable.Buttons = x; d.fn.DataTable.Buttons = x; d(C).on("init.dt plugin-init.dt", function (a, b) { "dt" === a.namespace && (a = b.oInit.buttons || u.defaults.buttons) && !b._buttons && (new x(b, a)).container() }); u.ext.feature.push({ fnInit: L, cFeature: "B" }); u.ext.features && u.ext.features.register("buttons", L); return x
  });
</script>
<script>/*!
HTML5 export buttons for Buttons and DataTables.
2016 SpryMedia Ltd - datatables.net/license

FileSaver.js (1.3.3) - MIT license
Copyright © 2016 Eli Grey - http://eligrey.com
*/
  (function (n) { "function" === typeof define && define.amd ? define(["jquery", "datatables.net", "datatables.net-buttons"], function (u) { return n(u, window, document) }) : "object" === typeof exports ? module.exports = function (u, x, E, F) { u || (u = window); x && x.fn.dataTable || (x = require("datatables.net")(u, x).$); x.fn.dataTable.Buttons || require("datatables.net-buttons")(u, x); return n(x, u, u.document, E, F) } : n(jQuery, window, document) })(function (n, u, x, E, F, B) {
    function I(a) {
      for (var c = ""; 0 <= a;)c = String.fromCharCode(a % 26 + 65) + c, a = Math.floor(a /
        26) - 1; return c
    } function O(a, c) {
      J === B && (J = -1 === M.serializeToString((new u.DOMParser).parseFromString(P["xl/worksheets/sheet1.xml"], "text/xml")).indexOf("xmlns:r")); n.each(c, function (d, b) {
        if (n.isPlainObject(b)) d = a.folder(d), O(d, b); else {
          if (J) {
            var m = b.childNodes[0], e, f = []; for (e = m.attributes.length - 1; 0 <= e; e--) { var g = m.attributes[e].nodeName; var p = m.attributes[e].nodeValue; -1 !== g.indexOf(":") && (f.push({ name: g, value: p }), m.removeAttribute(g)) } e = 0; for (g = f.length; e < g; e++)p = b.createAttribute(f[e].name.replace(":",
              "_dt_b_namespace_token_")), p.value = f[e].value, m.setAttributeNode(p)
          } b = M.serializeToString(b); J && (-1 === b.indexOf("<?xml") && (b = '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>' + b), b = b.replace(/_dt_b_namespace_token_/g, ":"), b = b.replace(/xmlns:NS[\d]+="" NS[\d]+:/g, "")); b = b.replace(/<([^<>]*?) xmlns=""([^<>]*?)>/g, "<$1 $2>"); a.file(d, b)
        }
      })
    } function y(a, c, d) {
      var b = a.createElement(c); d && (d.attr && n(b).attr(d.attr), d.children && n.each(d.children, function (m, e) { b.appendChild(e) }), null !== d.text && d.text !==
        B && b.appendChild(a.createTextNode(d.text))); return b
    } function V(a, c) { var d = a.header[c].length; a.footer && a.footer[c].length > d && (d = a.footer[c].length); for (var b = 0, m = a.body.length; b < m; b++) { var e = a.body[b][c]; e = null !== e && e !== B ? e.toString() : ""; -1 !== e.indexOf("\n") ? (e = e.split("\n"), e.sort(function (f, g) { return g.length - f.length }), e = e[0].length) : e = e.length; e > d && (d = e); if (40 < d) return 54 } d *= 1.35; return 6 < d ? d : 6 } var D = n.fn.dataTable; D.Buttons.pdfMake = function (a) { if (!a) return F || u.pdfMake; F = a }; D.Buttons.jszip =
      function (a) { if (!a) return E || u.JSZip; E = a }; var K = function (a) {
        if (!("undefined" === typeof a || "undefined" !== typeof navigator && /MSIE [1-9]\./.test(navigator.userAgent))) {
          var c = a.document.createElementNS("http://www.w3.org/1999/xhtml", "a"), d = "download" in c, b = /constructor/i.test(a.HTMLElement) || a.safari, m = /CriOS\/[\d]+/.test(navigator.userAgent), e = function (h) { (a.setImmediate || a.setTimeout)(function () { throw h; }, 0) }, f = function (h) {
            setTimeout(function () {
              "string" === typeof h ? (a.URL || a.webkitURL || a).revokeObjectURL(h) :
              h.remove()
            }, 4E4)
          }, g = function (h) { return /^\s*(?:text\/\S*|application\/xml|\S*\/\S*\+xml)\s*;.*charset\s*=\s*utf-8/i.test(h.type) ? new Blob([String.fromCharCode(65279), h], { type: h.type }) : h }, p = function (h, q, v) {
            v || (h = g(h)); var r = this, w = "application/octet-stream" === h.type, C = function () { var l = ["writestart", "progress", "write", "writeend"]; l = [].concat(l); for (var z = l.length; z--;) { var G = r["on" + l[z]]; if ("function" === typeof G) try { G.call(r, r) } catch (A) { e(A) } } }; r.readyState = r.INIT; if (d) {
              var k = (a.URL || a.webkitURL || a).createObjectURL(h);
              setTimeout(function () { c.href = k; c.download = q; var l = new MouseEvent("click"); c.dispatchEvent(l); C(); f(k); r.readyState = r.DONE })
            } else (function () {
              if ((m || w && b) && a.FileReader) { var l = new FileReader; l.onloadend = function () { var z = m ? l.result : l.result.replace(/^data:[^;]*;/, "data:attachment/file;"); a.open(z, "_blank") || (a.location.href = z); r.readyState = r.DONE; C() }; l.readAsDataURL(h); r.readyState = r.INIT } else k || (k = (a.URL || a.webkitURL || a).createObjectURL(h)), w ? a.location.href = k : a.open(k, "_blank") || (a.location.href =
                k), r.readyState = r.DONE, C(), f(k)
            })()
          }, t = p.prototype; if ("undefined" !== typeof navigator && navigator.msSaveOrOpenBlob) return function (h, q, v) { q = q || h.name || "download"; v || (h = g(h)); return navigator.msSaveOrOpenBlob(h, q) }; t.abort = function () { }; t.readyState = t.INIT = 0; t.WRITING = 1; t.DONE = 2; t.error = t.onwritestart = t.onprogress = t.onwrite = t.onabort = t.onerror = t.onwriteend = null; return function (h, q, v) { return new p(h, q || h.name || "download", v) }
        }
      }("undefined" !== typeof self && self || "undefined" !== typeof u && u || this.content); D.fileSave =
        K; var Q = function (a) { var c = "Sheet1"; a.sheetName && (c = a.sheetName.replace(/[\[\]\*\/\\\?:]/g, "")); return c }, R = function (a) { return a.newline ? a.newline : navigator.userAgent.match(/Windows/) ? "\r\n" : "\n" }, S = function (a, c) {
          var d = R(c); a = a.buttons.exportData(c.exportOptions); var b = c.fieldBoundary, m = c.fieldSeparator, e = new RegExp(b, "g"), f = c.escapeChar !== B ? c.escapeChar : "\\", g = function (v) { for (var r = "", w = 0, C = v.length; w < C; w++)0 < w && (r += m), r += b ? b + ("" + v[w]).replace(e, f + b) + b : v[w]; return r }, p = c.header ? g(a.header) + d : ""; c =
            c.footer && a.footer ? d + g(a.footer) : ""; for (var t = [], h = 0, q = a.body.length; h < q; h++)t.push(g(a.body[h])); return { str: p + t.join(d) + c, rows: t.length }
        }, T = function () { if (-1 === navigator.userAgent.indexOf("Safari") || -1 !== navigator.userAgent.indexOf("Chrome") || -1 !== navigator.userAgent.indexOf("Opera")) return !1; var a = navigator.userAgent.match(/AppleWebKit\/(\d+\.\d+)/); return a && 1 < a.length && 603.1 > 1 * a[1] ? !0 : !1 }; try { var M = new XMLSerializer, J } catch (a) { } var P = {
          "_rels/.rels": '<?xml version="1.0" encoding="UTF-8" standalone="yes"?><Relationships xmlns="http://schemas.openxmlformats.org/package/2006/relationships"><Relationship Id="rId1" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/officeDocument" Target="xl/workbook.xml"/></Relationships>',
          "xl/_rels/workbook.xml.rels": '<?xml version="1.0" encoding="UTF-8" standalone="yes"?><Relationships xmlns="http://schemas.openxmlformats.org/package/2006/relationships"><Relationship Id="rId1" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/worksheet" Target="worksheets/sheet1.xml"/><Relationship Id="rId2" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/styles" Target="styles.xml"/></Relationships>', "[Content_Types].xml": '<?xml version="1.0" encoding="UTF-8" standalone="yes"?><Types xmlns="http://schemas.openxmlformats.org/package/2006/content-types"><Default Extension="xml" ContentType="application/xml" /><Default Extension="rels" ContentType="application/vnd.openxmlformats-package.relationships+xml" /><Default Extension="jpeg" ContentType="image/jpeg" /><Override PartName="/xl/workbook.xml" ContentType="application/vnd.openxmlformats-officedocument.spreadsheetml.sheet.main+xml" /><Override PartName="/xl/worksheets/sheet1.xml" ContentType="application/vnd.openxmlformats-officedocument.spreadsheetml.worksheet+xml" /><Override PartName="/xl/styles.xml" ContentType="application/vnd.openxmlformats-officedocument.spreadsheetml.styles+xml" /></Types>',
          "xl/workbook.xml": '<?xml version="1.0" encoding="UTF-8" standalone="yes"?><workbook xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main" xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships"><fileVersion appName="xl" lastEdited="5" lowestEdited="5" rupBuild="24816"/><workbookPr showInkAnnotation="0" autoCompressPictures="0"/><bookViews><workbookView xWindow="0" yWindow="0" windowWidth="25600" windowHeight="19020" tabRatio="500"/></bookViews><sheets><sheet name="Sheet1" sheetId="1" r:id="rId1"/></sheets><definedNames/></workbook>',
          "xl/worksheets/sheet1.xml": '<?xml version="1.0" encoding="UTF-8" standalone="yes"?><worksheet xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main" xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" mc:Ignorable="x14ac" xmlns:x14ac="http://schemas.microsoft.com/office/spreadsheetml/2009/9/ac"><sheetData/><mergeCells count="0"/></worksheet>', "xl/styles.xml": '<?xml version="1.0" encoding="UTF-8"?><styleSheet xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" mc:Ignorable="x14ac" xmlns:x14ac="http://schemas.microsoft.com/office/spreadsheetml/2009/9/ac"><numFmts count="6"><numFmt numFmtId="164" formatCode="#,##0.00_- [$$-45C]"/><numFmt numFmtId="165" formatCode="&quot;£&quot;#,##0.00"/><numFmt numFmtId="166" formatCode="[$€-2] #,##0.00"/><numFmt numFmtId="167" formatCode="0.0%"/><numFmt numFmtId="168" formatCode="#,##0;(#,##0)"/><numFmt numFmtId="169" formatCode="#,##0.00;(#,##0.00)"/></numFmts><fonts count="5" x14ac:knownFonts="1"><font><sz val="11" /><name val="Calibri" /></font><font><sz val="11" /><name val="Calibri" /><color rgb="FFFFFFFF" /></font><font><sz val="11" /><name val="Calibri" /><b /></font><font><sz val="11" /><name val="Calibri" /><i /></font><font><sz val="11" /><name val="Calibri" /><u /></font></fonts><fills count="6"><fill><patternFill patternType="none" /></fill><fill><patternFill patternType="none" /></fill><fill><patternFill patternType="solid"><fgColor rgb="FFD9D9D9" /><bgColor indexed="64" /></patternFill></fill><fill><patternFill patternType="solid"><fgColor rgb="FFD99795" /><bgColor indexed="64" /></patternFill></fill><fill><patternFill patternType="solid"><fgColor rgb="ffc6efce" /><bgColor indexed="64" /></patternFill></fill><fill><patternFill patternType="solid"><fgColor rgb="ffc6cfef" /><bgColor indexed="64" /></patternFill></fill></fills><borders count="2"><border><left /><right /><top /><bottom /><diagonal /></border><border diagonalUp="false" diagonalDown="false"><left style="thin"><color auto="1" /></left><right style="thin"><color auto="1" /></right><top style="thin"><color auto="1" /></top><bottom style="thin"><color auto="1" /></bottom><diagonal /></border></borders><cellStyleXfs count="1"><xf numFmtId="0" fontId="0" fillId="0" borderId="0" /></cellStyleXfs><cellXfs count="68"><xf numFmtId="0" fontId="0" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="1" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="2" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="3" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="4" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="0" fillId="2" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="1" fillId="2" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="2" fillId="2" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="3" fillId="2" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="4" fillId="2" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="0" fillId="3" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="1" fillId="3" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="2" fillId="3" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="3" fillId="3" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="4" fillId="3" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="0" fillId="4" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="1" fillId="4" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="2" fillId="4" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="3" fillId="4" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="4" fillId="4" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="0" fillId="5" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="1" fillId="5" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="2" fillId="5" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="3" fillId="5" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="4" fillId="5" borderId="0" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="0" fillId="0" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="1" fillId="0" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="2" fillId="0" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="3" fillId="0" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="4" fillId="0" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="0" fillId="2" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="1" fillId="2" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="2" fillId="2" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="3" fillId="2" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="4" fillId="2" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="0" fillId="3" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="1" fillId="3" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="2" fillId="3" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="3" fillId="3" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="4" fillId="3" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="0" fillId="4" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="1" fillId="4" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="2" fillId="4" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="3" fillId="4" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="4" fillId="4" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="0" fillId="5" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="1" fillId="5" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="2" fillId="5" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="3" fillId="5" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="4" fillId="5" borderId="1" applyFont="1" applyFill="1" applyBorder="1"/><xf numFmtId="0" fontId="0" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1" xfId="0" applyAlignment="1"><alignment horizontal="left"/></xf><xf numFmtId="0" fontId="0" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1" xfId="0" applyAlignment="1"><alignment horizontal="center"/></xf><xf numFmtId="0" fontId="0" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1" xfId="0" applyAlignment="1"><alignment horizontal="right"/></xf><xf numFmtId="0" fontId="0" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1" xfId="0" applyAlignment="1"><alignment horizontal="fill"/></xf><xf numFmtId="0" fontId="0" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1" xfId="0" applyAlignment="1"><alignment textRotation="90"/></xf><xf numFmtId="0" fontId="0" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1" xfId="0" applyAlignment="1"><alignment wrapText="1"/></xf><xf numFmtId="9"   fontId="0" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1" xfId="0" applyNumberFormat="1"/><xf numFmtId="164" fontId="0" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1" xfId="0" applyNumberFormat="1"/><xf numFmtId="165" fontId="0" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1" xfId="0" applyNumberFormat="1"/><xf numFmtId="166" fontId="0" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1" xfId="0" applyNumberFormat="1"/><xf numFmtId="167" fontId="0" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1" xfId="0" applyNumberFormat="1"/><xf numFmtId="168" fontId="0" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1" xfId="0" applyNumberFormat="1"/><xf numFmtId="169" fontId="0" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1" xfId="0" applyNumberFormat="1"/><xf numFmtId="3" fontId="0" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1" xfId="0" applyNumberFormat="1"/><xf numFmtId="4" fontId="0" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1" xfId="0" applyNumberFormat="1"/><xf numFmtId="1" fontId="0" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1" xfId="0" applyNumberFormat="1"/><xf numFmtId="2" fontId="0" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1" xfId="0" applyNumberFormat="1"/><xf numFmtId="14" fontId="0" fillId="0" borderId="0" applyFont="1" applyFill="1" applyBorder="1" xfId="0" applyNumberFormat="1"/></cellXfs><cellStyles count="1"><cellStyle name="Normal" xfId="0" builtinId="0" /></cellStyles><dxfs count="0" /><tableStyles count="0" defaultTableStyle="TableStyleMedium9" defaultPivotStyle="PivotStyleMedium4" /></styleSheet>'
        },
          U = [{ match: /^\-?\d+\.\d%$/, style: 60, fmt: function (a) { return a / 100 } }, { match: /^\-?\d+\.?\d*%$/, style: 56, fmt: function (a) { return a / 100 } }, { match: /^\-?\$[\d,]+.?\d*$/, style: 57 }, { match: /^\-?£[\d,]+.?\d*$/, style: 58 }, { match: /^\-?€[\d,]+.?\d*$/, style: 59 }, { match: /^\-?\d+$/, style: 65 }, { match: /^\-?\d+\.\d{2}$/, style: 66 }, { match: /^\([\d,]+\)$/, style: 61, fmt: function (a) { return -1 * a.replace(/[\(\)]/g, "") } }, { match: /^\([\d,]+\.\d{2}\)$/, style: 62, fmt: function (a) { return -1 * a.replace(/[\(\)]/g, "") } }, { match: /^\-?[\d,]+$/, style: 63 },
          { match: /^\-?[\d,]+\.\d{2}$/, style: 64 }, { match: /^[\d]{4}\-[01][\d]\-[0123][\d]$/, style: 67, fmt: function (a) { return Math.round(25569 + Date.parse(a) / 864E5) } }]; D.ext.buttons.copyHtml5 = {
            className: "buttons-copy buttons-html5", text: function (a) { return a.i18n("buttons.copy", "Copy") }, action: function (a, c, d, b) {
              this.processing(!0); var m = this; a = S(c, b); var e = c.buttons.exportInfo(b), f = R(b), g = a.str; d = n("<div/>").css({ height: 1, width: 1, overflow: "hidden", position: "fixed", top: 0, left: 0 }); e.title && (g = e.title + f + f + g); e.messageTop &&
                (g = e.messageTop + f + f + g); e.messageBottom && (g = g + f + f + e.messageBottom); b.customize && (g = b.customize(g, b, c)); b = n("<textarea readonly/>").val(g).appendTo(d); if (x.queryCommandSupported("copy")) { d.appendTo(c.table().container()); b[0].focus(); b[0].select(); try { var p = x.execCommand("copy"); d.remove(); if (p) { c.buttons.info(c.i18n("buttons.copyTitle", "Copy to clipboard"), c.i18n("buttons.copySuccess", { 1: "Copied one row to clipboard", _: "Copied %d rows to clipboard" }, a.rows), 2E3); this.processing(!1); return } } catch (q) { } } p =
                  n("<span>" + c.i18n("buttons.copyKeys", "Press <i>ctrl</i> or <i>⌘</i> + <i>C</i> to copy the table data<br>to your system clipboard.<br><br>To cancel, click this message or press escape.") + "</span>").append(d); c.buttons.info(c.i18n("buttons.copyTitle", "Copy to clipboard"), p, 0); b[0].focus(); b[0].select(); var t = n(p).closest(".dt-button-info"), h = function () { t.off("click.buttons-copy"); n(x).off(".buttons-copy"); c.buttons.info(!1) }; t.on("click.buttons-copy", h); n(x).on("keydown.buttons-copy", function (q) {
                    27 ===
                    q.keyCode && (h(), m.processing(!1))
                  }).on("copy.buttons-copy cut.buttons-copy", function () { h(); m.processing(!1) })
            }, exportOptions: {}, fieldSeparator: "\t", fieldBoundary: "", header: !0, footer: !1, title: "*", messageTop: "*", messageBottom: "*"
          }; D.ext.buttons.csvHtml5 = {
            bom: !1, className: "buttons-csv buttons-html5", available: function () { return u.FileReader !== B && u.Blob }, text: function (a) { return a.i18n("buttons.csv", "CSV") }, action: function (a, c, d, b) {
              this.processing(!0); a = S(c, b).str; d = c.buttons.exportInfo(b); var m = b.charset;
              b.customize && (a = b.customize(a, b, c)); !1 !== m ? (m || (m = x.characterSet || x.charset), m && (m = ";charset=" + m)) : m = ""; b.bom && (a = String.fromCharCode(65279) + a); K(new Blob([a], { type: "text/csv" + m }), d.filename, !0); this.processing(!1)
            }, filename: "*", extension: ".csv", exportOptions: {}, fieldSeparator: ",", fieldBoundary: '"', escapeChar: '"', charset: null, header: !0, footer: !1
          }; D.ext.buttons.excelHtml5 = {
            className: "buttons-excel buttons-html5", available: function () { return u.FileReader !== B && (E || u.JSZip) !== B && !T() && M }, text: function (a) {
              return a.i18n("buttons.excel",
                "Excel")
            }, action: function (a, c, d, b) {
              this.processing(!0); var m = this, e = 0; a = function (k) { return n.parseXML(P[k]) }; var f = a("xl/worksheets/sheet1.xml"), g = f.getElementsByTagName("sheetData")[0]; a = { _rels: { ".rels": a("_rels/.rels") }, xl: { _rels: { "workbook.xml.rels": a("xl/_rels/workbook.xml.rels") }, "workbook.xml": a("xl/workbook.xml"), "styles.xml": a("xl/styles.xml"), worksheets: { "sheet1.xml": f } }, "[Content_Types].xml": a("[Content_Types].xml") }; var p = c.buttons.exportData(b.exportOptions), t, h, q = function (k) {
                t = e + 1; h =
                  y(f, "row", { attr: { r: t } }); for (var l = 0, z = k.length; l < z; l++) {
                    var G = I(l) + "" + t, A = null; if (null === k[l] || k[l] === B || "" === k[l]) if (!0 === b.createEmptyCells) k[l] = ""; else continue; var H = k[l]; k[l] = "function" === typeof k[l].trim ? k[l].trim() : k[l]; for (var N = 0, W = U.length; N < W; N++) { var L = U[N]; if (k[l].match && !k[l].match(/^0\d+/) && k[l].match(L.match)) { A = k[l].replace(/[^\d\.\-]/g, ""); L.fmt && (A = L.fmt(A)); A = y(f, "c", { attr: { r: G, s: L.style }, children: [y(f, "v", { text: A })] }); break } } A || ("number" === typeof k[l] || k[l].match && k[l].match(/^-?\d+(\.\d+)?([eE]\-?\d+)?$/) &&
                      !k[l].match(/^0\d+/) ? A = y(f, "c", { attr: { t: "n", r: G }, children: [y(f, "v", { text: k[l] })] }) : (H = H.replace ? H.replace(/[\x00-\x09\x0B\x0C\x0E-\x1F\x7F-\x9F]/g, "") : H, A = y(f, "c", { attr: { t: "inlineStr", r: G }, children: { row: y(f, "is", { children: { row: y(f, "t", { text: H, attr: { "xml:space": "preserve" } }) } }) } }))); h.appendChild(A)
                  } g.appendChild(h); e++
              }; b.customizeData && b.customizeData(p); var v = function (k, l) {
                var z = n("mergeCells", f); z[0].appendChild(y(f, "mergeCell", { attr: { ref: "A" + k + ":" + I(l) + k } })); z.attr("count", parseFloat(z.attr("count")) +
                  1); n("row:eq(" + (k - 1) + ") c", f).attr("s", "51")
              }, r = c.buttons.exportInfo(b); r.title && (q([r.title], e), v(e, p.header.length - 1)); r.messageTop && (q([r.messageTop], e), v(e, p.header.length - 1)); b.header && (q(p.header, e), n("row:last c", f).attr("s", "2")); d = e; var w = 0; for (var C = p.body.length; w < C; w++)q(p.body[w], e); w = e; b.footer && p.footer && (q(p.footer, e), n("row:last c", f).attr("s", "2")); r.messageBottom && (q([r.messageBottom], e), v(e, p.header.length - 1)); q = y(f, "cols"); n("worksheet", f).prepend(q); v = 0; for (C = p.header.length; v <
                C; v++)q.appendChild(y(f, "col", { attr: { min: v + 1, max: v + 1, width: V(p, v), customWidth: 1 } })); q = a.xl["workbook.xml"]; n("sheets sheet", q).attr("name", Q(b)); b.autoFilter && (n("mergeCells", f).before(y(f, "autoFilter", { attr: { ref: "A" + d + ":" + I(p.header.length - 1) + w } })), n("definedNames", q).append(y(q, "definedName", { attr: { name: "_xlnm._FilterDatabase", localSheetId: "0", hidden: 1 }, text: Q(b) + "!$A$" + d + ":" + I(p.header.length - 1) + w }))); b.customize && b.customize(a, b, c); 0 === n("mergeCells", f).children().length && n("mergeCells", f).remove();
              c = new (E || u.JSZip); d = { type: "blob", mimeType: "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet" }; O(c, a); c.generateAsync ? c.generateAsync(d).then(function (k) { K(k, r.filename); m.processing(!1) }) : (K(c.generate(d), r.filename), this.processing(!1))
            }, filename: "*", extension: ".xlsx", exportOptions: {}, header: !0, footer: !1, title: "*", messageTop: "*", messageBottom: "*", createEmptyCells: !1, autoFilter: !1, sheetName: ""
          }; D.ext.buttons.pdfHtml5 = {
            className: "buttons-pdf buttons-html5", available: function () {
              return u.FileReader !==
                B && (F || u.pdfMake)
            }, text: function (a) { return a.i18n("buttons.pdf", "PDF") }, action: function (a, c, d, b) {
              this.processing(!0); d = c.buttons.exportData(b.exportOptions); a = c.buttons.exportInfo(b); var m = []; b.header && m.push(n.map(d.header, function (g) { return { text: "string" === typeof g ? g : g + "", style: "tableHeader" } })); for (var e = 0, f = d.body.length; e < f; e++)m.push(n.map(d.body[e], function (g) { if (null === g || g === B) g = ""; return { text: "string" === typeof g ? g : g + "", style: e % 2 ? "tableBodyEven" : "tableBodyOdd" } })); b.footer && d.footer && m.push(n.map(d.footer,
                function (g) { return { text: "string" === typeof g ? g : g + "", style: "tableFooter" } })); d = { pageSize: b.pageSize, pageOrientation: b.orientation, content: [{ table: { headerRows: 1, body: m }, layout: "noBorders" }], styles: { tableHeader: { bold: !0, fontSize: 11, color: "white", fillColor: "#2d4154", alignment: "center" }, tableBodyEven: {}, tableBodyOdd: { fillColor: "#f3f3f3" }, tableFooter: { bold: !0, fontSize: 11, color: "white", fillColor: "#2d4154" }, title: { alignment: "center", fontSize: 15 }, message: {} }, defaultStyle: { fontSize: 10 } }; a.messageTop && d.content.unshift({
                  text: a.messageTop,
                  style: "message", margin: [0, 0, 0, 12]
                }); a.messageBottom && d.content.push({ text: a.messageBottom, style: "message", margin: [0, 0, 0, 12] }); a.title && d.content.unshift({ text: a.title, style: "title", margin: [0, 0, 0, 12] }); b.customize && b.customize(d, b, c); c = (F || u.pdfMake).createPdf(d); "open" !== b.download || T() ? c.download(a.filename) : c.open(); this.processing(!1)
            }, title: "*", filename: "*", extension: ".pdf", exportOptions: {}, orientation: "portrait", pageSize: "A4", header: !0, footer: !1, messageTop: "*", messageBottom: "*", customize: null,
            download: "download"
          }; return D.Buttons
  });
</script>
<script>/*!
Column visibility buttons for Buttons and DataTables.
2016 SpryMedia Ltd - datatables.net/license
*/
  (function (h) { "function" === typeof define && define.amd ? define(["jquery", "datatables.net", "datatables.net-buttons"], function (e) { return h(e, window, document) }) : "object" === typeof exports ? module.exports = function (e, g) { e || (e = window); g && g.fn.dataTable || (g = require("datatables.net")(e, g).$); g.fn.dataTable.Buttons || require("datatables.net-buttons")(e, g); return h(g, e, e.document) } : h(jQuery, window, document) })(function (h, e, g, l) {
    e = h.fn.dataTable; h.extend(e.ext.buttons, {
      colvis: function (b, a) {
        var c = null, d = {
          extend: "collection",
          init: function (f, k) { c = k }, text: function (f) { return f.i18n("buttons.colvis", "Column visibility") }, className: "buttons-colvis", closeButton: !1, buttons: [{ extend: "columnsToggle", columns: a.columns, columnText: a.columnText }]
        }; b.on("column-reorder.dt" + a.namespace, function (f, k, m) { b.button(null, b.button(null, c).node()).collectionRebuild([{ extend: "columnsToggle", columns: a.columns, columnText: a.columnText }]) }); return d
      }, columnsToggle: function (b, a) {
        return b.columns(a.columns).indexes().map(function (c) {
          return {
            extend: "columnToggle",
            columns: c, columnText: a.columnText
          }
        }).toArray()
      }, columnToggle: function (b, a) { return { extend: "columnVisibility", columns: a.columns, columnText: a.columnText } }, columnsVisibility: function (b, a) { return b.columns(a.columns).indexes().map(function (c) { return { extend: "columnVisibility", columns: c, visibility: a.visibility, columnText: a.columnText } }).toArray() }, columnVisibility: {
        columns: l, text: function (b, a, c) { return c._columnText(b, c) }, className: "buttons-columnVisibility", action: function (b, a, c, d) {
          b = a.columns(d.columns);
          a = b.visible(); b.visible(d.visibility !== l ? d.visibility : !(a.length && a[0]))
        }, init: function (b, a, c) { var d = this; a.attr("data-cv-idx", c.columns); b.on("column-visibility.dt" + c.namespace, function (f, k) { k.bDestroying || k.nTable != b.settings()[0].nTable || d.active(b.column(c.columns).visible()) }).on("column-reorder.dt" + c.namespace, function (f, k, m) { c.destroying || 1 !== b.columns(c.columns).count() || (d.text(c._columnText(b, c)), d.active(b.column(c.columns).visible())) }); this.active(b.column(c.columns).visible()) }, destroy: function (b,
          a, c) { b.off("column-visibility.dt" + c.namespace).off("column-reorder.dt" + c.namespace) }, _columnText: function (b, a) { var c = b.column(a.columns).index(), d = b.settings()[0].aoColumns[c].sTitle; d || (d = b.column(c).header().innerHTML); d = d.replace(/\n/g, " ").replace(/<br\s*\/?>/gi, " ").replace(/<select(.*?)<\/select>/g, "").replace(/<!\-\-.*?\-\->/g, "").replace(/<.*?>/g, "").replace(/^\s+|\s+$/g, ""); return a.columnText ? a.columnText(b, c, d) : d }
      }, colvisRestore: {
        className: "buttons-colvisRestore", text: function (b) {
          return b.i18n("buttons.colvisRestore",
            "Restore visibility")
        }, init: function (b, a, c) { c._visOriginal = b.columns().indexes().map(function (d) { return b.column(d).visible() }).toArray() }, action: function (b, a, c, d) { a.columns().every(function (f) { f = a.colReorder && a.colReorder.transpose ? a.colReorder.transpose(f, "toOriginal") : f; this.visible(d._visOriginal[f]) }) }
      }, colvisGroup: { className: "buttons-colvisGroup", action: function (b, a, c, d) { a.columns(d.show).visible(!0, !1); a.columns(d.hide).visible(!1, !1); a.columns.adjust() }, show: [], hide: [] }
    }); return e.Buttons
  });
</script>
<script>/*!
Print button for Buttons and DataTables.
2016 SpryMedia Ltd - datatables.net/license
*/
  (function (b) { "function" === typeof define && define.amd ? define(["jquery", "datatables.net", "datatables.net-buttons"], function (d) { return b(d, window, document) }) : "object" === typeof exports ? module.exports = function (d, h) { d || (d = window); h && h.fn.dataTable || (h = require("datatables.net")(d, h).$); h.fn.dataTable.Buttons || require("datatables.net-buttons")(d, h); return b(h, d, d.document) } : b(jQuery, window, document) })(function (b, d, h, y) {
    var u = b.fn.dataTable, n = h.createElement("a"), v = function (a) {
      n.href = a; a = n.host; -1 === a.indexOf("/") &&
        0 !== n.pathname.indexOf("/") && (a += "/"); return n.protocol + "//" + a + n.pathname + n.search
    }; u.ext.buttons.print = {
      className: "buttons-print", text: function (a) { return a.i18n("buttons.print", "Print") }, action: function (a, e, p, k) {
        a = e.buttons.exportData(b.extend({ decodeEntities: !1 }, k.exportOptions)); p = e.buttons.exportInfo(k); var w = e.columns(k.exportOptions.columns).flatten().map(function (f) { return e.settings()[0].aoColumns[e.column(f).index()].sClass }).toArray(), r = function (f, g) {
          for (var x = "<tr>", l = 0, z = f.length; l < z; l++)x +=
            "<" + g + " " + (w[l] ? 'class="' + w[l] + '"' : "") + ">" + (null === f[l] || f[l] === y ? "" : f[l]) + "</" + g + ">"; return x + "</tr>"
        }, m = '<table class="' + e.table().node().className + '">'; k.header && (m += "<thead>" + r(a.header, "th") + "</thead>"); m += "<tbody>"; for (var t = 0, A = a.body.length; t < A; t++)m += r(a.body[t], "td"); m += "</tbody>"; k.footer && a.footer && (m += "<tfoot>" + r(a.footer, "th") + "</tfoot>"); m += "</table>"; var c = d.open("", ""); if (c) {
          c.document.close(); var q = "<title>" + p.title + "</title>"; b("style, link").each(function () {
            var f = q, g = b(this).clone()[0];
            "link" === g.nodeName.toLowerCase() && (g.href = v(g.href)); q = f + g.outerHTML
          }); try { c.document.head.innerHTML = q } catch (f) { b(c.document.head).html(q) } c.document.body.innerHTML = "<h1>" + p.title + "</h1><div>" + (p.messageTop || "") + "</div>" + m + "<div>" + (p.messageBottom || "") + "</div>"; b(c.document.body).addClass("dt-print-view"); b("img", c.document.body).each(function (f, g) { g.setAttribute("src", v(g.getAttribute("src"))) }); k.customize && k.customize(c, k, e); a = function () { k.autoPrint && (c.print(), c.close()) }; navigator.userAgent.match(/Trident\/\d.\d/) ?
            a() : c.setTimeout(a, 1E3)
        } else e.buttons.info(e.i18n("buttons.printErrorTitle", "Unable to open print view"), e.i18n("buttons.printErrorMsg", "Please allow popups in your browser for this site to be able to view the print view."), 5E3)
      }, title: "*", messageTop: "*", messageBottom: "*", exportOptions: {}, header: !0, footer: !1, autoPrint: !0, customize: null
    }; return u.Buttons
  });
</script>
<style type="text/css">
  .container-fluid.crosstalk-bscols {
    margin-left: -30px;
    margin-right: -30px;
    white-space: normal
  }

  body>.container-fluid.crosstalk-bscols {
    margin-left: auto;
    margin-right: auto
  }

  .crosstalk-input-checkboxgroup .crosstalk-options-group .crosstalk-options-column {
    display: inline-block;
    padding-right: 12px;
    vertical-align: top
  }

  @media only screen and (max-width: 480px) {
    .crosstalk-input-checkboxgroup .crosstalk-options-group .crosstalk-options-column {
      display: block;
      padding-right: inherit
    }
  }

  .crosstalk-input {
    margin-bottom: 15px
  }

  .crosstalk-input .control-label {
    margin-bottom: 0;
    vertical-align: middle
  }

  .crosstalk-input input[type="checkbox"] {
    margin: 4px 0 0;
    margin-top: 1px;
    line-height: normal
  }

  .crosstalk-input .checkbox {
    position: relative;
    display: block;
    margin-top: 10px;
    margin-bottom: 10px
  }

  .crosstalk-input .checkbox>label {
    padding-left: 20px;
    margin-bottom: 0;
    font-weight: 400;
    cursor: pointer
  }

  .crosstalk-input .checkbox input[type="checkbox"],
  .crosstalk-input .checkbox-inline input[type="checkbox"] {
    position: absolute;
    margin-top: 2px;
    margin-left: -20px
  }

  .crosstalk-input .checkbox+.checkbox {
    margin-top: -5px
  }

  .crosstalk-input .checkbox-inline {
    position: relative;
    display: inline-block;
    padding-left: 20px;
    margin-bottom: 0;
    font-weight: 400;
    vertical-align: middle;
    cursor: pointer
  }

  .crosstalk-input .checkbox-inline+.checkbox-inline {
    margin-top: 0;
    margin-left: 10px
  }

</style>
<script>!function o(u, a, l) { function s(n, e) { if (!a[n]) { if (!u[n]) { var t = "function" == typeof require && require; if (!e && t) return t(n, !0); if (f) return f(n, !0); var r = new Error("Cannot find module '" + n + "'"); throw r.code = "MODULE_NOT_FOUND", r } var i = a[n] = { exports: {} }; u[n][0].call(i.exports, function (e) { var t = u[n][1][e]; return s(t || e) }, i, i.exports, o, u, a, l) } return a[n].exports } for (var f = "function" == typeof require && require, e = 0; e < l.length; e++)s(l[e]); return s }({ 1: [function (e, t, n) { "use strict"; Object.defineProperty(n, "__esModule", { value: !0 }); var r = function () { function r(e, t) { for (var n = 0; n < t.length; n++) { var r = t[n]; r.enumerable = r.enumerable || !1, r.configurable = !0, "value" in r && (r.writable = !0), Object.defineProperty(e, r.key, r) } } return function (e, t, n) { return t && r(e.prototype, t), n && r(e, n), e } }(); var i = function () { function e() { !function (e, t) { if (!(e instanceof t)) throw new TypeError("Cannot call a class as a function") }(this, e), this._types = {}, this._seq = 0 } return r(e, [{ key: "on", value: function (e, t) { var n = this._types[e]; n || (n = this._types[e] = {}); var r = "sub" + this._seq++; return n[r] = t, r } }, { key: "off", value: function (e, t) { var n = this._types[e]; if ("function" == typeof t) { for (var r in n) if (n.hasOwnProperty(r) && n[r] === t) return delete n[r], r; return !1 } if ("string" == typeof t) return !(!n || !n[t]) && (delete n[t], t); throw new Error("Unexpected type for listener") } }, { key: "trigger", value: function (e, t, n) { var r = this._types[e]; for (var i in r) r.hasOwnProperty(i) && r[i].call(n, t) } }]), e }(); n.default = i }, {}], 2: [function (e, t, n) { "use strict"; Object.defineProperty(n, "__esModule", { value: !0 }), n.FilterHandle = void 0; var r = function () { function r(e, t) { for (var n = 0; n < t.length; n++) { var r = t[n]; r.enumerable = r.enumerable || !1, r.configurable = !0, "value" in r && (r.writable = !0), Object.defineProperty(e, r.key, r) } } return function (e, t, n) { return t && r(e.prototype, t), n && r(e, n), e } }(), i = l(e("./events")), o = l(e("./filterset")), u = l(e("./group")), a = function (e) { { if (e && e.__esModule) return e; var t = {}; if (null != e) for (var n in e) Object.prototype.hasOwnProperty.call(e, n) && (t[n] = e[n]); return t.default = e, t } }(e("./util")); function l(e) { return e && e.__esModule ? e : { default: e } } var s = 1; n.FilterHandle = function () { function n(e, t) { !function (e, t) { if (!(e instanceof t)) throw new TypeError("Cannot call a class as a function") }(this, n), this._eventRelay = new i.default, this._emitter = new a.SubscriptionTracker(this._eventRelay), this._group = null, this._filterSet = null, this._filterVar = null, this._varOnChangeSub = null, this._extraInfo = a.extend({ sender: this }, t), this._id = "filter" + s++, this.setGroup(e) } return r(n, [{ key: "setGroup", value: function (e) { var t, n, r = this; if (this._group !== e && ((this._group || e) && (this._filterVar && (this._filterVar.off("change", this._varOnChangeSub), this.clear(), this._varOnChangeSub = null, this._filterVar = null, this._filterSet = null), this._group = e))) { e = (0, u.default)(e), this._filterSet = (t = e.var("filterset"), (n = t.get()) || (n = new o.default, t.set(n)), n), this._filterVar = (0, u.default)(e).var("filter"); var i = this._filterVar.on("change", function (e) { r._eventRelay.trigger("change", e, r) }); this._varOnChangeSub = i } } }, { key: "_mergeExtraInfo", value: function (e) { return a.extend({}, this._extraInfo ? this._extraInfo : null, e || null) } }, { key: "close", value: function () { this._emitter.removeAllListeners(), this.clear(), this.setGroup(null) } }, { key: "clear", value: function (e) { this._filterSet && (this._filterSet.clear(this._id), this._onChange(e)) } }, { key: "set", value: function (e, t) { this._filterSet && (this._filterSet.update(this._id, e), this._onChange(t)) } }, { key: "on", value: function (e, t) { return this._emitter.on(e, t) } }, { key: "off", value: function (e, t) { return this._emitter.off(e, t) } }, { key: "_onChange", value: function (e) { this._filterSet && this._filterVar.set(this._filterSet.value, this._mergeExtraInfo(e)) } }, { key: "filteredKeys", get: function () { return this._filterSet ? this._filterSet.value : null } }]), n }() }, { "./events": 1, "./filterset": 3, "./group": 4, "./util": 11 }], 3: [function (e, t, n) { "use strict"; Object.defineProperty(n, "__esModule", { value: !0 }); var r = function () { function r(e, t) { for (var n = 0; n < t.length; n++) { var r = t[n]; r.enumerable = r.enumerable || !1, r.configurable = !0, "value" in r && (r.writable = !0), Object.defineProperty(e, r.key, r) } } return function (e, t, n) { return t && r(e.prototype, t), n && r(e, n), e } }(), a = e("./util"); function l(e, t) { return e === t ? 0 : e < t ? -1 : t < e ? 1 : void 0 } var i = function () { function e() { !function (e, t) { if (!(e instanceof t)) throw new TypeError("Cannot call a class as a function") }(this, e), this.reset() } return r(e, [{ key: "reset", value: function () { this._handles = {}, this._keys = {}, this._value = null, this._activeHandles = 0 } }, { key: "update", value: function (e, t) { null !== t && (t = t.slice(0)).sort(l); var n = (0, a.diffSortedLists)(this._handles[e], t), r = n.added, i = n.removed; this._handles[e] = t; for (var o = 0; o < r.length; o++)this._keys[r[o]] = (this._keys[r[o]] || 0) + 1; for (var u = 0; u < i.length; u++)this._keys[i[u]]--; this._updateValue(t) } }, { key: "_updateValue", value: function () { var e = 0 < arguments.length && void 0 !== arguments[0] ? arguments[0] : this._allKeys, t = Object.keys(this._handles).length; if (0 === t) this._value = null; else { this._value = []; for (var n = 0; n < e.length; n++) { this._keys[e[n]] === t && this._value.push(e[n]) } } } }, { key: "clear", value: function (e) { if (void 0 !== this._handles[e]) { var t = this._handles[e]; t || (t = []); for (var n = 0; n < t.length; n++)this._keys[t[n]]--; delete this._handles[e], this._updateValue() } } }, { key: "value", get: function () { return this._value } }, { key: "_allKeys", get: function () { var e = Object.keys(this._keys); return e.sort(l), e } }]), e }(); n.default = i }, { "./util": 11 }], 4: [function (l, e, s) { (function (e) { "use strict"; Object.defineProperty(s, "__esModule", { value: !0 }); var n = function () { function r(e, t) { for (var n = 0; n < t.length; n++) { var r = t[n]; r.enumerable = r.enumerable || !1, r.configurable = !0, "value" in r && (r.writable = !0), Object.defineProperty(e, r.key, r) } } return function (e, t, n) { return t && r(e.prototype, t), n && r(e, n), e } }(), r = "function" == typeof Symbol && "symbol" == typeof Symbol.iterator ? function (e) { return typeof e } : function (e) { return e && "function" == typeof Symbol && e.constructor === Symbol && e !== Symbol.prototype ? "symbol" : typeof e }; s.default = function e(t) { { if (t && "string" == typeof t) return u.hasOwnProperty(t) || (u[t] = new a(t)), u[t]; if ("object" === (void 0 === t ? "undefined" : r(t)) && t._vars && t.var) return t; if (Array.isArray(t) && 1 == t.length && "string" == typeof t[0]) return e(t[0]); throw new Error("Invalid groupName argument") } }; var t, i = l("./var"), o = (t = i) && t.__esModule ? t : { default: t }; e.__crosstalk_groups = e.__crosstalk_groups || {}; var u = e.__crosstalk_groups; var a = function () { function t(e) { !function (e, t) { if (!(e instanceof t)) throw new TypeError("Cannot call a class as a function") }(this, t), this.name = e, this._vars = {} } return n(t, [{ key: "var", value: function (e) { if (!e || "string" != typeof e) throw new Error("Invalid var name"); return this._vars.hasOwnProperty(e) || (this._vars[e] = new o.default(this, e)), this._vars[e] } }, { key: "has", value: function (e) { if (!e || "string" != typeof e) throw new Error("Invalid var name"); return this._vars.hasOwnProperty(e) } }]), t }() }).call(this, "undefined" != typeof global ? global : "undefined" != typeof self ? self : "undefined" != typeof window ? window : {}) }, { "./var": 12 }], 5: [function (f, e, c) { (function (e) { "use strict"; Object.defineProperty(c, "__esModule", { value: !0 }); var t, n = f("./group"), r = (t = n) && t.__esModule ? t : { default: t }, i = f("./selection"), o = f("./filter"), u = f("./input"); f("./input_selectize"), f("./input_checkboxgroup"), f("./input_slider"); var a = (0, r.default)("default"); function l(e) { return a.var(e) } e.Shiny && e.Shiny.addCustomMessageHandler("update-client-value", function (e) { "string" == typeof e.group ? (0, r.default)(e.group).var(e.name).set(e.value) : l(e.name).set(e.value) }); var s = { group: r.default, var: l, has: function (e) { return a.has(e) }, SelectionHandle: i.SelectionHandle, FilterHandle: o.FilterHandle, bind: u.bind }; c.default = s, e.crosstalk = s }).call(this, "undefined" != typeof global ? global : "undefined" != typeof self ? self : "undefined" != typeof window ? window : {}) }, { "./filter": 2, "./group": 4, "./input": 6, "./input_checkboxgroup": 7, "./input_selectize": 8, "./input_slider": 9, "./selection": 10 }], 6: [function (e, t, a) { (function (t) { "use strict"; Object.defineProperty(a, "__esModule", { value: !0 }), a.register = function (e) { r[e.className] = e, t.document && "complete" !== t.document.readyState ? o(function () { n() }) : t.document && setTimeout(n, 100) }, a.bind = n; var o = t.jQuery, r = {}; function n() { Object.keys(r).forEach(function (e) { var n = r[e]; o("." + n.className).not(".crosstalk-input-bound").each(function (e, t) { i(n, t) }) }) } function i(e, t) { var n = o(t).find("script[type='application/json'][data-for='" + t.id.replace(/([!"#$%&'()*+,./:;<=>?@[\\\]^`{|}~])/g, "\\$1") + "']"), r = JSON.parse(n[0].innerText), i = e.factory(t, r); o(t).data("crosstalk-instance", i), o(t).addClass("crosstalk-input-bound") } if (t.Shiny) { var e = new t.Shiny.InputBinding, u = t.jQuery; u.extend(e, { find: function (e) { return u(e).find(".crosstalk-input") }, initialize: function (e) { var t, n; u(e).hasClass("crosstalk-input-bound") || (n = o(t = e), Object.keys(r).forEach(function (e) { n.hasClass(e) && !n.hasClass("crosstalk-input-bound") && i(r[e], t) })) }, getId: function (e) { return e.id }, getValue: function (e) { }, setValue: function (e, t) { }, receiveMessage: function (e, t) { }, subscribe: function (e, t) { u(e).data("crosstalk-instance").resume() }, unsubscribe: function (e) { u(e).data("crosstalk-instance").suspend() } }), t.Shiny.inputBindings.register(e, "crosstalk.inputBinding") } }).call(this, "undefined" != typeof global ? global : "undefined" != typeof self ? self : "undefined" != typeof window ? window : {}) }, {}], 7: [function (r, e, t) { (function (e) { "use strict"; var t = function (e) { { if (e && e.__esModule) return e; var t = {}; if (null != e) for (var n in e) Object.prototype.hasOwnProperty.call(e, n) && (t[n] = e[n]); return t.default = e, t } }(r("./input")), n = r("./filter"); var a = e.jQuery; t.register({ className: "crosstalk-input-checkboxgroup", factory: function (e, r) { var i = new n.FilterHandle(r.group), o = void 0, u = a(e); return u.on("change", "input[type='checkbox']", function () { var e = u.find("input[type='checkbox']:checked"); if (0 === e.length) o = null, i.clear(); else { var t = {}; e.each(function () { r.map[this.value].forEach(function (e) { t[e] = !0 }) }); var n = Object.keys(t); n.sort(), o = n, i.set(n) } }), { suspend: function () { i.clear() }, resume: function () { o && i.set(o) } } } }) }).call(this, "undefined" != typeof global ? global : "undefined" != typeof self ? self : "undefined" != typeof window ? window : {}) }, { "./filter": 2, "./input": 6 }], 8: [function (r, e, t) { (function (e) { "use strict"; var t = n(r("./input")), l = n(r("./util")), s = r("./filter"); function n(e) { if (e && e.__esModule) return e; var t = {}; if (null != e) for (var n in e) Object.prototype.hasOwnProperty.call(e, n) && (t[n] = e[n]); return t.default = e, t } var f = e.jQuery; t.register({ className: "crosstalk-input-select", factory: function (e, n) { var t = l.dataframeToD3(n.items), r = { options: [{ value: "", label: "(All)" }].concat(t), valueField: "value", labelField: "label", searchField: "label" }, i = f(e).find("select")[0], o = f(i).selectize(r)[0].selectize, u = new s.FilterHandle(n.group), a = void 0; return o.on("change", function () { if (0 === o.items.length) a = null, u.clear(); else { var t = {}; o.items.forEach(function (e) { n.map[e].forEach(function (e) { t[e] = !0 }) }); var e = Object.keys(t); e.sort(), a = e, u.set(e) } }), { suspend: function () { u.clear() }, resume: function () { a && u.set(a) } } } }) }).call(this, "undefined" != typeof global ? global : "undefined" != typeof self ? self : "undefined" != typeof window ? window : {}) }, { "./filter": 2, "./input": 6, "./util": 11 }], 9: [function (n, e, t) { (function (e) { "use strict"; var d = function (e, t) { if (Array.isArray(e)) return e; if (Symbol.iterator in Object(e)) return function (e, t) { var n = [], r = !0, i = !1, o = void 0; try { for (var u, a = e[Symbol.iterator](); !(r = (u = a.next()).done) && (n.push(u.value), !t || n.length !== t); r = !0); } catch (e) { i = !0, o = e } finally { try { !r && a.return && a.return() } finally { if (i) throw o } } return n }(e, t); throw new TypeError("Invalid attempt to destructure non-iterable instance") }, t = function (e) { { if (e && e.__esModule) return e; var t = {}; if (null != e) for (var n in e) Object.prototype.hasOwnProperty.call(e, n) && (t[n] = e[n]); return t.default = e, t } }(n("./input")), a = n("./filter"); var v = e.jQuery, p = e.strftime; function y(e, t) { for (var n = e.toString(); n.length < t;)n = "0" + n; return n } t.register({ className: "crosstalk-input-slider", factory: function (e, l) { var s = new a.FilterHandle(l.group), t = {}, f = v(e).find("input"), n = f.data("data-type"), r = f.data("time-format"), i = f.data("round"), o = void 0; if ("date" === n) o = p.utc(), t.prettify = function (e) { return o(r, new Date(e)) }; else if ("datetime" === n) { var u = f.data("timezone"); o = u ? p.timezone(u) : p, t.prettify = function (e) { return o(r, new Date(e)) } } else "number" === n && void 0 !== i && (t.prettify = function (e) { var t = Math.pow(10, i); return Math.round(e * t) / t }); function c() { var e = f.data("ionRangeSlider").result, t = void 0, n = f.data("data-type"); return t = "date" === n ? function (e) { return (t = new Date(+e)) instanceof Date ? t.getUTCFullYear() + "-" + y(t.getUTCMonth() + 1, 2) + "-" + y(t.getUTCDate(), 2) : null; var t } : "datetime" === n ? function (e) { return +e / 1e3 } : function (e) { return +e }, "double" === f.data("ionRangeSlider").options.type ? [t(e.from), t(e.to)] : t(e.from) } f.ionRangeSlider(t); var h = null; return f.on("change.crosstalkSliderInput", function (e) { if (!f.data("updating") && !f.data("animating")) { for (var t = c(), n = d(t, 2), r = n[0], i = n[1], o = [], u = 0; u < l.values.length; u++) { var a = l.values[u]; r <= a && a <= i && o.push(l.keys[u]) } o.sort(), s.set(o), h = o } }), { suspend: function () { s.clear() }, resume: function () { h && s.set(h) } } } }) }).call(this, "undefined" != typeof global ? global : "undefined" != typeof self ? self : "undefined" != typeof window ? window : {}) }, { "./filter": 2, "./input": 6 }], 10: [function (e, t, n) { "use strict"; Object.defineProperty(n, "__esModule", { value: !0 }), n.SelectionHandle = void 0; var r = function () { function r(e, t) { for (var n = 0; n < t.length; n++) { var r = t[n]; r.enumerable = r.enumerable || !1, r.configurable = !0, "value" in r && (r.writable = !0), Object.defineProperty(e, r.key, r) } } return function (e, t, n) { return t && r(e.prototype, t), n && r(e, n), e } }(), i = a(e("./events")), o = a(e("./group")), u = function (e) { { if (e && e.__esModule) return e; var t = {}; if (null != e) for (var n in e) Object.prototype.hasOwnProperty.call(e, n) && (t[n] = e[n]); return t.default = e, t } }(e("./util")); function a(e) { return e && e.__esModule ? e : { default: e } } n.SelectionHandle = function () { function n() { var e = 0 < arguments.length && void 0 !== arguments[0] ? arguments[0] : null, t = 1 < arguments.length && void 0 !== arguments[1] ? arguments[1] : null; !function (e, t) { if (!(e instanceof t)) throw new TypeError("Cannot call a class as a function") }(this, n), this._eventRelay = new i.default, this._emitter = new u.SubscriptionTracker(this._eventRelay), this._group = null, this._var = null, this._varOnChangeSub = null, this._extraInfo = u.extend({ sender: this }, t), this.setGroup(e) } return r(n, [{ key: "setGroup", value: function (e) { var t = this; if (this._group !== e && (this._group || e) && (this._var && (this._var.off("change", this._varOnChangeSub), this._var = null, this._varOnChangeSub = null), this._group = e)) { this._var = (0, o.default)(e).var("selection"); var n = this._var.on("change", function (e) { t._eventRelay.trigger("change", e, t) }); this._varOnChangeSub = n } } }, { key: "_mergeExtraInfo", value: function (e) { return u.extend({}, this._extraInfo ? this._extraInfo : null, e || null) } }, { key: "set", value: function (e, t) { this._var && this._var.set(e, this._mergeExtraInfo(t)) } }, { key: "clear", value: function (e) { this._var && this.set(void 0, this._mergeExtraInfo(e)) } }, { key: "on", value: function (e, t) { return this._emitter.on(e, t) } }, { key: "off", value: function (e, t) { return this._emitter.off(e, t) } }, { key: "close", value: function () { this._emitter.removeAllListeners(), this.setGroup(null) } }, { key: "value", get: function () { return this._var ? this._var.get() : null } }]), n }() }, { "./events": 1, "./group": 4, "./util": 11 }], 11: [function (e, t, n) { "use strict"; Object.defineProperty(n, "__esModule", { value: !0 }); var r = function () { function r(e, t) { for (var n = 0; n < t.length; n++) { var r = t[n]; r.enumerable = r.enumerable || !1, r.configurable = !0, "value" in r && (r.writable = !0), Object.defineProperty(e, r.key, r) } } return function (e, t, n) { return t && r(e.prototype, t), n && r(e, n), e } }(), l = "function" == typeof Symbol && "symbol" == typeof Symbol.iterator ? function (e) { return typeof e } : function (e) { return e && "function" == typeof Symbol && e.constructor === Symbol && e !== Symbol.prototype ? "symbol" : typeof e }; function u(e) { for (var t = 1; t < e.length; t++)if (e[t] <= e[t - 1]) throw new Error("List is not sorted or contains duplicate") } n.extend = function (e) { for (var t = arguments.length, n = Array(1 < t ? t - 1 : 0), r = 1; r < t; r++)n[r - 1] = arguments[r]; for (var i = 0; i < n.length; i++) { var o = n[i]; if (null != o) for (var u in o) o.hasOwnProperty(u) && (e[u] = o[u]) } return e }, n.checkSorted = u, n.diffSortedLists = function (e, t) { var n = 0, r = 0; e || (e = []); t || (t = []); var i = [], o = []; u(e), u(t); for (; n < e.length && r < t.length;)e[n] === t[r] ? (n++, r++) : e[n] < t[r] ? i.push(e[n++]) : o.push(t[r++]); n < e.length && (i = i.concat(e.slice(n))); r < t.length && (o = o.concat(t.slice(r))); return { removed: i, added: o } }, n.dataframeToD3 = function (e) { var t = [], n = void 0; for (var r in e) { if (e.hasOwnProperty(r) && t.push(r), "object" !== l(e[r]) || void 0 === e[r].length) throw new Error("All fields must be arrays"); if (void 0 !== n && n !== e[r].length) throw new Error("All fields must be arrays of the same length"); n = e[r].length } for (var i = [], o = void 0, u = 0; u < n; u++) { o = {}; for (var a = 0; a < t.length; a++)o[t[a]] = e[t[a]][u]; i.push(o) } return i }; n.SubscriptionTracker = function () { function t(e) { !function (e, t) { if (!(e instanceof t)) throw new TypeError("Cannot call a class as a function") }(this, t), this._emitter = e, this._subs = {} } return r(t, [{ key: "on", value: function (e, t) { var n = this._emitter.on(e, t); return this._subs[n] = e, n } }, { key: "off", value: function (e, t) { var n = this._emitter.off(e, t); return n && delete this._subs[n], n } }, { key: "removeAllListeners", value: function () { var t = this, n = this._subs; this._subs = {}, Object.keys(n).forEach(function (e) { t._emitter.off(n[e], e) }) } }]), t }() }, {}], 12: [function (a, e, l) { (function (o) { "use strict"; Object.defineProperty(l, "__esModule", { value: !0 }); var e, u = "function" == typeof Symbol && "symbol" == typeof Symbol.iterator ? function (e) { return typeof e } : function (e) { return e && "function" == typeof Symbol && e.constructor === Symbol && e !== Symbol.prototype ? "symbol" : typeof e }, t = function () { function r(e, t) { for (var n = 0; n < t.length; n++) { var r = t[n]; r.enumerable = r.enumerable || !1, r.configurable = !0, "value" in r && (r.writable = !0), Object.defineProperty(e, r.key, r) } } return function (e, t, n) { return t && r(e.prototype, t), n && r(e, n), e } }(), n = a("./events"), i = (e = n) && e.__esModule ? e : { default: e }; var r = function () { function r(e, t, n) { !function (e, t) { if (!(e instanceof t)) throw new TypeError("Cannot call a class as a function") }(this, r), this._group = e, this._name = t, this._value = n, this._events = new i.default } return t(r, [{ key: "get", value: function () { return this._value } }, { key: "set", value: function (e, t) { if (this._value !== e) { var n = this._value; this._value = e; var r = {}; if (t && "object" === (void 0 === t ? "undefined" : u(t))) for (var i in t) t.hasOwnProperty(i) && (r[i] = t[i]); r.oldValue = n, r.value = e, this._events.trigger("change", r, this), o.Shiny && o.Shiny.onInputChange && o.Shiny.onInputChange(".clientValue-" + (null !== this._group.name ? this._group.name + "-" : "") + this._name, void 0 === e ? null : e) } } }, { key: "on", value: function (e, t) { return this._events.on(e, t) } }, { key: "off", value: function (e, t) { return this._events.off(e, t) } }]), r }(); l.default = r }).call(this, "undefined" != typeof global ? global : "undefined" != typeof self ? self : "undefined" != typeof window ? window : {}) }, { "./events": 1 }] }, {}, [5]);
//# sourceMappingURL=crosstalk.min.js.map</script>


<!-- Global site tag (gtag.js) - Google Analytics -->
<script>
  // Copyright 2012 Google Inc. All rights reserved.

  (function () {

    var data = {
      "resource": {
        "version": "1",

        "macros": [{ "function": "__e" }, { "function": "__c", "vtp_value": "google.com.au" }, { "function": "__c", "vtp_value": 0 }, { "vtp_signal": 0, "function": "__c", "vtp_value": 0 }],
        "tags": [{ "function": "__gct", "vtp_trackingId": "G-T8GWK081T9", "vtp_sessionDuration": 0, "tag_id": 1 }, { "function": "__ogt_event_create", "vtp_eventName": "follow_to_galaxy", "vtp_isCopy": true, "vtp_instanceDestinationId": "G-T8GWK081T9", "vtp_precompiledRule": ["map", "new_event_name", "follow_to_galaxy", "merge_source_event_params", true, "event_name_predicate", ["map", "values", ["list", ["map", "type", "event_name"], ["map", "type", "const", "const_value", "follow_to_galaxy"]], "type", "eq"], "conditions", ["list", ["map", "predicates", ["list"]]]], "tag_id": 3 }, { "function": "__ccd_conversion_marking", "vtp_conversionRules": ["list", ["map", "matchingRules", "{\"type\":5,\"args\":[{\"stringValue\":\"purchase\"},{\"contextValue\":{\"namespaceType\":1,\"keyParts\":[\"eventName\"]}}]}"]], "vtp_instanceDestinationId": "G-T8GWK081T9", "tag_id": 5 }, { "function": "__ccd_em_download", "vtp_includeParams": true, "vtp_instanceDestinationId": "G-T8GWK081T9", "tag_id": 6 }, { "function": "__ccd_em_outbound_click", "priority": 0, "vtp_includeParams": true, "vtp_instanceDestinationId": "G-T8GWK081T9", "tag_id": 7 }, { "function": "__ccd_em_page_view", "vtp_historyEvents": true, "vtp_includeParams": true, "vtp_instanceDestinationId": "G-T8GWK081T9", "tag_id": 8 }, { "function": "__ccd_em_scroll", "vtp_includeParams": true, "vtp_instanceDestinationId": "G-T8GWK081T9", "tag_id": 9 }, { "function": "__ccd_em_site_search", "vtp_searchQueryParams": "q,s,search,query,keyword", "vtp_includeParams": true, "vtp_instanceDestinationId": "G-T8GWK081T9", "tag_id": 10 }, { "function": "__ccd_em_video", "vtp_includeParams": true, "vtp_instanceDestinationId": "G-T8GWK081T9", "tag_id": 11 }, { "function": "__set_product_settings", "vtp_instanceDestinationId": "G-T8GWK081T9", "vtp_foreignTldMacroResult": ["macro", 1], "vtp_isChinaVipRegionMacroResult": ["macro", 2], "tag_id": 12 }, { "function": "__ogt_google_signals", "vtp_googleSignals": "DISABLED", "vtp_instanceDestinationId": "G-T8GWK081T9", "vtp_serverMacroResult": ["macro", 3], "tag_id": 13 }],
        "predicates": [{ "function": "_eq", "arg0": ["macro", 0], "arg1": "gtm.js" }, { "function": "_eq", "arg0": ["macro", 0], "arg1": "gtm.init" }],
        "rules": [[["if", 0], ["add", 0]], [["if", 1], ["add", 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]]]
      },
      "runtime": [[50, "__ccd_conversion_marking", [46, "a"], [22, [30, [28, [17, [15, "a"], "conversionRules"]], [20, [17, [17, [15, "a"], "conversionRules"], "length"], 0]], [46, [2, [15, "a"], "gtmOnSuccess", [7]], [36]]], [52, "b", ["require", "internal.copyPreHit"]], [52, "c", ["require", "internal.evaluateBooleanExpression"]], [52, "d", ["require", "internal.registerCcdCallback"]], [52, "e", "is_conversion"], [52, "f", "is_first_visit"], [52, "g", "is_first_visit_conversion"], [52, "h", "is_session_start"], [52, "i", "is_session_start_conversion"], [52, "j", "first_visit"], [52, "k", "session_start"], [41, "l"], [41, "m"], ["d", [17, [15, "a"], "instanceDestinationId"], [51, "", [7, "n"], [52, "o", [8, "preHit", [15, "n"]]], [65, "p", [17, [15, "a"], "conversionRules"], [46, [22, ["c", [17, [15, "p"], "matchingRules"], [15, "o"]], [46, [2, [15, "n"], "setMetadata", [7, [15, "e"], true]], [4]]]]], [22, [2, [15, "n"], "getMetadata", [7, [15, "f"]]], [46, [22, [28, [15, "l"]], [46, [53, [52, "p", ["b", [15, "n"], [8, "omitHitData", true, "omitMetadata", true]]], [2, [15, "p"], "setEventName", [7, [15, "j"]]], [3, "l", [8, "preHit", [15, "p"]]]]]], [65, "p", [17, [15, "a"], "conversionRules"], [46, [22, ["c", [17, [15, "p"], "matchingRules"], [15, "l"]], [46, [2, [15, "n"], "setMetadata", [7, [15, "g"], true]], [4]]]]]]], [22, [2, [15, "n"], "getMetadata", [7, [15, "h"]]], [46, [22, [28, [15, "m"]], [46, [53, [52, "p", ["b", [15, "n"], [8, "omitHitData", true, "omitMetadata", true]]], [2, [15, "p"], "setEventName", [7, [15, "k"]]], [3, "m", [8, "preHit", [15, "p"]]]]]], [65, "p", [17, [15, "a"], "conversionRules"], [46, [22, ["c", [17, [15, "p"], "matchingRules"], [15, "m"]], [46, [2, [15, "n"], "setMetadata", [7, [15, "i"], true]], [4]]]]]]]]], [2, [15, "a"], "gtmOnSuccess", [7]], [36]]
        , [50, "__ccd_em_download", [46, "a"], [50, "r", [46, "x"], [36, [1, [15, "x"], [21, [2, [2, [15, "x"], "toLowerCase", [7]], "match", [7, [15, "q"]]], [45]]]]], [50, "s", [46, "x"], [52, "y", [2, [17, [15, "x"], "pathname"], "split", [7, "."]]], [52, "z", [39, [18, [17, [15, "y"], "length"], 1], [16, [15, "y"], [37, [17, [15, "y"], "length"], 1]], ""]], [36, [16, [2, [15, "z"], "split", [7, "/"]], 0]]], [50, "t", [46, "x"], [36, [39, [12, [2, [17, [15, "x"], "pathname"], "substring", [7, 0, 1]], "/"], [17, [15, "x"], "pathname"], [0, "/", [17, [15, "x"], "pathname"]]]]], [50, "u", [46, "x"], [41, "y"], [3, "y", ""], [22, [1, [15, "x"], [17, [15, "x"], "href"]], [46, [53, [41, "z"], [3, "z", [2, [17, [15, "x"], "href"], "indexOf", [7, "#"]]], [3, "y", [39, [23, [15, "z"], 0], [17, [15, "x"], "href"], [2, [17, [15, "x"], "href"], "substring", [7, 0, [15, "z"]]]]]]]], [36, [15, "y"]]], [50, "w", [46, "x"], [52, "y", [8]], [43, [15, "y"], [15, "j"], true], [43, [15, "y"], [15, "f"], true], [43, [15, "x"], "eventMetadata", [15, "y"]]], [52, "b", [13, [41, "$0"], [3, "$0", ["require", "internal.getFlags"]], ["$0"]]], [52, "c", ["require", "internal.getProductSettingsParameter"]], [52, "d", ["require", "templateStorage"]], [52, "e", [15, "__module_ccdEmDownloadActivity"]], [52, "f", "speculative"], [52, "g", "ae_block_downloads"], [52, "h", "file_download"], [52, "i", "isRegistered"], [52, "j", "em_event"], [52, "k", [17, [15, "a"], "instanceDestinationId"]], [22, ["c", [15, "k"], [15, "g"]], [46, [2, [15, "a"], "gtmOnSuccess", [7]], [36]]], [2, [15, "e"], "registerDownloadActivityCallback", [7, [15, "k"], [17, [15, "a"], "includeParams"]]], [22, [2, [15, "d"], "getItem", [7, [15, "i"]]], [46, [2, [15, "a"], "gtmOnSuccess", [7]], [36]]], [52, "l", ["require", "internal.addDataLayerEventListener"]], [52, "m", ["require", "internal.enableAutoEventOnLinkClick"]], [52, "n", ["require", "internal.getDestinationIds"]], [52, "o", ["require", "parseUrl"]], [52, "p", ["require", "internal.sendGtagEvent"]], [52, "q", [0, "^(pdf|xlsx?|docx?|txt|rtf|csv|exe|key|pp(s|t|tx)|7z|pkg|rar|gz|zip|avi|", "mov|mp4|mpe?g|wmv|midi?|mp3|wav|wma)$"]], [52, "v", ["m", [8, "checkValidation", true]]], [22, [28, [15, "v"]], [46, [2, [15, "a"], "gtmOnFailure", [7]], [36]]], [2, [15, "d"], "setItem", [7, [15, "i"], true]], ["l", "gtm.linkClick", [51, "", [7, "x", "y"], ["y"], [52, "z", [8, "eventId", [16, [15, "x"], "gtm.uniqueEventId"]]], [22, [16, [15, "b"], "enableDeferAllEnhancedMeasurement"], [46, [43, [15, "z"], "deferrable", true]]], [52, "ba", [16, [15, "x"], "gtm.elementUrl"]], [52, "bb", ["o", [15, "ba"]]], [22, [28, [15, "bb"]], [46, [36]]], [52, "bc", ["s", [15, "bb"]]], [22, [28, ["r", [15, "bc"]]], [46, [36]]], [52, "bd", [8, "link_id", [16, [15, "x"], "gtm.elementId"], "link_url", ["u", [15, "bb"]], "link_text", [16, [15, "x"], "gtm.elementText"], "file_name", ["t", [15, "bb"]], "file_extension", [15, "bc"]]], ["w", [15, "z"]], ["p", ["n"], [15, "h"], [15, "bd"], [15, "z"]]], [15, "v"]], [2, [15, "a"], "gtmOnSuccess", [7]]]
        , [50, "__ccd_em_outbound_click", [46, "a"], [50, "s", [46, "y"], [22, [28, [15, "y"]], [46, [36, [44]]]], [41, "z"], [3, "z", ""], [22, [1, [15, "y"], [17, [15, "y"], "href"]], [46, [53, [41, "ba"], [3, "ba", [2, [17, [15, "y"], "href"], "indexOf", [7, "#"]]], [3, "z", [39, [23, [15, "ba"], 0], [17, [15, "y"], "href"], [2, [17, [15, "y"], "href"], "substring", [7, 0, [15, "ba"]]]]]]]], [36, [15, "z"]]], [50, "t", [46, "y"], [22, [28, [15, "y"]], [46, [36, [44]]]], [41, "z"], [3, "z", [17, [15, "y"], "hostname"]], [52, "ba", [2, [15, "z"], "match", [7, "^www\\d*\\."]]], [22, [1, [15, "ba"], [16, [15, "ba"], 0]], [46, [3, "z", [2, [15, "z"], "substring", [7, [17, [16, [15, "ba"], 0], "length"]]]]]], [36, [15, "z"]]], [50, "u", [46, "y"], [22, [28, [15, "y"]], [46, [36, false]]], [52, "z", [2, [17, [15, "y"], "hostname"], "toLowerCase", [7]]], [41, "ba"], [3, "ba", [2, ["t", ["q", ["p"]]], "toLowerCase", [7]]], [41, "bb"], [3, "bb", [37, [17, [15, "z"], "length"], [17, [15, "ba"], "length"]]], [22, [1, [18, [15, "bb"], 0], [29, [2, [15, "ba"], "charAt", [7, 0]], "."]], [46, [32, [15, "bb"], [3, "bb", [37, [15, "bb"], 1]]], [3, "ba", [0, ".", [15, "ba"]]]]], [22, [1, [19, [15, "bb"], 0], [12, [2, [15, "z"], "indexOf", [7, [15, "ba"], [15, "bb"]]], [15, "bb"]]], [46, [36, false]]], [36, true]], [50, "x", [46, "y"], [52, "z", [8]], [43, [15, "z"], [15, "j"], true], [43, [15, "z"], [15, "f"], true], [43, [15, "y"], "eventMetadata", [15, "z"]]], [52, "b", [13, [41, "$0"], [3, "$0", ["require", "internal.getFlags"]], ["$0"]]], [52, "c", ["require", "internal.getProductSettingsParameter"]], [52, "d", ["require", "templateStorage"]], [52, "e", [15, "__module_ccdEmOutboundClickActivity"]], [52, "f", "speculative"], [52, "g", "ae_block_outbound_click"], [52, "h", "click"], [52, "i", "isRegistered"], [52, "j", "em_event"], [52, "k", [17, [15, "a"], "instanceDestinationId"]], [22, ["c", [15, "k"], [15, "g"]], [46, [2, [15, "a"], "gtmOnSuccess", [7]], [36]]], [2, [15, "e"], "registerOutbackClickActivityCallback", [7, [15, "k"], [17, [15, "a"], "includeParams"]]], [22, [2, [15, "d"], "getItem", [7, [15, "i"]]], [46, [2, [15, "a"], "gtmOnSuccess", [7]], [36]]], [52, "l", ["require", "internal.addDataLayerEventListener"]], [52, "m", ["require", "internal.enableAutoEventOnLinkClick"]], [52, "n", ["require", "internal.getDestinationIds"]], [52, "o", ["require", "internal.getRemoteConfigParameter"]], [52, "p", ["require", "getUrl"]], [52, "q", ["require", "parseUrl"]], [52, "r", ["require", "internal.sendGtagEvent"]], [52, "v", ["o", [15, "k"], "cross_domain_conditions"]], [52, "w", ["m", [8, "affiliateDomains", [15, "v"], "checkValidation", true, "waitForTags", false]]], [22, [28, [15, "w"]], [46, [2, [15, "a"], "gtmOnFailure", [7]], [36]]], [2, [15, "d"], "setItem", [7, [15, "i"], true]], ["l", "gtm.linkClick", [51, "", [7, "y", "z"], [52, "ba", ["q", [16, [15, "y"], "gtm.elementUrl"]]], [22, [28, ["u", [15, "ba"]]], [46, ["z"], [36]]], [52, "bb", [8, "link_id", [16, [15, "y"], "gtm.elementId"], "link_classes", [16, [15, "y"], "gtm.elementClasses"], "link_url", ["s", [15, "ba"]], "link_domain", ["t", [15, "ba"]], "outbound", true]], [43, [15, "bb"], "event_callback", [15, "z"]], [52, "bc", [8, "eventId", [16, [15, "y"], "gtm.uniqueEventId"]]], [22, [16, [15, "b"], "enableDeferAllEnhancedMeasurement"], [46, [43, [15, "bc"], "deferrable", true]]], ["x", [15, "bc"]], ["r", ["n"], [15, "h"], [15, "bb"], [15, "bc"]]], [15, "w"]], [2, [15, "a"], "gtmOnSuccess", [7]]]
        , [50, "__ccd_em_page_view", [46, "a"], [50, "r", [46, "s"], [52, "t", [8]], [43, [15, "t"], [15, "k"], true], [43, [15, "t"], [15, "g"], true], [43, [15, "s"], "eventMetadata", [15, "t"]]], [22, [28, [17, [15, "a"], "historyEvents"]], [46, [2, [15, "a"], "gtmOnSuccess", [7]], [36]]], [52, "b", [13, [41, "$0"], [3, "$0", ["require", "internal.getFlags"]], ["$0"]]], [52, "c", ["require", "internal.getProductSettingsParameter"]], [52, "d", ["require", "internal.setRemoteConfigParameter"]], [52, "e", ["require", "templateStorage"]], [52, "f", [15, "__module_ccdEmPageViewActivity"]], [52, "g", "speculative"], [52, "h", "ae_block_history"], [52, "i", "page_view"], [52, "j", "isRegistered"], [52, "k", "em_event"], [52, "l", [17, [15, "a"], "instanceDestinationId"]], [22, ["c", [15, "l"], [15, "h"]], [46, [2, [15, "a"], "gtmOnSuccess", [7]], [36]]], [2, [15, "f"], "registerPageViewActivityCallback", [7, [15, "l"], [17, [15, "a"], "includeParams"]]], [22, [2, [15, "e"], "getItem", [7, [15, "j"]]], [46, [2, [15, "a"], "gtmOnSuccess", [7]], [36]]], [52, "m", ["require", "internal.addDataLayerEventListener"]], [52, "n", ["require", "internal.enableAutoEventOnHistoryChange"]], [52, "o", ["require", "internal.getDestinationIds"]], [52, "p", ["require", "internal.sendGtagEvent"]], [52, "q", ["n", [8, "interval", 1000]]], [22, [28, [15, "q"]], [46, [2, [15, "a"], "gtmOnFailure", [7]], [36]]], [2, [15, "e"], "setItem", [7, [15, "j"], true]], ["m", "gtm.historyChange-v2", [51, "", [7, "s", "t"], ["t"], [52, "u", [16, [15, "s"], "gtm.oldUrl"]], [22, [20, [16, [15, "s"], "gtm.newUrl"], [15, "u"]], [46, [36]]], [52, "v", [16, [15, "s"], "gtm.historyChangeSource"]], [22, [1, [1, [21, [15, "v"], "pushState"], [21, [15, "v"], "popstate"]], [21, [15, "v"], "replaceState"]], [46, [36]]], [52, "w", [8]], [22, [17, [15, "a"], "includeParams"], [46, [43, [15, "w"], "page_location", [16, [15, "s"], "gtm.newUrl"]], [43, [15, "w"], "page_referrer", [15, "u"]]]], [52, "x", [8, "eventId", [16, [15, "s"], "gtm.uniqueEventId"]]], [22, [16, [15, "b"], "enableDeferAllEnhancedMeasurement"], [46, [43, [15, "x"], "deferrable", true]]], ["r", [15, "x"]], ["p", ["o"], [15, "i"], [15, "w"], [15, "x"]]], [15, "q"]], [2, [15, "a"], "gtmOnSuccess", [7]]]
        , [50, "__ccd_em_scroll", [46, "a"], [50, "q", [46, "r"], [52, "s", [8]], [43, [15, "s"], [15, "j"], true], [43, [15, "s"], [15, "f"], true], [43, [15, "r"], "eventMetadata", [15, "s"]]], [52, "b", [13, [41, "$0"], [3, "$0", ["require", "internal.getFlags"]], ["$0"]]], [52, "c", ["require", "internal.getProductSettingsParameter"]], [52, "d", ["require", "templateStorage"]], [52, "e", [15, "__module_ccdEmScrollActivity"]], [52, "f", "speculative"], [52, "g", "ae_block_scroll"], [52, "h", "scroll"], [52, "i", "isRegistered"], [52, "j", "em_event"], [52, "k", [17, [15, "a"], "instanceDestinationId"]], [22, ["c", [15, "k"], [15, "g"]], [46, [2, [15, "a"], "gtmOnSuccess", [7]], [36]]], [2, [15, "e"], "registerScrollActivityCallback", [7, [15, "k"], [17, [15, "a"], "includeParams"]]], [22, [2, [15, "d"], "getItem", [7, [15, "i"]]], [46, [2, [15, "a"], "gtmOnSuccess", [7]], [36]]], [52, "l", ["require", "internal.addDataLayerEventListener"]], [52, "m", ["require", "internal.enableAutoEventOnScroll"]], [52, "n", ["require", "internal.getDestinationIds"]], [52, "o", ["require", "internal.sendGtagEvent"]], [52, "p", ["m", [8, "verticalThresholdUnits", "PERCENT", "verticalThresholds", 90]]], [22, [28, [15, "p"]], [46, [2, [15, "a"], "gtmOnFailure", [7]], [36]]], [2, [15, "d"], "setItem", [7, [15, "i"], true]], ["l", "gtm.scrollDepth", [51, "", [7, "r", "s"], ["s"], [52, "t", [8, "eventId", [16, [15, "r"], "gtm.uniqueEventId"]]], [22, [16, [15, "b"], "enableDeferAllEnhancedMeasurement"], [46, [43, [15, "t"], "deferrable", true]]], [52, "u", [8, "percent_scrolled", [16, [15, "r"], "gtm.scrollThreshold"]]], ["q", [15, "t"]], ["o", ["n"], [15, "h"], [15, "u"], [15, "t"]]], [15, "p"]], [2, [15, "a"], "gtmOnSuccess", [7]]]
        , [50, "__ccd_em_site_search", [46, "a"], [50, "e", [46, "j"], [52, "k", [2, [30, [15, "j"], ""], "split", [7, ","]]], [53, [41, "l"], [3, "l", 0], [63, [7, "l"], [23, [15, "l"], [17, [15, "k"], "length"]], [33, [15, "l"], [3, "l", [0, [15, "l"], 1]]], [46, [53, [52, "m", ["b", [2, [16, [15, "k"], [15, "l"]], "trim", [7]]]], [22, [21, [15, "m"], [44]], [46, [36, [15, "m"]]]]]]]]], [50, "f", [46, "j", "k"], [52, "l", [8, "search_term", [15, "j"]]], [52, "m", [2, [30, [15, "k"], ""], "split", [7, ","]]], [53, [41, "n"], [3, "n", 0], [63, [7, "n"], [23, [15, "n"], [17, [15, "m"], "length"]], [33, [15, "n"], [3, "n", [0, [15, "n"], 1]]], [46, [53, [52, "o", [2, [16, [15, "m"], [15, "n"]], "trim", [7]]], [52, "p", ["b", [15, "o"]]], [22, [21, [15, "p"], [44]], [46, [43, [15, "l"], [0, "q_", [15, "o"]], [15, "p"]]]]]]]], [36, [15, "l"]]], [52, "b", ["require", "getQueryParameters"]], [52, "c", ["require", "internal.sendGtagEvent"]], [52, "d", ["require", "getContainerVersion"]], [52, "g", ["e", [17, [15, "a"], "searchQueryParams"]]], [52, "h", [30, [17, [15, "a"], "instanceDestinationId"], [17, ["d"], "containerId"]]], [52, "i", [8, "deferrable", true, "eventId", [17, [15, "a"], "gtmEventId"]]], [22, [15, "g"], [46, [53, [52, "j", [39, [28, [28, [17, [15, "a"], "includeParams"]]], ["f", [15, "g"], [17, [15, "a"], "additionalQueryParams"]], [8]]], ["c", [15, "h"], "view_search_results", [15, "j"], [15, "i"]]]]], [2, [15, "a"], "gtmOnSuccess", [7]]]
        , [50, "__ccd_em_video", [46, "a"], [50, "s", [46, "t"], [52, "u", [8]], [43, [15, "u"], [15, "l"], true], [43, [15, "u"], [15, "f"], true], [43, [15, "t"], "eventMetadata", [15, "u"]]], [52, "b", [13, [41, "$0"], [3, "$0", ["require", "internal.getFlags"]], ["$0"]]], [52, "c", ["require", "internal.getProductSettingsParameter"]], [52, "d", ["require", "templateStorage"]], [52, "e", [15, "__module_ccdEmVideoActivity"]], [52, "f", "speculative"], [52, "g", "ae_block_video"], [52, "h", "video_start"], [52, "i", "video_progress"], [52, "j", "video_complete"], [52, "k", "isRegistered"], [52, "l", "em_event"], [52, "m", [17, [15, "a"], "instanceDestinationId"]], [22, ["c", [15, "m"], [15, "g"]], [46, [2, [15, "a"], "gtmOnSuccess", [7]], [36]]], [2, [15, "e"], "registerVideoActivityCallback", [7, [15, "m"], [17, [15, "a"], "includeParams"]]], [22, [2, [15, "d"], "getItem", [7, [15, "k"]]], [46, [2, [15, "a"], "gtmOnSuccess", [7]], [36]]], [52, "n", ["require", "internal.addDataLayerEventListener"]], [52, "o", ["require", "internal.enableAutoEventOnYouTubeActivity"]], [52, "p", ["require", "internal.getDestinationIds"]], [52, "q", ["require", "internal.sendGtagEvent"]], [52, "r", ["o", [8, "captureComplete", true, "captureStart", true, "progressThresholdsPercent", [7, 10, 25, 50, 75]]]], [22, [28, [15, "r"]], [46, [2, [15, "a"], "gtmOnFailure", [7]], [36]]], [2, [15, "d"], "setItem", [7, [15, "k"], true]], ["n", "gtm.video", [51, "", [7, "t", "u"], ["u"], [52, "v", [16, [15, "t"], "gtm.videoStatus"]], [41, "w"], [22, [20, [15, "v"], "start"], [46, [3, "w", [15, "h"]]], [46, [22, [20, [15, "v"], "progress"], [46, [3, "w", [15, "i"]]], [46, [22, [20, [15, "v"], "complete"], [46, [3, "w", [15, "j"]]], [46, [36]]]]]]], [52, "x", [8, "video_current_time", [16, [15, "t"], "gtm.videoCurrentTime"], "video_duration", [16, [15, "t"], "gtm.videoDuration"], "video_percent", [16, [15, "t"], "gtm.videoPercent"], "video_provider", [16, [15, "t"], "gtm.videoProvider"], "video_title", [16, [15, "t"], "gtm.videoTitle"], "video_url", [16, [15, "t"], "gtm.videoUrl"], "visible", [16, [15, "t"], "gtm.videoVisible"]]], [52, "y", [8, "eventId", [16, [15, "t"], "gtm.uniqueEventId"]]], [22, [16, [15, "b"], "enableDeferAllEnhancedMeasurement"], [46, [43, [15, "y"], "deferrable", true]]], ["s", [15, "y"]], ["q", ["p"], [15, "w"], [15, "x"], [15, "y"]]], [15, "r"]], [2, [15, "a"], "gtmOnSuccess", [7]]]
        , [50, "__ogt_event_create", [46, "a"], [50, "r", [46, "s", "t"], [22, [28, [2, [15, "c"], "preHitMatchesRule", [7, [15, "s"], [16, [15, "t"], [15, "n"]], [30, [16, [15, "t"], [15, "o"]], [7]]]]], [46, [36, false]]], [52, "u", [16, [15, "t"], [15, "p"]]], [22, [2, [15, "c"], "isEventNameFalsyOrReserved", [7, [15, "u"]]], [46, [36]]], [52, "v", [28, [16, [15, "t"], [15, "q"]]]], [52, "w", [30, [2, [15, "s"], "getMetadata", [7, [15, "j"]]], [7]]], [22, [20, [2, [15, "w"], "indexOf", [7, [15, "k"]]], [27, 1]], [46, [2, [15, "w"], "push", [7, [15, "k"]]]]], [2, [15, "s"], "setMetadata", [7, [15, "j"], [15, "w"]]], [52, "x", ["b", [15, "s"], [8, "omitHitData", [15, "v"], "omitEventContext", [15, "v"], "omitMetadata", true]]], [2, [15, "c"], "applyParamOperations", [7, [15, "x"], [15, "t"]]], [2, [15, "x"], "setEventName", [7, [15, "u"]]], [2, [15, "x"], "setMetadata", [7, [15, "m"], true]], [2, [15, "x"], "setMetadata", [7, [15, "j"], [7, [15, "l"]]]], ["d", [15, "x"]]], [52, "b", ["require", "internal.copyPreHit"]], [52, "c", [15, "__module_eventEditingAndSynthesis"]], [52, "d", ["require", "internal.processAsNewEvent"]], [52, "e", ["require", "internal.registerCcdCallback"]], [52, "f", ["require", "templateStorage"]], [52, "g", [17, [15, "a"], "instanceDestinationId"]], [41, "h"], [3, "h", [2, [15, "f"], "getItem", [7, [15, "g"]]]], [41, "i"], [3, "i", [28, [28, [15, "h"]]]], [22, [15, "i"], [46, [2, [15, "h"], "push", [7, [17, [15, "a"], "precompiledRule"]]], [2, [15, "a"], "gtmOnSuccess", [7]], [36]]], [2, [15, "f"], "setItem", [7, [15, "g"], [7, [17, [15, "a"], "precompiledRule"]]]], [52, "j", "event_usage"], [52, "k", 1], [52, "l", 11], [52, "m", "is_syn"], [52, "n", "event_name_predicate"], [52, "o", "conditions"], [52, "p", "new_event_name"], [52, "q", "merge_source_event_params"], ["e", [15, "g"], [51, "", [7, "s"], [22, [2, [15, "s"], "getMetadata", [7, [15, "m"]]], [46, [36]]], [52, "t", [2, [15, "f"], "getItem", [7, [15, "g"]]]], [66, "u", [15, "t"], [46, ["r", [15, "s"], [15, "u"]]]]]], [2, [15, "a"], "gtmOnSuccess", [7]]]
        , [50, "__ogt_google_signals", [46, "a"], [52, "b", ["require", "internal.setProductSettingsParameter"]], [52, "c", ["require", "getContainerVersion"]], [52, "d", [13, [41, "$0"], [3, "$0", ["require", "internal.getFlags"]], ["$0"]]], [52, "e", [30, [17, [15, "a"], "instanceDestinationId"], [17, ["c"], "containerId"]]], ["b", [15, "e"], "google_signals", [20, [17, [15, "a"], "serverMacroResult"], 1]], [22, [17, [15, "d"], "enableGa4OnoRemarketing"], [46, ["b", [15, "e"], "google_ono", [20, [17, [15, "a"], "serverMacroResult"], 2]]]], [2, [15, "a"], "gtmOnSuccess", [7]]]
        , [50, "__set_product_settings", [46, "a"], [2, [15, "a"], "gtmOnSuccess", [7]]]
        , [52, "__module_eventEditingAndSynthesis", [13, [41, "$0"], [3, "$0", [51, "", [7], [50, "a", [46], [50, "bc", [46, "bp", "bq"], [52, "br", [30, [16, [15, "bq"], [15, "i"]], [7]]], [66, "bs", [15, "br"], [46, [22, [16, [15, "bs"], [15, "j"]], [46, [53, [52, "bt", [16, [16, [15, "bs"], [15, "j"]], [15, "l"]]], [52, "bu", ["bh", [15, "bp"], [16, [16, [15, "bs"], [15, "j"]], [15, "m"]]]], [2, [15, "bp"], "setHitData", [7, [15, "bt"], ["bd", [15, "bu"]]]]]], [46, [22, [16, [15, "bs"], [15, "k"]], [46, [53, [52, "bt", [16, [16, [15, "bs"], [15, "k"]], [15, "l"]]], [2, [15, "bp"], "setHitData", [7, [15, "bt"], [44]]]]]]]]]]], [50, "bd", [46, "bp"], [22, [28, [15, "bp"]], [46, [36, [15, "bp"]]]], [52, "bq", ["c", [15, "bp"]]], [52, "br", [21, [15, "bq"], [15, "bq"]]], [22, [15, "br"], [46, [36, [15, "bp"]]]], [36, [15, "bq"]]], [50, "be", [46, "bp", "bq", "br"], [22, [1, [15, "bq"], [28, ["bg", [15, "bp"], [15, "bq"]]]], [46, [36, false]]], [22, [30, [28, [15, "br"]], [20, [17, [15, "br"], "length"], 0]], [46, [36, true]]], [53, [41, "bs"], [3, "bs", 0], [63, [7, "bs"], [23, [15, "bs"], [17, [15, "br"], "length"]], [33, [15, "bs"], [3, "bs", [0, [15, "bs"], 1]]], [46, [53, [52, "bt", [30, [16, [16, [15, "br"], [15, "bs"]], [15, "q"]], [7]]], [22, ["bf", [15, "bp"], [15, "bt"]], [46, [36, true]]]]]]], [36, false]], [50, "bf", [46, "bp", "bq"], [53, [41, "br"], [3, "br", 0], [63, [7, "br"], [23, [15, "br"], [17, [15, "bq"], "length"]], [33, [15, "br"], [3, "br", [0, [15, "br"], 1]]], [46, [53, [52, "bs", [16, [15, "bq"], [15, "br"]]], [52, "bt", ["bg", [15, "bp"], [15, "bs"], false]], [22, [16, [15, "b"], "enableUrlDecodeEventUsage"], [46, [53, [52, "bu", [16, [30, [16, [15, "bs"], [15, "t"]], [7]], 0]], [22, [1, [20, [16, [15, "bu"], [15, "u"]], [15, "p"]], [21, [2, [15, "bb"], "indexOf", [7, [16, [16, [15, "bu"], [15, "p"]], [15, "o"]]]], [27, 1]]], [46, [53, [52, "bv", ["bg", [15, "bp"], [15, "bs"], true]], [22, [21, [15, "bt"], [15, "bv"]], [46, [53, [52, "bw", [30, [2, [15, "bp"], "getMetadata", [7, [15, "y"]]], [7]]], [2, [15, "bw"], "push", [7, [39, [15, "bt"], [15, "ba"], [15, "z"]]]], [2, [15, "bp"], "setMetadata", [7, [15, "y"], [15, "bw"]]]]]]]]]]]], [22, [28, [15, "bt"]], [46, [36, false]]]]]]], [36, true]], [50, "bg", [46, "bp", "bq", "br"], [52, "bs", [30, [16, [15, "bq"], [15, "t"]], [7]]], [41, "bt"], [3, "bt", ["bh", [15, "bp"], [16, [15, "bs"], 0]]], [41, "bu"], [3, "bu", ["bh", [15, "bp"], [16, [15, "bs"], 1]]], [22, [1, [15, "br"], [15, "bt"]], [46, [3, "bt", ["h", [15, "bt"]]]]], [52, "bv", [16, [15, "bq"], [15, "s"]]], [22, [30, [30, [30, [20, [15, "bv"], "eqi"], [20, [15, "bv"], "swi"]], [20, [15, "bv"], "ewi"]], [20, [15, "bv"], "cni"]], [46, [22, [15, "bt"], [46, [3, "bt", [2, ["e", [15, "bt"]], "toLowerCase", [7]]]]], [22, [15, "bu"], [46, [3, "bu", [2, ["e", [15, "bu"]], "toLowerCase", [7]]]]]]], [41, "bw"], [3, "bw", false], [38, [15, "bv"], [46, "eq", "eqi", "sw", "swi", "ew", "ewi", "cn", "cni", "lt", "le", "gt", "ge", "re", "rei"], [46, [5, [46]], [5, [46, [3, "bw", [20, ["e", [15, "bt"]], ["e", [15, "bu"]]]], [4]]], [5, [46]], [5, [46, [3, "bw", [20, [2, ["e", [15, "bt"]], "indexOf", [7, ["e", [15, "bu"]]]], 0]], [4]]], [5, [46]], [5, [46, [41, "bx"], [3, "bx", ["e", [15, "bt"]]], [41, "by"], [3, "by", ["e", [15, "bu"]]], [52, "bz", [37, [17, [15, "bx"], "length"], [17, [15, "by"], "length"]]], [3, "bw", [1, [19, [15, "bz"], 0], [20, [2, [15, "bx"], "indexOf", [7, [15, "by"], [15, "bz"]]], [15, "bz"]]]], [4]]], [5, [46]], [5, [46, [3, "bw", [19, [2, ["e", [15, "bt"]], "indexOf", [7, ["e", [15, "bu"]]]], 0]], [4]]], [5, [46, [3, "bw", [23, ["c", [15, "bt"]], ["c", [15, "bu"]]]], [4]]], [5, [46, [3, "bw", [24, ["c", [15, "bt"]], ["c", [15, "bu"]]]], [4]]], [5, [46, [3, "bw", [18, ["c", [15, "bt"]], ["c", [15, "bu"]]]], [4]]], [5, [46, [3, "bw", [19, ["c", [15, "bt"]], ["c", [15, "bu"]]]], [4]]], [5, [46, [22, [21, [15, "bt"], [44]], [46, [53, [52, "ca", ["f", [15, "bu"]]], [22, [15, "ca"], [46, [3, "bw", ["g", [15, "ca"], [15, "bt"]]]]]]]], [4]]], [5, [46, [22, [21, [15, "bt"], [44]], [46, [53, [52, "ca", ["f", [15, "bu"], "i"]], [22, [15, "ca"], [46, [3, "bw", ["g", [15, "ca"], [15, "bt"]]]]]]]], [4]]], [9, [46]]]], [22, [28, [28, [16, [15, "bq"], [15, "r"]]]], [46, [36, [28, [15, "bw"]]]]], [36, [15, "bw"]]], [50, "bh", [46, "bp", "bq"], [22, [28, [15, "bq"]], [46, [36, [44]]]], [38, [16, [15, "bq"], [15, "u"]], [46, "event_name", "const", "event_param"], [46, [5, [46, [36, [2, [15, "bp"], "getEventName", [7]]]]], [5, [46, [36, [16, [15, "bq"], [15, "n"]]]]], [5, [46, [52, "br", [16, [16, [15, "bq"], [15, "p"]], [15, "o"]]], [22, [1, [16, [15, "b"], "enableEesPagePath"], [20, [15, "br"], [15, "w"]]], [46, [36, ["bk", [15, "bp"]]]]], [22, [20, [15, "br"], [15, "v"]], [46, [36, ["bl", [15, "bp"]]]]], [36, [2, [15, "bp"], "getHitData", [7, [15, "br"]]]]]], [9, [46, [36, [44]]]]]]], [50, "bj", [46, "bp"], [22, [28, [15, "bp"]], [46, [36, [15, "bp"]]]], [52, "bq", [2, [15, "bp"], "split", [7, "&"]]], [52, "br", [7]], [43, [15, "bq"], 0, [2, [16, [15, "bq"], 0], "substring", [7, 1]]], [53, [41, "bs"], [3, "bs", 0], [63, [7, "bs"], [23, [15, "bs"], [17, [15, "bq"], "length"]], [33, [15, "bs"], [3, "bs", [0, [15, "bs"], 1]]], [46, [53, [52, "bt", [16, [15, "bq"], [15, "bs"]]], [52, "bu", [2, [15, "bt"], "indexOf", [7, "="]]], [52, "bv", [39, [19, [15, "bu"], 0], [2, [15, "bt"], "substring", [7, 0, [15, "bu"]]], [15, "bt"]]], [22, [28, [16, [15, "bi"], [15, "bv"]]], [46, [2, [15, "br"], "push", [7, [16, [15, "bq"], [15, "bs"]]]]]]]]]], [22, [17, [15, "br"], "length"], [46, [36, [0, "?", [2, [15, "br"], "join", [7, "&"]]]]]], [36, ""]], [50, "bk", [46, "bp"], [52, "bq", [2, [15, "bp"], "getHitData", [7, [15, "w"]]]], [22, [15, "bq"], [46, [36, [15, "bq"]]]], [52, "br", [2, [15, "bp"], "getHitData", [7, [15, "x"]]]], [22, [21, [40, [15, "br"]], "string"], [46, [36, [44]]]], [52, "bs", [16, [15, "b"], "includeQueryInEesPagePath"]], [52, "bt", ["d", [15, "br"]]], [22, [28, [15, "bt"]], [46, [36, [44]]]], [36, [0, [17, [15, "bt"], "pathname"], ["bj", [39, [15, "bs"], [17, [15, "bt"], "search"], ""]]]]], [50, "bl", [46, "bp"], [52, "bq", [2, [15, "bp"], "getHitData", [7, [15, "v"]]]], [22, [15, "bq"], [46, [36, [15, "bq"]]]], [52, "br", [2, [15, "bp"], "getHitData", [7, [15, "x"]]]], [22, [21, [40, [15, "br"]], "string"], [46, [36, [44]]]], [52, "bs", ["d", [15, "br"]]], [22, [28, [15, "bs"]], [46, [36, [44]]]], [36, [17, [15, "bs"], "hostname"]]], [50, "bo", [46, "bp"], [22, [28, [15, "bp"]], [46, [36, true]]], [3, "bp", ["e", [15, "bp"]]], [66, "bq", [15, "bn"], [46, [22, [20, [2, [15, "bp"], "indexOf", [7, [15, "bq"]]], 0], [46, [36, true]]]]], [22, [18, [2, [15, "bm"], "indexOf", [7, [15, "bp"]]], [27, 1]], [46, [36, true]]], [36, false]], [52, "b", [13, [41, "$0"], [3, "$0", ["require", "internal.getFlags"]], ["$0"]]], [52, "c", ["require", "makeNumber"]], [52, "d", ["require", "parseUrl"]], [52, "e", ["require", "makeString"]], [52, "f", ["require", "internal.createRegex"]], [52, "g", ["require", "internal.testRegex"]], [52, "h", ["require", "decodeUriComponent"]], [52, "i", "event_param_ops"], [52, "j", "edit_param"], [52, "k", "delete_param"], [52, "l", "param_name"], [52, "m", "param_value"], [52, "n", "const_value"], [52, "o", "param_name"], [52, "p", "event_param"], [52, "q", "predicates"], [52, "r", "negate"], [52, "s", "type"], [52, "t", "values"], [52, "u", "type"], [52, "v", "page_hostname"], [52, "w", "page_path"], [52, "x", "page_location"], [52, "y", "event_usage"], [52, "z", 20], [52, "ba", 21], [52, "bb", [7, [15, "w"], [15, "x"], "page_referrer"]], [52, "bi", [8, "__utma", 1, "__utmb", 1, "__utmc", 1, "__utmk", 1, "__utmv", 1, "__utmx", 1, "__utmz", 1, "__ga", 1, "_gac", 1, "_gl", 1, "dclid", 1, "gbraid", 1, "gclid", 1, "gclsrc", 1, "utm_campaign", 1, "utm_content", 1, "utm_expid", 1, "utm_id", 1, "utm_medium", 1, "utm_nooverride", 1, "utm_referrer", 1, "utm_source", 1, "utm_term", 1, "wbraid", 1]], [52, "bm", [7, "app_remove", "app_store_refund", "app_store_subscription_cancel", "app_store_subscription_convert", "app_store_subscription_renew", "first_open", "first_visit", "in_app_purchase", "session_start", "user_engagement"]], [52, "bn", [7, "_", "ga_", "google_", "gtag.", "firebase_"]], [36, [8, "applyParamOperations", [15, "bc"], "preHitMatchesRule", [15, "be"], "resolveValue", [15, "bh"], "isEventNameFalsyOrReserved", [15, "bo"]]]], [36, ["a"]]]], ["$0"]]]
        , [52, "__module_ccdEmDownloadActivity", [13, [41, "$0"], [3, "$0", [51, "", [7], [50, "a", [46], [50, "h", [46, "i", "j"], ["c", [15, "i"], [51, "", [7, "k"], [22, [30, [21, [2, [15, "k"], "getEventName", [7]], [15, "f"]], [28, [2, [15, "k"], "getMetadata", [7, [15, "g"]]]]], [46, [36]]], [22, ["b", [15, "i"], [15, "e"]], [46, [2, [15, "k"], "abort", [7]], [36]]], [2, [15, "k"], "setMetadata", [7, [15, "d"], false]], [22, [28, [15, "j"]], [46, [2, [15, "k"], "setHitData", [7, "link_id", [44]]], [2, [15, "k"], "setHitData", [7, "link_url", [44]]], [2, [15, "k"], "setHitData", [7, "link_text", [44]]], [2, [15, "k"], "setHitData", [7, "file_name", [44]]], [2, [15, "k"], "setHitData", [7, "file_extension", [44]]]]]]]], [52, "b", ["require", "internal.getProductSettingsParameter"]], [52, "c", ["require", "internal.registerCcdCallback"]], [52, "d", "speculative"], [52, "e", "ae_block_downloads"], [52, "f", "file_download"], [52, "g", "em_event"], [36, [8, "registerDownloadActivityCallback", [15, "h"]]]], [36, ["a"]]]], ["$0"]]]
        , [52, "__module_ccdEmOutboundClickActivity", [13, [41, "$0"], [3, "$0", [51, "", [7], [50, "a", [46], [50, "h", [46, "i", "j"], ["c", [15, "i"], [51, "", [7, "k"], [22, [30, [21, [2, [15, "k"], "getEventName", [7]], [15, "f"]], [28, [2, [15, "k"], "getMetadata", [7, [15, "g"]]]]], [46, [36]]], [22, ["b", [15, "i"], [15, "e"]], [46, [2, [15, "k"], "abort", [7]], [36]]], [2, [15, "k"], "setMetadata", [7, [15, "d"], false]], [22, [28, [15, "j"]], [46, [2, [15, "k"], "setHitData", [7, "link_id", [44]]], [2, [15, "k"], "setHitData", [7, "link_classes", [44]]], [2, [15, "k"], "setHitData", [7, "link_url", [44]]], [2, [15, "k"], "setHitData", [7, "link_domain", [44]]], [2, [15, "k"], "setHitData", [7, "outbound", [44]]]]]]]], [52, "b", ["require", "internal.getProductSettingsParameter"]], [52, "c", ["require", "internal.registerCcdCallback"]], [52, "d", "speculative"], [52, "e", "ae_block_outbound_click"], [52, "f", "click"], [52, "g", "em_event"], [36, [8, "registerOutbackClickActivityCallback", [15, "h"]]]], [36, ["a"]]]], ["$0"]]]
        , [52, "__module_ccdEmPageViewActivity", [13, [41, "$0"], [3, "$0", [51, "", [7], [50, "a", [46], [50, "i", [46, "j", "k"], ["c", [15, "j"], [51, "", [7, "l"], [22, [30, [21, [2, [15, "l"], "getEventName", [7]], [15, "g"]], [28, [2, [15, "l"], "getMetadata", [7, [15, "h"]]]]], [46, [36]]], [22, ["b", [15, "j"], [15, "f"]], [46, [2, [15, "l"], "abort", [7]], [36]]], [2, [15, "l"], "setMetadata", [7, [15, "e"], false]], ["d", [15, "j"], "page_referrer", [2, [15, "l"], "getHitData", [7, "page_referrer"]]]]]], [52, "b", ["require", "internal.getProductSettingsParameter"]], [52, "c", ["require", "internal.registerCcdCallback"]], [52, "d", ["require", "internal.setRemoteConfigParameter"]], [52, "e", "speculative"], [52, "f", "ae_block_history"], [52, "g", "page_view"], [52, "h", "em_event"], [36, [8, "registerPageViewActivityCallback", [15, "i"]]]], [36, ["a"]]]], ["$0"]]]
        , [52, "__module_ccdEmScrollActivity", [13, [41, "$0"], [3, "$0", [51, "", [7], [50, "a", [46], [50, "h", [46, "i", "j"], ["c", [15, "i"], [51, "", [7, "k"], [22, [30, [21, [2, [15, "k"], "getEventName", [7]], [15, "f"]], [28, [2, [15, "k"], "getMetadata", [7, [15, "g"]]]]], [46, [36]]], [22, ["b", [15, "i"], [15, "e"]], [46, [2, [15, "k"], "abort", [7]], [36]]], [2, [15, "k"], "setMetadata", [7, [15, "d"], false]], [22, [28, [15, "j"]], [46, [2, [15, "k"], "setHitData", [7, "percent_scrolled", [44]]]]]]]], [52, "b", ["require", "internal.getProductSettingsParameter"]], [52, "c", ["require", "internal.registerCcdCallback"]], [52, "d", "speculative"], [52, "e", "ae_block_scroll"], [52, "f", "scroll"], [52, "g", "em_event"], [36, [8, "registerScrollActivityCallback", [15, "h"]]]], [36, ["a"]]]], ["$0"]]]
        , [52, "__module_ccdEmVideoActivity", [13, [41, "$0"], [3, "$0", [51, "", [7], [50, "a", [46], [50, "j", [46, "k", "l"], ["c", [15, "k"], [51, "", [7, "m"], [52, "n", [2, [15, "m"], "getEventName", [7]]], [52, "o", [30, [30, [20, [15, "n"], [15, "f"]], [20, [15, "n"], [15, "g"]]], [20, [15, "n"], [15, "h"]]]], [22, [30, [28, [15, "o"]], [28, [2, [15, "m"], "getMetadata", [7, [15, "i"]]]]], [46, [36]]], [22, ["b", [15, "k"], [15, "e"]], [46, [2, [15, "m"], "abort", [7]], [36]]], [2, [15, "m"], "setMetadata", [7, [15, "d"], false]], [22, [28, [15, "l"]], [46, [2, [15, "m"], "setHitData", [7, "video_current_time", [44]]], [2, [15, "m"], "setHitData", [7, "video_duration", [44]]], [2, [15, "m"], "setHitData", [7, "video_percent", [44]]], [2, [15, "m"], "setHitData", [7, "video_provider", [44]]], [2, [15, "m"], "setHitData", [7, "video_title", [44]]], [2, [15, "m"], "setHitData", [7, "video_url", [44]]], [2, [15, "m"], "setHitData", [7, "visible", [44]]]]]]]], [52, "b", ["require", "internal.getProductSettingsParameter"]], [52, "c", ["require", "internal.registerCcdCallback"]], [52, "d", "speculative"], [52, "e", "ae_block_video"], [52, "f", "video_start"], [52, "g", "video_progress"], [52, "h", "video_complete"], [52, "i", "em_event"], [36, [8, "registerVideoActivityCallback", [15, "j"]]]], [36, ["a"]]]], ["$0"]]]

      ]
      , "entities": {
        "__ccd_conversion_marking": { "2": true, "4": true }
        ,
        "__ccd_em_download": { "2": true, "4": true }
        ,
        "__ccd_em_outbound_click": { "2": true, "4": true }
        ,
        "__ccd_em_page_view": { "2": true, "4": true }
        ,
        "__ccd_em_scroll": { "2": true, "4": true }
        ,
        "__ccd_em_site_search": { "2": true, "4": true }
        ,
        "__ccd_em_video": { "2": true, "4": true }
        ,
        "__ogt_event_create": { "2": true, "4": true }
        ,
        "__ogt_google_signals": { "2": true, "4": true }
        ,
        "__set_product_settings": { "2": true, "4": true }


      }
      , "permissions": {
        "__ccd_conversion_marking": {}
        ,
        "__ccd_em_download": { "listen_data_layer": { "accessType": "specific", "allowedEvents": ["gtm.linkClick"] }, "access_template_storage": {}, "detect_link_click_events": { "allowWaitForTags": "" } }
        ,
        "__ccd_em_outbound_click": { "get_url": { "urlParts": "any", "queriesAllowed": "any" }, "listen_data_layer": { "accessType": "specific", "allowedEvents": ["gtm.linkClick"] }, "access_template_storage": {}, "detect_link_click_events": { "allowWaitForTags": "" } }
        ,
        "__ccd_em_page_view": { "listen_data_layer": { "accessType": "specific", "allowedEvents": ["gtm.historyChange-v2"] }, "process_dom_events": { "targets": [{ "targetType": "window", "eventName": "pushstate" }, { "targetType": "window", "eventName": "popstate" }] }, "access_template_storage": {} }
        ,
        "__ccd_em_scroll": { "listen_data_layer": { "accessType": "specific", "allowedEvents": ["gtm.scrollDepth"] }, "process_dom_events": { "targets": [{ "targetType": "window", "eventName": "resize" }, { "targetType": "window", "eventName": "scroll" }] }, "access_template_storage": {} }
        ,
        "__ccd_em_site_search": { "get_url": { "urlParts": "any", "queriesAllowed": "any" }, "read_container_data": {} }
        ,
        "__ccd_em_video": { "listen_data_layer": { "accessType": "specific", "allowedEvents": ["gtm.video"] }, "process_dom_events": { "targets": [{ "targetType": "element", "eventName": "onStateChange" }, { "targetType": "element", "eventName": "onPlaybackRateChange" }] }, "access_template_storage": {}, "detect_youtube_activity_events": { "allowFixMissingJavaScriptApi": false } }
        ,
        "__ogt_event_create": { "access_template_storage": {} }
        ,
        "__ogt_google_signals": { "read_container_data": {} }
        ,
        "__set_product_settings": {}


      }



      , "security_groups": {
        "google": [
          "__ccd_conversion_marking"
          ,
          "__ccd_em_download"
          ,
          "__ccd_em_outbound_click"
          ,
          "__ccd_em_page_view"
          ,
          "__ccd_em_scroll"
          ,
          "__ccd_em_site_search"
          ,
          "__ccd_em_video"
          ,
          "__ogt_event_create"
          ,
          "__ogt_google_signals"
          ,
          "__set_product_settings"

        ]


      }



    };


    /*
    
     Copyright The Closure Library Authors.
     SPDX-License-Identifier: Apache-2.0
    */
    var aa, ba = function (a) { var b = 0; return function () { return b < a.length ? { done: !1, value: a[b++] } : { done: !0 } } }, da = function (a) { return a.raw = a }, ea = function (a, b) { a.raw = b; return a }, fa = function (a) { var b = "undefined" != typeof Symbol && Symbol.iterator && a[Symbol.iterator]; if (b) return b.call(a); if ("number" == typeof a.length) return { next: ba(a) }; throw Error(String(a) + " is not an iterable or ArrayLike"); }, ka = "function" == typeof Object.create ? Object.create : function (a) { var b = function () { }; b.prototype = a; return new b }, la;
    if ("function" == typeof Object.setPrototypeOf) la = Object.setPrototypeOf; else { var ma; a: { var na = { a: !0 }, oa = {}; try { oa.__proto__ = na; ma = oa.a; break a } catch (a) { } ma = !1 } la = ma ? function (a, b) { a.__proto__ = b; if (a.__proto__ !== b) throw new TypeError(a + " is not extensible"); return a } : null }
    var pa = la, qa = function (a, b) { a.prototype = ka(b.prototype); a.prototype.constructor = a; if (pa) pa(a, b); else for (var c in b) if ("prototype" != c) if (Object.defineProperties) { var d = Object.getOwnPropertyDescriptor(b, c); d && Object.defineProperty(a, c, d) } else a[c] = b[c]; a.fn = b.prototype }, ra = this || self, ta = function (a) { return a }; var ua = function (a, b) { this.h = a; this.B = b }; var va = function () { this.B = {}; this.F = {} }; aa = va.prototype; aa.get = function (a) { return this.B["dust." + a] }; aa.set = function (a, b) { a = "dust." + a; this.F.hasOwnProperty(a) || (this.B[a] = b) }; aa.Ih = function (a, b) { this.set(a, b); this.F["dust." + a] = !0 }; aa.has = function (a) { return this.B.hasOwnProperty("dust." + a) }; aa.remove = function (a) { a = "dust." + a; this.F.hasOwnProperty(a) || delete this.B[a] }; var wa = function () { this.quota = {} }; wa.prototype.reset = function () { this.quota = {} }; var xa = function (a, b) { this.W = a; this.K = function (c, d, e) { return c.apply(d, e) }; this.C = b; this.B = new va; this.h = this.F = void 0 }; xa.prototype.add = function (a, b) { ya(this, a, b, !1) }; var ya = function (a, b, c, d) { d ? a.B.Ih(b, c) : a.B.set(b, c) }; xa.prototype.set = function (a, b) { !this.B.has(a) && this.C && this.C.has(a) ? this.C.set(a, b) : this.B.set(a, b) }; xa.prototype.get = function (a) { return this.B.has(a) ? this.B.get(a) : this.C ? this.C.get(a) : void 0 }; xa.prototype.has = function (a) { return !!this.B.has(a) || !(!this.C || !this.C.has(a)) };
    var za = function (a) { var b = new xa(a.W, a); a.F && (b.F = a.F); b.K = a.K; b.h = a.h; return b }; var Aa = function () { }, Ba = function (a) { return "function" === typeof a }, h = function (a) { return "string" === typeof a }, Ca = function (a) { return "number" === typeof a && !isNaN(a) }, Da = Array.isArray, Fa = function (a, b) { if (a && Da(a)) for (var c = 0; c < a.length; c++)if (a[c] && b(a[c])) return a[c] }, Ga = function (a, b) { if (!Ca(a) || !Ca(b) || a > b) a = 0, b = 2147483647; return Math.floor(Math.random() * (b - a + 1) + a) }, Ja = function (a, b) { for (var c = new Ha, d = 0; d < a.length; d++)c.set(a[d], !0); for (var e = 0; e < b.length; e++)if (c.get(b[e])) return !0; return !1 }, l = function (a,
      b) { for (var c in a) Object.prototype.hasOwnProperty.call(a, c) && b(c, a[c]) }, Ka = function (a) { return !!a && ("[object Arguments]" === Object.prototype.toString.call(a) || Object.prototype.hasOwnProperty.call(a, "callee")) }, La = function (a) { return Math.round(Number(a)) || 0 }, Ma = function (a) { return "false" === String(a).toLowerCase() ? !1 : !!a }, Na = function (a) { var b = []; if (Da(a)) for (var c = 0; c < a.length; c++)b.push(String(a[c])); return b }, Oa = function (a) { return a ? a.replace(/^\s+|\s+$/g, "") : "" }, Pa = function () { return new Date(Date.now()) },
      Ra = function () { return Pa().getTime() }, Ha = function () { this.prefix = "gtm."; this.values = {} }; Ha.prototype.set = function (a, b) { this.values[this.prefix + a] = b }; Ha.prototype.get = function (a) { return this.values[this.prefix + a] };
    var Sa = function (a, b, c) { return a && a.hasOwnProperty(b) ? a[b] : c }, Ta = function (a) { var b = a; return function () { if (b) { var c = b; b = void 0; try { c() } catch (d) { } } } }, Ua = function (a, b) { for (var c in b) b.hasOwnProperty(c) && (a[c] = b[c]) }, Wa = function (a) { for (var b in a) if (a.hasOwnProperty(b)) return !0; return !1 }, Xa = function (a, b) { for (var c = [], d = 0; d < a.length; d++)c.push(a[d]), c.push.apply(c, b[a[d]] || []); return c }, Ya = function (a, b) {
      var c = z; b = b || []; for (var d = c, e = 0; e < a.length - 1; e++) {
        if (!d.hasOwnProperty(a[e])) return; d = d[a[e]]; if (0 <=
          b.indexOf(d)) return
      } return d
    }, Za = function (a, b) { for (var c = {}, d = c, e = a.split("."), f = 0; f < e.length - 1; f++)d = d[e[f]] = {}; d[e[e.length - 1]] = b; return c }, $a = /^\w{1,9}$/, ab = function (a, b) { a = a || {}; b = b || ","; var c = []; l(a, function (d, e) { $a.test(d) && e && c.push(d) }); return c.join(b) }, bb = function (a, b) { function c() { ++d === b && (e(), e = null, c.done = !0) } var d = 0, e = a; c.done = !1; return c }; function cb(a, b) { for (var c, d = 0; d < b.length && !(c = db(a, b[d]), c instanceof ua); d++); return c } function db(a, b) { try { var c = a.get(String(b[0])); if (!c || "function" !== typeof c.invoke) throw Error("Attempting to execute non-function " + b[0] + "."); return c.invoke.apply(c, [a].concat(b.slice(1))) } catch (e) { var d = a.F; d && d(e, b.context ? { id: b[0], line: b.context.line } : null); throw e; } }; var eb = function () { this.C = new wa; this.h = new xa(this.C) }; eb.prototype.execute = function (a, b) { var c = Array.prototype.slice.call(arguments, 0); return this.B(c) }; eb.prototype.B = function (a) { for (var b, c = 0; c < arguments.length; c++)b = db(this.h, arguments[c]); return b }; eb.prototype.F = function (a, b) { var c = za(this.h); c.h = a; for (var d, e = 1; e < arguments.length; e++)d = db(c, arguments[e]); return d }; var fb = function () { va.call(this); this.C = !1 }; qa(fb, va); var gb = function (a, b) { var c = [], d; for (d in a.B) if (a.B.hasOwnProperty(d)) switch (d = d.substr(5), b) { case 1: c.push(d); break; case 2: c.push(a.get(d)); break; case 3: c.push([d, a.get(d)]) }return c }; aa = fb.prototype; aa.set = function (a, b) { this.C || va.prototype.set.call(this, a, b) }; aa.Ih = function (a, b) { this.C || va.prototype.Ih.call(this, a, b) }; aa.remove = function (a) { this.C || va.prototype.remove.call(this, a) }; aa.Bb = function () { this.C = !0 }; aa.qj = function () { return this.C };/*
 jQuery (c) 2005, 2012 jQuery Foundation, Inc. jquery.org/license. */
    var hb = /\[object (Boolean|Number|String|Function|Array|Date|RegExp)\]/, ib = function (a) { if (null == a) return String(a); var b = hb.exec(Object.prototype.toString.call(Object(a))); return b ? b[1].toLowerCase() : "object" }, jb = function (a, b) { return Object.prototype.hasOwnProperty.call(Object(a), b) }, kb = function (a) {
      if (!a || "object" != ib(a) || a.nodeType || a == a.window) return !1; try { if (a.constructor && !jb(a, "constructor") && !jb(a.constructor.prototype, "isPrototypeOf")) return !1 } catch (c) { return !1 } for (var b in a); return void 0 ===
        b || jb(a, b)
    }, lb = function (a, b) { var c = b || ("array" == ib(a) ? [] : {}), d; for (d in a) if (jb(a, d)) { var e = a[d]; "array" == ib(e) ? ("array" != ib(c[d]) && (c[d] = []), c[d] = lb(e, c[d])) : kb(e) ? (kb(c[d]) || (c[d] = {}), c[d] = lb(e, c[d])) : c[d] = e } return c }; var mb = function (a) { for (var b = [], c = 0; c < a.length(); c++)a.has(c) && (b[c] = a.get(c)); return b }, nb = function (a) { if (void 0 == a || Da(a) || kb(a)) return !0; switch (typeof a) { case "boolean": case "number": case "string": case "function": return !0 }return !1 }, ob = function (a) { return "number" === typeof a && 0 <= a && isFinite(a) && 0 === a % 1 || "string" === typeof a && "-" !== a[0] && a === "" + parseInt(a, 10) }; var pb = function (a) { this.B = new fb; this.h = []; this.C = !1; a = a || []; for (var b in a) a.hasOwnProperty(b) && (ob(b) ? this.h[Number(b)] = a[Number(b)] : this.B.set(b, a[b])) }; aa = pb.prototype; aa.toString = function (a) { if (a && 0 <= a.indexOf(this)) return ""; for (var b = [], c = 0; c < this.h.length; c++) { var d = this.h[c]; null === d || void 0 === d ? b.push("") : d instanceof pb ? (a = a || [], a.push(this), b.push(d.toString(a)), a.pop()) : b.push(String(d)) } return b.join(",") };
    aa.set = function (a, b) { if (!this.C) if ("length" === a) { if (!ob(b)) throw Error("RangeError: Length property must be a valid integer."); this.h.length = Number(b) } else ob(a) ? this.h[Number(a)] = b : this.B.set(a, b) }; aa.get = function (a) { return "length" === a ? this.length() : ob(a) ? this.h[Number(a)] : this.B.get(a) }; aa.length = function () { return this.h.length }; aa.Pb = function () { for (var a = gb(this.B, 1), b = 0; b < this.h.length; b++)a.push(b + ""); return new pb(a) }; aa.remove = function (a) { ob(a) ? delete this.h[Number(a)] : this.B.remove(a) };
    aa.pop = function () { return this.h.pop() }; aa.push = function (a) { return this.h.push.apply(this.h, Array.prototype.slice.call(arguments)) }; aa.shift = function () { return this.h.shift() }; aa.splice = function (a, b, c) { return new pb(this.h.splice.apply(this.h, arguments)) }; aa.unshift = function (a) { return this.h.unshift.apply(this.h, Array.prototype.slice.call(arguments)) }; aa.has = function (a) { return ob(a) && this.h.hasOwnProperty(a) || this.B.has(a) }; aa.Bb = function () { this.C = !0; Object.freeze(this.h); this.B.Bb() }; aa.qj = function () { return this.C }; var qb = function () { fb.call(this) }; qa(qb, fb); qb.prototype.Pb = function () { return new pb(gb(this, 1)) }; function rb() { for (var a = sb, b = {}, c = 0; c < a.length; ++c)b[a[c]] = c; return b } function tb() { var a = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"; a += a.toLowerCase() + "0123456789-_"; return a + "." } var sb, ub; function vb(a) { sb = sb || tb(); ub = ub || rb(); for (var b = [], c = 0; c < a.length; c += 3) { var d = c + 1 < a.length, e = c + 2 < a.length, f = a.charCodeAt(c), g = d ? a.charCodeAt(c + 1) : 0, k = e ? a.charCodeAt(c + 2) : 0, m = f >> 2, n = (f & 3) << 4 | g >> 4, p = (g & 15) << 2 | k >> 6, q = k & 63; e || (q = 64, d || (p = 64)); b.push(sb[m], sb[n], sb[p], sb[q]) } return b.join("") }
    function wb(a) { function b(m) { for (; d < a.length;) { var n = a.charAt(d++), p = ub[n]; if (null != p) return p; if (!/^[\s\xa0]*$/.test(n)) throw Error("Unknown base64 encoding at char: " + n); } return m } sb = sb || tb(); ub = ub || rb(); for (var c = "", d = 0; ;) { var e = b(-1), f = b(0), g = b(64), k = b(64); if (64 === k && -1 === e) return c; c += String.fromCharCode(e << 2 | f >> 4); 64 != g && (c += String.fromCharCode(f << 4 & 240 | g >> 2), 64 != k && (c += String.fromCharCode(g << 6 & 192 | k))) } }; var xb = {}, yb = function (a, b) { xb[a] = xb[a] || []; xb[a][b] = !0 }, zb = function () { delete xb.GA4_EVENT }, Ab = function (a) { var b = xb[a]; if (!b || 0 === b.length) return ""; for (var c = [], d = 0, e = 0; e < b.length; e++)0 === e % 8 && 0 < e && (c.push(String.fromCharCode(d)), d = 0), b[e] && (d |= 1 << e % 8); 0 < d && c.push(String.fromCharCode(d)); return vb(c.join("")).replace(/\.+$/, "") }; var Bb = Array.prototype.indexOf ? function (a, b) { return Array.prototype.indexOf.call(a, b, void 0) } : function (a, b) { if ("string" === typeof a) return "string" !== typeof b || 1 != b.length ? -1 : a.indexOf(b, 0); for (var c = 0; c < a.length; c++)if (c in a && a[c] === b) return c; return -1 }; var Cb, Db = function () { if (void 0 === Cb) { var a = null, b = ra.trustedTypes; if (b && b.createPolicy) { try { a = b.createPolicy("goog#html", { createHTML: ta, createScript: ta, createScriptURL: ta }) } catch (c) { ra.console && ra.console.error(c.message) } Cb = a } else Cb = a } return Cb }; var Eb = function (a) { this.h = a }; Eb.prototype.toString = function () { return this.h + "" }; var Fb = function (a) { return a instanceof Eb && a.constructor === Eb ? a.h : "type_error:TrustedResourceUrl" }, Gb = {}, Hb = function (a) { var b = a, c = Db(), d = c ? c.createScriptURL(b) : b; return new Eb(d, Gb) }; var Ib = function (a) { this.h = a }; Ib.prototype.toString = function () { return this.h.toString() }; var Jb = function (a) { return a instanceof Ib && a.constructor === Ib ? a.h : "type_error:SafeUrl" }, Kb = /^(?:(?:https?|mailto|ftp):|[^:/?#]*(?:[/?#]|$))/i, Lb = {}, Mb = new Ib("about:invalid#zClosurez", Lb); var Nb, Ob; a: { for (var Pb = ["CLOSURE_FLAGS"], Qb = ra, Rb = 0; Rb < Pb.length; Rb++)if (Qb = Qb[Pb[Rb]], null == Qb) { Ob = null; break a } Ob = Qb } var Sb = Ob && Ob[610401301]; Nb = null != Sb ? Sb : !1; function Tb() { var a = ra.navigator; if (a) { var b = a.userAgent; if (b) return b } return "" } var Ub, Vb = ra.navigator; Ub = Vb ? Vb.userAgentData || null : null; function Wb(a) { return Nb ? Ub ? Ub.brands.some(function (b) { var c = b.brand; return c && -1 != c.indexOf(a) }) : !1 : !1 } function Xb(a) { return -1 != Tb().indexOf(a) }; function Yb() { return Nb ? !!Ub && 0 < Ub.brands.length : !1 } function Zb() { return Yb() ? !1 : Xb("Opera") } function $b() { return Xb("Firefox") || Xb("FxiOS") } function ac() { return Yb() ? Wb("Chromium") : (Xb("Chrome") || Xb("CriOS")) && !(Yb() ? 0 : Xb("Edge")) || Xb("Silk") }; var bc = {}, cc = function (a) { this.h = a }; cc.prototype.toString = function () { return this.h.toString() }; var dc = function (a) { return a instanceof cc && a.constructor === cc ? a.h : "type_error:SafeHtml" };/*

 SPDX-License-Identifier: Apache-2.0
*/
    var ec; try { new URL("s://g"), ec = !0 } catch (a) { ec = !1 } var fc = ec; function gc(a) { var b; a: if (fc) { var c; try { c = new URL(a) } catch (g) { b = "https:"; break a } b = c.protocol } else { var d; b: { var e = document.createElement("a"); try { e.href = a } catch (g) { d = void 0; break b } var f = e.protocol; d = ":" === f || "" === f ? "https:" : f } b = d } if ("javascript:" !== b) return a }; var hc = {}; var ic = function () { }, jc = function (a) { this.h = a }; qa(jc, ic); jc.prototype.toString = function () { return this.h }; function kc(a, b) { var c = [new jc(lc[0].toLowerCase(), hc)]; if (0 === c.length) throw Error(""); var d = c.map(function (f) { var g; if (f instanceof jc) g = f.h; else throw Error(""); return g }), e = b.toLowerCase(); if (d.every(function (f) { return 0 !== e.indexOf(f) })) throw Error('Attribute "' + b + '" does not match any of the allowed prefixes.'); a.setAttribute(b, "true") } function mc(a) { var b = a.tagName; if ("SCRIPT" === b || "STYLE" === b) throw Error(""); }; function nc(a, b) { var c = b instanceof Ib ? Jb(b) : gc(b); void 0 !== c && (a.action = c) }; var oc = da([""]), pc = ea(["\x00"], ["\\0"]), qc = ea(["\n"], ["\\n"]), rc = ea(["\x00"], ["\\u0000"]); function sc(a) { return -1 === a.toString().indexOf("`") } sc(function (a) { return a(oc) }) || sc(function (a) { return a(pc) }) || sc(function (a) { return a(qc) }) || sc(function (a) { return a(rc) }); var tc = function (a) { this.Jl = a }; function uc(a) { return new tc(function (b) { return b.substr(0, a.length + 1).toLowerCase() === a + ":" }) } var vc = [uc("data"), uc("http"), uc("https"), uc("mailto"), uc("ftp"), new tc(function (a) { return /^[^:]*([/?#]|$)/.test(a) })]; function wc(a, b) { b = void 0 === b ? vc : b; for (var c = 0; c < b.length; ++c) { var d = b[c]; if (d instanceof tc && d.Jl(a)) return new Ib(a, Lb) } } function xc(a) { var b; b = void 0 === b ? vc : b; return wc(a, b) || Mb }; function yc(a) { var b = a = Bc(a), c = Db(), d = c ? c.createHTML(b) : b; return new cc(d, bc) } function Bc(a) { return null === a ? "null" : void 0 === a ? "undefined" : a }; var z = window, C = document, Cc = navigator, Dc = C.currentScript && C.currentScript.src, Ec = function (a, b) { var c = z[a]; z[a] = void 0 === c ? b : c; return z[a] }, Fc = function (a, b) { b && (a.addEventListener ? a.onload = b : a.onreadystatechange = function () { a.readyState in { loaded: 1, complete: 1 } && (a.onreadystatechange = null, b()) }) }, Gc = { async: 1, nonce: 1, onerror: 1, onload: 1, src: 1, type: 1 }, Hc = { onload: 1, src: 1, width: 1, height: 1, style: 1 }; function Ic(a, b, c) { b && l(b, function (d, e) { d = d.toLowerCase(); c.hasOwnProperty(d) || a.setAttribute(d, e) }) }
    var Jc = function (a, b, c, d, e) { var f = C.createElement("script"); Ic(f, d, Gc); f.type = "text/javascript"; f.async = !0; var g; g = Hb(Bc(a)); f.src = Fb(g); var k, m, n, p = null == (n = (m = (f.ownerDocument && f.ownerDocument.defaultView || window).document).querySelector) ? void 0 : n.call(m, "script[nonce]"); (k = p ? p.nonce || p.getAttribute("nonce") || "" : "") && f.setAttribute("nonce", k); Fc(f, b); c && (f.onerror = c); if (e) e.appendChild(f); else { var q = C.getElementsByTagName("script")[0] || C.body || C.head; q.parentNode.insertBefore(f, q) } return f }, Kc = function () {
      if (Dc) {
        var a =
          Dc.toLowerCase(); if (0 === a.indexOf("https://")) return 2; if (0 === a.indexOf("http://")) return 3
      } return 1
    }, Lc = function (a, b, c, d, e) { var f; f = void 0 === f ? !0 : f; var g = e, k = !1; g || (g = C.createElement("iframe"), k = !0); Ic(g, c, Hc); d && l(d, function (n, p) { g.dataset[n] = p }); f && (g.height = "0", g.width = "0", g.style.display = "none", g.style.visibility = "hidden"); if (k) { var m = C.body && C.body.lastChild || C.body || C.head; m.parentNode.insertBefore(g, m) } Fc(g, b); void 0 !== a && (g.src = a); return g }, Mc = function (a, b, c, d) {
      var e = new Image(1, 1); Ic(e,
        d, {}); e.onload = function () { e.onload = null; b && b() }; e.onerror = function () { e.onerror = null; c && c() }; e.src = a
    }, Nc = function (a, b, c, d) { a.addEventListener ? a.addEventListener(b, c, !!d) : a.attachEvent && a.attachEvent("on" + b, c) }, Oc = function (a, b, c) { a.removeEventListener ? a.removeEventListener(b, c, !1) : a.detachEvent && a.detachEvent("on" + b, c) }, F = function (a) { z.setTimeout(a, 0) }, Qc = function (a, b) { return a && b && a.attributes && a.attributes[b] ? a.attributes[b].value : null }, Rc = function (a) {
      var b = a.innerText || a.textContent || ""; b && " " !=
        b && (b = b.replace(/^[\s\xa0]+|[\s\xa0]+$/g, "")); b && (b = b.replace(/(\xa0+|\s{2,}|\n|\r\t)/g, " ")); return b
    }, Sc = function (a) { var b = C.createElement("div"), c = b, d = yc("A<div>" + a + "</div>"); 1 === c.nodeType && mc(c); c.innerHTML = dc(d); b = b.lastChild; for (var e = []; b.firstChild;)e.push(b.removeChild(b.firstChild)); return e }, Tc = function (a, b, c) { c = c || 100; for (var d = {}, e = 0; e < b.length; e++)d[b[e]] = !0; for (var f = a, g = 0; f && g <= c; g++) { if (d[String(f.tagName).toLowerCase()]) return f; f = f.parentElement } return null }, Uc = function (a) {
      var b;
      try { b = Cc.sendBeacon && Cc.sendBeacon(a) } catch (c) { yb("TAGGING", 15) } b || Mc(a)
    }, Vc = function (a, b) { var c = a[b]; c && "string" === typeof c.animVal && (c = c.animVal); return c }, Wc = function (a) { var b = { headers: { "Attribution-Reporting-Eligible": "trigger" }, keepalive: !0, attributionReporting: { eventSourceEligible: !0, triggerEligible: !0 } }; try { z.fetch(a, b) } catch (c) { } }, Xc = function () { var a = z.performance; if (a && Ba(a.now)) return a.now() }, Yc = function () { return z.performance || void 0 }; var Zc = function (a, b) { return H(this, a) && H(this, b) }, $c = function (a, b) { return H(this, a) === H(this, b) }, ad = function (a, b) { return H(this, a) || H(this, b) }, bd = function (a, b) { a = H(this, a); b = H(this, b); return -1 < String(a).indexOf(String(b)) }, cd = function (a, b) { a = String(H(this, a)); b = String(H(this, b)); return a.substring(0, b.length) === b }, dd = function (a, b) { a = H(this, a); b = H(this, b); switch (a) { case "pageLocation": var c = z.location.href; b instanceof qb && b.get("stripProtocol") && (c = c.replace(/^https?:\/\//, "")); return c } }; var ed = function (a, b) { fb.call(this); this.K = a; this.W = b }; qa(ed, fb); ed.prototype.toString = function () { return this.K }; ed.prototype.Pb = function () { return new pb(gb(this, 1)) }; ed.prototype.invoke = function (a, b) { return this.W.apply(new hd(this, a), Array.prototype.slice.call(arguments, 1)) }; ed.prototype.h = function (a, b) { try { return this.invoke.apply(this, Array.prototype.slice.call(arguments, 0)) } catch (c) { } }; var hd = function (a, b) { this.B = a; this.h = b }, H = function (a, b) { return Da(b) ? db(a.h, b) : b }, J = function (a) { return a.B.K }; var id = function () { this.map = new Map }; id.prototype.set = function (a, b) { this.map.set(a, b) }; id.prototype.get = function (a) { return this.map.get(a) }; var jd = function () { this.keys = []; this.values = [] }; jd.prototype.set = function (a, b) { this.keys.push(a); this.values.push(b) }; jd.prototype.get = function (a) { var b = this.keys.indexOf(a); if (-1 < b) return this.values[b] }; function kd() { try { return Map ? new id : new jd } catch (a) { return new jd } }; var ld = function (a) { if (a instanceof ld) return a; this.h = a }; ld.prototype.toString = function () { return String(this.h) }; var nd = function (a) { fb.call(this); this.h = a; this.set("then", md(this)); this.set("catch", md(this, !0)); this.set("finally", md(this, !1, !0)) }; qa(nd, qb); var md = function (a, b, c) { b = void 0 === b ? !1 : b; c = void 0 === c ? !1 : c; return new ed("", function (d, e) { b && (e = d, d = void 0); c && (e = d); d instanceof ed || (d = void 0); e instanceof ed || (e = void 0); var f = za(this.h), g = function (m) { return function (n) { return c ? (m.invoke(f), a.h) : m.invoke(f, n) } }, k = a.h.then(d && g(d), e && g(e)); return new nd(k) }) }; var pd = function (a, b, c) {
      var d = kd(), e = function (g, k) { for (var m = gb(g, 1), n = 0; n < m.length; n++)k[m[n]] = f(g.get(m[n])) }, f = function (g) {
        var k = d.get(g); if (k) return k; if (g instanceof pb) { var m = []; d.set(g, m); for (var n = g.Pb(), p = 0; p < n.length(); p++)m[n.get(p)] = f(g.get(n.get(p))); return m } if (g instanceof nd) return g.h; if (g instanceof qb) { var q = {}; d.set(g, q); e(g, q); return q } if (g instanceof ed) {
          var r = function () {
            for (var u = Array.prototype.slice.call(arguments, 0), v = 0; v < u.length; v++)u[v] = od(u[v], b, c); var w = new xa(b ? b.W :
              new wa); b && (w.h = b.h); return f(g.invoke.apply(g, [w].concat(u)))
          }; d.set(g, r); e(g, r); return r
        } var t = !1; switch (c) { case 1: t = !0; break; case 2: t = !1; break; case 3: t = !1; break; default: }if (g instanceof ld && t) return g.h; switch (typeof g) { case "boolean": case "number": case "string": case "undefined": return g; case "object": if (null === g) return null }
      }; return f(a)
    }, od = function (a, b, c) {
      var d = kd(), e = function (g,
        k) { for (var m in g) g.hasOwnProperty(m) && k.set(m, f(g[m])) }, f = function (g) { var k = d.get(g); if (k) return k; if (Da(g) || Ka(g)) { var m = new pb([]); d.set(g, m); for (var n in g) g.hasOwnProperty(n) && m.set(n, f(g[n])); return m } if (kb(g)) { var p = new qb; d.set(g, p); e(g, p); return p } if ("function" === typeof g) { var q = new ed("", function (x) { for (var y = Array.prototype.slice.call(arguments, 0), A = 0; A < y.length; A++)y[A] = pd(H(this, y[A]), b, c); return f((0, this.h.K)(g, g, y)) }); d.set(g, q); e(g, q); return q } var v = typeof g; if (null === g || "string" === v || "number" === v || "boolean" === v) return g; var w = !1; switch (c) { case 1: w = !0; break; case 2: w = !1; break; default: }if (void 0 !== g && w) return new ld(g) }; return f(a)
    }; var qd = function () { var a = !1; return a }; var rd = {
      supportedMethods: "concat every filter forEach hasOwnProperty indexOf join lastIndexOf map pop push reduce reduceRight reverse shift slice some sort splice unshift toString".split(" "), concat: function (a, b) { for (var c = [], d = 0; d < this.length(); d++)c.push(this.get(d)); for (var e = 1; e < arguments.length; e++)if (arguments[e] instanceof pb) for (var f = arguments[e], g = 0; g < f.length(); g++)c.push(f.get(g)); else c.push(arguments[e]); return new pb(c) }, every: function (a, b) {
        for (var c = this.length(), d = 0; d < this.length() &&
          d < c; d++)if (this.has(d) && !b.invoke(a, this.get(d), d, this)) return !1; return !0
      }, filter: function (a, b) { for (var c = this.length(), d = [], e = 0; e < this.length() && e < c; e++)this.has(e) && b.invoke(a, this.get(e), e, this) && d.push(this.get(e)); return new pb(d) }, forEach: function (a, b) { for (var c = this.length(), d = 0; d < this.length() && d < c; d++)this.has(d) && b.invoke(a, this.get(d), d, this) }, hasOwnProperty: function (a, b) { return this.has(b) }, indexOf: function (a, b, c) {
        var d = this.length(), e = void 0 === c ? 0 : Number(c); 0 > e && (e = Math.max(d + e, 0)); for (var f =
          e; f < d; f++)if (this.has(f) && this.get(f) === b) return f; return -1
      }, join: function (a, b) { for (var c = [], d = 0; d < this.length(); d++)c.push(this.get(d)); return c.join(b) }, lastIndexOf: function (a, b, c) { var d = this.length(), e = d - 1; void 0 !== c && (e = 0 > c ? d + c : Math.min(c, e)); for (var f = e; 0 <= f; f--)if (this.has(f) && this.get(f) === b) return f; return -1 }, map: function (a, b) { for (var c = this.length(), d = [], e = 0; e < this.length() && e < c; e++)this.has(e) && (d[e] = b.invoke(a, this.get(e), e, this)); return new pb(d) }, pop: function () { return this.pop() }, push: function (a,
        b) { return this.push.apply(this, Array.prototype.slice.call(arguments, 1)) }, reduce: function (a, b, c) { var d = this.length(), e, f = 0; if (void 0 !== c) e = c; else { if (0 === d) throw Error("TypeError: Reduce on List with no elements."); for (var g = 0; g < d; g++)if (this.has(g)) { e = this.get(g); f = g + 1; break } if (g === d) throw Error("TypeError: Reduce on List with no elements."); } for (var k = f; k < d; k++)this.has(k) && (e = b.invoke(a, e, this.get(k), k, this)); return e }, reduceRight: function (a, b, c) {
          var d = this.length(), e, f = d - 1; if (void 0 !== c) e = c; else {
            if (0 ===
              d) throw Error("TypeError: ReduceRight on List with no elements."); for (var g = 1; g <= d; g++)if (this.has(d - g)) { e = this.get(d - g); f = d - (g + 1); break } if (g > d) throw Error("TypeError: ReduceRight on List with no elements.");
          } for (var k = f; 0 <= k; k--)this.has(k) && (e = b.invoke(a, e, this.get(k), k, this)); return e
        }, reverse: function () { for (var a = mb(this), b = a.length - 1, c = 0; 0 <= b; b--, c++)a.hasOwnProperty(b) ? this.set(c, a[b]) : this.remove(c); return this }, shift: function () { return this.shift() }, slice: function (a, b, c) {
          var d = this.length();
          void 0 === b && (b = 0); b = 0 > b ? Math.max(d + b, 0) : Math.min(b, d); c = void 0 === c ? d : 0 > c ? Math.max(d + c, 0) : Math.min(c, d); c = Math.max(b, c); for (var e = [], f = b; f < c; f++)e.push(this.get(f)); return new pb(e)
        }, some: function (a, b) { for (var c = this.length(), d = 0; d < this.length() && d < c; d++)if (this.has(d) && b.invoke(a, this.get(d), d, this)) return !0; return !1 }, sort: function (a, b) {
          var c = mb(this); void 0 === b ? c.sort() : c.sort(function (e, f) { return Number(b.invoke(a, e, f)) }); for (var d = 0; d < c.length; d++)c.hasOwnProperty(d) ? this.set(d, c[d]) : this.remove(d);
          return this
        }, splice: function (a, b, c, d) { return this.splice.apply(this, Array.prototype.splice.call(arguments, 1, arguments.length - 1)) }, toString: function () { return this.toString() }, unshift: function (a, b) { return this.unshift.apply(this, Array.prototype.slice.call(arguments, 1)) }
    }; var sd = function (a) { var b; b = Error.call(this, a); this.message = b.message; "stack" in b && (this.stack = b.stack) }; qa(sd, Error); var td = { charAt: 1, concat: 1, indexOf: 1, lastIndexOf: 1, match: 1, replace: 1, search: 1, slice: 1, split: 1, substring: 1, toLowerCase: 1, toLocaleLowerCase: 1, toString: 1, toUpperCase: 1, toLocaleUpperCase: 1, trim: 1 }, ud = new ua("break"), vd = new ua("continue"), wd = function (a, b) { return H(this, a) + H(this, b) }, xd = function (a, b) { return H(this, a) && H(this, b) }, yd = function (a, b, c) {
      a = H(this, a); b = H(this, b); c = H(this, c); if (!(c instanceof pb)) throw Error("Error: Non-List argument given to Apply instruction."); if (null === a || void 0 === a) {
        var d = "TypeError: Can't read property " +
          b + " of " + a + "."; if (qd()) throw new sd(d); throw Error(d);
      } var e = "number" === typeof a; if ("boolean" === typeof a || e) { if ("toString" === b) { if (e && c.length()) { var f = pd(c.get(0)); try { return a.toString(f) } catch (y) { } } return a.toString() } var g = "TypeError: " + a + "." + b + " is not a function."; if (qd()) throw new sd(g); throw Error(g); } if ("string" === typeof a) {
        if (td.hasOwnProperty(b)) {
          var k = 2; k = 1;
          var m = pd(c, void 0, k); return od(a[b].apply(a, m), this.h)
        } var n = "TypeError: " + b + " is not a function"; if (qd()) throw new sd(n); throw Error(n);
      } if (a instanceof pb) { if (a.has(b)) { var p = a.get(b); if (p instanceof ed) { var q = mb(c); q.unshift(this.h); return p.invoke.apply(p, q) } var r = "TypeError: " + b + " is not a function"; if (qd()) throw new sd(r); throw Error(r); } if (0 <= rd.supportedMethods.indexOf(b)) { var t = mb(c); t.unshift(this.h); return rd[b].apply(a, t) } } if (a instanceof ed || a instanceof qb) {
        if (a.has(b)) {
          var u = a.get(b);
          if (u instanceof ed) { var v = mb(c); v.unshift(this.h); return u.invoke.apply(u, v) } var w = "TypeError: " + b + " is not a function"; if (qd()) throw new sd(w); throw Error(w);
        } if ("toString" === b) return a instanceof ed ? a.K : a.toString(); if ("hasOwnProperty" === b) return a.has.apply(a, mb(c))
      } if (a instanceof ld && "toString" === b) return a.toString(); var x = "TypeError: Object has no '" + b + "' property."; if (qd()) throw new sd(x); throw Error(x);
    }, zd = function (a, b) {
      a = H(this, a); if ("string" !== typeof a) throw Error("Invalid key name given for assignment.");
      var c = this.h; if (!c.has(a)) throw Error("Attempting to assign to undefined value " + b); var d = H(this, b); c.set(a, d); return d
    }, Ad = function (a) { var b = za(this.h), c = cb(b, Array.prototype.slice.apply(arguments)); if (c instanceof ua) return c }, Bd = function () { return ud }, Cd = function (a) { for (var b = H(this, a), c = 0; c < b.length; c++) { var d = H(this, b[c]); if (d instanceof ua) return d } }, Dd = function (a) {
      for (var b = this.h, c = 0; c < arguments.length - 1; c += 2) {
        var d = arguments[c]; if ("string" === typeof d) {
          var e = H(this, arguments[c + 1]); ya(b, d, e,
            !0)
        }
      }
    }, Ed = function () { return vd }, Fd = function (a, b) { return new ua(a, H(this, b)) }, Gd = function (a, b, c) { var d = new pb; b = H(this, b); for (var e = 0; e < b.length; e++)d.push(b[e]); var f = [51, a, d].concat(Array.prototype.splice.call(arguments, 2, arguments.length - 2)); this.h.add(a, H(this, f)) }, Hd = function (a, b) { return H(this, a) / H(this, b) }, Id = function (a, b) { a = H(this, a); b = H(this, b); var c = a instanceof ld, d = b instanceof ld; return c || d ? c && d ? a.h == b.h : !1 : a == b }, Jd = function (a) {
      for (var b, c = 0; c < arguments.length; c++)b = H(this, arguments[c]);
      return b
    }; function Kd(a, b, c, d) { for (var e = 0; e < b(); e++) { var f = a(c(e)), g = cb(f, d); if (g instanceof ua) { if ("break" === g.h) break; if ("return" === g.h) return g } } } function Ld(a, b, c) { if ("string" === typeof b) return Kd(a, function () { return b.length }, function (f) { return f }, c); if (b instanceof qb || b instanceof pb || b instanceof ed) { var d = b.Pb(), e = d.length(); return Kd(a, function () { return e }, function (f) { return d.get(f) }, c) } }
    var Md = function (a, b, c) { a = H(this, a); b = H(this, b); c = H(this, c); var d = this.h; return Ld(function (e) { d.set(a, e); return d }, b, c) }, Nd = function (a, b, c) { a = H(this, a); b = H(this, b); c = H(this, c); var d = this.h; return Ld(function (e) { var f = za(d); ya(f, a, e, !0); return f }, b, c) }, Od = function (a, b, c) { a = H(this, a); b = H(this, b); c = H(this, c); var d = this.h; return Ld(function (e) { var f = za(d); f.add(a, e); return f }, b, c) }, Qd = function (a, b, c) { a = H(this, a); b = H(this, b); c = H(this, c); var d = this.h; return Pd(function (e) { d.set(a, e); return d }, b, c) }, Rd =
      function (a, b, c) { a = H(this, a); b = H(this, b); c = H(this, c); var d = this.h; return Pd(function (e) { var f = za(d); ya(f, a, e, !0); return f }, b, c) }, Sd = function (a, b, c) { a = H(this, a); b = H(this, b); c = H(this, c); var d = this.h; return Pd(function (e) { var f = za(d); f.add(a, e); return f }, b, c) };
    function Pd(a, b, c) { if ("string" === typeof b) return Kd(a, function () { return b.length }, function (d) { return b[d] }, c); if (b instanceof pb) return Kd(a, function () { return b.length() }, function (d) { return b.get(d) }, c); if (qd()) throw new sd("The value is not iterable."); throw new TypeError("The value is not iterable."); }
    var Td = function (a, b, c, d) { function e(p, q) { for (var r = 0; r < f.length(); r++) { var t = f.get(r); q.add(t, p.get(t)) } } var f = H(this, a); if (!(f instanceof pb)) throw Error("TypeError: Non-List argument given to ForLet instruction."); var g = this.h; d = H(this, d); var k = za(g); for (e(g, k); db(k, b);) { var m = cb(k, d); if (m instanceof ua) { if ("break" === m.h) break; if ("return" === m.h) return m } var n = za(g); e(k, n); db(n, c); k = n } }, Ud = function (a, b, c) {
      var d = this.h, e = H(this, b); if (!(e instanceof pb)) throw Error("Error: non-List value given for Fn argument names.");
      var f = Array.prototype.slice.call(arguments, 2); return new ed(a, function () { return function (g) { var k = za(d); void 0 === k.h && (k.h = this.h.h); for (var m = Array.prototype.slice.call(arguments, 0), n = 0; n < m.length; n++)if (m[n] = H(this, m[n]), m[n] instanceof ua) return m[n]; for (var p = e.get("length"), q = 0; q < p; q++)q < m.length ? k.add(e.get(q), m[q]) : k.add(e.get(q), void 0); k.add("arguments", new pb(m)); var r = cb(k, f); if (r instanceof ua) return "return" === r.h ? r.B : r } }())
    }, Vd = function (a) { a = H(this, a); var b = this.h, c = !1; if (c && !b.has(a)) throw new ReferenceError(a + " is not defined."); return b.get(a) }, Wd = function (a, b) { var c; a = H(this, a); b = H(this, b); if (void 0 === a || null === a) { var d = "TypeError: cannot access property of " + a + "."; if (qd()) throw new sd(d); throw Error(d); } if (a instanceof qb || a instanceof pb || a instanceof ed) c = a.get(b); else if ("string" === typeof a) "length" === b ? c = a.length : ob(b) && (c = a[b]); else if (a instanceof ld) return; return c }, Xd =
        function (a, b) { return H(this, a) > H(this, b) }, Yd = function (a, b) { return H(this, a) >= H(this, b) }, Zd = function (a, b) { a = H(this, a); b = H(this, b); a instanceof ld && (a = a.h); b instanceof ld && (b = b.h); return a === b }, $d = function (a, b) { return !Zd.call(this, a, b) }, ae = function (a, b, c) { var d = []; H(this, a) ? d = H(this, b) : c && (d = H(this, c)); var e = cb(this.h, d); if (e instanceof ua) return e }, be = function (a, b) { return H(this, a) < H(this, b) }, ce = function (a, b) { return H(this, a) <= H(this, b) }, de = function (a) {
          for (var b = new pb, c = 0; c < arguments.length; c++) {
            var d =
              H(this, arguments[c]); b.push(d)
          } return b
        }, ee = function (a) { for (var b = new qb, c = 0; c < arguments.length - 1; c += 2) { var d = H(this, arguments[c]) + "", e = H(this, arguments[c + 1]); b.set(d, e) } return b }, fe = function (a, b) { return H(this, a) % H(this, b) }, ge = function (a, b) { return H(this, a) * H(this, b) }, he = function (a) { return -H(this, a) }, ie = function (a) { return !H(this, a) }, je = function (a, b) { return !Id.call(this, a, b) }, ke = function () { return null }, le = function (a, b) { return H(this, a) || H(this, b) }, me = function (a, b) { var c = H(this, a); H(this, b); return c },
      ne = function (a) { return H(this, a) }, oe = function (a) { return Array.prototype.slice.apply(arguments) }, pe = function (a) { return new ua("return", H(this, a)) }, qe = function (a, b, c) { a = H(this, a); b = H(this, b); c = H(this, c); if (null === a || void 0 === a) { var d = "TypeError: Can't set property " + b + " of " + a + "."; if (qd()) throw new sd(d); throw Error(d); } (a instanceof ed || a instanceof pb || a instanceof qb) && a.set(b, c); return c }, re = function (a, b) { return H(this, a) - H(this, b) }, se = function (a, b, c) {
        a = H(this, a); var d = H(this, b), e = H(this, c); if (!Da(d) ||
          !Da(e)) throw Error("Error: Malformed switch instruction."); for (var f, g = !1, k = 0; k < d.length; k++)if (g || a === H(this, d[k])) if (f = H(this, e[k]), f instanceof ua) { var m = f.h; if ("break" === m) return; if ("return" === m || "continue" === m) return f } else g = !0; if (e.length === d.length + 1 && (f = H(this, e[e.length - 1]), f instanceof ua && ("return" === f.h || "continue" === f.h))) return f
      }, ve = function (a, b, c) { return H(this, a) ? H(this, b) : H(this, c) }, we = function (a) { a = H(this, a); return a instanceof ed ? "function" : typeof a }, xe = function (a) {
        for (var b =
          this.h, c = 0; c < arguments.length; c++) { var d = arguments[c]; "string" !== typeof d || b.add(d, void 0) }
      }, ye = function (a, b, c, d) { var e = H(this, d); if (H(this, c)) { var f = cb(this.h, e); if (f instanceof ua) { if ("break" === f.h) return; if ("return" === f.h) return f } } for (; H(this, a);) { var g = cb(this.h, e); if (g instanceof ua) { if ("break" === g.h) break; if ("return" === g.h) return g } H(this, b) } }, ze = function (a) { return ~Number(H(this, a)) }, Ae = function (a, b) { return Number(H(this, a)) << Number(H(this, b)) }, Be = function (a, b) {
        return Number(H(this, a)) >> Number(H(this,
          b))
      }, Ce = function (a, b) { return Number(H(this, a)) >>> Number(H(this, b)) }, De = function (a, b) { return Number(H(this, a)) & Number(H(this, b)) }, Ee = function (a, b) { return Number(H(this, a)) ^ Number(H(this, b)) }, Fe = function (a, b) { return Number(H(this, a)) | Number(H(this, b)) }, Ge = function () { }; var Ie = function () { this.h = new eb; He(this) }; Ie.prototype.execute = function (a) { return this.h.B(a) }; var He = function (a) { var b = function (c, d) { var e = new ed(String(c), d); e.Bb(); a.h.h.set(String(c), e) }; b("map", ee); b("and", Zc); b("contains", bd); b("equals", $c); b("or", ad); b("startsWith", cd); b("variable", dd) }; var Ke = function () { this.h = new eb; Je(this) }; Ke.prototype.execute = function (a) { return Le(this.h.B(a)) };
    var Me = function (a, b, c) { return Le(a.h.F(b, c)) }, Je = function (a) {
      var b = function (c, d) { var e = String(c), f = new ed(e, d); f.Bb(); a.h.h.set(e, f) }; b(0, wd); b(1, xd); b(2, yd); b(3, zd); b(56, De); b(57, Ae); b(58, ze); b(59, Fe); b(60, Be); b(61, Ce); b(62, Ee); b(53, Ad); b(4, Bd); b(5, Cd); b(52, Dd); b(6, Ed); b(49, Fd); b(7, de); b(8, ee); b(9, Cd); b(50, Gd); b(10, Hd); b(12, Id); b(13, Jd); b(51, Ud); b(47, Md); b(54, Nd); b(55, Od); b(63, Td); b(64, Qd); b(65, Rd); b(66, Sd); b(15, Vd); b(16, Wd); b(17, Wd); b(18, Xd); b(19, Yd); b(20, Zd); b(21, $d); b(22, ae); b(23, be); b(24, ce);
      b(25, fe); b(26, ge); b(27, he); b(28, ie); b(29, je); b(45, ke); b(30, le); b(32, me); b(33, me); b(34, ne); b(35, ne); b(46, oe); b(36, pe); b(43, qe); b(37, re); b(38, se); b(39, ve); b(40, we); b(44, Ge); b(41, xe); b(42, ye)
    }; function Le(a) { if (a instanceof ua || a instanceof ed || a instanceof pb || a instanceof qb || a instanceof ld || null === a || void 0 === a || "string" === typeof a || "number" === typeof a || "boolean" === typeof a) return a }; function Ne(a) { switch (a) { case 1: return "1"; case 2: case 4: return "0"; default: return "-" } } function Oe(a) { switch (a) { case 1: return "G"; case 3: return "g"; case 2: return "D"; case 4: return "d"; case 0: return "g"; default: return "g" } } function Pe(a, b) { var c = a[1] || 0, d = a[2] || 0; switch (b) { case 0: return "G1" + Ne(c) + Ne(d); case 1: return "G2" + Oe(c) + Oe(d); default: return "g1--" } }; var Qe = function () {
      var a = function (b) { return { toString: function () { return b } } }; return {
        Pj: a("consent"), Qh: a("convert_case_to"), Rh: a("convert_false_to"), Sh: a("convert_null_to"), Th: a("convert_true_to"), Uh: a("convert_undefined_to"), Lm: a("debug_mode_metadata"), xa: a("function"), Kg: a("instance_name"), yk: a("live_only"), zk: a("malware_disabled"), Ak: a("metadata"), Dk: a("original_activity_id"), Sm: a("original_vendor_template_id"), Rm: a("once_on_load"), Ck: a("once_per_event"), Qi: a("once_per_load"), Wm: a("priority_override"),
        Xm: a("respected_consent_types"), Ui: a("setup_tags"), he: a("tag_id"), Zi: a("teardown_tags")
      }
    }(); var mf = [], nf = function (a) { return void 0 == mf[a] ? !1 : mf[a] }; var of;
    var pf = [], rf = [], sf = [], tf = [], uf = [], vf = {}, wf, xf, yf = function (a) { xf = xf || a }, zf = function (a) { }, Af, Bf = [], Cf = function (a, b) { var c = {}; c[Qe.xa] = "__" + a; for (var d in b) b.hasOwnProperty(d) && (c["vtp_" + d] = b[d]); return c }, Df = function (a,
      b) {
        var c = a[Qe.xa], d = b && b.event; if (!c) throw Error("Error: No function name given for function call."); var e = vf[c], f = b && 2 === b.type && d.Dh && e && -1 !== Bf.indexOf(c), g = {}, k = {}, m; for (m in a) a.hasOwnProperty(m) && 0 === m.indexOf("vtp_") && (e && d && d.fj && d.fj(a[m]), e && (g[m] = a[m]), !e || f) && (k[m.substr(4)] = a[m]); e && d && d.ej && (g.vtp_gtmCachedValues = d.ej); if (b) {
          if (null == b.name) {
            var n; a: {
              var p = b.index; if (null == p) n = ""; else {
                var q; switch (b.type) { case 2: q = pf[p]; break; case 1: q = tf[p]; break; default: n = ""; break a }var r = q && q[Qe.Kg];
                n = r ? String(r) : ""
              }
            } b.name = n
          } e && (g.vtp_gtmEntityIndex = b.index, g.vtp_gtmEntityName = b.name)
        } var t, u; e && (t = e(g)); if (!e || f) u = of(c, k, b); f && d && (nb(t) ? typeof t !== typeof u && d.Dh(d.id, c) : t !== u && d.Dh(d.id, c)); return e ? t : u
    }, Ff = function (a, b, c) { c = c || []; var d = {}, e; for (e in a) a.hasOwnProperty(e) && (d[e] = Ef(a[e], b, c)); return d }, Ef = function (a, b, c) {
      if (Da(a)) {
        var d; switch (a[0]) {
          case "function_id": return a[1]; case "list": d = []; for (var e = 1; e < a.length; e++)d.push(Ef(a[e], b, c)); return d; case "macro": var f = a[1]; if (c[f]) return;
            var g = pf[f]; if (!g || b.ih(g)) return; c[f] = !0; var k = String(g[Qe.Kg]); try { var m = Ff(g, b, c); m.vtp_gtmEventId = b.id; b.priorityId && (m.vtp_gtmPriorityId = b.priorityId); d = Df(m, { event: b, index: f, type: 2, name: k }); Af && (d = Af.Ok(d, m)) } catch (y) { b.sj && b.sj(y, Number(f), k), d = !1 } c[f] = !1; return d; case "map": d = {}; for (var n = 1; n < a.length; n += 2)d[Ef(a[n], b, c)] = Ef(a[n + 1], b, c); return d; case "template": d = []; for (var p = !1, q = 1; q < a.length; q++) { var r = Ef(a[q], b, c); xf && (p = p || xf.Gl(r)); d.push(r) } return xf && p ? xf.Rk(d) : d.join(""); case "escape": d =
              Ef(a[1], b, c); if (xf && Da(a[1]) && "macro" === a[1][0] && xf.Hl(a)) return xf.fm(d); d = String(d); for (var t = 2; t < a.length; t++)Re[a[t]] && (d = Re[a[t]](d)); return d; case "tag": var u = a[1]; if (!tf[u]) throw Error("Unable to resolve tag reference " + u + "."); return d = { kj: a[2], index: u }; case "zb": var v = { arg0: a[2], arg1: a[3], ignore_case: a[5] }; v[Qe.xa] = a[1]; var w = Gf(v, b, c), x = !!a[4]; return x || 2 !== w ? x !== (1 === w) : null; default: throw Error("Attempting to expand unknown Value type: " + a[0] + ".");
        }
      } return a
    }, Gf = function (a, b, c) {
      try {
        return wf(Ff(a,
          b, c))
      } catch (d) { JSON.stringify(a) } return 2
    }, Hf = function (a) { var b = a[Qe.xa]; if (!b) throw Error("Error: No function name given for function call."); return !!vf[b] }; var If = function (a, b, c) { var d; d = Error.call(this, c); this.message = d.message; "stack" in d && (this.stack = d.stack); this.h = a }; qa(If, Error); function Jf(a, b) { if (Da(a)) { Object.defineProperty(a, "context", { value: { line: b[0] } }); for (var c = 1; c < a.length; c++)Jf(a[c], b[c]) } }; var Kf = function (a, b) { var c; c = Error.call(this); this.message = c.message; "stack" in c && (this.stack = c.stack); this.Xl = a; this.B = b; this.h = [] }; qa(Kf, Error); var Mf = function () { return function (a, b) { a instanceof Kf || (a = new Kf(a, Lf)); b && a.h.push(b); throw a; } }; function Lf(a) { if (!a.length) return a; a.push({ id: "main", line: 0 }); for (var b = a.length - 1; 0 < b; b--)Ca(a[b].id) && a.splice(b++, 1); for (var c = a.length - 1; 0 < c; c--)a[c].line = a[c - 1].line; a.splice(0, 1); return a }; var Pf = function (a) { function b(r) { for (var t = 0; t < r.length; t++)d[r[t]] = !0 } for (var c = [], d = [], e = Nf(a), f = 0; f < rf.length; f++) { var g = rf[f], k = Of(g, e); if (k) { for (var m = g.add || [], n = 0; n < m.length; n++)c[m[n]] = !0; b(g.block || []) } else null === k && b(g.block || []); } for (var p = [], q = 0; q < tf.length; q++)c[q] && !d[q] && (p[q] = !0); return p }, Of = function (a, b) {
      for (var c = a["if"] || [], d = 0; d < c.length; d++) { var e = b(c[d]); if (0 === e) return !1; if (2 === e) return null } for (var f =
        a.unless || [], g = 0; g < f.length; g++) { var k = b(f[g]); if (2 === k) return null; if (1 === k) return !1 } return !0
    }, Nf = function (a) { var b = []; return function (c) { void 0 === b[c] && (b[c] = Gf(sf[c], a)); return b[c] } }; var Qf = { Ok: function (a, b) { b[Qe.Qh] && "string" === typeof a && (a = 1 == b[Qe.Qh] ? a.toLowerCase() : a.toUpperCase()); b.hasOwnProperty(Qe.Sh) && null === a && (a = b[Qe.Sh]); b.hasOwnProperty(Qe.Uh) && void 0 === a && (a = b[Qe.Uh]); b.hasOwnProperty(Qe.Th) && !0 === a && (a = b[Qe.Th]); b.hasOwnProperty(Qe.Rh) && !1 === a && (a = b[Qe.Rh]); return a } }; var Rf = function () { this.h = {} }; function Sf(a, b, c, d) { if (a) for (var e = 0; e < a.length; e++) { var f = void 0, g = "A policy function denied the permission request"; try { f = a[e].call(void 0, b, c, d), g += "." } catch (k) { g = "string" === typeof k ? g + (": " + k) : k instanceof Error ? g + (": " + k.message) : g + "." } if (!f) throw new If(c, d, g); } } function Tf(a, b, c) { return function () { var d = arguments[0]; if (d) { var e = a.h[d], f = a.h.all; if (e || f) { var g = c.apply(void 0, Array.prototype.slice.call(arguments, 0)); Sf(e, b, d, g); Sf(f, b, d, g) } } } }; var Xf = function () { var a = data.permissions || {}, b = Uf.ctid, c = this; this.B = new Rf; this.h = {}; var d = {}, e = Tf(this.B, b, function () { var f = arguments[0]; return f && d[f] ? d[f].apply(void 0, Array.prototype.slice.call(arguments, 0)) : {} }); l(a, function (f, g) { var k = {}; l(g, function (m, n) { var p = Vf(m, n); k[m] = p.assert; d[m] || (d[m] = p.J) }); c.h[f] = function (m, n) { var p = k[m]; if (!p) throw Wf(m, {}, "The requested permission " + m + " is not configured."); var q = Array.prototype.slice.call(arguments, 0); p.apply(void 0, q); e.apply(void 0, q) } }) },
      Zf = function (a) { return Yf.h[a] || function () { } }; function Vf(a, b) { var c = Cf(a, b); c.vtp_permissionName = a; c.vtp_createPermissionError = Wf; try { return Df(c) } catch (d) { return { assert: function (e) { throw new If(e, {}, "Permission " + e + " is unknown."); }, J: function () { for (var e = {}, f = 0; f < arguments.length; ++f)e["arg" + (f + 1)] = arguments[f]; return e } } } } function Wf(a, b, c) { return new If(a, b, c) }; var $f = !1; var ag = {}; ag.Im = Ma(''); ag.Tk = Ma(''); var bg = $f, cg = ag.Tk, dg = ag.Im;
    var hg = function (a) { var b = {}, c = 0; l(a, function (e, f) { if (null != f) if (f = ("" + f).replace(/~/g, "~~"), eg.hasOwnProperty(e)) b[eg[e]] = f; else if (fg.hasOwnProperty(e)) { var g = fg[e], k = f; b.hasOwnProperty(g) || (b[g] = k) } else if ("category" === e) for (var m = f.split("/", 5), n = 0; n < m.length; n++) { var p = gg[n], q = m[n]; b.hasOwnProperty(p) || (b[p] = q) } else if (27 > c) { var r = String.fromCharCode(10 > c ? 48 + c : 65 + c - 10); b["k" + r] = ("" + String(e)).replace(/~/g, "~~"); b["v" + r] = f; c++ } }); var d = []; l(b, function (e, f) { d.push("" + e + f) }); return d.join("~") },
      eg = { item_id: "id", item_name: "nm", item_brand: "br", item_category: "ca", item_category2: "c2", item_category3: "c3", item_category4: "c4", item_category5: "c5", item_variant: "va", price: "pr", quantity: "qt", coupon: "cp", item_list_name: "ln", index: "lp", item_list_id: "li", discount: "ds", affiliation: "af", promotion_id: "pi", promotion_name: "pn", creative_name: "cn", creative_slot: "cs", location_id: "lo" }, fg = { id: "id", name: "nm", brand: "br", variant: "va", list_name: "ln", list_position: "lp", list: "ln", position: "lp", creative: "cn" }, gg = ["ca",
        "c2", "c3", "c4", "c5"]; var ig = function (a) { var b = []; l(a, function (c, d) { null != d && b.push(encodeURIComponent(c) + "=" + encodeURIComponent(String(d))) }); return b.join("&") }, jg = function (a, b, c, d) { this.Pa = a.Pa; this.yc = a.yc; this.Yg = a.Yg; this.B = b; this.F = c; this.C = ig(a.Pa); this.h = ig(a.Yg); this.K = this.h.length; if (d && 16384 < this.K) throw Error("EVENT_TOO_LARGE"); }; var kg = function () { this.events = []; this.h = this.Pa = ""; this.C = 0; this.B = !1 }; kg.prototype.add = function (a) { return this.F(a) ? (this.events.push(a), this.Pa = a.C, this.h = a.B, this.C += a.K, this.B = a.F, !0) : !1 }; kg.prototype.F = function (a) { var b = 20 > this.events.length && 16384 > a.K + this.C, c = this.Pa === a.C && this.h === a.B && this.B === a.F; return 0 == this.events.length || b && c };
    var lg = function (a, b) { l(a, function (c, d) { null != d && b.push(encodeURIComponent(c) + "=" + encodeURIComponent(d)) }) }, mg = function (a, b) { var c = []; a.C && c.push(a.C); b && c.push("_s=" + b); lg(a.yc, c); var d = !1; a.h && (c.push(a.h), d = !0); var e = c.join("&"), f = "", g = e.length + a.B.length + 1; d && 2048 < g && (f = c.pop(), e = c.join("&")); return { xh: e, body: f } }, ng = function (a, b) {
      var c = a.events; if (1 == c.length) return mg(c[0], b); var d = []; a.Pa && d.push(a.Pa); for (var e = {}, f = 0; f < c.length; f++)l(c[f].yc, function (t, u) {
        null != u && (e[t] = e[t] || {}, e[t][String(u)] =
          e[t][String(u)] + 1 || 1)
      }); var g = {}; l(e, function (t, u) { var v, w = -1, x = 0; l(u, function (y, A) { x += A; var B = (y.length + t.length + 2) * (A - 1); B > w && (v = y, w = B) }); x == c.length && (g[t] = v) }); lg(g, d); b && d.push("_s=" + b); for (var k = d.join("&"), m = [], n = {}, p = 0; p < c.length; n = { Cf: n.Cf }, p++) { var q = []; n.Cf = {}; l(c[p].yc, function (t) { return function (u, v) { g[u] != "" + v && (t.Cf[u] = v) } }(n)); c[p].h && q.push(c[p].h); lg(n.Cf, q); m.push(q.join("&")) } var r = m.join("\r\n"); return { xh: k, body: r }
    }; var rg = ["matches", "webkitMatchesSelector", "mozMatchesSelector", "msMatchesSelector", "oMatchesSelector"]; function sg(a, b) { a = String(a); b = String(b); var c = a.length - b.length; return 0 <= c && a.indexOf(b, c) === c } var tg = new Ha; function ug(a, b, c) { var d = c ? "i" : void 0; try { var e = String(b) + d, f = tg.get(e); f || (f = new RegExp(b, d), tg.set(e, f)); return f.test(a) } catch (g) { return !1 } } function vg(a, b) { return 0 <= String(a).indexOf(String(b)) } function wg(a, b) { return String(a) === String(b) }
    function xg(a, b) { return Number(a) >= Number(b) } function yg(a, b) { return Number(a) <= Number(b) } function zg(a, b) { return Number(a) > Number(b) } function Ag(a, b) { return Number(a) < Number(b) } function Bg(a, b) { return 0 === String(a).indexOf(String(b)) }; var Ig = /^[1-9a-zA-Z_-][1-9a-c][1-9a-v]\d$/; function Jg(a, b) { return "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ-_"[a << 2 | b] }; var Mg = /^([a-z][a-z0-9]*):(!|\?)(\*|string|boolean|number|Fn|PixieMap|List|OpaqueValue)$/i, Ng = { Fn: "function", PixieMap: "Object", List: "Array" }, K = function (a, b, c) {
      for (var d = 0; d < b.length; d++) {
        var e = Mg.exec(b[d]); if (!e) throw Error("Internal Error in " + a); var f = e[1], g = "!" === e[2], k = e[3], m = c[d]; if (null == m) { if (g) throw Error("Error in " + a + ". Required argument " + f + " not supplied."); } else if ("*" !== k) {
          var n = typeof m; m instanceof ed ? n = "Fn" : m instanceof pb ? n = "List" : m instanceof qb ? n = "PixieMap" : m instanceof ld && (n =
            "OpaqueValue"); if (n != k) throw Error("Error in " + a + ". Argument " + f + " has type " + (Ng[n] || n) + ", which does not match required type " + (Ng[k] || k) + ".");
        }
      }
    }; function Og(a) { return "" + a }
    function Pg(a, b) { var c = []; return c }; var Qg = function (a, b) { var c = new ed(a, function () { for (var d = Array.prototype.slice.call(arguments, 0), e = 0; e < d.length; e++)d[e] = H(this, d[e]); try { return b.apply(this, d) } catch (g) { if (qd()) throw new sd(g.message); throw g; } }); c.Bb(); return c }, Rg = function (a, b) {
      var c = new qb, d; for (d in b) if (b.hasOwnProperty(d)) {
        var e = b[d]; Ba(e) ? c.set(d, Qg(a + "_" + d, e)) : kb(e) ? c.set(d, Rg(a + "_" + d,
          e)) : (Ca(e) || h(e) || "boolean" === typeof e) && c.set(d, e)
      } c.Bb(); return c
    }; var Sg = function (a, b) { K(J(this), ["apiName:!string", "message:?string"], arguments); var c = {}, d = new qb; return d = Rg("AssertApiSubject", c) }; var Tg = function (a, b) { K(J(this), ["actual:?*", "message:?string"], arguments); if (a instanceof nd) throw Error("Argument actual cannot have type Promise. Assertions on asynchronous code aren't supported."); var c = {}, d = new qb; return d = Rg("AssertThatSubject", c) }; function Ug(a) { return function () { for (var b = [], c = this.h, d = 0; d < arguments.length; ++d)b.push(pd(arguments[d], c)); return od(a.apply(null, b)) } } var Wg = function () { for (var a = Math, b = Vg, c = {}, d = 0; d < b.length; d++) { var e = b[d]; a.hasOwnProperty(e) && (c[e] = Ug(a[e].bind(a))) } return c }; var Xg = function (a) { var b; return b }; var Yg = function (a) { var b; K(J(this), ["uri:!string"], arguments); try { b = decodeURIComponent(a) } catch (c) { } return b }; var Zg = function (a) { try { return encodeURI(a) } catch (b) { } }; var $g = function (a) { try { return encodeURIComponent(a) } catch (b) { } }; function ah(a, b) { var c = !1; K(J(this), ["booleanExpression:!string", "context:?PixieMap"], arguments); var d = JSON.parse(a); if (!d) throw Error("Invalid boolean expression string was given."); var e = b ? pd(b) : {}; c = bh(d, e); return c }
    var ch = function (a, b) { for (var c = 0; c < b.length; c++) { if (void 0 === a) return; a = a[b[c]] } return a }, dh = function (a, b) { var c = b.preHit; if (c) { var d = a[0]; switch (d) { case "hitData": return 2 > a.length ? void 0 : ch(c.getHitData(a[1]), a.slice(2)); case "metadata": return 2 > a.length ? void 0 : ch(c.getMetadata(a[1]), a.slice(2)); case "eventName": return c.getEventName(); case "destinationId": return c.getDestinationId(); default: throw Error(d + " is not a valid field that can be accessed\n                      from PreHit data."); } } },
      eh = function (a, b) {
        if (a) {
          if (void 0 !== a.contextValue) { var c; a: { var d = a.contextValue, e = d.keyParts; if (e && 0 !== e.length) { var f = d.namespaceType; switch (f) { case 1: c = dh(e, b); break a; case 2: var g = b.macro; c = g ? g[e[0]] : void 0; break a; default: throw Error("Unknown Namespace Type used: " + f); } } c = void 0 } return c } if (void 0 !== a.booleanExpressionValue) return bh(a.booleanExpressionValue, b); if (void 0 !== a.booleanValue) return !!a.booleanValue; if (void 0 !== a.stringValue) return String(a.stringValue); if (void 0 !== a.integerValue) return Number(a.integerValue);
          if (void 0 !== a.doubleValue) return Number(a.doubleValue); throw Error("Unknown field used for variable of type ExpressionValue:" + a);
        }
      }, bh = function (a, b) {
        var c = a.args; if (!Da(c) || 0 === c.length) throw Error('Invalid boolean expression format. Expected "args":' + c + " property to\n         be non-empty array."); var d = function (g) { return eh(g, b) }; switch (a.type) {
          case 1: for (var e = 0; e < c.length; e++)if (d(c[e])) return !0; return !1; case 2: for (var f = 0; f < c.length; f++)if (!d(c[f])) return !1; return 0 < c.length; case 3: return !d(c[0]);
          case 4: return ug(d(c[0]), d(c[1]), !1); case 5: return wg(d(c[0]), d(c[1])); case 6: return Bg(d(c[0]), d(c[1])); case 7: return sg(d(c[0]), d(c[1])); case 8: return vg(d(c[0]), d(c[1])); case 9: return Ag(d(c[0]), d(c[1])); case 10: return yg(d(c[0]), d(c[1])); case 11: return zg(d(c[0]), d(c[1])); case 12: return xg(d(c[0]), d(c[1])); default: throw Error('Invalid boolean expression format. Expected "type" property tobe a positive integer which is less than 13.');
        }
      };
    ah.D = "internal.evaluateBooleanExpression"; var fh = function (a) { K(J(this), ["message:?string"], arguments); }; var gh = function (a, b) { K(J(this), ["min:!number", "max:!number"], arguments); return Ga(a, b) }; var hh = function () { return (new Date).getTime() }; var ih = function (a) { if (null === a) return "null"; if (a instanceof pb) return "array"; if (a instanceof ed) return "function"; if (a instanceof ld) { a = a.h; if (void 0 === a.constructor || void 0 === a.constructor.name) { var b = String(a); return b.substring(8, b.length - 1) } return String(a.constructor.name) } return typeof a }; var jh = function (a) { function b(c) { return function (d) { try { return c(d) } catch (e) { (bg || dg) && a.call(this, e.message) } } } return { parse: b(function (c) { return od(JSON.parse(c)) }), stringify: b(function (c) { return JSON.stringify(pd(c)) }) } }; var kh = function (a) { return La(pd(a, this.h)) }; var lh = function (a) { return Number(pd(a, this.h)) }; var mh = function (a) { return null === a ? "null" : void 0 === a ? "undefined" : a.toString() }; var nh = function (a, b, c) { var d = null, e = !1; return e ? d : null }; var Vg = "floor ceil round max min abs pow sqrt".split(" "); var oh = function () { var a = {}; return { jl: function (b) { return a.hasOwnProperty(b) ? a[b] : void 0 }, zm: function (b, c) { a[b] = c }, reset: function () { a = {} } } }, ph = function (a, b) { return function () { var c = Array.prototype.slice.call(arguments, 0); c.unshift(b); return ed.prototype.invoke.apply(a, c) } }, qh = function (a, b) { K(J(this), ["apiName:!string", "mock:?*"], arguments); }; var rh = {};
    rh.keys = function (a) { return new pb };
    rh.values = function (a) { return new pb };
    rh.entries = function (a) { return new pb };
    rh.freeze = function (a) { return a }; rh.delete = function (a, b) { return !1 }; var L = function (a, b, c) { var d = a.h.h; if (!d) throw Error("Missing program state."); if (d.lm) { try { d.dj.apply(null, Array.prototype.slice.call(arguments, 1)) } catch (e) { throw yb("TAGGING", 21), e; } return } d.dj.apply(null, Array.prototype.slice.call(arguments, 1)) }; var th = function () { this.h = {}; this.B = {}; }; th.prototype.get = function (a, b) { var c = this.h.hasOwnProperty(a) ? this.h[a] : void 0; return c };
    th.prototype.add = function (a, b, c) { if (this.h.hasOwnProperty(a)) throw "Attempting to add a function which already exists: " + a + "."; if (this.B.hasOwnProperty(a)) throw "Attempting to add an API with an existing private API name: " + a + "."; this.h[a] = c ? void 0 : Ba(b) ? Qg(a, b) : Rg(a, b) }; function uh(a, b) { var c = void 0; return c }; function vh() {
      var a = {};
      return a
    }; var xh = function (a) { return wh ? C.querySelectorAll(a) : null }, yh = function (a, b) {
      if (!wh) return null; if (Element.prototype.closest) try { return a.closest(b) } catch (e) { return null } var c = Element.prototype.matches || Element.prototype.webkitMatchesSelector || Element.prototype.mozMatchesSelector || Element.prototype.msMatchesSelector || Element.prototype.oMatchesSelector, d = a; if (!C.documentElement.contains(d)) return null; do { try { if (c.call(d, b)) return d } catch (e) { break } d = d.parentElement || d.parentNode } while (null !== d && 1 === d.nodeType);
      return null
    }, zh = !1; if (C.querySelectorAll) try { var Ah = C.querySelectorAll(":root"); Ah && 1 == Ah.length && Ah[0] == C.documentElement && (zh = !0) } catch (a) { } var wh = zh; var M = function (a) { yb("GTM", a) };
    var Bh = function (a) { return null == a ? "" : h(a) ? Oa(String(a)) : "e0" }, Dh = function (a) { return a.replace(Ch, "") }, Fh = function (a) { return Eh(a.replace(/\s/g, "")) }, Eh = function (a) { return Oa(a.replace(Gh, "").toLowerCase()) }, Ih = function (a) { a = a.replace(/[\s-()/.]/g, ""); "+" !== a.charAt(0) && (a = "+" + a); return Hh.test(a) ? a : "e0" }, Kh = function (a) { var b = a.toLowerCase().split("@"); if (2 == b.length) { var c = b[0]; /^(gmail|googlemail)\./.test(b[1]) && (c = c.replace(/\./g, "")); c = c + "@" + b[1]; if (Jh.test(c)) return c } return "e0" }, Nh = function (a,
      b) { window.Promise || b([]); Promise.all(a.map(function (c) { return c.value && -1 !== Lh.indexOf(c.name) ? Mh(c.value).then(function (d) { c.value = d }) : Promise.resolve() })).then(function () { b(a) }).catch(function () { b([]) }) }, Mh = function (a) {
        if ("" === a || "e0" === a) return Promise.resolve(a); if (z.crypto && z.crypto.subtle) {
          if (Oh.test(a)) return Promise.resolve(a); try {
            var b = Ph(a); return z.crypto.subtle.digest("SHA-256", b).then(function (c) {
              var d = Array.from(new Uint8Array(c)).map(function (e) { return String.fromCharCode(e) }).join("");
              return z.btoa(d).replace(/\+/g, "-").replace(/\//g, "_").replace(/=+$/, "")
            }).catch(function () { return "e2" })
          } catch (c) { return Promise.resolve("e2") }
        } else return Promise.resolve("e1")
      }, Ph = function (a) {
        var b; if (z.TextEncoder) b = (new TextEncoder("utf-8")).encode(a); else {
          for (var c = [], d = 0; d < a.length; d++) {
            var e = a.charCodeAt(d); 128 > e ? c.push(e) : 2048 > e ? c.push(192 | e >> 6, 128 | e & 63) : 55296 > e || 57344 <= e ? c.push(224 | e >> 12, 128 | e >> 6 & 63, 128 | e & 63) : (e = 65536 + ((e & 1023) << 10 | a.charCodeAt(++d) & 1023), c.push(240 | e >> 18, 128 | e >> 12 & 63, 128 |
              e >> 6 & 63, 128 | e & 63))
          } b = new Uint8Array(c)
        } return b
      }, Gh = /[0-9`~!@#$%^&*()_\-+=:;<>,.?|/\\[\]]/g, Jh = /^\S+@\S+\.\S+$/, Hh = /^\+\d{10,15}$/, Ch = /[.~]/g, Qh = /^[0-9A-Za-z_-]{43}$/, Oh = /^[0-9A-Fa-f]{64}$/, Rh = {}, Sh = (Rh.email = "em", Rh.phone_number = "pn", Rh.first_name = "fn", Rh.last_name = "ln", Rh.street = "sa", Rh.city = "ct", Rh.region = "rg", Rh.country = "co", Rh.postal_code = "pc", Rh.error_code = "ec", Rh), Th = {}, Uh = (Th.email = "sha256_email_address", Th.phone_number = "sha256_phone_number", Th.first_name = "sha256_first_name", Th.last_name =
        "sha256_last_name", Th.street = "sha256_street", Th), Vh = function (a, b) {
          function c(t, u, v, w) { var x = Bh(t); "" !== x && (Oh.test(x) ? m.push({ name: u, value: x, index: w }) : m.push({ name: u, value: v(x), index: w })) } function d(t, u) { var v = t; if (h(v) || Da(v)) { v = Da(t) ? t : [t]; for (var w = 0; w < v.length; ++w) { var x = Bh(v[w]), y = Oh.test(x); u && !y && M(89); !u && y && M(88) } } } function e(t, u) { var v = t[u]; d(v, !1); var w = Uh[u]; t.hasOwnProperty(w) && (t.hasOwnProperty(u) && M(90), v = t[w], d(v, !0)); return v } function f(t, u, v) {
            var w = e(t, u); w = Da(w) ? w : [w]; for (var x =
              0; x < w.length; ++x)c(w[x], u, v)
          } function g(t, u, v, w) { var x = e(t, u); c(x, u, v, w) } function k(t) { return function (u) { M(64); return t(u) } } var m = []; if ("https:" === z.location.protocol) {
            f(a, "email", Kh); f(a, "phone_number", Ih); f(a, "first_name", k(Fh)); f(a, "last_name", k(Fh)); var n = a.home_address || {}; f(n, "street", k(Eh)); f(n, "city", k(Eh)); f(n, "postal_code", k(Dh)); f(n, "region", k(Eh)); f(n, "country", k(Dh)); var p = a.address || {}; p = Da(p) ? p : [p]; for (var q = 0; q < p.length; q++) {
              var r = p[q]; g(r, "first_name", Fh, q); g(r, "last_name", Fh, q);
              g(r, "street", Eh, q); g(r, "city", Eh, q); g(r, "postal_code", Dh, q); g(r, "region", Eh, q); g(r, "country", Dh, q)
            } Nh(m, b)
          } else m.push({ name: "error_code", value: "e3", index: void 0 }), b(m)
        }, Wh = function (a, b) { Vh(a, function (c) { for (var d = ["tv.1"], e = 0, f = 0; f < c.length; ++f) { var g = c[f].name, k = c[f].value, m = c[f].index, n = Sh[g]; n && k && (-1 === Lh.indexOf(g) || /^e\d+$/.test(k) || Qh.test(k) || Oh.test(k)) && (void 0 !== m && (n += m), d.push(n + "." + k), e++) } 1 === c.length && "error_code" === c[0].name && (e = 0); b(encodeURIComponent(d.join("~")), e) }) }, Xh = function (a) {
          if (z.Promise) try {
            return new Promise(function (b) {
              Wh(a,
                function (c, d) { b({ uj: c, bm: d }) })
            })
          } catch (b) { }
        }, Lh = Object.freeze(["email", "phone_number", "first_name", "last_name", "street"]); var N = {
          g: {
            nb: "ad_personalization", I: "ad_storage", N: "ad_user_data", R: "analytics_storage", Ra: "region", ld: "consent_updated", md: "wait_for_update", Tj: "ads", Sf: "all", Uj: "android", Vj: "chrome", Wj: "playstore", Xj: "search", Yj: "shopping", Zj: "youtube", Vh: "app_remove", Wh: "app_store_refund", Xh: "app_store_subscription_cancel", Yh: "app_store_subscription_convert", Zh: "app_store_subscription_renew", Uf: "add_payment_info", Vf: "add_shipping_info", ac: "add_to_cart", bc: "remove_from_cart", Wf: "view_cart", Fb: "begin_checkout",
            fc: "select_item", Xa: "view_item_list", ob: "select_promotion", Ya: "view_promotion", ma: "purchase", hc: "refund", Ba: "view_item", Xf: "add_to_wishlist", bk: "exception", ai: "first_open", bi: "first_visit", na: "gtag.config", Ja: "gtag.get", di: "in_app_purchase", ic: "page_view", dk: "screen_view", ei: "session_start", ek: "timing_complete", fk: "track_social", od: "user_engagement", pb: "gclid", oa: "ads_data_redaction", da: "allow_ad_personalization_signals", Pe: "allow_custom_scripts", Qe: "allow_display_features", pd: "allow_enhanced_conversions",
            qb: "allow_google_signals", Ga: "allow_interest_groups", gk: "app_id", hk: "app_installer_id", ik: "app_name", jk: "app_version", jc: "auid", fi: "auto_detection_enabled", Gb: "aw_remarketing", Re: "aw_remarketing_only", rd: "discount", sd: "aw_feed_country", ud: "aw_feed_language", Z: "items", vd: "aw_merchant_id", Yf: "aw_basket_type", zc: "campaign_content", Ac: "campaign_id", Bc: "campaign_medium", Cc: "campaign_name", Dc: "campaign", Ec: "campaign_source", Fc: "campaign_term", Za: "client_id", gi: "content_group", hi: "content_type", Ka: "conversion_cookie_prefix",
            Gc: "conversion_id", Ca: "conversion_linker", Hb: "conversion_api", Sa: "cookie_domain", Ha: "cookie_expires", Ta: "cookie_flags", kc: "cookie_name", Hc: "cookie_path", La: "cookie_prefix", rb: "cookie_update", mc: "country", qa: "currency", wd: "customer_lifetime_value", Ic: "custom_map", ii: "gcldc", ji: "debug_mode", aa: "developer_id", ki: "disable_merchant_reported_purchases", Jc: "dc_custom_params", li: "dc_natural_search", Zf: "dynamic_event_settings", ag: "affiliation", xd: "checkout_option", Se: "checkout_step", cg: "coupon", Kc: "item_list_name",
            Te: "list_name", mi: "promotions", Lc: "shipping", Ue: "tax", yd: "engagement_time_msec", Mc: "enhanced_client_id", zd: "enhanced_conversions", dg: "enhanced_conversions_automatic_settings", Ad: "estimated_delivery_date", Ve: "euid_logged_in_state", Nc: "event_callback", kk: "event_category", sb: "event_developer_id_string", lk: "event_label", eg: "event", Bd: "event_settings", Cd: "event_timeout", mk: "description", nk: "fatal", ni: "experiments", We: "firebase_id", Dd: "first_party_collection", Ed: "_x_20", Ib: "_x_19", fg: "fledge", gg: "flight_error_code",
            hg: "flight_error_message", oi: "fl_activity_category", ri: "fl_activity_group", ig: "fl_advertiser_id", si: "fl_ar_dedupe", ui: "fl_random_number", vi: "tran", wi: "u", Fd: "gac_gclid", nc: "gac_wbraid", jg: "gac_wbraid_multiple_conversions", kg: "ga_restrict_domain", Xe: "ga_temp_client_id", Gd: "gdpr_applies", lg: "geo_granularity", tb: "value_callback", cb: "value_key", pk: "google_ono", eb: "google_signals", mg: "google_tld", Hd: "groups", ng: "gsa_experiment_id", og: "iframe_state", Id: "ignore_referrer", Ye: "internal_traffic_results", Jb: "is_legacy_converted",
            vb: "is_legacy_loaded", Jd: "is_passthrough", xi: "_lps", Ia: "language", Ze: "legacy_developer_id_string", Da: "linker", oc: "accept_incoming", wb: "decorate_forms", T: "domains", Kb: "url_position", pg: "method", qk: "name", Oc: "new_customer", qg: "non_interaction", yi: "optimize_id", zi: "page_hostname", Pc: "page_path", Ea: "page_referrer", xb: "page_title", rg: "passengers", sg: "phone_conversion_callback", Ai: "phone_conversion_country_code", ug: "phone_conversion_css_class", Bi: "phone_conversion_ids", vg: "phone_conversion_number", wg: "phone_conversion_options",
            Qc: "quantity", Rc: "redact_device_info", af: "redact_enhanced_user_id", Ci: "redact_ga_client_id", Di: "redact_user_id", Kd: "referral_exclusion_definition", Lb: "restricted_data_processing", Ei: "retoken", rk: "sample_rate", bf: "screen_name", yb: "screen_resolution", Fi: "search_term", Ma: "send_page_view", Mb: "send_to", Ld: "server_container_url", Sc: "session_duration", Md: "session_engaged", cf: "session_engaged_time", zb: "session_id", Nd: "session_number", Tc: "delivery_postal_code", xg: "temporary_client_id", df: "topmost_url", Gi: "tracking_id",
            ef: "traffic_type", sa: "transaction_id", Nb: "transport_url", yg: "trip_type", Ob: "update", Ab: "url_passthrough", Pd: "_user_agent_architecture", Qd: "_user_agent_bitness", Rd: "_user_agent_full_version_list", Sd: "_user_agent_mobile", Td: "_user_agent_model", Ud: "_user_agent_platform", Vd: "_user_agent_platform_version", Wd: "_user_agent_wow64", wa: "user_data", zg: "user_data_auto_latency", Ag: "user_data_auto_meta", Bg: "user_data_auto_multi", Cg: "user_data_auto_selectors", Dg: "user_data_auto_status", ff: "user_data_mode", Xd: "user_data_settings",
            za: "user_id", Na: "user_properties", Hi: "_user_region", Eg: "us_privacy_string", fa: "value", qc: "wbraid", Fg: "wbraid_multiple_conversions", Ni: "_host_name", Oi: "_in_page_command", Pi: "_is_passthrough_cid", be: "non_personalized_ads", fe: "_sst_parameters", ab: "conversion_label", ra: "page_location", ub: "global_developer_id_string", Od: "tc_privacy_string"
          }
        }, Yh = {}, Zh = Object.freeze((Yh[N.g.da] = 1, Yh[N.g.Qe] = 1, Yh[N.g.pd] = 1, Yh[N.g.qb] = 1, Yh[N.g.Z] = 1, Yh[N.g.Sa] = 1, Yh[N.g.Ha] = 1, Yh[N.g.Ta] = 1, Yh[N.g.kc] = 1, Yh[N.g.Hc] = 1, Yh[N.g.La] =
          1, Yh[N.g.rb] = 1, Yh[N.g.Ic] = 1, Yh[N.g.aa] = 1, Yh[N.g.Zf] = 1, Yh[N.g.Nc] = 1, Yh[N.g.Bd] = 1, Yh[N.g.Cd] = 1, Yh[N.g.Dd] = 1, Yh[N.g.kg] = 1, Yh[N.g.eb] = 1, Yh[N.g.mg] = 1, Yh[N.g.Hd] = 1, Yh[N.g.Ye] = 1, Yh[N.g.Jb] = 1, Yh[N.g.vb] = 1, Yh[N.g.Da] = 1, Yh[N.g.af] = 1, Yh[N.g.Kd] = 1, Yh[N.g.Lb] = 1, Yh[N.g.Ma] = 1, Yh[N.g.Mb] = 1, Yh[N.g.Ld] = 1, Yh[N.g.Sc] = 1, Yh[N.g.cf] = 1, Yh[N.g.Tc] = 1, Yh[N.g.Nb] = 1, Yh[N.g.Ob] = 1, Yh[N.g.Xd] = 1, Yh[N.g.Na] = 1, Yh[N.g.fe] = 1, Yh)); Object.freeze([N.g.ra, N.g.Ea, N.g.xb, N.g.Ia, N.g.bf, N.g.za, N.g.We, N.g.gi]);
    var $h = {}, ai = Object.freeze(($h[N.g.Vh] = 1, $h[N.g.Wh] = 1, $h[N.g.Xh] = 1, $h[N.g.Yh] = 1, $h[N.g.Zh] = 1, $h[N.g.ai] = 1, $h[N.g.bi] = 1, $h[N.g.di] = 1, $h[N.g.ei] = 1, $h[N.g.od] = 1, $h)), bi = {}, ci = Object.freeze((bi[N.g.Uf] = 1, bi[N.g.Vf] = 1, bi[N.g.ac] = 1, bi[N.g.bc] = 1, bi[N.g.Wf] = 1, bi[N.g.Fb] = 1, bi[N.g.fc] = 1, bi[N.g.Xa] = 1, bi[N.g.ob] = 1, bi[N.g.Ya] = 1, bi[N.g.ma] = 1, bi[N.g.hc] = 1, bi[N.g.Ba] = 1, bi[N.g.Xf] = 1, bi)), di = Object.freeze([N.g.da, N.g.qb, N.g.rb, N.g.Ob]), ei = Object.freeze([].concat(di)), fi = Object.freeze([N.g.Ha, N.g.Cd, N.g.Sc, N.g.cf, N.g.yd]),
      gi = Object.freeze([].concat(fi)), hi = {}, ii = (hi[N.g.I] = "1", hi[N.g.R] = "2", hi[N.g.N] = "3", hi[N.g.nb] = "4", hi), ji = {}, ki = Object.freeze((ji[N.g.da] = 1, ji[N.g.pd] = 1, ji[N.g.Ga] = 1, ji[N.g.Gb] = 1, ji[N.g.Re] = 1, ji[N.g.rd] = 1, ji[N.g.sd] = 1, ji[N.g.ud] = 1, ji[N.g.Z] = 1, ji[N.g.vd] = 1, ji[N.g.Ka] = 1, ji[N.g.Ca] = 1, ji[N.g.Sa] = 1, ji[N.g.Ha] = 1, ji[N.g.Ta] = 1, ji[N.g.La] = 1, ji[N.g.qa] = 1, ji[N.g.wd] = 1, ji[N.g.aa] = 1, ji[N.g.ki] = 1, ji[N.g.zd] = 1, ji[N.g.Ad] = 1, ji[N.g.We] = 1, ji[N.g.Dd] = 1, ji[N.g.Jb] = 1, ji[N.g.vb] = 1, ji[N.g.Ia] = 1, ji[N.g.Oc] = 1, ji[N.g.ra] =
        1, ji[N.g.Ea] = 1, ji[N.g.sg] = 1, ji[N.g.ug] = 1, ji[N.g.vg] = 1, ji[N.g.wg] = 1, ji[N.g.Lb] = 1, ji[N.g.Ma] = 1, ji[N.g.Mb] = 1, ji[N.g.Ld] = 1, ji[N.g.Tc] = 1, ji[N.g.sa] = 1, ji[N.g.Nb] = 1, ji[N.g.Ob] = 1, ji[N.g.Ab] = 1, ji[N.g.wa] = 1, ji[N.g.za] = 1, ji[N.g.fa] = 1, ji)), li = {}, mi = Object.freeze((li[N.g.Xj] = "s", li[N.g.Zj] = "y", li[N.g.Uj] = "n", li[N.g.Vj] = "c", li[N.g.Wj] = "p", li[N.g.Yj] = "h", li[N.g.Tj] = "a", li)); Object.freeze(N.g); var ni = {}, oi = z.google_tag_manager = z.google_tag_manager || {}, pi = Math.random(); ni.Lg = "38u0"; ni.ee = Number("0") || 0; ni.ia = "dataLayer"; ni.Rj = "ChEI8PTApwYQ8sjgkqfb7u2KARIlAAN4hmsLXQwNbW+L3p6maq66nQ0Dh6SP46sRoFp5Ldxeqs6jMRoCnmM\x3d"; var qi = { __cl: 1, __ecl: 1, __ehl: 1, __evl: 1, __fal: 1, __fil: 1, __fsl: 1, __hl: 1, __jel: 1, __lcl: 1, __sdl: 1, __tl: 1, __ytl: 1 }, ri = { __paused: 1, __tg: 1 }, si; for (si in qi) qi.hasOwnProperty(si) && (ri[si] = 1); var ti = Ma(""), ui, vi = !1; vi = !0;
    ui = vi; var wi, xi = !1; wi = xi; var yi, zi = !1; yi = zi; var Ai, Bi = !1; Ai = Bi; ni.Oe = "www.googletagmanager.com";
    var Ci = "" + ni.Oe + (ui ? "/gtag/js" : "/gtm.js"), Di = null, Ei = null, Fi = {}, Gi = {}, Hi = {}, Ii = function () { var a = oi.sequence || 1; oi.sequence = a + 1; return a }; ni.Qj = "true"; var Pi = ""; ni.nf = Pi; var Qi = new Ha, Ri = {}, Si = {}, Vi = { name: ni.ia, set: function (a, b) { lb(Za(a, b), Ri); Ti() }, get: function (a) { return Ui(a, 2) }, reset: function () { Qi = new Ha; Ri = {}; Ti() } }, Ui = function (a, b) { return 2 != b ? Qi.get(a) : Wi(a) }, Wi = function (a, b) { var c = a.split("."); b = b || []; for (var d = Ri, e = 0; e < c.length; e++) { if (null === d) return !1; if (void 0 === d) break; d = d[c[e]]; if (-1 !== b.indexOf(d)) return } return d }, Xi = function (a, b) { Si.hasOwnProperty(a) || (Qi.set(a, b), lb(Za(a, b), Ri), Ti()) }, Yi = function () {
      for (var a = ["gtm.allowlist", "gtm.blocklist", "gtm.whitelist",
        "gtm.blacklist", "tagTypeBlacklist"], b = 0; b < a.length; b++) { var c = a[b], d = Ui(c, 1); if (Da(d) || kb(d)) d = lb(d); Si[c] = d }
    }, Ti = function (a) { l(Si, function (b, c) { Qi.set(b, c); lb(Za(b), Ri); lb(Za(b, c), Ri); a && delete Si[b] }) }, Zi = function (a, b) { var c, d = 1 !== (void 0 === b ? 2 : b) ? Wi(a) : Qi.get(a); "array" === ib(d) || "object" === ib(d) ? c = lb(d) : c = d; return c };
    var $i = function (a, b, c) {
      if (!c) return !1; var d = c.selector_type, e = String(c.value), f; if ("js_variable" === d) { e = e.replace(/\["?'?/g, ".").replace(/"?'?\]/g, ""); for (var g = e.split(","), k = 0; k < g.length; k++) { var m = g[k].trim(); if (m) { if (0 === m.indexOf("dataLayer.")) f = Ui(m.substring(10)); else { var n = m.split("."); f = z[n.shift()]; for (var p = 0; p < n.length; p++)f = f && f[n[p]] } if (void 0 !== f) break } } } else if ("css_selector" === d && wh) {
        var q = xh(e); if (q && 0 < q.length) {
          f = []; for (var r = 0; r < q.length && r < ("email" === b || "phone_number" === b ? 5 : 1); r++)f.push(Rc(q[r]) ||
            Oa(q[r].value)); f = 1 === f.length ? f[0] : f
        }
      } return f ? (a[b] = f, !0) : !1
    }, aj = function (a) {
      if (a) {
        var b = {}, c = !1; c = $i(b, "email", a.email) || c; c = $i(b, "phone_number", a.phone) || c; b.address = []; for (var d = a.name_and_address || [], e = 0; e < d.length; e++) { var f = {}; c = $i(f, "first_name", d[e].first_name) || c; c = $i(f, "last_name", d[e].last_name) || c; c = $i(f, "street", d[e].street) || c; c = $i(f, "city", d[e].city) || c; c = $i(f, "region", d[e].region) || c; c = $i(f, "country", d[e].country) || c; c = $i(f, "postal_code", d[e].postal_code) || c; b.address.push(f) } return c ?
          b : void 0
      }
    }, bj = function (a) { return kb(a) ? !!a.enable_code : !1 }; var P = []; P[5] = !0; P[6] = !0; P[11] = !0;
    P[7] = !0; P[8] = !0; P[21] = !0; P[9] = !0;
    P[10] = !0; P[13] = !0; P[14] = !0; P[15] = !0;
    P[17] = !0;
    P[23] = !0; P[24] = !0; P[25] = !0;
    P[27] = !0; P[19] = !0; P[28] = !0; P[31] = !0; P[32] = !0;
    P[33] = !0; P[34] = !0; P[36] = !0; P[37] = !0;
    P[39] = !0; P[41] = !0;
    P[42] = !0; P[43] = !0; P[44] = !0;


    P[55] = !0; P[56] = !0;
    P[57] = !0; P[58] = !0; P[60] = !0;
    P[61] = !0;
    P[69] = !0; P[72] = !0;
    a: { for (var fj = void 0, gj = void 0, hj = 0; fj === gj;)if (fj = Math.floor(2 * Math.random()), gj = Math.floor(2 * Math.random()), hj++, 20 < hj) break a; fj ? P[75] = !0 : P[76] = !0 } Q(76) ? mf[6] = !0 : Q(75) && (mf[5] = !0); P[77] = !0;
    P[80] = !0;
    P[81] = !0; P[83] = !0;
    P[86] = !0; P[90] = !0;
    P[93] = !0;
    P[99] = !0;
    function Q(a) { return !!P[a] } var ij = function (a) { yb("HEALTH", a) }; var jj; try { jj = JSON.parse(wb("eyIwIjoiQVUiLCIxIjoiQVUtU0EiLCIyIjpmYWxzZSwiMyI6Imdvb2dsZS5jb20uYXUiLCI0IjoiIiwiNSI6dHJ1ZSwiNiI6ZmFsc2UsIjciOiJhZF9zdG9yYWdlfGFuYWx5dGljc19zdG9yYWdlfGFkX3VzZXJfZGF0YXxhZF9wZXJzb25hbGl6YXRpb24ifQ")) } catch (a) { M(123), ij(2), jj = {} }
    var kj = function () { return jj["0"] || "" }, lj = function () { return jj["1"] || "" }, mj = function () { var a = !1; a = !!jj["2"]; return a }, nj = function () { var a = ""; a = jj["4"] || ""; return a }, oj = function () { var a = !1; a = !!jj["5"]; return a }, pj = function () {
      var a = ""; a = jj["3"] || "";
      return a
    }; var qj = new function (a, b) { this.h = a; this.defaultValue = void 0 === b ? !1 : b }(1933); var rj = function (a) { rj[" "](a); return a }; rj[" "] = function () { }; var tj = function () { var a = sj, b = "gh"; if (a.gh && a.hasOwnProperty(b)) return a.gh; var c = new a; return a.gh = c }; var sj = function () { var a = {}; this.h = function () { var b = qj.h, c = qj.defaultValue; return null != a[b] ? a[b] : c }; this.B = function () { a[qj.h] = !0 } }; var uj = !1, vj = !1, wj = {}, xj = {}, yj = !1, zj = { ad_storage: !1, ad_user_data: !1, ad_personalization: !1 }; function Aj() { var a = Ec("google_tag_data", {}); return a.ics = a.ics || new Bj } var Bj = function () { this.entries = {}; this.cps = {}; this.wasSetLate = this.accessedAny = this.accessedDefault = this.usedSetCps = this.usedImplicit = this.usedUpdate = this.usedDefault = this.usedDeclare = this.active = !1; this.h = [] };
    Bj.prototype.default = function (a, b, c, d, e, f) { this.usedDefault || !this.accessedDefault && !this.accessedAny || (this.wasSetLate = !0); this.usedDefault = this.active = !0; yb("TAGGING", 19); void 0 == b ? yb("TAGGING", 18) : Cj(this, a, "granted" === b, c, d, e, f) }; Bj.prototype.waitForUpdate = function (a, b) { for (var c = 0; c < a.length; c++)Cj(this, a[c], void 0, void 0, "", "", b) };
    var Cj = function (a, b, c, d, e, f, g) {
      var k = a.entries, m = k[b] || {}, n = m.region, p = d && h(d) ? d.toUpperCase() : void 0; e = e.toUpperCase(); f = f.toUpperCase(); if (Dj(p, n, e, f)) {
        var q = !!(g && 0 < g && void 0 === m.update), r = { region: p, declare_region: m.declare_region, implicit: m.implicit, default: void 0 !== c ? c : m.default, declare: m.declare, update: m.update, quiet: q }; if ("" !== e || !1 !== m.default) k[b] = r; q && z.setTimeout(function () {
          if (k[b] === r && r.quiet) {
            r.quiet = !1; var t = [b]; if (nf(4)) for (var u in wj) wj.hasOwnProperty(u) && wj[u] === b && t.push(u); for (var v =
              0; v < t.length; v++)Ej(a, t[v]); a.notifyListeners(); yb("TAGGING", 2)
          }
        }, g)
      }
    }; aa = Bj.prototype;
    aa.update = function (a, b) { this.usedDefault || this.usedUpdate || !this.accessedAny || (this.wasSetLate = !0); this.usedUpdate = this.active = !0; if (void 0 != b) { var c = this.getConsentState(a), d = this.entries, e = d[a] = d[a] || {}; e.update = "granted" === b; var f = this.getConsentState(a), g = [a]; if (nf(4)) for (var k in wj) wj.hasOwnProperty(k) && wj[k] === a && g.push(k); if (e.quiet) { e.quiet = !1; for (var m = 0; m < g.length; m++)Ej(this, g[m]) } else if (f !== c) for (var n = 0; n < g.length; n++)Ej(this, g[n]) } };
    aa.declare = function (a, b, c, d, e) { this.usedDeclare = this.active = !0; var f = this.entries, g = f[a] || {}, k = g.declare_region, m = c && h(c) ? c.toUpperCase() : void 0; d = d.toUpperCase(); e = e.toUpperCase(); if (Dj(m, k, d, e)) { var n = { region: g.region, declare_region: m, declare: "granted" === b, implicit: g.implicit, default: g.default, update: g.update, quiet: g.quiet }; if ("" !== d || !1 !== g.declare) f[a] = n } }; aa.implicit = function (a, b) { this.usedImplicit = !0; var c = this.entries, d = c[a] = c[a] || {}; !1 !== d.implicit && (d.implicit = "granted" === b) };
    aa.getConsentState = function (a) { var b = this.entries, c = b[a] || {}, d = c.update; if (void 0 !== d) return d ? 1 : 2; d = c.default; if (void 0 !== d) return d ? 1 : 2; if (nf(4) && wj.hasOwnProperty(a)) { var e = b[wj[a]] || {}; d = e.update; if (void 0 !== d) return d ? 1 : 2; d = e.default; if (void 0 !== d) return d ? 1 : 2 } d = c.declare; if (void 0 !== d) return d ? 1 : 2; if (nf(3)) { d = c.implicit; if (void 0 !== d) return d ? 3 : 4; if (zj.hasOwnProperty(a)) return zj[a] ? 3 : 4 } return 0 }; aa.setCps = function (a, b, c, d, e) { Fj(this.cps, a, b, c, d, e) && (this.usedSetCps = !0) };
    aa.addListener = function (a, b) { this.h.push({ consentTypes: a, Yk: b }) }; var Ej = function (a, b) { for (var c = 0; c < a.h.length; ++c) { var d = a.h[c]; Da(d.consentTypes) && -1 !== d.consentTypes.indexOf(b) && (d.wj = !0) } }; Bj.prototype.notifyListeners = function (a, b) { for (var c = 0; c < this.h.length; ++c) { var d = this.h[c]; if (d.wj) { d.wj = !1; try { d.Yk({ consentEventId: a, consentPriorityId: b }) } catch (e) { } } } }; function Dj(a, b, c, d) { return "" === c || a === d ? !0 : a === c ? b !== d : !a && !b }
    function Fj(a, b, c, d, e, f) { var g = a[b] || {}, k = g.region, m = d && h(d) ? d.toUpperCase() : void 0; e = e.toUpperCase(); f = f.toUpperCase(); if (Dj(m, k, e, f)) { var n = { enabled: "granted" === c, region: m }; if ("" !== e || !1 !== g.enabled) return a[b] = n, !0 } return !1 }
    var Gj = function (a) { var b = Aj(); b.accessedAny = !0; switch (b.getConsentState(a)) { case 1: case 3: return !0; case 2: case 4: return !1; default: return !0 } }, Hj = function (a) { var b = Aj(); b.accessedDefault = !0; switch ((b.entries[a] || {}).default) { case !0: return 3; case !1: return 2; default: return 1 } }, Ij = function (a) { var b = Aj(); b.accessedAny = !0; return !(b.entries[a] || {}).quiet }, Jj = function () { if (!tj().h()) return !1; var a = Aj(); a.accessedAny = !0; return a.active }, Kj = function () { var a = Aj(); a.accessedDefault = !0; return a.usedDefault },
      Lj = function (a, b) { Aj().addListener(a, b) }, Mj = function (a, b) { Aj().notifyListeners(a, b) }, Nj = function (a, b) { function c() { for (var e = 0; e < b.length; e++)if (!Ij(b[e])) return !0; return !1 } if (c()) { var d = !1; Lj(b, function (e) { d || c() || (d = !0, a(e)) }) } else a({}) }, Oj = function (a, b) { function c() { for (var f = [], g = 0; g < d.length; g++) { var k = d[g]; Gj(k) && !e[k] && (f.push(k), e[k] = !0) } return f } var d = h(b) ? [b] : b, e = {}; c().length !== d.length && Lj(d, function (f) { var g = c(); 0 < g.length && (f.consentTypes = g, a(f)) }) }; function Pj() { } function Qj() { }; var Rj = [N.g.I, N.g.R, N.g.N, N.g.nb], Sj = function (a) { for (var b = a[N.g.Ra], c = Array.isArray(b) ? b : [b], d = { xe: 0 }; d.xe < c.length; d = { xe: d.xe }, ++d.xe)l(a, function (e) { return function (f, g) { if (f !== N.g.Ra) { var k = c[e.xe], m = kj(), n = lj(); vj = !0; uj && yb("TAGGING", 20); Aj().declare(f, g, k, m, n) } } }(d)) }, Tj = function (a) {
        var b = a[N.g.Ra]; b && M(40); var c = a[N.g.md]; c && M(41); for (var d = Da(b) ? b : [b], e = { ye: 0 }; e.ye < d.length; e = { ye: e.ye }, ++e.ye)l(a, function (f) {
          return function (g, k) {
            if (g !== N.g.Ra && g !== N.g.md) {
              var m = d[f.ye], n = Number(c), p = kj(),
              q = lj(); uj = !0; vj && yb("TAGGING", 20); Aj().default(g, k, m, p, q, n)
            }
          }
        }(e))
      }, Uj = function (a, b) { l(a, function (c, d) { uj = !0; vj && yb("TAGGING", 20); Aj().update(c, d) }); Mj(b.eventId, b.priorityId) }, Vj = function (a) { for (var b = a[N.g.Ra], c = Array.isArray(b) ? b : [b], d = { ze: 0 }; d.ze < c.length; d = { ze: d.ze }, ++d.ze)l(a, function (e) { return function (f, g) { if (f !== N.g.Ra) { var k = c[e.ze], m = kj(), n = lj(); Aj().setCps(f, g, k, m, n) } } }(d)) }, Wj = function (a) {
        for (var b = a[N.g.Ra], c = Array.isArray(b) ? b : [b], d = { Zc: 0 }; d.Zc < c.length; d = { Zc: d.Zc }, ++d.Zc)a.hasOwnProperty(N.g.Sf) &&
          l(mi, function (e) { return function (f) { Fj(xj, f, a[N.g.Sf], c[e.Zc], kj(), lj()) && (yj = !0) } }(d)), l(a, function (e) { return function (f, g) { f !== N.g.Ra && f !== N.g.Sf && Fj(xj, f, g, c[e.Zc], kj(), lj()) && (yj = !0) } }(d))
      }, S = function (a) { Array.isArray(a) || (a = [a]); return a.every(function (b) { return Gj(b) }) }, Xj = function (a, b) { Lj(a, b) }, Yj = function (a, b) { Oj(a, b) }, Zj = function (a, b) { Nj(a, b) }, ak = function (a) { Aj().waitForUpdate(a, 500) }; var bk = function (a) { var b = String(a[Qe.xa] || "").replace(/_/g, ""); 0 === b.indexOf("cvt") && (b = "cvt"); return b }, ck = 0 <= z.location.search.indexOf("?gtm_latency=") || 0 <= z.location.search.indexOf("&gtm_latency="); var ek = function (a, b) { var c = dk(); c.pending || (c.pending = []); Fa(c.pending, function (d) { return d.target.ctid === a.ctid && d.target.isDestination === a.isDestination }) || c.pending.push({ target: a, onLoad: b }) }, fk = function () { this.container = {}; this.destination = {}; this.canonical = {}; this.pending = []; this.siloed = [] }, dk = function () { var a = Ec("google_tag_data", {}), b = a.tidr; b || (b = new fk, a.tidr = b); return b }; var gk = {}, hk = !1, Uf = { ctid: "G-T8GWK081T9", Og: "88536031", Ff: "G-T8GWK081T9", vj: "G-T8GWK081T9" }; gk.hf = Ma("");
    var jk = function () { var a = Uf.Ff ? Uf.Ff.split("|") : [Uf.ctid]; return hk ? a.map(ik) : a }, lk = function () { var a = kk(); return hk ? a.map(ik) : a }, nk = function () { return mk(Uf.ctid) }, ok = function () { return mk(Uf.Og || "_" + Uf.ctid) }, kk = function () { return Uf.vj ? Uf.vj.split("|") : [] }, pk = function (a) { var b = dk(); return a.isDestination ? b.destination[a.ctid] : b.container[a.ctid] }, mk = function (a) { return hk ? ik(a) : a }, ik = function (a) { return "siloed_" + a }, qk = function (a) { a = String(a); return hk && 0 === a.indexOf("siloed_") ? a.substring(7) : a }, rk = function () {
      var a =
        !1; if (a) { var b = dk(); if (b.siloed) { for (var c = [], d = Uf.Ff ? Uf.Ff.split("|") : [Uf.ctid], e = kk(), f = {}, g = 0; g < b.siloed.length; f = { me: f.me }, g++)f.me = b.siloed[g], !hk && Fa(f.me.isDestination ? e : d, function (k) { return function (m) { return m === k.me.ctid } }(f)) ? hk = !0 : c.push(f.me); b.siloed = c } }
    };
    function sk() { var a = dk(); if (a.pending) { for (var b, c = [], d = !1, e = jk(), f = lk(), g = {}, k = 0; k < a.pending.length; g = { fd: g.fd }, k++)g.fd = a.pending[k], Fa(g.fd.target.isDestination ? f : e, function (m) { return function (n) { return n === m.fd.target.ctid } }(g)) ? d || (b = g.fd.onLoad, d = !0) : c.push(g.fd); a.pending = c; if (b) try { b(ok()) } catch (m) { } } }
    var tk = function () { for (var a = dk(), b = jk(), c = 0; c < b.length; c++) { var d = a.container[b[c]]; d ? (d.state = 2, d.containers = jk(), d.destinations = lk()) : a.container[b[c]] = { state: 2, containers: jk(), destinations: lk() } } for (var e = lk(), f = 0; f < e.length; f++) { var g = a.destination[e[f]]; g && 0 === g.state && M(93); g ? (g.state = 2, g.containers = jk(), g.destinations = lk()) : a.destination[e[f]] = { state: 2, containers: jk(), destinations: lk() } } a.canonical[ok()] = {}; sk() }, uk = function (a) { return !!dk().container[a] }, vk = function () { return { ctid: nk(), isDestination: gk.hf } };
    function wk(a) { var b = dk(); (b.siloed = b.siloed || []).push(a) } var xk = function () { var a = dk().container, b; for (b in a) if (a.hasOwnProperty(b) && 1 === a[b].state) return !0; return !1 }, yk = function () { var a = {}; l(dk().destination, function (b, c) { 0 === c.state && (a[b] = c) }); return a }, zk = function (a) { return !!(a && a.parent && a.context && 1 === a.context.source && 0 !== a.parent.ctid.indexOf("GTM-")) }; var Ak = { sampleRate: "0.005000", Mj: "", Lj: Number("5"), gn: Number("") }, Bk = []; function Ck(a) { Bk.push(a) } var Dk = !1, Ek; if (!(Ek = ck)) { var Fk = Math.random(), Gk = Ak.sampleRate; Ek = Fk < Number(Gk) }
    var Hk = Ek, Ik = "https://www.googletagmanager.com/a?id=" + Uf.ctid, Jk = void 0, Kk = {}, Lk = void 0, Mk = new function () { var a = 5; 0 < Ak.Lj && (a = Ak.Lj); this.h = 0; this.C = []; this.B = a }, Nk = 1E3; function Ok(a, b) { var c = Jk; if (void 0 === c) if (b) c = Ii(); else return ""; for (var d = [Ik], e = 0; e < Bk.length; e++) { var f = Bk[e]({ eventId: c, Yb: !!a, Cj: function () { Dk = !0 } }); "&" === f[0] && d.push(f) } d.push("&z=0"); return d.join("") }
    function Pk() { Lk && (z.clearTimeout(Lk), Lk = void 0); if (void 0 !== Jk && Qk) { var a; (a = Kk[Jk]) || (a = Mk.h < Mk.B ? !1 : 1E3 > Ra() - Mk.C[Mk.h % Mk.B]); if (a || 0 >= Nk--) M(1), Kk[Jk] = !0; else { var b = Mk.h++ % Mk.B; Mk.C[b] = Ra(); var c = Ok(!0); Mc(c); if (Dk) { var d = c.replace("/a?", "/td?"); Mc(d) } Qk = Dk = !1 } } } var Qk = !1; function Rk(a) { Kk[a] || (a !== Jk && (Pk(), Jk = a), Qk = !0, Lk || (Lk = z.setTimeout(Pk, 500)), 2022 <= Ok().length && Pk()) } var Sk = Ga(); function Tk() { Sk = Ga() } function Uk() { return ["&v=3&t=t", "&pid=" + Sk].join("") }; var Vk = function (a, b, c, d, e, f, g, k, m, n, p, q) { this.eventId = a; this.priorityId = b; this.h = c; this.K = d; this.C = e; this.F = f; this.W = g; this.B = k; this.eventMetadata = m; this.onSuccess = n; this.onFailure = p; this.isGtmEvent = q }, U = function (a, b, c) { if (void 0 !== a.h[b]) return a.h[b]; if (void 0 !== a.K[b]) return a.K[b]; if (void 0 !== a.C[b]) return a.C[b]; Hk && Wk(a, a.F[b], a.W[b]) && (M(71), M(79)); return void 0 !== a.F[b] ? a.F[b] : void 0 !== a.B[b] ? a.B[b] : c }, Xk = function (a) {
      function b(g) { for (var k = Object.keys(g), m = 0; m < k.length; ++m)c[k[m]] = 1 } var c =
        {}; b(a.h); b(a.K); b(a.C); b(a.F); if (Hk) for (var d = Object.keys(a.W), e = 0; e < d.length; e++) { var f = d[e]; if ("event" !== f && "gtm" !== f && "tagTypeBlacklist" !== f && !c.hasOwnProperty(f)) { M(71); M(80); break } } return Object.keys(c)
    }, Yk = function (a, b, c) {
      function d(m) { kb(m) && l(m, function (n, p) { f = !0; e[n] = p }) } var e = {}, f = !1; c && 1 !== c || (d(a.B[b]), d(a.F[b]), d(a.C[b]), d(a.K[b])); c && 2 !== c || d(a.h[b]); if (Hk) {
        var g = f, k = e; e = {}; f = !1; c && 1 !== c || (d(a.B[b]), d(a.W[b]), d(a.C[b]), d(a.K[b])); c && 2 !== c || d(a.h[b]); if (f !== g || Wk(a, e, k)) M(71), M(81);
        f = g; e = k
      } return f ? e : void 0
    }, Zk = function (a) { var b = [N.g.Dc, N.g.zc, N.g.Ac, N.g.Bc, N.g.Cc, N.g.Ec, N.g.Fc], c = {}, d = !1, e = function (k) { for (var m = 0; m < b.length; m++)void 0 !== k[b[m]] && (c[b[m]] = k[b[m]], d = !0); return d }; if (e(a.h) || e(a.K) || e(a.C)) return c; e(a.F); if (Hk) { var f = c, g = d; c = {}; d = !1; e(a.W); Wk(a, c, f) && (M(71), M(82)); c = f; d = g } if (d) return c; e(a.B); return c }, Wk = function (a, b, c) {
      if (!Hk) return !1; try {
        if (b === c) return !1; var d = ib(b); if (d !== ib(c) || !(kb(b) && kb(c) || "array" === d)) return !0; if ("array" === d) {
          if (b.length !== c.length) return !0;
          for (var e = 0; e < b.length; e++)if (Wk(a, b[e], c[e])) return !0
        } else { for (var f in c) if (!b.hasOwnProperty(f)) return !0; for (var g in b) if (!c.hasOwnProperty(g) || Wk(a, b[g], c[g])) return !0 }
      } catch (k) { M(72) } return !1
    }, $k = function (a, b) { this.Ig = a; this.uk = b; this.F = {}; this.ae = {}; this.h = {}; this.K = {}; this.B = {}; this.Zd = {}; this.C = {}; this.nd = function () { }; this.Zb = function () { }; this.W = !1 }, al = function (a, b) { a.F = b; return a }, bl = function (a, b) { a.ae = b; return a }, cl = function (a, b) { a.h = b; return a }, dl = function (a, b) { a.K = b; return a }, el = function (a,
      b) { a.B = b; return a }, fl = function (a, b) { a.Zd = b; return a }, gl = function (a, b) { a.C = b || {}; return a }, hl = function (a, b) { a.nd = b; return a }, il = function (a, b) { a.Zb = b; return a }, jl = function (a, b) { a.W = b; return a }, kl = function (a) { return new Vk(a.Ig, a.uk, a.F, a.ae, a.h, a.K, a.B, a.Zd, a.C, a.nd, a.Zb, a.W) }; function ll(a, b) { if ("" === a) return b; var c = Number(a); return isNaN(c) ? b : c }; var ml = function (a, b) { var c = function () { }; c.prototype = a.prototype; var d = new c; a.apply(d, Array.prototype.slice.call(arguments, 1)); return d }, nl = function (a) { var b = a; return function () { if (b) { var c = b; b = null; c() } } }; var ol = function (a, b, c) { a.addEventListener && a.addEventListener(b, c, !1) }; function pl() { return Nb ? !!Ub && !!Ub.platform : !1 } function ql() { return Xb("iPhone") && !Xb("iPod") && !Xb("iPad") } function rl() { ql() || Xb("iPad") || Xb("iPod") }; Zb(); Yb() || Xb("Trident") || Xb("MSIE"); Xb("Edge"); !Xb("Gecko") || -1 != Tb().toLowerCase().indexOf("webkit") && !Xb("Edge") || Xb("Trident") || Xb("MSIE") || Xb("Edge"); -1 != Tb().toLowerCase().indexOf("webkit") && !Xb("Edge") && Xb("Mobile"); pl() || Xb("Macintosh"); pl() || Xb("Windows"); (pl() ? "Linux" === Ub.platform : Xb("Linux")) || pl() || Xb("CrOS"); var sl = ra.navigator || null; sl && (sl.appVersion || "").indexOf("X11"); pl() || Xb("Android"); ql(); Xb("iPad"); Xb("iPod"); rl(); Tb().toLowerCase().indexOf("kaios"); var tl = function (a, b, c, d) { for (var e = b, f = c.length; 0 <= (e = a.indexOf(c, e)) && e < d;) { var g = a.charCodeAt(e - 1); if (38 == g || 63 == g) { var k = a.charCodeAt(e + f); if (!k || 61 == k || 38 == k || 35 == k) return e } e += f + 1 } return -1 }, ul = /#|$/, vl = function (a, b) { var c = a.search(ul), d = tl(a, 0, b, c); if (0 > d) return null; var e = a.indexOf("&", d); if (0 > e || e > c) e = c; d += b.length + 1; return decodeURIComponent(a.slice(d, -1 !== e ? e : 0).replace(/\+/g, " ")) }, wl = /[?&]($|#)/, xl = function (a, b, c) {
        for (var d, e = a.search(ul), f = 0, g, k = []; 0 <= (g = tl(a, f, b, e));)k.push(a.substring(f,
          g)), f = Math.min(a.indexOf("&", g) + 1 || e, e); k.push(a.slice(f)); d = k.join("").replace(wl, "$1"); var m, n = null != c ? "=" + encodeURIComponent(String(c)) : ""; var p = b + n; if (p) { var q, r = d.indexOf("#"); 0 > r && (r = d.length); var t = d.indexOf("?"), u; 0 > t || t > r ? (t = r, u = "") : u = d.substring(t + 1, r); q = [d.slice(0, t), u, d.slice(r)]; var v = q[1]; q[1] = p ? v ? v + "&" + p : p : v; m = q[0] + (q[1] ? "?" + q[1] : "") + q[2] } else m = d; return m
      }; var yl = function (a) { try { var b; if (b = !!a && null != a.location.href) a: { try { rj(a.foo); b = !0; break a } catch (c) { } b = !1 } return b } catch (c) { return !1 } }, zl = function (a, b) { if (a) for (var c in a) Object.prototype.hasOwnProperty.call(a, c) && b(a[c], c, a) }; function Al(a) { if (!a || !C.head) return null; var b = Bl("META"); C.head.appendChild(b); b.httpEquiv = "origin-trial"; b.content = a; return b }
    var Cl = function () { if (z.top == z) return 0; var a = z.location.ancestorOrigins; return a ? a[a.length - 1] == z.location.origin ? 1 : 2 : yl(z.top) ? 1 : 2 }, Bl = function (a, b) { b = void 0 === b ? document : b; return b.createElement(String(a).toLowerCase()) }; function Dl(a, b, c, d) { d = void 0 === d ? !1 : d; a.google_image_requests || (a.google_image_requests = []); var e = Bl("IMG", a.document); if (c) { var f = function () { if (c) { var g = a.google_image_requests, k = Bb(g, e); 0 <= k && Array.prototype.splice.call(g, k, 1) } e.removeEventListener && e.removeEventListener("load", f, !1); e.removeEventListener && e.removeEventListener("error", f, !1) }; ol(e, "load", f); ol(e, "error", f) } d && (e.attributionSrc = ""); e.src = b; a.google_image_requests.push(e) }
    var Fl = function (a) { var b; b = void 0 === b ? !1 : b; var c = "https://pagead2.googlesyndication.com/pagead/gen_204?id=tcfe"; zl(a, function (d, e) { if (d || 0 === d) c += "&" + e + "=" + encodeURIComponent("" + d) }); El(c, b) }, El = function (a, b) {
      var c = window, d; b = void 0 === b ? !1 : b; d = void 0 === d ? !1 : d; if (c.fetch) {
        var e = { keepalive: !0, credentials: "include", redirect: "follow", method: "get", mode: "no-cors" }; d && (e.mode = "cors", "setAttributionReporting" in XMLHttpRequest.prototype ? e.attributionReporting = { eventSourceEligible: "true", triggerEligible: "false" } :
          e.headers = { "Attribution-Reporting-Eligible": "event-source" }); c.fetch(a, e)
      } else Dl(c, a, void 0 === b ? !1 : b, void 0 === d ? !1 : d)
    }; var Gl = function () { }; var Hl = function (a) { void 0 !== a.addtlConsent && "string" !== typeof a.addtlConsent && (a.addtlConsent = void 0); void 0 !== a.gdprApplies && "boolean" !== typeof a.gdprApplies && (a.gdprApplies = void 0); return void 0 !== a.tcString && "string" !== typeof a.tcString || void 0 !== a.listenerId && "number" !== typeof a.listenerId ? 2 : a.cmpStatus && "error" !== a.cmpStatus ? 0 : 3 }, Il = function (a, b) { b = void 0 === b ? {} : b; this.B = a; this.h = null; this.K = {}; this.Zb = 0; var c; this.W = null != (c = b.Dm) ? c : 500; var d; this.F = null != (d = b.Zm) ? d : !1; this.C = null }; qa(Il, Gl);
    var Kl = function (a) { return "function" === typeof a.B.__tcfapi || null != Jl(a) };
    Il.prototype.addEventListener = function (a) {
      var b = this, c = { internalBlockOnErrors: this.F }, d = nl(function () { return a(c) }), e = 0; -1 !== this.W && (e = setTimeout(function () { c.tcString = "tcunavailable"; c.internalErrorState = 1; d() }, this.W)); var f = function (g, k) { clearTimeout(e); g ? (c = g, c.internalErrorState = Hl(c), c.internalBlockOnErrors = b.F, k && 0 === c.internalErrorState || (c.tcString = "tcunavailable", k || (c.internalErrorState = 3))) : (c.tcString = "tcunavailable", c.internalErrorState = 3); a(c) }; try { Ll(this, "addEventListener", f) } catch (g) {
        c.tcString =
        "tcunavailable", c.internalErrorState = 3, e && (clearTimeout(e), e = 0), d()
      }
    }; Il.prototype.removeEventListener = function (a) { a && a.listenerId && Ll(this, "removeEventListener", null, a.listenerId) };
    var Nl = function (a, b, c) {
      var d; d = void 0 === d ? "755" : d; var e; a: { if (a.publisher && a.publisher.restrictions) { var f = a.publisher.restrictions[b]; if (void 0 !== f) { e = f[void 0 === d ? "755" : d]; break a } } e = void 0 } var g = e; if (0 === g) return !1; var k = c; 2 === c ? (k = 0, 2 === g && (k = 1)) : 3 === c && (k = 1, 1 === g && (k = 0)); var m; if (0 === k) if (a.purpose && a.vendor) { var n = Ml(a.vendor.consents, void 0 === d ? "755" : d); m = n && "1" === b && a.purposeOneTreatment && "CH" === a.publisherCC ? !0 : n && Ml(a.purpose.consents, b) } else m = !0; else m = 1 === k ? a.purpose && a.vendor ? Ml(a.purpose.legitimateInterests,
        b) && Ml(a.vendor.legitimateInterests, void 0 === d ? "755" : d) : !0 : !0; return m
    }, Ml = function (a, b) { return !(!a || !a[b]) }, Ll = function (a, b, c, d) { c || (c = function () { }); if ("function" === typeof a.B.__tcfapi) { var e = a.B.__tcfapi; e(b, 2, c, d) } else if (Jl(a)) { Ol(a); var f = ++a.Zb; a.K[f] = c; if (a.h) { var g = {}; a.h.postMessage((g.__tcfapiCall = { command: b, version: 2, callId: f, parameter: d }, g), "*") } } else c({}, !1) }, Jl = function (a) {
      if (a.h) return a.h; var b; a: {
        for (var c = a.B, d = 0; 50 > d; ++d) {
          var e; try { e = !(!c.frames || !c.frames.__tcfapiLocator) } catch (k) {
            e =
            !1
          } if (e) { b = c; break a } var f; b: { try { var g = c.parent; if (g && g != c) { f = g; break b } } catch (k) { } f = null } if (!(c = f)) break
        } b = null
      } a.h = b; return a.h
    }, Ol = function (a) { a.C || (a.C = function (b) { try { var c; c = ("string" === typeof b.data ? JSON.parse(b.data) : b.data).__tcfapiReturn; a.K[c.callId](c.returnValue, c.success) } catch (d) { } }, ol(a.B, "message", a.C)) }, Pl = function (a) {
      if (!1 === a.gdprApplies) return !0; void 0 === a.internalErrorState && (a.internalErrorState = Hl(a)); return "error" === a.cmpStatus || 0 !== a.internalErrorState ? a.internalBlockOnErrors ?
        (Fl({ e: String(a.internalErrorState) }), !1) : !0 : "loaded" !== a.cmpStatus || "tcloaded" !== a.eventStatus && "useractioncomplete" !== a.eventStatus ? !1 : !0
    }; var Ql = { 1: 0, 3: 0, 4: 0, 7: 3, 9: 3, 10: 3 }, Rl = ll('', 500); function Sl() { var a = oi.tcf || {}; return oi.tcf = a }
    var Tl = function () { return new Il(z, { Dm: -1 }) }, Zl = function () {
      var a = Sl(), b = Tl(); Kl(b) && Ul() && M(124); if ((Vl() || Q(85)) && !a.active && Kl(b)) {
        a.active = !0; Vl() ? (a.Wb = {}, a.cmpId = 0, a.tcfPolicyVersion = 0, Wl(), Q(85) && ak([N.g.N]), a.tcString = "tcunavailable") : Q(85) && ak([N.g.I, N.g.nb, N.g.N]); try {
          b.addEventListener(function (c) {
            if (0 !== c.internalErrorState) Xl(a), Yl(a); else {
              a.gdprApplies = c.gdprApplies; if (Q(85)) {
                a.cmpId = c.cmpId; a.enableAdvertiserConsentMode = c.enableAdvertiserConsentMode; if (Ul()) return; a.tcfPolicyVersion =
                  c.tcfPolicyVersion
              } var d; if (!1 === c.gdprApplies) { var e = {}, f; for (f in Ql) Ql.hasOwnProperty(f) && (e[f] = !0); d = e; b.removeEventListener(c) } else if ("tcloaded" === c.eventStatus || "useractioncomplete" === c.eventStatus || "cmpuishown" === c.eventStatus) {
                var g = {}, k; for (k in Ql) if (Ql.hasOwnProperty(k)) if ("1" === k) { var m, n = c, p = !0; p = void 0 === p ? !1 : p; m = Pl(n) ? !1 === n.gdprApplies || "tcunavailable" === n.tcString || void 0 === n.gdprApplies && !p || "string" !== typeof n.tcString || !n.tcString.length ? !0 : Nl(n, "1", 0) : !1; g["1"] = m } else g[k] = Nl(c,
                  k, Ql[k]); d = g
              } d && (a.tcString = c.tcString || "tcempty", a.Wb = d, Yl(a))
            }
          })
        } catch (c) { Xl(a), Yl(a) }
      }
    }; function Xl(a) { a.type = "e"; a.tcString = "tcunavailable" } function Wl() { var a = {}, b = (a[N.g.I] = "denied", a[N.g.md] = Rl, a); Q(85) && (b[N.g.nb] = "denied"); Tj(b) } var Vl = function () { return !0 === z.gtag_enable_tcf_support }, Ul = function () { var a = Vl(); if (Q(85)) { var b = Sl().enableAdvertiserConsentMode; return !a && !b } return !a };
    function Yl(a) { var b = {}, c = (b[N.g.I] = a.Wb["1"] ? "granted" : "denied", b); Q(85) && (c[N.g.nb] = a.Wb["3"] && a.Wb["4"] ? "granted" : "denied", "number" === typeof a.tcfPolicyVersion && 4 <= a.tcfPolicyVersion && (c[N.g.N] = a.Wb["1"] && a.Wb["7"] ? "granted" : "denied")); Uj(c, { eventId: 0 }, { gdprApplies: a ? a.gdprApplies : void 0, tcString: $l() || "" }) }
    var $l = function () { var a = Sl(); if (a.active) return a.tcString }, am = function () { var a = Sl(); if (a.active && void 0 !== a.gdprApplies) return a.gdprApplies ? "1" : "0" }, bm = function (a) { if (!Ql.hasOwnProperty(String(a))) return !0; var b = Sl(); return b.active && b.Wb ? !!b.Wb[String(a)] : !0 }; var cm = [N.g.I, N.g.R], dm = [N.g.I, N.g.R, N.g.N, N.g.nb], em = {}, fm = (em[N.g.I] = 1, em[N.g.R] = 2, em); function gm(a) { switch (U(a, N.g.da)) { case void 0: return 1; case !1: return 3; default: return 2 } }
    var hm = function (a) { var b = 3 !== gm(a); Q(68) && (b = b && Gj(N.g.nb)); return b }, im = function () { var a = {}, b; for (b in fm) if (fm.hasOwnProperty(b)) { var c = fm[b], d, e = Aj(); e.accessedAny = !0; d = e.getConsentState(b); a[c] = d } var f = Q(52) && cm.every(Gj), g = Q(47); return f || g ? Pe(a, 1) : Pe(a, 0) }, jm = {}, km = (jm[N.g.I] = 0, jm[N.g.R] = 1, jm[N.g.N] = 2, jm[N.g.nb] = 3, jm); function lm(a) { switch (a) { case void 0: return 1; case !0: return 3; case !1: return 2; default: return 0 } }
    var mm = function (a) {
      if (Q(49)) { for (var b = "1", c = 0; c < dm.length; c++) { var d = b, e, f = dm[c], g = wj[f]; e = void 0 === g ? 0 : km.hasOwnProperty(g) ? 12 | km[g] : 8; var k = Aj(); k.accessedAny = !0; var m = k.entries[f] || {}; e = e << 2 | lm(m.implicit); b = d + ("" + "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ-_"[e] + "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ-_"[lm(m.declare) << 4 | lm(m.default) << 2 | lm(m.update)]) } void 0 !== a && (b += gm(a)); return b } for (var n = "G1", p = 0; p < cm.length; p++)switch (Hj(cm[p])) {
        case 3: n += "1";
          break; case 2: n += "0"; break; case 1: n += "-"
      }return n
    }, nm = function () { if (!Gj(N.g.N)) return "-"; var a = Aj(), b = yj, c = a.cps, d = a.usedSetCps, e = []; if (b && d) for (var f in xj) xj.hasOwnProperty(f) && xj[f].enabled && c.hasOwnProperty(f) && c[f].enabled && e.push(f); else { var g = b ? xj : c, k; for (k in g) g.hasOwnProperty(k) && g[k].enabled && e.push(k) } for (var m = "", n = 0; n < e.length; n++) { var p = mi[e[n]]; p && (m += p) } return "" === m ? "-" : m }; function om() { var a = !!jj["6"], b = !1; Q(85) && (b = !Ul() && "1" === am()); return a || b }
    var pm = function () { return om() ? "1" : "0" }, qm = function () { return om() || Aj().usedSetCps || !S(N.g.N) }, rm = function () {
      var a = "0", b = "0", c; var d = Sl(); c = d.active && Q(85) ? d.cmpId : void 0; "number" === typeof c && 0 <= c && 4095 >= c && (a = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ-_"[c >> 6 & 63], b = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ-_"[c & 63]); var e = "0", f; var g = Sl(); f = g.active && Q(85) ? g.tcfPolicyVersion : void 0; "number" === typeof f && 0 <= f && 63 >= f && (e = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ-_"[f]);
      var k = 0; jj["6"] && (k |= 1); "1" === am() && (k |= 2); Vl() && (k |= 4); var m; var n = Sl(); m = n.active && void 0 !== n.enableAdvertiserConsentMode ? n.enableAdvertiserConsentMode ? "1" : "0" : void 0; "1" === m && (k |= 8); return "1" + a + b + e + "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ-_"[k]
    }; var sm = function (a) { var b = 1, c, d, e; if (a) for (b = 0, d = a.length - 1; 0 <= d; d--)e = a.charCodeAt(d), b = (b << 6 & 268435455) + e + (e << 14), c = b & 266338304, b = 0 !== c ? b ^ c >> 21 : b; return b }; var tm = function (a, b, c) { for (var d = [], e = b.split(";"), f = 0; f < e.length; f++) { var g = e[f].split("="), k = g[0].replace(/^\s*|\s*$/g, ""); if (k && k == a) { var m = g.slice(1).join("=").replace(/^\s*|\s*$/g, ""); m && c && (m = decodeURIComponent(m)); d.push(m) } } return d }; function um(a) { return "null" !== a.origin }; var xm = function (a, b, c, d) { return vm(d) ? tm(a, String(b || wm()), c) : [] }, Am = function (a, b, c, d, e) { if (vm(e)) { var f = ym(a, d, e); if (1 === f.length) return f[0].id; if (0 !== f.length) { f = zm(f, function (g) { return g.vf }, b); if (1 === f.length) return f[0].id; f = zm(f, function (g) { return g.He }, c); return f[0] ? f[0].id : void 0 } } }; function Bm(a, b, c, d) { var e = wm(), f = window; um(f) && (f.document.cookie = a); var g = wm(); return e != g || void 0 != c && 0 <= xm(b, g, !1, d).indexOf(c) }
    var Fm = function (a, b, c, d) {
      function e(w, x, y) { if (null == y) return delete k[x], w; k[x] = y; return w + "; " + x + "=" + y } function f(w, x) { if (null == x) return delete k[x], w; k[x] = !0; return w + "; " + x } if (!vm(c.Eb)) return 2; var g; void 0 == b ? g = a + "=deleted; expires=" + (new Date(0)).toUTCString() : (c.encode && (b = encodeURIComponent(b)), b = Cm(b), g = a + "=" + b); var k = {}; g = e(g, "path", c.path); var m; c.expires instanceof Date ? m = c.expires.toUTCString() : null != c.expires && (m = "" + c.expires); g = e(g, "expires", m); g = e(g, "max-age", c.Ul); g = e(g, "samesite",
        c.qm); c.vm && (g = f(g, "secure")); var n = c.domain; if (n && "auto" === n.toLowerCase()) { for (var p = Dm(), q = void 0, r = !1, t = 0; t < p.length; ++t) { var u = "none" !== p[t] ? p[t] : void 0, v = e(g, "domain", u); v = f(v, c.flags); try { d && d(a, k) } catch (w) { q = w; continue } r = !0; if (!Em(u, c.path) && Bm(v, a, b, c.Eb)) return 0 } if (q && !r) throw q; return 1 } n && "none" !== n.toLowerCase() && (g = e(g, "domain", n)); g = f(g, c.flags); d && d(a, k); return Em(n, c.path) ? 1 : Bm(g, a, b, c.Eb) ? 0 : 1
    }, Gm = function (a, b, c) {
      null == c.path && (c.path = "/"); c.domain || (c.domain = "auto"); return Fm(a,
        b, c)
    }; function zm(a, b, c) { for (var d = [], e = [], f, g = 0; g < a.length; g++) { var k = a[g], m = b(k); m === c ? d.push(k) : void 0 === f || m < f ? (e = [k], f = m) : m === f && e.push(k) } return 0 < d.length ? d : e } function ym(a, b, c) { for (var d = [], e = xm(a, void 0, void 0, c), f = 0; f < e.length; f++) { var g = e[f].split("."), k = g.shift(); if (!b || -1 !== b.indexOf(k)) { var m = g.shift(); m && (m = m.split("-"), d.push({ id: g.join("."), vf: 1 * m[0] || 1, He: 1 * m[1] || 1 })) } } return d }
    var Cm = function (a) { a && 1200 < a.length && (a = a.substring(0, 1200)); return a }, Hm = /^(www\.)?google(\.com?)?(\.[a-z]{2})?$/, Tm = /(^|\.)doubleclick\.net$/i, Em = function (a, b) { return Tm.test(window.document.location.hostname) || "/" === b && Hm.test(a) }, wm = function () { return um(window) ? window.document.cookie : "" }, Dm = function () {
      var a = [], b = window.document.location.hostname.split("."); if (4 === b.length) { var c = b[b.length - 1]; if (parseInt(c, 10).toString() === c) return ["none"] } for (var d = b.length - 2; 0 <= d; d--)a.push(b.slice(d).join("."));
      var e = window.document.location.hostname; Tm.test(e) || Hm.test(e) || a.push("none"); return a
    }, vm = function (a) { return a && tj().h() ? Ij(a) ? Gj(a) : !1 : !0 }; var Um = function (a) { var b = Math.round(2147483647 * Math.random()); return a ? String(b ^ sm(a) & 2147483647) : String(b) }, Vm = function (a) { return [Um(a), Math.round(Ra() / 1E3)].join(".") }, Ym = function (a, b, c, d, e) { var f = Wm(b); return Am(a, f, Xm(c), d, e) }, Zm = function (a, b, c, d) { var e = "" + Wm(c), f = Xm(d); 1 < f && (e += "-" + f); return [b, e, a].join(".") }, Wm = function (a) { if (!a) return 1; a = 0 === a.indexOf(".") ? a.substr(1) : a; return a.split(".").length }, Xm = function (a) {
      if (!a || "/" === a) return 1; "/" !== a[0] && (a = "/" + a); "/" !== a[a.length - 1] && (a += "/"); return a.split("/").length -
        1
    }; var $m = function () { oi.dedupe_gclid || (oi.dedupe_gclid = "" + Vm()); return oi.dedupe_gclid }; var an = function () { var a = !1; return a }; var bn = { UA: 1, AW: 2, DC: 3, G: 4, GF: 5, GT: 12, GTM: 14, HA: 6, MC: 7 }, cn = function (a) {
      var b = Uf.ctid.split("-")[0].toUpperCase(), c = {}; c.ctid = Uf.ctid; c.km = ni.ee; c.om = ni.Lg; c.Rl = gk.hf ? 2 : 1; ui ? (c.Jf = bn[b], c.Jf || (c.Jf = 0)) : c.Jf = Ai ? 13 : 10; yi ? c.sh = 1 : an() ? c.sh = 2 : c.sh = 3; var d; var e = c.Jf, f = c.sh; void 0 === e ? d = "" : (f || (f = 0), d = "" + Jg(1, 1) + "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ-_"[e << 2 | f]); var g = c.Ym, k = 4 + d + (g ? "" + Jg(2, 1) + "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ-_"[g] : ""), m, n = c.om; m = n &&
        Ig.test(n) ? "" + Jg(3, 2) + n : ""; var p, q = c.km; p = q ? "" + Jg(4, 1) + "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ-_"[q] : ""; var r; var t = c.ctid; if (t && a) { var u = t.split("-"), v = u[0].toUpperCase(); if ("GTM" !== v && "OPT" !== v) r = ""; else { var w = u[1]; r = "" + Jg(5, 3) + "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ-_"[1 + w.length] + (c.Rl || 0) + w } } else r = ""; return k + m + p + r
    }; var dn = void 0; function en(a) { var b = ""; return b }; var fn = function (a) { for (var b = [], c = 0, d = 0; d < a.length; d++) { var e = a.charCodeAt(d); 128 > e ? b[c++] = e : (2048 > e ? b[c++] = e >> 6 | 192 : (55296 == (e & 64512) && d + 1 < a.length && 56320 == (a.charCodeAt(d + 1) & 64512) ? (e = 65536 + ((e & 1023) << 10) + (a.charCodeAt(++d) & 1023), b[c++] = e >> 18 | 240, b[c++] = e >> 12 & 63 | 128) : b[c++] = e >> 12 | 224, b[c++] = e >> 6 & 63 | 128), b[c++] = e & 63 | 128) } return b }; $b(); ql() || Xb("iPod"); Xb("iPad"); !Xb("Android") || ac() || $b() || Zb() || Xb("Silk"); ac(); !Xb("Safari") || ac() || (Yb() ? 0 : Xb("Coast")) || Zb() || (Yb() ? 0 : Xb("Edge")) || (Yb() ? Wb("Microsoft Edge") : Xb("Edg/")) || (Yb() ? Wb("Opera") : Xb("OPR")) || $b() || Xb("Silk") || Xb("Android") || rl(); var gn = {}, hn = null, jn = function (a) {
      for (var b = [], c = 0, d = 0; d < a.length; d++) { var e = a.charCodeAt(d); 255 < e && (b[c++] = e & 255, e >>= 8); b[c++] = e } var f = 4; void 0 === f && (f = 0); if (!hn) { hn = {}; for (var g = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789".split(""), k = ["+/=", "+/", "-_=", "-_.", "-_"], m = 0; 5 > m; m++) { var n = g.concat(k[m].split("")); gn[m] = n; for (var p = 0; p < n.length; p++) { var q = n[p]; void 0 === hn[q] && (hn[q] = p) } } } for (var r = gn[f], t = Array(Math.floor(b.length / 3)), u = r[64] || "", v = 0, w = 0; v < b.length - 2; v += 3) {
        var x = b[v],
        y = b[v + 1], A = b[v + 2], B = r[x >> 2], D = r[(x & 3) << 4 | y >> 4], G = r[(y & 15) << 2 | A >> 6], E = r[A & 63]; t[w++] = "" + B + D + G + E
      } var I = 0, O = u; switch (b.length - v) { case 2: I = b[v + 1], O = r[(I & 15) << 2] || u; case 1: var R = b[v]; t[w] = "" + r[R >> 2] + r[(R & 3) << 4 | I >> 4] + O + u }return t.join("")
    }; var kn = "platform platformVersion architecture model uaFullVersion bitness fullVersionList wow64".split(" "); function ln(a) { var b; return null != (b = a.google_tag_data) ? b : a.google_tag_data = {} } function mn() { var a = z.google_tag_data, b; if (null != a && a.uach) { var c = a.uach, d = Object.assign({}, c); c.fullVersionList && (d.fullVersionList = c.fullVersionList.slice(0)); b = d } else b = null; return b } function nn() { var a, b; return null != (b = null == (a = z.google_tag_data) ? void 0 : a.uach_promise) ? b : null }
    function on(a) { var b, c; return "function" === typeof (null == (b = a.navigator) ? void 0 : null == (c = b.userAgentData) ? void 0 : c.getHighEntropyValues) } function pn() { var a = z; if (!on(a)) return null; var b = ln(a); if (b.uach_promise) return b.uach_promise; var c = a.navigator.userAgentData.getHighEntropyValues(kn).then(function (d) { null != b.uach || (b.uach = d); return d }); return b.uach_promise = c };
    var qn, rn = function () { if (on(z) && (qn = Ra(), !nn())) { var a = pn(); a && (a.then(function () { M(95); }), a.catch(function () { M(96) })) } }, tn = function (a) {
      var b = sn.Hm, c = function (g, k) { try { a(g, k) } catch (m) { } }, d = mn(); if (d) c(d); else {
        var e = nn(); if (e) {
          b =
          Math.min(Math.max(isFinite(b) ? b : 0, 0), 1E3); var f = z.setTimeout(function () { c.Ce || (c.Ce = !0, M(106), c(null, Error("Timeout"))) }, b); e.then(function (g) { c.Ce || (c.Ce = !0, M(104), z.clearTimeout(f), c(g)) }).catch(function (g) { c.Ce || (c.Ce = !0, M(105), z.clearTimeout(f), c(null, g)) })
        } else c(null)
      }
    }, un = function (a, b) {
      a && (b.h[N.g.Pd] = a.architecture, b.h[N.g.Qd] = a.bitness, a.fullVersionList && (b.h[N.g.Rd] = a.fullVersionList.map(function (c) { return encodeURIComponent(c.brand || "") + ";" + encodeURIComponent(c.version || "") }).join("|")),
        b.h[N.g.Sd] = a.mobile ? "1" : "0", b.h[N.g.Td] = a.model, b.h[N.g.Ud] = a.platform, b.h[N.g.Vd] = a.platformVersion, b.h[N.g.Wd] = a.wow64 ? "1" : "0")
    }; var vn = /:[0-9]+$/, wn = /^\d+\.fls\.doubleclick\.net$/, xn = function (a, b, c, d) { for (var e = [], f = a.split("&"), g = 0; g < f.length; g++) { var k = f[g].split("="); if (decodeURIComponent(k[0]).replace(/\+/g, " ") === b) { var m = k.slice(1).join("="); if (!c) return d ? m : decodeURIComponent(m).replace(/\+/g, " "); e.push(d ? m : decodeURIComponent(m).replace(/\+/g, " ")) } } return c ? e : void 0 }, An = function (a, b, c, d, e) {
      b && (b = String(b).toLowerCase()); if ("protocol" === b || "port" === b) a.protocol = yn(a.protocol) || yn(z.location.protocol); "port" === b ? a.port =
        String(Number(a.hostname ? a.port : z.location.port) || ("http" === a.protocol ? 80 : "https" === a.protocol ? 443 : "")) : "host" === b && (a.hostname = (a.hostname || z.location.hostname).replace(vn, "").toLowerCase()); return zn(a, b, c, d, e)
    }, zn = function (a, b, c, d, e) {
      var f, g = yn(a.protocol); b && (b = String(b).toLowerCase()); switch (b) {
        case "url_no_fragment": f = Bn(a); break; case "protocol": f = g; break; case "host": f = a.hostname.replace(vn, "").toLowerCase(); if (c) { var k = /^www\d*\./.exec(f); k && k[0] && (f = f.substr(k[0].length)) } break; case "port": f =
          String(Number(a.port) || ("http" === g ? 80 : "https" === g ? 443 : "")); break; case "path": a.pathname || a.hostname || yb("TAGGING", 1); f = "/" === a.pathname.substr(0, 1) ? a.pathname : "/" + a.pathname; var m = f.split("/"); 0 <= (d || []).indexOf(m[m.length - 1]) && (m[m.length - 1] = ""); f = m.join("/"); break; case "query": f = a.search.replace("?", ""); e && (f = xn(f, e, !1)); break; case "extension": var n = a.pathname.split("."); f = 1 < n.length ? n[n.length - 1] : ""; f = f.split("/")[0]; break; case "fragment": f = a.hash.replace("#", ""); break; default: f = a && a.href
      }return f
    },
      yn = function (a) { return a ? a.replace(":", "").toLowerCase() : "" }, Bn = function (a) { var b = ""; if (a && a.href) { var c = a.href.indexOf("#"); b = 0 > c ? a.href : a.href.substr(0, c) } return b }, Cn = function (a) { var b = C.createElement("a"); a && (b.href = a); var c = b.pathname; "/" !== c[0] && (a || yb("TAGGING", 1), c = "/" + c); var d = b.hostname.replace(vn, ""); return { href: b.href, protocol: b.protocol, host: b.host, hostname: d, pathname: c, search: b.search, hash: b.hash, port: b.port } }, Dn = function (a) {
        function b(n) {
          var p = n.split("=")[0]; return 0 > d.indexOf(p) ? n :
            p + "=0"
        } function c(n) { return n.split("&").map(b).filter(function (p) { return void 0 !== p }).join("&") } var d = "gclid dclid gbraid wbraid gclaw gcldc gclha gclgf gclgb _gl".split(" "), e = Cn(a), f = a.split(/[?#]/)[0], g = e.search, k = e.hash; "?" === g[0] && (g = g.substring(1)); "#" === k[0] && (k = k.substring(1)); g = c(g); k = c(k); "" !== g && (g = "?" + g); "" !== k && (k = "#" + k); var m = "" + f + g + k; "/" === m[m.length - 1] && (m = m.substring(0, m.length - 1)); return m
      }, En = function (a) {
        var b = Cn(z.location.href), c = An(b, "host", !1); if (c && c.match(wn)) {
          var d = An(b,
            "path").split(a + "="); if (1 < d.length) return d[1].split(";")[0].split("?")[0]
        }
      }; function Fn(a, b, c, d) { var e, f = Number(null != a.Tb ? a.Tb : void 0); 0 !== f && (e = new Date((b || Ra()) + 1E3 * (f || 7776E3))); return { path: a.path, domain: a.domain, flags: a.flags, encode: !!c, expires: e, Eb: d } }; var Gn; var Kn = function () { var a = Hn, b = In, c = Jn(), d = function (g) { a(g.target || g.srcElement || {}) }, e = function (g) { b(g.target || g.srcElement || {}) }; if (!c.init) { Nc(C, "mousedown", d); Nc(C, "keyup", d); Nc(C, "submit", e); var f = HTMLFormElement.prototype.submit; HTMLFormElement.prototype.submit = function () { b(this); f.call(this) }; c.init = !0 } }, Ln = function (a, b, c, d, e) { var f = { callback: a, domains: b, fragment: 2 === c, placement: c, forms: d, sameHost: e }; Jn().decorators.push(f) }, Mn = function (a, b, c) {
        for (var d = Jn().decorators, e = {}, f = 0; f < d.length; ++f) {
          var g =
            d[f], k; if (k = !c || g.forms) a: { var m = g.domains, n = a, p = !!g.sameHost; if (m && (p || n !== C.location.hostname)) for (var q = 0; q < m.length; q++)if (m[q] instanceof RegExp) { if (m[q].test(n)) { k = !0; break a } } else if (0 <= n.indexOf(m[q]) || p && 0 <= m[q].indexOf(n)) { k = !0; break a } k = !1 } if (k) { var r = g.placement; void 0 == r && (r = g.fragment ? 2 : 1); r === b && Ua(e, g.callback()) }
        } return e
      }; function Jn() { var a = Ec("google_tag_data", {}), b = a.gl; b && b.decorators || (b = { decorators: [] }, a.gl = b); return b }; var Nn = /(.*?)\*(.*?)\*(.*)/, On = /^https?:\/\/([^\/]*?)\.?cdn\.ampproject\.org\/?(.*)/, Pn = /^(?:www\.|m\.|amp\.)+/, Qn = /([^?#]+)(\?[^#]*)?(#.*)?/; function Rn(a) { return new RegExp("(.*?)(^|&)" + a + "=([^&]*)&?(.*)") } var Tn = function (a) { var b = [], c; for (c in a) if (a.hasOwnProperty(c)) { var d = a[c]; void 0 !== d && d === d && null !== d && "[object Object]" !== d.toString() && (b.push(c), b.push(vb(String(d)))) } var e = b.join("*"); return ["1", Sn(e), e].join("*") };
    function Sn(a, b) { var c = [Cc.userAgent, (new Date).getTimezoneOffset(), Cc.userLanguage || Cc.language, Math.floor(Ra() / 60 / 1E3) - (void 0 === b ? 0 : b), a].join("*"), d; if (!(d = Gn)) { for (var e = Array(256), f = 0; 256 > f; f++) { for (var g = f, k = 0; 8 > k; k++)g = g & 1 ? g >>> 1 ^ 3988292384 : g >>> 1; e[f] = g } d = e } Gn = d; for (var m = 4294967295, n = 0; n < c.length; n++)m = m >>> 8 ^ Gn[(m ^ c.charCodeAt(n)) & 255]; return ((m ^ -1) >>> 0).toString(36) }
    function Un() { return function (a) { var b = Cn(z.location.href), c = b.search.replace("?", ""), d = xn(c, "_gl", !1, !0) || ""; a.query = Vn(d) || {}; var e = An(b, "fragment").match(Rn("_gl")); a.fragment = Vn(e && e[3] || "") || {} } } function Wn(a, b) { var c = Rn(a).exec(b), d = b; if (c) { var e = c[2], f = c[4]; d = c[1]; f && (d = d + e + f) } return d }
    var Xn = function (a, b) { b || (b = "_gl"); var c = Qn.exec(a); if (!c) return ""; var d = c[1], e = Wn(b, (c[2] || "").slice(1)), f = Wn(b, (c[3] || "").slice(1)); e.length && (e = "?" + e); f.length && (f = "#" + f); return "" + d + e + f }, Yn = function (a) { var b = Un(), c = Jn(); c.data || (c.data = { query: {}, fragment: {} }, b(c.data)); var d = {}, e = c.data; e && (Ua(d, e.query), a && Ua(d, e.fragment)); return d }, Vn = function (a) {
      try { var b = Zn(a, 3); if (void 0 !== b) { for (var c = {}, d = b ? b.split("*") : [], e = 0; e + 1 < d.length; e += 2) { var f = d[e], g = wb(d[e + 1]); c[f] = g } yb("TAGGING", 6); return c } } catch (k) {
        yb("TAGGING",
          8)
      }
    }; function Zn(a, b) { if (a) { var c; a: { for (var d = a, e = 0; 3 > e; ++e) { var f = Nn.exec(d); if (f) { c = f; break a } d = decodeURIComponent(d) } c = void 0 } var g = c; if (g && "1" === g[1]) { var k = g[3], m; a: { for (var n = g[2], p = 0; p < b; ++p)if (n === Sn(k, p)) { m = !0; break a } m = !1 } if (m) return k; yb("TAGGING", 7) } } }
    function $n(a, b, c, d) { function e(p) { p = Wn(a, p); var q = p.charAt(p.length - 1); p && "&" !== q && (p += "&"); return p + n } d = void 0 === d ? !1 : d; var f = Qn.exec(c); if (!f) return ""; var g = f[1], k = f[2] || "", m = f[3] || "", n = a + "=" + b; d ? m = "#" + e(m.substring(1)) : k = "?" + e(k.substring(1)); return "" + g + k + m }
    function ao(a, b) { var c = "FORM" === (a.tagName || "").toUpperCase(), d = Mn(b, 1, c), e = Mn(b, 2, c), f = Mn(b, 3, c); if (Wa(d)) { var g = Tn(d); c ? bo("_gl", g, a) : co("_gl", g, a, !1) } if (!c && Wa(e)) { var k = Tn(e); co("_gl", k, a, !0) } for (var m in f) if (f.hasOwnProperty(m)) a: { var n = m, p = f[m], q = a; if (q.tagName) { if ("a" === q.tagName.toLowerCase()) { co(n, p, q); break a } if ("form" === q.tagName.toLowerCase()) { bo(n, p, q); break a } } "string" == typeof q && $n(n, p, q) } }
    function co(a, b, c, d) { if (c.href) { var e = $n(a, b, c.href, void 0 === d ? !1 : d); Kb.test(e) && (c.href = e) } }
    function bo(a, b, c) { if (c && c.action) { var d = (c.method || "").toLowerCase(); if ("get" === d) { for (var e = c.childNodes || [], f = !1, g = 0; g < e.length; g++) { var k = e[g]; if (k.name === a) { k.setAttribute("value", b); f = !0; break } } if (!f) { var m = C.createElement("input"); m.setAttribute("type", "hidden"); m.setAttribute("name", a); m.setAttribute("value", b); c.appendChild(m) } } else if ("post" === d) { var n = $n(a, b, c.action); Kb.test(n) && (c.action = n) } } }
    function Hn(a) { try { var b; a: { for (var c = a, d = 100; c && 0 < d;) { if (c.href && c.nodeName.match(/^a(?:rea)?$/i)) { b = c; break a } c = c.parentNode; d-- } b = null } var e = b; if (e) { var f = e.protocol; "http:" !== f && "https:" !== f || ao(e, e.hostname) } } catch (g) { } } function In(a) { try { if (a.action) { var b = An(Cn(a.action), "host"); ao(a, b) } } catch (c) { } }
    var eo = function (a, b, c, d) { Kn(); Ln(a, b, "fragment" === c ? 2 : 1, !!d, !1) }, fo = function (a, b) { Kn(); Ln(a, [zn(z.location, "host", !0)], b, !0, !0) }, go = function () {
      var a = C.location.hostname, b = On.exec(C.referrer); if (!b) return !1; var c = b[2], d = b[1], e = ""; if (c) { var f = c.split("/"), g = f[1]; e = "s" === g ? decodeURIComponent(f[2]) : decodeURIComponent(g) } else if (d) { if (0 === d.indexOf("xn--")) return !1; e = d.replace(/-/g, ".").replace(/\.\./g, "-") } var k = a.replace(Pn, ""), m = e.replace(Pn, ""), n; if (!(n = k === m)) {
        var p = "." + m; n = k.substring(k.length - p.length,
          k.length) === p
      } return n
    }, ho = function (a, b) { return !1 === a ? !1 : a || b || go() }; var io = ["1"], jo = {}, ko = {}, po = function (a, b) { b = void 0 === b ? !0 : b; var c = lo(a.prefix); if (!jo[c]) if (mo(c, a.path, a.domain)) { var d = ko[lo(a.prefix)]; no(a, d ? d.id : void 0, d ? d.qh : void 0) } else { var e = En("auiddc"); if (e) yb("TAGGING", 17), jo[c] = e; else if (b) { var f = lo(a.prefix), g = Vm(); if (0 === oo(f, g, a)) { var k = Ec("google_tag_data", {}); k._gcl_au || (k._gcl_au = g) } mo(c, a.path, a.domain) } } };
    function no(a, b, c) { var d = lo(a.prefix), e = jo[d]; if (e) { var f = e.split("."); if (2 === f.length) { var g = Number(f[1]) || 0; if (g) { var k = e; b && (k = e + "." + b + "." + (c ? c : Math.floor(Ra() / 1E3))); oo(d, k, a, 1E3 * g) } } } } function oo(a, b, c, d) { var e = Zm(b, "1", c.domain, c.path), f = Fn(c, d); f.Eb = "ad_storage"; return Gm(a, e, f) } function mo(a, b, c) { var d = Ym(a, b, c, io, "ad_storage"); if (!d) return !1; qo(a, d); return !0 }
    function qo(a, b) { var c = b.split("."); 5 === c.length ? (jo[a] = c.slice(0, 2).join("."), ko[a] = { id: c.slice(2, 4).join("."), qh: Number(c[4]) || 0 }) : 3 === c.length ? ko[a] = { id: c.slice(0, 2).join("."), qh: Number(c[2]) || 0 } : jo[a] = b } function lo(a) { return (a || "_gcl") + "_au" } function ro(a) { tj().h() ? Nj(function () { Gj("ad_storage") ? a() : Oj(a, "ad_storage") }, ["ad_storage"]) : a() }
    function so(a) { var b = Yn(!0), c = lo(a.prefix); ro(function () { var d = b[c]; if (d) { qo(c, d); var e = 1E3 * Number(jo[c].split(".")[1]); if (e) { yb("TAGGING", 16); var f = Fn(a, e); f.Eb = "ad_storage"; var g = Zm(d, "1", a.domain, a.path); Gm(c, g, f) } } }) } function to(a, b, c, d, e) { e = e || {}; var f = function () { var g = {}, k = Ym(a, e.path, e.domain, io, "ad_storage"); k && (g[a] = k); return g }; ro(function () { eo(f, b, c, d) }) }; var uo = function (a) { for (var b = [], c = C.cookie.split(";"), d = new RegExp("^\\s*" + (a || "_gac") + "_(UA-\\d+-\\d+)=\\s*(.+?)\\s*$"), e = 0; e < c.length; e++) { var f = c[e].match(d); f && b.push({ Kh: f[1], value: f[2], timestamp: Number(f[2].split(".")[1]) || 0 }) } b.sort(function (g, k) { return k.timestamp - g.timestamp }); return b };
    function vo(a, b) { var c = uo(a), d = {}; if (!c || !c.length) return d; for (var e = 0; e < c.length; e++) { var f = c[e].value.split("."); if (!("1" !== f[0] || b && 3 > f.length || !b && 3 !== f.length) && Number(f[1])) { d[c[e].Kh] || (d[c[e].Kh] = []); var g = { version: f[0], timestamp: 1E3 * Number(f[1]), X: f[2] }; b && 3 < f.length && (g.labels = f.slice(3)); d[c[e].Kh].push(g) } } return d }; var wo = /^\w+$/, xo = /^[\w-]+$/, yo = { aw: "_aw", dc: "_dc", gf: "_gf", ha: "_ha", gp: "_gp", gb: "_gb" }, zo = function () { return tj().h() ? Gj("ad_storage") : !0 }, Ao = function (a, b) { Ij("ad_storage") ? zo() ? a() : Oj(a, "ad_storage") : b ? yb("TAGGING", 3) : Nj(function () { Ao(a, !0) }, ["ad_storage"]) }, Co = function (a) { return Bo(a).map(function (b) { return b.X }) }, Bo = function (a) {
      var b = []; if (!um(z) || !C.cookie) return b; var c = xm(a, C.cookie, void 0, "ad_storage"); if (!c || 0 == c.length) return b; for (var d = {}, e = 0; e < c.length; d = { X: d.X }, e++) {
        var f = Do(c[e]); if (null !=
          f) { var g = f, k = g.version; d.X = g.X; var m = g.timestamp, n = g.labels, p = Fa(b, function (q) { return function (r) { return r.X === q.X } }(d)); p ? (p.timestamp = Math.max(p.timestamp, m), p.labels = Eo(p.labels, n || [])) : b.push({ version: k, X: d.X, timestamp: m, labels: n }) }
      } b.sort(function (q, r) { return r.timestamp - q.timestamp }); return Fo(b)
    }; function Eo(a, b) { for (var c = {}, d = [], e = 0; e < a.length; e++)c[a[e]] = !0, d.push(a[e]); for (var f = 0; f < b.length; f++)c[b[f]] || d.push(b[f]); return d }
    function Go(a) { return a && "string" == typeof a && a.match(wo) ? a : "_gcl" }
    var Io = function () { var a = Cn(z.location.href), b = An(a, "query", !1, void 0, "gclid"), c = An(a, "query", !1, void 0, "gclsrc"), d = An(a, "query", !1, void 0, "wbraid"), e = An(a, "query", !1, void 0, "dclid"); if (!b || !c || !d) { var f = a.hash.replace("#", ""); b = b || xn(f, "gclid", !1); c = c || xn(f, "gclsrc", !1); d = d || xn(f, "wbraid", !1) } return Ho(b, c, e, d) }, Ho = function (a, b, c, d) {
      var e = {}, f = function (g, k) { e[k] || (e[k] = []); e[k].push(g) }; e.gclid = a; e.gclsrc = b; e.dclid = c; void 0 !== d && xo.test(d) && (e.gbraid = d, f(d, "gb")); if (void 0 !== a && a.match(xo)) switch (b) {
        case void 0: f(a,
          "aw"); break; case "aw.ds": f(a, "aw"); f(a, "dc"); break; case "ds": f(a, "dc"); break; case "3p.ds": f(a, "dc"); break; case "gf": f(a, "gf"); break; case "ha": f(a, "ha")
      }c && f(c, "dc"); return e
    }, Ko = function (a) { var b = Io(); Ao(function () { Jo(b, !1, a) }) };
    function Jo(a, b, c, d, e) {
      function f(w, x) { var y = Lo(w, g); y && (Gm(y, x, k), m = !0) } c = c || {}; e = e || []; var g = Go(c.prefix); d = d || Ra(); var k = Fn(c, d, !0); k.Eb = "ad_storage"; var m = !1, n = Math.round(d / 1E3), p = function (w) { var x = ["GCL", n, w]; 0 < e.length && x.push(e.join(".")); return x.join(".") }; a.aw && f("aw", p(a.aw[0])); a.dc && f("dc", p(a.dc[0])); a.gf && f("gf", p(a.gf[0])); a.ha && f("ha", p(a.ha[0])); a.gp && f("gp", p(a.gp[0])); if (!m && a.gb) {
        var q = a.gb[0], r = Lo("gb", g), t = !1; if (!b) for (var u = Bo(r), v = 0; v < u.length; v++)u[v].X === q && u[v].labels &&
          0 < u[v].labels.length && (t = !0); t || f("gb", p(q))
      }
    }
    var No = function (a, b) { var c = Yn(!0); Ao(function () { for (var d = Go(b.prefix), e = 0; e < a.length; ++e) { var f = a[e]; if (void 0 !== yo[f]) { var g = Lo(f, d), k = c[g]; if (k) { var m = Math.min(Mo(k), Ra()), n; b: { var p = m; if (um(z)) for (var q = xm(g, C.cookie, void 0, "ad_storage"), r = 0; r < q.length; ++r)if (Mo(q[r]) > p) { n = !0; break b } n = !1 } if (!n) { var t = Fn(b, m, !0); t.Eb = "ad_storage"; Gm(g, k, t) } } } } Jo(Ho(c.gclid, c.gclsrc), !1, b) }) }, Lo = function (a, b) { var c = yo[a]; if (void 0 !== c) return b + c }, Mo = function (a) {
      return 0 !== Oo(a.split(".")).length ? 1E3 * (Number(a.split(".")[1]) ||
        0) : 0
    }; function Do(a) { var b = Oo(a.split(".")); return 0 === b.length ? null : { version: b[0], X: b[2], timestamp: 1E3 * (Number(b[1]) || 0), labels: b.slice(3) } } function Oo(a) { return 3 > a.length || "GCL" !== a[0] && "1" !== a[0] || !/^\d+$/.test(a[1]) || !xo.test(a[2]) ? [] : a }
    var Po = function (a, b, c, d, e) { if (Da(b) && um(z)) { var f = Go(e), g = function () { for (var k = {}, m = 0; m < a.length; ++m) { var n = Lo(a[m], f); if (n) { var p = xm(n, C.cookie, void 0, "ad_storage"); p.length && (k[n] = p.sort()[p.length - 1]) } } return k }; Ao(function () { eo(g, b, c, d) }) } }, Fo = function (a) { return a.filter(function (b) { return xo.test(b.X) }) }, Qo = function (a, b) {
      if (um(z)) {
        for (var c = Go(b.prefix), d = {}, e = 0; e < a.length; e++)yo[a[e]] && (d[a[e]] = yo[a[e]]); Ao(function () {
          l(d, function (f, g) {
            var k = xm(c + g, C.cookie, void 0, "ad_storage"); k.sort(function (t,
              u) { return Mo(u) - Mo(t) }); if (k.length) { var m = k[0], n = Mo(m), p = 0 !== Oo(m.split(".")).length ? m.split(".").slice(3) : [], q = {}, r; r = 0 !== Oo(m.split(".")).length ? m.split(".")[2] : void 0; q[f] = [r]; Jo(q, !0, b, n, p) }
          })
        })
      }
    }; function Ro(a, b) { for (var c = 0; c < b.length; ++c)if (a[b[c]]) return !0; return !1 }
    var So = function (a) { function b(e, f, g) { g && (e[f] = g) } if (Jj()) { var c = Io(); if (Ro(c, a)) { var d = {}; b(d, "gclid", c.gclid); b(d, "dclid", c.dclid); b(d, "gclsrc", c.gclsrc); b(d, "wbraid", c.gbraid); fo(function () { return d }, 3); fo(function () { var e = {}; return e._up = "1", e }, 1) } } }, To = function (a, b, c, d) {
      var e = []; c = c || {}; if (!zo()) return e; var f = Bo(a); if (!f.length) return e; for (var g = 0; g < f.length; g++)-1 === (f[g].labels || []).indexOf(b) ? e.push(0) : e.push(1); if (d) return e; if (1 !== e[0]) {
        var k = f[0], m = f[0].timestamp, n = [k.version, Math.round(m /
          1E3), k.X].concat(k.labels || [], [b]).join("."), p = Fn(c, m, !0); p.Eb = "ad_storage"; Gm(a, n, p)
      } return e
    }; function Uo(a, b) { var c = Go(b), d = Lo(a, c); if (!d) return 0; for (var e = Bo(d), f = 0, g = 0; g < e.length; g++)f = Math.max(f, e[g].timestamp); return f } function Vo(a) { var b = 0, c; for (c in a) for (var d = a[c], e = 0; e < d.length; e++)b = Math.max(b, Number(d[e].timestamp)); return b } var Wo = function (a) { var b = Math.max(Uo("aw", a), Vo(zo() ? vo() : {})); return Math.max(Uo("gb", a), Vo(zo() ? vo("_gac_gb", !0) : {})) > b }; var ap = /[A-Z]+/, bp = /\s/, cp = function (a, b) { if (h(a)) { a = Oa(a); var c = a.indexOf("-"); if (!(0 > c)) { var d = a.substring(0, c); if (ap.test(d)) { var e = a.substring(c + 1), f; if (b) { var g = function (n) { var p = n.indexOf("/"); return 0 > p ? [n] : [n.substring(0, p), n.substring(p + 1)] }; f = g(e); if ("DC" === d && 2 === f.length) { var k = g(f[1]); 2 === k.length && (f[1] = k[0], f.push(k[1])) } } else { f = e.split("/"); for (var m = 0; m < f.length; m++)if (!f[m] || bp.test(f[m]) && ("AW" !== d || 1 !== m)) return } return { id: a, prefix: d, ba: d + "-" + f[0], O: f } } } } }, ep = function (a, b) {
      for (var c =
        {}, d = 0; d < a.length; ++d) { var e = cp(a[d], b); e && (c[e.id] = e) } dp(c); var f = []; l(c, function (g, k) { f.push(k) }); return f
    }; function dp(a) { var b = [], c; for (c in a) if (a.hasOwnProperty(c)) { var d = a[c]; "AW" === d.prefix && d.O[1] && b.push(d.ba) } for (var e = 0; e < b.length; ++e)delete a[b[e]] }; var fp = function (a, b, c, d) { var e = Kc(), f; if (1 === e) a: { var g = Ci; g = g.toLowerCase(); for (var k = "https://" + g, m = "http://" + g, n = 1, p = C.getElementsByTagName("script"), q = 0; q < p.length && 100 > q; q++) { var r = p[q].src; if (r) { r = r.toLowerCase(); if (0 === r.indexOf(m)) { f = 3; break a } 1 === n && 0 === r.indexOf(k) && (n = 2) } } f = n } else f = e; return (2 === f || d || "http:" != z.location.protocol ? a : b) + c }; var rp, sp = !1; function tp() { sp = !0; rp = rp || {} } var up = function (a) { sp || tp(); return rp[a] }; var vp = function (a, b, c) { this.target = a; this.eventName = b; this.s = c; this.h = {}; this.metadata = lb(c.eventMetadata || {}); this.isAborted = !1 }; vp.prototype.copyToHitData = function (a, b) { var c = U(this.s, a); void 0 !== c ? this.h[a] = c : void 0 !== b && (this.h[a] = b) }; var wp = function (a, b, c) { var d = up(a.target.ba); return d && d.hasOwnProperty(b) ? d[b] : c }; function xp(a) {
      return {
        getDestinationId: function () { return a.target.ba }, getEventName: function () { return a.eventName }, setEventName: function (b) { a.eventName = b }, getHitData: function (b) { return a.h[b] }, setHitData: function (b, c) { a.h[b] = c }, setHitDataIfNotDefined: function (b, c) { void 0 === a.h[b] && (a.h[b] = c) }, copyToHitData: function (b, c) { a.copyToHitData(b, c) }, getMetadata: function (b) { return a.metadata[b] }, setMetadata: function (b, c) { a.metadata[b] = c }, isAborted: function () { return a.isAborted }, abort: function () {
          a.isAborted =
          !0
        }, getFromEventContext: function (b) { return U(a.s, b) }, nj: function () { return a }, getHitKeys: function () { return Object.keys(a.h) }
      }
    }; var zp = function (a) { var b = yp[a.target.ba]; if (!a.isAborted && b) for (var c = xp(a), d = 0; d < b.length; ++d) { try { b[d](c) } catch (e) { a.isAborted = !0 } if (a.isAborted) break } }, Ap = function (a, b) { var c = yp[a]; c || (c = yp[a] = []); c.push(b) }, yp = {}; function Up(a) { var b = U(a.s, N.g.vb), c = U(a.s, N.g.Jb); b && !c ? (a.eventName !== N.g.na && a.eventName !== N.g.od && M(131), a.isAborted = !0) : !b && c && (M(132), a.isAborted = !0) }; var Wp = function () { var a = z.screen; return { width: a ? a.width : 0, height: a ? a.height : 0 } }, Xp = function (a) {
      if (C.hidden) return !0; var b = a.getBoundingClientRect(); if (b.top == b.bottom || b.left == b.right || !z.getComputedStyle) return !0; var c = z.getComputedStyle(a, null); if ("hidden" === c.visibility) return !0; for (var d = a, e = c; d;) {
        if ("none" === e.display) return !0; var f = e.opacity, g = e.filter; if (g) {
          var k = g.indexOf("opacity("); 0 <= k && (g = g.substring(k + 8, g.indexOf(")", k)), "%" == g.charAt(g.length - 1) && (g = g.substring(0, g.length - 1)), f = Math.min(g,
            f))
        } if (void 0 !== f && 0 >= f) return !0; (d = d.parentElement) && (e = z.getComputedStyle(d, null))
      } return !1
    };
    var Yp = function () { var a = C.body, b = C.documentElement || a && a.parentElement, c, d; if (C.compatMode && "BackCompat" !== C.compatMode) c = b ? b.clientHeight : 0, d = b ? b.clientWidth : 0; else { var e = function (f, g) { return f && g ? Math.min(f, g) : Math.max(f, g) }; c = e(b ? b.clientHeight : 0, a ? a.clientHeight : 0); d = e(b ? b.clientWidth : 0, a ? a.clientWidth : 0) } return { width: d, height: c } }, Zp = function (a) {
      var b = Yp(), c = b.height, d = b.width, e = a.getBoundingClientRect(), f = e.bottom - e.top, g = e.right - e.left; return f && g ? (1 - Math.min((Math.max(0 - e.left, 0) + Math.max(e.right -
        d, 0)) / g, 1)) * (1 - Math.min((Math.max(0 - e.top, 0) + Math.max(e.bottom - c, 0)) / f, 1)) : 0
    }; var $p = [], aq = !(!z.IntersectionObserver || !z.IntersectionObserverEntry), bq = function (a, b, c) { for (var d = new z.IntersectionObserver(a, { threshold: c }), e = 0; e < b.length; e++)d.observe(b[e]); for (var f = 0; f < $p.length; f++)if (!$p[f]) return $p[f] = d, f; return $p.push(d) - 1 }, cq = function (a, b, c) {
      function d(k, m) {
        var n = { top: 0, bottom: 0, right: 0, left: 0, width: 0, height: 0 }, p = {
          boundingClientRect: k.getBoundingClientRect(),
          intersectionRatio: m, intersectionRect: n, isIntersecting: 0 < m, rootBounds: n, target: k, time: Ra()
        }; F(function () { return a(p) })
      } for (var e = [], f = [], g = 0; g < b.length; g++)e.push(0), f.push(-1); c.sort(function (k, m) { return k - m }); return function () { for (var k = 0; k < b.length; k++) { var m = Zp(b[k]); if (m > e[k]) for (; f[k] < c.length - 1 && m >= c[f[k] + 1];)d(b[k], m), f[k]++; else if (m < e[k]) for (; 0 <= f[k] && m <= c[f[k]];)d(b[k], m), f[k]--; e[k] = m } }
    }, dq = function (a, b, c) {
      for (var d = 0; d < c.length; d++)1 < c[d] ? c[d] = 1 : 0 > c[d] && (c[d] = 0); if (aq) {
        var e = !1; F(function () {
          e ||
          cq(a, b, c)()
        }); return bq(function (f) { e = !0; for (var g = { Ae: 0 }; g.Ae < f.length; g = { Ae: g.Ae }, g.Ae++)F(function (k) { return function () { return a(f[k.Ae]) } }(g)) }, b, c)
      } return z.setInterval(cq(a, b, c), 1E3)
    }, eq = function (a) { aq ? 0 <= a && a < $p.length && $p[a] && ($p[a].disconnect(), $p[a] = void 0) : z.clearInterval(a) };
    var gq = function (a, b, c) { var d = a.element, e = { U: a.U, type: a.ja, tagName: d.tagName }; b && (e.querySelector = fq(d)); c && (e.isVisible = !Xp(d)); return e }, hq = function (a, b, c) { return gq({ element: a.element, U: a.U, ja: "1" }, b, c) }, iq = function (a) { var b = !!a.bd + "." + !!a.dd; a && a.oe && a.oe.length && (b += "." + a.oe.join(".")); a && a.hb && (b += "." + a.hb.email + "." + a.hb.phone + "." + a.hb.address); return b }, lq = function (a) {
      if (0 != a.length) {
        var b; b = jq(a, function (c) { return !kq.test(c.U) }); b = jq(b, function (c) { return "INPUT" === c.element.tagName.toUpperCase() });
        b = jq(b, function (c) { return !Xp(c.element) }); return b[0]
      }
    }, mq = function (a, b) { if (!b || 0 === b.length) return a; for (var c = [], d = 0; d < a.length; d++) { for (var e = !0, f = 0; f < b.length; f++) { var g = b[f]; if (g && yh(a[d].element, g)) { e = !1; break } } e && c.push(a[d]) } return c }, jq = function (a, b) { if (1 >= a.length) return a; var c = a.filter(b); return 0 == c.length ? a : c }, fq = function (a) {
      var b; if (a === C.body) b = "body"; else {
        var c; if (a.id) c = "#" + a.id; else {
          var d; if (a.parentElement) {
            var e; a: {
              var f = a.parentElement; if (f) {
                for (var g = 0; g < f.childElementCount; g++)if (f.children[g] ===
                  a) { e = g + 1; break a } e = -1
              } else e = 1
            } d = fq(a.parentElement) + ">:nth-child(" + e + ")"
          } else d = ""; c = d
        } b = c
      } return b
    }, oq = function (a) { for (var b = [], c = 0; c < a.length; c++) { var d = a[c], e = d.textContent; "INPUT" === d.tagName.toUpperCase() && d.value && (e = d.value); if (e) { var f = e.match(nq); if (f) { var g = f[0], k; if (z.location) { var m = zn(z.location, "host", !0); k = 0 <= g.toLowerCase().indexOf(m) } else k = !1; k || b.push({ element: d, U: g }) } } } return b }, sq = function () {
      var a = [], b = C.body; if (!b) return { elements: a, status: "4" }; for (var c = b.querySelectorAll("*"),
        d = 0; d < c.length && 1E4 > d; d++) { var e = c[d]; if (!(0 <= pq.indexOf(e.tagName.toUpperCase())) && e.children instanceof HTMLCollection) { for (var f = !1, g = 0; g < e.childElementCount && 1E4 > g; g++)if (!(0 <= qq.indexOf(e.children[g].tagName.toUpperCase()))) { f = !0; break } (!f || Q(54) && -1 !== rq.indexOf(e.tagName)) && a.push(e) } } return { elements: a, status: 1E4 < c.length ? "2" : "1" }
    }, tq = !0, uq = !1; var nq = /[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}/i, vq = /@(gmail|googlemail)\./i, kq = /support|noreply/i, pq = "SCRIPT STYLE IMG SVG PATH BR NOSCRIPT TEXTAREA".split(" "), qq = ["BR"], wq = { Mm: "1", Um: "2", Nm: "3", Pm: "4", Km: "5", Vm: "6", Qm: "7" }, xq = {}, rq = ["INPUT", "SELECT"]; var Qq = function (a) {
      a = a || { bd: !0, dd: !0 }; a.hb = a.hb || { email: !0, phone: !1, address: !1 }; var b = iq(a), c = xq[b]; if (c && 200 > Ra() - c.timestamp) return c.result; var d = sq(), e = d.status,
        f = [], g, k, m = []; if (!Q(54)) { if (a.hb && a.hb.email) { var n = oq(d.elements); f = mq(n, a && a.oe); g = lq(f); 10 < n.length && (e = "3") } !a.Hh && g && (f = [g]); for (var p = 0; p < f.length; p++)m.push(hq(f[p], a.bd, a.dd)); m = m.slice(0, 10) } else if (a.hb) { } g && (k = hq(g, a.bd, a.dd)); var D = { elements: m, zh: k, status: e }; xq[b] = { timestamp: Ra(), result: D }; return D
    }, Rq = function (a) { return a.tagName + ":" + a.isVisible + ":" + a.U.length + ":" + vq.test(a.U) }; function Sq() { return "attribution-reporting" } function Tq(a) { var b; b = void 0 === b ? document : b; var c; return !(null == (c = b.featurePolicy) || !c.allowedFeatures().includes(a)) }; var Uq = !1; function Vq() { if (Tq("join-ad-interest-group") && Ba(Cc.joinAdInterestGroup)) return !0; Uq || (Al('AymqwRC7u88Y4JPvfIF2F37QKylC04248hLCdJAsh8xgOfe/dVJPV3XS3wLFca1ZMVOtnBfVjaCMTVudWM//5g4AAAB7eyJvcmlnaW4iOiJodHRwczovL3d3dy5nb29nbGV0YWdtYW5hZ2VyLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjk1MTY3OTk5LCJpc1RoaXJkUGFydHkiOnRydWV9'), Uq = !0); return Tq("join-ad-interest-group") && Ba(Cc.joinAdInterestGroup) }
    function Wq(a, b) { var c = void 0; try { c = C.querySelector('iframe[data-tagging-id="' + b + '"]') } catch (e) { } if (c) { var d = Number(c.dataset.loadTime); if (d && 6E4 > Ra() - d) { yb("TAGGING", 9); return } } else try { if (50 <= C.querySelectorAll('iframe[allow="join-ad-interest-group"][data-tagging-id*="-"]').length) { yb("TAGGING", 10); return } } catch (e) { } Lc(a, void 0, { allow: "join-ad-interest-group" }, { taggingId: b, loadTime: Ra() }, c) } function Xq() { return "https://td.doubleclick.net" }; var Yq = RegExp("^UA-\\d+-\\d+%3A[\\w-]+(?:%2C[\\w-]+)*(?:%3BUA-\\d+-\\d+%3A[\\w-]+(?:%2C[\\w-]+)*)*$"), Zq = /^~?[\w-]+(?:\.~?[\w-]+)*$/, $q = /^\d+\.fls\.doubleclick\.net$/, ar = /;gac=([^;?]+)/, br = /;gacgb=([^;?]+)/, cr = /;gclaw=([^;?]+)/, dr = /;gclgb=([^;?]+)/;
    function er(a, b) { if ($q.test(C.location.host)) { var c = C.location.href.match(b); return c && 2 == c.length && c[1].match(Yq) ? decodeURIComponent(c[1]) : "" } var d = [], e; for (e in a) { for (var f = [], g = a[e], k = 0; k < g.length; k++)f.push(g[k].X); d.push(e + ":" + f.join(",")) } return 0 < d.length ? d.join(";") : "" } var fr = function (a, b, c) { var d = zo() ? vo("_gac_gb", !0) : {}, e = [], f = !1, g; for (g in d) { var k = To("_gac_gb_" + g, a, b, c); f = f || 0 !== k.length && k.some(function (m) { return 1 === m }); e.push(g + ":" + k.join(",")) } return { bl: f ? e.join(";") : "", al: er(d, br) } };
    function gr(a, b, c) { if ($q.test(C.location.host)) { var d = C.location.href.match(c); if (d && 2 == d.length && d[1].match(Zq)) return [{ X: d[1] }] } else return Bo((a || "_gcl") + b); return [] } var hr = function (a) { return gr(a, "_aw", cr).map(function (b) { return b.X }).join(".") }, ir = function (a) { return gr(a, "_gb", dr).map(function (b) { return b.X }).join(".") }, jr = function (a, b) { var c = To((b && b.prefix || "_gcl") + "_gb", a, b); return 0 === c.length || c.every(function (d) { return 0 === d }) ? "" : c.join(".") }; var kr = function () { if (Ba(z.__uspapi)) { var a = ""; try { z.__uspapi("getUSPData", 1, function (b, c) { if (c && b) { var d = b.uspString; d && RegExp("^[\\da-zA-Z-]{1,20}$").test(d) && (a = d) } }) } catch (b) { } return a } }; var Wr = { H: { Mh: "ads_conversion_hit", Pf: "container_execute_start", Oh: "container_setup_end", Qf: "container_setup_start", Nh: "container_execute_end", Ph: "container_yield_end", Rf: "container_yield_start", Ji: "event_execute_end", Ii: "event_evaluation_end", Gg: "event_evaluation_start", Ki: "event_setup_end", Yd: "event_setup_start", Li: "ga4_conversion_hit", ce: "page_load", Tm: "pageview", Qb: "snippet_load", Vi: "tag_callback_error", Wi: "tag_callback_failure", Xi: "tag_callback_success", Yi: "tag_execute_end", Vc: "tag_execute_start" } }; function Xr() { function a(c, d) { var e = Ab(d); e && b.push(c + "=" + e) } var b = []; a("&u", "GTM"); a("&ut", "TAGGING"); a("&h", "HEALTH"); return b.join("") }; var Yr = !1, Zr = "L S Y E EC TC HTC".split(" "), $r = ["S", "V", "E"], as = ["TS", "TI", "TE"];
    var Ds = function (a) { }, Es = function (a) { }, Fs = function () { }, Gs = function (a, b) { }, Hs = function (a, b) { }, Is = function (a, b) { },
      Js = function (a, b) { }, bs = function (a, b, c, d, e, f) {
        var g; g = void 0 === g ? !1 : g; var k = {};
        return k
      }, cs = function (a) {
        var b = !1;
        return b
      }, ds = function (a, b) { },
      Ks = function () { var a = {}; return a }, ws = function (a) { a = void 0 === a ? !0 : a; var b = {}; return b }, Ls = function () { }, Ms = function (a, b) { }, Ns = function (a, b, c) { }, Os = function () { var a = bs("PAGEVIEW", nk()); if (ns(a.entry, "mark")[0]) { var b = Yc(); b.clearMarks(a.entry); b.clearMeasures("GTM-" + nk() + ":" + Wr.H.ce + ":to:PAGEVIEW") } var c = bs(Wr.H.ce, nk()); cs(a) && ds(a, c) }; var Ps = function (a, b) { var c = z, d, e = c.GooglebQhCsO; e || (e = {}, c.GooglebQhCsO = e); d = e; if (d[a]) return !1; d[a] = []; d[a][0] = b; return !0 }; var Qs = function (a, b, c) { var d = vl(a, "fmt"); if (b) { var e = vl(a, "random"), f = vl(a, "label") || ""; if (!e) return !1; var g = jn(decodeURIComponent(f.replace(/\+/g, " ")) + ":" + decodeURIComponent(e.replace(/\+/g, " "))); if (!Ps(g, b)) return !1 } d && 4 != d && (a = xl(a, "rfmt", d)); var k = xl(a, "fmt", 4); Jc(k, function () { z.google_noFurtherRedirects && b && b.call && (z.google_noFurtherRedirects = null, b()) }, void 0, c, C.getElementsByTagName("script")[0].parentElement || void 0); return !0 }; var ft = function () { this.h = {} }, gt = function (a, b, c) { null != c && (a.h[b] = c) }, ht = function (a) { return Object.keys(a.h).map(function (b) { return encodeURIComponent(b) + "=" + encodeURIComponent(a.h[b]) }).join("&") }, jt = function (a, b, c, d) { }; function lt(a, b) { if (data.entities && data.entities[a]) return data.entities[a][b] }; var nt = function (a, b) { mt(a).entity.push(b) }, ot = function (a, b) { mt(a).event && mt(a).event.push(b) }, pt = function () { var a = mt(ok()); return a.event ? a.event : [] }; function mt(a) { var b, c = oi.r; c || (c = { container: {} }, oi.r = c); b = c; var d = b.container[a]; d || (d = { entity: [], event: [] }, b.container[a] = d); return d }; var qt = new RegExp(/^(.*\.)?(google|youtube|blogger|withgoogle)(\.com?)?(\.[a-z]{2})?\.?$/), rt = { cl: ["ecl"], customPixels: ["nonGooglePixels"], ecl: ["cl"], ehl: ["hl"], hl: ["ehl"], html: ["customScripts", "customPixels", "nonGooglePixels", "nonGoogleScripts", "nonGoogleIframes"], customScripts: ["html", "customPixels", "nonGooglePixels", "nonGoogleScripts", "nonGoogleIframes"], nonGooglePixels: [], nonGoogleScripts: ["nonGooglePixels"], nonGoogleIframes: ["nonGooglePixels"] }, st = {
        cl: ["ecl"], customPixels: ["customScripts", "html"],
        ecl: ["cl"], ehl: ["hl"], hl: ["ehl"], html: ["customScripts"], customScripts: ["html"], nonGooglePixels: ["customPixels", "customScripts", "html", "nonGoogleScripts", "nonGoogleIframes"], nonGoogleScripts: ["customScripts", "html"], nonGoogleIframes: ["customScripts", "html", "nonGoogleScripts"]
      }, tt = "google customPixels customScripts html nonGooglePixels nonGoogleScripts nonGoogleIframes".split(" "), wt = function (a) {
        var b = Ui("gtm.allowlist") || Ui("gtm.whitelist"); b && M(9); ui && (b = ["google", "gtagfl", "lcl", "zone"]); ut() && (ui ?
          M(116) : M(117), vt && (b = [], window.console && window.console.log && window.console.log("GTM blocked. See go/13687728."))); var c = b && Xa(Na(b), rt), d = Ui("gtm.blocklist") || Ui("gtm.blacklist"); d || (d = Ui("tagTypeBlacklist")) && M(3); d ? M(8) : d = []; ut() && (d = Na(d), d.push("nonGooglePixels", "nonGoogleScripts", "sandboxedScripts")); 0 <= Na(d).indexOf("google") && M(2); var e = d && Xa(Na(d), st), f = {}; return function (g) {
            var k = g && g[Qe.xa]; if (!k || "string" != typeof k) return !0; k = k.replace(/^_*/, ""); if (void 0 !== f[k]) return f[k]; var m = Gi[k] ||
              [], n = a(k, m), p; p = mt(ok()).entity; for (var q = 0; q < p.length; q++)try { n = n && p[q](k, m) } catch (y) { n = !1 } if (b) { var r; if (r = n) a: { if (0 > c.indexOf(k)) if (m && 0 < m.length) for (var t = 0; t < m.length; t++) { if (0 > c.indexOf(m[t])) { M(11); r = !1; break a } } else { r = !1; break a } r = !0 } n = r } var u = !1; if (d) { var v = 0 <= e.indexOf(k); if (v) u = v; else { var w = Ja(e, m || []); w && M(10); u = w } } var x = !n || u; x || !(0 <= m.indexOf("sandboxedScripts")) || c && -1 !== c.indexOf("sandboxedScripts") || (x = Ja(e, tt)); return f[k] = x
          }
      }, vt = !1; var ut = function () { return qt.test(z.location && z.location.hostname) }, xt = function () { hk && nt(ok(), function (a) { var b = Cf(a), c; if (Hf(b)) { var d = b[Qe.xa]; if (!d) throw "Error: No function name given for function call."; var e = vf[d]; c = !!e && !!e.runInSiloedMode } else c = !!lt(b[Qe.xa], 4); return c }) }; function zt(a, b) { if (a) { var c = "" + a; 0 !== c.indexOf("http://") && 0 !== c.indexOf("https://") && (c = "https://" + c); "/" === c[c.length - 1] && (c = c.substring(0, c.length - 1)); return Cn("" + c + b).href } } function At() { return !!ni.nf && "SGTM_TOKEN" !== ni.nf.split("@@").join("") } function Bt(a) { for (var b = fa([N.g.Ld, N.g.Nb]), c = b.next(); !c.done; c = b.next()) { var d = U(a, c.value); if (d) return d } }; var Dt = function (a, b, c, d, e) {
        if (!Ct() && !uk(a)) {
          var f = "?id=" + encodeURIComponent(a) + "&l=" + ni.ia, g = 0 === a.indexOf("GTM-"); g || (f += "&cx=c"); Q(67) && (f += "&gtm=" + cn()); var k = At(); k && (f += "&sign=" + ni.nf); var m = c ? "/gtag/js" : "/gtm.js", n = wi || yi ? zt(b, m + f) : void 0; if (!n) { var p = ni.Oe + m; k && Dc && g && (p = Dc.replace(/^(?:https?:\/\/)?/i, "").split(/[?#]/)[0]); n = fp("https://", "http://", p + f) } var q = a; d.siloed && (wk({ ctid: q, isDestination: !1 }), q = ik(q)); var r = q, t = vk(); dk().container[r] = { state: 1, context: d, parent: t }; ek({ ctid: r, isDestination: !1 },
            e); Jc(n)
        }
      }, Et = function (a, b, c) {
        var d; if (d = !Ct()) { var e = dk().destination[a]; d = !(e && e.state) } if (d) if (xk()) dk().destination[a] = { state: 0, transportUrl: b, context: c, parent: vk() }, ek({ ctid: a, isDestination: !0 }), M(91); else {
          var f = "/gtag/destination?id=" + encodeURIComponent(a) + "&l=" + ni.ia + "&cx=c"; Q(67) && (f += "&gtm=" + cn()); At() && (f += "&sign=" + ni.nf); var g = wi || yi ? zt(b, f) : void 0; g || (g = fp("https://", "http://", ni.Oe + f)); var k = a; c.siloed && (wk({ ctid: k, isDestination: !0 }), k = ik(k)); dk().destination[k] = { state: 1, context: c, parent: vk() };
          ek({ ctid: k, isDestination: !0 }); Jc(g)
        }
      }; function Ct() { if (an()) { return !0 } return !1 }; var Ft = "", Gt = []; function Ht(a) { var b = ""; Ft && (b = "&dl=" + encodeURIComponent(Ft)); 0 < Gt.length && (b += "&tdp=" + Gt.join(".")); a.Yb && (Ft = "", Gt.length = 0, b && a.Cj()); return b }; var It = []; function Jt(a) { if (!It.length) return ""; var b = "&tdc=" + It.join("!"); a.Yb && (a.Cj(), It.length = 0); return b }; var Kt = { initialized: 11, complete: 12, interactive: 13 }, Lt = {}, Mt = Object.freeze((Lt[N.g.Ma] = !0, Lt)), Nt = 0 <= C.location.search.indexOf("?gtm_diagnostics=") || 0 <= C.location.search.indexOf("&gtm_diagnostics="), Pt = function (a, b, c) {
        if (Hk && "config" === a && !(1 < cp(b).O.length)) {
          var d, e = Ec("google_tag_data", {}); e.td || (e.td = {}); d = e.td; var f = lb(c.F); lb(c.h, f); var g = [], k; for (k in d) { var m = Ot(d[k], f); m.length && (Nt && console.log(m), g.push(k)) } g.length && (g.length && Hk && It.push(b + "*" + g.join(".")), yb("TAGGING", Kt[C.readyState] ||
            14)); d[b] = f
        }
      }; function Qt(a, b) { var c = {}, d; for (d in b) b.hasOwnProperty(d) && (c[d] = !0); for (var e in a) a.hasOwnProperty(e) && (c[e] = !0); return c } function Ot(a, b, c, d) { c = void 0 === c ? {} : c; d = void 0 === d ? "" : d; if (a === b) return []; var e = function (q, r) { var t = r[q]; return void 0 === t ? Mt[q] : t }, f; for (f in Qt(a, b)) { var g = (d ? d + "." : "") + f, k = e(f, a), m = e(f, b), n = "object" === ib(k) || "array" === ib(k), p = "object" === ib(m) || "array" === ib(m); if (n && p) Ot(k, m, c, g); else if (n || p || k !== m) c[g] = !0 } return Object.keys(c) }; var Rt = !1, St = 0, Tt = []; function Ut(a) { if (!Rt) { var b = C.createEventObject, c = "complete" == C.readyState, d = "interactive" == C.readyState; if (!a || "readystatechange" != a.type || c || !b && d) { Rt = !0; for (var e = 0; e < Tt.length; e++)F(Tt[e]) } Tt.push = function () { for (var f = 0; f < arguments.length; f++)F(arguments[f]); return 0 } } } function Vt() { if (!Rt && 140 > St) { St++; try { C.documentElement.doScroll("left"), Ut() } catch (a) { z.setTimeout(Vt, 50) } } } var Wt = function (a) { Rt ? a() : Tt.push(a) }; var Xt = function () { this.F = 0; this.h = {} }; Xt.prototype.addListener = function (a, b, c) { var d = ++this.F; this.h[a] = this.h[a] || {}; this.h[a][String(d)] = { listener: b, jb: c }; return d }; Xt.prototype.B = function (a, b) { var c = this.h[a], d = String(b); if (!c || !c[d]) return !1; delete c[d]; return !0 }; Xt.prototype.C = function (a, b) { var c = []; l(this.h[a], function (d, e) { 0 > c.indexOf(e.listener) && (void 0 === e.jb || 0 <= b.indexOf(e.jb)) && c.push(e.listener) }); return c }; var Yt = function (a, b, c) { return { entityType: a, indexInOriginContainer: b, nameInOriginContainer: c, originContainerId: nk() } }; var $t = function (a, b) { this.h = !1; this.F = []; this.K = { tags: [] }; this.W = !1; this.B = this.C = 0; Zt(this, a, b) }, au = function (a, b, c, d) { if (ri.hasOwnProperty(b) || "__zone" === b) return -1; var e = {}; kb(d) && (e = lb(d, e)); e.id = c; e.status = "timeout"; return a.K.tags.push(e) - 1 }, bu = function (a, b, c, d) { var e = a.K.tags[b]; e && (e.status = c, e.executionTime = d) }, cu = function (a) { if (!a.h) { for (var b = a.F, c = 0; c < b.length; c++)b[c](); a.h = !0; a.F.length = 0 } }, Zt = function (a, b, c) { void 0 !== b && a.qf(b); c && z.setTimeout(function () { return cu(a) }, Number(c)) };
    $t.prototype.qf = function (a) { var b = this, c = Ta(function () { return F(function () { a(nk(), b.K) }) }); this.h ? c() : this.F.push(c) }; var du = function (a) { a.C++; return Ta(function () { a.B++; a.W && a.B >= a.C && cu(a) }) }, eu = function (a) { a.W = !0; a.B >= a.C && cu(a) }; var fu = {}, hu = function () { return z[gu()] }, iu = !1;
    function gu() { return z.GoogleAnalyticsObject || "ga" }
    var lu = function (a) { }, mu = function (a, b) {
      return function () {
        var c = hu(), d = c && c.getByName && c.getByName(a); if (d) {
          var e = d.get("sendHitTask"); d.set("sendHitTask", function (f) {
            var g = f.get("hitPayload"), k = f.get("hitCallback"), m = 0 > g.indexOf("&tid=" + b); m && (f.set("hitPayload", g.replace(/&tid=UA-[0-9]+-[0-9]+/, "&tid=" + b), !0), f.set("hitCallback", void 0, !0)); e(f); m && (f.set("hitPayload",
              g, !0), f.set("hitCallback", k, !0), f.set("_x_19", void 0, !0), e(f))
          })
        }
      }
    }; var ru = {}, su = {}; function tu(a, b) { if (Hk) { var c; c = b.match(/^(gtm|gtag)\./) ? encodeURIComponent(b) : "*"; ru[a] = "&e=" + c + "&eid=" + a; Rk(a) } } function uu(a) { var b = a.eventId, c = a.Yb; if (!ru[b]) return ""; var d = su[b] ? "" : "&es=1"; d += ru[b]; c && (su[b] = !0); return d }; var vu = {}; function wu(a, b) { Hk && (vu[a] = vu[a] || {}, vu[a][b] = (vu[a][b] || 0) + 1) } function xu(a) { var b = a.eventId, c = a.Yb, d = vu[b] || {}, e = [], f; for (f in d) d.hasOwnProperty(f) && e.push("" + f + d[f]); c && delete vu[b]; return e.length ? "&md=" + e.join(".") : "" }; var yu = {}, zu = { aev: "1", c: "2", jsm: "3", v: "4", j: "5", smm: "6", rmm: "7", input: "8" };
    function Au(a, b, c) { if (Hk) { yu[a] = yu[a] || []; var d = zu[b] || "0", e; e = c instanceof z.Element ? "1" : c instanceof z.Event ? "2" : c instanceof z.RegExp ? "3" : c === z ? "4" : c === C ? "5" : c instanceof z.Promise ? "6" : c instanceof z.Storage ? "7" : c instanceof z.Date ? "8" : c instanceof z.History ? "9" : c instanceof z.Performance ? "a" : c === z.crypto ? "b" : c instanceof z.Location ? "c" : c instanceof z.Navigator ? "d" : "object" !== typeof c || kb(c) ? "0" : "e"; yu[a].push("" + d + e) } }
    function Bu(a) { var b = a.eventId, c = yu[b] || []; if (!c.length) return ""; a.Yb && delete yu[b]; return "&pcr=" + c.join(".") }; var Cu = {}, Du = {}; function Eu(a, b, c) { if (Hk && b) { var d = bk(b); Cu[a] = Cu[a] || []; Cu[a].push(c + d); var e = (Hf(b) ? "1" : "2") + d; Du[a] = Du[a] || []; Du[a].push(e); Rk(a) } } function Fu(a) { var b = a.eventId, c = a.Yb, d = "", e = Cu[b] || []; e.length && (d += "&tr=" + e.join(".")); var f = Du[b] || []; f.length && (d += "&ti=" + f.join(".")); c && (delete Cu[b], delete Du[b]); return d }; function Gu(a, b, c, d) { var e = tf[a], f = Hu(a, b, c, d); if (!f) return null; var g = Ef(e[Qe.Ui], c, []); if (g && g.length) { var k = g[0]; f = Gu(k.index, { onSuccess: f, onFailure: 1 === k.kj ? b.terminate : f, terminate: b.terminate }, c, d) } return f }
    function Hu(a, b, c, d) {
      function e() {
        if (f[Qe.zk]) k(); else {
          var w = Ff(f, c, []), x = w[Qe.Pj]; if (null != x) for (var y = 0; y < x.length; y++)if (!S(x[y])) { k(); return } var A = au(c.Rb, String(f[Qe.xa]), Number(f[Qe.he]), w[Qe.Ak]), B = !1; w.vtp_gtmOnSuccess = function () { if (!B) { B = !0; var E = Ra() - G; Eu(c.id, tf[a], "5"); bu(c.Rb, A, "success", E); Q(29) && Ns(c, f, Wr.H.Xi); g() } }; w.vtp_gtmOnFailure = function () { if (!B) { B = !0; var E = Ra() - G; Eu(c.id, tf[a], "6"); bu(c.Rb, A, "failure", E); Q(29) && Ns(c, f, Wr.H.Wi); k() } }; w.vtp_gtmTagId = f.tag_id; w.vtp_gtmEventId =
            c.id; c.priorityId && (w.vtp_gtmPriorityId = c.priorityId); Eu(c.id, f, "1"); var D = function () { ij(3); var E = Ra() - G; Eu(c.id, f, "7"); bu(c.Rb, A, "exception", E); Q(29) && Ns(c, f, Wr.H.Vi); B || (B = !0, k()) }; Q(29) && Ms(c, f); var G = Ra(); try { Df(w, { event: c, index: a, type: 1 }) } catch (E) { D(E) } Q(29) && Ns(c, f, Wr.H.Yi)
        }
      } var f = tf[a], g = b.onSuccess, k = b.onFailure, m = b.terminate; if (c.ih(f)) return null; var n = Ef(f[Qe.Zi], c, []); if (n && n.length) {
        var p = n[0], q = Gu(p.index, { onSuccess: g, onFailure: k, terminate: m }, c, d); if (!q) return null; g = q; k = 2 === p.kj ? m :
          q
      } if (f[Qe.Qi] || f[Qe.Ck]) { var r = f[Qe.Qi] ? uf : c.Am, t = g, u = k; if (!r[a]) { e = Ta(e); var v = Iu(a, r, e); g = v.onSuccess; k = v.onFailure } return function () { r[a](t, u) } } return e
    } function Iu(a, b, c) { var d = [], e = []; b[a] = Ju(d, e, c); return { onSuccess: function () { b[a] = Ku; for (var f = 0; f < d.length; f++)d[f]() }, onFailure: function () { b[a] = Lu; for (var f = 0; f < e.length; f++)e[f]() } } } function Ju(a, b, c) { return function (d, e) { a.push(d); b.push(e); c() } } function Ku(a) { a() } function Lu(a, b) { b() }; var Nu = function (a, b) { return 1 === arguments.length ? Mu("set", a) : Mu("set", a, b) }, Ou = function (a, b) { return 1 === arguments.length ? Mu("config", a) : Mu("config", a, b) }, Pu = function (a, b, c) { c = c || {}; c[N.g.Mb] = a; return Mu("event", b, c) }; function Mu(a) { return arguments } var Qu = function () { this.h = []; this.B = [] }; Qu.prototype.enqueue = function (a, b, c) { var d = this.h.length + 1; a["gtm.uniqueEventId"] = b; a["gtm.priorityId"] = d; c.eventId = b; c.fromContainerExecution = !0; c.priorityId = d; var e = { message: a, notBeforeEventId: b, priorityId: d, messageContext: c }; this.h.push(e); for (var f = 0; f < this.B.length; f++)try { this.B[f](e) } catch (g) { } }; Qu.prototype.listen = function (a) { this.B.push(a) };
    Qu.prototype.get = function () { for (var a = {}, b = 0; b < this.h.length; b++) { var c = this.h[b], d = a[c.notBeforeEventId]; d || (d = [], a[c.notBeforeEventId] = d); d.push(c) } return a }; Qu.prototype.prune = function (a) { for (var b = [], c = [], d = 0; d < this.h.length; d++) { var e = this.h[d]; e.notBeforeEventId === a ? b.push(e) : c.push(e) } this.h = c; return b }; var Su = function (a, b, c) { Ru().enqueue(a, b, c) }, Uu = function () { var a = Tu; Ru().listen(a) }; function Ru() { var a = oi.mb; a || (a = new Qu, oi.mb = a); return a } var bv = function (a) { var b = oi.zones; return b ? b.getIsAllowedFn(jk(), a) : function () { return !0 } }, cv = function () { ot(ok(), function (a, b) { var c = oi.zones; return c ? c.isActive(jk(), b) : !0 }) }; var fv = function (a, b) { for (var c = [], d = 0; d < tf.length; d++)if (a[d]) { var e = tf[d]; var f = du(b.Rb); try { var g = Gu(d, { onSuccess: f, onFailure: f, terminate: f }, b, d); if (g) { var k = e[Qe.xa]; if (!k) throw "Error: No function name given for function call."; var m = vf[k]; c.push({ Ej: d, xj: (m ? m.priorityOverride || 0 : 0) || lt(e[Qe.xa], 1) || 0, execute: g }) } else dv(d, b), f() } catch (p) { f() } } c.sort(ev); for (var n = 0; n < c.length; n++)c[n].execute(); return 0 < c.length };
    var Cv = function (a, b) { if (!gv) return !1; var c = a["gtm.triggers"] && String(a["gtm.triggers"]), d = gv.C(a.event, c ? String(c).split(",") : []); if (!d.length) return !1; for (var e = 0; e < d.length; ++e) { var f = du(b); try { d[e](a, f) } catch (g) { f() } } return !0 }; function ev(a, b) { var c, d = b.xj, e = a.xj; c = d > e ? 1 : d < e ? -1 : 0; var f; if (0 !== c) f = c; else { var g = a.Ej, k = b.Ej; f = g > k ? 1 : g < k ? -1 : 0 } return f }
    function dv(a, b) { if (Hk) { var c = function (d) { var e = b.ih(tf[d]) ? "3" : "4", f = Ef(tf[d][Qe.Ui], b, []); f && f.length && c(f[0].index); Eu(b.id, tf[d], e); var g = Ef(tf[d][Qe.Zi], b, []); g && g.length && c(g[0].index) }; c(a) } } var Iv = !1, gv; var Jv = function () { gv || (gv = new Xt); return gv };
    var Ov = function (a) {
      var b = a["gtm.uniqueEventId"], c = a["gtm.priorityId"], d = a.event; Q(29) && Gs(b, d); if ("gtm.js" === d) { if (Iv) return !1; Iv = !0 } var e, f = !1; if (pt().every(function (r) { return r(d, b) })) e = bv(b); else { if ("gtm.js" !== d && "gtm.init" !== d && "gtm.init_consent" !== d) return !1; f = !0; e = bv(Number.MAX_SAFE_INTEGER) } tu(b, d); var g = a.eventCallback, k = a.eventTimeout, m = { id: b, priorityId: c, name: d, ih: wt(e), Am: [], sj: function () { M(6); ij(0) }, ej: Kv(), fj: Lv(b), Rb: new $t(function () { if (Q(29)) { } g && g.apply(g, [].slice.call(arguments, 0)) }, k) }; Q(57) && (m.Dh = wu); Q(29) && Is(m.id, m.name); var n = Pf(m); Q(29) && Js(m.id, m.name);
      f && (n = Mv(n)); Q(29) && Hs(b, d); var p = fv(n, m), q = !1; q = Cv(a, m.Rb); eu(m.Rb); "gtm.js" !== d && "gtm.sync" !== d || lu(nk()); return Nv(n, p) || q
    }; function Lv(a) { return function (b) { nb(b) || Au(a, "input", b) } }
    function Kv() { var a = {}; a.event = Zi("event", 1); a.ecommerce = Zi("ecommerce", 1); a.gtm = Zi("gtm"); a.eventModel = Zi("eventModel"); return a } function Mv(a) { for (var b = [], c = 0; c < a.length; c++)if (a[c]) { var d = String(tf[c][Qe.xa]); if (qi[d] || void 0 !== tf[c][Qe.Dk] || Hi[d] || lt(d, 2)) b[c] = !0 } return b } function Nv(a, b) { if (!b) return b; for (var c = 0; c < a.length; c++)if (a[c] && tf[c] && !ri[String(tf[c][Qe.xa])]) return !0; return !1 } var Pv = {}; function Qv(a, b, c) { Hk && void 0 !== a && (Pv[a] = Pv[a] || [], Pv[a].push(c + b), Rk(a)) } function Rv(a) { var b = a.eventId, c = a.Yb, d = "", e = Pv[b] || []; e.length && (d += "&epr=" + e.join(".")); c && delete Pv[b]; return d }; var Tv = function (a, b) { var c = cp(mk(a), !0); c && Sv.register(c, b) }, Uv = function (a, b, c, d) { var e = cp(c, d.isGtmEvent); e && Sv.push("event", [b, a], e, d) }, Vv = function (a, b, c, d) { var e = cp(c, d.isGtmEvent); e && Sv.push("get", [a, b], e, d) }, Xv = function (a) { var b = cp(mk(a), !0), c; b ? c = Wv(Sv, b).h : c = {}; return c }, Yv = function (a, b) { var c = cp(mk(a), !0); if (c) { var d = Sv, e = lb(b); lb(Wv(d, c).h, e); Wv(d, c).h = e } }, Zv = function () { this.status = 1; this.K = {}; this.h = {}; this.B = {}; this.W = null; this.F = {}; this.C = !1 }, $v = function (a, b, c, d) {
      var e = Ra(); this.type =
        a; this.C = e; this.h = b; this.B = c; this.messageContext = d
    }, aw = function () { this.B = {}; this.C = {}; this.h = [] }, Wv = function (a, b) { var c = b.ba; return a.B[c] = a.B[c] || new Zv }, bw = function (a, b, c, d) {
      if (d.h) {
        var e = Wv(a, d.h), f = e.W; if (f) {
          var g = lb(c), k = lb(e.K[d.h.id]), m = lb(e.F), n = lb(e.h), p = lb(a.C), q = {}; if (Hk) try { q = lb(Ri) } catch (v) { M(72) } var r = d.h.prefix, t = function (v) { Qv(d.messageContext.eventId, r, v) }, u = kl(jl(il(hl(gl(el(dl(fl(cl(bl(al(new $k(d.messageContext.eventId, d.messageContext.priorityId), g), k), m), n), p), q), d.messageContext.eventMetadata),
            function () { if (t) { var v = t; t = void 0; v("2"); if (d.messageContext.onSuccess) d.messageContext.onSuccess() } }), function () { if (t) { var v = t; t = void 0; v("3"); if (d.messageContext.onFailure) d.messageContext.onFailure() } }), !!d.messageContext.isGtmEvent)); try { Qv(d.messageContext.eventId, r, "1"), Pt(d.type, d.h.id, u), f(d.h.id, b, d.C, u) } catch (v) { Qv(d.messageContext.eventId, r, "4") }
        }
      }
    }; aw.prototype.register = function (a, b, c) { var d = Wv(this, a); 3 !== d.status && (d.W = b, d.status = 3, c && (lb(d.h, c), d.h = c), this.flush()) };
    aw.prototype.push = function (a, b, c, d) { void 0 !== c && (1 === Wv(this, c).status && (Wv(this, c).status = 2, this.push("require", [{}], c, {})), Wv(this, c).C && (d.deferrable = !1)); this.h.push(new $v(a, c, b, d)); d.deferrable || this.flush() };
    aw.prototype.flush = function (a) {
      for (var b = this, c = [], d = !1, e = {}; this.h.length;) {
        e = { Sb: e.Sb, wf: e.wf }; var f = this.h[0], g = f.h; if (f.messageContext.deferrable) !g || Wv(this, g).C ? (f.messageContext.deferrable = !1, this.h.push(f)) : c.push(f), this.h.shift(); else {
          switch (f.type) {
            case "require": if (3 !== Wv(this, g).status && !a) { this.h.push.apply(this.h, c); return } break; case "set": l(f.B[0], function (r, t) { lb(Za(r, t), b.C) }); break; case "config": var k = Wv(this, g); e.Sb = {}; l(f.B[0], function (r) { return function (t, u) { lb(Za(t, u), r.Sb) } }(e));
              var m = !!e.Sb[N.g.Ob]; delete e.Sb[N.g.Ob]; var n = g.ba === g.id; m || (n ? k.F = {} : k.K[g.id] = {}); k.C && m || bw(this, N.g.na, e.Sb, f); k.C = !0; n ? lb(e.Sb, k.F) : (lb(e.Sb, k.K[g.id]), M(70)); d = !0; break; case "event": e.wf = {}; l(f.B[0], function (r) { return function (t, u) { lb(Za(t, u), r.wf) } }(e)); bw(this, f.B[1], e.wf, f); break; case "get": var p = {}, q = (p[N.g.cb] = f.B[0], p[N.g.tb] = f.B[1], p); bw(this, N.g.Ja, q, f)
          }this.h.shift(); cw(this, f)
        }
      } this.h.push.apply(this.h, c); d && this.flush()
    };
    var cw = function (a, b) { if ("require" !== b.type) if (b.h) for (var c = Wv(a, b.h).B[b.type] || [], d = 0; d < c.length; d++)c[d](); else for (var e in a.B) if (a.B.hasOwnProperty(e)) { var f = a.B[e]; if (f && f.B) for (var g = f.B[b.type] || [], k = 0; k < g.length; k++)g[k]() } }, Sv = new aw; var Yf; var dw = {}, ew = {}, fw = function (a, b) { for (var c = [], d = [], e = {}, f = 0; f < a.length; e = { Ef: e.Ef, zf: e.zf }, f++) { var g = a[f]; if (0 <= g.indexOf("-")) { if (e.Ef = cp(g, b), e.Ef) { var k = lk(); Fa(k, function (r) { return function (t) { return r.Ef.ba === t } }(e)) ? c.push(g) : d.push(g) } } else { var m = dw[g] || []; e.zf = {}; m.forEach(function (r) { return function (t) { return r.zf[t] = !0 } }(e)); for (var n = jk(), p = 0; p < n.length; p++)if (e.zf[n[p]]) { c = c.concat(lk()); break } var q = ew[g] || []; q.length && (c = c.concat(q)) } } return { Tl: c, Vl: d } }, gw = function (a) {
      l(dw, function (b,
        c) { var d = c.indexOf(a); 0 <= d && c.splice(d, 1) })
    }, hw = function (a) { l(ew, function (b, c) { var d = c.indexOf(a); 0 <= d && c.splice(d, 1) }) }; var iw = "HA GF G UA AW DC MC".split(" "), jw = !1, kw = !1; function lw(a, b) { a.hasOwnProperty("gtm.uniqueEventId") || Object.defineProperty(a, "gtm.uniqueEventId", { value: Ii() }); b.eventId = a["gtm.uniqueEventId"]; b.priorityId = a["gtm.priorityId"]; return { eventId: b.eventId, priorityId: b.priorityId } } var mw = void 0, nw = void 0; function ow(a, b, c) { var d = lb(a); d.eventId = void 0; d.inheritParentConfig = void 0; Object.keys(b).some(function (f) { return void 0 !== b[f] }) && M(136); var e = lb(b); lb(c, e); Su(Ou(jk()[0], e), a.eventId, d) }
    function pw(a) { for (var b = fa([N.g.Ld, N.g.Nb]), c = b.next(); !c.done; c = b.next()) { var d = c.value, e = a && a[d] || Sv.C[d]; if (e) return e } }
    var qw = {
      config: function (a, b) {
        var c = Q(58), d = lw(a, b); if (!(2 > a.length) && h(a[1])) {
          var e = {}; if (2 < a.length) { if (void 0 != a[2] && !kb(a[2]) || 3 < a.length) return; e = a[2] } var f = cp(a[1], b.isGtmEvent); if (f) {
            var g, k, m; a: { if (!gk.hf) { var n = pk(vk()); if (zk(n)) { var p = n.parent, q = p.isDestination; m = { Yl: pk(p), Sl: q }; break a } } m = void 0 } var r = m; r && (g = r.Yl, k = r.Sl); tu(d.eventId, "gtag.config"); var t = f.ba, u = f.id !== t; if (u ? -1 === lk().indexOf(t) : -1 === jk().indexOf(t)) {
              if (!(c && b.inheritParentConfig || Q(25) && e[N.g.vb])) {
                var v = pw(e); if (u) Et(t,
                  v, { source: 2, fromContainerExecution: b.fromContainerExecution }); else if (c && void 0 !== g && -1 !== g.containers.indexOf(t)) { var w = e; mw ? ow(b, w, mw) : nw || (nw = lb(w)) } else Dt(t, v, !0, { source: 2, fromContainerExecution: b.fromContainerExecution })
              }
            } else {
              if (g && (M(128), k && M(130), c && b.inheritParentConfig)) { var x = e; nw ? ow(b, nw, x) : !x[N.g.Ob] && ti && mw || (mw = lb(x)); return } if (ti && !u && !e[N.g.Ob]) { var y = kw; kw = !0; if (y) return } jw || M(43); if (!b.noTargetGroup) if (u) {
                hw(f.id); var A = f.id, B = e[N.g.Hd] || "default"; B = String(B).split(","); for (var D =
                  0; D < B.length; D++) { var G = ew[B[D]] || []; ew[B[D]] = G; 0 > G.indexOf(A) && G.push(A) }
              } else { gw(f.id); var E = f.id, I = e[N.g.Hd] || "default"; I = I.toString().split(","); for (var O = 0; O < I.length; O++) { var R = dw[I[O]] || []; dw[I[O]] = R; 0 > R.indexOf(E) && R.push(E) } } delete e[N.g.Hd]; var W = b.eventMetadata || {}; W.hasOwnProperty("is_external_event") || (W.is_external_event = !b.fromContainerExecution); b.eventMetadata = W; delete e[N.g.Nc]; for (var ha = u ? [f.id] : lk(), X = 0; X < ha.length; X++) {
                var T = e, ca = lb(b), ja = cp(ha[X], ca.isGtmEvent); ja && Sv.push("config",
                  [T], ja, ca)
              }
            }
          }
        }
      }, consent: function (a, b) { if (3 === a.length) { M(39); var c = lw(a, b), d = a[1]; "default" === d ? Tj(a[2]) : "update" === d ? Uj(a[2], c) : "declare" === d ? b.fromContainerExecution && Sj(a[2]) : "core_platform_services" === d && Vj(a[2]) } }, event: function (a, b) {
        var c = a[1]; if (!(2 > a.length) && h(c)) {
          var d; if (2 < a.length) { if (!kb(a[2]) && void 0 != a[2] || 3 < a.length) return; d = a[2] } var e = d, f = {}, g = (f.event = c, f); e && (g.eventModel = lb(e), e[N.g.Nc] && (g.eventCallback = e[N.g.Nc]), e[N.g.Cd] && (g.eventTimeout = e[N.g.Cd])); var k = lw(a, b), m = k.eventId,
            n = k.priorityId; g["gtm.uniqueEventId"] = m; n && (g["gtm.priorityId"] = n); if ("optimize.callback" === c) return g.eventModel = g.eventModel || {}, g; var p; var q = d, r = q && q[N.g.Mb]; void 0 === r && (r = Ui(N.g.Mb, 2), void 0 === r && (r = "default")); if (h(r) || Da(r)) {
              var t; b.isGtmEvent ? t = h(r) ? [r] : r : t = r.toString().replace(/\s+/g, "").split(","); var u = fw(t, b.isGtmEvent), v = u.Tl, w = u.Vl; if (w.length) for (var x = pw(q), y = 0; y < w.length; y++) { var A = cp(w[y], b.isGtmEvent); A && Et(A.ba, x, { source: 3, fromContainerExecution: b.fromContainerExecution }) } p = ep(v,
                b.isGtmEvent)
            } else p = void 0; var B = p; if (B) {
              tu(m, c); for (var D = [], G = 0; G < B.length; G++) { var E = B[G], I = lb(b); if (-1 !== iw.indexOf(qk(E.prefix))) { var O = lb(d), R = I.eventMetadata || {}; R.hasOwnProperty("is_external_event") || (R.is_external_event = !I.fromContainerExecution); I.eventMetadata = R; delete O[N.g.Nc]; Uv(c, O, E.id, I) } D.push(E.id) } g.eventModel = g.eventModel || {}; 0 < B.length ? g.eventModel[N.g.Mb] = D.join() : delete g.eventModel[N.g.Mb]; jw || M(43); void 0 === b.noGtmEvent && b.eventMetadata && b.eventMetadata.syn_or_mod && (b.noGtmEvent =
                !0); g.eventModel[N.g.Jb] && (b.noGtmEvent = !0); return b.noGtmEvent ? void 0 : g
            }
        }
      }, get: function (a, b) { M(53); if (4 === a.length && h(a[1]) && h(a[2]) && Ba(a[3])) { var c = cp(a[1], b.isGtmEvent), d = String(a[2]), e = a[3]; if (c) { jw || M(43); var f = pw(); if (!Fa(lk(), function (k) { return c.ba === k })) Et(c.ba, f, { source: 4, fromContainerExecution: b.fromContainerExecution }); else if (-1 !== iw.indexOf(qk(c.prefix))) { lw(a, b); var g = {}; Pj(lb((g[N.g.cb] = d, g[N.g.tb] = e, g))); Vv(d, function (k) { F(function () { return e(k) }) }, c.id, b) } } } }, js: function (a, b) {
        if (2 ==
          a.length && a[1].getTime) { jw = !0; var c = lw(a, b), d = c.eventId, e = c.priorityId, f = {}; return f.event = "gtm.js", f["gtm.start"] = a[1].getTime(), f["gtm.uniqueEventId"] = d, f["gtm.priorityId"] = e, f }
      }, policy: function (a) {
        if (3 === a.length && h(a[1]) && Ba(a[2])) { var b = a[1], c = a[2], d = Yf.B; d.h[b] ? d.h[b].push(c) : d.h[b] = [c]; if (M(74), "all" === a[1]) { M(75); var e = !1; try { e = a[2](nk(), "unknown", {}) } catch (f) { } e || M(76) } } else {
          M(73);
        }
      }, set: function (a, b) { var c; 2 == a.length && kb(a[1]) ? c = lb(a[1]) : 3 == a.length && h(a[1]) && (c = {}, kb(a[2]) || Da(a[2]) ? c[a[1]] = lb(a[2]) : c[a[1]] = a[2]); if (c) { var d = lw(a, b), e = d.eventId, f = d.priorityId; lb(c); var g = lb(c); Sv.push("set", [g], void 0, b); c["gtm.uniqueEventId"] = e; f && (c["gtm.priorityId"] = f); Q(13) && delete c.event; b.overwriteModelFields = !0; return c } }
    }, rw = { policy: !0 }; var sw = function (a) { var b = z[ni.ia].hide; if (b && void 0 !== b[a] && b.end) { b[a] = !1; var c = !0, d; for (d in b) if (b.hasOwnProperty(d) && !0 === b[d]) { c = !1; break } c && (b.end(), b.end = null) } }, tw = function (a) { var b = z[ni.ia], c = b && b.hide; c && c.end && (c[a] = !0) }; var uw = !1, vw = []; function ww() { if (!uw) { uw = !0; for (var a = 0; a < vw.length; a++)F(vw[a]) } } var xw = function (a) { uw ? F(a) : vw.push(a) }; var Ow = function (a) { if (Nw(a)) return a; this.h = a }; Ow.prototype.getUntrustedMessageValue = function () { return this.h }; var Nw = function (a) { return !a || "object" !== ib(a) || kb(a) ? !1 : "getUntrustedMessageValue" in a }; Ow.prototype.getUntrustedMessageValue = Ow.prototype.getUntrustedMessageValue; var Pw = 0, Qw = {}, Rw = [], Sw = [], Tw = !1, Uw = !1; function Vw(a, b) { return a.messageContext.eventId - b.messageContext.eventId || a.messageContext.priorityId - b.messageContext.priorityId }
    var Ww = function (a) { return z[ni.ia].push(a) }, Xw = function (a, b, c) { a.eventCallback = b; c && (a.eventTimeout = c); return Ww(a) }, Yw = function (a, b) { if (!Ca(b) || 0 > b) b = 0; var c = oi[ni.ia], d = 0, e = !1, f = void 0; f = z.setTimeout(function () { e || (e = !0, a()); f = void 0 }, b); return function () { var g = c ? c.subscribers : 1; ++d === g && (f && (z.clearTimeout(f), f = void 0), e || (a(), e = !0)) } };
    function Zw(a, b) { var c = a._clear || b.overwriteModelFields; l(a, function (e, f) { "_clear" !== e && (c && Xi(e), Xi(e, f)) }); Di || (Di = a["gtm.start"]); var d = a["gtm.uniqueEventId"]; if (!a.event) return !1; "number" !== typeof d && (d = Ii(), a["gtm.uniqueEventId"] = d, Xi("gtm.uniqueEventId", d)); return Ov(a) }
    function $w(a) { if (null == a || "object" !== typeof a) return !1; if (a.event) return !0; if (Ka(a)) { var b = a[0]; if ("config" === b || "event" === b || "js" === b || "get" === b) return !0 } return !1 }
    function ax() {
      var a; if (Sw.length) a = Sw.shift(); else if (Rw.length) a = Rw.shift(); else return; var b; var c = a; if (Tw || !$w(c.message)) b = c; else {
        Tw = !0; var d = c.message["gtm.uniqueEventId"]; "number" !== typeof d && (d = c.message["gtm.uniqueEventId"] = Ii()); var e = {}, f = { message: (e.event = "gtm.init_consent", e["gtm.uniqueEventId"] = d - 2, e), messageContext: { eventId: d - 2 } }, g = {}, k = { message: (g.event = "gtm.init", g["gtm.uniqueEventId"] = d - 1, g), messageContext: { eventId: d - 1 } }; Rw.unshift(k, c); if (Hk) {
          var m = Uf.ctid; if (m) {
            var n, p = pk(vk());
            n = p && p.context; var q, r = Cn(z.location.href); q = r.hostname + r.pathname; var t = n && n.fromContainerExecution, u = n && n.source, v = Uf.Og, w = gk.hf; Hk && (Ft || (Ft = q), Gt.push(m + ";" + v + ";" + (t ? 1 : 0) + ";" + (u || 0) + ";" + (w ? 1 : 0)))
          }
        } b = f
      } return b
    }
    function bx() {
      for (var a = !1, b; !Uw && (b = ax());) {
        Uw = !0; delete Ri.eventModel; Ti(); var c = b, d = c.message, e = c.messageContext; if (null == d) Uw = !1; else {
          e.fromContainerExecution && Yi(); try {
            if (Ba(d)) try { d.call(Vi) } catch (x) { } else if (Da(d)) { var f = d; if (h(f[0])) { var g = f[0].split("."), k = g.pop(), m = f.slice(1), n = Ui(g.join("."), 2); if (null != n) try { n[k].apply(n, m) } catch (x) { } } } else {
              var p = void 0, q = !1; if (Ka(d)) {
                a: { if (d.length && h(d[0])) { var r = qw[d[0]]; if (r && (!e.fromContainerExecution || !rw[d[0]])) { p = r(d, e); break a } } p = void 0 } (q = p &&
                  "set" === d[0] && !!p.event) && M(101)
              } else p = d; if (p) { var t = Zw(p, e); a = t || a; q && t && M(113) }
            }
          } finally { e.fromContainerExecution && Ti(!0); var u = d["gtm.uniqueEventId"]; if ("number" === typeof u) { for (var v = Qw[String(u)] || [], w = 0; w < v.length; w++)Sw.push(cx(v[w])); v.length && Sw.sort(Vw); delete Qw[String(u)]; u > Pw && (Pw = u) } Uw = !1 }
        }
      } return !a
    }
    function dx() { if (Q(29)) { var a = ex(); } var e = bx(); try { sw(nk()) } catch (f) { } return e } function Tu(a) { if (Pw < a.notBeforeEventId) { var b = String(a.notBeforeEventId); Qw[b] = Qw[b] || []; Qw[b].push(a) } else Sw.push(cx(a)), Sw.sort(Vw), F(function () { Uw || bx() }) }
    function cx(a) { return { message: a.message, messageContext: a.messageContext } }
    var fx = function () {
      function a(f) { var g = {}; if (Nw(f)) { var k = f; f = Nw(k) ? k.getUntrustedMessageValue() : void 0; g.fromContainerExecution = !0 } return { message: f, messageContext: g } } var b = Ec(ni.ia, []), c = oi[ni.ia] = oi[ni.ia] || {}; !0 === c.pruned && M(83); Qw = Ru().get(); Uu(); Wt(function () { if (!c.gtmDom) { c.gtmDom = !0; var f = {}; b.push((f.event = "gtm.dom", f)) } }); xw(function () { if (!c.gtmLoad) { c.gtmLoad = !0; var f = {}; b.push((f.event = "gtm.load", f)) } }); c.subscribers = (c.subscribers || 0) + 1; var d = b.push; b.push = function () {
        var f; if (0 < oi.SANDBOXED_JS_SEMAPHORE) {
          f =
          []; for (var g = 0; g < arguments.length; g++)f[g] = new Ow(arguments[g])
        } else f = [].slice.call(arguments, 0); var k = f.map(function (q) { return a(q) }); Rw.push.apply(Rw, k); var m = d.apply(b, f), n = Math.max(100, Number("1000") || 300); if (this.length > n) for (M(4), c.pruned = !0; this.length > n;)this.shift(); var p = "boolean" !== typeof m || m; return bx() && p
      }; var e = b.slice(0).map(function (f) { return a(f) }); Rw.push.apply(Rw, e); ex() && (Q(29) && Fs(), F(dx))
    }, ex = function () { var a = !0; return a }; function gx(a) { if (null == a || 0 === a.length) return !1; var b = Number(a), c = Ra(); return b < c + 3E5 && b > c - 9E5 } function hx(a) { return a && 0 === a.indexOf("pending:") ? gx(a.substr(8)) : !1 };

    var Cx = function () { }; var Dx = new String("undefined"), Ex = function () { }; Ex.prototype.toString = function () { return "undefined" }; var Fx = new Ex;
    var Nx = function (a, b, c) { var d = { event: b, "gtm.element": a, "gtm.elementClasses": Vc(a, "className"), "gtm.elementId": a["for"] || Qc(a, "id") || "", "gtm.elementTarget": a.formTarget || Vc(a, "target") || "" }; c && (d["gtm.triggers"] = c.join(",")); d["gtm.elementUrl"] = (a.attributes && a.attributes.formaction ? a.formAction : "") || a.action || Vc(a, "href") || a.src || a.code || a.codebase || ""; return d }, Ox = function (a) {
      oi.hasOwnProperty("autoEventsSettings") || (oi.autoEventsSettings = {}); var b = oi.autoEventsSettings; b.hasOwnProperty(a) || (b[a] =
        {}); return b[a]
    }, Px = function (a, b, c) { Ox(a)[b] = c }, Qx = function (a, b, c, d) { var e = Ox(a), f = Sa(e, b, d); e[b] = c(f) }, Rx = function (a, b, c) { var d = Ox(a); return Sa(d, b, c) }, Sx = function (a) { Rx("jel", "init", !1) || (Px("jel", "init", !0), a()) }, Tx = function (a) { return "string" === typeof a ? a : String(Ii()) }; var Zx = !!z.MutationObserver, $x = void 0, ay = function (a) { if (!$x) { var b = function () { var c = C.body; if (c) if (Zx) (new MutationObserver(function () { for (var e = 0; e < $x.length; e++)F($x[e]) })).observe(c, { childList: !0, subtree: !0 }); else { var d = !1; Nc(c, "DOMNodeInserted", function () { d || (d = !0, F(function () { d = !1; for (var e = 0; e < $x.length; e++)F($x[e]) })) }) } }; $x = []; C.body ? b() : F(b) } $x.push(a) }, by = function (a) { if ($x) for (var b = 0; b < $x.length; b++)$x[b] === a && $x.splice(b, 1) };
    var my = function (a, b, c) {
      function d() { var g = a(); f += e ? (Ra() - e) * g.playbackRate / 1E3 : 0; e = Ra() } var e = 0, f = 0; return {
        createEvent: function (g, k, m) {
          var n = a(), p = n.Wg, q = void 0 !== m ? Math.round(m) : void 0 !== k ? Math.round(n.Wg * k) : Math.round(n.ij), r = void 0 !== k ? Math.round(100 * k) : 0 >= p ? 0 : Math.round(q / p * 100), t = C.hidden ? !1 : .5 <= Zp(c); d(); var u = void 0; void 0 !== b && (u = [b]); var v = Nx(c, "gtm.video", u); v["gtm.videoProvider"] = "youtube"; v["gtm.videoStatus"] = g; v["gtm.videoUrl"] = n.url; v["gtm.videoTitle"] = n.title; v["gtm.videoDuration"] =
            Math.round(p); v["gtm.videoCurrentTime"] = Math.round(q); v["gtm.videoElapsedTime"] = Math.round(f); v["gtm.videoPercent"] = r; v["gtm.videoVisible"] = t; return v
        }, Bj: function () { e = Ra() }, Wc: function () { d() }
      }
    }; var ny = z.clearTimeout, oy = z.setTimeout, py = function (a, b, c, d) { if (an()) { b && F(b) } else return Jc(a, b, c, d) }, qy = function () { return new Date }, ry = function () { return z.location.href }, sy = function (a) { return An(Cn(a), "fragment") }, ty = function (a) { return Bn(Cn(a)) }, uy = function (a, b) { return Ui(a, b || 2) }, vy = function (a, b, c) { return b ? Xw(a, b, c) : Ww(a) }, wy = function (a, b) { z[a] = b }, V = function (a, b, c) { b && (void 0 === z[a] || c && !z[a]) && (z[a] = b); return z[a] },
      xy = function (a, b, c) { return xm(a, b, void 0 === c ? !0 : !!c) }, yy = function (a, b, c) { return 0 === Gm(a, b, c) }, zy = function (a, b) { if (an()) { b && F(b) } else Lc(a, b) }, Ay = function (a) { return !!Rx(a, "init", !1) }, By = function (a) { Px(a, "init", !0) }, Cy = function (a, b, c) { nb(a) || Au(c, b, a) };

    function Zy(a, b) { function c(g) { var k = Cn(g), m = An(k, "protocol"), n = An(k, "host", !0), p = An(k, "port"), q = An(k, "path").toLowerCase().replace(/\/$/, ""); if (void 0 === m || "http" === m && "80" === p || "https" === m && "443" === p) m = "web", p = "default"; return [m, n, p, q] } for (var d = c(String(a)), e = c(String(b)), f = 0; f < d.length; f++)if (d[f] !== e[f]) return !1; return !0 }
    function $y(a) { return az(a) ? 1 : 0 }
    function az(a) {
      var b = a.arg0, c = a.arg1; if (a.any_of && Array.isArray(c)) { for (var d = 0; d < c.length; d++) { var e = lb(a, {}); lb({ arg1: c[d], any_of: void 0 }, e); if ($y(e)) return !0 } return !1 } switch (a["function"]) {
        case "_cn": return vg(b, c); case "_css": var f; a: { if (b) try { for (var g = 0; g < rg.length; g++) { var k = rg[g]; if (b[k]) { f = b[k](c); break a } } } catch (m) { } f = !1 } return f; case "_ew": return sg(b, c); case "_eq": return wg(b, c); case "_ge": return xg(b, c); case "_gt": return zg(b, c); case "_lc": return 0 <= String(b).split(",").indexOf(String(c));
        case "_le": return yg(b, c); case "_lt": return Ag(b, c); case "_re": return ug(b, c, a.ignore_case); case "_sw": return Bg(b, c); case "_um": return Zy(b, c)
      }return !1
    }; function bz() { var a = ["&cv=1", "&rv=" + ni.Lg, "&tc=" + tf.filter(function (b) { return b }).length]; ni.ee && a.push("&x=" + ni.ee); return a.join("") }; var cz = function () { return !1 }, dz = function () { var a = {}; return function (b, c, d) { } }; function ez() {
      var a = fz; return function (b, c, d) {
        var e = d && d.event; gz(c); var f = new qb; l(c, function (q, r) { var t = od(r, void 0, 1); void 0 === t && void 0 !== r && M(44); f.set(q, t) }); a.h.h.F = Mf(); var g = { dj: Zf(b), eventId: void 0 !== e ? e.id : void 0, priorityId: void 0 !== e ? e.priorityId : void 0, qf: void 0 !== e ? function (q) { return e.Rb.qf(q) } : void 0, Yc: function () { return b }, log: function () { }, Xk: { index: d && d.index, type: d && d.type, name: d && d.name }, lm: !!lt(b, 3) }; if (cz()) {
          var k = dz(), m = void 0, n = void 0; g.Wa = {
            Jh: [], ie: {}, ib: function (q, r, t) {
              1 ===
              r && (m = q); 7 === r && (n = t); k(q, r, t)
            }, th: oh()
          }; g.log = function (q, r) { if (m) { var t = Array.prototype.slice.call(arguments, 1); k(m, 4, { level: q, source: n, message: t }) } }
        } var p = Me(a, g, [b, f]); a.h.h.F = void 0; p instanceof ua && "return" === p.h && (p = p.B); return pd(p, void 0, 1)
      }
    } function gz(a) { var b = a.gtmOnSuccess, c = a.gtmOnFailure; Ba(b) && (a.gtmOnSuccess = function () { F(b) }); Ba(c) && (a.gtmOnFailure = function () { F(c) }) }; function hz(a, b) { var c = this; } hz.M = "addConsentListener"; var iz; var jz = function (a) { for (var b = 0; b < a.length; ++b)if (iz) try { a[b]() } catch (c) { M(77) } else a[b]() }; function kz(a, b, c) { var d = this, e; K(J(this), ["eventName:!string", "callback:!Fn", "triggerId:?string"], arguments), jz([function () { return L(d, "listen_data_layer", a) }]), e = Jv().addListener(a, pd(b), c); return e } kz.D = "internal.addDataLayerEventListener"; function lz(a, b, c) { } lz.M = "addDocumentEventListener"; function mz(a, b, c, d) { } mz.M = "addElementEventListener"; function nz(a) { } nz.M = "addEventCallback";
    function rz(a) { } rz.D = "internal.addFormAbandonmentListener"; function sz(a, b, c, d) { } sz.D = "internal.addFormData"; var tz = {}, uz = [], vz = {}, wz = 0, xz = 0;
    function Ez(a, b) { } Ez.D = "internal.addFormInteractionListener";
    function Lz(a, b) { } Lz.D = "internal.addFormSubmitListener";
    function Qz(a) { } Qz.D = "internal.addGaSendListener"; var Rz = function (a, b) { this.tagId = a; this.h = b }; function Sz(a, b, c) { var d = this; } Sz.D = "internal.loadGoogleTag"; function Tz(a, b, c) { } Tz.D = "internal.addGoogleTagRestriction"; var Uz = {}, Vz = [];
    var bA = function (a, b) { };
    bA.D = "internal.addHistoryChangeListener"; function cA(a, b, c) { } cA.M = "addWindowEventListener"; function dA(a, b) { return !0 } dA.M = "aliasInWindow"; function eA(a, b, c) { } eA.D = "internal.appendRemoteConfigParameter"; function fA() { var a = 2; return a }; function gA(a, b) { var c; return c } gA.M = "callInWindow"; function hA(a) { } hA.M = "callLater"; function iA(a) { } iA.D = "callOnDomReady"; function jA(a) { } jA.D = "callOnWindowLoad"; function kA(a) { var b; return b } kA.D = "internal.computeGtmParameter"; function lA(a, b) { var c; var d = od(c, this.h, fA()); void 0 === d && void 0 !== c && M(45); return d } lA.M = "copyFromDataLayer"; function mA(a) { var b; return b } mA.M = "copyFromWindow"; function nA(a, b) { var c; K(J(this), ["preHit:!PixieMap", "dustOptions:?PixieMap"], arguments); var d = pd(b) || {}, e = pd(a, this.h, 1).nj(), f = e.s; d.omitEventContext && (f = kl(new $k(e.s.eventId, e.s.priorityId))); var g = new vp(e.target, e.eventName, f); d.omitHitData || lb(e.h, g.h); d.omitMetadata ? g.metadata = {} : lb(e.metadata, g.metadata); g.isAborted = e.isAborted; c = od(xp(g), this.h, 1); return c } nA.D = "internal.copyPreHit"; function oA(a, b) { var c = null, d = fA(); return od(c, this.h, d) } oA.M = "createArgumentsQueue"; function pA(a) {
      var b; return od(b, this.h,
        fA())
    } pA.M = "createQueue"; function qA(a, b) { var c = null; K(J(this), ["pattern:!string", "flags:?string"], arguments); try { var d = (b || "").split("").filter(function (e) { return "ig".includes(e) }).join(""); c = new ld(new RegExp(a, d)) } catch (e) { } return c } qA.D = "internal.createRegex"; function rA(a) { if (!a) return {}; var b = a.Xk; return Yt(b.type, b.index, b.name) } function sA(a) { return a ? { originatingEntity: rA(a) } : {} }; function tA(a) { } tA.D = "internal.declareConsentState"; function uA(a) { var b = ""; return b } uA.D = "internal.decodeUrlHtmlEntities"; function vA(a) {
      var b;
      return b
    } vA.D = "internal.detectUserProvidedData";
    function BA(a, b) { return b } BA.D = "internal.enableAutoEventOnElementVisibility"; function CA() { } CA.D = "internal.enableAutoEventOnError"; var DA = {}, EA = [], FA = {}, GA = 0, HA = 0;
    function NA(a, b) { var c = this; return b } NA.D = "internal.enableAutoEventOnFormInteraction";
    function SA(a, b) { var c = this; return b } SA.D = "internal.enableAutoEventOnFormSubmit";
    function XA() { var a = this; } XA.D = "internal.enableAutoEventOnGaSend"; var YA = {}, ZA = [];
    var aB = function (a, b) { var c = "" + b; if (YA[c]) YA[c].push(a); else { var d = [a]; YA[c] = d; var e = $A(), f = -1; ZA.push(function (g) { 0 <= f && z.clearTimeout(f); b ? f = z.setTimeout(function () { e(g, d); f = -1 }, b) : e(g, d) }) } }, $A = function () {
      var a = z.location.href, b = { source: null, state: z.history.state || null, url: Bn(Cn(a)), P: An(Cn(a), "fragment") }; return function (c, d) {
        var e = b, f = {}; f[e.source] = !0; f[c.source] = !0; if (!f.popstate || !f.hashchange || e.P != c.P) {
          var g = {}, k = (g.event = "gtm.historyChange-v2", g["gtm.historyChangeSource"] = c.source, g["gtm.oldUrlFragment"] =
            b.P, g["gtm.newUrlFragment"] = c.P, g["gtm.oldHistoryState"] = b.state, g["gtm.newHistoryState"] = c.state, g["gtm.oldUrl"] = b.url, g["gtm.newUrl"] = c.url, g["gtm.triggers"] = d.join(","), g); b = c; Ww(k)
        }
      }
    }, bB = function (a, b) { var c = z.history, d = c[a]; if (Ba(d)) try { c[a] = function (e, f, g) { d.apply(c, [].slice.call(arguments, 0)); var k = z.location.href; b({ source: a, state: e, url: Bn(Cn(k)), P: An(Cn(k), "fragment") }) } } catch (e) { } }, dB = function (a) {
      z.addEventListener("popstate", function (b) {
        var c = cB(b); a({
          source: "popstate", state: b.state, url: Bn(Cn(c)),
          P: An(Cn(c), "fragment")
        })
      })
    }, eB = function (a) { z.addEventListener("hashchange", function (b) { var c = cB(b); a({ source: "hashchange", state: null, url: Bn(Cn(c)), P: An(Cn(c), "fragment") }) }) }, cB = function (a) { return a.target && a.target.location && a.target.location.href ? a.target.location.href : z.location.href };
    function fB(a, b) {
      var c = this; K(J(this), ["options:?PixieMap", "triggerId:?*"], arguments); jz([function () { return L(c, "process_dom_events", "window", "popstate") }, function () { return L(c, "process_dom_events", "window", "pushstate") }]); b = Tx(b); var d = Number(a && a.get("interval")); 0 < d && isFinite(d) || (d = 0); if (Rx("ehl", "init", !1)) { var e = Rx("ehl", "reg"); e && e(b, d) } else {
        var f = function (g) { for (var k = 0; k < ZA.length; k++)ZA[k](g) }; eB(f); dB(f); bB("pushState",
          f); bB("replaceState", f); aB(b, d); Px("ehl", "reg", aB); Px("ehl", "init", !0)
      } return b
    } fB.D = "internal.enableAutoEventOnHistoryChange";
    var gB = function (a, b) { if (2 === a.which || a.ctrlKey || a.shiftKey || a.altKey || a.metaKey) return !1; var c = Vc(b, "href"), d = c.indexOf("#"), e = Vc(b, "target"); if (e && "_self" !== e && "_parent" !== e && "_top" !== e || 0 === d) return !1; if (0 < d) { var f = Bn(Cn(c)), g = Bn(Cn(z.location.href)); return f !== g } return !0 }, hB = function (a, b) { for (var c = An(Cn((b.attributes && b.attributes.formaction ? b.formAction : "") || b.action || Vc(b, "href") || b.src || b.code || b.codebase || ""), "host"), d = 0; d < a.length; d++)try { if ((new RegExp(a[d])).test(c)) return !1 } catch (e) { } return !0 },
      iB = function () {
        var a = 0, b = function (c) {
          var d = c.target; if (d && 3 !== c.which && !(c.jh || c.timeStamp && c.timeStamp === a)) {
            a = c.timeStamp; d = Tc(d, ["a", "area"], 100); if (!d) return c.returnValue; var e = c.defaultPrevented || !1 === c.returnValue, f = Rx("aelc", e ? "nv.mwt" : "mwt", 0), g; g = e ? Rx("aelc", "nv.ids", []) : Rx("aelc", "ids", []); for (var k = [], m = 0; m < g.length; m++) { var n = g[m], p = Rx("aelc", "aff.map", {})[n]; p && !hB(p, d) || k.push(n) } if (k.length) {
              var q = gB(c, d), r = Nx(d, "gtm.linkClick", k); r["gtm.elementText"] = Rc(d); r["gtm.willOpenInNewWindow"] =
                !q; if (q && !e && f && d.href) {
                  var t = !!Fa(String(Vc(d, "rel") || "").split(" "), function (x) { return "noreferrer" === x.toLowerCase() }), u = z[(Vc(d, "target") || "_self").substring(1)], v = !0, w = Yw(function () { var x; if (x = v && u) { var y; a: if (t) { var A; try { A = new MouseEvent(c.type, { bubbles: !0 }) } catch (B) { if (!C.createEvent) { y = !1; break a } A = C.createEvent("MouseEvents"); A.initEvent(c.type, !0, !0) } A.jh = !0; c.target.dispatchEvent(A); y = !0 } else y = !1; x = !y } x && (u.location.href = Vc(d, "href")) }, f); if (Xw(r, w, f)) v = !1; else return c.preventDefault &&
                    c.preventDefault(), c.returnValue = !1
                } else Xw(r, function () { }, f || 2E3); return !0
            }
          }
        }; Nc(C, "click", b, !1); Nc(C, "auxclick", b, !1)
      };
    function jB(a, b) {
      var c = this; K(J(this), ["dustOptions:?PixieMap", "triggerId:?*"], arguments); var d = pd(a); jz([function () { return L(c, "detect_link_click_events", d) }]); var e = d && !!d.waitForTags, f = d && !!d.checkValidation, g = d ? d.affiliateDomains : void 0; b = Tx(b); if (e) { var k = Number(d.waitForTagsTimeout); 0 < k && isFinite(k) || (k = 2E3); var m = function (p) { return Math.max(k, p) }; Qx("aelc", "mwt", m, 0); f || Qx("aelc", "nv.mwt", m, 0) } var n = function (p) {
        p.push(b);
        return p
      }; Qx("aelc", "ids", n, []); f || Qx("aelc", "nv.ids", n, []); g && Qx("aelc", "aff.map", function (p) { p[b] = g; return p }, {}); Rx("aelc", "init", !1) || (iB(), Px("aelc", "init", !0)); return b
    } jB.D = "internal.enableAutoEventOnLinkClick"; var kB, lB;
    var mB = function (a) { return Rx("sdl", a, {}) }, nB = function (a, b, c) { b && (Array.isArray(a) || (a = [a]), Qx("sdl", c, function (d) { for (var e = 0; e < a.length; e++) { var f = String(a[e]); d.hasOwnProperty(f) || (d[f] = []); d[f].push(b) } return d }, {})) }, qB = function () {
      var a = 250, b = !1; C.scrollingElement && C.documentElement && z.addEventListener && (a = 50, b = !0); var c = 0, d = !1, e = function () { d ? c = z.setTimeout(e, a) : (c = 0, oB(), Rx("sdl", "init", !1) && !pB() && (Oc(z, "scroll", f), Oc(z, "resize", f), Px("sdl", "init", !1))); d = !1 }, f = function () {
        b && kB(); c ? d = !0 : (c =
          z.setTimeout(e, a), Px("sdl", "pending", !0))
      }; return f
    }, oB = function () { var a = kB(), b = a.Ug, c = a.Vg, d = b / lB.scrollWidth * 100, e = c / lB.scrollHeight * 100; rB(b, "horiz.pix", "PIXELS", "horizontal"); rB(d, "horiz.pct", "PERCENT", "horizontal"); rB(c, "vert.pix", "PIXELS", "vertical"); rB(e, "vert.pct", "PERCENT", "vertical"); Px("sdl", "pending", !1) }, rB = function (a, b, c, d) {
      var e = mB(b), f = {}, g; for (g in e) if (f = { jd: f.jd }, f.jd = g, e.hasOwnProperty(f.jd)) {
        var k = Number(f.jd); if (!(a < k)) {
          var m = {}; Ww((m.event = "gtm.scrollDepth", m["gtm.scrollThreshold"] =
            k, m["gtm.scrollUnits"] = c.toLowerCase(), m["gtm.scrollDirection"] = d, m["gtm.triggers"] = e[f.jd].join(","), m)); Qx("sdl", b, function (n) { return function (p) { delete p[n.jd]; return p } }(f), {})
        }
      }
    }, tB = function () { Qx("sdl", "scr", function (a) { a || (a = C.scrollingElement || C.body && C.body.parentNode); return lB = a }, !1); Qx("sdl", "depth", function (a) { a || (a = sB()); return kB = a }, !1) }, sB = function () { var a = 0, b = 0; return function () { var c = Yp(), d = c.height; a = Math.max(lB.scrollLeft + c.width, a); b = Math.max(lB.scrollTop + d, b); return { Ug: a, Vg: b } } },
      pB = function () { return !!(Object.keys(mB("horiz.pix")).length || Object.keys(mB("horiz.pct")).length || Object.keys(mB("vert.pix")).length || Object.keys(mB("vert.pct")).length) };
    function uB(a, b) {
      var c = this; K(J(this), ["options:!PixieMap", "triggerId:?*"], arguments); jz([function () { return L(c, "process_dom_events", "window", "resize") }, function () { return L(c, "process_dom_events", "window", "scroll") }]); tB(); if (!lB) return; b = Tx(b); var d = pd(a); switch (d.horizontalThresholdUnits) { case "PIXELS": nB(d.horizontalThresholds, b, "horiz.pix"); break; case "PERCENT": nB(d.horizontalThresholds, b, "horiz.pct") }switch (d.verticalThresholdUnits) {
        case "PIXELS": nB(d.verticalThresholds,
          b, "vert.pix"); break; case "PERCENT": nB(d.verticalThresholds, b, "vert.pct")
      }Rx("sdl", "init", !1) ? Rx("sdl", "pending", !1) || F(function () { return oB() }) : (Px("sdl", "init", !0), Px("sdl", "pending", !0), F(function () { oB(); if (pB()) { var e = qB(); Nc(z, "scroll", e); Nc(z, "resize", e) } else Px("sdl", "init", !1) })); return b
    } uB.D = "internal.enableAutoEventOnScroll"; var lc = da(["data-gtm-yt-inspected-"]), vB = ["www.youtube.com", "www.youtube-nocookie.com"], wB, xB = !1;
    var yB = function (a, b, c) { var d = a.map(function (g) { return { Aa: g, Ke: g, Ie: void 0 } }); if (!b.length) return d; var e = b.map(function (g) { return { Aa: g * c, Ke: void 0, Ie: g } }); if (!d.length) return e; var f = d.concat(e); f.sort(function (g, k) { return g.Aa - k.Aa }); return f }, zB = function (a) { a = void 0 === a ? [] : a; for (var b = [], c = 0; c < a.length; c++)0 > a[c] || b.push(a[c]); b.sort(function (d, e) { return d - e }); return b }, AB = function (a) {
      a = void 0 === a ? [] : a; for (var b = [], c = 0; c < a.length; c++)100 < a[c] || 0 > a[c] || (b[c] = a[c] / 100); b.sort(function (d, e) {
        return d -
          e
      }); return b
    }, BB = function (a, b) {
      var c, d; function e() { t = my(function () { return { url: w, title: x, Wg: v, ij: a.getCurrentTime(), playbackRate: y } }, b.jb, a.getIframe()); v = 0; x = w = ""; y = 1; return f } function f(G) { switch (G) { case 1: v = Math.round(a.getDuration()); w = a.getVideoUrl(); if (a.getVideoData) { var E = a.getVideoData(); x = E ? E.title : "" } y = a.getPlaybackRate(); b.Rg ? Ww(t.createEvent("start")) : t.Wc(); u = yB(b.Bh, b.Ah, a.getDuration()); return g(G); default: return f } } function g() { A = a.getCurrentTime(); B = Pa().getTime(); t.Bj(); r(); return k }
      function k(G) { var E; switch (G) { case 0: return n(G); case 2: E = "pause"; case 3: var I = a.getCurrentTime() - A; E = 1 < Math.abs((Pa().getTime() - B) / 1E3 * y - I) ? "seek" : E || "buffering"; a.getCurrentTime() && (b.Qg ? Ww(t.createEvent(E)) : t.Wc()); q(); return m; case -1: return e(G); default: return k } } function m(G) { switch (G) { case 0: return n(G); case 1: return g(G); case -1: return e(G); default: return m } } function n() { for (; d;) { var G = c; z.clearTimeout(d); G() } b.Pg && Ww(t.createEvent("complete", 1)); return e(-1) } function p() { } function q() {
        d &&
        (z.clearTimeout(d), d = 0, c = p)
      } function r() { if (u.length && 0 !== y) { var G = -1, E; do { E = u[0]; if (E.Aa > a.getDuration()) return; G = (E.Aa - a.getCurrentTime()) / y; if (0 > G && (u.shift(), 0 === u.length)) return } while (0 > G); c = function () { d = 0; c = p; 0 < u.length && u[0].Aa === E.Aa && (u.shift(), Ww(t.createEvent("progress", E.Ie, E.Ke))); r() }; d = z.setTimeout(c, 1E3 * G) } } var t, u = [], v, w, x, y, A, B, D = e(-1); d = 0; c = p; return { onStateChange: function (G) { D = D(G) }, onPlaybackRateChange: function (G) { A = a.getCurrentTime(); B = Pa().getTime(); t.Wc(); y = G; q(); r() } }
    }, DB =
        function (a) { F(function () { function b() { for (var d = c.getElementsByTagName("iframe"), e = d.length, f = 0; f < e; f++)CB(d[f], a) } var c = C; b(); ay(b) }) }, CB = function (a, b) { if (!a.getAttribute("data-gtm-yt-inspected-" + b.jb) && (kc(a, "data-gtm-yt-inspected-" + b.jb), EB(a, b.se))) { a.id || (a.id = FB()); var c = z.YT, d = c.get(a.id); d || (d = new c.Player(a.id)); var e = BB(d, b), f = {}, g; for (g in e) f = { De: f.De }, f.De = g, e.hasOwnProperty(f.De) && d.addEventListener(f.De, function (k) { return function (m) { return e[k.De](m.data) } }(f)) } }, EB = function (a, b) {
          var c =
            a.getAttribute("src"); if (GB(c, "embed/")) { if (0 < c.indexOf("enablejsapi=1")) return !0; if (b) { var d; var e = -1 !== c.indexOf("?") ? "&" : "?"; -1 < c.indexOf("origin=") ? d = c + e + "enablejsapi=1" : (wB || (wB = C.location.protocol + "//" + C.location.hostname, C.location.port && (wB += ":" + C.location.port)), d = c + e + "enablejsapi=1&origin=" + encodeURIComponent(wB)); var f; f = Hb(d); a.src = Fb(f).toString(); return !0 } } return !1
        }, GB = function (a, b) { if (!a) return !1; for (var c = 0; c < vB.length; c++)if (0 <= a.indexOf("//" + vB[c] + "/" + b)) return !0; return !1 }, FB = function () {
          var a =
            Math.round(1E9 * Math.random()) + ""; return C.getElementById(a) ? FB() : a
        };
    function HB(a, b) {
      var c = this; K(J(this), ["dustOptions:!PixieMap", "triggerId:?*"], arguments); Q(81) ? jz([function () { return L(c, "detect_youtube_activity_events", { fixMissingApi: !!a.get("fixMissingApi") }) }]) : jz([function () { return L(c, "process_dom_events", "element", "onStateChange") }, function () { return L(c, "process_dom_events", "element", "onPlaybackRateChange") }]); b = Tx(b); var d = !!a.get("captureStart"), e = !!a.get("captureComplete"),
        f = !!a.get("capturePause"), g = AB(pd(a.get("progressThresholdsPercent"))), k = zB(pd(a.get("progressThresholdsTimeInSeconds"))), m = !!a.get("fixMissingApi"); if (!(d || e || f || g.length || k.length)) return; var n = { Rg: d, Pg: e, Qg: f, Ah: g, Bh: k, se: m, jb: b }, p = z.YT, q = function () { DB(n) }; if (p) return p.ready && p.ready(q), b; var r = z.onYouTubeIframeAPIReady; z.onYouTubeIframeAPIReady = function () { r && r(); q() }; F(function () {
          for (var t = C.getElementsByTagName("script"), u = t.length, v = 0; v < u; v++) {
            var w = t[v].getAttribute("src"); if (GB(w, "iframe_api") ||
              GB(w, "player_api")) return b
          } for (var x = C.getElementsByTagName("iframe"), y = x.length, A = 0; A < y; A++)if (!xB && EB(x[A], n.se)) return Jc("https://www.youtube.com/iframe_api"), xB = !0, b
        }); return b
    } HB.D = "internal.enableAutoEventOnYouTubeActivity"; var IB; function JB(a) { var b = !1; return b } JB.D = "internal.evaluateMatchingRules"; var mC = function () { var a = !0; bm(7) && bm(9) && bm(10) || (a = !1); return a }, nC = function () { var a = !0; bm(3) && bm(4) || (a = !1); return a }; function iD(a, b, c, d) { } iD.D = "internal.executeEventProcessor"; function jD(a) { var b = void 0; return od(b) } jD.D = "internal.executeJavascriptString"; var kD = function (a) { var b; return b }; function lD() { var a = new qb; L(this, "read_container_data"), a.set("containerId", 'G-T8GWK081T9'), a.set("version", '1'), a.set("environmentName", ''), a.set("debugMode", bg), a.set("previewMode", dg), a.set("environmentMode", cg), Q(92) && a.set("firstPartyServing", wi || yi), Q(94) && a.set("containerUrl", Dc), a.Bb(); return a }
    lD.M = "getContainerVersion"; function mD(a, b) { b = void 0 === b ? !0 : b; var c; return c } mD.M = "getCookieValues"; function nD() { return kj() } nD.D = "internal.getCountryCode"; function oD() { var a = []; a = lk(); return od(a) } oD.D = "internal.getDestinationIds"; function pD(a, b) { var c = ""; return c } pD.D = "internal.getElementAttribute"; function qD(a) { var b = null; return b } qD.M = "getElementById"; function rD(a) { var b = ""; return b } rD.D = "internal.getElementInnerText"; function sD(a, b) { var c = null; return c } sD.D = "internal.getElementProperty"; function tD(a) { var b; return b } tD.D = "internal.getElementValue"; function uD(a) { var b = 0; return b } uD.D = "internal.getElementVisibilityRatio"; function vD(a) { var b = null; return b } vD.D = "internal.getElementsByCssSelector"; var wD = {}; wD.deferGaGamLink = Q(65); wD.enableAddGoogleTagRestrictionApi = Q(40); wD.enableAdsConversionValidation = Q(37); wD.enableAdsHistoryChangeEvents = Q(15); wD.enableAutoPiiOnPhoneAndAddress = Q(54); wD.enableCcdAutoRedaction = Q(41); wD.enableCcdPreAutoPiiDetection = Q(19); wD.enableConsentDisclosureActivity = Q(60); wD.enableDeferAllEnhancedMeasurement = Q(62); wD.enableEesPagePath = Q(17); wD.enableEuidAutoMode = Q(16); wD.enableFormSkipValidation = Q(55); wD.enableGa4OnoRemarketing = Q(14); wD.enableGetElementAttribute = Q(70);
    wD.enableGetElementInnerText = Q(78); wD.enableRemoteDomainListSettingsInPixie = Q(101); wD.enableUrlDecodeEventUsage = Q(80); wD.includeQueryInEesPagePath = Q(23); wD.pixieSetCorePlatformServices = Q(48); wD.useEnableAutoEventOnFormApis = Q(40); wD.autoPiiEligible = oj(); function xD() { return od(wD) } xD.D = "internal.getFlags"; function yD() { return new ld(Fx) } yD.D = "internal.getHtmlId"; function zD(a, b) { var c; K(J(this), ["targetId:!string", "name:!string"], arguments); var d = up(a) || {}; c = od(d[b], this.h); return c } zD.D = "internal.getProductSettingsParameter"; function AD(a, b) { var c; K(J(this), ["queryKey:!string", "retrieveAll:?boolean"], arguments); L(this, "get_url", "query", a); var d = An(Cn(z.location.href), "query"), e = xn(d, a, b); c = od(e, this.h); return c } AD.M = "getQueryParameters"; function BD(a, b) { var c; return c } BD.M = "getReferrerQueryParameters"; function CD(a) { var b = ""; return b } CD.M = "getReferrerUrl"; function DD() { return lj() } DD.D = "internal.getRegionCode"; function ED(a, b) { var c; K(J(this), ["targetId:!string", "name:!string"], arguments); var d = Xv(a); c = od(d[b], this.h); return c } ED.D = "internal.getRemoteConfigParameter"; function FD(a) { var b = ""; K(J(this), ["component:?string"], arguments), L(this, "get_url", a), b = An(Cn(z.location.href), a); return b } FD.M = "getUrl"; function GD() { L(this, "get_user_agent"); return Cc.userAgent } GD.M = "getUserAgent";
    var HD = function (a) { var b = !1; b = wp(a, "google_ono", !1); return b }, ID = function (a) { if (a.metadata.is_merchant_center || !Bt(a.s)) return !1; if (!U(a.s, N.g.Ld)) { var b = U(a.s, N.g.Dd); return !0 === b || "true" === b } return !0 }, JD = function (a) { var b = a.metadata.user_data; if (kb(b)) return b }, KD = function (a, b) { var c = wp(a, N.g.Bd, a.s.B[N.g.Bd]); if (c && void 0 !== c[b || a.eventName]) return c[b || a.eventName] }, LD = function (a,
      b, c) { a.h[N.g.fe] || (a.h[N.g.fe] = {}); a.h[N.g.fe][b] = c }; var MD = !1, ND = function (a) { var b = a.eventName === N.g.ic && Jj() && ID(a), c = a.metadata.batch_on_navigation, d = a.metadata.is_conversion, e = a.metadata.is_session_start, f = a.metadata.create_dc_join, g = a.metadata.create_google_join, k = a.metadata.euid_mode_enabled && !!JD(a); return !(!Cc.sendBeacon || d || k || e || f || g || b || !c && MD) };
    var OD = function (a) { var b = 0, c = 0; return { start: function () { b = Ra() }, stop: function () { c = this.get() }, get: function () { var d = 0; a.lh() && (d = Ra() - b); return d + c } } }, PD = function () { this.h = void 0; this.B = 0; this.isActive = this.isVisible = this.C = !1; this.K = this.F = void 0 }; aa = PD.prototype; aa.vk = function (a) {
      var b = this; if (!this.h) {
        this.C = C.hasFocus(); this.isVisible = !C.hidden; this.isActive = !0; var c = function (d, e, f) { Nc(d, e, function (g) { b.h.stop(); f(g); b.lh() && b.h.start() }) }; c(z, "focus", function () { b.C = !0 }); c(z, "blur", function () {
          b.C =
          !1
        }); c(z, "pageshow", function (d) { b.isActive = !0; d.persisted && M(56); b.K && b.K() }); c(z, "pagehide", function () { b.isActive = !1; b.F && b.F() }); c(C, "visibilitychange", function () { b.isVisible = !C.hidden }); ID(a) && -1 === (Cc.userAgent || "").indexOf("Firefox") && -1 === (Cc.userAgent || "").indexOf("FxiOS") && c(z, "beforeunload", function () { MD = !0 }); this.Eh(); this.B = 0
      }
    }; aa.Eh = function () { this.B += this.yf(); this.h = OD(this); this.lh() && this.h.start() }; aa.Em = function (a) { var b = this.yf(); 0 < b && (a.h[N.g.yd] = b) }; aa.wl = function (a) {
      a.h[N.g.yd] =
      void 0; this.Eh(); this.B = 0
    }; aa.lh = function () { return this.C && this.isVisible && this.isActive }; aa.ml = function () { return this.B + this.yf() }; aa.yf = function () { return this.h && this.h.get() || 0 }; aa.jm = function (a) { this.F = a }; aa.zj = function (a) { this.K = a }; var QD = function (a) { yb("GA4_EVENT", a) }; function RD() { return z.gaGlobal = z.gaGlobal || {} } var SD = function () { var a = RD(); a.hid = a.hid || Ga(); return a.hid }, TD = function (a, b) { var c = RD(); if (void 0 == c.vid || b && !c.from_cookie) c.vid = a, c.from_cookie = b };
    var UD = function (a, b, c) { var d = a.metadata.client_id_source; if (void 0 === d || c <= d) a.h[N.g.Za] = b, a.metadata.client_id_source = c }, XD = function (a, b) { var c; var d = b.metadata.cookie_options, e = d.prefix + "_ga", f = Fn(d, void 0, void 0, N.g.R); if (!1 === U(b.s, N.g.rb) && VD(b) === a) c = !0; else { var g = Zm(a, WD[0], d.domain, d.path); c = 1 !== Gm(e, g, f) } return c }, VD = function (a) { var b = a.metadata.cookie_options, c = b.prefix + "_ga", d = Ym(c, b.domain, b.path, WD, N.g.R); if (!d) { var e = String(U(a.s, N.g.kc, "")); e && e != c && (d = Ym(e, b.domain, b.path, WD, N.g.R)) } return d },
      WD = ["GA1"], YD = function (a, b) { var c = a.h[N.g.Za]; if (U(a.s, N.g.vb) && U(a.s, N.g.Jb) || b && c === b) return c; if (c) { c = "" + c; if (!XD(c, a)) return M(31), a.isAborted = !0, ""; TD(c, S(N.g.R)); return c } M(32); a.isAborted = !0; return "" };
    var aE = function (a, b, c) { if (!b) return a; if (!a) return b; var d = ZD(a); if (!d) return b; var e, f = La(null != (e = U(c.s, N.g.Sc)) ? e : 30); if (!(Math.floor(c.metadata.event_start_timestamp_ms / 1E3) > d.Fe + 60 * f)) return a; var g = ZD(b); if (!g) return a; g.xc = d.xc + 1; var k; return null != (k = $D(g.sessionId, g.xc, g.hd, g.Fe, g.ph, g.vc, g.ne)) ? k : b }, dE = function (a, b) {
      var c = b.metadata.cookie_options, d = bE(b, c), e = Zm(a, cE[0], c.domain, c.path), f = { Eb: N.g.R, domain: c.domain, path: c.path, expires: c.Tb ? new Date(Ra() + 1E3 * c.Tb) : void 0, flags: c.flags }; Gm(d,
        void 0, f); return 1 !== Gm(d, e, f)
    }, eE = function (a) { var b = a.metadata.cookie_options, c = bE(a, b), d = Ym(c, b.domain, b.path, cE, N.g.R); if (!d) return d; var e = xm(c, void 0, void 0, N.g.R); if (d && 1 < e.length) { M(114); for (var f = void 0, g = void 0, k = 0; k < e.length; k++) { var m = e[k].split("."); if (!(7 > m.length)) { var n = Number(m[5]); n && (!g || n > g) && (g = n, f = e[k]) } } f && !f.endsWith(d) && (M(115), d = f.split(".").slice(2).join(".")) } return d }, $D = function (a, b, c, d, e, f, g) { if (a && b) { var k = [a, b, La(c), d, e]; k.push(f ? "1" : "0"); k.push(g || "0"); return k.join(".") } },
      cE = ["GS1"], bE = function (a, b) { return b.prefix + "_ga_" + a.target.O[0] }, ZD = function (a) { if (a) { var b = a.split("."); if (!(5 > b.length || 7 < b.length)) { 7 > b.length && M(67); var c = Number(b[1]), d = Number(b[3]), e = Number(b[4] || 0); c || M(118); d || M(119); isNaN(e) && M(120); if (c && d && !isNaN(e)) return { sessionId: b[0], xc: c, hd: !!Number(b[2]), Fe: d, ph: e, vc: "1" === b[5], ne: "0" !== b[6] ? b[6] : void 0 } } } }, fE = function (a) {
        return $D(a.h[N.g.zb], a.h[N.g.Nd], a.h[N.g.Md], Math.floor(a.metadata.event_start_timestamp_ms / 1E3), a.metadata.join_timer_sec ||
          0, !!a.metadata[N.g.Ve], a.h[N.g.Mc])
      };
    var gE = function (a) { var b = U(a.s, N.g.Da), c = a.s.B[N.g.Da]; if (c === b) return c; var d = lb(b); c && c[N.g.T] && (d[N.g.T] = (d[N.g.T] || []).concat(c[N.g.T])); return d }, hE = function (a, b) { var c = Yn(!0); return "1" !== c._up ? {} : { clientId: c[a], Kf: c[b] } }, iE = function (a, b, c) { var d = Yn(!0), e = d[b]; e && (UD(a, e, 2), XD(e, a)); var f = d[c]; f && dE(f, a); return { clientId: e, Kf: f } }, jE = !1, kE = function (a) {
      var b = gE(a) || {}, c = a.metadata.cookie_options, d = c.prefix + "_ga", e = bE(a, c), f = {}; ho(b[N.g.oc], !!b[N.g.T]) && (f = iE(a, d, e), f.clientId && f.Kf && (jE = !0)); b[N.g.T] &&
        eo(function () { var g = {}, k = VD(a); k && (g[d] = k); var m = eE(a); m && (g[e] = m); var n = xm("FPLC", void 0, void 0, N.g.R); n.length && (g._fplc = n[0]); return g }, b[N.g.T], b[N.g.Kb], !!b[N.g.wb]); return f
    }, mE = function (a) { if (!U(a.s, N.g.Ab)) return {}; var b = a.metadata.cookie_options, c = b.prefix + "_ga", d = bE(a, b); fo(function () { var e; if (S("analytics_storage")) e = {}; else { var f = {}; e = (f._up = "1", f[c] = a.h[N.g.Za], f[d] = fE(a), f) } return e }, 1); return !S("analytics_storage") && lE() ? hE(c, d) : {} }, lE = function () {
      var a = zn(z.location, "host"), b = zn(Cn(C.referrer),
        "host"); return a && b ? a === b || 0 <= a.indexOf("." + b) || 0 <= b.indexOf("." + a) ? !0 : !1 : !1
    }, nE = function (a) { if (!a) return a; var b = String(a); b = Xn(b); return b = Xn(b, "_ga") }; var oE = function () { var a = Ra(), b = a + 864E5, c = 20, d = 5E3; return function () { var e = Ra(); e >= b && (b = e + 864E5, d = 5E3); if (1 > d) return !1; c = Math.min(c + (e - a) / 1E3 * 5, 20); a = e; if (1 > c) return !1; d--; c--; return !0 } };
    var pE = function (a, b) { if (Jj() || Q(45)) a.gcs = im(), Q(49) && (a.gcd = mm(b.s)), b.metadata.is_consent_update && (a.gcu = "1"); Q(104) && (hm(b.s) && (Q(85) || nC()) ? Q(53) && (a.npa = "0") : a.npa = "1") }, sE = function (a) {
      if (a.metadata.is_merchant_center) return "https://www.merchant-center-analytics.goog/mc/collect"; var b = zt(Bt(a.s), "/g/collect"); if (b) return b; var c = wp(a, N.g.eb, U(a.s, N.g.eb)); c = c || HD(a); var d = U(a.s,
        N.g.qb); return c && !mj() && !1 !== d && mC() && S(N.g.I) && S(N.g.R) ? qE() : rE()
    }, tE = !1; tE = !0; var uE = {}; uE[N.g.Za] = "cid"; uE[N.g.We] = "_fid"; uE[N.g.lg] = "_geo"; uE[N.g.ub] = "gdid"; uE[N.g.Id] = "ir"; uE[N.g.Ia] = "ul"; uE[N.g.Rc] = "_rdi"; uE[N.g.yb] = "sr"; uE[N.g.Gi] = "tid"; uE[N.g.ef] = "tt"; uE[N.g.ff] = "ec_mode"; uE[N.g.Pi] = "gtm_up"; uE[N.g.Pd] = "uaa", uE[N.g.Qd] = "uab", uE[N.g.Rd] = "uafvl", uE[N.g.Sd] = "uamb", uE[N.g.Td] = "uam", uE[N.g.Ud] = "uap", uE[N.g.Vd] = "uapv", uE[N.g.Wd] = "uaw";
    uE[N.g.xi] = "lps"; var vE = {}; vE[N.g.zc] = "cc"; vE[N.g.Ac] = "ci"; vE[N.g.Bc] = "cm"; vE[N.g.Cc] = "cn"; vE[N.g.Ec] = "cs"; vE[N.g.Fc] = "ck"; vE[N.g.qa] = "cu"; vE[N.g.ra] = "dl"; vE[N.g.Ea] = "dr"; vE[N.g.xb] = "dt"; vE[N.g.Md] = "seg"; vE[N.g.zb] = "sid"; vE[N.g.Nd] = "sct"; vE[N.g.za] = "uid"; Q(36) &&
      (vE[N.g.Pc] = "dp"); var wE = {}; wE[N.g.yd] = "_et"; wE[N.g.sb] = "edid"; var xE = {}; xE[N.g.zc] = "cc"; xE[N.g.Ac] = "ci"; xE[N.g.Bc] = "cm"; xE[N.g.Cc] = "cn"; xE[N.g.Ec] = "cs"; xE[N.g.Fc] = "ck"; var yE = {}, zE = Object.freeze((yE[N.g.wa] = 1, yE)), rE = function () { var a = "www"; tE && nj() && (a = nj()); return "https://" + a + ".google-analytics.com/g/collect" }, qE = function () { var a; tE && "" !== nj() && (a = nj()); return "https://" + (a ? a + "." : "") + "analytics.google.com/g/collect" }, AE = function (a, b, c) {
        var d = {}, e = {}, f = {}; d.v = "2"; d.tid = a.target.ba;
        HD(a) && !mj() && (d._ono = 1); d.gtm = cn(); d._p = SD(); c && (d.em = c); a.metadata.create_google_join && (d._gaz = 1); pE(d, a); Q(51) && (qm() && (d.dma_cps = nm()), d.dma = pm()); Q(85) && Kl(Tl()) && (d.tcfd = rm()); var g = a.h[N.g.ub]; g && (d.gdid = g); e.en = String(a.eventName); a.metadata.is_first_visit && (e._fv = a.metadata.is_first_visit_conversion ? 2 : 1); a.metadata.is_new_to_site && (e._nsi = 1); a.metadata.is_session_start && (e._ss = a.metadata.is_session_start_conversion ? 2 : 1); a.metadata.is_conversion &&
          (e._c = 1); a.metadata.is_external_event && (e._ee = 1); if (a.metadata.is_ecommerce) { var k = a.h[N.g.Z] || U(a.s, N.g.Z); if (Da(k)) for (var m = 0; m < k.length && 200 > m; m++)e["pr" + (m + 1)] = hg(k[m]) } var n = a.h[N.g.sb]; n && (e.edid = n); var p = function (r, t) { if ("object" !== typeof t || !zE[r]) { var u = "ep." + r, v = "epn." + r; r = Ca(t) ? v : u; var w = Ca(t) ? u : v; e.hasOwnProperty(w) && delete e[w]; e[r] = String(t) } }; l(a.h, function (r, t) {
            if (void 0 !== t && !Zh.hasOwnProperty(r)) {
              null === t && (t = ""); var u; r !== N.g.Mc ? u = !1 : a.metadata.euid_mode_enabled ? (d.ecid = t, u = !0) :
                u = void 0; if (!u && r !== N.g.Ve) { var v = t; !0 === t && (v = "1"); !1 === t && (v = "0"); v = String(v); var w; if (uE[r]) w = uE[r], d[w] = v; else if (vE[r]) w = vE[r], f[w] = v; else if (wE[r]) w = wE[r], e[w] = v; else if ("_" === r.charAt(0)) d[r] = v; else { var x; xE[r] ? x = !0 : r !== N.g.Dc ? x = !1 : ("object" !== typeof t && p(r, t), x = !0); x || p(r, t) } }
            }
          }); (function (r) { ID(a) && "object" === typeof r && l(r || {}, function (t, u) { "object" !== typeof u && (d["sst." + t] = String(u)) }) })(a.h[N.g.fe]); var q = a.h[N.g.Na] || {}; Q(104) || (hm(a.s) && (Q(85) || nC()) ? Q(53) && (q._npa = "0") : q._npa = "1"); Q(12) &&
            !1 === U(a.s, N.g.qb) && (d.ngs = "1"); l(q, function (r, t) { void 0 !== t && ((null === t && (t = ""), r !== N.g.za || f.uid) ? b[r] !== t && (e[(Ca(t) ? "upn." : "up.") + String(r)] = String(t), b[r] = t) : f.uid = String(t)) }); return jg.call(this, { Pa: d, yc: f, Yg: e }, sE(a), ID(a)) || this
      }; qa(AE, jg);
    var BE = function (a, b) { return a.replace(/\$\{([^\}]+)\}/g, function (c, d) { return b[d] || c }) }, CE = function (a) { var b = a.search; return a.protocol + "//" + a.hostname + a.pathname + (b ? b + "&richsstsse" : "?richsstsse") }, DE = function (a) { var b = {}, c = "", d = a.pathname.indexOf("/g/collect"); 0 <= d && (c = a.pathname.substring(0, d)); b.transport_url = a.protocol + "//" + a.hostname + c; return b }, EE = function (a, b) {
      var c = new z.XMLHttpRequest; c.withCredentials = !0; var d = b ? "POST" : "GET", e = "", f = 0, g = Cn(a), k = DE(g), m = CE(g); c.onprogress = function (n) {
        if (200 ===
          c.status) {
            e += c.responseText.substring(f); f = n.loaded; for (var p = BE(e, k), q = p.indexOf("\n\n"); -1 !== q;) {
              var r; a: { var t = fa(p.substring(0, q).split("\n")), u = t.next().value, v = t.next().value; if (u.startsWith("event: message") && v.startsWith("data: ")) try { r = JSON.parse(v.substring(v.indexOf(":") + 1)); break a } catch (D) { } r = void 0 } var w = r; if (w) { var x = w.send_pixel || []; if (Array.isArray(x)) for (var y = 0; y < x.length; y++)Mc(x[y]); if (Q(27)) { var A = w.send_beacon || []; if (Array.isArray(A)) for (var B = 0; B < A.length; B++)Uc(A[B]) } } p = p.substring(q +
                2); q = p.indexOf("\n\n")
            } e = p
        }
      }; c.open(d, m); c.send(b)
    };
    var HE = function (a, b, c, d) { var e = Q(88) && d; if (Q(87) || e) { var f = b, g = Xc(); void 0 !== g && (f += "&tfd=" + Math.round(g)); b = f } var k = a + "?" + b; FE && (d = !(0 === k.indexOf(rE()) || 0 === k.indexOf(qE()))); d && !MD ? EE(k, c) : GE(a, b, c) }, IE = function (a, b) {
      function c(r) { p.push(r + "=" + encodeURIComponent("" + a.Pa[r])) } var d = b.wm, e = b.xm, f = b.pl, g = b.Kk, k = b.Jk, m = b.Fl, n = b.El; if (d || e) {
        var p = []; a.Pa._ono && c("_ono"); c("tid"); c("cid");
        c("gtm"); p.push("aip=1"); a.yc.uid && !n && p.push("uid=" + encodeURIComponent("" + a.yc.uid)); d && (GE("https://stats.g.doubleclick.net/g/collect", "v=2&" + p.join("&")), Qj("https://stats.g.doubleclick.net/g/collect?v=2&" + p.join("&"))); if (e) {
          p.push("z=" + Ga()); if (!m) { var q = f && 0 === f.indexOf("google.") && "google.com" != f ? "https://www.%/ads/ga-audiences?v=1&t=sr&slf_rd=1&_r=4&".replace("%", f) : void 0; q && Mc(q + p.join("&")) } Q(12) && !MD && g && k && Vq() && function () {
            var r = Xq() + "/td/ga/rul?"; p = []; c("tid"); p.push("gacid=" + encodeURIComponent(String(a.Pa.cid)));
            c("gtm"); p.push("aip=1"); p.push("fledge=1"); p.push("z=" + Ga()); Wq(r + p.join("&"), a.Pa.tid)
          }()
        }
      }
    }, FE = !1; var JE = function () { this.F = 1; this.K = {}; this.h = new kg; this.B = -1 }; JE.prototype.C = function (a, b) {
      var c = this, d = new AE(a, this.K, b), e = ND(a); e && this.h.F(d) || this.flush(); if (e && this.h.add(d)) {
        if (0 > this.B) {
          var f = z.setTimeout, g; ID(a) ? KE ? (KE = !1, g = LE) : g = ME : g = 5E3; this.B = f.call(z,
            function () { return c.flush() }, g)
        }
      } else { var k = mg(d, this.F++); HE(d.B, k.xh, k.body, d.F); var m = a.metadata.create_dc_join, n = a.metadata.create_google_join, p = !1 !== U(a.s, N.g.Ga), q = hm(a.s), r = { eventId: a.s.eventId, priorityId: a.s.priorityId }, t = { wm: m, xm: n, pl: pj(), Kk: p, Jk: q, Fl: mj(), El: a.metadata.euid_mode_enabled, dn: r }; IE(d, t) } Ds(a)
    }; JE.prototype.add = function (a) { a.metadata.euid_mode_enabled && !MD ? this.W(a) : this.C(a) }; JE.prototype.flush = function () {
      if (this.h.events.length) {
        var a = ng(this.h, this.F++); HE(this.h.h, a.xh,
          a.body, this.h.B); this.h = new kg; 0 <= this.B && (z.clearTimeout(this.B), this.B = -1)
      }
    }; JE.prototype.W = function (a) { var b = this, c = JD(a); c ? Wh(c, function (d) { b.C(a, 1 === d.split("~").length ? void 0 : d) }) : this.C(a) }; var GE = function (a, b, c) { var d = a + "?" + b; if (c) try { Cc.sendBeacon && Cc.sendBeacon(d, c) } catch (e) { yb("TAGGING", 15) } else Uc(d) }, LE = ll('', 500), ME = ll('', 5E3), KE = !0;
    var NE = function (a, b, c) { void 0 === c && (c = {}); if ("object" === typeof b) for (var d in b) NE(a + "." + d, b[d], c); else c[a] = b; return c }, OE = function (a) { if (ID(a)) { var b = function (d) { var e = NE(N.g.wa, d); l(e, function (f, g) { a.h[f] = g }) }, c = U(a.s, N.g.wa); void 0 !== c ? b(c) : b(a.metadata.user_data); a.metadata.user_data = void 0 } }; var PE = /^(www\.)?google(\.com?)?(\.[a-z]{2}t?)?$/, QE = /^www.googleadservices.com$/; var RE = window, SE = document, TE = function (a) { var b = RE._gaUserPrefs; if (b && b.ioo && b.ioo() || SE.documentElement.hasAttribute("data-google-analytics-opt-out") || a && !0 === RE["ga-disable-" + a]) return !0; try { var c = RE.external; if (c && c._gaUserPrefs && "oo" == c._gaUserPrefs) return !0 } catch (f) { } for (var d = tm("AMP_TOKEN", String(SE.cookie), !0), e = 0; e < d.length; e++)if ("$OPT_OUT" == d[e]) return !0; return SE.getElementById("__gaOptOutExtension") ? !0 : !1 };
    var VE = function (a) { return !a || UE.test(a) || ai.hasOwnProperty(a) }, WE = function (a, b, c) { c || (c = function () { }); void 0 !== a.h[b] && (a.h[b] = c(a.h[b])) }, XE = function (a, b, c) { S(c) || Yj(function () { b.metadata.is_consent_update = !0; var d = ii[c || ""]; d && LD(b, "gcut", d); a.bj(b) }, c) }, sn = { Vk: "", Hm: Number("") }, YE = {}, ZE = (YE[N.g.zc] = 1, YE[N.g.Ac] = 1, YE[N.g.Bc] = 1, YE[N.g.Cc] = 1, YE[N.g.Ec] = 1, YE[N.g.Fc] = 1, YE), UE = /^(_|ga_|google_|gtag\.|firebase_).*$/, $E = function (a) {
      this.Zb = a;
      this.nd = new JE; this.h = void 0; this.F = new PD; this.B = this.C = void 0; this.W = !1; this.ae = void 0; this.Zd = !1; this.Ig = 0; this.K = !1
    }; aa = $E.prototype; aa.gm = function (a, b, c) {
      var d = this, e = cp(this.Zb); if (e) if (c.eventMetadata.is_external_event && "_" === a.charAt(0)) c.onFailure(); else {
        a !== N.g.na && a !== N.g.Ja && VE(a) && M(58); aF(c.h); var f = new vp(e, a, c); f.metadata.event_start_timestamp_ms = b; var g = [N.g.R]; (wp(f, N.g.eb, U(f.s, N.g.eb)) || ID(f)) && g.push(N.g.I); Q(71) && ID(f) && g.push(N.g.N); var k = function () { Zj(function () { d.hm(f) }, g) };
        Q(7) && Q(10) ? tn(k) : k()
      } else c.onFailure()
    }; aa.hm = function (a) {
      this.B = a; try {
        if (TE(a.target.ba)) M(28), a.isAborted = !0; else if (Q(44)) { var b; var c = pk(vk()), d = c && c.parent; b = d ? pk(d) : void 0; if (b && Da(b.destinations)) for (var e = 0; e < b.destinations.length; e++)if (TE(b.destinations[e])) { M(125); a.isAborted = !0; break } } if (0 <= sn.Vk.replace(/\s+/g, "").split(",").indexOf(a.eventName)) a.isAborted = !0; else { var f = KD(a); f && f.blacklisted && (a.isAborted = !0) } var g = C.location.protocol; "http:" != g && "https:" != g && (M(29), a.isAborted = !0);
        Cc && "preview" == Cc.loadPurpose && (M(30), a.isAborted = !0); Q(50) && (a.isAborted = !0); Up(a); var k = oi.grl; k || (k = oE(), oi.grl = k); k() || (M(35), a.isAborted = !0); if (a.isAborted) { a.s.onFailure(); zb(); return } var m = { prefix: String(U(a.s, N.g.La, "")), path: String(U(a.s, N.g.Hc, "/")), flags: String(U(a.s, N.g.Ta, "")), domain: String(U(a.s, N.g.Sa, "auto")), Tb: Number(U(a.s, N.g.Ha, 63072E3)) }; a.metadata.cookie_options = m; bF(a); this.wk(a); this.F.Em(a); a.metadata.is_merchant_center ? a.metadata.euid_mode_enabled = !1 : U(a.s, N.g.af) ? a.metadata.euid_mode_enabled =
          !1 : wp(a, "ccd_add_1p_data", !1) && (a.metadata.euid_mode_enabled = !0); if (a.metadata.euid_mode_enabled && wp(a, "ccd_add_1p_data", !1)) { var n = a.s.B[N.g.Xd]; if (bj(n)) { var p = U(a.s, N.g.wa); null === p ? a.metadata.user_data_from_code = null : (n.enable_code && kb(p) && (a.metadata.user_data_from_code = p), kb(n.selectors) && !a.metadata.user_data_from_manual && (a.metadata.user_data_from_manual = aj(n.selectors))) } } var q = this.yj, r; U(a.s, N.g.Ab) && (S(N.g.R) || U(a.s, N.g.Za) || (a.h[N.g.Pi] = !0)); var t; var u; u = void 0 === u ? 3 : u; var v = z.location.href;
        if (v) { var w = Cn(v).search.replace("?", ""), x = xn(w, "_gl", !1, !0) || ""; t = x ? void 0 !== Zn(x, u) : !1 } else t = !1; t && ID(a) && LD(a, "glv", 1); if (a.eventName !== N.g.na) r = {}; else { U(a.s, N.g.Ab) && So(["aw", "dc"]); var y = kE(a), A = mE(a); r = Q(56) && Object.keys(y).length ? y : A } q.call(this, r); var B = a.eventName === N.g.na; B && (this.K = !0); a.eventName == N.g.na && (U(a.s, N.g.Ma, !0) ? (a.s.h[N.g.aa] && (a.s.C[N.g.aa] = a.s.h[N.g.aa], a.s.h[N.g.aa] = void 0, a.h[N.g.aa] = void 0), a.eventName = N.g.ic) : a.isAborted = !0); B && !a.isAborted && 0 < this.Ig++ && QD(17); var D =
          ab(Yk(a.s, N.g.aa, 1), "."); D && (a.h[N.g.ub] = D); var G = ab(Yk(a.s, N.g.aa, 2), "."); G && (a.h[N.g.sb] = G); var E = this.C, I = this.F, O = !this.Zd, R = this.h, W = U(a.s, N.g.Za); if (U(a.s, N.g.vb) && U(a.s, N.g.Jb)) W ? UD(a, W, 1) : (M(127), a.isAborted = !0); else { var ha = W ? 1 : 8; a.metadata.is_new_to_site = !1; W || (W = VD(a), ha = 3); W || (W = R, ha = 5); if (!W) { var X = S(N.g.R), T = RD(); W = !T.from_cookie || X ? T.vid : void 0; ha = 6 } W ? W = "" + W : (W = Vm(), ha = 7, a.metadata.is_first_visit = a.metadata.is_new_to_site = !0); UD(a, W, ha) } var ca = Math.floor(a.metadata.event_start_timestamp_ms /
            1E3), ja = void 0; a.metadata.is_new_to_site || (ja = eE(a) || E); var ia = La(U(a.s, N.g.Sc, 30)); ia = Math.min(475, ia); ia = Math.max(5, ia); var Ia = La(U(a.s, N.g.cf, 1E4)), sa = ZD(ja); a.metadata.is_first_visit = !1; a.metadata.is_session_start = !1; a.metadata.join_timer_sec = 0; sa && sa.ph && (a.metadata.join_timer_sec = Math.max(0, sa.ph - Math.max(0, ca - sa.Fe))); var Ea = !1; sa || (Ea = a.metadata.is_first_visit = !0, sa = { sessionId: String(ca), xc: 1, hd: !1, Fe: ca, vc: !1, ne: void 0 }); ca > sa.Fe + 60 * ia && (Ea = !0, sa.sessionId = String(ca), sa.xc++, sa.hd = !1, sa.ne =
              void 0); if (Ea) a.metadata.is_session_start = !0, I.wl(a); else if (I.ml() > Ia || a.eventName == N.g.ic) sa.hd = !0; a.metadata.euid_mode_enabled ? U(a.s, N.g.za) ? sa.vc = !0 : (sa.vc && (sa.ne = void 0), sa.vc = !1) : sa.vc = !1; var Qa = sa.ne; if (a.metadata.euid_mode_enabled) { var Va = U(a.s, N.g.Mc), zc = Va ? 1 : 8; Va || (Va = Qa, zc = 4); Va || (Va = Um(), zc = 7); var Ac = zc, fd = a.metadata.enhanced_client_id_source; if (void 0 === fd || Ac <= fd) a.h[N.g.Mc] = Va.toString(), a.metadata.enhanced_client_id_source = Ac } O ? (a.copyToHitData(N.g.zb, sa.sessionId), a.copyToHitData(N.g.Nd,
                sa.xc), a.copyToHitData(N.g.Md, sa.hd ? 1 : 0)) : (a.h[N.g.zb] = sa.sessionId, a.h[N.g.Nd] = sa.xc, a.h[N.g.Md] = sa.hd ? 1 : 0); a.metadata[N.g.Ve] = sa.vc ? 1 : 0; cF(a); if (!U(a.s, N.g.Jb) || !U(a.s, N.g.vb)) {
                  var te = "", ue = C.location; if (ue) { var qf = ue.pathname || ""; "/" != qf.charAt(0) && (qf = "/" + qf); te = ue.protocol + "//" + ue.hostname + qf + ue.search; if (Q(100)) try { te = decodeURI(te) } catch (hv) { } } a.copyToHitData(N.g.ra, te); var Ji = a.copyToHitData, cH = N.g.Ea, Ki; a: {
                    var iv = xm("_opt_expid", void 0, void 0, N.g.R)[0]; if (iv) {
                      var jv = decodeURIComponent(iv).split("$");
                      if (3 === jv.length) { Ki = jv[2]; break a }
                    } if (void 0 !== oi.ga4_referrer_override) Ki = oi.ga4_referrer_override; else { var kv = Ui("gtm.gtagReferrer." + a.target.ba), Im = C.referrer; if (Q(100)) try { Im = decodeURI(Im) } catch (hv) { } Ki = kv ? "" + kv : Im }
                  } Ji.call(a, cH, Ki || void 0); a.copyToHitData(N.g.xb, C.title); a.copyToHitData(N.g.Ia, (Cc.language || "").toLowerCase()); var lv = Wp(); a.copyToHitData(N.g.yb, lv.width + "x" + lv.height); Q(36) && a.copyToHitData(N.g.Pc); if (Q(90)) {
                    var gd; if (!gd) {
                      var Pc = {}, dH = Yn(!0); Pc.Jm = !!dH._up; var Jm = Io(); Pc.xl =
                        void 0 !== Jm.aw; Pc.yl = void 0 !== Jm.dc; Pc.zl = void 0 !== Jm.gbraid; var mv = Cn(z.location.href), eH = An(mv, "query", !1, void 0, "gad"); Pc.eh = void 0 !== eH; if (!Pc.eh) { var fH = mv.hash.replace("#", ""), gH = xn(fH, "gad", !1); Pc.eh = void 0 !== gH } var nv = C.referrer ? An(Cn(C.referrer), "host") : ""; Pc.ol = PE.test(nv); Pc.fl = QE.test(nv); gd = Pc
                    } (gd.Jm ? 0 : gd.xl || gd.yl || gd.zl || gd.eh || gd.fl || gd.ol) && a.copyToHitData(N.g.xi, "1")
                  }
                } a.metadata.create_dc_join = !1; a.metadata.create_google_join = !1; if (!(Q(61) && ID(a) || a.metadata.is_merchant_center ||
                  !1 === U(a.s, N.g.qb)) && mC() && S(N.g.I)) { var Li = wp(a, N.g.eb, U(a.s, N.g.eb)); Li = Li || HD(a); (a.metadata.is_session_start || U(a.s, N.g.Xe)) && (a.metadata.create_dc_join = !!Li); var ov; ov = a.metadata.join_timer_sec; Li && 0 === (ov || 0) && (a.metadata.join_timer_sec = 60, a.metadata.create_google_join = !0) } dF(a); ci.hasOwnProperty(a.eventName) && (a.metadata.is_ecommerce = !0, a.copyToHitData(N.g.Z), a.copyToHitData(N.g.qa));
        a.copyToHitData(N.g.ef); for (var pv = U(a.s, N.g.Ye) || [], Km = 0; Km < pv.length; Km++) { var qv = pv[Km]; if (qv.rule_result) { a.copyToHitData(N.g.ef, qv.traffic_type); QD(3); break } } if (!a.metadata.is_merchant_center && Bt(a.s)) { var rv = gE(a) || {}, iH = (ho(rv[N.g.oc], !!rv[N.g.T]) ? Yn(!0)._fplc : void 0) || (0 < xm("FPLC", void 0, void 0, N.g.R).length ? void 0 : "0"); a.h._fplc = iH } if (void 0 !== U(a.s, N.g.Id)) a.copyToHitData(N.g.Id); else {
          var sv = U(a.s, N.g.Kd), Lm, Mi; a: {
            if (jE) {
              var Mm = gE(a) || {}; if (Mm && Mm[N.g.T]) for (var tv = An(Cn(a.h[N.g.Ea]), "host",
                !0), Ni = Mm[N.g.T], Kg = 0; Kg < Ni.length; Kg++)if (Ni[Kg] instanceof RegExp) { if (Ni[Kg].test(tv)) { Mi = !0; break a } } else if (0 <= tv.indexOf(Ni[Kg])) { Mi = !0; break a }
            } Mi = !1
          } if (!(Lm = Mi)) { var Oi; if (Oi = sv) a: { for (var uv = sv.include_conditions || [], jH = An(Cn(a.h[N.g.Ea]), "host", !0), Nm = 0; Nm < uv.length; Nm++)if (uv[Nm].test(jH)) { Oi = !0; break a } Oi = !1 } Lm = Oi } Lm && (a.h[N.g.Id] = "1", QD(4))
        } ID(a) && (LD(a, "uc", kj()), (Jj() || Q(45)) && LD(a, "rnd", $m())); if (Q(27) && ID(a)) { wp(a, N.g.eb, !1) && LD(a, "gse", 1); !1 === U(a.s, N.g.qb) && LD(a, "ngs", 1); mj() && LD(a, "ga_rd", 1); mC() || LD(a, "ngst", 1); var vv = pj(); vv && LD(a, "etld", vv) } if (ID(a)) { var wv = tE ? nj() : ""; wv && LD(a, "gcsub", wv) } ID(a) && (Jj() || Q(45)) && ((Kj() || Q(49)) && LD(a, "gcd", mm(a.s)), U(a.s, N.g.oa) && LD(a, "adr", 1)); if (ID(a)) { var xv = kr(); xv && LD(a, "us_privacy", xv); var yv = am(); yv && LD(a, "gdpr", yv); var zv = $l(); zv && LD(a, "gdpr_consent", zv) } Q(74) && ID(a) &&
          (a.h[N.g.Hi] = lj() || kj()); if (ID(a) && Q(87)) { var Av = Di; Av && LD(a, "tft", Number(Av)) } a: if (Q(7)) if (!on(z)) M(87); else if (void 0 !== qn) { M(85); var Bv = mn(); if (Bv) { if (Q(24)) { if (U(a.s, N.g.Rc) && !ID(a)) break a } else if (U(a.s, N.g.Rc)) break a; un(Bv, a) } else M(86) } if (Q(35)) { var Om = Tq(Sq()); Om || eF || (eF = !0, Al('AymqwRC7u88Y4JPvfIF2F37QKylC04248hLCdJAsh8xgOfe/dVJPV3XS3wLFca1ZMVOtnBfVjaCMTVudWM//5g4AAAB7eyJvcmlnaW4iOiJodHRwczovL3d3dy5nb29nbGV0YWdtYW5hZ2VyLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjk1MTY3OTk5LCJpc1RoaXJkUGFydHkiOnRydWV9'), Om = Tq(Sq())); Om && (a.h[N.g.Hb] = "1") } if (a.eventName == N.g.Ja) { var Dv = U(a.s, N.g.cb), kH = U(a.s, N.g.tb), Ev = void 0; Ev = a.h[Dv]; kH(Ev || U(a.s, Dv)); a.isAborted = !0 } a.copyToHitData(N.g.za);
        a.copyToHitData(N.g.Na); zp(a); OE(a); var Fv = U(a.s, N.g.vb); Fv && QD(12); a.metadata.em_event && QD(14); var Lg = pk(vk()); (Fv || zk(Lg) || Lg && Lg.parent && Lg.context && 5 === Lg.context.source) && QD(19); !this.K && a.metadata.em_event && QD(18); var Pm = a.metadata.event_usage; if (Da(Pm)) for (var Qm = 0; Qm < Pm.length; Qm++)QD(Pm[Qm]); var Gv = Ab("GA4_EVENT"); Gv && (a.h._eu = Gv); if (a.metadata.speculative || a.isAborted) { a.s.onFailure(); zb(); return } var lH = this.yj, Hv, mH = this.h, Rm; a: {
          var Sm = fE(a); if (Sm) {
            if (dE(Sm, a)) { Rm = Sm; break a } M(25); a.isAborted =
              !0
          } Rm = void 0
        } var nH = Rm; Hv = { clientId: YD(a, mH), Kf: nH }; lH.call(this, Hv); this.Zd = !0; this.Bm(a); if (ID(a)) { var oH = a.metadata.is_conversion; if ("page_view" === a.eventName || oH) XE(this, a, N.g.I), Q(71) && XE(this, a, N.g.N) } this.F.Eh(); this.ae = fF(a, this.ae); a.copyToHitData(N.g.lg); U(a.s, N.g.Rc) && (a.h[N.g.Rc] = !0, Q(32) && ID(a) || WE(a, N.g.yb)); if (a.isAborted) { a.s.onFailure(); zb(); return } this.bj(a); a.s.onSuccess()
      } catch (hv) { a.s.onFailure() } zb()
    }; aa.bj = function (a) { this.nd.add(a) }; aa.yj = function (a) {
      var b = a.clientId, c = a.Kf;
      b && c && (this.h = b, this.C = c)
    }; aa.flush = function () { this.nd.flush() }; aa.Bm = function (a) {
      var b = this; if (!this.W) {
        var c = S(N.g.R); Xj([N.g.R], function () {
          var d = S(N.g.R); if (c ^ d && b.B && b.C && b.h) {
            var e = b.h; if (d) { var f = VD(b.B); if (f) { b.h = f; var g = eE(b.B); g && (b.C = aE(g, b.C, b.B)) } else XD(b.h, b.B), TD(b.h, !0); dE(b.C, b.B); var k = {}; k[N.g.Xe] = e; var m = Pu(b.Zb, N.g.od, k); Su(m, a.s.eventId, {}); } else {
              b.C = void 0; b.h = void 0;
              z.gaGlobal = {};
            } c = d
          }
        }); this.W = !0
      }
    }; aa.wk = function (a) { a.eventName !== N.g.Ja && this.F.vk(a) }; var bF = function (a) { function b(c, d) { Zh[c] || void 0 === d || (a.h[c] = d) } l(a.s.C, b); l(a.s.h, b) }, cF = function (a) { var b = Zk(a.s), c = function (d, e) { ZE[d] && (a.h[d] = e) }; kb(b[N.g.Dc]) ? l(b[N.g.Dc], function (d, e) { c((N.g.Dc + "_" + d).toLowerCase(), e) }) : l(b, c) }, dF = function (a) {
      var b = function (c) { return !!c && c.conversion }; a.metadata.is_conversion = b(KD(a)); a.metadata.is_first_visit && (a.metadata.is_first_visit_conversion =
        b(KD(a, "first_visit"))); a.metadata.is_session_start && (a.metadata.is_session_start_conversion = b(KD(a, "session_start")))
    }, fF = function (a, b) {
      var c = void 0;
      return c
    }, eF = !1; function aF(a) { l(a, function (c) { "_" === c.charAt(0) && delete a[c] }); var b = a[N.g.Na] || {}; l(b, function (c) { "_" === c.charAt(0) && delete b[c] }) }; var gF = function (a) { if (Q(66) && "prerendering" in C ? C.prerendering : "prerender" === C.visibilityState) return !1; a(); return !0 }, hF = function (a) { if (!gF(a)) { var b = !1, c = function () { !b && gF(a) && (b = !0, Oc(C, "visibilitychange", c), Q(66) && Oc(C, "prerenderingchange", c), M(55)) }; Nc(C, "visibilitychange", c); Q(66) && Nc(C, "prerenderingchange", c); M(54) } }; var jF = function (a, b) { hF(function () { var c = cp(a); if (c) { var d = iF(c, b); Tv(a, d) } }); }; function iF(a, b) { var c = function () { }; var d = new $E(a.id), e = "MC" === a.prefix; c = function (f, g, k, m) { e && (m.eventMetadata.is_merchant_center = !0); d.gm(g, k, m) }; hk || kF(a, d, b); return c }
    function kF(a, b, c) { var d = b.F, e = {}, f = { eventId: c, eventMetadata: (e.batch_on_navigation = !0, e) }; d.jm(function () { MD = !0; Sv.flush(); 1E3 <= d.yf() && Cc.sendBeacon && Uv(N.g.od, {}, a.id, f); b.flush(); d.zj(function () { MD = !1; d.zj() }) }); }; var lF = iF; function nF(a, b, c) { var d = this; } nF.D = "internal.gtagConfig"; function oF() { var a = {}; return a };
    function qF(a, b) { } qF.M = "gtagSet"; function rF(a, b) { } rF.M = "injectHiddenIframe"; var sF = {};
    function uF(a, b, c, d) { } var vF = Object.freeze({ dl: 1, id: 1 }), wF = {};
    function xF(a, b, c, d) { } uF.M = "injectScript"; xF.D = "internal.injectScript"; function yF(a) { var b = !0; return b } yF.M = "isConsentGranted"; var zF = function () { var a = jh(function (b) { this.h.h.log("error", b) }); a.M = "JSON"; return a }; function AF(a) { var b = void 0; return od(b) } AF.D = "internal.legacyParseUrl"; var BF = function () { return !1 }, CF = {
      getItem: function (a) { var b = null; return b }, setItem: function (a,
        b) { return !1 }, removeItem: function (a) { }
    }; function DF() { } DF.M = "logToConsole"; function EF(a, b) { } EF.D = "internal.mergeRemoteConfig"; function FF(a, b, c) { c = void 0 === c ? !0 : c; var d = []; return d } FF.D = "internal.parseCookieValuesFromString"; function GF(a) {
      var b = void 0; if ("string" !== typeof a) return; a && 0 === a.indexOf("//") && (a = C.location.protocol + a); if ("function" === typeof URL) {
        var c; a: {
          var d; try { d = new URL(a) } catch (w) { c = void 0; break a } for (var e = {}, f = Array.from(d.searchParams), g = 0; g < f.length; g++) { var k = f[g][0], m = f[g][1]; e.hasOwnProperty(k) ? "string" === typeof e[k] ? e[k] = [e[k], m] : e[k].push(m) : e[k] = m } c = od({
            href: d.href, origin: d.origin, protocol: d.protocol, username: d.username, password: d.password, host: d.host,
            hostname: d.hostname, port: d.port, pathname: d.pathname, search: d.search, searchParams: e, hash: d.hash
          })
        } return c
      } var n; try { n = Cn(a) } catch (w) { return } if (!n.protocol || !n.host) return; var p = {}; if (n.search) for (var q = n.search.replace("?", "").split("&"), r = 0; r < q.length; r++) { var t = q[r].split("="), u = t[0], v = decodeURIComponent(t.splice(1).join("=")).replace(/\+/g, " "); p.hasOwnProperty(u) ? "string" === typeof p[u] ? p[u] = [p[u], v] : p[u].push(v) : p[u] = v } n.searchParams = p; n.origin = n.protocol + "//" + n.host; n.username = ""; n.password =
        ""; b = od(n); return b
    } GF.M = "parseUrl"; function HF(a) { K(J(this), ["preHit:!PixieMap"], arguments); var b = pd(a, this.h, 1).nj(), c = lb(b.s.h); lb(b.h, c); var d = lb(b.metadata); d.syn_or_mod = !0; var e = { eventMetadata: d }, f = b.s.eventId, g = Pu(b.target.ba, b.eventName, c); Su(g, f, e); } HF.D = "internal.processAsNewEvent"; function IF(a, b, c) { var d; return d } IF.D = "internal.pushToDataLayer"; function JF(a, b) { var c = !1; return c } JF.M = "queryPermission"; function KF() { var a = ""; return a } KF.M = "readCharacterSet"; function LF() { return ni.ia } LF.D = "internal.readDataLayerName"; function MF() { var a = ""; return a } MF.M = "readTitle"; function NF(a, b) { var c = this; K(J(this), ["destinationId:!string", "callback:!Fn"], arguments), Ap(a, function (d) { b.invoke(c.h, od(d, c.h, 1)) }); } NF.D = "internal.registerCcdCallback"; function OF(a) { return !0 } OF.D = "internal.registerDestination"; var PF = Object.freeze(["config", "event", "get", "set"]); function QF(a, b, c) { } QF.D = "internal.registerGtagCommandListener"; function RF(a, b) { var c = !1; return c } RF.D = "internal.removeDataLayerEventListener"; function SF(a, b) { }
    SF.D = "internal.removeFormData"; function TF() { } TF.M = "resetDataLayer"; function UF(a, b, c, d) {
      K(J(this), ["destinationIds:!*", "eventName:!*", "eventParameters:?PixieMap", "messageContext:?PixieMap"], arguments); var e = c ? pd(c) : {}, f = pd(a); Array.isArray(f) || (f = [f]); b = String(b); var g = d ? pd(d) : {}, k = this.h.h; g.originatingEntity = rA(k); for (var m = 0; m < f.length; m++) {
        var n = f[m]; if ("string" === typeof n) {
          var p =
            lb(e), q = lb(g), r = Pu(n, b, p); Su(r, g.eventId || k.eventId, q)
        }
      }
    } UF.D = "internal.sendGtagEvent"; function VF(a, b, c) { } VF.M = "sendPixel"; function WF(a, b) { } WF.D = "internal.setAnchorHref"; function XF(a, b, c, d) { var e = this; d = void 0 === d ? !0 : d; var f = !1; return f } XF.M = "setCookie"; function YF(a, b) { } YF.M = "setCorePlatformServices"; function ZF(a) { } ZF.M = "setDefaultConsentState"; function $F(a, b) { } $F.D = "internal.setDelegatedConsentType"; function aG(a, b) { } aG.D = "internal.setFormAction"; function bG(a, b, c) { return !1 } bG.M = "setInWindow"; function cG(a, b, c) { K(J(this), ["targetId:!string", "name:!string", "value:!*"], arguments); var d = up(a) || {}; d[b] = pd(c, this.h); var e = a; sp || tp(); rp[e] = d; } cG.D = "internal.setProductSettingsParameter"; function dG(a, b, c) { K(J(this), ["targetId:!string", "name:!string", "value:!*"], arguments); for (var d = b.split("."), e = Xv(a), f = 0; f < d.length - 1; f++) { if (void 0 === e[d[f]]) e[d[f]] = {}; else if (!kb(e[d[f]])) throw Error("setRemoteConfigParameter failed, path contains a non-object type: " + d[f]); e = e[d[f]] } e[d[f]] = pd(c, this.h, 1); } dG.D = "internal.setRemoteConfigParameter"; function eG(a, b, c, d) { var e = this; } eG.M = "sha256"; function fG(a, b, c) { }
    fG.D = "internal.sortRemoteConfigParameters"; var gG = {}, hG = {}; gG.M = "templateStorage"; gG.getItem = function (a) { var b = null; L(this, "access_template_storage"); var c = this.h.h; if (!c) throw Error("invalid program state"); var d = c.Yc(); hG[d] && (b = hG[d].hasOwnProperty("gtm." + a) ? hG[d]["gtm." + a] : null); return b };
    gG.setItem = function (a, b) { L(this, "access_template_storage"); var c = this.h.h; if (!c) throw Error("invalid program state"); var d = c.Yc(); hG[d] = hG[d] || {}; hG[d]["gtm." + a] = b; };
    gG.removeItem = function (a) { L(this, "access_template_storage"); var b = this.h.h; if (!b) throw Error("invalid program state"); var c = b.Yc(); if (!hG[c] || !hG[c].hasOwnProperty("gtm." + a)) return; delete hG[c]["gtm." + a]; }; gG.clear = function () { L(this, "access_template_storage"); var a = this.h.h; if (!a) throw Error("invalid program state"); delete hG[a.Yc()]; }; function iG(a, b) { var c = !1; K(J(this), ["regex:!OpaqueValue", "testString:!string"], arguments); if (!(a.h instanceof RegExp)) return !1; c = a.h.test(b); return c } iG.D = "internal.testRegex"; var jG = function (a) { var b; return b }; function kG(a) { } kG.M = "updateConsentState"; var lG; function mG(a, b, c) { lG = lG || new th; lG.add(a, b, c) } function nG(a, b) { var c = lG = lG || new th; if (c.B.hasOwnProperty(a)) throw "Attempting to add a private function which already exists: " + a + "."; if (c.h.hasOwnProperty(a)) throw "Attempting to add a private function with an existing API name: " + a + "."; c.B[a] = Ba(b) ? Qg(a, b) : Rg(a, b) }
    function oG() { return function (a) { var b; var c = lG; if (c.h.hasOwnProperty(a)) b = c.get(a, this); else { var d; if (d = c.B.hasOwnProperty(a)) { var e = !1, f = this.h.h; if (f) { var g = f.Yc(); if (g) { 0 !== g.indexOf("__cvt_") && (e = !0); } } else e = !0; d = e } if (d) { var k = c.B.hasOwnProperty(a) ? c.B[a] : void 0; b = k } else throw Error(a + " is not a valid API name."); } return b } }; var pG = function () {
      var a = function (c) { return nG(c.D, c) }, b = function (c) { return mG(c.M, c) }; b(hz); b(nz); b(dA); b(gA); b(hA); b(lA); b(mA); b(oA); b(zF()); b(pA); b(lD); b(mD); b(AD); b(BD); b(CD); b(FD); b(qF); b(rF); b(uF); b(yF); b(DF); b(GF); b(JF); b(KF); b(MF); b(VF); b(XF); b(ZF); b(bG); b(eG); b(gG); b(kG); mG("Math", Wg()); mG("Object", rh); mG("TestHelper", vh()); mG("assertApi", Sg); mG("assertThat", Tg); mG("decodeUri", Xg); mG("decodeUriComponent", Yg); mG("encodeUri", Zg); mG("encodeUriComponent", $g); mG("fail", fh); mG("generateRandom",
        gh); mG("getTimestamp", hh); mG("getTimestampMillis", hh); mG("getType", ih); mG("makeInteger", kh); mG("makeNumber", lh); mG("makeString", mh); mG("makeTableMap", nh); mG("mock", qh); mG("fromBase64", kD, !("atob" in z)); mG("localStorage", CF, !BF()); mG("toBase64", jG, !("btoa" in z)); a(kz); a(sz); a(Ez); a(Lz); a(Qz); a(Tz); a(bA); a(eA); a(jA); a(nA); a(qA); a(tA); a(vA); a(CA); a(NA); a(SA); a(XA); a(fB); a(jB); a(uB); a(HB); a(ah); a(JB); a(iD); a(nD); a(oD); a(tD); a(vD); a(xD); a(zD); a(DD); a(ED); a(nF); a(xF); a(Sz); a(EF); a(HF); a(NF); a(OF); a(QF);
      a(RF); a(SF); a(UF); a($F); a(cG); a(dG); a(fG); a(iG); nG("internal.GtagSchema", oF()); Q(48) && b(YF); Q(77) && a(BA); Q(70) &&
        a(pD); Q(78) && a(rD); Q(79) && a(sD); Q(83) && a(aG); Q(86) && a(WF); Q(89) && a(uD); Q(93) && a(FF); Q(96) && a(yD); Q(98) && a(LF); Q(99) && a(uA); Q(102) && a(AF); Q(105) && a(IF); Q(106) && a(jD); return oG()
    }; var fz; function qG() { fz.h.h.K = function (a, b, c) { oi.SANDBOXED_JS_SEMAPHORE = oi.SANDBOXED_JS_SEMAPHORE || 0; oi.SANDBOXED_JS_SEMAPHORE++; try { return a.apply(b, c) } finally { oi.SANDBOXED_JS_SEMAPHORE-- } } } function rG(a) { void 0 !== a && l(a, function (b, c) { for (var d = 0; d < c.length; d++) { var e = c[d].replace(/^_*/, ""); Gi[e] = Gi[e] || []; Gi[e].push(b) } }) }; var sG = encodeURI, Y = encodeURIComponent, tG = function (a, b, c) { Mc(a, b, c) }, uG = function (a, b) { if (!a) return !1; var c = An(Cn(a), "host"); if (!c) return !1; for (var d = 0; b && d < b.length; d++) { var e = b[d] && b[d].toLowerCase(); if (e) { var f = c.length - e.length; 0 < f && "." != e.charAt(0) && (f--, e = "." + e); if (0 <= f && c.indexOf(e, f) == f) return !0 } } return !1 }, vG = function (a, b, c) {
      for (var d = {}, e = !1, f = 0; a && f < a.length; f++)a[f] && a[f].hasOwnProperty(b) &&
        a[f].hasOwnProperty(c) && (d[a[f][b]] = a[f][c], e = !0); return e ? d : null
    }; var Z = { m: {} };
    Z.m.access_template_storage = ["google"], function () { (function (a) { Z.__access_template_storage = a; Z.__access_template_storage.o = "access_template_storage"; Z.__access_template_storage.isVendorTemplate = !0; Z.__access_template_storage.priorityOverride = 0; Z.__access_template_storage.isInfrastructure = !1; Z.__access_template_storage.runInSiloedMode = !1 })(function () { return { assert: function () { }, J: function () { return {} } } }) }();
    Z.m.c = ["google"], function () { (function (a) { Z.__c = a; Z.__c.o = "c"; Z.__c.isVendorTemplate = !0; Z.__c.priorityOverride = 0; Z.__c.isInfrastructure = !1; Z.__c.runInSiloedMode = !0 })(function (a) { Cy(a.vtp_value, "c", a.vtp_gtmEventId); return a.vtp_value }) }();
    Z.m.e = ["google"], function () { (function (a) { Z.__e = a; Z.__e.o = "e"; Z.__e.isVendorTemplate = !0; Z.__e.priorityOverride = 0; Z.__e.isInfrastructure = !1; Z.__e.runInSiloedMode = !0 })(function (a) { return String(a.vtp_gtmCachedValues.event) }) }(); Z.m.v = ["google"], function () { (function (a) { Z.__v = a; Z.__v.o = "v"; Z.__v.isVendorTemplate = !0; Z.__v.priorityOverride = 0; Z.__v.isInfrastructure = !1; Z.__v.runInSiloedMode = !1 })(function (a) { var b = a.vtp_name; if (!b || !b.replace) return !1; var c = uy(b.replace(/\\\./g, "."), a.vtp_dataLayerVersion || 1), d = void 0 !== c ? c : a.vtp_defaultValue; Cy(d, "v", a.vtp_gtmEventId); return d }) }();






    Z.m.process_dom_events = ["google"], function () {
      function a(b, c, d) { return { targetType: c, eventName: d } } (function (b) { Z.__process_dom_events = b; Z.__process_dom_events.o = "process_dom_events"; Z.__process_dom_events.isVendorTemplate = !0; Z.__process_dom_events.priorityOverride = 0; Z.__process_dom_events.isInfrastructure = !1; Z.__process_dom_events.runInSiloedMode = !1 })(function (b) {
        for (var c = b.vtp_targets || [], d = b.vtp_createPermissionError, e = {}, f = 0; f < c.length; f++) { var g = c[f]; e[g.targetType] = e[g.targetType] || []; e[g.targetType].push(g.eventName) } return {
          assert: function (k,
            m, n) { if (!e[m]) throw d(k, {}, "Prohibited event target " + m + "."); if (-1 === e[m].indexOf(n)) throw d(k, {}, "Prohibited listener registration for DOM event " + n + "."); }, J: a
        }
      })
    }();
    Z.m.detect_youtube_activity_events = ["google"], function () {
      function a(b, c) { return { options: { fixMissingApi: !!c.fixMissingApi } } } (function (b) { Z.__detect_youtube_activity_events = b; Z.__detect_youtube_activity_events.o = "detect_youtube_activity_events"; Z.__detect_youtube_activity_events.isVendorTemplate = !0; Z.__detect_youtube_activity_events.priorityOverride = 0; Z.__detect_youtube_activity_events.isInfrastructure = !1; Z.__detect_youtube_activity_events.runInSiloedMode = !1 })(function (b) {
        var c = !!b.vtp_allowFixMissingJavaScriptApi,
        d = b.vtp_createPermissionError; return { assert: function (e, f) { if (!c && f && f.fixMissingApi) throw d(e, {}, "Prohibited option: fixMissingApi."); }, J: a }
      })
    }();




    Z.m.detect_link_click_events = ["google"], function () {
      function a(b, c) { return { options: c } } (function (b) { Z.__detect_link_click_events = b; Z.__detect_link_click_events.o = "detect_link_click_events"; Z.__detect_link_click_events.isVendorTemplate = !0; Z.__detect_link_click_events.priorityOverride = 0; Z.__detect_link_click_events.isInfrastructure = !1; Z.__detect_link_click_events.runInSiloedMode = !1 })(function (b) {
        var c = b.vtp_allowWaitForTags, d = b.vtp_createPermissionError; return {
          assert: function (e, f) {
            if (!c && f && f.waitForTags) throw d(e,
              {}, "Prohibited option waitForTags.");
          }, J: a
        }
      })
    }();
    Z.m.read_container_data = ["google"], function () { (function (a) { Z.__read_container_data = a; Z.__read_container_data.o = "read_container_data"; Z.__read_container_data.isVendorTemplate = !0; Z.__read_container_data.priorityOverride = 0; Z.__read_container_data.isInfrastructure = !1; Z.__read_container_data.runInSiloedMode = !1 })(function () { return { assert: function () { }, J: function () { return {} } } }) }();
    Z.m.listen_data_layer = ["google"], function () {
      function a(b, c) { return { eventName: c } } (function (b) { Z.__listen_data_layer = b; Z.__listen_data_layer.o = "listen_data_layer"; Z.__listen_data_layer.isVendorTemplate = !0; Z.__listen_data_layer.priorityOverride = 0; Z.__listen_data_layer.isInfrastructure = !1; Z.__listen_data_layer.runInSiloedMode = !1 })(function (b) {
        var c = b.vtp_accessType, d = b.vtp_allowedEvents || [], e = b.vtp_createPermissionError; return {
          assert: function (f, g) {
            if (!h(g)) throw e(f, { eventName: g }, "Event name must be a string.");
            if (!("any" === c || "specific" === c && 0 <= d.indexOf(g))) throw e(f, { eventName: g }, "Prohibited listen on data layer event.");
          }, J: a
        }
      })
    }();


    Z.m.get_url = ["google"], function () {
      function a(b, c, d) { return { component: c, queryKey: d } } (function (b) { Z.__get_url = b; Z.__get_url.o = "get_url"; Z.__get_url.isVendorTemplate = !0; Z.__get_url.priorityOverride = 0; Z.__get_url.isInfrastructure = !1; Z.__get_url.runInSiloedMode = !1 })(function (b) {
        var c = "any" === b.vtp_urlParts ? null : []; c && (b.vtp_protocol && c.push("protocol"), b.vtp_host && c.push("host"), b.vtp_port && c.push("port"), b.vtp_path && c.push("path"), b.vtp_extension && c.push("extension"), b.vtp_query && c.push("query"), b.vtp_fragment &&
          c.push("fragment")); var d = c && "any" !== b.vtp_queriesAllowed ? b.vtp_queryKeys || [] : null, e = b.vtp_createPermissionError; return {
            assert: function (f, g, k) {
              if (g) { if (!h(g)) throw e(f, {}, "URL component must be a string."); if (c && 0 > c.indexOf(g)) throw e(f, {}, "Prohibited URL component: " + g); if ("query" === g && d) { if (!k) throw e(f, {}, "Prohibited from getting entire URL query when query keys are specified."); if (!h(k)) throw e(f, {}, "Query key must be a string."); if (0 > d.indexOf(k)) throw e(f, {}, "Prohibited query key: " + k); } } else if (c) throw e(f,
                {}, "Prohibited from getting entire URL when components are specified.");
            }, J: a
          }
      })
    }();


    Z.m.gct = ["google"], function () {
      function a(d) { for (var e = [], f = 0; f < d.length; f++)try { e.push(new RegExp(d[f])) } catch (g) { } return e } function b(d) { return d.replace(/[.*+\-?^${}()|[\]\\]/g, "\\$&") } function c(d) { for (var e = [], f = 0; f < d.length; f++) { var g = d[f].matchValue, k; switch (d[f].matchType) { case "BEGINS_WITH": k = "^" + b(g); break; case "ENDS_WITH": k = b(g) + "$"; break; case "EQUALS": k = "^" + b(g) + "$"; break; case "REGEX": k = g; break; default: k = b(g) }e.push(k) } return e } (function (d) {
        Z.__gct = d; Z.__gct.o = "gct"; Z.__gct.isVendorTemplate =
          !0; Z.__gct.priorityOverride = 0; Z.__gct.isInfrastructure = !1; Z.__gct.runInSiloedMode = !0
      })(function (d) {
        var e = {}, f = d.vtp_sessionDuration; 0 < f && (e[N.g.Sc] = f); e[N.g.Bd] = d.vtp_eventSettings; e[N.g.Zf] = d.vtp_dynamicEventSettings; e[N.g.eb] = 1 === d.vtp_googleSignals; e[N.g.mg] = d.vtp_foreignTld; e[N.g.kg] = 1 === d.vtp_restrictDomain; e[N.g.Ye] = d.vtp_internalTrafficResults; var g = N.g.Da, k = d.vtp_linker; k && k[N.g.T] && (k[N.g.T] = a(k[N.g.T])); e[g] = k; var m = N.g.Kd, n = d.vtp_referralExclusionDefinition; n && n.include_conditions && (n.include_conditions =
          a(n.include_conditions)); e[m] = n; var p = qk(d.vtp_trackingId), q = Xv(p); if (!Q(101)) { var r = q.cross_domain_conditions; if (r) { r.length && "object" === typeof r[0] && (r = c(r)); var t = {}; e[N.g.Da] = (t[N.g.T] = a(r), t[N.g.wb] = !0, t[N.g.oc] = !0, t[N.g.Kb] = "query", t) } var u = q.referral_exclusion_conditions; u && (u.length && "object" === typeof u[0] && (u = c(u)), e[N.g.Kd] = { include_conditions: a(u) }) } Yv(p, e); jF(p, d.vtp_gtmEventId); F(d.vtp_gtmOnSuccess)
      })
    }();


    Z.m.get = ["google"], function () { (function (a) { Z.__get = a; Z.__get.o = "get"; Z.__get.isVendorTemplate = !0; Z.__get.priorityOverride = 0; Z.__get.isInfrastructure = !1; Z.__get.runInSiloedMode = !1 })(function (a) { var b = a.vtp_settings, c = b.eventParameters || {}, d = String(a.vtp_eventName), e = {}; e.eventId = a.vtp_gtmEventId; e.priorityId = a.vtp_gtmPriorityId; a.vtp_deferrable && (e.deferrable = !0); var f = Pu(String(b.streamId), d, c); Su(f, e.eventId, e); a.vtp_gtmOnSuccess() }) }();





    var QH = {}; QH.dataLayer = Vi; QH.callback = function (a) { Fi.hasOwnProperty(a) && Ba(Fi[a]) && Fi[a](); delete Fi[a] }; QH.bootstrap = 0; QH._spx = !1;
    function RH() { oi[nk()] = oi[nk()] || QH; tk(); xk() || l(yk(), function (a, b) { Et(a, b.transportUrl, b.context); M(92) }); Ua(Gi, Z.m); Q(97) && pk(vk()); Af = Qf }
    (function (a) {
      function b() { m = C.documentElement.getAttribute("data-tag-assistant-present"); gx(m) && (k = g.sk) } if (!z["__TAGGY_INSTALLED"]) { var c = !1; if (C.referrer) { var d = Cn(C.referrer); c = "cct.google" === zn(d, "host") } if (!c) { var e = xm("googTaggyReferrer"); c = e.length && e[0].length } c && (z["__TAGGY_INSTALLED"] = !0, Jc("https://cct.google/taggy/agent.js")) } if (Ai) a(); else {
        var f = function (u) {
          var v = "GTM", w = "GTM"; ui ? (v = "OGT", w = "GTAG") : Ai && (w = v = "OPT"); var x = z["google.tagmanager.debugui2.queue"]; x || (x = [],
            z["google.tagmanager.debugui2.queue"] = x, Jc("https://" + ni.Oe + "/debug/bootstrap?id=" + Uf.ctid + "&src=" + w + "&cond=" + u + "&gtm=" + cn())); var y = { messageType: "CONTAINER_STARTING", data: { scriptSource: Dc, containerProduct: v, debug: !1, id: Uf.ctid, destinations: kk() } }; y.data.resume = function () { a() }; ni.Qj && (y.data.initialPublish = !0); x.push(y)
        }, g = { Om: 1, tk: 2, Ek: 3, Sj: 4, sk: 5 }, k = void 0, m = void 0, n = An(z.location, "query", !1, void 0, "gtm_debug"); gx(n) && (k = g.tk); if (!k && C.referrer) { var p = Cn(C.referrer); "tagassistant.google.com" === zn(p, "host") && (k = g.Ek) } if (!k) {
          var q =
            xm("__TAG_ASSISTANT"); q.length && q[0].length && (k = g.Sj)
        } k || b(); if (!k && hx(m)) { var r = function () { if (t) return !0; t = !0; b(); k && Dc ? f(k) : a() }, t = !1; Nc(C, "TADebugSignal", function () { r() }, !1); z.setTimeout(function () { r() }, 200) } else k && Dc ? f(k) : a()
      }
    })(function () {
      try {
        rk(); if (Q(29)) { } tj().B(); Zl(); var b = ok(); if (dk().canonical[b]) {
          var c = oi.zones; c && c.unregisterChild(jk());
        } else {
          (Q(7) || Q(8) || Q(21) || Q(18)) && rn(); xt(); for (var d = data.resource || {}, e = d.macros || [], f = 0; f < e.length; f++)pf.push(e[f]); for (var g = d.tags || [], k = 0; k < g.length; k++)tf.push(g[k]); for (var m = d.predicates || [], n = 0; n < m.length; n++)sf.push(m[n]); for (var p = d.rules || [], q = 0; q < p.length; q++) {
            for (var r = p[q], t = {}, u = 0; u < r.length; u++) {
              var v = r[u][0];
              t[v] = Array.prototype.slice.call(r[u], 1); nf(5) || nf(7) ? ("if" !== v && "unless" !== v || zf(t[v]), yb("TAGGING", 22)) : nf(6) && yb("TAGGING", 23)
            } rf.push(t)
          } vf = Z; wf = $y; Yf = new Xf; var w = data.sandboxed_scripts, x = data.security_groups, y = data.infra; a: {
            var A = data.runtime || [], B = data.runtime_lines; fz = new Ke; qG(); of = ez(); var D = fz, G = pG(), E = new ed("require", G); E.Bb(); D.h.h.set("require", E); for (var I = [], O = 0; O < A.length; O++) {
              var R = A[O]; if (!Da(R) || 3 > R.length) { if (0 === R.length) continue; break a } B && B[O] && B[O].length && Jf(R, B[O]); try {
                fz.execute(R),
                Q(57) && Hk && 50 === R[0] && I.push(R[1])
              } catch (Ji) { }
            } Q(57) && (Bf = I)
          } if (void 0 !== w) for (var W = ["sandboxedScripts"], ha = 0; ha < w.length; ha++) { var X = w[ha].replace(/^_*/, ""); Gi[X] = W } rG(x); if (void 0 !== y) for (var T = 0; T < y.length; T++)Hi[y[T]] = !0; RH(); if (Q(45)) { for (var ca = jj["7"], ja = ca ? ca.split("|") : [], ia = {}, Ia = 0; Ia < ja.length; Ia++)ia[ja[Ia]] = !0; for (var sa = 0; sa < Rj.length; sa++) { var Ea = Rj[sa], Qa = ia[Ea] ? "granted" : "denied"; Aj().implicit(Ea, Qa) } } fx(); Rt = !1; St = 0; if ("interactive" == C.readyState && !C.createEventObject || "complete" ==
            C.readyState) Ut(); else { Nc(C, "DOMContentLoaded", Ut); Nc(C, "readystatechange", Ut); if (C.createEventObject && C.documentElement.doScroll) { var Va = !0; try { Va = !z.frameElement } catch (Ji) { } Va && Vt() } Nc(z, "load", Ut) } uw = !1; "complete" === C.readyState ? ww() : Nc(z, "load", ww); Hk && (Ck(Uk), z.setInterval(Tk, 864E5)); Ck(bz); Ck(uu); Ck(Xr); Ck(Rv); Ck(Fu); Ck(Jt); Ck(en);
          Ck(Ht); Ck(Bu); Q(57) && Ck(xu); Cx(); ij(1); cv(); Ei = Ra(); QH.bootstrap = Ei; if (Q(29)) { }
        }
      } catch (Ji) { if (ij(4), Hk) { var qf = Ok(!0, !0); Mc(qf) } }
    });

  })()

</script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag() { dataLayer.push(arguments); }
  gtag('js', new Date());

  gtag('config', 'G-T8GWK081T9');
</script>
<!--/html_preserve-->

<style type="text/css">
  code {
    white-space: pre-wrap;
  }

  span.smallcaps {
    font-variant: small-caps;
  }

  span.underline {
    text-decoration: underline;
  }

  div.column {
    display: inline-block;
    vertical-align: top;
    width: 50%;
  }

  div.hanging-indent {
    margin-left: 1.5em;
    text-indent: -1.5em;
  }

  ul.task-list {
    list-style: none;
  }

</style>







<style type="text/css">


  button.galaxy-link {
    font-size: 1.1rem;
    overflow-wrap: anywhere;
    padding: 3px 3px 3px;
    margin: 4px;
    border-radius: 12px;
    border: outset;
    background-color: white;
    border-color: #D3D3D3;
    color: #2780e3;
  }

  .galaxy-links-holder {
    display: inline-block;
    padding: 0px;
    color: #2780e3;
  }

  .galaxy-links-holder li {
    display: inline-block;
    padding: 0px 0px;
  }

  .galaxy-links {
    padding: 0;
    color: #2780e3;
  }

  .open span.button {
    background-image: url(data:image/svg+xml;charset=US-ASCII,%3C%3Fxml%20version%3D%221.0%22%20encoding%3D%22iso-8859-1%22%3F%3E%3C!DOCTYPE%20svg%20PUBLIC%20%22-%2F%2FW3C%2F%2FDTD%20SVG%201.1%2F%2FEN%22%20%22http%3A%2F%2Fwww.w3.org%2FGraphics%2FSVG%2F1.1%2FDTD%2Fsvg11.dtd%22%3E%3Csvg%20version%3D%221.1%22%20id%3D%22Layer_1%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20xmlns%3Axlink%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2Fxlink%22%20x%3D%220px%22%20y%3D%220px%22%20%20width%3D%2214px%22%20height%3D%2214px%22%20viewBox%3D%220%200%2014%2014%22%20style%3D%22enable-background%3Anew%200%200%2014%2014%3B%22%20xml%3Aspace%3D%22preserve%22%3E%3Crect%20y%3D%225%22%20style%3D%22fill%3A%23FFFFFF%3B%22%20width%3D%2214%22%20height%3D%224%22%2F%3E%3C%2Fsvg%3E);
  }

  .closed span.button {
    background-image: url(data:image/svg+xml;charset=US-ASCII,%3C%3Fxml%20version%3D%221.0%22%20encoding%3D%22iso-8859-1%22%3F%3E%3C!DOCTYPE%20svg%20PUBLIC%20%22-%2F%2FW3C%2F%2FDTD%20SVG%201.1%2F%2FEN%22%20%22http%3A%2F%2Fwww.w3.org%2FGraphics%2FSVG%2F1.1%2FDTD%2Fsvg11.dtd%22%3E%3Csvg%20version%3D%221.1%22%20id%3D%22Layer_1%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20xmlns%3Axlink%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2Fxlink%22%20x%3D%220px%22%20y%3D%220px%22%20%20width%3D%2214px%22%20height%3D%2214px%22%20viewBox%3D%220%200%2014%2014%22%20style%3D%22enable-background%3Anew%200%200%2014%2014%3B%22%20xml%3Aspace%3D%22preserve%22%3E%3Cpolygon%20fill%3D%22%23FFF%22%20points%3D%2214%2C5%209%2C5%209%2C0%205%2C0%205%2C5%200%2C5%200%2C9%205%2C9%205%2C14%209%2C14%209%2C9%2014%2C9%20%22%2F%3E%3Cg%3E%3C%2Fg%3E%3Cg%3E%3C%2Fg%3E%3Cg%3E%3C%2Fg%3E%3Cg%3E%3C%2Fg%3E%3Cg%3E%3C%2Fg%3E%3Cg%3E%3C%2Fg%3E%3Cg%3E%3C%2Fg%3E%3Cg%3E%3C%2Fg%3E%3Cg%3E%3C%2Fg%3E%3Cg%3E%3C%2Fg%3E%3Cg%3E%3C%2Fg%3E%3Cg%3E%3C%2Fg%3E%3Cg%3E%3C%2Fg%3E%3Cg%3E%3C%2Fg%3E%3Cg%3E%3C%2Fg%3E%3C%2Fsvg%3E);
  }

  span.button {
    width: 18px;
    height: 18px;
    vertical-align: middle;
    position: absolute;

    background-color: rgba(0, 0, 0, .3);
    border-radius: 0.5em;
    background-position: center center;
    background-repeat: no-repeat;
    transform: scale(0.7);
  }

  button.edam-button {
    overflow-wrap: anywhere;
    font-size: 1.1rem;
    display: inline-block;
    padding: 3px 3px 3px;
    margin: 4px;
    border-radius: 12px;
    border: outset;
    color: #2780e3;
    border-color: #D3D3D3;
    background-color: white;
  }

  .tags {
    font-size: 1.2rem;
    display: inline-block;
    padding: 3px 3px 3px;
    margin: 4px;
    border-radius: 12px;

    color: black;
    border-color: #D3D3D3;
    background-color: white;
  }

  a.publication-title {
    overflow: hidden;
    width: 150px;
    display: -webkit-box;
    -webkit-line-clamp: 2;
    -webkit-box-orient: vertical;
    font-size: 1.1rem;
  }

  .title {
    font-size: 1.3rem;
  }

  .version {
    font-size: 1.1rem;
    overflow-wrap: anywhere;
    display: inline-block;
    padding: 3px 3px 3px;
    margin: 4px;
    border-radius: 12px;
    border-color: #D3D3D3;
    background-color: white;
  }

  .containers {
    overflow-wrap: anywhere;
    font-size: 1.1rem;
  }


  .description-text {
    font-size: 1.1rem;
    display: -webkit-box;
    -webkit-line-clamp: 5;
    -webkit-box-orient: vertical;
    max-width: 200px;
    overflow: hidden;
    text-overflow: ellipsis;
  }

  .biotools {
    padding: 10px 10px 10px;
    border-radius: 12px;
    background-color: #FFA500;
    border: outset;
  }

</style>



<style type="text/css">
  .main-container {
    max-width: 940px;
    margin-left: auto;
    margin-right: auto;
  }

  img {
    max-width: 100%;
  }

  .tabbed-pane {
    padding-top: 12px;
  }

  .html-widget {
    margin-bottom: 20px;
  }

  button.code-folding-btn:focus {
    outline: none;
  }

  summary {
    display: list-item;
  }

  details>summary>p:only-child {
    display: inline;
  }

  pre code {
    padding: 0;
  }

</style>


<style type="text/css">
  .dropdown-submenu {
    position: relative;
  }

  .dropdown-submenu>.dropdown-menu {
    top: 0;
    left: 100%;
    margin-top: -6px;
    margin-left: -1px;
    border-radius: 0 6px 6px 6px;
  }

  .dropdown-submenu:hover>.dropdown-menu {
    display: block;
  }

  .dropdown-submenu>a:after {
    display: block;
    content: " ";
    float: right;
    width: 0;
    height: 0;
    border-color: transparent;
    border-style: solid;
    border-width: 5px 0 5px 5px;
    border-left-color: #cccccc;
    margin-top: 5px;
    margin-right: -10px;
  }

  .dropdown-submenu:hover>a:after {
    border-left-color: #adb5bd;
  }

  .dropdown-submenu.pull-left {
    float: none;
  }

  .dropdown-submenu.pull-left>.dropdown-menu {
    left: -100%;
    margin-left: 10px;
    border-radius: 6px 0 6px 6px;
  }

</style>



<style type="text/css">
  .main-container {
    max-width: 1500px;
    margin-left: auto;
    margin-right: auto;
  }

</style>

<p>
  <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapse1" role="button" aria-expanded="false" aria-controls="collapse1">
    Can’t find the software you need? Click here to see other options for
    finding software </a>
</p>
<div class="collapse" id="collapse1">
  <div class="card card-body">
    <ul>
      <li>
        <p><a href="https://biocontainers.pro/registry">BioContainers
            registry</a>: discover and access containers for software tools</p>
      </li>
      <li>
        <p><a href="https://toolshed.g2.bx.psu.edu/">Galaxy toolshed</a>:
          discover tools which can be requested for installation on Galaxy
          Australia</p>
      </li>
      <li>
        <p><a href="https://dockstore.org/search?entryType=tools&amp;searchMode=Files">Dockstore
            tool search</a>: discover containers for single tools</p>


<p>
  <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapse2" role="button" aria-expanded="false" aria-controls="collapse2">
    Click to find out how to request installation of software on ToolFinder
    listed infrastructures </a>
</p>
<div id="collapse2" class="collapse">
  <div class="card card-body">
    <table class="table table-striped" style="width: auto !important; ">
      <thead>
        <tr>
          <th style="text-align:left;">
            Infrastructure
          </th>
          <th style="text-align:left;">
            Process
          </th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td style="text-align:left;">
            <a href="https://usegalaxy.org.au/">Galaxy Australia</a>
          </td>
          <td style="text-align:left;">
            Complete a <a href="https://request.usegalaxy.org.au/">tool install
              request</a>
          </td>
        </tr>
        <tr>
          <td style="text-align:left;">
            <a href="https://nci.org.au/">NCI</a>
          </td>
          <td style="text-align:left;">
            <a href="https://opus.nci.org.au/display/Help/5.+Software+Applications">Contact
              NCI</a>
          </td>
        </tr>
        <tr>
          <td style="text-align:left;">
            <a href="https://pawsey.org.au/">Pawsey</a>
          </td>
          <td style="text-align:left;">
            Visit the <a href="https://support.pawsey.org.au/portal/servicedesk/customer/portal">Helpdesk
              Portal</a> or email: <a href="help@pawsey.org.au">help@pawsey.org.au</a>
            with your request
          </td>
        </tr>
        <tr>
          <td style="text-align:left;">
            <a href="https://www.qriscloud.org.au/">QRIScloud</a> / <a href="https://rcc.uq.edu.au/">UQ-RCC</a>
          </td>
          <td style="text-align:left;">
            Complete a <a href="https://support.qcif.edu.au/Request-for-installation-of-QRIScloud-HPC">tool
              install request</a>
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
<p><strong>Please note:</strong></p>
<ul>
  <li><code>NCI (if89)</code> below refers to the <a href="https://australianbiocommons.github.io/ables/if89/"><code>Australian BioCommons Tools and Workflows repository</code></a>,
    located in project allocation if89 at the National Computational
    Infrastructure (NCI).</li>
</ul>
<script>
  $(document).ready(function () {
    $('[data-toggle="popover"]').popover();
  });
</script>
<p><a name="tool_information"></a></p>

<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-76e9f3a78aaa5b0ec715" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-76e9f3a78aaa5b0ec715">{"x":{"filter":"none","vertical":false,"class":"cell-border stripe","extensions":["Buttons"],"data":[["<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Mass screening of contigs for antimicrobial resistance or virulence genes.\">ABRicate<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"De novo genome sequence assembler.\">ABySS<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">aggregate scores in intervals2<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"The Assisted Model Building with Energy Refinement tool refers to two things: a set of molecular mechanical force fields for the simulation of biomolecules (which are in the public domain, and are used in a variety of simulation programs); and a package of molecular simulation programs which includes source code and demos.\">amber<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Software package specially developed for the study of genes’ primary structure. It uses gene sequences downloaded from public databases, as FASTA and GenBank, and it applies a set of statistical and visualization methods in different ways, to reveal information about codon context, codon usage, nucleotide repeats within open reading frames (ORFeome) and others.\">anaconda<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">anndata<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"This tool can get annotation for a generic set of IDs, using the Bioconductor annotation data packages. Supported organisms are human, mouse, rat, fruit fly and zebrafish. The org.db packages that are used here are primarily based on mapping using Entrez Gene identifiers. More information on the annotation packages can be found at the Bioconductor website, for example, information on the human annotation package (org.Hs.eg.db) can be found here.\">annotatemyids<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Rapid genome-wide identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genomes. It integrates and cross-links with a large number of in silico secondary metabolite analysis tools that have been published earlier.\">antiSMASH<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Convert various sequence formats to FASTA\">any2fasta<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Apollo is a genome annotation viewer and editor. Apollo allows researchers to explore genomic annotations at many levels of detail, and to perform expert annotation curation, all in a graphical environment.\">Apollo<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"ARAGORN detects tRNA, mtRNA info about tmRNA, and tmRNA genes\">aragorn<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Arriba is a command-line tool to detect gene fusions from RNA-Seq data based on the STAR aligner. In addition to fusions, it can detect exon duplications/inversions and truncations of genes (i.e., breakpoints in introns and intergenic regions). Arriba is the winner of the DREAM SMC-RNA Challenge.\">Arriba<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Java program for estimating a species tree given a set of unrooted gene trees.\">ASTRAL<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"AUGUSTUS is a eukaryotic gene prediction tool. It can integrate evidence, e.g. from RNA-Seq, ESTs, proteomics, but can also predict genes ab initio. The PPX extension to AUGUSTUS can take a protein sequence multiple sequence alignment as input to find new members of the family in a genome. It can be run through a web interface (see https://bio.tools/webaugustus), or downloaded and run locally.\">Augustus<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"AutoDock Vina is a new open-source program for drug discovery, molecular docking and virtual screening, offering multi-core capability, high performance and enhanced accuracy and ease of use.\">autodock vina<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">axt to concat fasta<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">axt to fasta<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">axt to lav 1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Rapid & standardized annotation of bacterial genomes, MAGs & plasmids\">Bakta<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"BamTools provides a fast, flexible C++ API & toolkit for reading, writing, and managing BAM files.\">bamtools<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Bamutil provides a serie of programs to work on SAM/BAM files.\">bamutil diff<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"GUI program that allows users to interact with the assembly graphs made by de novo assemblers such as Velvet, SPAdes, MEGAHIT and others. It visualises assembly graphs, with connections, using graph layout algorithms.\">bandage<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">barchart gnuplot<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Predict the location of ribosomal RNA genes in genomes. It supports bacteria (5S,23S,16S), archaea (5S,5.8S,23S,16S), mitochondria (12S,16S) and eukaryotes (5S,5.8S,28S,18S).\">barrnap<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"BBMap is a fast splice-aware aligner for RNA and DNA.  It is faster than almost all short-read aligners, yet retains unrivaled sensitivity and specificity, particularly for reads with many errors and indels.\">bbmap<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A tool for filling the gap created by genomic data processing/analysis by rebasing some analysis results against the parent features which were originally analysed.\">bcbiogff<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"BCFtools is a set of utilities that manipulate variant calls in the Variant Call Format (VCF) and its binary counterpart BCF. All commands work transparently with both VCFs and BCFs, both uncompressed and BGZF-compressed.\">BCFtools<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Beagle is a software package that performs genotype calling, genotype phasing, imputation of ungenotyped markers, and identity-by-descent segment detection.\">Beagle<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"BEAGLE is a high-performance library that can perform the core calculations at the heart of most Bayesian and Maximum Likelihood phylogenetics packages.\">beagle-lib<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"The Bayesian Evolutionary Analysis Sampling Trees is a cross-platform program for Bayesian analysis of molecular sequences using MCMC (Markov chain Monte Carlo). It is entirely orientated towards rooted, time-measured phylogenies inferred using strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies but is also a framework for testing evolutionary hypotheses without conditioning on a single tree topology.\">BEAST<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Bayesian phylogenetic analysis of molecular sequences. It estimates rooted, time-measured phylogenies using strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies but is also a framework for testing evolutionary hypotheses without conditioning on a single tree topology. It uses Markov chain Monte Carlo (MCMC) to average over tree space, so that each tree is weighted proportional to its posterior probability. It includes a graphical user-interface for setting up standard analyses and a suit of programs for analysing the results.\">BEAST2<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">bed to bigBed<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Convert a BED format file of the proteins from a proteomics search database into a tabular format for the Multiomics Visualization Platform (MVP).\">bed to protein map<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">bed2gff1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"BEDTools is an extensive suite of utilities for comparing genomic features in BED format.\">BEDTools<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"The Bellerophon pipeline, improving de novo transcriptomes and removing chimeras.\n\nBellerophon is a pipeline created to remove falsely assembled chimeric transcripts in de novo transcriptome assemblies.\n\nThe pipeline can be downloaded as a vragrant virtual machine (https://app.vagrantup.com/bellerophon/boxes/bellerophon). This is recommended, as it avoids backwards compatibility problems with TransRate\">bellerophon<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Trim, circularise, orient and filter long read bacterial genome assemblies\">berokka<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">bftools<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">bio-searchio-hmmer<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Bio3D is an R package containing utilities for the analysis of protein structure, sequence and trajectory data.\">bio3d<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">biobakery workflows<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Tools for early stage NGS alignment file processing including fast sorting and duplicate marking.\">biobambam<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"This package includes basic tools for reading biom-format files, accessing and subsetting data tables from a biom object, as well as limited support for writing a biom-object back to a biom-format file. The design of this API is intended to match the python API and other tools included with the biom-format project, but with a decidedly \"R flavor\" that should be familiar to R users. This includes S4 classes and methods, as well as extensions of common core functions/methods.\">biom-format<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">Bionano Solve<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A collection of Perl modules that facilitate the development of Perl scripts for bioinformatics applications.  It provides software modules for many of the typical tasks of bioinformatics programming.\">bioperl<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Biopython is a set of freely available tools for biological computation written in Python by an international team of developers.\">biopython<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"BioTransformer is a freely available web server that supports accurate, rapid and comprehensive in silico metabolism prediction.\">BioTransformer<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A tool that finds regions of similarity between biological sequences. The program compares nucleotide or protein sequences to sequence databases and calculates the statistical significance.\">BLAST<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A tool that finds regions of similarity between biological sequences. The program compares nucleotide or protein sequences to sequence databases and calculates the statistical significance.\">BLAST+<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A tool that finds regions of similarity between biological sequences. The program compares nucleotide or protein sequences to sequence databases and calculates the statistical significance.\">BLAST2<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Fast, accurate spliced alignment of DNA sequences.\">blat<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"detect blocks of overlapping reads using a gaussian-distribution approach\">Blockbuster<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">BlockClust<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Boost is a set of libraries for the C++ programming language that provides support for tasks and structures such as linear algebra, pseudorandom number generation, multithreading, image processing, regular expressions, and unit testing.\">Boost<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Bowtie is an ultrafast, memory-efficient short read aligner.\">Bowtie<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.\">Bowtie2<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">bp genbank2gff3<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Runs Breseq software on a set of fastq files.\">breseq<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">bundle collections<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Provides measures for quantitative assessment of genome assembly, gene set, and transcriptome completeness based on evolutionarily informed expectations of gene content from near-universal single-copy orthologs.\">BUSCO<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Fast, accurate, memory-efficient aligner for short and long sequencing reads\">bwa<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Bwa-mem2 is the next version of the bwa-mem algorithm in bwa. It produces alignment identical to bwa and is ~1.3-3.1x faster depending on the use-case,  dataset and the running machine.\">bwa mem2<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">BWA-meth<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">bwakit<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Tools for manipulating biological data, particularly multiple sequence alignments.\">bx-python<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Consensus Cancer Driver Gene Caller | Next-generation sequencing has allowed identification of millions of somatic mutations in human cancer cells. A key challenge in interpreting cancer genomes is to distinguish drivers of cancer development among available genetic mutations. To address this issue, we present the first web-based application, consensus cancer driver gene caller (C3), to identify the consensus driver genes using six different complementary strategies, i.e., frequency-based, machine learning-based, functional bias-based, clustering-based, statistics model-based, and network-based strategies. This application allows users to specify customized operations when calling driver genes, and provides solid statistical evaluations and interpretable visualizations on the integration results\">c3s<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Cactus is a reference-free whole-genome multiple alignment program.\">Cactus<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Annotation of peaklists generated by xcms, rule based annotation of isotopes and adducts, isotope validation, EIC correlation based tagging of unknown adducts and fragments.\">CAMERA<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"De-novo assembly tool for long read chemistry like Nanopore data and PacBio data.\">Canu<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Web-based contig assembly.\">cap3<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Implements statistical and computational tools for analyzing mass spectrometry imaging datasets, including methods for efficient pre-processing, spatial segmentation, and classification.\">Cardinal<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Contig Annotation Tool (CAT) and Bin Annotation Tool (BAT) are pipelines for the taxonomic classification of long DNA sequences and metagenome assembled genomes (MAGs/bins) of both known and (highly) unknown microorganisms, as generated by contemporary metagenomics studies. The core algorithm of both programs involves gene calling, mapping of predicted ORFs against the nr protein database, and voting-based classification of the entire contig / MAG based on classification of the individual ORFs.\">CAT<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">categorize elements satisfying criteria<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Cluster a nucleotide dataset into representative sequences.\">cd-hit<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Tool for quantifying data from biological images, particularly in high-throughput experiments.\">CellProfiler<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">cellranger<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"cellxgene (pronounced \"cell-by-gene\") is an interactive data explorer for single-cell transcriptomics datasets, such as those coming from the Human Cell Atlas.\">cellxgene<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"CheckM provides a set of tools for assessing the quality of genomes recovered from isolates, single cells, or metagenomes.\">checkm<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Database of bioactive compounds, their quantitative properties and bioactivities (binding constants, pharmacology and ADMET, etc). The data is abstracted and curated from the primary scientific literature.\">ChEMBL<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Fast cheminformatics fingerprint search, at your fingertips.\n\nChemfp is a set of command-line tools and a Python library for fingerprint generation and high-performance similarity search.\n\nThere are two ways to try out chemfp. From the download page page you can request an evaluation copy of the most recent version of chemfp, or you can download an earlier version for no cost under the MIT license\">chemfp<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">chemical data sources<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"This package implements functions to retrieve the nearest genes around the peak, annotate the genomic region of the peak, statistical methods for estimating the significance of overlap among ChIP peak data sets, and incorporate GEO database to compare the own dataset with those deposited in database. The comparison can be used to infer cooperative regulation and thus can be used to generate hypotheses.\">chipseeker<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"An ultra fast, heuristic approach to detect conserved signals in extremely large pairwise genome comparisons (dotplot).\">chromeister<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">circexplorer<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Circos is tool for visualizing data in a circular format. It was developed for genomic data but can work for many other kinds of data as well.\">circos<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">clifinder<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">climate stripes<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Automatic generation of gene cluster comparison figures.\n\nGene cluster comparison figure generator.\n\nA d3 chart for generating gene cluster comparison figures.\n\nclinker is a pipeline for easily generating publication-quality gene cluster comparison figures.\n\nGiven a set of GenBank files, clinker will automatically extract protein translations, perform global alignments between sequences in each cluster, determine the optimal display order based on cluster similarity, and generate an interactive visualisation (using clustermap.js) that can be extensively tweaked before being exported as an SVG file.\n\nclustermap.js is an interactive, reusable d3 chart designed to visualise homology between multiple gene clusters.\">clinker<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Multiple sequence alignment software. The name is occassionally spelled as ClustalOmega, Clustal Ω, ClustalΩ, Clustal O, ClustalO.\">Clustal Omega<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Multiple sequence alignment software. Old deprecated versions. Even older versions were CLUSTAL and CLUSTAL V (ClustalV).\">clustalw<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"ColabFold databases are MMseqs2 expandable profile databases to generate diverse multiple sequence alignments to predict protein structures.\">ColabFold<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">collapse collections<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">collection column join<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">collection element identifiers<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">column arrange by header<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">column maker<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">compose text param<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">compress file<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">compute motif frequencies for all motifs<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">compute motifs frequency<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">concatenate multiple datasets<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">condense characters<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">CONVERTER neostorezip to neostore<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">correlation<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">Count occurrences of each record<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"CPAT (Coding-Potential Assessment Tool) is a logistic regression model-based classifier that can accurately and quickly distinguish protein-coding and noncoding RNAs using pure linguistic features calculated from the RNA sequences. CPAT takes as input the nucleotides sequences or genomic coordinates of RNAs and outputs the probabilities p (0 ≤ p ≤ 1), which measure the likelihood of protein coding.\">CPAT (Coding-Potential Assessment Tool)<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">ctsm fates<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">cuda<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"The NVIDIA CUDA® Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for deep neural networks. cuDNN provides highly tuned implementations for standard routines such as forward and backward convolution, pooling, normalization, and activation layers.\">cudnn<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Cufflinks assembles transcripts and estimates their abundances in RNA-Seq samples. It accepts aligned RNA-Seq reads and assembles the alignments into a parsimonious set of transcripts. Cufflinks then estimates the relative abundances of these transcripts based on how many reads support each one.\">Cufflinks<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Allows for persistent storage, access, exploration, and manipulation of Cufflinks high-throughput sequencing data. In addition, provides numerous plotting functions for commonly used visualizations.\">cummeRbund<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Generate customized protein sequence database from RNA-Seq data for proteomics search.\">customProDB<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">cut include exclude<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Find and remove adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.\">cutadapt<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Long Read based Human Genomic Structural Variation Detection with cuteSV | Long-read sequencing technologies enable to comprehensively discover structural variations (SVs). However, it is still non-trivial for state-of-the-art approaches to detect SVs with high sensitivity or high performance or both. Herein, we propose cuteSV, a sensitive, fast and lightweight SV detection approach. cuteSV uses tailored methods to comprehensively collect various types of SV signatures, and a clustering-and-refinement method to implement a stepwise SV detection, which enables to achieve high sensitivity without loss of accuracy. Benchmark results demonstrate that cuteSV has better yields on real datasets. Further, its speed and scalability are outstanding and promising to large-scale data analysis\">cuteSV<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"This package infers exact sequence variants (SVs) from amplicon data, replacing the commonly used and coarser OTU clustering approach. This pipeline inputs demultiplexed fastq files, and outputs the sequence variants and their sample-wise abundances after removing substitution and chimera errors. Taxonomic classification is available via a native implementation of the RDP naive Bayesian classifier.\">DADA2<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">datamash<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">dbbuilder<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">dcm2niix<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">DeepConsensus (CPU)<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">DeepConsensus (GPU)<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"User-friendly tools for the normalization and visualization of deep-sequencing data.\">DeepTools<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"DendroPy is a Python library for phylogenetic computing. It provides classes and functions for the simulation, processing, and manipulation of phylogenetic trees and character matrices, and supports the reading and writing of phylogenetic data in a range of formats.\">dendropy<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"R/Bioconductor package for differential gene expression analysis based on the negative binomial distribution. Estimate variance-mean dependence in count data from high-throughput sequencing assays and test for differential expression based on a model using the negative binomial distribution.\">deseq2<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"The package is focused on finding differential exon usage using RNA-seq exon counts between samples with different experimental designs. It provides functions that allows the user to make the necessary statistical tests based on a model that uses the negative binomial distribution to estimate the variance between biological replicates and generalized linear models for testing. The package also provides functions for the visualization and exploration of the results.\">DEXSeq<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Neural networks and interference correction enable deep proteome coverage in high throughput.\n\nDIA-NN - a fast and easy to use tool for processing data independent acquisition (DIA) proteomics data.\n\nNone required (for .raw, .mzML and .dia processing). Two executables are provided: DiaNN.exe (a command-line tool) and DIA-NN.exe (a GUI implemented as a wrapper for DiaNN.exe)\">DIA-NN<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"DIA-Umpire is an open source Java program for computational analysis of data independent acquisition (DIA) mass spectrometry-based proteomics data. It enables untargeted peptide and protein identification and quantitation using DIA data, and also incorporates targeted extraction to reduce the number of cases of missing quantitation.\">DIA-Umpire<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Sequence aligner for protein and translated DNA searches and functions as a drop-in replacement for the NCBI BLAST software tools. It is suitable for protein-protein search as well as DNA-protein search on short reads and longer sequences including contigs and assemblies, providing a speedup of BLAST ranging up to x20,000.\">diamond<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"diaPASEF is an appproch for parallel accumulation-serial fragmentation combined with data-independent acquisition.\">diapysef<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Compute differentially bound sites from multiple ChIP-seq experiments using affinity (quantitative) data. Also enables occupancy (overlap) analysis and plotting functions.\">diffbind<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">draw stacked barplots<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">drishti<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Provides a number of utility functions for handling single-cell (RNA-seq) data from droplet technologies such as 10X Genomics. This includes data loading, identification of cells from empty droplets, removal of barcode-swapped pseudo-cells, and downsampling of the count matrix.\">dropletutils<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">dwt<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"EasyBuild is a software build and installation framework that allows you to manage (scientific) software on High Performance Computing (HPC) systems in an efficient way.\">easybuild<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">ebi sra main<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Differential expression analysis of RNA-seq expression profiles with biological replication. Implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi-likelihood tests. As well as RNA-seq, it be applied to differential signal analysis of other types of genomic data that produce counts, including ChIP-seq, SAGE and CAGE.\">edger<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Differential expression analysis of RNA-seq expression profiles with biological replication. Implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi-likelihood tests. As well as RNA-seq, it be applied to differential signal analysis of other types of genomic data that produce counts, including ChIP-seq, SAGE and CAGE.\">edger-repenrich<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"For fast functional annotation of novel sequences. It uses precomputed orthologous groups and phylogenies from the eggNOG database to transfer functional information from fine-grained orthologs only. Its common uses include the annotation of novel genomes, transcriptomes or even metagenomic gene catalogs.\n\nThe use of orthology predictions for functional annotation permits a higher precision than traditional homology searches, as it avoids transferring annotations from close paralogs.\">eggNOG-mapper<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"This package implements the Ensemble of Gene Set Enrichment Analyses method for gene set testing.\">egsea<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Evaluation of an Open Source Registration Package for Automatic Contour Propagation in Online Adaptive Intensity-Modulated Proton Therapy of Prostate Cancer.\n\nHome : About : FAQ : wiki : Download : News : Legal stuff : Documentation.\n\nWelcome to elastix : a toolbox for rigid and nonrigid registration of images.\n\nelastix is open source software, based on the well-known Insight Segmentation and Registration Toolkit (ITK). The software consists of a collection of algorithms that are commonly used to solve (medical) image registration problems. The modular design of elastix allows the user to quickly configure, test, and compare different registration methods for a specific application. A command-line interface enables automated processing of large numbers of data sets, by means of scripting. Nowadays elastix is accompanied by SimpleElastix , making it available in languages like C++, Python, Java, R, Ruby, C# and Lua.\">elastix<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.\">EMBOSS (European Molecular Biology Open Software Suite)<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">ena upload<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"EncyclopeDIA is library search engine comprised of several algorithms for DIA data analysis and can search for peptides using either DDA-based spectrum libraries or DIA-based chromatogram libraries.\">EncyclopeDIA<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">ete3<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">ethercalc<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Integrated database covering the eukaryotic pathogens of the genera Cryptosporidium, Giardia, Leishmania, Neospora, Plasmodium, Toxoplasma, Trichomonas and Trypanosoma. The database portal offers an entry point to all these resources, and the opportunity to leverage orthology for searches across genera.\">eupathdb<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"ExaBayes is a software package for Bayesian phylogenetic tree inference. It is particularly suitable for large-scale analyses on computer clusters.\">ExaBayes<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Tool for phylogenomic analyses on supercomputers.\">ExaML<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A tool for pairwise sequence alignment. It enables alignment for DNA-DNA and DNA-protein pairs and also gapped and ungapped alignment.\">Exonerate<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">export remote<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Streaming tool for quantifying the abundances of a set of target sequences from sampled subsequences. Example applications include transcript-level RNA-Seq quantification, allele-specific/haplotype expression analysis (from RNA-Seq), transcription factor binding quantification in ChIP-Seq, and analysis of metagenomic data. It can be used to resolve ambiguous mappings in other high-throughput sequencing based applications.\">eXpress<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">f5c<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Experimental PacBio diploid assembler.\">Falcon (pb-assembly)<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">fasta compute length<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">fasta concatenate by species<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">fasta filter by length<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">fasta merge files and filter unique sequences<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">fasta stats<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">fasta to tabular<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">fastahack<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"FastANI is developed for fast alignment-free computation of whole-genome Average Nucleotide Identity (ANI). ANI is defined as mean nucleotide identity of orthologous gene pairs shared between two microbial genomes. FastANI supports pairwise comparison of both complete and draft genome assemblies.\">fastani<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Read huge FastQ and FastA files (both normal and gzipped) an demanipulate them.\">fastool<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A tool designed to provide fast all-in-one preprocessing for FastQ files. This tool is developed in C++ with multithreading supported to afford high performance.\">fastp<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"This tool aims to provide a QC report which can spot problems or biases which originate either in the sequencer or in the starting library material. It can be run in one of two modes. It can either run as a stand alone interactive application for the immediate analysis of small numbers of FastQ files, or it can be run in a non-interactive mode where it would be suitable for integrating into a larger analysis pipeline for the systematic processing of large numbers of files.\">FASTQC<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">fastqe<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences.\">FastTree<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Collection of command line tools for Short-Reads FASTA/FASTQ files preprocessing.\">FASTX-Toolkit<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Collection of command line tools for Short-Reads FASTA/FASTQ files preprocessing.\">FASTX-Toolkit<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">fermi-lite<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"HMM-based gene structure prediction (multiple genes, both chains); Program for predicting multiple genes in genomic DNA sequences.\">Fgenesh++<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"The package implements an algorithm for fast gene set enrichment analysis. Using the fast algorithm allows to make more permutations and get more fine grained p-values, which allows to use accurate stantard approaches to multiple hypothesis correction.\">fgsea<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">filter by fasta ids<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">filter tabular<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">filter transcripts via tracking<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Filtlong is a tool for filtering long reads by quality. It can take a set of long reads and produce a smaller, better subset. It uses both read length (longer is better) and read identity (higher is better) when choosing which reads pass the filter.\">filtlong<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"FlashLFQ is an ultrafast label-free quantification algorithm for mass-spectrometry proteomics.\">flashLFQ<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">flex<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Flye is a de novo assembler for single molecule sequencing reads, such as those produced by PacBio and Oxford Nanopore Technologies. It is designed for a wide range of datasets, from small bacterial projects to large mammalian-scale assemblies. The package represents a complete pipeline: it takes raw PB / ONT reads as input and outputs polished contigs.\">Flye<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">flymine<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Foldseek enables fast and sensitive comparisons of large structure sets. It reaches sensitivities similar to state-of-the-art structural aligners while being at least 20,000 times faster.\">Foldseek<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Web server which detects small molecule pockets by relying on the geometric alpha sphere theory. It also tracks pockets during molecular dynamics so to provide insight on pocket dynamics (mdpocket) and transposes mdpocket to the combined analysis of homologous structures (hpocket).\">fpocket<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Application for finding (fragmented) genes in short reads\">FragGeneScan<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Bayesian genetic variant detector designed to find small polymorphisms, specifically SNPs, indels, multi-nucleotide polymorphisms, and complex events (composite insertion and substitution events) smaller than the length of a short-read sequencing alignment.\">freebayes<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">fsom<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"funannotate is a pipeline for genome annotation (built specifically for fungi, but will also work with higher eukaryotes).\">funannotate<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Gene Annotation EVAluation.\">gaeval<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">Galaxy collection operations<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">Galaxy converters<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">Galaxy filter and sort<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">Galaxy join subtract and group<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">Galaxy text manipulation<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">galaxy sequence utils<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">galaxy-ops<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"The Genome Analysis Toolkit (GATK) is a set of bioinformatic tools for analyzing high-throughput sequencing (HTS) and variant call format (VCF) data. The toolkit is well established for germline short variant discovery from whole genome and exome sequencing data. GATK4 expands functionality into copy number and somatic analyses and offers pipeline scripts for workflows.  \n\nVersion 4 (GATK4) is open-source at https://github.com/broadinstitute/gatk.\">GATK<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Cleaning aligned sequences.\">gblocks<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Genome-wide Complex Trait Analysis. Estimate the proportion of phenotypic variance explained by genome- or chromosome-wide SNPs for complex traits (the GREML method), and has subsequently extended for many other analyses to better understand the genetic architecture of complex traits.\">gcta<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Software aimed at pairwise sequence comparison generating high quality results (equivalent to MUMmer) with controlled memory consumption and comparable or faster execution times particularly with long sequences.\">gecko<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"GEMINI (GEnome MINIng) is a flexible framework for exploring genetic variation in the context of the wealth of genome annotations available for the human genome. By placing genetic variants, sample phenotypes and genotypes, as well as genome annotations into an integrated database framework, GEMINI provides a simple, flexible, and powerful system for exploring genetic variation for disease and population genetics.\">gemini<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Gene Model Mapper is a homology-based gene prediction program. GeMoMa uses the annotation of protein-coding genes in a reference genome to infer the annotation of protein-coding genes in a target genome. Thereby, it utilizes amino acid sequence and intron position conservation. In addition, it allows to incorporate RNA-seq evidence for splice site prediction.\">gemoma<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">gene2exon1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">GeneBed Maf Fasta2<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">generate count matrix<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">generate pc lda matrix<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Reference-free profiling of polyploid genomes | We have developed GenomeScope 2.0, which applies classical insights from combinatorial theory to establish a detailed mathematical model of how k-mer frequencies will be distributed in heterozygous and polyploid genomes | Average k-mer coverage for polyploid genome | Upload results from running Jellyfish or KMC\">GenomeScope<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Free collection of bioinformatics tools for genome informatics.\">GenomeTools<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">genrich<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">get pdb file<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A fast and versatile toolkit for accurate de novo assembly of organelle genomes.\n\nThis toolkit assemblies organelle genome from genomic skimming data.\">getorganelle<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">gfa to fa<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"gfastats is a single fast and exhaustive tool for summary statistics and simultaneous genome assembly file manipulation. gfastats also allows seamless fasta/fastq/gfa conversion.\">gfastats<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">gff2bed1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">gff3sort<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Program for comparing, annotating, merging and tracking transcripts in GFF files.\">gffcompare<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"program for filtering, converting and manipulating GFF files\">gffread<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Plotting system for R, based on the grammar of graphics.\">ggplot2<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">ghostscript<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">GMAJ<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"GNU parallel is a shell tool for executing jobs in parallel using one or more computers. A job can be a single command or a small script that has to be run for each of the lines in the input. The typical input is a list of files, a list of hosts, a list of users, a list of URLs, or a list of tables. A job can also be a command that reads from a pipe. GNU parallel can then split the input and pipe it into commands in parallel.\">GNU parallel<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.\">GNU scientific library<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"GOEnrichment is a tool for performing GO enrichment analysis of gene sets, such as those obtained from RNA-seq or Microarray experiments, to help characterize them at the functional level. It is available in Galaxy Europe and as a stand-alone tool.\nGOEnrichment is flexible in that it allows the user to use any version of the Gene Ontology and any GO annotation file they desire. To enable the use of GO slims, it is accompanied by a sister tool GOSlimmer, which can convert annotation files from full GO to any specified GO slim.\nThe tool features an optional graph clustering algorithm to reduce the redundancy in the set of enriched GO terms and simplify its output.\nIt was developed by the BioData.pt / ELIXIR-PT team at the Instituto Gulbenkian de Ciência.\">goenrichment<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Detect Gene Ontology and/or other user defined categories which are over/under represented in RNA-seq data.\">goseq<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">gramenemart<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"GraPhlAn is a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees. GraPhlAn focuses on concise, integrative, informative, and publication-ready representations of phylogenetically- and taxonomically-driven investigation.\">GraPhlAn<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles. It is primarily designed for biochemical molecules like proteins, lipids and nucleic acids that have a lot of complicated bonded interactions, but since it is extremely fast at calculating the nonbonded interactions (that usually dominate simulations) many groups are also using it for research on non-biological systems, e.g. polymers.\">gromacs<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"a toolkit to classify genomes with the Genome Taxonomy Database.\n\nGTDB-Tk: a toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.\n\nGTDB-Tk is a software toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes based on the Genome Database Taxonomy GTDB. It is designed to work with recent advances that allow hundreds or thousands of metagenome-assembled genomes (MAGs) to be obtained directly from environmental samples. It can also be applied to isolate and single-cell genomes. The GTDB-Tk is open source and released under the GNU General Public License (Version 3).\">GTDB-TK<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">gtf2bedgraph<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">gtf2gene list<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Gubbins is a tool for rapid phylogenetic analysis of large samples of recombinant bacterial whole genome sequences.\">gubbins<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"HapCUT2 is a maximum-likelihood-based tool for assembling haplotypes from DNA sequence reads, designed to \"just work\" with excellent speed and accuracy across a range of long- and short-read sequencing technologies.\nThe output is in Haplotype block format described here: https://github.com/vibansal/HapCUT2/blob/master/outputformat.md\">hapcut2<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">hbvar<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Tool for single-species active module discovery.\">heinz<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"This tool provides functional annotation for a list of genes by connecting with DAVID database.\">hgv david<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"This tool can be used to analyze the patterns of linkage disequilibrium (LD) between polymorphic sites in a locus.\">hgv ldtools<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"This tool creates a link to the g:GOSt tool (Gene Group Functional Profiling), which provides functional profiling of gene lists.\">hgv linkToGProfile<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A web server for reproducible Hi-C, capture Hi-C and single-cell Hi-C data analysis, quality control and visualization.\n\nHiCExplorer — HiCExplorer 3.6 documentation.\n\nscHiCExplorer — scHiCExplorer 7 documentation.\n\nFree document hosting provided by Read the Docs.\">hicexplorer<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Remove CCS reads with remnant PacBio adapter sequences and convert outputs to a compressed .fastq (.fastq.gz).\">hifiadapterfilt<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Hifiasm: a haplotype-resolved assembler for accurate Hifi reads\">hifiasm<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">hifiasm meta<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Alignment program for mapping next-generation sequencing reads (both DNA and RNA) to a population of human genomes (as well as to a single reference genome).\">HISAT2<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">histogram<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">hmmcleaner<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"This tool is used for searching sequence databases for homologs of protein sequences, and for making protein sequence alignments. It implements methods using probabilistic models called profile hidden Markov models. The new HMMER3 project, HMMER is now as fast as BLAST for protein search.\">HMMER<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Horovod is a distributed deep learning training framework for TensorFlow, Keras, PyTorch, and Apache MXNet. The goal of Horovod is to make distributed deep learning fast and easy to use.\">horovod<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Python framework to process and analyse high-throughput sequencing (HTS) data\">HTSeq<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"The main purpose of HTSlib is to provide access to genomic information files, both alignment data (SAM, BAM, and CRAM formats) and variant data (VCF and BCF formats). The library also provides interfaces to access and index genome reference data in FASTA format and tab-delimited files with genomic coordinates. It is utilized and incorporated into both SAMtools and BCFtools.\">HTSlib<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"HUMAnN is a pipeline for efficiently and accurately profiling the presence/absence and abundance of microbial pathways in a community from metagenomic or metatranscriptomic sequencing data (typically millions of short DNA/RNA reads). This process, referred to as functional profiling, aims to describe the metabolic potential of a microbial community and its members. More generally, functional profiling answers the question “What are the microbes in my community-of-interest doing (or are capable of doing)?”\">HUMAnN<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"HUMAnN 2.0 is a pipeline for efficiently and accurately profiling the presence/absence and abundance of microbial pathways in a community from metagenomic or metatranscriptomic sequencing data (typically millions of short DNA/RNA reads). This process, referred to as functional profiling, aims to describe the metabolic potential of a microbial community and its members. More generally, functional profiling answers the question “What are the microbes in my community-of-interest doing (or capable of doing)?”\">HUMAnN2<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Software package for the analysis of genetic sequences using techniques in phylogenetics, molecular evolution, and machine learning.\">HyPhy<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">IDL<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">ImageMagick<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Improved Phased Assembler (IPA) is the official PacBio software for HiFi genome assembly. IPA was designed to utilize the accuracy of PacBio HiFi reads to produce high-quality phased genome assemblies\">Improved Phased Assembler (pbipa)<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Infernal (\"INFERence of RNA ALignment\") is for searching DNA sequence databases for RNA structure and sequence similarities. It is an implementation of a special case of profile stochastic context-free grammars called covariance models (CMs). A CM is like a sequence profile, but it scores a combination of sequence consensus and RNA secondary structure consensus, so in many cases, it is more capable of identifying RNA homologs that conserve their secondary structure more than their primary sequence.\">Infernal<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">insight-toolkit<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"High-performance visualization tool for interactive exploration of large, integrated datasets. It supports a wide variety of data types and format, including short-read alignments in the SAM/BAM format. Data can be viewed from local files or over the web via http.\">Integrated Genomics Viewer<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A tool to detect Integron in DNA sequences.\">integron finder<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">intermine<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">intermine galaxy exchange<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Scan sequences against the InterPro protein signature databases.\">InterProScan<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">Interval Maf Merged Fasta2<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">Interval2Maf pairwise1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">Interval2Maf1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Interactive assembly and analysis of RADseq datasets.\n\nipyrad: interactive assembly and analysis of RAD-seq data sets.\n\nWelcome to ipyrad, an interactive toolkit for assembly and analysis of restriction-site associated genomic data sets (e.g., RAD, ddRAD, GBS) for population genetic and phylogenetic studies.\n\nWelcome to ipyrad — ipyrad documentation.\">ipyrad<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Very efficient phylogenetic software for reconstructing maximum-likelihood trees and assessing branch supports with the ultrafast bootstrap approximation. It is based on the IQPNNI algorithm with 10-fold speedup together with substantially additional features.\">IQ-TREE<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Provides functions for creating an interactive Shiny-based graphical user interface for exploring data stored in SummarizedExperiment objects, including row- and column-level metadata. Particular attention is given to single-cell data in a SingleCellExperiment object with visualization of dimensionality reduction results.\">iSEE<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Automated identification of insertion sequence elements in prokaryotic genomes.\">ISEScan<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Enables identification of isoform switches with predicted functional consequences from RNA-seq data. Consequences can be chosen from a long list but includes protein domains gain/loss changes in NMD sensitivity etc. It directly supports import of data from Cufflinks/Cuffdiff, Kallisto, Salmon and RSEM but other transcript qunatification tools are easy to import as well.\">IsoformSwitchAnalyzeR<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Interpretation-oriented tool to manage the update and revision of variant annotation and classification.\n\niVar - DataBase of Genomics Variants.\">ivar<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Implementation of the Interval-Wise Testing (IWT) for omics data. This inferential procedure tests for differences in \"Omics\" data between two groups of genomic regions (or between a group of genomic regions and a reference center of symmetry), and does not require fixing location and scale at the outset.\">IWTomics<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"JAGS is Just Another Gibbs Sampler. It is a program for analysis of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation.\">jags<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"JASMINE (Jointly Accurate Sv Merging with Intersample Network Edges) is an automated pipeline for alignment and SV calling in long-read datasets. The tool is used to merge structural variants (SVs) across samples. Each sample has a number of SV calls, consisting of position information (chromosome, start, end, length), type and strand information, and a number of other values. Jasmine represents the set of all SVs across samples as a network, and uses a modified minimum spanning forest algorithm to determine the best way of merging the variants such that each merged variants represents a set of analogous variants occurring in different samples.\">jasminesv<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">Java Development Kit (JDK)<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">jbigkit<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Slick, speedy genome browser with a responsive and dynamic AJAX interface for visualization of genome data. Being developed by the GMOD project as a successor to GBrowse.\">JBrowse<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">JCVI<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A command-line algorithm for counting k-mers in DNA sequence.\">jellyfish<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">join files on column fuzzy<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">jq<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">jupyterlab<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A program for quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads. It is based on the novel idea of pseudoalignment for rapidly determining the compatibility of reads with targets, without the need for alignment.\">Kallisto<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Suite of tools that generate, analyse and compare k-mer spectra produced from sequence files\">KAT<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">kentutils<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"khmer is a set of command-line tools for working with DNA shotgun sequencing data from genomes, transcriptomes, metagenomes, and single cells. khmer can make de novo assemblies faster, and sometimes better. khmer can also identify (and fix) problems with shotgun data.\">khmer<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"KMC is a utility designed for counting k-mers (sequences of consecutive k symbols) in a set of reads from genome sequencing projects.\">KMC<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">kofamscan<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"System for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies. Previous attempts by other bioinformatics software to accomplish this task have often used sequence alignment or machine learning techniques that were quite slow, leading to the development of less sensitive but much faster abundance estimation programs. It aims to achieve high sensitivity and high speed by utilizing exact alignments of k-mers and a novel classification algorithm.\">kraken<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Kraken 2 is the newest version of Kraken, a taxonomic classification system using exact k-mer matches to achieve high accuracy and fast classification speeds. This classifier matches each k-mer within a query sequence to the lowest common ancestor (LCA) of all genomes containing the given k-mer. The k-mer assignments inform the classification algorithm.\">kraken2<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Krona creates interactive HTML5 charts of hierarchical data (such as taxonomic abundance in a metagenome).\">krona<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">kronatools<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Automated image analysis for developmental phenotyping of mouse embryos.\n\nLAMA (Lightweight Analysis of Morphological Abnormalities).\n\nWelcome to LAMA, an open source pipeline to automatically identify embryo dysmorphology from 3D volumetric images.\">LAMA (Lightweight Analysis of Morphological Abnormalities)<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A tool for (1) aligning two DNA sequences, and (2) inferring appropriate scoring parameters automatically.\">lastz<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">lav to bed1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"An Easy-To-Use Interactive Web Platform To Analyze and Visualize Label-Free Proteomics Data Preprocessed with MaxQuant.\n\nA tool for analysing label-free quantitative proteomics dataset https://bioinformatics.erc.monash.edu/apps/LFQ-Analyst/.\n\nLFQ-Analyst: An easy-to-use interactive web-platform to analyze and visualize proteomics data preprocessed with MaxQuant.\n\nLFQ-Analyst is an easy-to-use, interactive web application developed to perform differential expression analysis with “one click” and to visualize label-free quantitative proteomic datasets preprocessed with MaxQuant. LFQ-Analyst provides a wealth of user-analytic features and offers numerous publication-quality result output graphics and tables to facilitate statistical and exploratory analysis of label-free quantitative datasets\">LFQ-Analyst<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">liftOver1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Data analysis, linear models and differential expression for microarray data.\">limma<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"LINKS (Long Interval Nucleotide K-mer Scaffolder) is a genomics application for scaffolding genome assemblies with long reads, such as those produced by Oxford Nanopore Technologies Ltd. It can be used to scaffold high-quality draft genome assemblies with any long sequences (eg. ONT reads, PacBio reads, other draft genomes, etc). It is also used to scaffold contig pairs linked by ARCS/ARKS.\">LINKS<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"LoFreq* (i.e. LoFreq version 2) is a fast and sensitive variant-caller for inferring SNVs and indels from next-generation sequencing data. It makes full use of base-call qualities and other sources of errors inherent in sequencing (e.g. mapping or base/indel alignment uncertainty), which are usually ignored by other methods or only used for filtering.\">lofreq<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Longshot is a variant calling tool for diploid genomes using long error prone reads such as Pacific Biosciences (PacBio) SMRT and Oxford Nanopore Technologies (ONT). It takes as input an aligned BAM file and outputs a phased VCF file with variants and haplotype information. It can also genotype and phase input VCF files. It can output haplotype-separated BAM files that can be used for downstream analysis. Currently, it only calls single nucleotide variants (SNVs), but it can genotype indels if they are given in an input VCF.\">longshot<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">lpsolve<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"LTR_retriever is a highly accurate and sensitive program for identification of LTR retrotransposons; The LTR Assembly Index (LAI) is also included in this package.\">LTR retriever<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Model-based Analysis of ChIP-seq data.\">macs<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Model-based Analysis of ChIP-seq data.\">macs2<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">maf by block number1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">MAF filter<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">maf limit size1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">MAF Limit To Species1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">MAF Reverse Complement 1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">MAF split blocks by species1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">maf stats1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">MAF Thread For Species1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">MAF To BED1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">MAF To Fasta1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">MAF To Interval1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"MAFFT (Multiple Alignment using Fast Fourier Transform) is a high speed multiple sequence alignment program.\">mafft<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Computational tool to identify important genes from the recent genome-scale CRISPR-Cas9 knockout screens technology.\">MAGeCK<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Portable and easily configurable genome annotation pipeline. It’s purpose is to allow smaller eukaryotic and prokaryotic genome projects to independently annotate their genomes and to create genome databases.\">MAKER<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"MALDIquant is a complete analysis pipeline for matrix-assisted laser desorption/ionization-time-of-flight (MALDI-TOF) and other two-dimensional mass spectrometry data. In addition to commonly used plotting and processing methods it includes distinctive features, namely baseline subtraction methods such as morphological filters (TopHat) or the statistics-sensitive non-linear iterative peak-clipping algorithm (SNIP), peak alignment using warping functions, handling of replicated measurements as well as allowing spectra with different resolutions.\">MALDIquant<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Fast genome and metagenome distance estimation using MinHash.\">mash<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">master2pgSnp<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Whole genome assembly software. It combines the efficiency of the de Bruijn graph and Overlap-Layout-Consensus (OLC) approaches. MaSuRCA can assemble data sets containing only short reads from Illumina sequencing or a mixture of short reads and long reads (Sanger, 454).\">MaSuRCA<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Tool to import, process, clean, and compare mass spectrometry data.\">Matchms<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Matplotlib is a comprehensive library for creating static, animated, and interactive visualizations in Python. Matplotlib makes easy things easy and hard things possible.\">matplotlib<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Software for binning assembled metagenomic sequences based on an Expectation-Maximization algorithm.\">MaxBin2<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Quantitative proteomics software package designed for analyzing large mass-spectrometric data sets. It is specifically aimed at high-resolution MS data.\">MaxQuant<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Quantitative proteomics software package designed for analyzing large mass-spectrometric data sets. It is specifically aimed at high-resolution MS data.\">MaxQuant (biocontainers)<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"MCL is a clustering algorithm widely used in bioinformatics and gaining traction in other fields.\">mcl<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"MDAnalysis is an object-oriented python toolkit to analyze molecular dynamics trajectories generated by CHARMM, Gromacs, NAMD, LAMMPS, Amber or DL_POLY; it also reads other formats (e.g. PDB files and XYZ format trajectories; see the supported coordinate formats for the full list). It can write most of these formats, too, together with atom selections for use in Gromacs, CHARMM, VMD and PyMol\">mdanalysis<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">MDTraj<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"medaka is a tool to create consensus sequences and variant calls from nanopore sequencing data. This task is performed using neural networks applied a pileup of individual sequencing reads against a draft assembly.\">Medaka<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Single node assembler for large and complex metagenomics NGS reads, such as soil. It makes use of succinct de Bruijn graph to achieve low memory usage, whereas its goal is not to make memory usage as low as possible.\">megahit<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">merge cols<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Reference-free quality, completeness, and phasing assessment for genome assemblies.\n\nEvaluate genome assemblies with k-mers and more.\nOften, genome assembly projects have illumina whole genome sequencing reads available for the assembled individual.\n\nMerqury provides a set of tools for this purpose.\">Merqury<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Meryl is a tool for counting and working with sets of k-mers that was originally developed for use in the Celera Assembler and has since been migrated and maintained as part of Canu.\">meryl<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies | MetaBAT2 clusters metagenomic contigs into different \"bins\", each of which should correspond to a putative genome | MetaBAT2 uses nucleotide composition information and source strain abundance (measured by depth-of-coverage by aligning the reads to the contigs) to perform binning\">metabat<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">metabolicmine<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Galaxy workflow for differential abundance analysis of 16s metagenomic data.\n\nYou are over your disk quota. Tool execution is on hold until your disk usage drops below your allocated quota.\n\nThis history is empty. You can load your own data or get data from an external source\">MetaDEGalaxy<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"MetaEuk - sensitive, high-throughput gene discovery and annotation for large-scale eukaryotic metagenomics\">metaeuk<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Computational tool for profiling the composition of microbial communities from metagenomic shotgun sequencing data.\">MetaPhlAn<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"metaQuantome software suite analyzes the state of a microbiome by leveraging complex taxonomic and functional hierarchies to summarize peptide-level quantitative information. metaQuantome offers differential abundance analysis, principal components analysis, and clustered heat map visualizations, as well as exploratory analysis for a single sample or experimental condition.\">metaQuantome<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Genome assembler for metagenomics datasets.\">metaspades<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"MetaWRAP aims to be an easy-to-use metagenomic wrapper suite that accomplishes the core tasks of metagenomic analysis from start to finish: read quality control, assembly, visualization, taxonomic profiling, extracting draft genomes (binning), and functional annotation.\">metawrap<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A (mostly) universal methylation extractor for BS-seq experiments.\">MethylDackel<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">metilene<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Estimates effective population sizes,past migration rates between n population assuming a migration matrix model with asymmetric migration rates and different subpopulation sizes, and population divergences or admixture.\">MIGRATE<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">MiModD<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Short-read assembler based on a de Bruijn graph, capable of assembling a human genome on a desktop computer in a day.\">minia<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Miniasm is a very fast OLC-based de novo assembler for noisy long reads. It takes all-vs-all read self-mappings (typically by minimap) as input and outputs an assembly graph in the GFA format.\">miniasm<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">miniconda<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Pairwise aligner for genomic and spliced nucleotide sequences\">minimap2<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Miniprot aligns a protein sequence against a genome with affine gap penalty, splicing and frameshift. It is primarily intended for annotating protein-coding genes in a new species using known genes from other species.\">miniprot<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">mircounts<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"miRDeep2 discovers active known or novel miRNAs from deep sequencing data.\">mirdeep2<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">mitobim<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"De novo metazoan mitochondrial genome annotation.\">mitos<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Multi Locus Sequence Typing from an assembled genome or from a set of reads.\">MLST<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"MMseqs2 (Many-against-Many sequence searching) is a software suite to search and cluster huge protein and nucleotide sequence sets. MMseqs2 is open source GPL-licensed software implemented in C++ for Linux, MacOS, and (as beta version, via cygwin) Windows. The software is designed to run on multiple cores and servers and exhibits very good scalability. MMseqs2 can run 10000 times faster than BLAST. At 100 times its speed it achieves almost the same sensitivity. It can perform profile searches with the same sensitivity as PSI-BLAST at over 400 times its speed.\n\nMMseqs2 includes Linclust, the first clustering algorithm whose runtime scales linearly With Linclust we clustered 1.6 billion metagenomic sequence fragments in 10 h on a single server to 50% sequence identity.\">MMseqs2<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Universal whole-sequence-based plasmid typing and its utility to prediction of host range and epidemiological surveillance.\n\nMOB-suite: Software tools for clustering, reconstruction and typing of plasmids from draft assemblies.\n\nPlasmids are mobile genetic elements (MGEs), which allow for rapid evolution and adaption of bacteria to new niches through horizontal transmission of novel traits to different genetic backgrounds. The MOB-suite is designed to be a modular set of tools for the typing and reconstruction of plasmid sequences from WGS assemblies.\">MOB-suite<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">modENCODEfly<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">modENCODEworm<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">modmine<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A modest Feature Finder to extract features in MS1 Data.\">moFF<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">MONAI Label<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A proteomics search algorithm specifically designed for high-resolution tandem mass spectra.\">Morpheus<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Open-source, platform-independent, community-supported software for describing and comparing microbial communities\">Mothur<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Data warehouse for accessing mouse data from Mouse Genome Informatics (MGI). Supports powerful query, reporting, and analysis capabilities, the ability to save and combine results from different queries, easy integration into larger workflows, and a comprehensive Web Services layer.\">mousemine<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Program for Bayesian inference and model choice across a wide range of phylogenetic and evolutionary models. It uses Markov chain Monte Carlo (MCMC) methods to estimate the posterior distribution of model parameters.\">MrBayes<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"msConvert is a command-line utility for converting between various mass spectrometry data formats, including from raw data from several commercial companies (with vendor libraries, Windows-only). For Windows users, there is also a GUI, msConvertGUI.\">msconvert<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Tool for mass spectra metadata annotation.\">MSMetaEnhancer<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Statistical tool for quantitative mass spectrometry-based proteomics.\">MSstats<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"MultiQC aggregates results from multiple bioinformatics analyses across many samples into a single report. It searches a given directory for analysis logs and compiles a HTML report. It's a general use tool, perfect for summarising the output from numerous bioinformatics tools.\">MultiQC<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"MUMmer is a modular system for the rapid whole genome alignment of finished or draft sequence. Basically it is a ultra-fast alignment of large-scale DNA and protein sequences\">MUMmer<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">music deconvolution<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">mztosqlite<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">nag<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"RNA modifications detection by comparative Nanopore direct RNA sequencing.\n\nRNA modifications detection from Nanopore dRNA-Seq data.\n\nNanocompore identifies differences in ONT nanopore sequencing raw signal corresponding to RNA modifications by comparing 2 samples.\n\nAnalyses performed for the nanocompore paper.\n\nNanocompore compares 2 ONT nanopore direct RNA sequencing datasets from different experimental conditions expected to have a significant impact on RNA modifications. It is recommended to have at least 2 replicates per condition. For example one can use a control condition with a significantly reduced number of modifications such as a cell line for which a modification writing enzyme was knocked-down or knocked-out. Alternatively, on a smaller scale transcripts of interests could be synthesized in-vitro\">Nanocompore<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">nanofilt<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"NanoPlot is a tool with various visualizations of sequencing data in bam, cram, fastq, fasta or platform-specific TSV summaries, mainly intended for long-read sequencing from Oxford Nanopore Technologies and Pacific Biosciences\">nanoplot<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A package for detecting cytosine methylations and genetic variations from nanopore MinION sequencing data.\">Nanopolish<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">NanoSV<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">natural product likeness scorer<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"NCBI Datasets is a new resource that lets you easily gather data from across NCBI databases. Find and download sequence, annotation, and metadata for genes and genomes using our command-line tools or web interface.\">NCBI Datasets<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">ncbi acc download<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"nccmp compares two NetCDF files bitwise, semantically or with a user defined tolerance (absolute or relative percentage).  Parallel comparisons are done in local memory without requiring temporary files.  Highly recommended for regression testing scientific models or datasets in a test-driven development environment.\">nccmp<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">NCL<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"The NCO toolkit manipulates and analyzes data stored in netCDF-accessible formats, including DAP, HDF4, and HDF5.\">NCO<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Ncview is a netCDF visual browser.\">Ncview<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"NetCDF (Network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data.\">NetCDF<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"The Newick Utilities are a set of command-line tools for processing phylogenetic trees. They can process arbitrarily large amounts of data and do not require user interaction, which makes them suitable for automating phylogeny processing tasks.\">newick utils<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">nextclade<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Nextflow enables scalable and reproducible scientific workflows using software containers. It allows the adaptation of pipelines written in the most common scripting languages.\">nextflow<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"NGSUtils is a suite of software tools for working with next-generation sequencing datasets\">ngsutils<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Nearly Infinite Neighbor Joining Application\">ninja<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">NOVOPlasty<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">nseg<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">nvc<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Set of python programs developed to simplify the manipulation of sequence files. They were mainly designed to help us for analyzing Next Generation Sequencer outputs (454 or Illumina) in the context of DNA Metabarcoding.\">OBITools<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">ont-fast5-api<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Open source library and a collection of tools and interfaces for the analysis of mass spectrometry data. Includes over 200 standalone (TOPP) tools that can be combined to a workflow with the integrated workflow editor TOPPAS. Raw and intermediate mass spectrometry data can be visualised with the included viewer TOPPView.\">OpenMS<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">DRAGEN ORA<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">PAMPA-Galaxy<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Pangolin is a deep-learning based method for predicting splice site strengths (for details, see Zeng and Li, Genome Biology 2022). It is available as a command-line tool that can be run on a VCF or CSV file containing variants of interest; Pangolin will predict changes in splice site strength due to each variant, and return a file of the same format. Pangolin's models can also be used with custom sequences.\">Pangolin<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">param value from file<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"ParaView is an open-source, multi-platform data analysis and visualization application. ParaView users can quickly build visualizations to analyze their data using qualitative and quantitative techniques. The data exploration can be done interactively in 3D or programmatically using ParaView’s batch processing capabilities.\">ParaView<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">parse mito blast<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Tool set for pathway based data integration and visualization that maps and renders a wide variety of biological data on relevant pathway graphs. It downloads the pathway graph data, parses the data file, maps user data to the pathway, and render pathway graph with the mapped data. In addition, it integrates with pathway and gene set (enrichment) analysis tools for large-scale and fully automated analysis.\">Pathview<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Multithread blat algorithm speeding up aligning sequences to genomes.\">pblat<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Productive visualization of high-throughput sequencing data using the SeqCode open portable platform.\">pe histogram<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Paired-end read merger. PEAR evaluates all possible paired-end read overlaps without requiring the target fragment size as input. In addition, it implements a statistical test for minimizing false-positive results.\">PEAR<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">PepPointer<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"PeptideShaker is a search engine independent platform for interpretation of proteomics identification results from multiple search engines, currently supporting X!Tandem, MS-GF+, MS Amanda, OMSSA, MyriMatch, Comet, Tide, Mascot, Andromeda and mzIdentML. By combining the results from multiple search engines, while re-calculating PTM localization scores and redoing the protein inference, PeptideShaker attempts to give you the best possible understanding of your proteomics data\">Peptide Shaker<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">peptide genomic coordinate<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Semi-supervised learning for peptide identification from MS/MS data.\">Percolator<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">perllib<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"This tool is used to search a FASTA sequence against a library of Pfam HMM.\">PfamScan<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">phinch<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">PHP<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Provides a set of classes and tools to facilitate the import, storage, analysis, and graphical display of microbiome census data.\">Phyloseq<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Phylogenetic estimation software using Maximum Likelihood\">PhyML<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A set of command line tools for manipulating high-throughput sequencing (HTS) data in formats such as SAM/BAM/CRAM and VCF. Available as a standalone program or within the GATK4 program.\">PICARD<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"PICRUSt (Phylogenetic Investigation of Communities by Reconstruction of Unobserved States) is a bioinformatics software package designed to predict metagenome functional content from marker gene (e.g., 16S rRNA) surveys and full genomes.\">PICRUSt<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"PICRUSt2 (Phylogenetic Investigation of Communities by Reconstruction of Unobserved States) is a software for predicting functional abundances based only on marker gene sequences.\">PICRUSt 2<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">pileup interval<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">pileup parser<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Read alignment analysis to diagnose, report, and automatically improve de novo genome assemblies.\">Pilon<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"PIPE-T is a Galaxy Workflow for processing and analyzing miR expression profiles by RTqPCR.\n\nIt is a tool that offers several state-of-the-art options for parsing, filtering, normalizing, imputing and analyzing RT-qPCR expression data. Integration of PIPE-T into Galaxy allows experimentalists with strong bioinformatic background, as well as those without any programming or development expertise, to perform complex analysis in a simple to use, transparent, accessible, reproducible, and user-friendly environment\">pipe t<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">PlasFlow<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.\">plink<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">plot from lda<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">plotly<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"PnetCDF: A Parallel I/O Library for NetCDF File Access\">PnetCDF<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">poisson2test<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">porechop<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Flexible toolkit for exploring datasets generated by nanopore sequencing devices from MinION for the purposes of quality control and downstream analysis.\">poretools<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Tools for performing taxonomic assignment based on phylogeny using pplacer and clst.\">pplacer<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">pretext map<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Pretext is an OpenGL-powered pretext contact map viewer.\">PretextView / Pretext Suite<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"The pipeline runs PRODIGAL gene predictions on all genomes, runs pan-reciprocal BLAST, and identifies ortholog sets. For a set of orthologous genes, if the positions of the PRODIGAL selected starts coincide in a multiple sequence alignment, they are accepted. If they do not coincide, a consistent start position is sought where a majority of the highest-scoring PRODIGAL selected sites coincide. If such a position is found, it is accepted, and the predictions are changed for the outlying genes.\">Prodigal<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Software tool to annotate bacterial, archaeal and viral genomes quickly and produce standards-compliant output files.\">Prokka<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">prop venn<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">pslcdnafilter<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Identifying and removing haplotypic duplication in primary genome assemblies | haplotypic duplication identification tool | scripts/pd_config.py: script to generate a configuration file used by run_purge_dups.py | purge haplotigs and overlaps in an assembly based on read depth | Given a primary assembly pri_asm and an alternative assembly hap_asm (optional, if you have one), follow the steps shown below to build your own purge_dups pipeline, steps with same number can be run simultaneously. Among all the steps, although step 4 is optional, we highly recommend our users to do so, because assemblers may produce overrepresented seqeuences. In such a case, The final step 4 can be applied to remove those seqeuences\">Purge Dups<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">pybigwig<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"PycoQC computes metrics and generates interactive QC plots for Oxford Nanopore technologies sequencing data.\">pycoqc<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"reproducible plots for multivariate genomic data sets.\n\nStandalone program and library to plot beautiful genome browser tracks.\n\npyGenomeTracks aims to produce high-quality genome browser tracks that are highly customizable. Currently, it is possible to plot:.\">pygenometracks<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">pyprophet<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A Python module for reading and manipulating SAM/BAM/VCF/BCF files.\">pysam<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"PyTorch is an optimized tensor library for deep learning using GPUs and CPUs.\">PyTorch<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"QIIME 2™ is a next-generation microbiome bioinformatics platform that is extensible, free, open source, and community developed.\">QIIME2.0<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">qual stats boxplot<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Platform-independent application written in Java and R that provides both a Graphical User Inteface (GUI) and a command-line interface to facilitate the quality control of alignment sequencing data.\">qualimap<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A Collection of Tools for Viral Quasispecies Analysis | Abstract Summary quasitools is a collection of newly-developed, open-source tools for analyzing viral quasispcies data. The application suite includes tools with the ability to create consensus sequences, call nucleotide, codon, and amino acid variants, calculate the complexity of a quasispecies, and measure the genetic distance between two similar quasispecies. These tools may be run independently or in user-created workflows. Availability The quasitools suite is a freely available application licensed under the Apache License, Version 2.0. The source code, documentation, and file specifications are available at: https: phac-nml.github.io quasitools Contact gary.vandomselaar@canada.ca\">quasitools<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"QUAST stands for QUality ASsessment Tool.  \nIt evaluates a quality of genome assemblies by computing various metrics and providing nice reports.\">QUAST<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Query Tabular is a Galaxy-based tool which manipulates tabular files.  Query Tabular automatically creates a SQLite database directly from a tabular file within a Galaxy workflow.  The SQLite database can be saved to the Galaxy history, and further process to generate tabular outputs containing desired information and formatting.\">query tabular<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Free software environment for statistical computing and graphics.\">R<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">r-raceid<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"The Possibility to Use Oxford Nanopore Technology | Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads. http://genome.cshlp.org/content/early/2017/01/18/gr.214270.116 Note: This was the original repository which will no longer be officially maintained. Please use the new official repository here: | Racon is intended as a standalone consensus module to correct raw contigs generated by rapid assembly methods which do not include a consensus step | Consensus module for raw de novo DNA assembly of long uncorrected reads\">Racon<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A feature clustering algorithm for non-targeted mass spectrometric metabolomics data.\">RAMClustR<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">ratmine<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"a de novo genome assembler for long reads.\n\nRaven is a de novo genome assembler for long uncorrected reads.\">raven<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A standalone tool for extracting data directly from raw files generated by Thermo Orbitrap family instruments.\">Raw Tools<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A tool for Phylogenetic Analysis and Post-Analysis of Large Phylogenies.\">RAxML<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"RDKit is an Open-Source Cheminformatics Software. Fast, Efficient Fragment-Based Coordinate Generation for Open Babel.\">rdkit<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">rdock<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Tool for calculating the probability of nucleosome formation along a DNA sequence input by the user.\">recon<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"recetox-aplcms is a tool for peak detection in mass spectrometry data. The tool performs (1) noise removal, (2) peak detection, (3) retention time drift correction, (4) peak alignment and (5) weaker signal recovery as well as (6) suspect screening.\">recetox-aplcms<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"This is a program to detect and visualize RNA editing events at genomic scale using next-generation sequencing data.\">RED<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A program that screens DNA sequences for interspersed repeats and low complexity DNA sequences. The output of the program is a detailed annotation of the repeats that are present in the query sequence as well as a modified version of the query sequence in which all the annotated repeats have been masked (default: replaced by Ns).\">RepeatMasker<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"De-novo repeat family identification and modeling package. At the heart of RepeatModeler are two de-novo repeat finding programs ( RECON and RepeatScout ) which employ complementary computational methods for identifying repeat element boundaries and family relationships from sequence data. RepeatModeler assists in automating the runs of RECON and RepeatScout given a genomic database and uses the output to build, refine and classify consensus models of putative interspersed repeats.\">RepeatModeler<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"RepeatScout is a tool to discover repetitive substrings in DNA.\">RepeatScout<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">repenrich<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">replace column with key value file<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"The rjags package provides an interface from R to the JAGS library for Bayesian data analysis. JAGS uses Markov Chain Monte Carlo (MCMC) to generate a sequence of dependent samples\nfrom the posterior distribution of the parameters.\">rjags<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Workflow to process tandem MS files and build MassBank records. Functions include automated extraction of tandem MS spectra, formula assignment to tandem MS fragments, recalibration of tandem MS spectra with assigned fragments, spectrum cleanup, automated retrieval of compound information from Internet databases, and export to MassBank records.\">RMassBank<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">rmats-turbo<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"RMBlast is a RepeatMasker compatible version of the standard NCBI blastn program. The primary difference between this distribution and the NCBI distribution is the addition of a new program \"rmblastn\" for use with RepeatMasker and RepeatModeler.\">rmblast<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">rnachipintegrator<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A high speed stand alone pan genome pipeline, which takes annotated assemblies in GFF3 format (produced by Prokka (Seemann, 2014)) and calculates the pan genome.\">roary<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">row means<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"We present a generative statistical model and associated inference methods that handle read mapping uncertainty in a principled manner. Through simulations parameterized by real RNASeq data, we show that our method is more accurate than previous methods. Our improved accuracy is the result of handling read mapping uncertainty with a statistical model and the estimation of gene expression levels as the sum of isoform expression levels.\">RSEM<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Provides a number of useful modules that can comprehensively evaluate high throughput sequence data especially RNA-seq data. Some basic modules quickly inspect sequence quality, nucleotide composition bias, PCR bias and GC bias, while RNA-seq specific modules evaluate sequencing saturation, mapped reads distribution, coverage uniformity, strand specificity, transcript level RNA integrity etc.\">rseqc<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Integrated development environment (IDE) for the R programming language.\">Rstudio<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"RTG Core: Software for alignment and analysis of next-gen sequencing data.\">rtg-tools<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">rxdock<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A software tool that implements a novel, is an alignment-free algorithm for the estimation of isoform abundances directly from a set of reference sequences and RNA-seq reads.\">sailfish<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A tool for transcript expression quantification from RNA-seq data\">Salmon<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">salmon kallisto mtx to 10x<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"> VERY_LOW CONFIDENCE! | > CORRECT NAME OF TOOL COULD ALSO BE 'chromosome-scale', 'reference-quality', 'Hi-C', 'scaffolder' | Integrating Hi-C links with assembly graphs for chromosome-scale assembly | SALSA: A tool to scaffold long read assemblies with Hi-C data | SALSA: A tool to scaffold long read assemblies with Hi-C | This code is used to scaffold your assemblies using Hi-C data. This version implements some improvements in the original SALSA algorithm. If you want to use the old version, it can be found in the old_salsa branch\">salsa<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">sam bitwise flag filter<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">sam2interval<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"This tool is a high performance modern robust and fast tool (and library), written in the D programming language, for working with SAM, BAM and CRAM formats.\">sambamba<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A tool to mark duplicates and extract discordant and split reads from SAM files.\">samblaster<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">sample seqs<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"SAMtools and BCFtools are widely used programs for processing and analysing high-throughput sequencing data. They include tools for file format conversion and manipulation, sorting, querying, statistics, variant calling, and effect analysis amongst other methods.\">SAMTools<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Scalable toolkit for analyzing single-cell gene expression data. It includes preprocessing, visualization, clustering, pseudotime and trajectory inference and differential expression testing. The Python-based implementation efficiently deals with datasets of more than one million cells.\">scanpy<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Pre-processing, quality control, normalization and visualization of single-cell RNA-seq data.\">scater<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">scikit-build<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Scikit-image contains image processing algorithms for SciPy, including IO, morphology, filtering, warping, color manipulation, object detection, etc.\">scikit-image<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">Scikit-learn<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">scipio<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">send to cloud<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"SEPP stands for SATé-Enabled Phylogenetic Placement and addresses the problem of phylogenetic placement for meta-genomic short reads\">sepp<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">seq filter by id<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">seq select by id<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"FASTA and FASTQ are basic and ubiquitous formats for storing nucleotide and protein sequences. Common manipulations of FASTA/Q file include converting, searching, filtering, deduplication, splitting, shuffling, and sampling. Existing tools only implement some of these manipulations, and not particularly efficiently, and some are only available for certain operating systems. Furthermore, the complicated installation process of required packages and running environments can render these programs less user friendly. SeqKit demonstrates competitive performance in execution time and memory usage compared to similar tools. The efficiency and usability of SeqKit enable researchers to rapidly accomplish common FASTA/Q file manipulations.\">SeqKit<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">seqlib<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A tool for processing sequences in the FASTA or FASTQ format. It parses both FASTA and FASTQ files which can also be optionally compressed by gzip.\">seqtk<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Seurat is an R package designed for QC, analysis, and exploration of single-cell RNA-seq data. Seurat aims to enable users to identify and interpret sources of heterogeneity from single-cell transcriptomic measurements, and to integrate diverse types of single-cell data.\">seurat<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">Sff extractor<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"De novo assembly from Oxford Nanopore reads.\">shasta<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Shovill is a pipeline for assembly of bacterial isolate genomes from Illumina paired-end reads.  Shovill uses SPAdes at its core, but alters the steps before and after the primary assembly step to get similar results in less time. Shovill also supports other assemblers like SKESA, Velvet and Megahit, so you can take advantage of the pre- and post-processing the Shovill provides with those too.\">shovill<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A clustering approach for identification of enriched domains from histone modification ChIP-seq data.\">sicer<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A text mining framework for interactive analysis and visualization of similarities among biomedical entities.\n\nFor each search query, PMIDs or abstracts from PubMed are saved.\n\n$ git clone https://github.com/dlal-group/simtext.\n\nFor all PMIDs in each row of a table the according abstracts are saved in additional columns.\">SimText<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">SIP<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"The Salmonella In Silico Typing Resource (SISTR) is an open-source and freely available web application for rapid in silico typing and serovar prediction from Salmonella genome assemblies using cgMLST and O and H antigen gene searching.\">sistr<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">slow5-guppy<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">slow5tools<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">smithwaterman<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Reference-free profiling of polyploid genomes | Inference of ploidy and heterozygosity structure using whole genome sequencing data | Smudgeplots are computed from raw or even better from trimmed reads and show the haplotype structure using heterozygous kmer pairs. For example: | This tool extracts heterozygous kmer pairs from kmer dump files and performs gymnastics with them. We are able to disentangle genome structure by comparing the sum of kmer pair coverages (CovA + CovB) to their relative coverage (CovA / (CovA + CovB)). Such an approach also allows us to analyze obscure genomes with duplications, various ploidy levels, etc | GenomeScope 2.0 and Smudgeplots: Reference-free profiling of polyploid genomes Timothy Rhyker Ranallo-Benavidez, Kamil S. Jaron, Michael C. Schatz bioRxiv 747568; doi: https://doi.org/10.1101/747568\">Smudgeplots<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Workflow engine and language. It aims to reduce the complexity of creating workflows by providing a fast and comfortable execution environment, together with a clean and modern domain specific specification language (DSL) in python style.\">Snakemake<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"The Semi-HMM-based Nucleic Acid Parser is a gene prediction tool.\">SNAP<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"An algorithm for structural variation detection from third generation sequencing alignment.\">sniffles<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Rapid haploid variant calling and core SNP phylogeny generation.\">snippy<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">snp sites<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">snp-dists<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Variant annotation and effect prediction tool. It annotates and predicts the effects of variants on genes and proteins (such as amino acid changes).\">snpeff<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">snpfreq<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">snpfreqplot<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Toolbox that allows you to filter and manipulate annotated vcf files.\">snpsift<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"SOAPdenovo2 is a next generation sequencing reads de novo assembler.\">SOAPdenovo2<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Sequence analysis tool for filtering, mapping and OTU-picking NGS reads.\">sortmerna<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"St. Petersburg genome assembler – is intended for both standard isolates and single-cell MDA bacteria assemblies. SPAdes 3.9 works with Illumina or IonTorrent reads and is capable of providing hybrid assemblies using PacBio, Oxford Nanopore and Sanger reads. Additional contigs can be provided and can be used as long reads.\">SPAdes<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">spectra<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Spectral Repeat Finder (SRF) is a program to find repeats through an analysis of the power spectrum of a given DNA sequence.\">Spectral Repeat Finder (SRF)<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">split file to collection<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">sqlite<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">sqlite to tabular<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">sra source<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for using data in the INSDC Sequence Read Archives.\">sra-tools<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A fast implementation of the Smith-Waterman algorithm whose API that can be flexibly used by programs written in C, C++ and other languages.\">ssw<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Developed to work with restriction enzyme based sequence data, such as RADseq, for building genetic maps and conducting population genomics and phylogeography analysis.\">Stacks<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Ultrafast universal RNA-seq data aligner\">star<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">staramr<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"STAR-Fusion, a method that is both fast and accurate in identifying fusion transcripts from RNA-Seq data\">STAR-Fusion<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Fast and highly efficient assembler of RNA-Seq alignments into potential transcripts. It uses a novel network flow algorithm as well as an optional de novo assembly step to assemble and quantitate full-length transcripts representing multiple splice variants for each gene locus.\">stringtie<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Subread is a general-purpose read aligner which can be used to map both genomic DNA-seq reads and RNA-seq reads. It uses a new mapping paradigm called \"seed-and-vote\" to achieve fast, accurate and scalable read mapping. It automatically determines if a read should be globally or locally aligned, therefore particularly powerful in mapping RNA-seq reads. It supports indel detection and can map reads with both fixed and variable lengths.\">subread<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">Summary Statistics1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"An agile homology-based approach using a reduced SEED database to report the subsystems present in metagenomic samples and profile their abundances.\">SUPER-FOCUS<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Structural variant detection from haploid and diploid genome assemblies.\n\nSVIM-asm - Structural variant identification method (Assembly edition).\n\nSVIM-asm (pronounced SWIM-assem) is a structural variant caller for haploid or diploid genome-genome alignments. It analyzes a given sorted BAM file (preferably from minimap2) and detects five different variant classes between the query assembly and the reference: deletions, insertions, tandem and interspersed duplications and inversions.\">SVIM-asm<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"SyRI is tool for finding genomic rearrangements and local sequence differences from whole-genome assemblies.\n\nGenomic differences range from single nucleotide differences to complex structural variations. Current methods typically annotate sequence differences ranging from SNPs to large indels accurately but do not unravel the full complexity of structural rearrangements, including inversions, translocations, and duplications, where highly similar sequence changes in location, orientation, or copy number. Here, we present SyRI, a pairwise whole-genome comparison tool for chromosome-level assemblies. SyRI starts by finding rearranged regions and then searches for differences in the sequences, which are distinguished for residing in syntenic or rearranged regions. This distinction is important as rearranged regions are inherited differently compared to syntenic regions.\">SyRI<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">t test two samples<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">table compute<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">tabular to fasta<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Targetfinder.org provides a web based resource that allows users to find genes that have a similar expression to a query gene signature.\">TargetFinder<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">tb variant filter<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">tbl2asn<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">tbprofiler<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">tbvcfreport<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Prediction of cognitive impairment via deep learning trained with multi-center neuropsychological test data.\n\nAn end-to-end open source machine learning platform.\n\nAnnouncing the TensorFlow Dev Summit 2020 Learn more.\n\nTensorFlow is an end-to-end open source platform for machine learning. It has a comprehensive, flexible ecosystem of tools, libraries and community resources that lets researchers push the state-of-the-art in ML and developers easily build and deploy ML powered applications\">TensorFlow<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"lineage-level classification of transposable elements using conserved protein domains.\n\nNote: do not move or hard link TEsorter.py alone to anywhere else, as it rely on database/ and bin/. You can add the directory to PATH or soft link TEsorter.py to PATH\">TEsorter<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">text processing<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Open-source, crossplatform tool that converts Thermo RAW files into open file formats such as MGF and to the HUPO-PSI standard file format mzML\">ThermoRawFileParser<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Program that aligns RNA-Seq reads to a genome in order to identify exon-exon splice junctions. It is built on the ultrafast short read mapping program Bowtie. A stable SAMtools version is now packaged with the program.\">tophat<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"TotalView is a debugger for High Performance Computing applications.\">TotalView<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">trams<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"TransDecoder identifies candidate coding regions within transcript sequences, such as those generated by de novo RNA-Seq transcript assembly using Trinity, or constructed based on RNA-Seq alignments to the genome using Tophat and Cufflinks.\">TransDecoder<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A tool for the analysis of Tn-Seq data. It provides an easy to use graphical interface and access to three different analysis methods that allow the user to determine essentiality in a single condition as well as between conditions.\">transit<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">translate bed<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Institute for Systems Biology \"Trans-Proteomic Pipeline\"\">Trans-Proteomic Pipeline (TPP)<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">TreeShrink<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Tandem Repeats Finder. Find tandem repeats in DNA sequences without the need to specify either the pattern or pattern size. It uses the method of k-tuple matching to avoid the need for full scale alignment matrix computations. It requires no a priori knowledge of the pattern, pattern size or number of copies. There are no restrictions on the size of the repeats that can be detected. It determines a consensus pattern for the smallest repetitive unit in the tandem repeat.\">trf<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Tool for the automated removal of spurious sequences or poorly aligned regions from a multiple sequence alignment.\">trimAl<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A wrapper tool around Cutadapt and FastQC to consistently apply quality and adapter trimming to FastQ files, with some extra functionality for MspI-digested RRBS-type (Reduced Representation Bisufite-Seq) libraries.\">Trim Galore<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A flexible read trimming tool for Illumina NGS data\">Trimmomatic<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Trinity is a transcriptome assembler which relies on three different tools, inchworm an assembler, chrysalis which pools contigs and butterfly which amongst others compacts a graph resulting from butterfly with reads.\">Trinity / trinityrnaseq<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Comprehensive annotation suite designed for automatic functional annotation of transcriptomes, particularly de novo assembled transcriptomes, from model or non-model organisms.\">Trinotate<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A program for improved detection of transfer RNA genes in genomic sequence.\">trnascan-se<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Trycycler: consensus long-read assemblies for bacterial genomes\">Trycycler toolkit<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Utilities for handling sequences and assemblies from the UCSC Genome Browser project.\">UCSC tools<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">ucsc table direct archaea1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">ucsc table direct1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Tools for handling Unique Molecular Identifiers in NGS data sets.\">umi tools<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A tool for assembling bacterial genomes from a combination of short (2nd generation) and long (3rd generation) sequencing reads.\">Unicycler<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Metaproteomics data analysis with a focus on interactive data visualizations.\">unipept<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">UniProt Downloader<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">uniprot rest interface<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">unzip<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Tool for predicting effects of variants for any genome in Ensembl or with genome annotation (via GFF). This includes vertebrates and also plants, fungi, protists, metazoa and bacteria.  There is a web and a REST API version but the most powerful is the Perl script version. See McLaren et al., 2016, Genome Biology\">Variant Effect Predictor (VEP)<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"VarScan, an open source tool for variant detection that is compatible with several short read align-ers.\">VarScan<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">vcf to maf customtrack1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">vcf2maf<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"API and command line utilities for the manipulation of VCF files.\">vcflib<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Provide easily accessible methods for working with complex genetic variation data in the form of VCF files.\">VCFTools<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"A de novo genomic assembler specially designed for short read sequencing technologies, such as Solexa or 454 or SOLiD.\">Velvet<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">verkko<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">VisIt<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">volcanoplot<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"High-throughput search and clustering sequence analysis tool. It supports de novo and reference based chimera detection, clustering, full-length and prefix dereplication, reverse complementation, masking, all-vs-all pairwise global alignment, exact and global alignment searching, shuffling, subsampling and sorting. It also supports FASTQ file analysis, filtering and conversion.\">vsearch<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">weblogo3<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Software for phasing genomic variants using DNA sequencing reads, also called haplotype assembly. It is especially suitable for long reads, but works also well with short reads.\">WhatsHap<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">wig to bigWig<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">wiggle2simple1<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Winnowmap is a long-read mapping algorithm optimized for mapping ONT and PacBio reads to repetitive reference sequences. Winnowmap development began on top of minimap2 codebase, and since then we have incorporated the following two ideas to improve mapping accuracy within repeats\">winnowmap<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"First fully open-source and collaborative online platform for computational metabolomics. It includes preprocessing, normalization, quality control, statistical analysis of LC/MS, FIA-MS, GC/MS and NMR data.\">Workflow 4 Metabolomics<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">wormbase<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">xarray<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Framework for processing and visualization of chromatographically separated and single-spectra mass spectral data.  The packages enables imports from AIA/ANDI NetCDF, mzXML, mzData and mzML files and preprocesses data for high-throughput, untargeted analyte profiling.\">xcms<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">xml4ena<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Detection of differential RNA modifications from direct RNA sequencing of human cell lines.\n\nPython package for detection of differential RNA modifications from direct RNA sequencing.\">xpore<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"Matches tandem mass spectra with peptide sequences.\">xtandem<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">xy plot<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"YaHS is scaffolding tool using Hi-C data. It relies on a new algorithm for contig joining detection which considers the topological distribution of Hi-C signals aiming to distinguish real interaction signals from mapping noises.\">YaHS<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">yeastmine<\/p>","<p class=\"name\" data-toggle=\"popover\" data-trigger=\"hover\" title=\"Description\" data-content=\"No metadata available.\">zebrafishmine<\/p>"],["Mass screening of contigs for antimicrobial resistance or virulence genes.","De novo genome sequence assembler.","No description available.","The Assisted Model Building with Energy Refinement tool refers to two things: a set of molecular mechanical force fields for the simulation of biomolecules (which are in the public domain, and are used in a variety of simulation programs); and a package of molecular simulation programs which includes source code and demos.","Software package specially developed for the study of genes’ primary structure. It uses gene sequences downloaded from public databases, as FASTA and GenBank, and it applies a set of statistical and visualization methods in different ways, to reveal information about codon context, codon usage, nucleotide repeats within open reading frames (ORFeome) and others.","No description available.","This tool can get annotation for a generic set of IDs, using the Bioconductor annotation data packages. Supported organisms are human, mouse, rat, fruit fly and zebrafish. The org.db packages that are used here are primarily based on mapping using Entrez Gene identifiers. More information on the annotation packages can be found at the Bioconductor website, for example, information on the human annotation package (org.Hs.eg.db) can be found here.","Rapid genome-wide identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genomes. It integrates and cross-links with a large number of in silico secondary metabolite analysis tools that have been published earlier.","Convert various sequence formats to FASTA","Apollo is a genome annotation viewer and editor. Apollo allows researchers to explore genomic annotations at many levels of detail, and to perform expert annotation curation, all in a graphical environment.","ARAGORN detects tRNA, mtRNA info about tmRNA, and tmRNA genes","Arriba is a command-line tool to detect gene fusions from RNA-Seq data based on the STAR aligner. In addition to fusions, it can detect exon duplications/inversions and truncations of genes (i.e., breakpoints in introns and intergenic regions). Arriba is the winner of the DREAM SMC-RNA Challenge.","Java program for estimating a species tree given a set of unrooted gene trees.","AUGUSTUS is a eukaryotic gene prediction tool. It can integrate evidence, e.g. from RNA-Seq, ESTs, proteomics, but can also predict genes ab initio. The PPX extension to AUGUSTUS can take a protein sequence multiple sequence alignment as input to find new members of the family in a genome. It can be run through a web interface (see https://bio.tools/webaugustus), or downloaded and run locally.","AutoDock Vina is a new open-source program for drug discovery, molecular docking and virtual screening, offering multi-core capability, high performance and enhanced accuracy and ease of use.","No description available.","No description available.","No description available.","Rapid & standardized annotation of bacterial genomes, MAGs & plasmids","BamTools provides a fast, flexible C++ API & toolkit for reading, writing, and managing BAM files.","Bamutil provides a serie of programs to work on SAM/BAM files.","GUI program that allows users to interact with the assembly graphs made by de novo assemblers such as Velvet, SPAdes, MEGAHIT and others. It visualises assembly graphs, with connections, using graph layout algorithms.","No description available.","Predict the location of ribosomal RNA genes in genomes. It supports bacteria (5S,23S,16S), archaea (5S,5.8S,23S,16S), mitochondria (12S,16S) and eukaryotes (5S,5.8S,28S,18S).","BBMap is a fast splice-aware aligner for RNA and DNA.  It is faster than almost all short-read aligners, yet retains unrivaled sensitivity and specificity, particularly for reads with many errors and indels.","A tool for filling the gap created by genomic data processing/analysis by rebasing some analysis results against the parent features which were originally analysed.","BCFtools is a set of utilities that manipulate variant calls in the Variant Call Format (VCF) and its binary counterpart BCF. All commands work transparently with both VCFs and BCFs, both uncompressed and BGZF-compressed.","Beagle is a software package that performs genotype calling, genotype phasing, imputation of ungenotyped markers, and identity-by-descent segment detection.","BEAGLE is a high-performance library that can perform the core calculations at the heart of most Bayesian and Maximum Likelihood phylogenetics packages.","The Bayesian Evolutionary Analysis Sampling Trees is a cross-platform program for Bayesian analysis of molecular sequences using MCMC (Markov chain Monte Carlo). It is entirely orientated towards rooted, time-measured phylogenies inferred using strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies but is also a framework for testing evolutionary hypotheses without conditioning on a single tree topology.","Bayesian phylogenetic analysis of molecular sequences. It estimates rooted, time-measured phylogenies using strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies but is also a framework for testing evolutionary hypotheses without conditioning on a single tree topology. It uses Markov chain Monte Carlo (MCMC) to average over tree space, so that each tree is weighted proportional to its posterior probability. It includes a graphical user-interface for setting up standard analyses and a suit of programs for analysing the results.","No description available.","Convert a BED format file of the proteins from a proteomics search database into a tabular format for the Multiomics Visualization Platform (MVP).","No description available.","BEDTools is an extensive suite of utilities for comparing genomic features in BED format.","The Bellerophon pipeline, improving de novo transcriptomes and removing chimeras.\n\nBellerophon is a pipeline created to remove falsely assembled chimeric transcripts in de novo transcriptome assemblies.\n\nThe pipeline can be downloaded as a vragrant virtual machine (https://app.vagrantup.com/bellerophon/boxes/bellerophon). This is recommended, as it avoids backwards compatibility problems with TransRate","Trim, circularise, orient and filter long read bacterial genome assemblies","No description available.","No description available.","Bio3D is an R package containing utilities for the analysis of protein structure, sequence and trajectory data.","No description available.","Tools for early stage NGS alignment file processing including fast sorting and duplicate marking.","This package includes basic tools for reading biom-format files, accessing and subsetting data tables from a biom object, as well as limited support for writing a biom-object back to a biom-format file. The design of this API is intended to match the python API and other tools included with the biom-format project, but with a decidedly \"R flavor\" that should be familiar to R users. This includes S4 classes and methods, as well as extensions of common core functions/methods.","No description available.","A collection of Perl modules that facilitate the development of Perl scripts for bioinformatics applications.  It provides software modules for many of the typical tasks of bioinformatics programming.","Biopython is a set of freely available tools for biological computation written in Python by an international team of developers.","BioTransformer is a freely available web server that supports accurate, rapid and comprehensive in silico metabolism prediction.","A tool that finds regions of similarity between biological sequences. The program compares nucleotide or protein sequences to sequence databases and calculates the statistical significance.","A tool that finds regions of similarity between biological sequences. The program compares nucleotide or protein sequences to sequence databases and calculates the statistical significance.","A tool that finds regions of similarity between biological sequences. The program compares nucleotide or protein sequences to sequence databases and calculates the statistical significance.","Fast, accurate spliced alignment of DNA sequences.","detect blocks of overlapping reads using a gaussian-distribution approach","No description available.","Boost is a set of libraries for the C++ programming language that provides support for tasks and structures such as linear algebra, pseudorandom number generation, multithreading, image processing, regular expressions, and unit testing.","Bowtie is an ultrafast, memory-efficient short read aligner.","Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.","No description available.","Runs Breseq software on a set of fastq files.","No description available.","Provides measures for quantitative assessment of genome assembly, gene set, and transcriptome completeness based on evolutionarily informed expectations of gene content from near-universal single-copy orthologs.","Fast, accurate, memory-efficient aligner for short and long sequencing reads","Bwa-mem2 is the next version of the bwa-mem algorithm in bwa. It produces alignment identical to bwa and is ~1.3-3.1x faster depending on the use-case,  dataset and the running machine.","No description available.","No description available.","Tools for manipulating biological data, particularly multiple sequence alignments.","Consensus Cancer Driver Gene Caller | Next-generation sequencing has allowed identification of millions of somatic mutations in human cancer cells. A key challenge in interpreting cancer genomes is to distinguish drivers of cancer development among available genetic mutations. To address this issue, we present the first web-based application, consensus cancer driver gene caller (C3), to identify the consensus driver genes using six different complementary strategies, i.e., frequency-based, machine learning-based, functional bias-based, clustering-based, statistics model-based, and network-based strategies. This application allows users to specify customized operations when calling driver genes, and provides solid statistical evaluations and interpretable visualizations on the integration results","Cactus is a reference-free whole-genome multiple alignment program.","Annotation of peaklists generated by xcms, rule based annotation of isotopes and adducts, isotope validation, EIC correlation based tagging of unknown adducts and fragments.","De-novo assembly tool for long read chemistry like Nanopore data and PacBio data.","Web-based contig assembly.","Implements statistical and computational tools for analyzing mass spectrometry imaging datasets, including methods for efficient pre-processing, spatial segmentation, and classification.","Contig Annotation Tool (CAT) and Bin Annotation Tool (BAT) are pipelines for the taxonomic classification of long DNA sequences and metagenome assembled genomes (MAGs/bins) of both known and (highly) unknown microorganisms, as generated by contemporary metagenomics studies. The core algorithm of both programs involves gene calling, mapping of predicted ORFs against the nr protein database, and voting-based classification of the entire contig / MAG based on classification of the individual ORFs.","No description available.","Cluster a nucleotide dataset into representative sequences.","Tool for quantifying data from biological images, particularly in high-throughput experiments.","No description available.","cellxgene (pronounced \"cell-by-gene\") is an interactive data explorer for single-cell transcriptomics datasets, such as those coming from the Human Cell Atlas.","CheckM provides a set of tools for assessing the quality of genomes recovered from isolates, single cells, or metagenomes.","Database of bioactive compounds, their quantitative properties and bioactivities (binding constants, pharmacology and ADMET, etc). The data is abstracted and curated from the primary scientific literature.","Fast cheminformatics fingerprint search, at your fingertips.\n\nChemfp is a set of command-line tools and a Python library for fingerprint generation and high-performance similarity search.\n\nThere are two ways to try out chemfp. From the download page page you can request an evaluation copy of the most recent version of chemfp, or you can download an earlier version for no cost under the MIT license","No description available.","This package implements functions to retrieve the nearest genes around the peak, annotate the genomic region of the peak, statistical methods for estimating the significance of overlap among ChIP peak data sets, and incorporate GEO database to compare the own dataset with those deposited in database. The comparison can be used to infer cooperative regulation and thus can be used to generate hypotheses.","An ultra fast, heuristic approach to detect conserved signals in extremely large pairwise genome comparisons (dotplot).","No description available.","Circos is tool for visualizing data in a circular format. It was developed for genomic data but can work for many other kinds of data as well.","No description available.","No description available.","Automatic generation of gene cluster comparison figures.\n\nGene cluster comparison figure generator.\n\nA d3 chart for generating gene cluster comparison figures.\n\nclinker is a pipeline for easily generating publication-quality gene cluster comparison figures.\n\nGiven a set of GenBank files, clinker will automatically extract protein translations, perform global alignments between sequences in each cluster, determine the optimal display order based on cluster similarity, and generate an interactive visualisation (using clustermap.js) that can be extensively tweaked before being exported as an SVG file.\n\nclustermap.js is an interactive, reusable d3 chart designed to visualise homology between multiple gene clusters.","Multiple sequence alignment software. The name is occassionally spelled as ClustalOmega, Clustal Ω, ClustalΩ, Clustal O, ClustalO.","Multiple sequence alignment software. Old deprecated versions. Even older versions were CLUSTAL and CLUSTAL V (ClustalV).","ColabFold databases are MMseqs2 expandable profile databases to generate diverse multiple sequence alignments to predict protein structures.","No description available.","No description available.","No description available.","No description available.","No description available.","No description available.","No description available.","No description available.","No description available.","No description available.","No description available.","No description available.","No description available.","No description available.","CPAT (Coding-Potential Assessment Tool) is a logistic regression model-based classifier that can accurately and quickly distinguish protein-coding and noncoding RNAs using pure linguistic features calculated from the RNA sequences. CPAT takes as input the nucleotides sequences or genomic coordinates of RNAs and outputs the probabilities p (0 ≤ p ≤ 1), which measure the likelihood of protein coding.","No description available.","No description available.","The NVIDIA CUDA® Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for deep neural networks. cuDNN provides highly tuned implementations for standard routines such as forward and backward convolution, pooling, normalization, and activation layers.","Cufflinks assembles transcripts and estimates their abundances in RNA-Seq samples. It accepts aligned RNA-Seq reads and assembles the alignments into a parsimonious set of transcripts. Cufflinks then estimates the relative abundances of these transcripts based on how many reads support each one.","Allows for persistent storage, access, exploration, and manipulation of Cufflinks high-throughput sequencing data. In addition, provides numerous plotting functions for commonly used visualizations.","Generate customized protein sequence database from RNA-Seq data for proteomics search.","No description available.","Find and remove adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.","Long Read based Human Genomic Structural Variation Detection with cuteSV | Long-read sequencing technologies enable to comprehensively discover structural variations (SVs). However, it is still non-trivial for state-of-the-art approaches to detect SVs with high sensitivity or high performance or both. Herein, we propose cuteSV, a sensitive, fast and lightweight SV detection approach. cuteSV uses tailored methods to comprehensively collect various types of SV signatures, and a clustering-and-refinement method to implement a stepwise SV detection, which enables to achieve high sensitivity without loss of accuracy. Benchmark results demonstrate that cuteSV has better yields on real datasets. Further, its speed and scalability are outstanding and promising to large-scale data analysis","This package infers exact sequence variants (SVs) from amplicon data, replacing the commonly used and coarser OTU clustering approach. This pipeline inputs demultiplexed fastq files, and outputs the sequence variants and their sample-wise abundances after removing substitution and chimera errors. Taxonomic classification is available via a native implementation of the RDP naive Bayesian classifier.","No description available.","No description available.","No description available.","No description available.","No description available.","User-friendly tools for the normalization and visualization of deep-sequencing data.","DendroPy is a Python library for phylogenetic computing. It provides classes and functions for the simulation, processing, and manipulation of phylogenetic trees and character matrices, and supports the reading and writing of phylogenetic data in a range of formats.","R/Bioconductor package for differential gene expression analysis based on the negative binomial distribution. Estimate variance-mean dependence in count data from high-throughput sequencing assays and test for differential expression based on a model using the negative binomial distribution.","The package is focused on finding differential exon usage using RNA-seq exon counts between samples with different experimental designs. It provides functions that allows the user to make the necessary statistical tests based on a model that uses the negative binomial distribution to estimate the variance between biological replicates and generalized linear models for testing. The package also provides functions for the visualization and exploration of the results.","Neural networks and interference correction enable deep proteome coverage in high throughput.\n\nDIA-NN - a fast and easy to use tool for processing data independent acquisition (DIA) proteomics data.\n\nNone required (for .raw, .mzML and .dia processing). Two executables are provided: DiaNN.exe (a command-line tool) and DIA-NN.exe (a GUI implemented as a wrapper for DiaNN.exe)","DIA-Umpire is an open source Java program for computational analysis of data independent acquisition (DIA) mass spectrometry-based proteomics data. It enables untargeted peptide and protein identification and quantitation using DIA data, and also incorporates targeted extraction to reduce the number of cases of missing quantitation.","Sequence aligner for protein and translated DNA searches and functions as a drop-in replacement for the NCBI BLAST software tools. It is suitable for protein-protein search as well as DNA-protein search on short reads and longer sequences including contigs and assemblies, providing a speedup of BLAST ranging up to x20,000.","diaPASEF is an appproch for parallel accumulation-serial fragmentation combined with data-independent acquisition.","Compute differentially bound sites from multiple ChIP-seq experiments using affinity (quantitative) data. Also enables occupancy (overlap) analysis and plotting functions.","No description available.","No description available.","Provides a number of utility functions for handling single-cell (RNA-seq) data from droplet technologies such as 10X Genomics. This includes data loading, identification of cells from empty droplets, removal of barcode-swapped pseudo-cells, and downsampling of the count matrix.","No description available.","EasyBuild is a software build and installation framework that allows you to manage (scientific) software on High Performance Computing (HPC) systems in an efficient way.","No description available.","Differential expression analysis of RNA-seq expression profiles with biological replication. Implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi-likelihood tests. As well as RNA-seq, it be applied to differential signal analysis of other types of genomic data that produce counts, including ChIP-seq, SAGE and CAGE.","Differential expression analysis of RNA-seq expression profiles with biological replication. Implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi-likelihood tests. As well as RNA-seq, it be applied to differential signal analysis of other types of genomic data that produce counts, including ChIP-seq, SAGE and CAGE.","For fast functional annotation of novel sequences. It uses precomputed orthologous groups and phylogenies from the eggNOG database to transfer functional information from fine-grained orthologs only. Its common uses include the annotation of novel genomes, transcriptomes or even metagenomic gene catalogs.\n\nThe use of orthology predictions for functional annotation permits a higher precision than traditional homology searches, as it avoids transferring annotations from close paralogs.","This package implements the Ensemble of Gene Set Enrichment Analyses method for gene set testing.","Evaluation of an Open Source Registration Package for Automatic Contour Propagation in Online Adaptive Intensity-Modulated Proton Therapy of Prostate Cancer.\n\nHome : About : FAQ : wiki : Download : News : Legal stuff : Documentation.\n\nWelcome to elastix : a toolbox for rigid and nonrigid registration of images.\n\nelastix is open source software, based on the well-known Insight Segmentation and Registration Toolkit (ITK). The software consists of a collection of algorithms that are commonly used to solve (medical) image registration problems. The modular design of elastix allows the user to quickly configure, test, and compare different registration methods for a specific application. A command-line interface enables automated processing of large numbers of data sets, by means of scripting. Nowadays elastix is accompanied by SimpleElastix , making it available in languages like C++, Python, Java, R, Ruby, C# and Lua.","Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.","No description available.","EncyclopeDIA is library search engine comprised of several algorithms for DIA data analysis and can search for peptides using either DDA-based spectrum libraries or DIA-based chromatogram libraries.","No description available.","No description available.","Integrated database covering the eukaryotic pathogens of the genera Cryptosporidium, Giardia, Leishmania, Neospora, Plasmodium, Toxoplasma, Trichomonas and Trypanosoma. The database portal offers an entry point to all these resources, and the opportunity to leverage orthology for searches across genera.","ExaBayes is a software package for Bayesian phylogenetic tree inference. It is particularly suitable for large-scale analyses on computer clusters.","Tool for phylogenomic analyses on supercomputers.","A tool for pairwise sequence alignment. It enables alignment for DNA-DNA and DNA-protein pairs and also gapped and ungapped alignment.","No description available.","Streaming tool for quantifying the abundances of a set of target sequences from sampled subsequences. Example applications include transcript-level RNA-Seq quantification, allele-specific/haplotype expression analysis (from RNA-Seq), transcription factor binding quantification in ChIP-Seq, and analysis of metagenomic data. It can be used to resolve ambiguous mappings in other high-throughput sequencing based applications.","No description available.","Experimental PacBio diploid assembler.","No description available.","No description available.","No description available.","No description available.","No description available.","No description available.","No description available.","FastANI is developed for fast alignment-free computation of whole-genome Average Nucleotide Identity (ANI). ANI is defined as mean nucleotide identity of orthologous gene pairs shared between two microbial genomes. FastANI supports pairwise comparison of both complete and draft genome assemblies.","Read huge FastQ and FastA files (both normal and gzipped) an demanipulate them.","A tool designed to provide fast all-in-one preprocessing for FastQ files. This tool is developed in C++ with multithreading supported to afford high performance.","This tool aims to provide a QC report which can spot problems or biases which originate either in the sequencer or in the starting library material. It can be run in one of two modes. It can either run as a stand alone interactive application for the immediate analysis of small numbers of FastQ files, or it can be run in a non-interactive mode where it would be suitable for integrating into a larger analysis pipeline for the systematic processing of large numbers of files.","No description available.","Infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences.","Collection of command line tools for Short-Reads FASTA/FASTQ files preprocessing.","Collection of command line tools for Short-Reads FASTA/FASTQ files preprocessing.","No description available.","HMM-based gene structure prediction (multiple genes, both chains); Program for predicting multiple genes in genomic DNA sequences.","The package implements an algorithm for fast gene set enrichment analysis. Using the fast algorithm allows to make more permutations and get more fine grained p-values, which allows to use accurate stantard approaches to multiple hypothesis correction.","No description available.","No description available.","No description available.","Filtlong is a tool for filtering long reads by quality. It can take a set of long reads and produce a smaller, better subset. It uses both read length (longer is better) and read identity (higher is better) when choosing which reads pass the filter.","FlashLFQ is an ultrafast label-free quantification algorithm for mass-spectrometry proteomics.","No description available.","Flye is a de novo assembler for single molecule sequencing reads, such as those produced by PacBio and Oxford Nanopore Technologies. It is designed for a wide range of datasets, from small bacterial projects to large mammalian-scale assemblies. The package represents a complete pipeline: it takes raw PB / ONT reads as input and outputs polished contigs.","No description available.","Foldseek enables fast and sensitive comparisons of large structure sets. It reaches sensitivities similar to state-of-the-art structural aligners while being at least 20,000 times faster.","Web server which detects small molecule pockets by relying on the geometric alpha sphere theory. It also tracks pockets during molecular dynamics so to provide insight on pocket dynamics (mdpocket) and transposes mdpocket to the combined analysis of homologous structures (hpocket).","Application for finding (fragmented) genes in short reads","Bayesian genetic variant detector designed to find small polymorphisms, specifically SNPs, indels, multi-nucleotide polymorphisms, and complex events (composite insertion and substitution events) smaller than the length of a short-read sequencing alignment.","No description available.","funannotate is a pipeline for genome annotation (built specifically for fungi, but will also work with higher eukaryotes).","Gene Annotation EVAluation.","No description available.","No description available.","No description available.","No description available.","No description available.","No description available.","No description available.","The Genome Analysis Toolkit (GATK) is a set of bioinformatic tools for analyzing high-throughput sequencing (HTS) and variant call format (VCF) data. The toolkit is well established for germline short variant discovery from whole genome and exome sequencing data. GATK4 expands functionality into copy number and somatic analyses and offers pipeline scripts for workflows.  \n\nVersion 4 (GATK4) is open-source at https://github.com/broadinstitute/gatk.","Cleaning aligned sequences.","Genome-wide Complex Trait Analysis. Estimate the proportion of phenotypic variance explained by genome- or chromosome-wide SNPs for complex traits (the GREML method), and has subsequently extended for many other analyses to better understand the genetic architecture of complex traits.","Software aimed at pairwise sequence comparison generating high quality results (equivalent to MUMmer) with controlled memory consumption and comparable or faster execution times particularly with long sequences.","GEMINI (GEnome MINIng) is a flexible framework for exploring genetic variation in the context of the wealth of genome annotations available for the human genome. By placing genetic variants, sample phenotypes and genotypes, as well as genome annotations into an integrated database framework, GEMINI provides a simple, flexible, and powerful system for exploring genetic variation for disease and population genetics.","Gene Model Mapper is a homology-based gene prediction program. GeMoMa uses the annotation of protein-coding genes in a reference genome to infer the annotation of protein-coding genes in a target genome. Thereby, it utilizes amino acid sequence and intron position conservation. In addition, it allows to incorporate RNA-seq evidence for splice site prediction.","No description available.","No description available.","No description available.","No description available.","Reference-free profiling of polyploid genomes | We have developed GenomeScope 2.0, which applies classical insights from combinatorial theory to establish a detailed mathematical model of how k-mer frequencies will be distributed in heterozygous and polyploid genomes | Average k-mer coverage for polyploid genome | Upload results from running Jellyfish or KMC","Free collection of bioinformatics tools for genome informatics.","No description available.","No description available.","A fast and versatile toolkit for accurate de novo assembly of organelle genomes.\n\nThis toolkit assemblies organelle genome from genomic skimming data.","No description available.","gfastats is a single fast and exhaustive tool for summary statistics and simultaneous genome assembly file manipulation. gfastats also allows seamless fasta/fastq/gfa conversion.","No description available.","No description available.","Program for comparing, annotating, merging and tracking transcripts in GFF files.","program for filtering, converting and manipulating GFF files","Plotting system for R, based on the grammar of graphics.","No description available.","No description available.","GNU parallel is a shell tool for executing jobs in parallel using one or more computers. A job can be a single command or a small script that has to be run for each of the lines in the input. The typical input is a list of files, a list of hosts, a list of users, a list of URLs, or a list of tables. A job can also be a command that reads from a pipe. GNU parallel can then split the input and pipe it into commands in parallel.","The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.","GOEnrichment is a tool for performing GO enrichment analysis of gene sets, such as those obtained from RNA-seq or Microarray experiments, to help characterize them at the functional level. It is available in Galaxy Europe and as a stand-alone tool.\nGOEnrichment is flexible in that it allows the user to use any version of the Gene Ontology and any GO annotation file they desire. To enable the use of GO slims, it is accompanied by a sister tool GOSlimmer, which can convert annotation files from full GO to any specified GO slim.\nThe tool features an optional graph clustering algorithm to reduce the redundancy in the set of enriched GO terms and simplify its output.\nIt was developed by the BioData.pt / ELIXIR-PT team at the Instituto Gulbenkian de Ciência.","Detect Gene Ontology and/or other user defined categories which are over/under represented in RNA-seq data.","No description available.","GraPhlAn is a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees. GraPhlAn focuses on concise, integrative, informative, and publication-ready representations of phylogenetically- and taxonomically-driven investigation.","Versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles. It is primarily designed for biochemical molecules like proteins, lipids and nucleic acids that have a lot of complicated bonded interactions, but since it is extremely fast at calculating the nonbonded interactions (that usually dominate simulations) many groups are also using it for research on non-biological systems, e.g. polymers.","a toolkit to classify genomes with the Genome Taxonomy Database.\n\nGTDB-Tk: a toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.\n\nGTDB-Tk is a software toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes based on the Genome Database Taxonomy GTDB. It is designed to work with recent advances that allow hundreds or thousands of metagenome-assembled genomes (MAGs) to be obtained directly from environmental samples. It can also be applied to isolate and single-cell genomes. The GTDB-Tk is open source and released under the GNU General Public License (Version 3).","No description available.","No description available.","Gubbins is a tool for rapid phylogenetic analysis of large samples of recombinant bacterial whole genome sequences.","HapCUT2 is a maximum-likelihood-based tool for assembling haplotypes from DNA sequence reads, designed to \"just work\" with excellent speed and accuracy across a range of long- and short-read sequencing technologies.\nThe output is in Haplotype block format described here: https://github.com/vibansal/HapCUT2/blob/master/outputformat.md","No description available.","Tool for single-species active module discovery.","This tool provides functional annotation for a list of genes by connecting with DAVID database.","This tool can be used to analyze the patterns of linkage disequilibrium (LD) between polymorphic sites in a locus.","This tool creates a link to the g:GOSt tool (Gene Group Functional Profiling), which provides functional profiling of gene lists.","A web server for reproducible Hi-C, capture Hi-C and single-cell Hi-C data analysis, quality control and visualization.\n\nHiCExplorer — HiCExplorer 3.6 documentation.\n\nscHiCExplorer — scHiCExplorer 7 documentation.\n\nFree document hosting provided by Read the Docs.","Remove CCS reads with remnant PacBio adapter sequences and convert outputs to a compressed .fastq (.fastq.gz).","Hifiasm: a haplotype-resolved assembler for accurate Hifi reads","No description available.","Alignment program for mapping next-generation sequencing reads (both DNA and RNA) to a population of human genomes (as well as to a single reference genome).","No description available.","No description available.","This tool is used for searching sequence databases for homologs of protein sequences, and for making protein sequence alignments. It implements methods using probabilistic models called profile hidden Markov models. The new HMMER3 project, HMMER is now as fast as BLAST for protein search.","Horovod is a distributed deep learning training framework for TensorFlow, Keras, PyTorch, and Apache MXNet. The goal of Horovod is to make distributed deep learning fast and easy to use.","Python framework to process and analyse high-throughput sequencing (HTS) data","The main purpose of HTSlib is to provide access to genomic information files, both alignment data (SAM, BAM, and CRAM formats) and variant data (VCF and BCF formats). The library also provides interfaces to access and index genome reference data in FASTA format and tab-delimited files with genomic coordinates. It is utilized and incorporated into both SAMtools and BCFtools.","HUMAnN is a pipeline for efficiently and accurately profiling the presence/absence and abundance of microbial pathways in a community from metagenomic or metatranscriptomic sequencing data (typically millions of short DNA/RNA reads). This process, referred to as functional profiling, aims to describe the metabolic potential of a microbial community and its members. More generally, functional profiling answers the question “What are the microbes in my community-of-interest doing (or are capable of doing)?”","HUMAnN 2.0 is a pipeline for efficiently and accurately profiling the presence/absence and abundance of microbial pathways in a community from metagenomic or metatranscriptomic sequencing data (typically millions of short DNA/RNA reads). This process, referred to as functional profiling, aims to describe the metabolic potential of a microbial community and its members. More generally, functional profiling answers the question “What are the microbes in my community-of-interest doing (or capable of doing)?”","Software package for the analysis of genetic sequences using techniques in phylogenetics, molecular evolution, and machine learning.","No description available.","No description available.","Improved Phased Assembler (IPA) is the official PacBio software for HiFi genome assembly. IPA was designed to utilize the accuracy of PacBio HiFi reads to produce high-quality phased genome assemblies","Infernal (\"INFERence of RNA ALignment\") is for searching DNA sequence databases for RNA structure and sequence similarities. It is an implementation of a special case of profile stochastic context-free grammars called covariance models (CMs). A CM is like a sequence profile, but it scores a combination of sequence consensus and RNA secondary structure consensus, so in many cases, it is more capable of identifying RNA homologs that conserve their secondary structure more than their primary sequence.","No description available.","High-performance visualization tool for interactive exploration of large, integrated datasets. It supports a wide variety of data types and format, including short-read alignments in the SAM/BAM format. Data can be viewed from local files or over the web via http.","A tool to detect Integron in DNA sequences.","No description available.","No description available.","Scan sequences against the InterPro protein signature databases.","No description available.","No description available.","No description available.","Interactive assembly and analysis of RADseq datasets.\n\nipyrad: interactive assembly and analysis of RAD-seq data sets.\n\nWelcome to ipyrad, an interactive toolkit for assembly and analysis of restriction-site associated genomic data sets (e.g., RAD, ddRAD, GBS) for population genetic and phylogenetic studies.\n\nWelcome to ipyrad — ipyrad documentation.","Very efficient phylogenetic software for reconstructing maximum-likelihood trees and assessing branch supports with the ultrafast bootstrap approximation. It is based on the IQPNNI algorithm with 10-fold speedup together with substantially additional features.","Provides functions for creating an interactive Shiny-based graphical user interface for exploring data stored in SummarizedExperiment objects, including row- and column-level metadata. Particular attention is given to single-cell data in a SingleCellExperiment object with visualization of dimensionality reduction results.","Automated identification of insertion sequence elements in prokaryotic genomes.","Enables identification of isoform switches with predicted functional consequences from RNA-seq data. Consequences can be chosen from a long list but includes protein domains gain/loss changes in NMD sensitivity etc. It directly supports import of data from Cufflinks/Cuffdiff, Kallisto, Salmon and RSEM but other transcript qunatification tools are easy to import as well.","Interpretation-oriented tool to manage the update and revision of variant annotation and classification.\n\niVar - DataBase of Genomics Variants.","Implementation of the Interval-Wise Testing (IWT) for omics data. This inferential procedure tests for differences in \"Omics\" data between two groups of genomic regions (or between a group of genomic regions and a reference center of symmetry), and does not require fixing location and scale at the outset.","JAGS is Just Another Gibbs Sampler. It is a program for analysis of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation.","JASMINE (Jointly Accurate Sv Merging with Intersample Network Edges) is an automated pipeline for alignment and SV calling in long-read datasets. The tool is used to merge structural variants (SVs) across samples. Each sample has a number of SV calls, consisting of position information (chromosome, start, end, length), type and strand information, and a number of other values. Jasmine represents the set of all SVs across samples as a network, and uses a modified minimum spanning forest algorithm to determine the best way of merging the variants such that each merged variants represents a set of analogous variants occurring in different samples.","No description available.","No description available.","Slick, speedy genome browser with a responsive and dynamic AJAX interface for visualization of genome data. Being developed by the GMOD project as a successor to GBrowse.","No description available.","A command-line algorithm for counting k-mers in DNA sequence.","No description available.","No description available.","No description available.","A program for quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads. It is based on the novel idea of pseudoalignment for rapidly determining the compatibility of reads with targets, without the need for alignment.","Suite of tools that generate, analyse and compare k-mer spectra produced from sequence files","No description available.","khmer is a set of command-line tools for working with DNA shotgun sequencing data from genomes, transcriptomes, metagenomes, and single cells. khmer can make de novo assemblies faster, and sometimes better. khmer can also identify (and fix) problems with shotgun data.","KMC is a utility designed for counting k-mers (sequences of consecutive k symbols) in a set of reads from genome sequencing projects.","No description available.","System for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies. Previous attempts by other bioinformatics software to accomplish this task have often used sequence alignment or machine learning techniques that were quite slow, leading to the development of less sensitive but much faster abundance estimation programs. It aims to achieve high sensitivity and high speed by utilizing exact alignments of k-mers and a novel classification algorithm.","Kraken 2 is the newest version of Kraken, a taxonomic classification system using exact k-mer matches to achieve high accuracy and fast classification speeds. This classifier matches each k-mer within a query sequence to the lowest common ancestor (LCA) of all genomes containing the given k-mer. The k-mer assignments inform the classification algorithm.","Krona creates interactive HTML5 charts of hierarchical data (such as taxonomic abundance in a metagenome).","No description available.","Automated image analysis for developmental phenotyping of mouse embryos.\n\nLAMA (Lightweight Analysis of Morphological Abnormalities).\n\nWelcome to LAMA, an open source pipeline to automatically identify embryo dysmorphology from 3D volumetric images.","A tool for (1) aligning two DNA sequences, and (2) inferring appropriate scoring parameters automatically.","No description available.","An Easy-To-Use Interactive Web Platform To Analyze and Visualize Label-Free Proteomics Data Preprocessed with MaxQuant.\n\nA tool for analysing label-free quantitative proteomics dataset https://bioinformatics.erc.monash.edu/apps/LFQ-Analyst/.\n\nLFQ-Analyst: An easy-to-use interactive web-platform to analyze and visualize proteomics data preprocessed with MaxQuant.\n\nLFQ-Analyst is an easy-to-use, interactive web application developed to perform differential expression analysis with “one click” and to visualize label-free quantitative proteomic datasets preprocessed with MaxQuant. LFQ-Analyst provides a wealth of user-analytic features and offers numerous publication-quality result output graphics and tables to facilitate statistical and exploratory analysis of label-free quantitative datasets","No description available.","Data analysis, linear models and differential expression for microarray data.","LINKS (Long Interval Nucleotide K-mer Scaffolder) is a genomics application for scaffolding genome assemblies with long reads, such as those produced by Oxford Nanopore Technologies Ltd. It can be used to scaffold high-quality draft genome assemblies with any long sequences (eg. ONT reads, PacBio reads, other draft genomes, etc). It is also used to scaffold contig pairs linked by ARCS/ARKS.","LoFreq* (i.e. LoFreq version 2) is a fast and sensitive variant-caller for inferring SNVs and indels from next-generation sequencing data. It makes full use of base-call qualities and other sources of errors inherent in sequencing (e.g. mapping or base/indel alignment uncertainty), which are usually ignored by other methods or only used for filtering.","Longshot is a variant calling tool for diploid genomes using long error prone reads such as Pacific Biosciences (PacBio) SMRT and Oxford Nanopore Technologies (ONT). It takes as input an aligned BAM file and outputs a phased VCF file with variants and haplotype information. It can also genotype and phase input VCF files. It can output haplotype-separated BAM files that can be used for downstream analysis. Currently, it only calls single nucleotide variants (SNVs), but it can genotype indels if they are given in an input VCF.","No description available.","LTR_retriever is a highly accurate and sensitive program for identification of LTR retrotransposons; The LTR Assembly Index (LAI) is also included in this package.","Model-based Analysis of ChIP-seq data.","Model-based Analysis of ChIP-seq data.","No description available.","No description available.","No description available.","No description available.","No description available.","No description available.","No description available.","No description available.","No description available.","No description available.","No description available.","MAFFT (Multiple Alignment using Fast Fourier Transform) is a high speed multiple sequence alignment program.","Computational tool to identify important genes from the recent genome-scale CRISPR-Cas9 knockout screens technology.","Portable and easily configurable genome annotation pipeline. It’s purpose is to allow smaller eukaryotic and prokaryotic genome projects to independently annotate their genomes and to create genome databases.","MALDIquant is a complete analysis pipeline for matrix-assisted laser desorption/ionization-time-of-flight (MALDI-TOF) and other two-dimensional mass spectrometry data. In addition to commonly used plotting and processing methods it includes distinctive features, namely baseline subtraction methods such as morphological filters (TopHat) or the statistics-sensitive non-linear iterative peak-clipping algorithm (SNIP), peak alignment using warping functions, handling of replicated measurements as well as allowing spectra with different resolutions.","Fast genome and metagenome distance estimation using MinHash.","No description available.","Whole genome assembly software. It combines the efficiency of the de Bruijn graph and Overlap-Layout-Consensus (OLC) approaches. MaSuRCA can assemble data sets containing only short reads from Illumina sequencing or a mixture of short reads and long reads (Sanger, 454).","Tool to import, process, clean, and compare mass spectrometry data.","Matplotlib is a comprehensive library for creating static, animated, and interactive visualizations in Python. Matplotlib makes easy things easy and hard things possible.","Software for binning assembled metagenomic sequences based on an Expectation-Maximization algorithm.","Quantitative proteomics software package designed for analyzing large mass-spectrometric data sets. It is specifically aimed at high-resolution MS data.","Quantitative proteomics software package designed for analyzing large mass-spectrometric data sets. It is specifically aimed at high-resolution MS data.","MCL is a clustering algorithm widely used in bioinformatics and gaining traction in other fields.","MDAnalysis is an object-oriented python toolkit to analyze molecular dynamics trajectories generated by CHARMM, Gromacs, NAMD, LAMMPS, Amber or DL_POLY; it also reads other formats (e.g. PDB files and XYZ format trajectories; see the supported coordinate formats for the full list). It can write most of these formats, too, together with atom selections for use in Gromacs, CHARMM, VMD and PyMol","No description available.","medaka is a tool to create consensus sequences and variant calls from nanopore sequencing data. This task is performed using neural networks applied a pileup of individual sequencing reads against a draft assembly.","Single node assembler for large and complex metagenomics NGS reads, such as soil. It makes use of succinct de Bruijn graph to achieve low memory usage, whereas its goal is not to make memory usage as low as possible.","No description available.","Reference-free quality, completeness, and phasing assessment for genome assemblies.\n\nEvaluate genome assemblies with k-mers and more.\nOften, genome assembly projects have illumina whole genome sequencing reads available for the assembled individual.\n\nMerqury provides a set of tools for this purpose.","Meryl is a tool for counting and working with sets of k-mers that was originally developed for use in the Celera Assembler and has since been migrated and maintained as part of Canu.","an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies | MetaBAT2 clusters metagenomic contigs into different \"bins\", each of which should correspond to a putative genome | MetaBAT2 uses nucleotide composition information and source strain abundance (measured by depth-of-coverage by aligning the reads to the contigs) to perform binning","No description available.","Galaxy workflow for differential abundance analysis of 16s metagenomic data.\n\nYou are over your disk quota. Tool execution is on hold until your disk usage drops below your allocated quota.\n\nThis history is empty. You can load your own data or get data from an external source","MetaEuk - sensitive, high-throughput gene discovery and annotation for large-scale eukaryotic metagenomics","Computational tool for profiling the composition of microbial communities from metagenomic shotgun sequencing data.","metaQuantome software suite analyzes the state of a microbiome by leveraging complex taxonomic and functional hierarchies to summarize peptide-level quantitative information. metaQuantome offers differential abundance analysis, principal components analysis, and clustered heat map visualizations, as well as exploratory analysis for a single sample or experimental condition.","Genome assembler for metagenomics datasets.","MetaWRAP aims to be an easy-to-use metagenomic wrapper suite that accomplishes the core tasks of metagenomic analysis from start to finish: read quality control, assembly, visualization, taxonomic profiling, extracting draft genomes (binning), and functional annotation.","A (mostly) universal methylation extractor for BS-seq experiments.","No description available.","Estimates effective population sizes,past migration rates between n population assuming a migration matrix model with asymmetric migration rates and different subpopulation sizes, and population divergences or admixture.","No description available.","Short-read assembler based on a de Bruijn graph, capable of assembling a human genome on a desktop computer in a day.","Miniasm is a very fast OLC-based de novo assembler for noisy long reads. It takes all-vs-all read self-mappings (typically by minimap) as input and outputs an assembly graph in the GFA format.","No description available.","Pairwise aligner for genomic and spliced nucleotide sequences","Miniprot aligns a protein sequence against a genome with affine gap penalty, splicing and frameshift. It is primarily intended for annotating protein-coding genes in a new species using known genes from other species.","No description available.","miRDeep2 discovers active known or novel miRNAs from deep sequencing data.","No description available.","De novo metazoan mitochondrial genome annotation.","Multi Locus Sequence Typing from an assembled genome or from a set of reads.","MMseqs2 (Many-against-Many sequence searching) is a software suite to search and cluster huge protein and nucleotide sequence sets. MMseqs2 is open source GPL-licensed software implemented in C++ for Linux, MacOS, and (as beta version, via cygwin) Windows. The software is designed to run on multiple cores and servers and exhibits very good scalability. MMseqs2 can run 10000 times faster than BLAST. At 100 times its speed it achieves almost the same sensitivity. It can perform profile searches with the same sensitivity as PSI-BLAST at over 400 times its speed.\n\nMMseqs2 includes Linclust, the first clustering algorithm whose runtime scales linearly With Linclust we clustered 1.6 billion metagenomic sequence fragments in 10 h on a single server to 50% sequence identity.","Universal whole-sequence-based plasmid typing and its utility to prediction of host range and epidemiological surveillance.\n\nMOB-suite: Software tools for clustering, reconstruction and typing of plasmids from draft assemblies.\n\nPlasmids are mobile genetic elements (MGEs), which allow for rapid evolution and adaption of bacteria to new niches through horizontal transmission of novel traits to different genetic backgrounds. The MOB-suite is designed to be a modular set of tools for the typing and reconstruction of plasmid sequences from WGS assemblies.","No description available.","No description available.","No description available.","A modest Feature Finder to extract features in MS1 Data.","No description available.","A proteomics search algorithm specifically designed for high-resolution tandem mass spectra.","Open-source, platform-independent, community-supported software for describing and comparing microbial communities","Data warehouse for accessing mouse data from Mouse Genome Informatics (MGI). Supports powerful query, reporting, and analysis capabilities, the ability to save and combine results from different queries, easy integration into larger workflows, and a comprehensive Web Services layer.","Program for Bayesian inference and model choice across a wide range of phylogenetic and evolutionary models. It uses Markov chain Monte Carlo (MCMC) methods to estimate the posterior distribution of model parameters.","msConvert is a command-line utility for converting between various mass spectrometry data formats, including from raw data from several commercial companies (with vendor libraries, Windows-only). For Windows users, there is also a GUI, msConvertGUI.","Tool for mass spectra metadata annotation.","Statistical tool for quantitative mass spectrometry-based proteomics.","MultiQC aggregates results from multiple bioinformatics analyses across many samples into a single report. It searches a given directory for analysis logs and compiles a HTML report. It's a general use tool, perfect for summarising the output from numerous bioinformatics tools.","MUMmer is a modular system for the rapid whole genome alignment of finished or draft sequence. Basically it is a ultra-fast alignment of large-scale DNA and protein sequences","No description available.","No description available.","No description available.","RNA modifications detection by comparative Nanopore direct RNA sequencing.\n\nRNA modifications detection from Nanopore dRNA-Seq data.\n\nNanocompore identifies differences in ONT nanopore sequencing raw signal corresponding to RNA modifications by comparing 2 samples.\n\nAnalyses performed for the nanocompore paper.\n\nNanocompore compares 2 ONT nanopore direct RNA sequencing datasets from different experimental conditions expected to have a significant impact on RNA modifications. It is recommended to have at least 2 replicates per condition. For example one can use a control condition with a significantly reduced number of modifications such as a cell line for which a modification writing enzyme was knocked-down or knocked-out. Alternatively, on a smaller scale transcripts of interests could be synthesized in-vitro","No description available.","NanoPlot is a tool with various visualizations of sequencing data in bam, cram, fastq, fasta or platform-specific TSV summaries, mainly intended for long-read sequencing from Oxford Nanopore Technologies and Pacific Biosciences","A package for detecting cytosine methylations and genetic variations from nanopore MinION sequencing data.","No description available.","No description available.","NCBI Datasets is a new resource that lets you easily gather data from across NCBI databases. Find and download sequence, annotation, and metadata for genes and genomes using our command-line tools or web interface.","No description available.","nccmp compares two NetCDF files bitwise, semantically or with a user defined tolerance (absolute or relative percentage).  Parallel comparisons are done in local memory without requiring temporary files.  Highly recommended for regression testing scientific models or datasets in a test-driven development environment.","No description available.","The NCO toolkit manipulates and analyzes data stored in netCDF-accessible formats, including DAP, HDF4, and HDF5.","Ncview is a netCDF visual browser.","NetCDF (Network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data.","The Newick Utilities are a set of command-line tools for processing phylogenetic trees. They can process arbitrarily large amounts of data and do not require user interaction, which makes them suitable for automating phylogeny processing tasks.","No description available.","Nextflow enables scalable and reproducible scientific workflows using software containers. It allows the adaptation of pipelines written in the most common scripting languages.","NGSUtils is a suite of software tools for working with next-generation sequencing datasets","Nearly Infinite Neighbor Joining Application","No description available.","No description available.","No description available.","Set of python programs developed to simplify the manipulation of sequence files. They were mainly designed to help us for analyzing Next Generation Sequencer outputs (454 or Illumina) in the context of DNA Metabarcoding.","No description available.","Open source library and a collection of tools and interfaces for the analysis of mass spectrometry data. Includes over 200 standalone (TOPP) tools that can be combined to a workflow with the integrated workflow editor TOPPAS. Raw and intermediate mass spectrometry data can be visualised with the included viewer TOPPView.","No description available.","No description available.","Pangolin is a deep-learning based method for predicting splice site strengths (for details, see Zeng and Li, Genome Biology 2022). It is available as a command-line tool that can be run on a VCF or CSV file containing variants of interest; Pangolin will predict changes in splice site strength due to each variant, and return a file of the same format. Pangolin's models can also be used with custom sequences.","No description available.","ParaView is an open-source, multi-platform data analysis and visualization application. ParaView users can quickly build visualizations to analyze their data using qualitative and quantitative techniques. The data exploration can be done interactively in 3D or programmatically using ParaView’s batch processing capabilities.","No description available.","Tool set for pathway based data integration and visualization that maps and renders a wide variety of biological data on relevant pathway graphs. It downloads the pathway graph data, parses the data file, maps user data to the pathway, and render pathway graph with the mapped data. In addition, it integrates with pathway and gene set (enrichment) analysis tools for large-scale and fully automated analysis.","Multithread blat algorithm speeding up aligning sequences to genomes.","Productive visualization of high-throughput sequencing data using the SeqCode open portable platform.","Paired-end read merger. PEAR evaluates all possible paired-end read overlaps without requiring the target fragment size as input. In addition, it implements a statistical test for minimizing false-positive results.","No description available.","PeptideShaker is a search engine independent platform for interpretation of proteomics identification results from multiple search engines, currently supporting X!Tandem, MS-GF+, MS Amanda, OMSSA, MyriMatch, Comet, Tide, Mascot, Andromeda and mzIdentML. By combining the results from multiple search engines, while re-calculating PTM localization scores and redoing the protein inference, PeptideShaker attempts to give you the best possible understanding of your proteomics data","No description available.","Semi-supervised learning for peptide identification from MS/MS data.","No description available.","This tool is used to search a FASTA sequence against a library of Pfam HMM.","No description available.","No description available.","Provides a set of classes and tools to facilitate the import, storage, analysis, and graphical display of microbiome census data.","Phylogenetic estimation software using Maximum Likelihood","A set of command line tools for manipulating high-throughput sequencing (HTS) data in formats such as SAM/BAM/CRAM and VCF. Available as a standalone program or within the GATK4 program.","PICRUSt (Phylogenetic Investigation of Communities by Reconstruction of Unobserved States) is a bioinformatics software package designed to predict metagenome functional content from marker gene (e.g., 16S rRNA) surveys and full genomes.","PICRUSt2 (Phylogenetic Investigation of Communities by Reconstruction of Unobserved States) is a software for predicting functional abundances based only on marker gene sequences.","No description available.","No description available.","Read alignment analysis to diagnose, report, and automatically improve de novo genome assemblies.","PIPE-T is a Galaxy Workflow for processing and analyzing miR expression profiles by RTqPCR.\n\nIt is a tool that offers several state-of-the-art options for parsing, filtering, normalizing, imputing and analyzing RT-qPCR expression data. Integration of PIPE-T into Galaxy allows experimentalists with strong bioinformatic background, as well as those without any programming or development expertise, to perform complex analysis in a simple to use, transparent, accessible, reproducible, and user-friendly environment","No description available.","Free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.","No description available.","No description available.","PnetCDF: A Parallel I/O Library for NetCDF File Access","No description available.","No description available.","Flexible toolkit for exploring datasets generated by nanopore sequencing devices from MinION for the purposes of quality control and downstream analysis.","Tools for performing taxonomic assignment based on phylogeny using pplacer and clst.","No description available.","Pretext is an OpenGL-powered pretext contact map viewer.","The pipeline runs PRODIGAL gene predictions on all genomes, runs pan-reciprocal BLAST, and identifies ortholog sets. For a set of orthologous genes, if the positions of the PRODIGAL selected starts coincide in a multiple sequence alignment, they are accepted. If they do not coincide, a consistent start position is sought where a majority of the highest-scoring PRODIGAL selected sites coincide. If such a position is found, it is accepted, and the predictions are changed for the outlying genes.","Software tool to annotate bacterial, archaeal and viral genomes quickly and produce standards-compliant output files.","No description available.","No description available.","Identifying and removing haplotypic duplication in primary genome assemblies | haplotypic duplication identification tool | scripts/pd_config.py: script to generate a configuration file used by run_purge_dups.py | purge haplotigs and overlaps in an assembly based on read depth | Given a primary assembly pri_asm and an alternative assembly hap_asm (optional, if you have one), follow the steps shown below to build your own purge_dups pipeline, steps with same number can be run simultaneously. Among all the steps, although step 4 is optional, we highly recommend our users to do so, because assemblers may produce overrepresented seqeuences. In such a case, The final step 4 can be applied to remove those seqeuences","No description available.","PycoQC computes metrics and generates interactive QC plots for Oxford Nanopore technologies sequencing data.","reproducible plots for multivariate genomic data sets.\n\nStandalone program and library to plot beautiful genome browser tracks.\n\npyGenomeTracks aims to produce high-quality genome browser tracks that are highly customizable. Currently, it is possible to plot:.","No description available.","A Python module for reading and manipulating SAM/BAM/VCF/BCF files.","PyTorch is an optimized tensor library for deep learning using GPUs and CPUs.","QIIME 2™ is a next-generation microbiome bioinformatics platform that is extensible, free, open source, and community developed.","No description available.","Platform-independent application written in Java and R that provides both a Graphical User Inteface (GUI) and a command-line interface to facilitate the quality control of alignment sequencing data.","A Collection of Tools for Viral Quasispecies Analysis | Abstract Summary quasitools is a collection of newly-developed, open-source tools for analyzing viral quasispcies data. The application suite includes tools with the ability to create consensus sequences, call nucleotide, codon, and amino acid variants, calculate the complexity of a quasispecies, and measure the genetic distance between two similar quasispecies. These tools may be run independently or in user-created workflows. Availability The quasitools suite is a freely available application licensed under the Apache License, Version 2.0. The source code, documentation, and file specifications are available at: https: phac-nml.github.io quasitools Contact gary.vandomselaar@canada.ca","QUAST stands for QUality ASsessment Tool.  \nIt evaluates a quality of genome assemblies by computing various metrics and providing nice reports.","Query Tabular is a Galaxy-based tool which manipulates tabular files.  Query Tabular automatically creates a SQLite database directly from a tabular file within a Galaxy workflow.  The SQLite database can be saved to the Galaxy history, and further process to generate tabular outputs containing desired information and formatting.","Free software environment for statistical computing and graphics.","No description available.","The Possibility to Use Oxford Nanopore Technology | Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads. http://genome.cshlp.org/content/early/2017/01/18/gr.214270.116 Note: This was the original repository which will no longer be officially maintained. Please use the new official repository here: | Racon is intended as a standalone consensus module to correct raw contigs generated by rapid assembly methods which do not include a consensus step | Consensus module for raw de novo DNA assembly of long uncorrected reads","A feature clustering algorithm for non-targeted mass spectrometric metabolomics data.","No description available.","a de novo genome assembler for long reads.\n\nRaven is a de novo genome assembler for long uncorrected reads.","A standalone tool for extracting data directly from raw files generated by Thermo Orbitrap family instruments.","A tool for Phylogenetic Analysis and Post-Analysis of Large Phylogenies.","RDKit is an Open-Source Cheminformatics Software. Fast, Efficient Fragment-Based Coordinate Generation for Open Babel.","No description available.","Tool for calculating the probability of nucleosome formation along a DNA sequence input by the user.","recetox-aplcms is a tool for peak detection in mass spectrometry data. The tool performs (1) noise removal, (2) peak detection, (3) retention time drift correction, (4) peak alignment and (5) weaker signal recovery as well as (6) suspect screening.","This is a program to detect and visualize RNA editing events at genomic scale using next-generation sequencing data.","A program that screens DNA sequences for interspersed repeats and low complexity DNA sequences. The output of the program is a detailed annotation of the repeats that are present in the query sequence as well as a modified version of the query sequence in which all the annotated repeats have been masked (default: replaced by Ns).","De-novo repeat family identification and modeling package. At the heart of RepeatModeler are two de-novo repeat finding programs ( RECON and RepeatScout ) which employ complementary computational methods for identifying repeat element boundaries and family relationships from sequence data. RepeatModeler assists in automating the runs of RECON and RepeatScout given a genomic database and uses the output to build, refine and classify consensus models of putative interspersed repeats.","RepeatScout is a tool to discover repetitive substrings in DNA.","No description available.","No description available.","The rjags package provides an interface from R to the JAGS library for Bayesian data analysis. JAGS uses Markov Chain Monte Carlo (MCMC) to generate a sequence of dependent samples\nfrom the posterior distribution of the parameters.","Workflow to process tandem MS files and build MassBank records. Functions include automated extraction of tandem MS spectra, formula assignment to tandem MS fragments, recalibration of tandem MS spectra with assigned fragments, spectrum cleanup, automated retrieval of compound information from Internet databases, and export to MassBank records.","No description available.","RMBlast is a RepeatMasker compatible version of the standard NCBI blastn program. The primary difference between this distribution and the NCBI distribution is the addition of a new program \"rmblastn\" for use with RepeatMasker and RepeatModeler.","No description available.","A high speed stand alone pan genome pipeline, which takes annotated assemblies in GFF3 format (produced by Prokka (Seemann, 2014)) and calculates the pan genome.","No description available.","We present a generative statistical model and associated inference methods that handle read mapping uncertainty in a principled manner. Through simulations parameterized by real RNASeq data, we show that our method is more accurate than previous methods. Our improved accuracy is the result of handling read mapping uncertainty with a statistical model and the estimation of gene expression levels as the sum of isoform expression levels.","Provides a number of useful modules that can comprehensively evaluate high throughput sequence data especially RNA-seq data. Some basic modules quickly inspect sequence quality, nucleotide composition bias, PCR bias and GC bias, while RNA-seq specific modules evaluate sequencing saturation, mapped reads distribution, coverage uniformity, strand specificity, transcript level RNA integrity etc.","Integrated development environment (IDE) for the R programming language.","RTG Core: Software for alignment and analysis of next-gen sequencing data.","No description available.","A software tool that implements a novel, is an alignment-free algorithm for the estimation of isoform abundances directly from a set of reference sequences and RNA-seq reads.","A tool for transcript expression quantification from RNA-seq data","No description available.","> VERY_LOW CONFIDENCE! | > CORRECT NAME OF TOOL COULD ALSO BE 'chromosome-scale', 'reference-quality', 'Hi-C', 'scaffolder' | Integrating Hi-C links with assembly graphs for chromosome-scale assembly | SALSA: A tool to scaffold long read assemblies with Hi-C data | SALSA: A tool to scaffold long read assemblies with Hi-C | This code is used to scaffold your assemblies using Hi-C data. This version implements some improvements in the original SALSA algorithm. If you want to use the old version, it can be found in the old_salsa branch","No description available.","No description available.","This tool is a high performance modern robust and fast tool (and library), written in the D programming language, for working with SAM, BAM and CRAM formats.","A tool to mark duplicates and extract discordant and split reads from SAM files.","No description available.","SAMtools and BCFtools are widely used programs for processing and analysing high-throughput sequencing data. They include tools for file format conversion and manipulation, sorting, querying, statistics, variant calling, and effect analysis amongst other methods.","Scalable toolkit for analyzing single-cell gene expression data. It includes preprocessing, visualization, clustering, pseudotime and trajectory inference and differential expression testing. The Python-based implementation efficiently deals with datasets of more than one million cells.","Pre-processing, quality control, normalization and visualization of single-cell RNA-seq data.","No description available.","Scikit-image contains image processing algorithms for SciPy, including IO, morphology, filtering, warping, color manipulation, object detection, etc.","No description available.","No description available.","No description available.","SEPP stands for SATé-Enabled Phylogenetic Placement and addresses the problem of phylogenetic placement for meta-genomic short reads","No description available.","No description available.","FASTA and FASTQ are basic and ubiquitous formats for storing nucleotide and protein sequences. Common manipulations of FASTA/Q file include converting, searching, filtering, deduplication, splitting, shuffling, and sampling. Existing tools only implement some of these manipulations, and not particularly efficiently, and some are only available for certain operating systems. Furthermore, the complicated installation process of required packages and running environments can render these programs less user friendly. SeqKit demonstrates competitive performance in execution time and memory usage compared to similar tools. The efficiency and usability of SeqKit enable researchers to rapidly accomplish common FASTA/Q file manipulations.","No description available.","A tool for processing sequences in the FASTA or FASTQ format. It parses both FASTA and FASTQ files which can also be optionally compressed by gzip.","Seurat is an R package designed for QC, analysis, and exploration of single-cell RNA-seq data. Seurat aims to enable users to identify and interpret sources of heterogeneity from single-cell transcriptomic measurements, and to integrate diverse types of single-cell data.","No description available.","De novo assembly from Oxford Nanopore reads.","Shovill is a pipeline for assembly of bacterial isolate genomes from Illumina paired-end reads.  Shovill uses SPAdes at its core, but alters the steps before and after the primary assembly step to get similar results in less time. Shovill also supports other assemblers like SKESA, Velvet and Megahit, so you can take advantage of the pre- and post-processing the Shovill provides with those too.","A clustering approach for identification of enriched domains from histone modification ChIP-seq data.","A text mining framework for interactive analysis and visualization of similarities among biomedical entities.\n\nFor each search query, PMIDs or abstracts from PubMed are saved.\n\n$ git clone https://github.com/dlal-group/simtext.\n\nFor all PMIDs in each row of a table the according abstracts are saved in additional columns.","No description available.","The Salmonella In Silico Typing Resource (SISTR) is an open-source and freely available web application for rapid in silico typing and serovar prediction from Salmonella genome assemblies using cgMLST and O and H antigen gene searching.","No description available.","No description available.","No description available.","Reference-free profiling of polyploid genomes | Inference of ploidy and heterozygosity structure using whole genome sequencing data | Smudgeplots are computed from raw or even better from trimmed reads and show the haplotype structure using heterozygous kmer pairs. For example: | This tool extracts heterozygous kmer pairs from kmer dump files and performs gymnastics with them. We are able to disentangle genome structure by comparing the sum of kmer pair coverages (CovA + CovB) to their relative coverage (CovA / (CovA + CovB)). Such an approach also allows us to analyze obscure genomes with duplications, various ploidy levels, etc | GenomeScope 2.0 and Smudgeplots: Reference-free profiling of polyploid genomes Timothy Rhyker Ranallo-Benavidez, Kamil S. Jaron, Michael C. Schatz bioRxiv 747568; doi: https://doi.org/10.1101/747568","Workflow engine and language. It aims to reduce the complexity of creating workflows by providing a fast and comfortable execution environment, together with a clean and modern domain specific specification language (DSL) in python style.","The Semi-HMM-based Nucleic Acid Parser is a gene prediction tool.","An algorithm for structural variation detection from third generation sequencing alignment.","Rapid haploid variant calling and core SNP phylogeny generation.","No description available.","No description available.","Variant annotation and effect prediction tool. It annotates and predicts the effects of variants on genes and proteins (such as amino acid changes).","No description available.","No description available.","Toolbox that allows you to filter and manipulate annotated vcf files.","SOAPdenovo2 is a next generation sequencing reads de novo assembler.","Sequence analysis tool for filtering, mapping and OTU-picking NGS reads.","St. Petersburg genome assembler – is intended for both standard isolates and single-cell MDA bacteria assemblies. SPAdes 3.9 works with Illumina or IonTorrent reads and is capable of providing hybrid assemblies using PacBio, Oxford Nanopore and Sanger reads. Additional contigs can be provided and can be used as long reads.","No description available.","Spectral Repeat Finder (SRF) is a program to find repeats through an analysis of the power spectrum of a given DNA sequence.","No description available.","No description available.","No description available.","No description available.","The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for using data in the INSDC Sequence Read Archives.","A fast implementation of the Smith-Waterman algorithm whose API that can be flexibly used by programs written in C, C++ and other languages.","Developed to work with restriction enzyme based sequence data, such as RADseq, for building genetic maps and conducting population genomics and phylogeography analysis.","Ultrafast universal RNA-seq data aligner","No description available.","STAR-Fusion, a method that is both fast and accurate in identifying fusion transcripts from RNA-Seq data","Fast and highly efficient assembler of RNA-Seq alignments into potential transcripts. It uses a novel network flow algorithm as well as an optional de novo assembly step to assemble and quantitate full-length transcripts representing multiple splice variants for each gene locus.","Subread is a general-purpose read aligner which can be used to map both genomic DNA-seq reads and RNA-seq reads. It uses a new mapping paradigm called \"seed-and-vote\" to achieve fast, accurate and scalable read mapping. It automatically determines if a read should be globally or locally aligned, therefore particularly powerful in mapping RNA-seq reads. It supports indel detection and can map reads with both fixed and variable lengths.","No description available.","An agile homology-based approach using a reduced SEED database to report the subsystems present in metagenomic samples and profile their abundances.","Structural variant detection from haploid and diploid genome assemblies.\n\nSVIM-asm - Structural variant identification method (Assembly edition).\n\nSVIM-asm (pronounced SWIM-assem) is a structural variant caller for haploid or diploid genome-genome alignments. It analyzes a given sorted BAM file (preferably from minimap2) and detects five different variant classes between the query assembly and the reference: deletions, insertions, tandem and interspersed duplications and inversions.","SyRI is tool for finding genomic rearrangements and local sequence differences from whole-genome assemblies.\n\nGenomic differences range from single nucleotide differences to complex structural variations. Current methods typically annotate sequence differences ranging from SNPs to large indels accurately but do not unravel the full complexity of structural rearrangements, including inversions, translocations, and duplications, where highly similar sequence changes in location, orientation, or copy number. Here, we present SyRI, a pairwise whole-genome comparison tool for chromosome-level assemblies. SyRI starts by finding rearranged regions and then searches for differences in the sequences, which are distinguished for residing in syntenic or rearranged regions. This distinction is important as rearranged regions are inherited differently compared to syntenic regions.","No description available.","No description available.","No description available.","Targetfinder.org provides a web based resource that allows users to find genes that have a similar expression to a query gene signature.","No description available.","No description available.","No description available.","No description available.","Prediction of cognitive impairment via deep learning trained with multi-center neuropsychological test data.\n\nAn end-to-end open source machine learning platform.\n\nAnnouncing the TensorFlow Dev Summit 2020 Learn more.\n\nTensorFlow is an end-to-end open source platform for machine learning. It has a comprehensive, flexible ecosystem of tools, libraries and community resources that lets researchers push the state-of-the-art in ML and developers easily build and deploy ML powered applications","lineage-level classification of transposable elements using conserved protein domains.\n\nNote: do not move or hard link TEsorter.py alone to anywhere else, as it rely on database/ and bin/. You can add the directory to PATH or soft link TEsorter.py to PATH","No description available.","Open-source, crossplatform tool that converts Thermo RAW files into open file formats such as MGF and to the HUPO-PSI standard file format mzML","Program that aligns RNA-Seq reads to a genome in order to identify exon-exon splice junctions. It is built on the ultrafast short read mapping program Bowtie. A stable SAMtools version is now packaged with the program.","TotalView is a debugger for High Performance Computing applications.","No description available.","TransDecoder identifies candidate coding regions within transcript sequences, such as those generated by de novo RNA-Seq transcript assembly using Trinity, or constructed based on RNA-Seq alignments to the genome using Tophat and Cufflinks.","A tool for the analysis of Tn-Seq data. It provides an easy to use graphical interface and access to three different analysis methods that allow the user to determine essentiality in a single condition as well as between conditions.","No description available.","Institute for Systems Biology \"Trans-Proteomic Pipeline\"","No description available.","Tandem Repeats Finder. Find tandem repeats in DNA sequences without the need to specify either the pattern or pattern size. It uses the method of k-tuple matching to avoid the need for full scale alignment matrix computations. It requires no a priori knowledge of the pattern, pattern size or number of copies. There are no restrictions on the size of the repeats that can be detected. It determines a consensus pattern for the smallest repetitive unit in the tandem repeat.","Tool for the automated removal of spurious sequences or poorly aligned regions from a multiple sequence alignment.","A wrapper tool around Cutadapt and FastQC to consistently apply quality and adapter trimming to FastQ files, with some extra functionality for MspI-digested RRBS-type (Reduced Representation Bisufite-Seq) libraries.","A flexible read trimming tool for Illumina NGS data","Trinity is a transcriptome assembler which relies on three different tools, inchworm an assembler, chrysalis which pools contigs and butterfly which amongst others compacts a graph resulting from butterfly with reads.","Comprehensive annotation suite designed for automatic functional annotation of transcriptomes, particularly de novo assembled transcriptomes, from model or non-model organisms.","A program for improved detection of transfer RNA genes in genomic sequence.","Trycycler: consensus long-read assemblies for bacterial genomes","Utilities for handling sequences and assemblies from the UCSC Genome Browser project.","No description available.","No description available.","Tools for handling Unique Molecular Identifiers in NGS data sets.","A tool for assembling bacterial genomes from a combination of short (2nd generation) and long (3rd generation) sequencing reads.","Metaproteomics data analysis with a focus on interactive data visualizations.","No description available.","No description available.","No description available.","Tool for predicting effects of variants for any genome in Ensembl or with genome annotation (via GFF). This includes vertebrates and also plants, fungi, protists, metazoa and bacteria.  There is a web and a REST API version but the most powerful is the Perl script version. See McLaren et al., 2016, Genome Biology","VarScan, an open source tool for variant detection that is compatible with several short read align-ers.","No description available.","No description available.","API and command line utilities for the manipulation of VCF files.","Provide easily accessible methods for working with complex genetic variation data in the form of VCF files.","A de novo genomic assembler specially designed for short read sequencing technologies, such as Solexa or 454 or SOLiD.","No description available.","No description available.","No description available.","High-throughput search and clustering sequence analysis tool. It supports de novo and reference based chimera detection, clustering, full-length and prefix dereplication, reverse complementation, masking, all-vs-all pairwise global alignment, exact and global alignment searching, shuffling, subsampling and sorting. It also supports FASTQ file analysis, filtering and conversion.","No description available.","Software for phasing genomic variants using DNA sequencing reads, also called haplotype assembly. It is especially suitable for long reads, but works also well with short reads.","No description available.","No description available.","Winnowmap is a long-read mapping algorithm optimized for mapping ONT and PacBio reads to repetitive reference sequences. Winnowmap development began on top of minimap2 codebase, and since then we have incorporated the following two ideas to improve mapping accuracy within repeats","First fully open-source and collaborative online platform for computational metabolomics. It includes preprocessing, normalization, quality control, statistical analysis of LC/MS, FIA-MS, GC/MS and NMR data.","No description available.","No description available.","Framework for processing and visualization of chromatographically separated and single-spectra mass spectral data.  The packages enables imports from AIA/ANDI NetCDF, mzXML, mzData and mzML files and preprocesses data for high-throughput, untargeted analyte profiling.","No description available.","Detection of differential RNA modifications from direct RNA sequencing of human cell lines.\n\nPython package for detection of differential RNA modifications from direct RNA sequencing.","Matches tandem mass spectra with peptide sequences.","No description available.","YaHS is scaffolding tool using Hi-C data. It relies on a new algorithm for contig joining detection which considers the topological distribution of Hi-C signals aiming to distinguish real interaction signals from mapping noises.","No description available.","No description available."],["<a class=\"homepage\" href=\"https://github.com/tseemann/abricate\" ga-product=\"homepage\" ga-id=\"abricate\">ABRicate<\/a>","<a class=\"homepage\" href=\"https://github.com/bcgsc/abyss/\" ga-product=\"homepage\" ga-id=\"abyss\">ABySS<\/a>",null,"<a class=\"homepage\" href=\"http://ambermd.org/\" ga-product=\"homepage\" ga-id=\"amber\">amber<\/a>","<a class=\"homepage\" href=\"http://bioinformatics.ua.pt/software/anaconda/\" ga-product=\"homepage\" ga-id=\"anaconda\">anaconda<\/a>",null,"<a class=\"homepage\" href=\"https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fannotatemyids%2Fannotatemyids%2F3.12.0%2Bgalaxy1\" ga-product=\"homepage\" ga-id=\"annotatemyids\">annotatemyids<\/a>","<a class=\"homepage\" href=\"http://antismash.secondarymetabolites.org\" ga-product=\"homepage\" ga-id=\"antismash\">antiSMASH<\/a>","<a class=\"homepage\" href=\"https://github.com/tseemann/any2fasta\" ga-product=\"homepage\" ga-id=\"any2fasta\">any2fasta<\/a>","<a class=\"homepage\" href=\"http://apollo.berkeleybop.org\" ga-product=\"homepage\" ga-id=\"apollo\">Apollo<\/a>","<a class=\"homepage\" href=\"http://www.ansikte.se/ARAGORN/\" ga-product=\"homepage\" ga-id=\"aragorn\">aragorn<\/a>","<a class=\"homepage\" href=\"https://github.com/suhrig/arriba/\" ga-product=\"homepage\" ga-id=\"arriba\">Arriba<\/a>","<a class=\"homepage\" href=\"https://github.com/smirarab/ASTRAL\" ga-product=\"homepage\" ga-id=\"astral\">ASTRAL<\/a>","<a class=\"homepage\" href=\"http://bioinf.uni-greifswald.de/augustus\" ga-product=\"homepage\" ga-id=\"augustus\">Augustus<\/a>","<a class=\"homepage\" href=\"http://vina.scripps.edu/\" ga-product=\"homepage\" ga-id=\"autodock_vina\">autodock_vina<\/a>",null,null,null,"<a class=\"homepage\" href=\"https://github.com/oschwengers/bakta\" ga-product=\"homepage\" ga-id=\"bakta\">Bakta<\/a>","<a class=\"homepage\" href=\"https://github.com/pezmaster31/bamtools\" ga-product=\"homepage\" ga-id=\"bamtools\">bamtools<\/a>","<a class=\"homepage\" href=\"http://genome.sph.umich.edu/wiki/BamUtil\" ga-product=\"homepage\" ga-id=\"bamutil_diff\">bamutil_diff<\/a>","<a class=\"homepage\" href=\"https://rrwick.github.io/Bandage/\" ga-product=\"homepage\" ga-id=\"bandage\">bandage<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/tseemann/barrnap\" ga-product=\"homepage\" ga-id=\"barrnap\">barrnap<\/a>","<a class=\"homepage\" href=\"https://jgi.doe.gov/data-and-tools/bbtools/\" ga-product=\"homepage\" ga-id=\"bbmap\">bbmap<\/a>","<a class=\"homepage\" href=\"https://toolshed.g2.bx.psu.edu/view/iuc/gff3_rebase\" ga-product=\"homepage\" ga-id=\"bcbiogff\">bcbiogff<\/a>","<a class=\"homepage\" href=\"http://www.htslib.org/\" ga-product=\"homepage\" ga-id=\"bcftools\">BCFtools<\/a>","<a class=\"homepage\" href=\"https://faculty.washington.edu/browning/beagle/beagle.html\" ga-product=\"homepage\" ga-id=\"beagle\">Beagle<\/a>","<a class=\"homepage\" href=\"https://github.com/beagle-dev/beagle-lib\" ga-product=\"homepage\" ga-id=\"beagle-lib\">beagle-lib<\/a>","<a class=\"homepage\" href=\"http://beast.bio.ed.ac.uk/\" ga-product=\"homepage\" ga-id=\"beast\">BEAST<\/a>","<a class=\"homepage\" href=\"http://www.beast2.org\" ga-product=\"homepage\" ga-id=\"beast2\">BEAST2<\/a>",null,"<a class=\"homepage\" href=\"https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/bed_to_protein_map/bed_to_protein_map/0.2.0\" ga-product=\"homepage\" ga-id=\"bed_to_protein_map\">bed_to_protein_map<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/arq5x/bedtools2\" ga-product=\"homepage\" ga-id=\"bedtools\">BEDTools<\/a>","<a class=\"homepage\" href=\"https://github.com/JesseKerkvliet/Bellerophon\" ga-product=\"homepage\" ga-id=\"bellerophon\">bellerophon<\/a>","<a class=\"homepage\" href=\"https://github.com/tseemann/berokka\" ga-product=\"homepage\" ga-id=\"berokka\">berokka<\/a>",null,null,"<a class=\"homepage\" href=\"http://thegrantlab.org/bio3d/\" ga-product=\"homepage\" ga-id=\"bio3d\">bio3d<\/a>",null,"<a class=\"homepage\" href=\"http://www.sanger.ac.uk/science/tools/biobambam\" ga-product=\"homepage\" ga-id=\"biobambam\">biobambam<\/a>","<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/biomformat.html\" ga-product=\"homepage\" ga-id=\"biom-format\">biom-format<\/a>",null,"<a class=\"homepage\" href=\"http://bioperl.org/\" ga-product=\"homepage\" ga-id=\"bioperl\">bioperl<\/a>","<a class=\"homepage\" href=\"http://biopython.org/\" ga-product=\"homepage\" ga-id=\"biopython\">biopython<\/a>","<a class=\"homepage\" href=\"https://biotransformer.ca\" ga-product=\"homepage\" ga-id=\"biotransformer\">BioTransformer<\/a>","<a class=\"homepage\" href=\"https://blast.ncbi.nlm.nih.gov/Blast.cgi\" ga-product=\"homepage\" ga-id=\"blast\">BLAST<\/a>","<a class=\"homepage\" href=\"https://blast.ncbi.nlm.nih.gov/Blast.cgi\" ga-product=\"homepage\" ga-id=\"blast+\">BLAST+<\/a>","<a class=\"homepage\" href=\"https://blast.ncbi.nlm.nih.gov/Blast.cgi\" ga-product=\"homepage\" ga-id=\"blast2\">BLAST2<\/a>","<a class=\"homepage\" href=\"http://genome.ucsc.edu/cgi-bin/hgBlat?command=start\" ga-product=\"homepage\" ga-id=\"blat\">blat<\/a>","<a class=\"homepage\" href=\"http://hoffmann.bioinf.uni-leipzig.de/LIFE/blockbuster.html\" ga-product=\"homepage\" ga-id=\"Blockbuster\">Blockbuster<\/a>",null,"<a class=\"homepage\" href=\"https://www.boost.org/\" ga-product=\"homepage\" ga-id=\"boost\">Boost<\/a>","<a class=\"homepage\" href=\"http://bowtie-bio.sourceforge.net/index.shtml\" ga-product=\"homepage\" ga-id=\"bowtie\">Bowtie<\/a>","<a class=\"homepage\" href=\"http://bowtie-bio.sourceforge.net/bowtie2/index.shtml\" ga-product=\"homepage\" ga-id=\"bowtie2\">Bowtie2<\/a>",null,"<a class=\"homepage\" href=\"http://barricklab.org/twiki/bin/view/Lab/ToolsBacterialGenomeResequencing\" ga-product=\"homepage\" ga-id=\"breseq\">breseq<\/a>",null,"<a class=\"homepage\" href=\"http://busco.ezlab.org/\" ga-product=\"homepage\" ga-id=\"busco\">BUSCO<\/a>","<a class=\"homepage\" href=\"http://bio-bwa.sourceforge.net\" ga-product=\"homepage\" ga-id=\"bwa\">bwa<\/a>","<a class=\"homepage\" href=\"https://github.com/bwa-mem2/bwa-mem2\" ga-product=\"homepage\" ga-id=\"bwa-mem2\">bwa_mem2<\/a>",null,"<a class=\"homepage\" href=\"https://sourceforge.net/projects/bio-bwa/files/bwakit/\" ga-product=\"homepage\" ga-id=\"bwakit\">bwakit<\/a>","<a class=\"homepage\" href=\"https://github.com/bxlab/bx-python\" ga-product=\"homepage\" ga-id=\"bx-python\">bx-python<\/a>","<a class=\"homepage\" href=\"http://drivergene.rwebox.com/c3/index.html\" ga-product=\"homepage\" ga-id=\"c3s\">c3s<\/a>","<a class=\"homepage\" href=\"https://github.com/ComparativeGenomicsToolkit/cactus\" ga-product=\"homepage\" ga-id=\"cactus\">Cactus<\/a>","<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/CAMERA.html\" ga-product=\"homepage\" ga-id=\"camera\">CAMERA<\/a>","<a class=\"homepage\" href=\"https://github.com/marbl/canu\" ga-product=\"homepage\" ga-id=\"canu\">Canu<\/a>","<a class=\"homepage\" href=\"http://biosrv.cab.unina.it/webcap3/\" ga-product=\"homepage\" ga-id=\"cap3\">cap3<\/a>","<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/Cardinal.html\" ga-product=\"homepage\" ga-id=\"cardinal\">Cardinal<\/a>","<a class=\"homepage\" href=\"https://github.com/dutilh/CAT\" ga-product=\"homepage\" ga-id=\"cat_bins\">CAT<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/weizhongli/cdhit\" ga-product=\"homepage\" ga-id=\"cd-hit\">cd-hit<\/a>","<a class=\"homepage\" href=\"https://cellprofiler.org/\" ga-product=\"homepage\" ga-id=\"CellProfiler\">CellProfiler<\/a>","<a class=\"homepage\" href=\"https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger\" ga-product=\"homepage\" ga-id=\"cellranger\">cellranger<\/a>","<a class=\"homepage\" href=\"https://github.com/chanzuckerberg/cellxgene\" ga-product=\"homepage\" ga-id=\"cellxgene\">cellxgene<\/a>","<a class=\"homepage\" href=\"https://ecogenomics.github.io/CheckM\" ga-product=\"homepage\" ga-id=\"checkm\">checkm<\/a>","<a class=\"homepage\" href=\"https://www.ebi.ac.uk/chembldb/\" ga-product=\"homepage\" ga-id=\"chembl\">ChEMBL<\/a>","<a class=\"homepage\" href=\"http://www.chemfp.com/\" ga-product=\"homepage\" ga-id=\"chemfp\">chemfp<\/a>",null,"<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/ChIPseeker.html\" ga-product=\"homepage\" ga-id=\"chipseeker\">chipseeker<\/a>","<a class=\"homepage\" href=\"https://github.com/estebanpw/chromeister\" ga-product=\"homepage\" ga-id=\"chromeister\">chromeister<\/a>","<a class=\"homepage\" href=\"https://github.com/YangLab/CIRCexplorer\" ga-product=\"homepage\" ga-id=\"circexplorer\">circexplorer<\/a>","<a class=\"homepage\" href=\"http://circos.ca/\" ga-product=\"homepage\" ga-id=\"circos\">circos<\/a>",null,null,"<a class=\"homepage\" href=\"http://github.com/gamcil/clinker\" ga-product=\"homepage\" ga-id=\"clinker\">clinker<\/a>","<a class=\"homepage\" href=\"http://www.clustal.org/omega/\" ga-product=\"homepage\" ga-id=\"clustalo\">Clustal Omega<\/a>","<a class=\"homepage\" href=\"http://www.clustal.org/download/1.X/ftp-igbmc.u-strasbg.fr/pub/\" ga-product=\"homepage\" ga-id=\"clustalw\">clustalw<\/a>","<a class=\"homepage\" href=\"http://colabfold.mmseqs.com\" ga-product=\"homepage\" ga-id=\"colabfold_batch\">ColabFold<\/a>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<a class=\"homepage\" href=\"http://lilab.research.bcm.edu/cpat/\" ga-product=\"homepage\" ga-id=\"cpat\">CPAT (Coding-Potential Assessment Tool)<\/a>","<a class=\"homepage\" href=\"https://github.com/NordicESMhub/ctsm/tree/fates_emerald_api\" ga-product=\"homepage\" ga-id=\"ctsm_fates\">ctsm_fates<\/a>","<a class=\"homepage\" href=\"https://developer.nvidia.com/cuda-toolkit\" ga-product=\"homepage\" ga-id=\"cuda\">cuda<\/a>","<a class=\"homepage\" href=\"https://developer.nvidia.com/cudnn\" ga-product=\"homepage\" ga-id=\"cudnn\">cudnn<\/a>","<a class=\"homepage\" href=\"http://cole-trapnell-lab.github.io/cufflinks\" ga-product=\"homepage\" ga-id=\"cufflinks\">Cufflinks<\/a>","<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/cummeRbund.html\" ga-product=\"homepage\" ga-id=\"cummeRbund\">cummeRbund<\/a>","<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/customProDB.html\" ga-product=\"homepage\" ga-id=\"customprodb\">customProDB<\/a>",null,"<a class=\"homepage\" href=\"https://pypi.python.org/pypi/cutadapt\" ga-product=\"homepage\" ga-id=\"cutadapt\">cutadapt<\/a>","<a class=\"homepage\" href=\"https://github.com/tjiangHIT/cuteSV\" ga-product=\"homepage\" ga-id=\"cutesv\">cuteSV<\/a>","<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/dada2.html\" ga-product=\"homepage\" ga-id=\"dada2\">DADA2<\/a>",null,null,null,null,null,"<a class=\"homepage\" href=\"https://deeptools.readthedocs.io/en/latest/\" ga-product=\"homepage\" ga-id=\"deeptools\">DeepTools<\/a>","<a class=\"homepage\" href=\"https://dendropy.org/\" ga-product=\"homepage\" ga-id=\"dendropy\">dendropy<\/a>","<a class=\"homepage\" href=\"http://bioconductor.org/packages/DESeq2/\" ga-product=\"homepage\" ga-id=\"deseq2\">deseq2<\/a>","<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/DEXSeq.html\" ga-product=\"homepage\" ga-id=\"DEXSeq\">DEXSeq<\/a>","<a class=\"homepage\" href=\"https://github.com/vdemichev/diann\" ga-product=\"homepage\" ga-id=\"diann\">DIA-NN<\/a>","<a class=\"homepage\" href=\"https://sourceforge.net/projects/diaumpire/\" ga-product=\"homepage\" ga-id=\"diaumpire\">DIA-Umpire<\/a>","<a class=\"homepage\" href=\"https://github.com/bbuchfink/diamond\" ga-product=\"homepage\" ga-id=\"diamond\">diamond<\/a>","<a class=\"homepage\" href=\"https://github.com/Roestlab/dia-pasef\" ga-product=\"homepage\" ga-id=\"diapysef\">diapysef<\/a>","<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/DiffBind.html\" ga-product=\"homepage\" ga-id=\"diffbind\">diffbind<\/a>",null,null,"<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/DropletUtils.html\" ga-product=\"homepage\" ga-id=\"dropletutils\">dropletutils<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/easybuilders/easybuild\" ga-product=\"homepage\" ga-id=\"easybuild\">easybuild<\/a>",null,"<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/edgeR.html\" ga-product=\"homepage\" ga-id=\"edger\">edger<\/a>","<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/edgeR.html\" ga-product=\"homepage\" ga-id=\"edger-repenrich\">edger-repenrich<\/a>","<a class=\"homepage\" href=\"http://beta-eggnogdb.embl.de/#/app/emapper\" ga-product=\"homepage\" ga-id=\"eggnog-mapper\">eggNOG-mapper<\/a>","<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/EGSEA.html\" ga-product=\"homepage\" ga-id=\"egsea\">egsea<\/a>","<a class=\"homepage\" href=\"http://elastix.isi.uu.nl\" ga-product=\"homepage\" ga-id=\"elastix\">elastix<\/a>","<a class=\"homepage\" href=\"http://emboss.bioinformatics.nl/\" ga-product=\"homepage\" ga-id=\"emboss\">EMBOSS (European Molecular Biology Open Software Suite)<\/a>",null,"<a class=\"homepage\" href=\"https://bitbucket.org/searleb/encyclopedia/\" ga-product=\"homepage\" ga-id=\"encyclopedia\">EncyclopeDIA<\/a>",null,null,"<a class=\"homepage\" href=\"http://EuPathDB.org\" ga-product=\"homepage\" ga-id=\"eupathdb\">eupathdb<\/a>","<a class=\"homepage\" href=\"https://github.com/aberer/exabayes\" ga-product=\"homepage\" ga-id=\"exabayes\">ExaBayes<\/a>","<a class=\"homepage\" href=\"https://github.com/stamatak/ExaML\" ga-product=\"homepage\" ga-id=\"examl\">ExaML<\/a>","<a class=\"homepage\" href=\"http://www.ebi.ac.uk/%7Eguy/exonerate\" ga-product=\"homepage\" ga-id=\"exonerate\">Exonerate<\/a>",null,"<a class=\"homepage\" href=\"https://pachterlab.github.io/eXpress/index.html\" ga-product=\"homepage\" ga-id=\"eXpress\">eXpress<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/pratas/falcon\" ga-product=\"homepage\" ga-id=\"pb-assembly\">Falcon (pb-assembly)<\/a>",null,null,null,null,null,null,null,"<a class=\"homepage\" href=\"https://github.com/ParBLiSS/FastANI\" ga-product=\"homepage\" ga-id=\"fastani\">fastani<\/a>","<a class=\"homepage\" href=\"https://github.com/fstrozzi/Fastool\" ga-product=\"homepage\" ga-id=\"fastool\">fastool<\/a>","<a class=\"homepage\" href=\"https://github.com/OpenGene/fastp\" ga-product=\"homepage\" ga-id=\"fastp\">fastp<\/a>","<a class=\"homepage\" href=\"http://www.bioinformatics.babraham.ac.uk/projects/fastqc/\" ga-product=\"homepage\" ga-id=\"fastqc\">FASTQC<\/a>",null,"<a class=\"homepage\" href=\"http://www.microbesonline.org/fasttree/\" ga-product=\"homepage\" ga-id=\"fasttree\">FastTree<\/a>","<a class=\"homepage\" href=\"http://hannonlab.cshl.edu/fastx_toolkit/\" ga-product=\"homepage\" ga-id=\"fastx\">FASTX-Toolkit<\/a>","<a class=\"homepage\" href=\"http://hannonlab.cshl.edu/fastx_toolkit/\" ga-product=\"homepage\" ga-id=\"fastx_toolkit\">FASTX-Toolkit<\/a>",null,"<a class=\"homepage\" href=\"http://nhjy.hzau.edu.cn/kech/swxxx/jakj/dianzi/Bioinf6/GeneFinding/GeneFinding2.htm\" ga-product=\"homepage\" ga-id=\"fgenesh\">Fgenesh++<\/a>","<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/fgsea.html\" ga-product=\"homepage\" ga-id=\"fgsea\">fgsea<\/a>",null,null,null,"<a class=\"homepage\" href=\"https://github.com/rrwick/Filtlong\" ga-product=\"homepage\" ga-id=\"filtlong\">filtlong<\/a>","<a class=\"homepage\" href=\"https://github.com/smith-chem-wisc/FlashLFQ\" ga-product=\"homepage\" ga-id=\"flashlfq\">flashLFQ<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/fenderglass/Flye\" ga-product=\"homepage\" ga-id=\"flye\">Flye<\/a>",null,"<a class=\"homepage\" href=\"https://foldseek.com\" ga-product=\"homepage\" ga-id=\"foldseek\">Foldseek<\/a>","<a class=\"homepage\" href=\"https://mobyle.rpbs.univ-paris-diderot.fr/cgi-bin/portal.py#forms::fpocket\" ga-product=\"homepage\" ga-id=\"fpocket\">fpocket<\/a>","<a class=\"homepage\" href=\"https://sourceforge.net/projects/fraggenescan/\" ga-product=\"homepage\" ga-id=\"fraggenescan\">FragGeneScan<\/a>","<a class=\"homepage\" href=\"https://github.com/ekg/freebayes\" ga-product=\"homepage\" ga-id=\"freebayes\">freebayes<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/nextgenusfs/funannotate\" ga-product=\"homepage\" ga-id=\"funannotate\">funannotate<\/a>","<a class=\"homepage\" href=\"http://www.plantgdb.org/GAEVAL/docs/index.html\" ga-product=\"homepage\" ga-id=\"gaeval\">gaeval<\/a>",null,null,null,null,null,null,null,"<a class=\"homepage\" href=\"https://software.broadinstitute.org/gatk/\" ga-product=\"homepage\" ga-id=\"gatk\">GATK<\/a>","<a class=\"homepage\" href=\"http://www.embl-heidelberg.de/~castresa/Gblocks/Gblocks.html\" ga-product=\"homepage\" ga-id=\"gblock\">gblocks<\/a>","<a class=\"homepage\" href=\"http://cnsgenomics.com/software/gcta/\" ga-product=\"homepage\" ga-id=\"gcta\">gcta<\/a>","<a class=\"homepage\" href=\"http://bitlab-es.com/gecko/\" ga-product=\"homepage\" ga-id=\"gecko\">gecko<\/a>","<a class=\"homepage\" href=\"http://gemini.readthedocs.io/en/latest/index.html\" ga-product=\"homepage\" ga-id=\"gemini\">gemini<\/a>","<a class=\"homepage\" href=\"http://www.jstacs.de/index.php/GeMoMa\" ga-product=\"homepage\" ga-id=\"gemoma\">gemoma<\/a>",null,null,null,null,"<a class=\"homepage\" href=\"https://github.com/tbenavi1/genomescope2.0\" ga-product=\"homepage\" ga-id=\"genomescope\">GenomeScope<\/a>","<a class=\"homepage\" href=\"http://genometools.org/\" ga-product=\"homepage\" ga-id=\"genometools\">GenomeTools<\/a>",null,null,"<a class=\"homepage\" href=\"https://github.com/Kinggerm/GetOrganelle\" ga-product=\"homepage\" ga-id=\"getorganelle\">getorganelle<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/vgl-hub/gfastats\" ga-product=\"homepage\" ga-id=\"gfastats\">gfastats<\/a>",null,null,"<a class=\"homepage\" href=\"https://ccb.jhu.edu/software/stringtie/gff.shtml\" ga-product=\"homepage\" ga-id=\"gffcompare\">gffcompare<\/a>","<a class=\"homepage\" href=\"https://ccb.jhu.edu/software/stringtie/gff.shtml\" ga-product=\"homepage\" ga-id=\"gffread\">gffread<\/a>","<a class=\"homepage\" href=\"http://ggplot2.org/\" ga-product=\"homepage\" ga-id=\"ggplot2\">ggplot2<\/a>",null,null,"<a class=\"homepage\" href=\"https://www.gnu.org/software/parallel/\" ga-product=\"homepage\" ga-id=\"parallel\">GNU parallel<\/a>","<a class=\"homepage\" href=\"https://www.gnu.org/software/gsl/\" ga-product=\"homepage\" ga-id=\"gsl\">GNU scientific library<\/a>","<a class=\"homepage\" href=\"https://github.com/DanFaria/GOEnrichment\" ga-product=\"homepage\" ga-id=\"goenrichment\">goenrichment<\/a>","<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/goseq.html\" ga-product=\"homepage\" ga-id=\"goseq\">goseq<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/biobakery/graphlan\" ga-product=\"homepage\" ga-id=\"graphlan\">GraPhlAn<\/a>","<a class=\"homepage\" href=\"http://www.gromacs.org/\" ga-product=\"homepage\" ga-id=\"gromacs\">gromacs<\/a>","<a class=\"homepage\" href=\"https://github.com/ecogenomics/gtdbtk\" ga-product=\"homepage\" ga-id=\"gtdb-tk\">GTDB-TK<\/a>",null,null,"<a class=\"homepage\" href=\"https://github.com/sanger-pathogens/gubbins\" ga-product=\"homepage\" ga-id=\"gubbins\">gubbins<\/a>","<a class=\"homepage\" href=\"https://github.com/vibansal/HapCUT2\" ga-product=\"homepage\" ga-id=\"hapcut2\">hapcut2<\/a>",null,"<a class=\"homepage\" href=\"https://software.cwi.nl/software/heinz\" ga-product=\"homepage\" ga-id=\"heinz\">heinz<\/a>","<a class=\"homepage\" href=\"https://github.com/galaxyproject/galaxy/tree/dev/tools\" ga-product=\"homepage\" ga-id=\"hgv_david\">hgv_david<\/a>","<a class=\"homepage\" href=\"https://github.com/galaxyproject/galaxy/tree/dev/tools\" ga-product=\"homepage\" ga-id=\"hgv_ldtools\">hgv_ldtools<\/a>","<a class=\"homepage\" href=\"https://github.com/galaxyproject/galaxy/tree/dev/tools\" ga-product=\"homepage\" ga-id=\"hgv_linkToGProfile\">hgv_linkToGProfile<\/a>","<a class=\"homepage\" href=\"https://hicexplorer.usegalaxy.eu\" ga-product=\"homepage\" ga-id=\"hicexplorer\">hicexplorer<\/a>","<a class=\"homepage\" href=\"https://github.com/sheinasim/HiFiAdapterFilt\" ga-product=\"homepage\" ga-id=\"hifiadapterfilt\">hifiadapterfilt<\/a>","<a class=\"homepage\" href=\"https://github.com/chhylp123/hifiasm\" ga-product=\"homepage\" ga-id=\"hifiasm\">hifiasm<\/a>",null,"<a class=\"homepage\" href=\"https://ccb.jhu.edu/software/hisat2/index.shtml\" ga-product=\"homepage\" ga-id=\"hisat2\">HISAT2<\/a>",null,null,"<a class=\"homepage\" href=\"http://hmmer.org/\" ga-product=\"homepage\" ga-id=\"hmmer\">HMMER<\/a>","<a class=\"homepage\" href=\"https://github.com/horovod/horovod\" ga-product=\"homepage\" ga-id=\"horovod\">horovod<\/a>","<a class=\"homepage\" href=\"http://htseq.readthedocs.io/en/release_0.9.1/\" ga-product=\"homepage\" ga-id=\"htseq\">HTSeq<\/a>","<a class=\"homepage\" href=\"http://www.htslib.org/\" ga-product=\"homepage\" ga-id=\"htslib\">HTSlib<\/a>","<a class=\"homepage\" href=\"https://huttenhower.sph.harvard.edu/humann\" ga-product=\"homepage\" ga-id=\"humann\">HUMAnN<\/a>","<a class=\"homepage\" href=\"http://huttenhower.sph.harvard.edu/humann2\" ga-product=\"homepage\" ga-id=\"humann2\">HUMAnN2<\/a>","<a class=\"homepage\" href=\"http://hyphy.org\" ga-product=\"homepage\" ga-id=\"HyPhy\">HyPhy<\/a>","<a class=\"homepage\" href=\"https://www.l3harrisgeospatial.com/Software-Technology/IDL\" ga-product=\"homepage\" ga-id=\"idl\">IDL<\/a>","<a class=\"homepage\" href=\"https://imagemagick.org/script/index.php\" ga-product=\"homepage\" ga-id=\"imagemagick\">ImageMagick<\/a>","<a class=\"homepage\" href=\"https://github.com/PacificBiosciences/pbipa\" ga-product=\"homepage\" ga-id=\"pbipa\">Improved Phased Assembler (pbipa)<\/a>","<a class=\"homepage\" href=\"http://eddylab.org/infernal/\" ga-product=\"homepage\" ga-id=\"infernal\">Infernal<\/a>",null,"<a class=\"homepage\" href=\"http://www.broadinstitute.org/igv/\" ga-product=\"homepage\" ga-id=\"igv\">Integrated Genomics Viewer<\/a>","<a class=\"homepage\" href=\"https://github.com/gem-pasteur/Integron_Finder\" ga-product=\"homepage\" ga-id=\"integron_finder\">integron_finder<\/a>",null,null,"<a class=\"homepage\" href=\"http://www.ebi.ac.uk/Tools/pfa/iprscan/\" ga-product=\"homepage\" ga-id=\"interproscan\">InterProScan<\/a>",null,null,null,"<a class=\"homepage\" href=\"https://github.com/dereneaton/ipyrad/\" ga-product=\"homepage\" ga-id=\"ipyrad\">ipyrad<\/a>","<a class=\"homepage\" href=\"http://www.cibiv.at/software/iqtree/\" ga-product=\"homepage\" ga-id=\"iq-tree\">IQ-TREE<\/a>","<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/iSEE.html\" ga-product=\"homepage\" ga-id=\"isee\">iSEE<\/a>","<a class=\"homepage\" href=\"https://github.com/xiezhq/ISEScan\" ga-product=\"homepage\" ga-id=\"isescan\">ISEScan<\/a>","<a class=\"homepage\" href=\"http://bioconductor.org/packages/IsoformSwitchAnalyzeR/\" ga-product=\"homepage\" ga-id=\"isoformswitchanalyzer\">IsoformSwitchAnalyzeR<\/a>","<a class=\"homepage\" href=\"https://github.com/CGR-UNIMORE/iVar\" ga-product=\"homepage\" ga-id=\"ivar\">ivar<\/a>","<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/IWTomics.html\" ga-product=\"homepage\" ga-id=\"iwtomics\">IWTomics<\/a>","<a class=\"homepage\" href=\"https://mcmc-jags.sourceforge.io/\" ga-product=\"homepage\" ga-id=\"jags\">jags<\/a>","<a class=\"homepage\" href=\"https://github.com/mkirsche/Jasmine\" ga-product=\"homepage\" ga-id=\"jasminesv\">jasminesv<\/a>",null,null,"<a class=\"homepage\" href=\"http://gmod.org/wiki/JBrowse\" ga-product=\"homepage\" ga-id=\"jbrowse\">JBrowse<\/a>","<a class=\"homepage\" href=\"https://github.com/tanghaibao/jcvi\" ga-product=\"homepage\" ga-id=\"jcvi\">JCVI<\/a>","<a class=\"homepage\" href=\"http://www.genome.umd.edu/jellyfish.html\" ga-product=\"homepage\" ga-id=\"jellyfish\">jellyfish<\/a>",null,"<a class=\"homepage\" href=\"https://stedolan.github.io/jq/\" ga-product=\"homepage\" ga-id=\"jq\">jq<\/a>",null,"<a class=\"homepage\" href=\"https://pachterlab.github.io/kallisto/about.html\" ga-product=\"homepage\" ga-id=\"kallisto\">Kallisto<\/a>","<a class=\"homepage\" href=\"http://www.earlham.ac.uk/kat-tools\" ga-product=\"homepage\" ga-id=\"kat\">KAT<\/a>","<a class=\"homepage\" href=\"https://github.com/ucscGenomeBrowser/kent\" ga-product=\"homepage\" ga-id=\"kentutils\">kentutils<\/a>","<a class=\"homepage\" href=\"http://khmer.readthedocs.org/en/v2.0/\" ga-product=\"homepage\" ga-id=\"khmer\">khmer<\/a>","<a class=\"homepage\" href=\"http://sun.aei.polsl.pl/REFRESH/index.php?page=projects&project=kmc&subpage=about\" ga-product=\"homepage\" ga-id=\"kmc\">KMC<\/a>",null,"<a class=\"homepage\" href=\"https://ccb.jhu.edu/software/kraken/\" ga-product=\"homepage\" ga-id=\"kraken\">kraken<\/a>","<a class=\"homepage\" href=\"https://ccb.jhu.edu/software/kraken2/\" ga-product=\"homepage\" ga-id=\"kraken2\">kraken2<\/a>","<a class=\"homepage\" href=\"https://github.com/marbl/Krona/wiki\" ga-product=\"homepage\" ga-id=\"krona\">krona<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/mpi2/LAMA\" ga-product=\"homepage\" ga-id=\"lama\">LAMA (Lightweight Analysis of Morphological Abnormalities)<\/a>","<a class=\"homepage\" href=\"http://www.bx.psu.edu/~rsharris/lastz/\" ga-product=\"homepage\" ga-id=\"lastz\">lastz<\/a>",null,"<a class=\"homepage\" href=\"https://bioinformatics.erc.monash.edu/apps/LFQ-Analyst\" ga-product=\"homepage\" ga-id=\"LFQ-Analyst\">LFQ-Analyst<\/a>",null,"<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/limma.html\" ga-product=\"homepage\" ga-id=\"limma\">limma<\/a>","<a class=\"homepage\" href=\"https://github.com/bcgsc/LINKS\" ga-product=\"homepage\" ga-id=\"links\">LINKS<\/a>","<a class=\"homepage\" href=\"https://csb5.github.io/lofreq/\" ga-product=\"homepage\" ga-id=\"lofreq\">lofreq<\/a>","<a class=\"homepage\" href=\"https://github.com/pjedge/longshot\" ga-product=\"homepage\" ga-id=\"longshot\">longshot<\/a>","<a class=\"homepage\" href=\"http://lpsolve.sourceforge.net/5.5/\" ga-product=\"homepage\" ga-id=\"lpsolve\">lpsolve<\/a>","<a class=\"homepage\" href=\"https://github.com/oushujun/LTR_retriever\" ga-product=\"homepage\" ga-id=\"ltr_retriever\">LTR_retriever<\/a>","<a class=\"homepage\" href=\"http://liulab.dfci.harvard.edu/MACS/\" ga-product=\"homepage\" ga-id=\"macs\">macs<\/a>","<a class=\"homepage\" href=\"http://liulab.dfci.harvard.edu/MACS/\" ga-product=\"homepage\" ga-id=\"macs2\">macs2<\/a>",null,null,null,null,null,null,null,null,null,null,null,"<a class=\"homepage\" href=\"http://mafft.cbrc.jp/alignment/server/index.html\" ga-product=\"homepage\" ga-id=\"mafft\">mafft<\/a>","<a class=\"homepage\" href=\"https://bitbucket.org/johanneskoester/mageck\" ga-product=\"homepage\" ga-id=\"MAGeCK\">MAGeCK<\/a>","<a class=\"homepage\" href=\"http://www.yandell-lab.org/software/maker.html\" ga-product=\"homepage\" ga-id=\"maker\">MAKER<\/a>","<a class=\"homepage\" href=\"https://cran.r-project.org/package=MALDIquant\" ga-product=\"homepage\" ga-id=\"maldi_quant\">MALDIquant<\/a>","<a class=\"homepage\" href=\"https://github.com/marbl/mash\" ga-product=\"homepage\" ga-id=\"mash\">mash<\/a>",null,"<a class=\"homepage\" href=\"http://www.genome.umd.edu/masurca.html\" ga-product=\"homepage\" ga-id=\"masurca\">MaSuRCA<\/a>","<a class=\"homepage\" href=\"https://github.com/matchms/matchms\" ga-product=\"homepage\" ga-id=\"matchms\">Matchms<\/a>","<a class=\"homepage\" href=\"https://matplotlib.org/\" ga-product=\"homepage\" ga-id=\"matplotlib\">matplotlib<\/a>","<a class=\"homepage\" href=\"http://downloads.jbei.org/data/microbial_communities/MaxBin/MaxBin.html\" ga-product=\"homepage\" ga-id=\"maxbin\">MaxBin2<\/a>","<a class=\"homepage\" href=\"http://www.maxquant.org/\" ga-product=\"homepage\" ga-id=\"maxquant\">MaxQuant<\/a>","<a class=\"homepage\" href=\"http://www.maxquant.org/\" ga-product=\"homepage\" ga-id=\"biocontainers-maxquant\">MaxQuant (biocontainers)<\/a>","<a class=\"homepage\" href=\"http://micans.org/mcl/\" ga-product=\"homepage\" ga-id=\"mcl\">mcl<\/a>","<a class=\"homepage\" href=\"https://www.mdanalysis.org\" ga-product=\"homepage\" ga-id=\"mdanalysis\">mdanalysis<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/nanoporetech/medaka\" ga-product=\"homepage\" ga-id=\"medaka\">Medaka<\/a>","<a class=\"homepage\" href=\"https://github.com/voutcn/megahit\" ga-product=\"homepage\" ga-id=\"megahit\">megahit<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/marbl/merqury\" ga-product=\"homepage\" ga-id=\"merqury\">Merqury<\/a>","<a class=\"homepage\" href=\"https://github.com/marbl/meryl\" ga-product=\"homepage\" ga-id=\"meryl\">meryl<\/a>","<a class=\"homepage\" href=\"https://bitbucket.org/berkeleylab/metabat\" ga-product=\"homepage\" ga-id=\"metabat\">metabat<\/a>",null,"<a class=\"homepage\" href=\"http://203.101.224.202/galaxy/\" ga-product=\"homepage\" ga-id=\"MetaDEGalaxy\">MetaDEGalaxy<\/a>","<a class=\"homepage\" href=\"https://metaeuk.soedinglab.org/\" ga-product=\"homepage\" ga-id=\"metaeuk\">metaeuk<\/a>","<a class=\"homepage\" href=\"http://huttenhower.sph.harvard.edu/metaphlan\" ga-product=\"homepage\" ga-id=\"metaphlan\">MetaPhlAn<\/a>","<a class=\"homepage\" href=\"https://github.com/galaxyproteomics/metaquantome\" ga-product=\"homepage\" ga-id=\"metaQuantome\">metaQuantome<\/a>","<a class=\"homepage\" href=\"http://cab.spbu.ru/software/spades/\" ga-product=\"homepage\" ga-id=\"metaspades\">metaspades<\/a>","<a class=\"homepage\" href=\"https://github.com/bxlab/metaWRAP\" ga-product=\"homepage\" ga-id=\"metawrap\">metawrap<\/a>","<a class=\"homepage\" href=\"https://github.com/dpryan79/MethylDackel\" ga-product=\"homepage\" ga-id=\"MethylDackel\">MethylDackel<\/a>",null,"<a class=\"homepage\" href=\"https://peterbeerli.com/migrate-html5/index.html\" ga-product=\"homepage\" ga-id=\"MIGRATE\">MIGRATE<\/a>","<a class=\"homepage\" href=\"https://mimodd.readthedocs.io/en/latest/nacreousmap.html\" ga-product=\"homepage\" ga-id=\"mimodd\">MiModD<\/a>","<a class=\"homepage\" href=\"http://minia.genouest.org/\" ga-product=\"homepage\" ga-id=\"minia\">minia<\/a>","<a class=\"homepage\" href=\"https://github.com/lh3/miniasm\" ga-product=\"homepage\" ga-id=\"miniasm\">miniasm<\/a>","<a class=\"homepage\" href=\"https://docs.conda.io/en/latest/miniconda.html\" ga-product=\"homepage\" ga-id=\"miniconda\">miniconda<\/a>","<a class=\"homepage\" href=\"https://github.com/lh3/minimap2\" ga-product=\"homepage\" ga-id=\"minimap2\">minimap2<\/a>","<a class=\"homepage\" href=\"https://github.com/lh3/miniprot\" ga-product=\"homepage\" ga-id=\"miniprot\">miniprot<\/a>","<a class=\"homepage\" href=\"https://github.com/ARTbio/tools-artbio/tree/master/tools/mircounts\" ga-product=\"homepage\" ga-id=\"mircounts\">mircounts<\/a>","<a class=\"homepage\" href=\"https://github.com/rajewsky-lab/mirdeep2\" ga-product=\"homepage\" ga-id=\"mirdeep2\">mirdeep2<\/a>",null,"<a class=\"homepage\" href=\"http://mitos.bioinf.uni-leipzig.de/index.py\" ga-product=\"homepage\" ga-id=\"mitos2\">mitos<\/a>","<a class=\"homepage\" href=\"http://cge.cbs.dtu.dk/services/MLST/\" ga-product=\"homepage\" ga-id=\"mlst\">MLST<\/a>","<a class=\"homepage\" href=\"https://mmseqs.com\" ga-product=\"homepage\" ga-id=\"mmseqs2\">MMseqs2<\/a>","<a class=\"homepage\" href=\"https://github.com/phac-nml/mob-suite\" ga-product=\"homepage\" ga-id=\"mob-suite\">MOB-suite<\/a>",null,null,null,"<a class=\"homepage\" href=\"https://github.com/compomics/moFF\" ga-product=\"homepage\" ga-id=\"moff\">moFF<\/a>",null,"<a class=\"homepage\" href=\"https://cwenger.github.io/Morpheus/\" ga-product=\"homepage\" ga-id=\"morpheus\">Morpheus<\/a>","<a class=\"homepage\" href=\"http://mothur.org/\" ga-product=\"homepage\" ga-id=\"mothur\">Mothur<\/a>","<a class=\"homepage\" href=\"http://www.mousemine.org/mousemine/begin.do\" ga-product=\"homepage\" ga-id=\"mousemine\">mousemine<\/a>","<a class=\"homepage\" href=\"http://nbisweden.github.io/MrBayes/\" ga-product=\"homepage\" ga-id=\"mrbayes\">MrBayes<\/a>","<a class=\"homepage\" href=\"http://proteowizard.sourceforge.net/tools.shtml\" ga-product=\"homepage\" ga-id=\"msconvert\">msconvert<\/a>","<a class=\"homepage\" href=\"https://github.com/RECETOX/MSMetaEnhancer\" ga-product=\"homepage\" ga-id=\"msmetaenhancer\">MSMetaEnhancer<\/a>","<a class=\"homepage\" href=\"http://www.msstats.org/\" ga-product=\"homepage\" ga-id=\"msstats\">MSstats<\/a>","<a class=\"homepage\" href=\"http://multiqc.info/\" ga-product=\"homepage\" ga-id=\"multiqc\">MultiQC<\/a>","<a class=\"homepage\" href=\"http://mummer.sourceforge.net/\" ga-product=\"homepage\" ga-id=\"mummer\">MUMmer<\/a>",null,null,"<a class=\"homepage\" href=\"https://www.nag.com/\" ga-product=\"homepage\" ga-id=\"nag\">nag<\/a>","<a class=\"homepage\" href=\"https://github.com/tleonardi/nanocompore\" ga-product=\"homepage\" ga-id=\"nanocompore\">Nanocompore<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/wdecoster/NanoPlot\" ga-product=\"homepage\" ga-id=\"nanoplot\">nanoplot<\/a>","<a class=\"homepage\" href=\"https://github.com/jts/nanopolish\" ga-product=\"homepage\" ga-id=\"nanopolish\">Nanopolish<\/a>",null,null,"<a class=\"homepage\" href=\"https://www.ncbi.nlm.nih.gov/datasets/\" ga-product=\"homepage\" ga-id=\"ncbi-datasets-cli\">NCBI Datasets<\/a>",null,"<a class=\"homepage\" href=\"https://gitlab.com/remikz/nccmp\" ga-product=\"homepage\" ga-id=\"nccmp\">nccmp<\/a>","<a class=\"homepage\" href=\"https://www.ncl.ucar.edu/index.shtml\" ga-product=\"homepage\" ga-id=\"ncl\">NCL<\/a>","<a class=\"homepage\" href=\"http://nco.sourceforge.net/\" ga-product=\"homepage\" ga-id=\"nco\">NCO<\/a>","<a class=\"homepage\" href=\"https://cirrus.ucsd.edu/ncview/\" ga-product=\"homepage\" ga-id=\"ncview\">Ncview<\/a>","<a class=\"homepage\" href=\"https://www.unidata.ucar.edu/software/netcdf/\" ga-product=\"homepage\" ga-id=\"netcdf\">NetCDF<\/a>","<a class=\"homepage\" href=\"http://cegg.unige.ch/newick_utils\" ga-product=\"homepage\" ga-id=\"newick_utils\">newick_utils<\/a>",null,"<a class=\"homepage\" href=\"https://www.nextflow.io/\" ga-product=\"homepage\" ga-id=\"nextflow\">nextflow<\/a>","<a class=\"homepage\" href=\"http://ngsutils.org\" ga-product=\"homepage\" ga-id=\"ngsutils\">ngsutils<\/a>","<a class=\"homepage\" href=\"https://github.com/TravisWheelerLab/NINJA\" ga-product=\"homepage\" ga-id=\"ninja\">ninja<\/a>","<a class=\"homepage\" href=\"https://github.com/ndierckx/NOVOPlasty\" ga-product=\"homepage\" ga-id=\"NOVOPlasty\">NOVOPlasty<\/a>",null,null,"<a class=\"homepage\" href=\"https://git.metabarcoding.org/obitools/obitools/wikis/home\" ga-product=\"homepage\" ga-id=\"obitools\">OBITools<\/a>",null,"<a class=\"homepage\" href=\"http://www.openms.de\" ga-product=\"homepage\" ga-id=\"openms\">OpenMS<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/ColineRoyaux/PAMPA-Galaxy\" ga-product=\"homepage\" ga-id=\"pampa\">PAMPA-Galaxy<\/a>","<a class=\"homepage\" href=\"https://github.com/tkzeng/Pangolin\" ga-product=\"homepage\" ga-id=\"pangolin\">Pangolin<\/a>",null,"<a class=\"homepage\" href=\"https://www.paraview.org/\" ga-product=\"homepage\" ga-id=\"paraview\">ParaView<\/a>",null,"<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/pathview.html\" ga-product=\"homepage\" ga-id=\"pathview\">Pathview<\/a>","<a class=\"homepage\" href=\"https://github.com/icebert/pblat\" ga-product=\"homepage\" ga-id=\"pblat\">pblat<\/a>","<a class=\"homepage\" href=\"https://github.com/eblancoga/seqcode\" ga-product=\"homepage\" ga-id=\"pe_histogram\">pe_histogram<\/a>","<a class=\"homepage\" href=\"http://sco.h-its.org/exelixis/web/software/pear/\" ga-product=\"homepage\" ga-id=\"pear\">PEAR<\/a>",null,"<a class=\"homepage\" href=\"http://compomics.github.io/projects/peptide-shaker.html\" ga-product=\"homepage\" ga-id=\"peptideshaker\">Peptide Shaker<\/a>",null,"<a class=\"homepage\" href=\"http://noble.gs.washington.edu/proj/percolator/\" ga-product=\"homepage\" ga-id=\"percolator\">Percolator<\/a>",null,"<a class=\"homepage\" href=\"http://ftp.ebi.ac.uk/pub/databases/Pfam/Tools/\" ga-product=\"homepage\" ga-id=\"pfamscan\">PfamScan<\/a>",null,"<a class=\"homepage\" href=\"https://www.php.net/\" ga-product=\"homepage\" ga-id=\"php\">PHP<\/a>","<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/phyloseq.html\" ga-product=\"homepage\" ga-id=\"phyloseq\">Phyloseq<\/a>","<a class=\"homepage\" href=\"http://www.atgc-montpellier.fr/phyml/\" ga-product=\"homepage\" ga-id=\"phyml\">PhyML<\/a>","<a class=\"homepage\" href=\"https://github.com/broadinstitute/picard\" ga-product=\"homepage\" ga-id=\"picard\">PICARD<\/a>","<a class=\"homepage\" href=\"http://picrust.github.io/picrust/\" ga-product=\"homepage\" ga-id=\"picrust\">PICRUSt<\/a>","<a class=\"homepage\" href=\"https://github.com/picrust/picrust2\" ga-product=\"homepage\" ga-id=\"picrust2\">PICRUSt 2<\/a>",null,null,"<a class=\"homepage\" href=\"http://www.broadinstitute.org/software/pilon/\" ga-product=\"homepage\" ga-id=\"pilon\">Pilon<\/a>","<a class=\"homepage\" href=\"https://github.com/igg-molecular-biology-lab/pipe-t\" ga-product=\"homepage\" ga-id=\"pipe_t\">pipe_t<\/a>",null,"<a class=\"homepage\" href=\"http://zzz.bwh.harvard.edu/plink/\" ga-product=\"homepage\" ga-id=\"plink\">plink<\/a>",null,null,"<a class=\"homepage\" href=\"https://parallel-netcdf.github.io/\" ga-product=\"homepage\" ga-id=\"pnetcdf\">PnetCDF<\/a>",null,null,"<a class=\"homepage\" href=\"https://github.com/arq5x/poretools\" ga-product=\"homepage\" ga-id=\"poretools\">poretools<\/a>","<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/clstutils.html\" ga-product=\"homepage\" ga-id=\"pplacer\">pplacer<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/wtsi-hpag/PretextView\" ga-product=\"homepage\" ga-id=\"pretextview\">PretextView / Pretext Suite<\/a>","<a class=\"homepage\" href=\"https://code.google.com/p/gmv/\" ga-product=\"homepage\" ga-id=\"prodigal\">Prodigal<\/a>","<a class=\"homepage\" href=\"https://github.com/tseemann/prokka\" ga-product=\"homepage\" ga-id=\"prokka\">Prokka<\/a>",null,"<a class=\"homepage\" href=\"https://anaconda.org/bioconda/ucsc-pslcdnafilter\" ga-product=\"homepage\" ga-id=\"pslcdnafilter\">pslcdnafilter<\/a>","<a class=\"homepage\" href=\"https://github.com/dfguan/purge_dups\" ga-product=\"homepage\" ga-id=\"purge_dups\">Purge_Dups<\/a>",null,"<a class=\"homepage\" href=\"https://a-slide.github.io/pycoQC/\" ga-product=\"homepage\" ga-id=\"pycoqc\">pycoqc<\/a>","<a class=\"homepage\" href=\"https://github.com/deeptools/pyGenomeTracks\" ga-product=\"homepage\" ga-id=\"pygenometracks\">pygenometracks<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/pysam-developers/pysam\" ga-product=\"homepage\" ga-id=\"pysam\">pysam<\/a>","<a class=\"homepage\" href=\"https://pytorch.org/\" ga-product=\"homepage\" ga-id=\"pytorch\">PyTorch<\/a>","<a class=\"homepage\" href=\"https://qiime2.org/\" ga-product=\"homepage\" ga-id=\"qiime2\">QIIME2.0<\/a>",null,"<a class=\"homepage\" href=\"http://qualimap.bioinfo.cipf.es/\" ga-product=\"homepage\" ga-id=\"qualimap\">qualimap<\/a>","<a class=\"homepage\" href=\"https://phac-nml.github.io/quasitools/\" ga-product=\"homepage\" ga-id=\"quasitools\">quasitools<\/a>","<a class=\"homepage\" href=\"http://quast.sourceforge.net/quast\" ga-product=\"homepage\" ga-id=\"quast\">QUAST<\/a>","<a class=\"homepage\" href=\"https://github.com/galaxyproject/tools-iuc/tree/master/tools/query_tabular\" ga-product=\"homepage\" ga-id=\"query_tabular\">query_tabular<\/a>","<a class=\"homepage\" href=\"http://www.r-project.org/\" ga-product=\"homepage\" ga-id=\"r\">R<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/isovic/racon\" ga-product=\"homepage\" ga-id=\"racon\">Racon<\/a>","<a class=\"homepage\" href=\"https://github.com/cbroeckl/RAMClustR\" ga-product=\"homepage\" ga-id=\"ramclustr\">RAMClustR<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/lbcb-sci/raven\" ga-product=\"homepage\" ga-id=\"raven\">raven<\/a>","<a class=\"homepage\" href=\"https://github.com/kevinkovalchik/RawTools\" ga-product=\"homepage\" ga-id=\"rawtools\">Raw Tools<\/a>","<a class=\"homepage\" href=\"http://sco.h-its.org/exelixis/web/software/raxml/\" ga-product=\"homepage\" ga-id=\"raxml\">RAxML<\/a>","<a class=\"homepage\" href=\"http://www.rdkit.org\" ga-product=\"homepage\" ga-id=\"rdkit\">rdkit<\/a>",null,"<a class=\"homepage\" href=\"http://wwwmgs.bionet.nsc.ru/mgs/programs/recon/\" ga-product=\"homepage\" ga-id=\"recon\">recon<\/a>","<a class=\"homepage\" href=\"https://github.com/RECETOX/recetox-aplcms\" ga-product=\"homepage\" ga-id=\"recetox-aplcms\">recetox-aplcms<\/a>","<a class=\"homepage\" href=\"https://github.com/REDetector/RED\" ga-product=\"homepage\" ga-id=\"red\">RED<\/a>","<a class=\"homepage\" href=\"http://www.repeatmasker.org/\" ga-product=\"homepage\" ga-id=\"repeatmasker\">RepeatMasker<\/a>","<a class=\"homepage\" href=\"http://www.repeatmasker.org/RepeatModeler/\" ga-product=\"homepage\" ga-id=\"repeatmodeler\">RepeatModeler<\/a>","<a class=\"homepage\" href=\"http://bix.ucsd.edu/repeatscout/\" ga-product=\"homepage\" ga-id=\"repeatscout\">RepeatScout<\/a>",null,null,"<a class=\"homepage\" href=\"https://cran.r-project.org/package=rjags\" ga-product=\"homepage\" ga-id=\"rjags\">rjags<\/a>","<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/RMassBank.html\" ga-product=\"homepage\" ga-id=\"rmassbank\">RMassBank<\/a>",null,"<a class=\"homepage\" href=\"http://www.repeatmasker.org/RMBlast.html\" ga-product=\"homepage\" ga-id=\"rmblast\">rmblast<\/a>",null,"<a class=\"homepage\" href=\"http://sanger-pathogens.github.io/Roary/\" ga-product=\"homepage\" ga-id=\"roary\">roary<\/a>",null,"<a class=\"homepage\" href=\"http://deweylab.github.io/RSEM/\" ga-product=\"homepage\" ga-id=\"rsem\">RSEM<\/a>","<a class=\"homepage\" href=\"http://rseqc.sourceforge.net\" ga-product=\"homepage\" ga-id=\"rseqc\">rseqc<\/a>","<a class=\"homepage\" href=\"https://www.rstudio.com/\" ga-product=\"homepage\" ga-id=\"rstudio\">Rstudio<\/a>","<a class=\"homepage\" href=\"https://github.com/RealTimeGenomics/rtg-core\" ga-product=\"homepage\" ga-id=\"rtg-tools\">rtg-tools<\/a>",null,"<a class=\"homepage\" href=\"http://www.cs.cmu.edu/~ckingsf/software/sailfish/\" ga-product=\"homepage\" ga-id=\"sailfish\">sailfish<\/a>","<a class=\"homepage\" href=\"https://combine-lab.github.io/salmon\" ga-product=\"homepage\" ga-id=\"salmon\">Salmon<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/machinegun/SALSA\" ga-product=\"homepage\" ga-id=\"salsa\">salsa<\/a>",null,null,"<a class=\"homepage\" href=\"http://www.open-bio.org/wiki/Sambamba\" ga-product=\"homepage\" ga-id=\"sambamba\">sambamba<\/a>","<a class=\"homepage\" href=\"https://github.com/GregoryFaust/samblaster\" ga-product=\"homepage\" ga-id=\"samblaster\">samblaster<\/a>",null,"<a class=\"homepage\" href=\"http://www.htslib.org/\" ga-product=\"homepage\" ga-id=\"samtools\">SAMTools<\/a>","<a class=\"homepage\" href=\"https://github.com/theislab/Scanpy\" ga-product=\"homepage\" ga-id=\"scanpy\">scanpy<\/a>","<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/scater.html\" ga-product=\"homepage\" ga-id=\"scater\">scater<\/a>",null,"<a class=\"homepage\" href=\"http://scikit-image.org\" ga-product=\"homepage\" ga-id=\"scikit-image\">scikit-image<\/a>","<a class=\"homepage\" href=\"http://scikit-learn.org\" ga-product=\"homepage\" ga-id=\"scikit\">Scikit-learn<\/a>","<a class=\"homepage\" href=\"https://www.webscipio.org/\" ga-product=\"homepage\" ga-id=\"scipio\">scipio<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/smirarab/sepp\" ga-product=\"homepage\" ga-id=\"sepp\">sepp<\/a>",null,null,"<a class=\"homepage\" href=\"https://bioinf.shenwei.me/seqkit/\" ga-product=\"homepage\" ga-id=\"seqkit\">SeqKit<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/lh3/seqtk\" ga-product=\"homepage\" ga-id=\"seqtk\">seqtk<\/a>","<a class=\"homepage\" href=\"https://satijalab.org/seurat/\" ga-product=\"homepage\" ga-id=\"seurat\">seurat<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/chanzuckerberg/shasta\" ga-product=\"homepage\" ga-id=\"shasta\">shasta<\/a>","<a class=\"homepage\" href=\"https://github.com/tseemann/shovill\" ga-product=\"homepage\" ga-id=\"shovill\">shovill<\/a>","<a class=\"homepage\" href=\"http://home.gwu.edu/%7Ewpeng/Software.htm\" ga-product=\"homepage\" ga-id=\"sicer\">sicer<\/a>","<a class=\"homepage\" href=\"https://github.com/mgramm1/simtext\" ga-product=\"homepage\" ga-id=\"simtext\">SimText<\/a>",null,"<a class=\"homepage\" href=\"https://lfz.corefacility.ca/sistr-app/\" ga-product=\"homepage\" ga-id=\"sistr\">sistr<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/hasindu2008/slow5tools\" ga-product=\"homepage\" ga-id=\"slow5tools\">slow5tools<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/KamilSJaron/smudgeplot\" ga-product=\"homepage\" ga-id=\"Smudgeplots\">Smudgeplots<\/a>","<a class=\"homepage\" href=\"https://snakemake.readthedocs.io/en/stable/index.html\" ga-product=\"homepage\" ga-id=\"snakemake\">Snakemake<\/a>","<a class=\"homepage\" href=\"http://korflab.ucdavis.edu/software.html\" ga-product=\"homepage\" ga-id=\"snap\">SNAP<\/a>","<a class=\"homepage\" href=\"https://github.com/fritzsedlazeck/Sniffles\" ga-product=\"homepage\" ga-id=\"sniffles\">sniffles<\/a>","<a class=\"homepage\" href=\"https://github.com/tseemann/snippy\" ga-product=\"homepage\" ga-id=\"snippy\">snippy<\/a>",null,null,"<a class=\"homepage\" href=\"http://snpeff.sourceforge.net/\" ga-product=\"homepage\" ga-id=\"snpeff\">snpeff<\/a>",null,null,"<a class=\"homepage\" href=\"http://snpeff.sourceforge.net/SnpSift.html\" ga-product=\"homepage\" ga-id=\"snpsift\">snpsift<\/a>","<a class=\"homepage\" href=\"https://github.com/aquaskyline/SOAPdenovo2\" ga-product=\"homepage\" ga-id=\"soapdenovo2\">SOAPdenovo2<\/a>","<a class=\"homepage\" href=\"https://bioinfo.cristal.univ-lille.fr/RNA/sortmerna/\" ga-product=\"homepage\" ga-id=\"sortmerna\">sortmerna<\/a>","<a class=\"homepage\" href=\"http://cab.spbu.ru/software/spades/\" ga-product=\"homepage\" ga-id=\"spades\">SPAdes<\/a>",null,"<a class=\"homepage\" href=\"https://webs.iiitd.edu.in/raghava/srf/\" ga-product=\"homepage\" ga-id=\"srf\">Spectral Repeat Finder (SRF)<\/a>",null,"<a class=\"homepage\" href=\"https://www.sqlite.org/index.html\" ga-product=\"homepage\" ga-id=\"sqlite\">sqlite<\/a>",null,null,"<a class=\"homepage\" href=\"https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=toolkit_doc\" ga-product=\"homepage\" ga-id=\"sra-tools\">sra-tools<\/a>","<a class=\"homepage\" href=\"http://blog.gkno.me/post/42508822069/ssw-library\" ga-product=\"homepage\" ga-id=\"ssw\">ssw<\/a>","<a class=\"homepage\" href=\"http://catchenlab.life.illinois.edu/stacks/\" ga-product=\"homepage\" ga-id=\"stacks\">Stacks<\/a>","<a class=\"homepage\" href=\"http://code.google.com/p/rna-star/\" ga-product=\"homepage\" ga-id=\"star\">star<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/STAR-Fusion/STAR-Fusion\" ga-product=\"homepage\" ga-id=\"star_fusion\">STAR-Fusion<\/a>","<a class=\"homepage\" href=\"https://ccb.jhu.edu/software/stringtie/\" ga-product=\"homepage\" ga-id=\"stringtie\">stringtie<\/a>","<a class=\"homepage\" href=\"http://subread.sourceforge.net/\" ga-product=\"homepage\" ga-id=\"subread\">subread<\/a>",null,"<a class=\"homepage\" href=\"http://edwards.sdsu.edu/superfocus/\" ga-product=\"homepage\" ga-id=\"superfocus\">SUPER-FOCUS<\/a>","<a class=\"homepage\" href=\"http://github.com/eldariont/svim-asm\" ga-product=\"homepage\" ga-id=\"svim-asm\">SVIM-asm<\/a>","<a class=\"homepage\" href=\"https://schneebergerlab.github.io/syri/\" ga-product=\"homepage\" ga-id=\"syri\">SyRI<\/a>",null,null,null,"<a class=\"homepage\" href=\"https://github.com/carringtonlab/TargetFinder\" ga-product=\"homepage\" ga-id=\"targetfinder\">TargetFinder<\/a>",null,null,null,null,"<a class=\"homepage\" href=\"https://www.tensorflow.org\" ga-product=\"homepage\" ga-id=\"tensorflow\">TensorFlow<\/a>","<a class=\"homepage\" href=\"https://github.com/zhangrengang/TEsorter\" ga-product=\"homepage\" ga-id=\"tesorter\">TEsorter<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/compomics/ThermoRawFileParser\" ga-product=\"homepage\" ga-id=\"ThermoRawFileParser\">ThermoRawFileParser<\/a>","<a class=\"homepage\" href=\"http://ccb.jhu.edu/software/tophat/index.shtml\" ga-product=\"homepage\" ga-id=\"tophat\">tophat<\/a>","<a class=\"homepage\" href=\"https://totalview.io/\" ga-product=\"homepage\" ga-id=\"totalview\">TotalView<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/TransDecoder/TransDecoder/wiki\" ga-product=\"homepage\" ga-id=\"transdecoder\">TransDecoder<\/a>","<a class=\"homepage\" href=\"http://saclab.tamu.edu/essentiality/transit/\" ga-product=\"homepage\" ga-id=\"transit\">transit<\/a>",null,"<a class=\"homepage\" href=\"http://tools.proteomecenter.org/software.php\" ga-product=\"homepage\" ga-id=\"tpp\">Trans-Proteomic Pipeline (TPP)<\/a>",null,"<a class=\"homepage\" href=\"http://tandem.bu.edu/trf/trf.html\" ga-product=\"homepage\" ga-id=\"trf\">trf<\/a>","<a class=\"homepage\" href=\"http://trimal.cgenomics.org\" ga-product=\"homepage\" ga-id=\"trimal\">trimAl<\/a>","<a class=\"homepage\" href=\"http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/\" ga-product=\"homepage\" ga-id=\"trim_galore\">Trim Galore<\/a>","<a class=\"homepage\" href=\"http://www.usadellab.org/cms/index.php?page=trimmomatic\" ga-product=\"homepage\" ga-id=\"trimmomatic\">Trimmomatic<\/a>","<a class=\"homepage\" href=\"https://github.com/trinityrnaseq/trinityrnaseq/wiki\" ga-product=\"homepage\" ga-id=\"trinity\">Trinity / trinityrnaseq<\/a>","<a class=\"homepage\" href=\"https://github.com/Trinotate/Trinotate.github.io\" ga-product=\"homepage\" ga-id=\"trinotate\">Trinotate<\/a>","<a class=\"homepage\" href=\"http://trna.ucsc.edu/tRNAscan-SE/\" ga-product=\"homepage\" ga-id=\"trnascan-se\">trnascan-se<\/a>","<a class=\"homepage\" href=\"http://github.com/rrwick/Trycycler-paper\" ga-product=\"homepage\" ga-id=\"trycycler\">Trycycler toolkit<\/a>","<a class=\"homepage\" href=\"http://hgdownload.cse.ucsc.edu/downloads.html\" ga-product=\"homepage\" ga-id=\"ucsc\">UCSC tools<\/a>",null,null,"<a class=\"homepage\" href=\"https://github.com/CGATOxford/UMI-tools\" ga-product=\"homepage\" ga-id=\"umi_tools\">umi_tools<\/a>","<a class=\"homepage\" href=\"https://github.com/rrwick/Unicycler\" ga-product=\"homepage\" ga-id=\"unicycler\">Unicycler<\/a>","<a class=\"homepage\" href=\"http://unipept.ugent.be/\" ga-product=\"homepage\" ga-id=\"unipept\">unipept<\/a>",null,null,null,"<a class=\"homepage\" href=\"http://www.ensembl.org/vep\" ga-product=\"homepage\" ga-id=\"ensembl-vep\">Variant Effect Predictor (VEP)<\/a>","<a class=\"homepage\" href=\"http://dkoboldt.github.io/varscan/\" ga-product=\"homepage\" ga-id=\"varscan\">VarScan<\/a>",null,null,"<a class=\"homepage\" href=\"https://github.com/vcflib/vcflib\" ga-product=\"homepage\" ga-id=\"vcflib\">vcflib<\/a>","<a class=\"homepage\" href=\"https://vcftools.github.io/index.html\" ga-product=\"homepage\" ga-id=\"vcftools\">VCFTools<\/a>","<a class=\"homepage\" href=\"https://github.com/dzerbino/velvet\" ga-product=\"homepage\" ga-id=\"velvet\">Velvet<\/a>","<a class=\"homepage\" href=\"https://github.com/marbl/verkko\" ga-product=\"homepage\" ga-id=\"verkko\">verkko<\/a>","<a class=\"homepage\" href=\"https://wci.llnl.gov/simulation/computer-codes/visit/\" ga-product=\"homepage\" ga-id=\"visit\">VisIt<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/torognes/vsearch\" ga-product=\"homepage\" ga-id=\"vsearch\">vsearch<\/a>",null,"<a class=\"homepage\" href=\"http://whatshap.readthedocs.io\" ga-product=\"homepage\" ga-id=\"whatshap\">WhatsHap<\/a>",null,null,"<a class=\"homepage\" href=\"https://github.com/marbl/Winnowmap\" ga-product=\"homepage\" ga-id=\"winnowmap\">winnowmap<\/a>","<a class=\"homepage\" href=\"https://workflow4metabolomics.org/\" ga-product=\"homepage\" ga-id=\"workflow4metabolomics\">Workflow 4 Metabolomics<\/a>",null,"<a class=\"homepage\" href=\"http://xarray.pydata.org/en/stable/\" ga-product=\"homepage\" ga-id=\"xarray\">xarray<\/a>","<a class=\"homepage\" href=\"http://bioconductor.org/packages/release/bioc/html/xcms.html\" ga-product=\"homepage\" ga-id=\"xcms\">xcms<\/a>","<a class=\"homepage\" href=\"https://github.com/AustralianBioCommons/xml4ena\" ga-product=\"homepage\" ga-id=\"xml4ena\">xml4ena<\/a>","<a class=\"homepage\" href=\"https://github.com/GoekeLab/xpore\" ga-product=\"homepage\" ga-id=\"xpore\">xpore<\/a>","<a class=\"homepage\" href=\"http://www.thegpm.org/TANDEM/\" ga-product=\"homepage\" ga-id=\"xtandem\">xtandem<\/a>",null,"<a class=\"homepage\" href=\"https://github.com/c-zhou/yahs\" ga-product=\"homepage\" ga-id=\"yahs\">YaHS<\/a>",null,null],["<a class=\"biotools\" href=\"https://bio.tools/ABRicate\" ga-product=\"biotool\" ga-id=\"abricate\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/abyss\" ga-product=\"biotool\" ga-id=\"abyss\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/amber\" ga-product=\"biotool\" ga-id=\"amber\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/anaconda\" ga-product=\"biotool\" ga-id=\"anaconda\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/annotatemyids\" ga-product=\"biotool\" ga-id=\"annotatemyids\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/antismash\" ga-product=\"biotool\" ga-id=\"antismash\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/any2fasta\" ga-product=\"biotool\" ga-id=\"any2fasta\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/apollo\" ga-product=\"biotool\" ga-id=\"apollo\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/aragorn\" ga-product=\"biotool\" ga-id=\"aragorn\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/Arriba\" ga-product=\"biotool\" ga-id=\"arriba\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/astral\" ga-product=\"biotool\" ga-id=\"astral\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/augustus\" ga-product=\"biotool\" ga-id=\"augustus\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/autodock_vina\" ga-product=\"biotool\" ga-id=\"autodock_vina\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,null,"<a class=\"biotools\" href=\"https://bio.tools/bakta\" ga-product=\"biotool\" ga-id=\"bakta\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/bamtools\" ga-product=\"biotool\" ga-id=\"bamtools\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/bamutil\" ga-product=\"biotool\" ga-id=\"bamutil_diff\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/bandage\" ga-product=\"biotool\" ga-id=\"bandage\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/barrnap\" ga-product=\"biotool\" ga-id=\"barrnap\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/bbmap\" ga-product=\"biotool\" ga-id=\"bbmap\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/gff3_rebase\" ga-product=\"biotool\" ga-id=\"bcbiogff\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/bcftools\" ga-product=\"biotool\" ga-id=\"bcftools\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/beagle\" ga-product=\"biotool\" ga-id=\"beagle\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/beagle-lib\" ga-product=\"biotool\" ga-id=\"beagle-lib\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/beast\" ga-product=\"biotool\" ga-id=\"beast\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/beast2\" ga-product=\"biotool\" ga-id=\"beast2\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/bed_to_protein_map\" ga-product=\"biotool\" ga-id=\"bed_to_protein_map\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/bedtools\" ga-product=\"biotool\" ga-id=\"bedtools\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/Bellerophon\" ga-product=\"biotool\" ga-id=\"bellerophon\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/berokka\" ga-product=\"biotool\" ga-id=\"berokka\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/bio3d\" ga-product=\"biotool\" ga-id=\"bio3d\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/biobambam\" ga-product=\"biotool\" ga-id=\"biobambam\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/biomformat\" ga-product=\"biotool\" ga-id=\"biom-format\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/bioperl\" ga-product=\"biotool\" ga-id=\"bioperl\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/biopython\" ga-product=\"biotool\" ga-id=\"biopython\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/biotransformer\" ga-product=\"biotool\" ga-id=\"biotransformer\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/blast\" ga-product=\"biotool\" ga-id=\"blast\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/blast\" ga-product=\"biotool\" ga-id=\"blast+\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/blast\" ga-product=\"biotool\" ga-id=\"blast2\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/blat\" ga-product=\"biotool\" ga-id=\"blat\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/blockbuster\" ga-product=\"biotool\" ga-id=\"Blockbuster\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/boost\" ga-product=\"biotool\" ga-id=\"boost\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/bowtie\" ga-product=\"biotool\" ga-id=\"bowtie\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/bowtie2\" ga-product=\"biotool\" ga-id=\"bowtie2\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/breseq\" ga-product=\"biotool\" ga-id=\"breseq\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/busco\" ga-product=\"biotool\" ga-id=\"busco\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/bwa\" ga-product=\"biotool\" ga-id=\"bwa\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/bwa-mem2\" ga-product=\"biotool\" ga-id=\"bwa-mem2\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/bx-python\" ga-product=\"biotool\" ga-id=\"bx-python\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/C3\" ga-product=\"biotool\" ga-id=\"c3s\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/cactus\" ga-product=\"biotool\" ga-id=\"cactus\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/camera\" ga-product=\"biotool\" ga-id=\"camera\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/canu\" ga-product=\"biotool\" ga-id=\"canu\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/cap3\" ga-product=\"biotool\" ga-id=\"cap3\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/cardinal\" ga-product=\"biotool\" ga-id=\"cardinal\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/cat_bins\" ga-product=\"biotool\" ga-id=\"cat_bins\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/cd-hit\" ga-product=\"biotool\" ga-id=\"cd-hit\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/CellProfiler\" ga-product=\"biotool\" ga-id=\"CellProfiler\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/cellxgene\" ga-product=\"biotool\" ga-id=\"cellxgene\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/checkm\" ga-product=\"biotool\" ga-id=\"checkm\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/chembl\" ga-product=\"biotool\" ga-id=\"chembl\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/chemfp\" ga-product=\"biotool\" ga-id=\"chemfp\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/chipseeker\" ga-product=\"biotool\" ga-id=\"chipseeker\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/chromeister\" ga-product=\"biotool\" ga-id=\"chromeister\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/circos\" ga-product=\"biotool\" ga-id=\"circos\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/clinker\" ga-product=\"biotool\" ga-id=\"clinker\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/clustalo\" ga-product=\"biotool\" ga-id=\"clustalo\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/clustalw\" ga-product=\"biotool\" ga-id=\"clustalw\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/colabfold\" ga-product=\"biotool\" ga-id=\"colabfold_batch\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<a class=\"biotools\" href=\"https://bio.tools/cpat\" ga-product=\"biotool\" ga-id=\"cpat\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/cuDNN\" ga-product=\"biotool\" ga-id=\"cudnn\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/cufflinks\" ga-product=\"biotool\" ga-id=\"cufflinks\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/cummerbund\" ga-product=\"biotool\" ga-id=\"cummeRbund\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/customprodb\" ga-product=\"biotool\" ga-id=\"customprodb\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/cutadapt\" ga-product=\"biotool\" ga-id=\"cutadapt\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/cuteSV\" ga-product=\"biotool\" ga-id=\"cutesv\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/dada2\" ga-product=\"biotool\" ga-id=\"dada2\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,null,null,null,"<a class=\"biotools\" href=\"https://bio.tools/deeptools\" ga-product=\"biotool\" ga-id=\"deeptools\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/dendropy\" ga-product=\"biotool\" ga-id=\"dendropy\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/deseq2\" ga-product=\"biotool\" ga-id=\"deseq2\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/dexseq\" ga-product=\"biotool\" ga-id=\"DEXSeq\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/DIA-NN\" ga-product=\"biotool\" ga-id=\"diann\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/DIA-Umpire\" ga-product=\"biotool\" ga-id=\"diaumpire\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/diamond\" ga-product=\"biotool\" ga-id=\"diamond\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/diapasef\" ga-product=\"biotool\" ga-id=\"diapysef\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/diffbind\" ga-product=\"biotool\" ga-id=\"diffbind\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/dropletutils\" ga-product=\"biotool\" ga-id=\"dropletutils\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/easybuild\" ga-product=\"biotool\" ga-id=\"easybuild\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/edger\" ga-product=\"biotool\" ga-id=\"edger\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/edger\" ga-product=\"biotool\" ga-id=\"edger-repenrich\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/eggnog-mapper\" ga-product=\"biotool\" ga-id=\"eggnog-mapper\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/egsea\" ga-product=\"biotool\" ga-id=\"egsea\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/elastix\" ga-product=\"biotool\" ga-id=\"elastix\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/emboss\" ga-product=\"biotool\" ga-id=\"emboss\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/EncyclopeDIA\" ga-product=\"biotool\" ga-id=\"encyclopedia\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/eupathdb\" ga-product=\"biotool\" ga-id=\"eupathdb\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/exabayes\" ga-product=\"biotool\" ga-id=\"exabayes\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/ExaML\" ga-product=\"biotool\" ga-id=\"examl\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/exonerate\" ga-product=\"biotool\" ga-id=\"exonerate\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/express\" ga-product=\"biotool\" ga-id=\"eXpress\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/falcon\" ga-product=\"biotool\" ga-id=\"pb-assembly\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,null,null,null,null,null,"<a class=\"biotools\" href=\"https://bio.tools/fastani\" ga-product=\"biotool\" ga-id=\"fastani\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/fastool\" ga-product=\"biotool\" ga-id=\"fastool\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/fastp\" ga-product=\"biotool\" ga-id=\"fastp\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/fastqc\" ga-product=\"biotool\" ga-id=\"fastqc\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/fasttree\" ga-product=\"biotool\" ga-id=\"fasttree\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/fastx-toolkit\" ga-product=\"biotool\" ga-id=\"fastx\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/fastx-toolkit\" ga-product=\"biotool\" ga-id=\"fastx_toolkit\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/fgenesh\" ga-product=\"biotool\" ga-id=\"fgenesh\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/fgsea\" ga-product=\"biotool\" ga-id=\"fgsea\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,null,"<a class=\"biotools\" href=\"https://bio.tools/filtlong\" ga-product=\"biotool\" ga-id=\"filtlong\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/flashlfq\" ga-product=\"biotool\" ga-id=\"flashlfq\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/Flye\" ga-product=\"biotool\" ga-id=\"flye\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/foldseek\" ga-product=\"biotool\" ga-id=\"foldseek\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/fpocket\" ga-product=\"biotool\" ga-id=\"fpocket\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/fraggenescan\" ga-product=\"biotool\" ga-id=\"fraggenescan\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/freebayes\" ga-product=\"biotool\" ga-id=\"freebayes\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/funannotate\" ga-product=\"biotool\" ga-id=\"funannotate\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/gaeval\" ga-product=\"biotool\" ga-id=\"gaeval\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,null,null,null,null,null,"<a class=\"biotools\" href=\"https://bio.tools/gatk\" ga-product=\"biotool\" ga-id=\"gatk\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/gblocks\" ga-product=\"biotool\" ga-id=\"gblock\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/gcta\" ga-product=\"biotool\" ga-id=\"gcta\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/gecko\" ga-product=\"biotool\" ga-id=\"gecko\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/gemini\" ga-product=\"biotool\" ga-id=\"gemini\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/gemoma\" ga-product=\"biotool\" ga-id=\"gemoma\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,null,null,"<a class=\"biotools\" href=\"https://bio.tools/GenomeScope_2.0\" ga-product=\"biotool\" ga-id=\"genomescope\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/genometools\" ga-product=\"biotool\" ga-id=\"genometools\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/getorganelle\" ga-product=\"biotool\" ga-id=\"getorganelle\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/gfastats\" ga-product=\"biotool\" ga-id=\"gfastats\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/gffcompare\" ga-product=\"biotool\" ga-id=\"gffcompare\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/gffread\" ga-product=\"biotool\" ga-id=\"gffread\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/ggplot2\" ga-product=\"biotool\" ga-id=\"ggplot2\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/parallel\" ga-product=\"biotool\" ga-id=\"parallel\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/gsl\" ga-product=\"biotool\" ga-id=\"gsl\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/goenrichment\" ga-product=\"biotool\" ga-id=\"goenrichment\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/goseq\" ga-product=\"biotool\" ga-id=\"goseq\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/graphlan\" ga-product=\"biotool\" ga-id=\"graphlan\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/gromacs\" ga-product=\"biotool\" ga-id=\"gromacs\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/GTDB-Tk\" ga-product=\"biotool\" ga-id=\"gtdb-tk\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/gubbins\" ga-product=\"biotool\" ga-id=\"gubbins\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/hapcut2\" ga-product=\"biotool\" ga-id=\"hapcut2\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/heinz\" ga-product=\"biotool\" ga-id=\"heinz\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/hgv_david\" ga-product=\"biotool\" ga-id=\"hgv_david\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/hgv_ldtools\" ga-product=\"biotool\" ga-id=\"hgv_ldtools\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/hgv_linktogprofile\" ga-product=\"biotool\" ga-id=\"hgv_linkToGProfile\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/hicexplorer\" ga-product=\"biotool\" ga-id=\"hicexplorer\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/hifiadapterfilt\" ga-product=\"biotool\" ga-id=\"hifiadapterfilt\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/hifiasm\" ga-product=\"biotool\" ga-id=\"hifiasm\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/hisat2\" ga-product=\"biotool\" ga-id=\"hisat2\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/hmmer3\" ga-product=\"biotool\" ga-id=\"hmmer\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/horovod\" ga-product=\"biotool\" ga-id=\"horovod\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/htseq\" ga-product=\"biotool\" ga-id=\"htseq\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/htslib\" ga-product=\"biotool\" ga-id=\"htslib\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/humann\" ga-product=\"biotool\" ga-id=\"humann\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/humann2\" ga-product=\"biotool\" ga-id=\"humann2\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/HyPhy\" ga-product=\"biotool\" ga-id=\"HyPhy\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/ipa_hifi\" ga-product=\"biotool\" ga-id=\"pbipa\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/infernal\" ga-product=\"biotool\" ga-id=\"infernal\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/igv\" ga-product=\"biotool\" ga-id=\"igv\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/integron_finder\" ga-product=\"biotool\" ga-id=\"integron_finder\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/interproscan_4\" ga-product=\"biotool\" ga-id=\"interproscan\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,null,"<a class=\"biotools\" href=\"https://bio.tools/ipyrad\" ga-product=\"biotool\" ga-id=\"ipyrad\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/iq-tree\" ga-product=\"biotool\" ga-id=\"iq-tree\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/isee\" ga-product=\"biotool\" ga-id=\"isee\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/isescan\" ga-product=\"biotool\" ga-id=\"isescan\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/isoformswitchanalyzer\" ga-product=\"biotool\" ga-id=\"isoformswitchanalyzer\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/ivar\" ga-product=\"biotool\" ga-id=\"ivar\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/iwtomics\" ga-product=\"biotool\" ga-id=\"iwtomics\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/jags\" ga-product=\"biotool\" ga-id=\"jags\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/jasmine-sv\" ga-product=\"biotool\" ga-id=\"jasminesv\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/jbrowse\" ga-product=\"biotool\" ga-id=\"jbrowse\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/jellyfish\" ga-product=\"biotool\" ga-id=\"jellyfish\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,null,"<a class=\"biotools\" href=\"https://bio.tools/kallisto\" ga-product=\"biotool\" ga-id=\"kallisto\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/kat\" ga-product=\"biotool\" ga-id=\"kat\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/khmer\" ga-product=\"biotool\" ga-id=\"khmer\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/kmc\" ga-product=\"biotool\" ga-id=\"kmc\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/kraken\" ga-product=\"biotool\" ga-id=\"kraken\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/kraken2\" ga-product=\"biotool\" ga-id=\"kraken2\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/krona\" ga-product=\"biotool\" ga-id=\"krona\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/lama\" ga-product=\"biotool\" ga-id=\"lama\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/lastz\" ga-product=\"biotool\" ga-id=\"lastz\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/LFQ-Analyst\" ga-product=\"biotool\" ga-id=\"LFQ-Analyst\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/limma\" ga-product=\"biotool\" ga-id=\"limma\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/links\" ga-product=\"biotool\" ga-id=\"links\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/lofreq\" ga-product=\"biotool\" ga-id=\"lofreq\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/longshot\" ga-product=\"biotool\" ga-id=\"longshot\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/ltr_retriever\" ga-product=\"biotool\" ga-id=\"ltr_retriever\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/macs\" ga-product=\"biotool\" ga-id=\"macs\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/macs\" ga-product=\"biotool\" ga-id=\"macs2\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,null,null,null,null,null,null,null,null,null,"<a class=\"biotools\" href=\"https://bio.tools/MAFFT\" ga-product=\"biotool\" ga-id=\"mafft\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/mageck\" ga-product=\"biotool\" ga-id=\"MAGeCK\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/maker\" ga-product=\"biotool\" ga-id=\"maker\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/maldi_quant\" ga-product=\"biotool\" ga-id=\"maldi_quant\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/mash\" ga-product=\"biotool\" ga-id=\"mash\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/masurca\" ga-product=\"biotool\" ga-id=\"masurca\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/matchms\" ga-product=\"biotool\" ga-id=\"matchms\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/matplotlib\" ga-product=\"biotool\" ga-id=\"matplotlib\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/maxbin\" ga-product=\"biotool\" ga-id=\"maxbin\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/maxquant\" ga-product=\"biotool\" ga-id=\"maxquant\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/maxquant\" ga-product=\"biotool\" ga-id=\"biocontainers-maxquant\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/mcl\" ga-product=\"biotool\" ga-id=\"mcl\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/mdanalysis\" ga-product=\"biotool\" ga-id=\"mdanalysis\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/medaka\" ga-product=\"biotool\" ga-id=\"medaka\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/megahit\" ga-product=\"biotool\" ga-id=\"megahit\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/merqury\" ga-product=\"biotool\" ga-id=\"merqury\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/meryl\" ga-product=\"biotool\" ga-id=\"meryl\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/MetaBAT_2\" ga-product=\"biotool\" ga-id=\"metabat\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/MetaDEGalaxy\" ga-product=\"biotool\" ga-id=\"MetaDEGalaxy\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/MetaEuk\" ga-product=\"biotool\" ga-id=\"metaeuk\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/metaphlan\" ga-product=\"biotool\" ga-id=\"metaphlan\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/metaQuantome\" ga-product=\"biotool\" ga-id=\"metaQuantome\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/metaspades\" ga-product=\"biotool\" ga-id=\"metaspades\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/metawrap\" ga-product=\"biotool\" ga-id=\"metawrap\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/methyldackel\" ga-product=\"biotool\" ga-id=\"MethylDackel\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/MIGRATE\" ga-product=\"biotool\" ga-id=\"MIGRATE\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/minia\" ga-product=\"biotool\" ga-id=\"minia\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/miniasm\" ga-product=\"biotool\" ga-id=\"miniasm\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/minimap2\" ga-product=\"biotool\" ga-id=\"minimap2\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/miniprot\" ga-product=\"biotool\" ga-id=\"miniprot\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/mirdeep2\" ga-product=\"biotool\" ga-id=\"mirdeep2\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/mitos\" ga-product=\"biotool\" ga-id=\"mitos2\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/mlst\" ga-product=\"biotool\" ga-id=\"mlst\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/MMseqs2\" ga-product=\"biotool\" ga-id=\"mmseqs2\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/mob-suite\" ga-product=\"biotool\" ga-id=\"mob-suite\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,null,"<a class=\"biotools\" href=\"https://bio.tools/moff\" ga-product=\"biotool\" ga-id=\"moff\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/morpheus\" ga-product=\"biotool\" ga-id=\"morpheus\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/mothur\" ga-product=\"biotool\" ga-id=\"mothur\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/mousemine\" ga-product=\"biotool\" ga-id=\"mousemine\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/mrbayes\" ga-product=\"biotool\" ga-id=\"mrbayes\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/msconvert\" ga-product=\"biotool\" ga-id=\"msconvert\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/msmetaenhancer\" ga-product=\"biotool\" ga-id=\"msmetaenhancer\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/msstats\" ga-product=\"biotool\" ga-id=\"msstats\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/multiqc\" ga-product=\"biotool\" ga-id=\"multiqc\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/mummer\" ga-product=\"biotool\" ga-id=\"mummer\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,null,"<a class=\"biotools\" href=\"https://bio.tools/Nanocompore\" ga-product=\"biotool\" ga-id=\"nanocompore\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/nanoplot\" ga-product=\"biotool\" ga-id=\"nanoplot\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/nanopolish\" ga-product=\"biotool\" ga-id=\"nanopolish\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/ncbi_datasets\" ga-product=\"biotool\" ga-id=\"ncbi-datasets-cli\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/nccmp\" ga-product=\"biotool\" ga-id=\"nccmp\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/nco\" ga-product=\"biotool\" ga-id=\"nco\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/ncview\" ga-product=\"biotool\" ga-id=\"ncview\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/netcdf\" ga-product=\"biotool\" ga-id=\"netcdf\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/newick_utilities\" ga-product=\"biotool\" ga-id=\"newick_utils\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/nextflow\" ga-product=\"biotool\" ga-id=\"nextflow\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/ngsutils\" ga-product=\"biotool\" ga-id=\"ngsutils\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/ninja\" ga-product=\"biotool\" ga-id=\"ninja\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,null,"<a class=\"biotools\" href=\"https://bio.tools/obitools\" ga-product=\"biotool\" ga-id=\"obitools\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/openms\" ga-product=\"biotool\" ga-id=\"openms\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/pangolin\" ga-product=\"biotool\" ga-id=\"pangolin\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/paraview\" ga-product=\"biotool\" ga-id=\"paraview\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/pathview\" ga-product=\"biotool\" ga-id=\"pathview\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/pblat\" ga-product=\"biotool\" ga-id=\"pblat\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/seqcode\" ga-product=\"biotool\" ga-id=\"pe_histogram\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/pear\" ga-product=\"biotool\" ga-id=\"pear\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/peptideshaker\" ga-product=\"biotool\" ga-id=\"peptideshaker\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/percolator\" ga-product=\"biotool\" ga-id=\"percolator\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/pfamscan\" ga-product=\"biotool\" ga-id=\"pfamscan\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/phyloseq\" ga-product=\"biotool\" ga-id=\"phyloseq\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/phyml\" ga-product=\"biotool\" ga-id=\"phyml\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/picard_tools\" ga-product=\"biotool\" ga-id=\"picard\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/picrust\" ga-product=\"biotool\" ga-id=\"picrust\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/picrust2\" ga-product=\"biotool\" ga-id=\"picrust2\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/pilon\" ga-product=\"biotool\" ga-id=\"pilon\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/PIPE-T\" ga-product=\"biotool\" ga-id=\"pipe_t\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/plink\" ga-product=\"biotool\" ga-id=\"plink\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/pnetcdf\" ga-product=\"biotool\" ga-id=\"pnetcdf\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/poretools\" ga-product=\"biotool\" ga-id=\"poretools\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/clstutils\" ga-product=\"biotool\" ga-id=\"pplacer\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/pretextview\" ga-product=\"biotool\" ga-id=\"pretextview\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/gmv\" ga-product=\"biotool\" ga-id=\"prodigal\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/prokka\" ga-product=\"biotool\" ga-id=\"prokka\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/purge_dups\" ga-product=\"biotool\" ga-id=\"purge_dups\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/pycoqc\" ga-product=\"biotool\" ga-id=\"pycoqc\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/pygenometracks\" ga-product=\"biotool\" ga-id=\"pygenometracks\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/pysam\" ga-product=\"biotool\" ga-id=\"pysam\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/pytorch\" ga-product=\"biotool\" ga-id=\"pytorch\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/qiime2\" ga-product=\"biotool\" ga-id=\"qiime2\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/qualimap\" ga-product=\"biotool\" ga-id=\"qualimap\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/quasitools\" ga-product=\"biotool\" ga-id=\"quasitools\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/quast\" ga-product=\"biotool\" ga-id=\"quast\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/Query_Tabular\" ga-product=\"biotool\" ga-id=\"query_tabular\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/r\" ga-product=\"biotool\" ga-id=\"r\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/Racon\" ga-product=\"biotool\" ga-id=\"racon\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/ramclustr\" ga-product=\"biotool\" ga-id=\"ramclustr\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/raven\" ga-product=\"biotool\" ga-id=\"raven\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/rawtools\" ga-product=\"biotool\" ga-id=\"rawtools\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/raxml\" ga-product=\"biotool\" ga-id=\"raxml\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/RDKit\" ga-product=\"biotool\" ga-id=\"rdkit\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/recon\" ga-product=\"biotool\" ga-id=\"recon\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/recetox-aplcms\" ga-product=\"biotool\" ga-id=\"recetox-aplcms\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/red\" ga-product=\"biotool\" ga-id=\"red\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/repeatmasker\" ga-product=\"biotool\" ga-id=\"repeatmasker\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/repeatmodeler\" ga-product=\"biotool\" ga-id=\"repeatmodeler\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/RepeatScout\" ga-product=\"biotool\" ga-id=\"repeatscout\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/rjags\" ga-product=\"biotool\" ga-id=\"rjags\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/rmassbank\" ga-product=\"biotool\" ga-id=\"rmassbank\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/rmblast\" ga-product=\"biotool\" ga-id=\"rmblast\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/roary\" ga-product=\"biotool\" ga-id=\"roary\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/rsem\" ga-product=\"biotool\" ga-id=\"rsem\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/rseqc\" ga-product=\"biotool\" ga-id=\"rseqc\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/rstudio\" ga-product=\"biotool\" ga-id=\"rstudio\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/rtg_core\" ga-product=\"biotool\" ga-id=\"rtg-tools\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/sailfish\" ga-product=\"biotool\" ga-id=\"sailfish\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/salmon\" ga-product=\"biotool\" ga-id=\"salmon\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/SALSA\" ga-product=\"biotool\" ga-id=\"salsa\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/sambamba\" ga-product=\"biotool\" ga-id=\"sambamba\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/samblaster\" ga-product=\"biotool\" ga-id=\"samblaster\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/samtools\" ga-product=\"biotool\" ga-id=\"samtools\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/scanpy\" ga-product=\"biotool\" ga-id=\"scanpy\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/scater\" ga-product=\"biotool\" ga-id=\"scater\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/scikit-image\" ga-product=\"biotool\" ga-id=\"scikit-image\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,null,"<a class=\"biotools\" href=\"https://bio.tools/sepp\" ga-product=\"biotool\" ga-id=\"sepp\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/seqkit\" ga-product=\"biotool\" ga-id=\"seqkit\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/seqtk\" ga-product=\"biotool\" ga-id=\"seqtk\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/seurat\" ga-product=\"biotool\" ga-id=\"seurat\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/shasta\" ga-product=\"biotool\" ga-id=\"shasta\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/shovill\" ga-product=\"biotool\" ga-id=\"shovill\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/sicer\" ga-product=\"biotool\" ga-id=\"sicer\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/simtext\" ga-product=\"biotool\" ga-id=\"simtext\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/SISTR\" ga-product=\"biotool\" ga-id=\"sistr\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,null,"<a class=\"biotools\" href=\"https://bio.tools/Smudgeplots\" ga-product=\"biotool\" ga-id=\"Smudgeplots\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/snakemake\" ga-product=\"biotool\" ga-id=\"snakemake\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/snap\" ga-product=\"biotool\" ga-id=\"snap\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/sniffles\" ga-product=\"biotool\" ga-id=\"sniffles\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/snippy\" ga-product=\"biotool\" ga-id=\"snippy\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/snpeff\" ga-product=\"biotool\" ga-id=\"snpeff\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/snpsift\" ga-product=\"biotool\" ga-id=\"snpsift\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/soapdenovo2\" ga-product=\"biotool\" ga-id=\"soapdenovo2\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/sortmerna\" ga-product=\"biotool\" ga-id=\"sortmerna\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/spades\" ga-product=\"biotool\" ga-id=\"spades\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/srf\" ga-product=\"biotool\" ga-id=\"srf\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,null,null,"<a class=\"biotools\" href=\"https://bio.tools/sra-tools\" ga-product=\"biotool\" ga-id=\"sra-tools\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/ssw\" ga-product=\"biotool\" ga-id=\"ssw\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/stacks\" ga-product=\"biotool\" ga-id=\"stacks\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/star\" ga-product=\"biotool\" ga-id=\"star\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/star-fusion\" ga-product=\"biotool\" ga-id=\"star_fusion\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/stringtie\" ga-product=\"biotool\" ga-id=\"stringtie\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/subread\" ga-product=\"biotool\" ga-id=\"subread\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/super-focus\" ga-product=\"biotool\" ga-id=\"superfocus\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/svim-asm\" ga-product=\"biotool\" ga-id=\"svim-asm\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/SyRI\" ga-product=\"biotool\" ga-id=\"syri\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,null,"<a class=\"biotools\" href=\"https://bio.tools/targetfinder.org\" ga-product=\"biotool\" ga-id=\"targetfinder\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,null,null,"<a class=\"biotools\" href=\"https://bio.tools/tensorflow\" ga-product=\"biotool\" ga-id=\"tensorflow\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/TEsorter\" ga-product=\"biotool\" ga-id=\"tesorter\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/ThermoRawFileParser\" ga-product=\"biotool\" ga-id=\"ThermoRawFileParser\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/tophat\" ga-product=\"biotool\" ga-id=\"tophat\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/totalview\" ga-product=\"biotool\" ga-id=\"totalview\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/TransDecoder\" ga-product=\"biotool\" ga-id=\"transdecoder\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/transit\" ga-product=\"biotool\" ga-id=\"transit\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/TPP_SPC\" ga-product=\"biotool\" ga-id=\"tpp\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/trf\" ga-product=\"biotool\" ga-id=\"trf\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/trimal\" ga-product=\"biotool\" ga-id=\"trimal\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/trim_galore\" ga-product=\"biotool\" ga-id=\"trim_galore\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/trimmomatic\" ga-product=\"biotool\" ga-id=\"trimmomatic\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/trinity\" ga-product=\"biotool\" ga-id=\"trinity\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/trinotate\" ga-product=\"biotool\" ga-id=\"trinotate\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/trnascan-se\" ga-product=\"biotool\" ga-id=\"trnascan-se\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/trycycler\" ga-product=\"biotool\" ga-id=\"trycycler\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/UCSC_Genome_Browser_Utilities\" ga-product=\"biotool\" ga-id=\"ucsc\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/umi-tools\" ga-product=\"biotool\" ga-id=\"umi_tools\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/unicycler\" ga-product=\"biotool\" ga-id=\"unicycler\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/unipept\" ga-product=\"biotool\" ga-id=\"unipept\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,null,"<a class=\"biotools\" href=\"https://bio.tools/vep\" ga-product=\"biotool\" ga-id=\"ensembl-vep\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/varscan\" ga-product=\"biotool\" ga-id=\"varscan\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/vcflib\" ga-product=\"biotool\" ga-id=\"vcflib\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/vcftools\" ga-product=\"biotool\" ga-id=\"vcftools\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/velvet\" ga-product=\"biotool\" ga-id=\"velvet\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,null,"<a class=\"biotools\" href=\"https://bio.tools/vsearch\" ga-product=\"biotool\" ga-id=\"vsearch\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/whatshap\" ga-product=\"biotool\" ga-id=\"whatshap\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/winnowmap\" ga-product=\"biotool\" ga-id=\"winnowmap\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/workflow4metabolomics\" ga-product=\"biotool\" ga-id=\"workflow4metabolomics\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null,"<a class=\"biotools\" href=\"https://bio.tools/xcms\" ga-product=\"biotool\" ga-id=\"xcms\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/xpore\" ga-product=\"biotool\" ga-id=\"xpore\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>","<a class=\"biotools\" href=\"https://bio.tools/xtandem\" ga-product=\"biotool\" ga-id=\"xtandem\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,"<a class=\"biotools\" href=\"https://bio.tools/yahs\" ga-product=\"biotool\" ga-id=\"yahs\"><img src=\"./images/elixir_biotools_transparent.png\" style=\"width:50px;\"><\/a>",null,null],["abricate","abyss","aggregate_scores_in_intervals2","amber","anaconda","anndata","annotatemyids","antismash","any2fasta","apollo","aragorn","arriba","astral","augustus","autodock_vina","axt_to_concat_fasta","axt_to_fasta","axt_to_lav_1","bakta","bamtools","bamutil_diff","bandage","barchart_gnuplot","barrnap","bbmap","bcbiogff","bcftools","beagle","beagle-lib","beast","beast2","bed_to_bigBed","bed_to_protein_map","bed2gff1","bedtools","bellerophon","berokka","bftools","bio-searchio-hmmer","bio3d","biobakery_workflows","biobambam","biom-format","bionano_solve","bioperl","biopython","biotransformer","blast","blast+","blast2","blat","Blockbuster","BlockClust","boost","bowtie","bowtie2","bp_genbank2gff3","breseq","bundle_collections","busco","bwa","bwa-mem2","BWA-meth","bwakit","bx-python","c3s","cactus","camera","canu","cap3","cardinal","cat_bins","categorize_elements_satisfying_criteria","cd-hit","CellProfiler","cellranger","cellxgene","checkm","chembl","chemfp","chemical_data_sources","chipseeker","chromeister","circexplorer","circos","clifinder","climate_stripes","clinker","clustalo","clustalw","colabfold_batch","collapse_collections","collection_column_join","collection_element_identifiers","column_arrange_by_header","column_maker","compose_text_param","compress_file","compute_motif_frequencies_for_all_motifs","compute_motifs_frequency","concatenate_multiple_datasets","condense_characters","CONVERTER_neostorezip_to_neostore","correlation","Count1","cpat","ctsm_fates","cuda","cudnn","cufflinks","cummeRbund","customprodb","cut_include_exclude","cutadapt","cutesv","dada2","datamash","dbbuilder","dcm2niix","deepconsensus-cpu","deepconsensus-gpu","deeptools","dendropy","deseq2","DEXSeq","diann","diaumpire","diamond","diapysef","diffbind","draw_stacked_barplots","drishti","dropletutils","dwt","easybuild","ebi_sra_main","edger","edger-repenrich","eggnog-mapper","egsea","elastix","emboss","ena_upload","encyclopedia","ete3","ethercalc","eupathdb","exabayes","examl","exonerate","export_remote","eXpress","f5c","pb-assembly","fasta_compute_length","fasta_concatenate_by_species","fasta_filter_by_length","fasta_merge_files_and_filter_unique_sequences","fasta_stats","fasta_to_tabular","fastahack","fastani","fastool","fastp","fastqc","fastqe","fasttree","fastx","fastx_toolkit","fermi-lite","fgenesh","fgsea","filter_by_fasta_ids","filter_tabular","filter_transcripts_via_tracking","filtlong","flashlfq","flex","flye","flymine","foldseek","fpocket","fraggenescan","freebayes","fsom","funannotate","gaeval","galaxy_collection_operations","Galaxy CONVERTER","galaxy_filter_and_sort","galaxy_join_subtract_and_group","galaxy_text_manipulation","galaxy_sequence_utils","galaxy-ops","gatk","gblock","gcta","gecko","gemini","gemoma","gene2exon1","GeneBed_Maf_Fasta2","generate_count_matrix","generate_pc_lda_matrix","genomescope","genometools","genrich","get_pdb_file","getorganelle","gfa_to_fa","gfastats","gff2bed1","gff3sort","gffcompare","gffread","ggplot2","ghostscript","GMAJ","parallel","gsl","goenrichment","goseq","gramenemart","graphlan","gromacs","gtdb-tk","gtf2bedgraph","gtf2gene_list","gubbins","hapcut2","hbvar","heinz","hgv_david","hgv_ldtools","hgv_linkToGProfile","hicexplorer","hifiadapterfilt","hifiasm","hifiasm_meta","hisat2","histogram","hmmcleaner","hmmer","horovod","htseq","htslib","humann","humann2","HyPhy","idl","imagemagick","pbipa","infernal","insight-toolkit","igv","integron_finder","intermine","intermine_galaxy_exchange","interproscan","Interval_Maf_Merged_Fasta2","Interval2Maf_pairwise1","Interval2Maf1","ipyrad","iq-tree","isee","isescan","isoformswitchanalyzer","ivar","iwtomics","jags","jasminesv","jdk","jbigkit","jbrowse","jcvi","jellyfish","join_files_on_column_fuzzy","jq","jupyterlab","kallisto","kat","kentutils","khmer","kmc","kofamscan","kraken","kraken2","krona","kronatools","lama","lastz","lav_to_bed1","LFQ-Analyst","liftOver1","limma","links","lofreq","longshot","lpsolve","ltr_retriever","macs","macs2","maf_by_block_number1","MAF_filter","maf_limit_size1","MAF_Limit_To_Species1","MAF_Reverse_Complement_1","MAF_split_blocks_by_species1","maf_stats1","MAF_Thread_For_Species1","MAF_To_BED1","MAF_To_Fasta1","MAF_To_Interval1","mafft","MAGeCK","maker","maldi_quant","mash","master2pgSnp","masurca","matchms","matplotlib","maxbin","maxquant","biocontainers-maxquant","mcl","mdanalysis","MDTraj","medaka","megahit","merge_cols","merqury","meryl","metabat","metabolicmine","MetaDEGalaxy","metaeuk","metaphlan","metaQuantome","metaspades","metawrap","MethylDackel","metilene","MIGRATE","mimodd","minia","miniasm","miniconda","minimap2","miniprot","mircounts","mirdeep2","mitobim","mitos2","mlst","mmseqs2","mob-suite","modENCODEfly","modENCODEworm","modmine","moff","monailabel","morpheus","mothur","mousemine","mrbayes","msconvert","msmetaenhancer","msstats","multiqc","mummer","music_deconvolution","mztosqlite","nag","nanocompore","nanofilt","nanoplot","nanopolish","nanosv","natural_product_likeness_scorer","ncbi-datasets-cli","ncbi_acc_download","nccmp","ncl","nco","ncview","netcdf","newick_utils","nextclade","nextflow","ngsutils","ninja","NOVOPlasty","nseg","nvc","obitools","ont-fast5-api","openms","orad","pampa","pangolin","param_value_from_file","paraview","parse_mito_blast","pathview","pblat","pe_histogram","pear","PepPointer","peptideshaker","peptide_genomic_coordinate","percolator","perllib","pfamscan","phinch","php","phyloseq","phyml","picard","picrust","picrust2","pileup_interval","pileup_parser","pilon","pipe_t","PlasFlow","plink","plot_from_lda","plotly","pnetcdf","poisson2test","porechop","poretools","pplacer","pretext_map","pretextview","prodigal","prokka","prop_venn","pslcdnafilter","purge_dups","pybigwig","pycoqc","pygenometracks","pyprophet","pysam","pytorch","qiime2","qual_stats_boxplot","qualimap","quasitools","quast","query_tabular","r","r-raceid","racon","ramclustr","ratmine","raven","rawtools","raxml","rdkit","rdock","recon","recetox-aplcms","red","repeatmasker","repeatmodeler","repeatscout","repenrich","replace_column_with_key_value_file","rjags","rmassbank","rmats-turbo","rmblast","rnachipintegrator","roary","row_means","rsem","rseqc","rstudio","rtg-tools","rxdock","sailfish","salmon","salmon_kallisto_mtx_to_10x","salsa","sam_bitwise_flag_filter","sam2interval","sambamba","samblaster","sample_seqs","samtools","scanpy","scater","scikit-build","scikit-image","scikit","scipio","send_to_cloud","sepp","seq_filter_by_id","seq_select_by_id","seqkit","seqlib","seqtk","seurat","Sff_extractor","shasta","shovill","sicer","simtext","sip","sistr","slow5-guppy","slow5tools","smithwaterman","Smudgeplots","snakemake","snap","sniffles","snippy","snp_sites","snp-dists","snpeff","snpfreq","snpfreqplot","snpsift","soapdenovo2","sortmerna","spades","spectra","srf","split_file_to_collection","sqlite","sqlite_to_tabular","sra_source","sra-tools","ssw","stacks","star","staramr","star_fusion","stringtie","subread","Summary_Statistics1","superfocus","svim-asm","syri","t_test_two_samples","table_compute","tabular_to_fasta","targetfinder","tb_variant_filter","tbl2asn","tbprofiler","tbvcfreport","tensorflow","tesorter","text_processing","ThermoRawFileParser","tophat","totalview","trams","transdecoder","transit","translate_bed","tpp","treeshrink","trf","trimal","trim_galore","trimmomatic","trinity","trinotate","trnascan-se","trycycler","ucsc","ucsc_table_direct_archaea1","ucsc_table_direct1","umi_tools","unicycler","unipept","UniProt_Downloader","uniprot_rest_interface","unzip","ensembl-vep","varscan","vcf_to_maf_customtrack1","vcf2maf","vcflib","vcftools","velvet","verkko","visit","volcanoplot","vsearch","weblogo3","whatshap","wig_to_bigWig","wiggle2simple1","winnowmap","workflow4metabolomics","wormbase","xarray","xcms","xml4ena","xpore","xtandem","xy_plot","yahs","yeastmine","zebrafishmine"],["<p class=\"tags\">Genomics<\/p><p class=\"tags\">Microbiology<\/p>","<p class=\"tags\">Sequence assembly<\/p>",null,"<p class=\"tags\">Molecular modelling<\/p><p class=\"tags\">Computational biology<\/p>","<p class=\"tags\">DNA<\/p><p class=\"tags\">Gene expression<\/p>",null,null,"<p class=\"tags\">Molecular interactions, pathways and networks<\/p><p class=\"tags\">Gene and protein families<\/p>","<p class=\"tags\">Data quality management<\/p><p class=\"tags\">Genomics<\/p>","<p class=\"tags\">Data submission, annotation and curation<\/p><p class=\"tags\">Gene transcripts<\/p><p class=\"tags\">Genomics<\/p><p class=\"tags\">Mapping<\/p><p class=\"tags\">Molecular interactions, pathways and networks<\/p>","<p class=\"tags\">Functional, regulatory and non-coding RNA<\/p><p class=\"tags\">Genomics<\/p>","<p class=\"tags\">Gene transcripts<\/p><p class=\"tags\">RNA splicing<\/p><p class=\"tags\">DNA structural variation<\/p>","<p class=\"tags\">Phylogenetics<\/p><p class=\"tags\">Molecular genetics<\/p><p class=\"tags\">Statistics and probability<\/p>","<p class=\"tags\">Gene transcripts<\/p><p class=\"tags\">Gene and protein families<\/p>","<p class=\"tags\">Drug discovery<\/p><p class=\"tags\">Molecular modelling<\/p><p class=\"tags\">Molecular docking<\/p>",null,null,null,"<p class=\"tags\">Genomics<\/p><p class=\"tags\">Data submission, annotation and curation<\/p><p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Sequencing<\/p><p class=\"tags\">Data management<\/p><p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Data management<\/p>","<p class=\"tags\">Genomics<\/p><p class=\"tags\">Sequence assembly<\/p>",null,"<p class=\"tags\">Genomics<\/p><p class=\"tags\">Model organisms<\/p><p class=\"tags\">Model organisms<\/p>","<p class=\"tags\">Sequencing<\/p><p class=\"tags\">RNA splicing<\/p><p class=\"tags\">Whole genome sequencing<\/p><p class=\"tags\">Phylogenetics<\/p><p class=\"tags\">Metagenomics<\/p><p class=\"tags\">RNA-Seq<\/p>",null,"<p class=\"tags\">Genetic variation<\/p><p class=\"tags\">DNA polymorphism<\/p><p class=\"tags\">GWAS study<\/p><p class=\"tags\">Genotyping experiment<\/p>","<p class=\"tags\">Population genetics<\/p>","<p class=\"tags\">Phylogenetics<\/p><p class=\"tags\">Statistics and probability<\/p><p class=\"tags\">Computer science<\/p>","<p class=\"tags\">Phylogenetics<\/p><p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Phylogenetics<\/p><p class=\"tags\">Phylogenomics<\/p>",null,"<p class=\"tags\">Literature and language<\/p>",null,"<p class=\"tags\">Genomics<\/p>","<p class=\"tags\">Transcriptomics<\/p><p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">RNA-Seq<\/p><p class=\"tags\">Workflows<\/p>","<p class=\"tags\">Computer science<\/p>",null,null,"<p class=\"tags\">Structure analysis<\/p><p class=\"tags\">Gene and protein families<\/p><p class=\"tags\">Bioinformatics<\/p>",null,"<p class=\"tags\">Sequencing<\/p>","<p class=\"tags\">Laboratory information management<\/p><p class=\"tags\">Sequence analysis<\/p>",null,"<p class=\"tags\">Genomics<\/p><p class=\"tags\">Software engineering<\/p><p class=\"tags\">Data management<\/p>","<p class=\"tags\">Bioinformatics<\/p><p class=\"tags\">Data management<\/p><p class=\"tags\">Software engineering<\/p><p class=\"tags\">Computational biology<\/p>","<p class=\"tags\">Small molecules<\/p><p class=\"tags\">Endocrinology and metabolism<\/p><p class=\"tags\">Metabolomics<\/p><p class=\"tags\">Carbohydrates<\/p><p class=\"tags\">NMR<\/p>","<p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Bioinformatics<\/p>","<p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Bioinformatics<\/p>","<p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Bioinformatics<\/p>","<p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Genomics<\/p>",null,"<p class=\"tags\">Pure mathematics<\/p><p class=\"tags\">Imaging<\/p><p class=\"tags\">Software engineering<\/p>","<p class=\"tags\">Mapping<\/p><p class=\"tags\">Genomics<\/p><p class=\"tags\">Sequencing<\/p><p class=\"tags\">Sequence assembly<\/p>","<p class=\"tags\">Mapping<\/p><p class=\"tags\">Genomics<\/p><p class=\"tags\">Mapping<\/p>",null,"<p class=\"tags\">Sequencing<\/p><p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">DNA mutation<\/p>",null,"<p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Genomics<\/p><p class=\"tags\">Transcriptomics<\/p><p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Mapping<\/p>","<p class=\"tags\">Mapping<\/p>",null,null,"<p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Genetic variation<\/p><p class=\"tags\">Oncology<\/p><p class=\"tags\">Sequencing<\/p>","<p class=\"tags\">Genomics<\/p><p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Phylogeny<\/p><p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Mapping<\/p><p class=\"tags\">Phylogenetics<\/p>","<p class=\"tags\">Metabolomics<\/p>","<p class=\"tags\">Genomics<\/p>","<p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Sequencing<\/p><p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Genetic variation<\/p>","<p class=\"tags\">Proteomics<\/p><p class=\"tags\">Data visualisation<\/p>","<p class=\"tags\">Metagenomics<\/p><p class=\"tags\">Metagenomic sequencing<\/p><p class=\"tags\">Taxonomy<\/p><p class=\"tags\">Sequence assembly<\/p>",null,"<p class=\"tags\">Sequencing<\/p>","<p class=\"tags\">Imaging<\/p><p class=\"tags\">Microarray experiment<\/p><p class=\"tags\">Genotype and phenotype<\/p>",null,"<p class=\"tags\">Transcriptomics<\/p><p class=\"tags\">Cell biology<\/p>","<p class=\"tags\">Genomics<\/p><p class=\"tags\">Phylogenomics<\/p><p class=\"tags\">Phylogenetics<\/p><p class=\"tags\">Taxonomy<\/p><p class=\"tags\">Metagenomics<\/p>","<p class=\"tags\">Small molecules<\/p><p class=\"tags\">Chemical biology<\/p><p class=\"tags\">Cheminformatics<\/p>","<p class=\"tags\">Cheminformatics<\/p><p class=\"tags\">Chemistry<\/p><p class=\"tags\">Transcription factors and regulatory sites<\/p><p class=\"tags\">Literature and language<\/p>",null,"<p class=\"tags\">ChIP-seq<\/p><p class=\"tags\">Data visualisation<\/p><p class=\"tags\">Data mining<\/p>","<p class=\"tags\">Comparative genomics<\/p><p class=\"tags\">Sequence analysis<\/p>",null,"<p class=\"tags\">Comparative genomics<\/p><p class=\"tags\">Data visualisation<\/p>",null,null,"<p class=\"tags\">Molecular interactions, pathways and networks<\/p><p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Workflows<\/p><p class=\"tags\">Imaging<\/p>","<p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Protein folds and structural domains<\/p><p class=\"tags\">Protein folding, stability and design<\/p><p class=\"tags\">Structure prediction<\/p><p class=\"tags\">Sequence sites, features and motifs<\/p><p class=\"tags\">Metagenomics<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"tags\">Functional, regulatory and non-coding RNA<\/p><p class=\"tags\">Gene transcripts<\/p><p class=\"tags\">RNA-Seq<\/p><p class=\"tags\">Transcription factors and regulatory sites<\/p>",null,null,"<p class=\"tags\">Machine learning<\/p>","<p class=\"tags\">RNA splicing<\/p><p class=\"tags\">Transcriptomics<\/p><p class=\"tags\">RNA-Seq<\/p>","<p class=\"tags\">Sequencing<\/p>","<p class=\"tags\">Data integration and warehousing<\/p><p class=\"tags\">RNA-Seq<\/p><p class=\"tags\">Sequencing<\/p>",null,"<p class=\"tags\">Genomics<\/p><p class=\"tags\">Probes and primers<\/p><p class=\"tags\">Sequencing<\/p>","<p class=\"tags\">DNA structural variation<\/p><p class=\"tags\">Sequencing<\/p><p class=\"tags\">Computer science<\/p>","<p class=\"tags\">Sequencing<\/p><p class=\"tags\">Genetic variation<\/p><p class=\"tags\">Microbial ecology<\/p><p class=\"tags\">Metagenomics<\/p>",null,null,null,null,null,"<p class=\"tags\">Genomics<\/p><p class=\"tags\">ChIP-seq<\/p>","<p class=\"tags\">Phylogenetics<\/p><p class=\"tags\">Phylogenomics<\/p>","<p class=\"tags\">Transcriptomics<\/p>","<p class=\"tags\">RNA-Seq<\/p>","<p class=\"tags\">Proteomics<\/p><p class=\"tags\">Proteomics experiment<\/p><p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Machine learning<\/p><p class=\"tags\">Small molecules<\/p>","<p class=\"tags\">Proteomics<\/p><p class=\"tags\">Proteomics experiment<\/p>","<p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Proteins<\/p>","<p class=\"tags\">Proteomics experiment<\/p><p class=\"tags\">Proteomics<\/p><p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Sequencing<\/p><p class=\"tags\">Small molecules<\/p>","<p class=\"tags\">ChIP-seq<\/p>",null,null,"<p class=\"tags\">Gene expression<\/p><p class=\"tags\">RNA-seq<\/p><p class=\"tags\">Sequencing<\/p><p class=\"tags\">Transcriptomics<\/p>",null,"<p class=\"tags\">Computer science<\/p>",null,"<p class=\"tags\">Genetics<\/p><p class=\"tags\">RNA-Seq<\/p><p class=\"tags\">ChIP-seq<\/p>","<p class=\"tags\">Genetics<\/p><p class=\"tags\">RNA-Seq<\/p><p class=\"tags\">ChIP-seq<\/p>","<p class=\"tags\">Metagenomics<\/p><p class=\"tags\">Genomics<\/p>","<p class=\"tags\">Systems biology<\/p>","<p class=\"tags\">Medical imaging<\/p><p class=\"tags\">Oncology<\/p><p class=\"tags\">Model organisms<\/p>","<p class=\"tags\">Molecular biology<\/p><p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Biology<\/p>",null,"<p class=\"tags\">Proteomics<\/p><p class=\"tags\">Proteomics experiment<\/p>",null,null,"<p class=\"tags\">Model organisms<\/p><p class=\"tags\">Model organisms<\/p><p class=\"tags\">Zoology<\/p><p class=\"tags\">Gene transcripts<\/p>","<p class=\"tags\">Phylogenetics<\/p>","<p class=\"tags\">Phylogeny<\/p><p class=\"tags\">Phylogenomics<\/p><p class=\"tags\">Literature and language<\/p>","<p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Sequence sites, features and motifs<\/p><p class=\"tags\">Molecular interactions, pathways and networks<\/p>",null,"<p class=\"tags\">Gene expression<\/p><p class=\"tags\">Metagenomics<\/p><p class=\"tags\">RNA-seq<\/p><p class=\"tags\">ChIP-seq<\/p>",null,"<p class=\"tags\">Sequence assembly<\/p>",null,null,null,null,null,null,null,"<p class=\"tags\">Microbiology<\/p><p class=\"tags\">Genetic variation<\/p>","<p class=\"tags\">Data management<\/p>","<p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Probes and primers<\/p>","<p class=\"tags\">Sequencing<\/p><p class=\"tags\">Data quality management<\/p><p class=\"tags\">Sequence analysis<\/p>",null,"<p class=\"tags\">Phylogenetics<\/p><p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Sequence analysis<\/p>",null,"<p class=\"tags\">Gene structure<\/p>","<p class=\"tags\">Genetics<\/p>",null,null,null,null,"<p class=\"tags\">Proteomics experiment<\/p><p class=\"tags\">Proteomics<\/p>",null,"<p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Metagenomics<\/p><p class=\"tags\">Whole genome sequencing<\/p>",null,"<p class=\"tags\">Proteins<\/p>","<p class=\"tags\">Protein structural motifs and surfaces<\/p>","<p class=\"tags\">Genetics<\/p><p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Genomics<\/p><p class=\"tags\">Genetic variation<\/p><p class=\"tags\">Rare diseases<\/p>",null,"<p class=\"tags\">Genomics<\/p>","<p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Gene structure<\/p>",null,null,null,null,null,null,null,"<p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Genetic variation<\/p><p class=\"tags\">Sequencing<\/p><p class=\"tags\">Workflows<\/p>","<p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">GWAS study<\/p>","<p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Genomics<\/p><p class=\"tags\">Homology modeling<\/p>",null,null,null,null,"<p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Mathematics<\/p><p class=\"tags\">Mapping<\/p>","<p class=\"tags\">Nucleic acid sites, features and motifs<\/p><p class=\"tags\">Sequence analysis<\/p>",null,null,"<p class=\"tags\">Cell biology<\/p><p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Whole genome sequencing<\/p><p class=\"tags\">Plant biology<\/p><p class=\"tags\">Model organisms<\/p>",null,"<p class=\"tags\">Computational biology<\/p>",null,null,"<p class=\"tags\">Nucleic acids<\/p><p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Nucleic acids<\/p><p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Data visualisation<\/p>",null,null,null,null,"<p class=\"tags\">Transcriptomics<\/p>","<p class=\"tags\">RNA-seq<\/p>",null,"<p class=\"tags\">Metagenomics<\/p><p class=\"tags\">Phylogenetics<\/p><p class=\"tags\">Phylogenomics<\/p><p class=\"tags\">Cladistics<\/p>","<p class=\"tags\">Molecular dynamics<\/p>","<p class=\"tags\">Metagenomics<\/p><p class=\"tags\">Taxonomy<\/p><p class=\"tags\">Phylogenetics<\/p><p class=\"tags\">Database management<\/p><p class=\"tags\">Proteins<\/p>",null,null,"<p class=\"tags\">Phylogeny<\/p><p class=\"tags\">Genotype and phenotype<\/p><p class=\"tags\">Whole genome sequencing<\/p>",null,null,"<p class=\"tags\">Genetics<\/p><p class=\"tags\">Gene expression<\/p><p class=\"tags\">Molecular interactions, pathways and networks<\/p>","<p class=\"tags\">Ontology and terminology<\/p><p class=\"tags\">Gene expression<\/p>","<p class=\"tags\">Mapping<\/p><p class=\"tags\">Molecular interactions, pathways and networks<\/p><p class=\"tags\">DNA polymorphism<\/p><p class=\"tags\">Data mining<\/p>","<p class=\"tags\">Ontology and terminology<\/p><p class=\"tags\">Function analysis<\/p><p class=\"tags\">Data mining<\/p>","<p class=\"tags\">Workflows<\/p><p class=\"tags\">DNA<\/p><p class=\"tags\">Chromosome conformation capture<\/p><p class=\"tags\">ChIP-seq<\/p><p class=\"tags\">Epigenetics<\/p>","<p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Agricultural science<\/p><p class=\"tags\">Workflows<\/p><p class=\"tags\">Whole genome sequencing<\/p>","<p class=\"tags\">Sequence assembly<\/p>",null,"<p class=\"tags\">RNA-seq<\/p>",null,null,"<p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Sequence sites, features and motifs<\/p><p class=\"tags\">Gene and protein families<\/p>","<p class=\"tags\">Machine learning<\/p>","<p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Data management<\/p>","<p class=\"tags\">Metagenomics<\/p><p class=\"tags\">Phylogenomics<\/p>",null,"<p class=\"tags\">Phylogeny<\/p><p class=\"tags\">Small molecules<\/p><p class=\"tags\">Molecular interactions, pathways and networks<\/p>",null,null,"<p class=\"tags\">Sequence assembly<\/p>","<p class=\"tags\">Sequence sites, features and motifs<\/p>",null,"<p class=\"tags\">Genomics<\/p><p class=\"tags\">Data visualisation<\/p><p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Functional genomics<\/p><p class=\"tags\">Mobile genetic elements<\/p><p class=\"tags\">Molecular biology<\/p><p class=\"tags\">Sequence analysis<\/p>",null,null,"<p class=\"tags\">Gene and protein families<\/p><p class=\"tags\">Sequence analysis<\/p>",null,null,null,"<p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Phylogenetics<\/p><p class=\"tags\">Population genetics<\/p>","<p class=\"tags\">Phylogenetics<\/p><p class=\"tags\">Statistics and probability<\/p><p class=\"tags\">Phylogenomics<\/p>","<p class=\"tags\">Gene expression<\/p><p class=\"tags\">Transcriptomics<\/p>","<p class=\"tags\">Genomics<\/p><p class=\"tags\">DNA structural variation<\/p><p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Genetic variation<\/p>","<p class=\"tags\">Computational biology<\/p><p class=\"tags\">Gene transcripts<\/p>","<p class=\"tags\">Biological databases<\/p><p class=\"tags\">Evolutionary biology<\/p><p class=\"tags\">Sequencing<\/p>","<p class=\"tags\">Statistics and probability<\/p>","<p class=\"tags\">Statistics and probability<\/p>","<p class=\"tags\">Structural variation<\/p><p class=\"tags\">DNA polymorphism<\/p><p class=\"tags\">Workflows<\/p><p class=\"tags\">Mapping<\/p><p class=\"tags\">RNA-Seq<\/p>",null,null,"<p class=\"tags\">Genomics<\/p>",null,"<p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Genomics<\/p>",null,null,null,"<p class=\"tags\">Transcriptomics<\/p><p class=\"tags\">RNA-seq<\/p><p class=\"tags\">Gene expression<\/p>","<p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Genomics<\/p>",null,"<p class=\"tags\">Sequence assembly<\/p>","<p class=\"tags\">Whole genome sequencing<\/p><p class=\"tags\">Genomics<\/p><p class=\"tags\">Sequence assembly<\/p>",null,"<p class=\"tags\">Taxonomy<\/p><p class=\"tags\">Metagenomics<\/p>","<p class=\"tags\">Taxonomy<\/p><p class=\"tags\">Metagenomics<\/p>","<p class=\"tags\">Metagenomics<\/p>",null,"<p class=\"tags\">Tomography<\/p><p class=\"tags\">Genotype and phenotype<\/p><p class=\"tags\">MRI<\/p><p class=\"tags\">Workflows<\/p>","<p class=\"tags\">Genomics<\/p>",null,"<p class=\"tags\">Proteomics<\/p><p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Small molecules<\/p><p class=\"tags\">Metabolomics<\/p><p class=\"tags\">Proteomics experiment<\/p>",null,"<p class=\"tags\">Molecular biology<\/p><p class=\"tags\">Genetics<\/p>","<p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Mapping<\/p><p class=\"tags\">Sequencing<\/p>","<p class=\"tags\">Genomics<\/p><p class=\"tags\">Genetic variation<\/p>","<p class=\"tags\">Genetic variation<\/p>",null,"<p class=\"tags\">Sequence assembly<\/p>","<p class=\"tags\">ChIP-seq<\/p><p class=\"tags\">Molecular interactions, pathways and networks<\/p><p class=\"tags\">Transcription factors and regulatory sites<\/p>","<p class=\"tags\">ChIP-seq<\/p><p class=\"tags\">Molecular interactions, pathways and networks<\/p><p class=\"tags\">Transcription factors and regulatory sites<\/p>",null,null,null,null,null,null,null,null,null,null,null,"<p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Genetics<\/p><p class=\"tags\">Genetic variation<\/p><p class=\"tags\">Genomics<\/p>","<p class=\"tags\">Genomics<\/p><p class=\"tags\">DNA<\/p><p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Proteomics experiment<\/p><p class=\"tags\">Proteomics<\/p>","<p class=\"tags\">Genomics<\/p><p class=\"tags\">Metagenomics<\/p><p class=\"tags\">Statistics and probability<\/p><p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">DNA mutation<\/p>",null,"<p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Whole genome sequencing<\/p><p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Metabolomics<\/p>",null,"<p class=\"tags\">Metagenomics<\/p><p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Microbiology<\/p>","<p class=\"tags\">Proteomics experiment<\/p><p class=\"tags\">Proteomics<\/p><p class=\"tags\">Statistics and probability<\/p>","<p class=\"tags\">Proteomics experiment<\/p><p class=\"tags\">Proteomics<\/p><p class=\"tags\">Statistics and probability<\/p>","<p class=\"tags\">Molecular interactions, pathways and networks<\/p><p class=\"tags\">Computer science<\/p>","<p class=\"tags\">Molecular dynamics<\/p>",null,"<p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Machine learning<\/p>","<p class=\"tags\">Metagenomics<\/p><p class=\"tags\">Sequencing<\/p><p class=\"tags\">Ecology<\/p><p class=\"tags\">Sequence assembly<\/p>",null,"<p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Whole genome sequencing<\/p><p class=\"tags\">Plant biology<\/p>","<p class=\"tags\">Whole genome sequencing<\/p><p class=\"tags\">Genomics<\/p><p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Metagenomics<\/p><p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Metagenomic sequencing<\/p>",null,"<p class=\"tags\">Metagenomics<\/p><p class=\"tags\">Metagenomic sequencing<\/p><p class=\"tags\">Workflows<\/p><p class=\"tags\">Microbial ecology<\/p><p class=\"tags\">RNA-Seq<\/p>","<p class=\"tags\">Metagenomics<\/p><p class=\"tags\">Gene and protein families<\/p>","<p class=\"tags\">Metagenomics<\/p><p class=\"tags\">Phylogenomics<\/p>","<p class=\"tags\">Proteomics<\/p><p class=\"tags\">Metatranscriptomics<\/p><p class=\"tags\">Microbial ecology<\/p><p class=\"tags\">Proteomics experiment<\/p><p class=\"tags\">Metagenomics<\/p>","<p class=\"tags\">Metagenomics<\/p><p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Whole genome sequencing<\/p><p class=\"tags\">Metagenomic sequencing<\/p><p class=\"tags\">Metagenomics<\/p>","<p class=\"tags\">Epigenomics<\/p>",null,"<p class=\"tags\">Population genetics<\/p><p class=\"tags\">Nucleic acid structure analysis<\/p><p class=\"tags\">Zoology<\/p>",null,"<p class=\"tags\">Sequence assembly<\/p>","<p class=\"tags\">Genomics<\/p><p class=\"tags\">Sequence assembly<\/p>",null,"<p class=\"tags\">Mapping<\/p>","<p class=\"tags\">Sequence sites, features and motifs<\/p><p class=\"tags\">Sequence analysis<\/p>",null,"<p class=\"tags\">Functional, regulatory and non-coding RNA<\/p><p class=\"tags\">RNA immunoprecipitation<\/p>",null,"<p class=\"tags\">Zoology<\/p><p class=\"tags\">Whole genome sequencing<\/p>","<p class=\"tags\">Immunoproteins, genes and antigens<\/p>","<p class=\"tags\">Metagenomics<\/p><p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Proteins<\/p><p class=\"tags\">Nucleic acids<\/p><p class=\"tags\">Gene and protein families<\/p><p class=\"tags\">Taxonomy<\/p>","<p class=\"tags\">Mobile genetic elements<\/p><p class=\"tags\">Whole genome sequencing<\/p><p class=\"tags\">Public health and epidemiology<\/p><p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Biomarkers<\/p>",null,null,null,"<p class=\"tags\">Proteomics<\/p>",null,"<p class=\"tags\">Proteomics<\/p>","<p class=\"tags\">Microbial ecology<\/p>","<p class=\"tags\">Model organisms<\/p><p class=\"tags\">Genotype and phenotype<\/p>","<p class=\"tags\">Phylogenetics<\/p>","<p class=\"tags\">Proteomics<\/p><p class=\"tags\">Proteomics experiment<\/p>","<p class=\"tags\">Metabolomics<\/p><p class=\"tags\">Compound libraries and screening<\/p><p class=\"tags\">Data submission, annotation and curation<\/p>","<p class=\"tags\">Proteomics<\/p><p class=\"tags\">Proteomics experiment<\/p><p class=\"tags\">Statistics and probability<\/p><p class=\"tags\">Workflows<\/p>","<p class=\"tags\">Sequencing<\/p><p class=\"tags\">Bioinformatics<\/p>","<p class=\"tags\">Genomics<\/p><p class=\"tags\">Mapping<\/p><p class=\"tags\">Sequencing<\/p>",null,null,null,"<p class=\"tags\">Functional, regulatory and non-coding RNA<\/p><p class=\"tags\">RNA-Seq<\/p><p class=\"tags\">Gene transcripts<\/p><p class=\"tags\">Transcriptomics<\/p><p class=\"tags\">Transcription factors and regulatory sites<\/p>",null,"<p class=\"tags\">Genomics<\/p>","<p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Sequencing<\/p><p class=\"tags\">Genetic variation<\/p><p class=\"tags\">Statistics and probability<\/p>",null,null,"<p class=\"tags\">Biological databases<\/p>",null,null,null,null,null,null,"<p class=\"tags\">Phylogeny<\/p><p class=\"tags\">Genomics<\/p><p class=\"tags\">Computer science<\/p>",null,"<p class=\"tags\">Software engineering<\/p><p class=\"tags\">Workflows<\/p><p class=\"tags\">Bioinformatics<\/p><p class=\"tags\">Genomics<\/p>","<p class=\"tags\">Genomics<\/p><p class=\"tags\">Transcriptomics<\/p>","<p class=\"tags\">Phylogeny<\/p>",null,null,null,"<p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">DNA<\/p><p class=\"tags\">Sequencing<\/p>",null,"<p class=\"tags\">Proteomics experiment<\/p><p class=\"tags\">Proteomics<\/p><p class=\"tags\">Metabolomics<\/p>",null,null,"<p class=\"tags\">RNA splicing<\/p><p class=\"tags\">Genetic variation<\/p><p class=\"tags\">Gene transcripts<\/p><p class=\"tags\">RNA-Seq<\/p><p class=\"tags\">Exome sequencing<\/p>",null,"<p class=\"tags\">Data visualisation<\/p><p class=\"tags\">Imaging<\/p>",null,"<p class=\"tags\">Molecular interactions, pathways and networks<\/p><p class=\"tags\">Systems biology<\/p><p class=\"tags\">Data visualisation<\/p>","<p class=\"tags\">RNA-seq<\/p><p class=\"tags\">Mapping<\/p><p class=\"tags\">Gene transcripts<\/p>","<p class=\"tags\">ChIP-seq<\/p><p class=\"tags\">RNA-Seq<\/p><p class=\"tags\">Epigenetics<\/p><p class=\"tags\">Methylated DNA immunoprecipitation<\/p><p class=\"tags\">Transcription factors and regulatory sites<\/p>","<p class=\"tags\">Sequence assembly<\/p>",null,"<p class=\"tags\">Proteomics<\/p><p class=\"tags\">Proteomics experiment<\/p><p class=\"tags\">Proteome<\/p>",null,"<p class=\"tags\">Proteomics<\/p><p class=\"tags\">Proteomics experiment<\/p><p class=\"tags\">Statistics and probability<\/p><p class=\"tags\">Machine learning<\/p>",null,"<p class=\"tags\">Sequence analysis<\/p>",null,null,"<p class=\"tags\">Microbiology<\/p><p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Metagenomics<\/p>","<p class=\"tags\">Phylogenetics<\/p><p class=\"tags\">Bioinformatics<\/p><p class=\"tags\">Phylogenetics<\/p>","<p class=\"tags\">Sequencing<\/p><p class=\"tags\">Data management<\/p>","<p class=\"tags\">Metagenomics<\/p><p class=\"tags\">Microbial ecology<\/p><p class=\"tags\">Functional, regulatory and non-coding RNA<\/p><p class=\"tags\">Metagenomic sequencing<\/p>","<p class=\"tags\">Metagenomics<\/p><p class=\"tags\">Microbiology<\/p><p class=\"tags\">Phylogenetics<\/p><p class=\"tags\">Metagenomic sequencing<\/p>",null,null,"<p class=\"tags\">Assembly<\/p>","<p class=\"tags\">PCR experiment<\/p><p class=\"tags\">Transcriptomics<\/p><p class=\"tags\">Biomarkers<\/p><p class=\"tags\">Microarray experiment<\/p>",null,"<p class=\"tags\">GWAS study<\/p>",null,null,"<p class=\"tags\">Computer science<\/p>",null,null,"<p class=\"tags\">DNA<\/p><p class=\"tags\">Sequencing<\/p>","<p class=\"tags\">Taxonomy<\/p>",null,"<p class=\"tags\">Mapping<\/p>","<p class=\"tags\">DNA<\/p><p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Nucleic acid sites, features and motifs<\/p>","<p class=\"tags\">Genomics<\/p><p class=\"tags\">Model organisms<\/p><p class=\"tags\">Virology<\/p>",null,null,"<p class=\"tags\">Sequence assembly<\/p>",null,"<p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Data quality management<\/p><p class=\"tags\">Sequencing<\/p>","<p class=\"tags\">Model organisms<\/p><p class=\"tags\">Imaging<\/p><p class=\"tags\">Workflows<\/p>",null,"<p class=\"tags\">Sequencing<\/p><p class=\"tags\">Mapping<\/p>","<p class=\"tags\">Machine learning<\/p><p class=\"tags\">Computer science<\/p>","<p class=\"tags\">Microbial ecology<\/p><p class=\"tags\">Phylogeny<\/p><p class=\"tags\">Metatranscriptomics<\/p><p class=\"tags\">Metagenomics<\/p>",null,"<p class=\"tags\">Data quality management<\/p>","<p class=\"tags\">Virology<\/p><p class=\"tags\">Small molecules<\/p><p class=\"tags\">Sequencing<\/p>","<p class=\"tags\">Sequence assembly<\/p>","<p class=\"tags\">Bioinformatics<\/p><p class=\"tags\">Workflows<\/p>","<p class=\"tags\">Mathematics<\/p><p class=\"tags\">Data architecture, analysis and design<\/p><p class=\"tags\">Statistics and probability<\/p>",null,"<p class=\"tags\">Whole genome sequencing<\/p><p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Plant biology<\/p>","<p class=\"tags\">Metabolomics<\/p>",null,"<p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Whole genome sequencing<\/p><p class=\"tags\">Genomics<\/p>","<p class=\"tags\">Proteomics experiment<\/p><p class=\"tags\">Proteomics<\/p>","<p class=\"tags\">Phylogenetics<\/p><p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Cheminformatics<\/p><p class=\"tags\">Medicinal chemistry<\/p><p class=\"tags\">Molecular biology<\/p>",null,"<p class=\"tags\">DNA<\/p><p class=\"tags\">DNA binding sites<\/p><p class=\"tags\">Statistics and probability<\/p><p class=\"tags\">DNA packaging<\/p><p class=\"tags\">Transcription factors and regulatory sites<\/p>","<p class=\"tags\">Metabolomics<\/p>","<p class=\"tags\">RNA<\/p><p class=\"tags\">Sequencing<\/p><p class=\"tags\">Data visualisation<\/p>","<p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Sequence composition, complexity and repeats<\/p>","<p class=\"tags\">Sequence composition, complexity and repeats<\/p><p class=\"tags\">Sequence composition, complexity and repeats<\/p>","<p class=\"tags\">DNA<\/p>",null,null,"<p class=\"tags\">Statistics and probability<\/p>","<p class=\"tags\">Metabolomics<\/p>",null,"<p class=\"tags\">Genomics<\/p>",null,"<p class=\"tags\">DNA<\/p><p class=\"tags\">Genomics<\/p><p class=\"tags\">Mapping<\/p>",null,"<p class=\"tags\">RNA-Seq<\/p>","<p class=\"tags\">Sequencing<\/p>","<p class=\"tags\">Mathematics<\/p><p class=\"tags\">Data architecture, analysis and design<\/p><p class=\"tags\">Statistics and probability<\/p>","<p class=\"tags\">Genomics<\/p>",null,"<p class=\"tags\">RNA-Seq<\/p><p class=\"tags\">RNA<\/p><p class=\"tags\">Sequencing<\/p><p class=\"tags\">Transcriptomics<\/p>","<p class=\"tags\">RNA-Seq<\/p><p class=\"tags\">Gene expression<\/p><p class=\"tags\">Transcriptomics<\/p>",null,"<p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">DNA binding sites<\/p><p class=\"tags\">Mapping<\/p>",null,null,"<p class=\"tags\">DNA<\/p><p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">RNA-Seq<\/p>","<p class=\"tags\">DNA<\/p><p class=\"tags\">Sequencing<\/p><p class=\"tags\">Mapping<\/p>",null,"<p class=\"tags\">Mapping<\/p><p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Sequencing<\/p><p class=\"tags\">Rare diseases<\/p>","<p class=\"tags\">Gene expression<\/p><p class=\"tags\">Cell biology<\/p><p class=\"tags\">Genetics<\/p>","<p class=\"tags\">RNA-seq<\/p><p class=\"tags\">Quality affairs<\/p><p class=\"tags\">Molecular genetics<\/p>",null,"<p class=\"tags\">Imaging<\/p><p class=\"tags\">Software engineering<\/p><p class=\"tags\">Literature and language<\/p>",null,null,null,"<p class=\"tags\">Phylogenetics<\/p>",null,null,"<p class=\"tags\">Database management<\/p><p class=\"tags\">Sequence analysis<\/p>",null,"<p class=\"tags\">Data management<\/p>","<p class=\"tags\">RNA-Seq<\/p><p class=\"tags\">Transcriptomics<\/p>",null,"<p class=\"tags\">Genomics<\/p><p class=\"tags\">Sequence assembly<\/p>","<p class=\"tags\">Genomics<\/p><p class=\"tags\">Microbiology<\/p><p class=\"tags\">Sequence assembly<\/p>","<p class=\"tags\">ChIP-seq<\/p><p class=\"tags\">Epigenomics<\/p>","<p class=\"tags\">Natural language processing<\/p><p class=\"tags\">Workflows<\/p><p class=\"tags\">Machine learning<\/p><p class=\"tags\">Molecular interactions, pathways and networks<\/p>",null,"<p class=\"tags\">Whole genome sequencing<\/p><p class=\"tags\">Immunoproteins and antigens<\/p><p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Zoology<\/p><p class=\"tags\">Public health and epidemiology<\/p>",null,null,null,"<p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Genetic variation<\/p><p class=\"tags\">Mathematics<\/p>","<p class=\"tags\">Bioinformatics<\/p><p class=\"tags\">Workflows<\/p><p class=\"tags\">Software engineering<\/p>","<p class=\"tags\">DNA<\/p><p class=\"tags\">DNA polymorphism<\/p><p class=\"tags\">Genetics<\/p>","<p class=\"tags\">DNA structural variation<\/p><p class=\"tags\">Sequencing<\/p>","<p class=\"tags\">Genomics<\/p><p class=\"tags\">Model organisms<\/p><p class=\"tags\">DNA polymorphism<\/p><p class=\"tags\">Phylogenetics<\/p>",null,null,"<p class=\"tags\">DNA polymorphism<\/p><p class=\"tags\">Genetic variation<\/p><p class=\"tags\">Nucleic acid sites, features and motifs<\/p>",null,null,"<p class=\"tags\">Data management<\/p><p class=\"tags\">Data architecture, analysis and design<\/p>","<p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Sequencing<\/p><p class=\"tags\">Metagenomics<\/p>","<p class=\"tags\">Metatranscriptomics<\/p><p class=\"tags\">Metagenomics<\/p>","<p class=\"tags\">Sequence assembly<\/p>",null,"<p class=\"tags\">Computational biology<\/p>",null,null,null,null,"<p class=\"tags\">DNA<\/p><p class=\"tags\">Genomics<\/p><p class=\"tags\">Sequencing<\/p>","<p class=\"tags\">Sample collections<\/p><p class=\"tags\">Genomics<\/p><p class=\"tags\">Data governance<\/p><p class=\"tags\">Nucleic acid structure analysis<\/p>","<p class=\"tags\">Mapping<\/p><p class=\"tags\">Population genetics<\/p>","<p class=\"tags\">RNA-Seq<\/p><p class=\"tags\">Transcriptomics<\/p>",null,"<p class=\"tags\">Gene structure<\/p><p class=\"tags\">RNA-Seq<\/p>","<p class=\"tags\">Transcriptomics<\/p><p class=\"tags\">RNA-seq<\/p>","<p class=\"tags\">Sequencing<\/p><p class=\"tags\">Mapping<\/p><p class=\"tags\">Whole genome sequencing<\/p><p class=\"tags\">RNA-Seq<\/p>",null,"<p class=\"tags\">DNA<\/p><p class=\"tags\">Metagenomics<\/p><p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Genotype and phenotype<\/p><p class=\"tags\">Sequencing<\/p><p class=\"tags\">DNA polymorphism<\/p>","<p class=\"tags\">DNA polymorphism<\/p><p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">DNA structural variation<\/p><p class=\"tags\">Mapping<\/p><p class=\"tags\">Sequencing<\/p>",null,null,null,"<p class=\"tags\">Gene expression<\/p><p class=\"tags\">Transcription factors and regulatory sites<\/p><p class=\"tags\">Gene transcripts<\/p><p class=\"tags\">Small molecules<\/p><p class=\"tags\">Gene regulation<\/p>",null,null,null,null,"<p class=\"tags\">Machine learning<\/p><p class=\"tags\">Pathology<\/p><p class=\"tags\">Biomarkers<\/p>","<p class=\"tags\">Protein folds and structural domains<\/p><p class=\"tags\">Phylogenetics<\/p><p class=\"tags\">Transcription factors and regulatory sites<\/p><p class=\"tags\">Plant biology<\/p><p class=\"tags\">Model organisms<\/p>",null,"<p class=\"tags\">Proteomics<\/p><p class=\"tags\">Proteomics experiment<\/p><p class=\"tags\">Metabolomics<\/p>","<p class=\"tags\">RNA-Seq<\/p><p class=\"tags\">Mapping<\/p><p class=\"tags\">Rare diseases<\/p>","<p class=\"tags\">Computer science<\/p>",null,"<p class=\"tags\">Genomics<\/p><p class=\"tags\">Gene transcripts<\/p><p class=\"tags\">RNA-Seq<\/p><p class=\"tags\">Gene expression<\/p><p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">Whole genome sequencing<\/p>","<p class=\"tags\">DNA<\/p><p class=\"tags\">Sequencing<\/p><p class=\"tags\">Mobile genetic elements<\/p>",null,"<p class=\"tags\">Proteomics<\/p><p class=\"tags\">Proteomics experiment<\/p>",null,"<p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Sequence composition, complexity and repeats<\/p>","<p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Sequencing<\/p><p class=\"tags\">Sequence sites, features and motifs<\/p>","<p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Sequencing<\/p><p class=\"tags\">Sequence analysis<\/p><p class=\"tags\">Data quality management<\/p>","<p class=\"tags\">Transcriptomics<\/p><p class=\"tags\">Gene expression<\/p><p class=\"tags\">Gene transcripts<\/p>","<p class=\"tags\">Gene expression<\/p><p class=\"tags\">Transcriptomics<\/p>","<p class=\"tags\">RNA<\/p><p class=\"tags\">Genomics<\/p><p class=\"tags\">Sequence analysis<\/p>","<p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">DNA<\/p><p class=\"tags\">Human biology<\/p><p class=\"tags\">Whole genome sequencing<\/p><p class=\"tags\">Genomics<\/p>","<p class=\"tags\">Sequence analysis<\/p>",null,null,"<p class=\"tags\">NGS<\/p><p class=\"tags\">Sequence sites, features and motifs<\/p><p class=\"tags\">Quality affairs<\/p>","<p class=\"tags\">Microbiology<\/p><p class=\"tags\">Genomics<\/p><p class=\"tags\">Sequencing<\/p><p class=\"tags\">Sequence assembly<\/p>","<p class=\"tags\">Proteomics<\/p><p class=\"tags\">Proteogenomics<\/p><p class=\"tags\">Biodiversity<\/p><p class=\"tags\">Workflows<\/p>",null,null,null,"<p class=\"tags\">Genetic variation<\/p><p class=\"tags\">Genomics<\/p>","<p class=\"tags\">Sequencing<\/p><p class=\"tags\">DNA polymorphism<\/p>",null,null,"<p class=\"tags\">Genomics<\/p><p class=\"tags\">Data management<\/p>","<p class=\"tags\">Genetic variation<\/p>","<p class=\"tags\">Sequence assembly<\/p>",null,null,null,"<p class=\"tags\">Metagenomics<\/p><p class=\"tags\">Sequence analysis<\/p>",null,"<p class=\"tags\">DNA polymorphism<\/p><p class=\"tags\">Sequence assembly<\/p>",null,null,"<p class=\"tags\">Mapping<\/p>","<p class=\"tags\">Systems biology<\/p><p class=\"tags\">Metabolomics<\/p><p class=\"tags\">Computational biology<\/p>",null,null,"<p class=\"tags\">Biological imaging<\/p><p class=\"tags\">Data visualisation<\/p><p class=\"tags\">Metabolomics<\/p>",null,"<p class=\"tags\">RNA-Seq<\/p><p class=\"tags\">RNA<\/p><p class=\"tags\">Gene expression<\/p><p class=\"tags\">Genotype and phenotype<\/p><p class=\"tags\">Molecular interactions, pathways and networks<\/p>","<p class=\"tags\">Proteomics<\/p><p class=\"tags\">Proteomics experiment<\/p>",null,"<p class=\"tags\">Sequence assembly<\/p><p class=\"tags\">DNA binding sites<\/p>",null,null],[null,"<a class=\"publication-title\" href=\"https://doi.org/10.1101/gr.089532.108\" ga-product=\"publication\" ga-id=\"abyss\"\">ABySS: A parallel assembler for short read sequence data<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1101/gr.214346.116\" ga-product=\"publication\" ga-id=\"abyss\"\">ABySS 2.0: Resource-efficient assembly of large genomes using a Bloom filter<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1002/jcc.20290\" ga-product=\"publication\" ga-id=\"amber\"\">The Amber biomolecular simulation programs<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1002/wcms.1121\" ga-product=\"publication\" ga-id=\"amber\"\">An overview of the Amber biomolecular simulation package<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/16538282\" ga-product=\"publication\" ga-id=\"anaconda\">Statistical, computational and visualization methodologies to unveil gene primary structure features<\/a>",null,null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/21672958\" ga-product=\"publication\" ga-id=\"antismash\">AntiSMASH: Rapid identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genome sequences<\/a><br \\><a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/23737449\" ga-product=\"publication\" ga-id=\"antismash\">antiSMASH 2.0--a versatile platform for genome mining of secondary metabolite producers.<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkv437\" ga-product=\"publication\" ga-id=\"antismash\"\">AntiSMASH 3.0-A comprehensive resource for the genome mining of biosynthetic gene clusters<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/NAR/GKX319\" ga-product=\"publication\" ga-id=\"antismash\"\">AntiSMASH 4.0 - improvements in chemistry prediction and gene cluster boundary identification<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/NAR/GKAA978\" ga-product=\"publication\" ga-id=\"antismash\"\">The antiSMASH database version 3: Increased taxonomic coverage and new query features for modular enzymes<\/a>",null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/12537571\" ga-product=\"publication\" ga-id=\"apollo\">Apollo: a sequence annotation editor.<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkh152\" ga-product=\"publication\" ga-id=\"aragorn\"\">ARAGORN, a program to detect tRNA genes and tmRNA genes in nucleotide sequences<\/a>",null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/26072508\" ga-product=\"publication\" ga-id=\"astral\">ASTRAL-II: Coalescent-based species tree estimation with many hundreds of taxa and thousands of genes<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btu462\" ga-product=\"publication\" ga-id=\"astral\"\">ASTRAL: Genome-scale coalescent-based species tree estimation<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1186/s12859-018-2129-y\" ga-product=\"publication\" ga-id=\"astral\"\">ASTRAL-III: Polynomial time species tree reconstruction from partially resolved gene trees<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btg1080\" ga-product=\"publication\" ga-id=\"augustus\"\">Gene prediction with a hidden Markov model and a new intron submodel<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1186/1471-2105-7-62\" ga-product=\"publication\" ga-id=\"augustus\"\">Gene prediction in eukaryotes with a generalized hidden Markov model that uses hints from external sources<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1186/gb-2006-7-s1-s11\" ga-product=\"publication\" ga-id=\"augustus\"\">DOI:10.1186/gb-2006-7-s1-s11<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btn013\" ga-product=\"publication\" ga-id=\"augustus\"\">Using native and syntenically mapped cDNA alignments to improve de novo gene finding<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btr010\" ga-product=\"publication\" ga-id=\"augustus\"\">A novel hybrid gene prediction method employing protein multiple sequence alignments<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btw494\" ga-product=\"publication\" ga-id=\"augustus\"\">Simultaneous gene finding in multiple genomes<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1002/cpbi.57\" ga-product=\"publication\" ga-id=\"augustus\"\">Predicting Genes in Single Genomes with AUGUSTUS<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1007/978-1-4939-9173-0_5\" ga-product=\"publication\" ga-id=\"augustus\"\">Whole-genome annotation with BRAKER<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1007/978-1-4939-9173-0_8\" ga-product=\"publication\" ga-id=\"augustus\"\">Multi-genome annotation with AUGUSTUS<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1002/jcc.21334\" ga-product=\"publication\" ga-id=\"autodock_vina\"\">DOI:10.1002/jcc.21334<\/a>",null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1099/mgen.0.000685\" ga-product=\"publication\" ga-id=\"bakta\"\">Bakta: Rapid and standardized annotation of bacterial genomes via alignment-free sequence identification<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/21493652\" ga-product=\"publication\" ga-id=\"bamtools\">Bamtools: A C++ API and toolkit for analyzing and managing BAM files<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1101/gr.176552.114\" ga-product=\"publication\" ga-id=\"bamutil_diff\"\">An efficient and scalable analysis framework for variant extraction and refinement from population-scale DNA sequence data<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btv383\" ga-product=\"publication\" ga-id=\"bandage\"\">Bandage: Interactive visualization of de novo genome assemblies<\/a>",null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btp163\" ga-product=\"publication\" ga-id=\"bcbiogff\"\">Biopython: Freely available Python tools for computational molecular biology and bioinformatics<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btp352\" ga-product=\"publication\" ga-id=\"bcftools\"\">The Sequence Alignment/Map format and SAMtools<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/gigascience/giab008\" ga-product=\"publication\" ga-id=\"bcftools\"\">Twelve years of SAMtools and BCFtools<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1086/521987\" ga-product=\"publication\" ga-id=\"beagle\"\">Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1016/j.ajhg.2015.11.020\" ga-product=\"publication\" ga-id=\"beagle\"\">Genotype Imputation with Millions of Reference Samples<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1016/j.ajhg.2021.08.005\" ga-product=\"publication\" ga-id=\"beagle\"\">Fast two-stage phasing of large-scale sequence data<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1016/j.ajhg.2018.07.015\" ga-product=\"publication\" ga-id=\"beagle\"\">A One-Penny Imputed Genome from Next-Generation Reference Panels<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/sysbio/syr100\" ga-product=\"publication\" ga-id=\"beagle-lib\"\">BEAGLE: An application programming interface and high-performance computing library for statistical phylogenetics<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/molbev/mss075\" ga-product=\"publication\" ga-id=\"beast\"\">Bayesian phylogenetics with BEAUti and the BEAST 1.7<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/ve/vey016\" ga-product=\"publication\" ga-id=\"beast\"\">DOI:10.1093/ve/vey016<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1186/1471-2148-7-214\" ga-product=\"publication\" ga-id=\"beast\"\">BEAST: Bayesian evolutionary analysis by sampling trees<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1371/JOURNAL.PCBI.1003537\" ga-product=\"publication\" ga-id=\"beast2\"\">BEAST 2: A Software Platform for Bayesian Evolutionary Analysis<\/a>",null,null,null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/20110278\" ga-product=\"publication\" ga-id=\"bedtools\">BEDTools: A flexible suite of utilities for comparing genomic features<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1002/ECE3.5571\" ga-product=\"publication\" ga-id=\"bellerophon\"\">The Bellerophon pipeline, improving de novo transcriptomes and removing chimeras<\/a>",null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1002/PRO.3923\" ga-product=\"publication\" ga-id=\"bio3d\"\">The Bio3D packages for structural bioinformatics<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1186/1751-0473-9-13\" ga-product=\"publication\" ga-id=\"biobambam\"\">Biobambam: Tools for read pair collation based algorithms on BAM files<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/nmeth.3252\" ga-product=\"publication\" ga-id=\"biom-format\"\">Orchestrating high-throughput genomic analysis with Bioconductor<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1007/978-1-59745-535-0_26\" ga-product=\"publication\" ga-id=\"bioperl\"\">An introduction to BioPerl<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/19304878\" ga-product=\"publication\" ga-id=\"biopython\">Biopython: Freely available Python tools for computational molecular biology and bioinformatics<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/NAR/GKAC313\" ga-product=\"publication\" ga-id=\"biotransformer\"\">BioTransformer 3.0 - a web server for accurately predicting metabolic transformation products<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1016/S0022-2836(05)80360-2\" ga-product=\"publication\" ga-id=\"blast\"\">Basic local alignment search tool<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btw305\" ga-product=\"publication\" ga-id=\"blast\"\">Benchmarking the next generation of homology inference tools<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkn201\" ga-product=\"publication\" ga-id=\"blast\"\">NCBI BLAST: a better web interface.<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkt282\" ga-product=\"publication\" ga-id=\"blast\"\">BLAST: a more efficient report with usability improvements.<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1016/S0022-2836(05)80360-2\" ga-product=\"publication\" ga-id=\"blast+\"\">Basic local alignment search tool<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btw305\" ga-product=\"publication\" ga-id=\"blast+\"\">Benchmarking the next generation of homology inference tools<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkn201\" ga-product=\"publication\" ga-id=\"blast+\"\">NCBI BLAST: a better web interface.<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkt282\" ga-product=\"publication\" ga-id=\"blast+\"\">BLAST: a more efficient report with usability improvements.<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1016/S0022-2836(05)80360-2\" ga-product=\"publication\" ga-id=\"blast2\"\">Basic local alignment search tool<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btw305\" ga-product=\"publication\" ga-id=\"blast2\"\">Benchmarking the next generation of homology inference tools<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkn201\" ga-product=\"publication\" ga-id=\"blast2\"\">NCBI BLAST: a better web interface.<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkt282\" ga-product=\"publication\" ga-id=\"blast2\"\">BLAST: a more efficient report with usability improvements.<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/11932250\" ga-product=\"publication\" ga-id=\"blat\">BLAT - The BLAST-like alignment tool<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btp419\" ga-product=\"publication\" ga-id=\"Blockbuster\"\">Evidence for human microRNA-offset RNAs in small RNA sequencing data<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1186/gb-2009-10-3-r25\" ga-product=\"publication\" ga-id=\"bowtie\"\">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1186/1471-2105-14-184\" ga-product=\"publication\" ga-id=\"bowtie\"\">Benchmarking short sequence mapping tools<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.7490/f1000research.1114334.1\" ga-product=\"publication\" ga-id=\"bowtie\"\">DOI:10.7490/f1000research.1114334.1<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/nmeth.1923\" ga-product=\"publication\" ga-id=\"bowtie2\"\">Fast gapped-read alignment with Bowtie 2<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1186/1471-2105-14-184\" ga-product=\"publication\" ga-id=\"bowtie2\"\">Benchmarking short sequence mapping tools<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1016/j.ygeno.2017.03.001\" ga-product=\"publication\" ga-id=\"bowtie2\"\">Evaluation and assessment of read-mapping by multiple next-generation sequencing aligners based on genome-wide characteristics<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1186/1471-2164-15-264\" ga-product=\"publication\" ga-id=\"bowtie2\"\">Comparison of mapping algorithms used in high-throughput sequencing: Application to Ion Torrent data<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btu146\" ga-product=\"publication\" ga-id=\"bowtie2\"\">Lacking alignments? The next-generation sequencing mapper segemehl revisited<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.7490/f1000research.1114334.1\" ga-product=\"publication\" ga-id=\"bowtie2\"\">DOI:10.7490/f1000research.1114334.1<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1186/1471-2164-15-1039\" ga-product=\"publication\" ga-id=\"breseq\"\">Identifying structural variation in haploid microbial genomes from short-read resequencing data using breseq<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkw343\" ga-product=\"publication\" ga-id=\"breseq\"\">The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2016 update<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.7490/f1000research.1114334.1\" ga-product=\"publication\" ga-id=\"breseq\"\">DOI:10.7490/f1000research.1114334.1<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btv351\" ga-product=\"publication\" ga-id=\"busco\"\">BUSCO: Assessing genome assembly and annotation completeness with single-copy orthologs<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1002/CPZ1.323\" ga-product=\"publication\" ga-id=\"busco\"\">BUSCO: Assessing Genomic Data Quality and Beyond<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/molbev/msx319\" ga-product=\"publication\" ga-id=\"busco\"\">BUSCO applications from quality assessments to gene prediction and phylogenomics<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/molbev/msab199\" ga-product=\"publication\" ga-id=\"busco\"\">BUSCO Update: Novel and Streamlined Workflows along with Broader and Deeper Phylogenetic Coverage for Scoring of Eukaryotic, Prokaryotic, and Viral Genomes<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btp324\" ga-product=\"publication\" ga-id=\"bwa\"\">Fast and accurate short read alignment with Burrows-Wheeler transform<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1186/1471-2105-14-184\" ga-product=\"publication\" ga-id=\"bwa\"\">Benchmarking short sequence mapping tools<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1016/j.ygeno.2017.03.001\" ga-product=\"publication\" ga-id=\"bwa\"\">Evaluation and assessment of read-mapping by multiple next-generation sequencing aligners based on genome-wide characteristics<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1186/1471-2164-15-264\" ga-product=\"publication\" ga-id=\"bwa\"\">Comparison of mapping algorithms used in high-throughput sequencing: Application to Ion Torrent data<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btu146\" ga-product=\"publication\" ga-id=\"bwa\"\">Lacking alignments? The next-generation sequencing mapper segemehl revisited<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btp698\" ga-product=\"publication\" ga-id=\"bwa\"\">Fast and accurate long-read alignment with Burrows-Wheeler transform<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1109/IPDPS.2019.00041\" ga-product=\"publication\" ga-id=\"bwa-mem2\"\">Efficient architecture-aware acceleration of BWA-MEM for multicore systems<\/a>",null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1016/J.GPB.2018.10.004\" ga-product=\"publication\" ga-id=\"c3s\"\">C3: Consensus Cancer Driver Gene Caller<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/s41586-020-2871-y\" ga-product=\"publication\" ga-id=\"cactus\"\">Progressive Cactus is a multiple-genome aligner for the thousand-genome era<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1101/gr.123356.111\" ga-product=\"publication\" ga-id=\"cactus\"\">Cactus: Algorithms for genome multiple sequence alignment<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1089/cmb.2010.0252\" ga-product=\"publication\" ga-id=\"cactus\"\">Cactus graphs for genome comparisons<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1021/ac202450g\" ga-product=\"publication\" ga-id=\"camera\"\">CAMERA: An integrated strategy for compound spectra extraction and annotation of liquid chromatography/mass spectrometry data sets<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1101/gr.215087.116\" ga-product=\"publication\" ga-id=\"canu\"\">Canu: Scalable and accurate long-read assembly via adaptive κ-mer weighting and repeat separation<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/10508846\" ga-product=\"publication\" ga-id=\"cap3\">CAP3: A DNA sequence assembly program<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btv146\" ga-product=\"publication\" ga-id=\"cardinal\"\">DOI:10.1093/bioinformatics/btv146<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1186/s13059-019-1817-x\" ga-product=\"publication\" ga-id=\"cat_bins\"\">Robust taxonomic classification of uncharted microbial sequences and bins with CAT and BAT<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/bts565\" ga-product=\"publication\" ga-id=\"cd-hit\"\">CD-HIT: Accelerated for clustering the next-generation sequencing data<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btl158\" ga-product=\"publication\" ga-id=\"cd-hit\"\">Cd-hit: A fast program for clustering and comparing large sets of protein or nucleotide sequences<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/17.3.282\" ga-product=\"publication\" ga-id=\"cd-hit\"\">Clustering of highly homologous sequences to reduce the size of large protein databases<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/18.1.77\" ga-product=\"publication\" ga-id=\"cd-hit\"\">Tolerating some redundancy significantly speeds up clustering of large protein databases<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btq003\" ga-product=\"publication\" ga-id=\"cd-hit\"\">CD-HIT Suite: A web server for clustering and comparing biological sequences<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.2144/000112257\" ga-product=\"publication\" ga-id=\"CellProfiler\"\">CellProfiler™: Free, versatile software for automated biological image analysis<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1002/cpz1.89\" ga-product=\"publication\" ga-id=\"CellProfiler\"\">ImageJ and CellProfiler: Complements in Open-Source Bioimage Analysis<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1101/2021.04.05.438318\" ga-product=\"publication\" ga-id=\"cellxgene\"\">DOI:10.1101/2021.04.05.438318<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1101/gr.186072.114\" ga-product=\"publication\" ga-id=\"checkm\"\">CheckM: Assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1007/s10822-009-9260-9\" ga-product=\"publication\" ga-id=\"chembl\"\">ChEMBL. An interview with John Overington, team leader, chemogenomics at the European Bioinformatics Institute Outstation of the European Molecular Biology Laboratory (EMBL-EBI)<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkv1352\" ga-product=\"publication\" ga-id=\"chembl\"\">The European Bioinformatics Institute in 2016: Data growth and integration<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.26434/CHEMRXIV.7877846.V2\" ga-product=\"publication\" ga-id=\"chemfp\"\">DOI:10.26434/CHEMRXIV.7877846.V2<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1186/s13321-019-0398-8\" ga-product=\"publication\" ga-id=\"chemfp\"\">The chemfp project<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btv145\" ga-product=\"publication\" ga-id=\"chipseeker\"\">ChIP seeker: An R/Bioconductor package for ChIP peak annotation, comparison and visualization<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/s41598-019-46773-w\" ga-product=\"publication\" ga-id=\"chromeister\"\">Ultra-fast genome comparison for large-scale genomic experiments<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1101/gr.092759.109\" ga-product=\"publication\" ga-id=\"circos\"\">Circos: An information aesthetic for comparative genomics<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.7490/f1000research.1114334.1\" ga-product=\"publication\" ga-id=\"circos\"\">DOI:10.7490/f1000research.1114334.1<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1101/2020.11.08.370650\" ga-product=\"publication\" ga-id=\"clinker\"\">DOI:10.1101/2020.11.08.370650<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/msb.2011.75\" ga-product=\"publication\" ga-id=\"clustalo\"\">DOI:10.1038/msb.2011.75<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1002/pro.3290\" ga-product=\"publication\" ga-id=\"clustalo\"\">DOI:10.1002/pro.3290<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1002/0471250953.bi0313s48\" ga-product=\"publication\" ga-id=\"clustalo\"\">DOI:10.1002/0471250953.bi0313s48<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/22.22.4673\" ga-product=\"publication\" ga-id=\"clustalw\"\">CLUSTAL W: Improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/25.24.4876\" ga-product=\"publication\" ga-id=\"clustalw\"\">The CLUSTAL X windows interface: Flexible strategies for multiple sequence alignment aided by quality analysis tools<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkg500\" ga-product=\"publication\" ga-id=\"clustalw\"\">Multiple sequence alignment with the Clustal series of programs<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/8.2.189\" ga-product=\"publication\" ga-id=\"clustalw\"\">CLUSTAL V: Improved software for multiple sequence alignment<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1016/0378-1119(88)90330-7\" ga-product=\"publication\" ga-id=\"clustalw\"\">CLUSTAL: a package for performing multiple sequence alignment on a microcomputer<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/s41592-022-01488-1\" ga-product=\"publication\" ga-id=\"colabfold_batch\"\">ColabFold: making protein folding accessible to all<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1101/2021.08.15.456425\" ga-product=\"publication\" ga-id=\"colabfold_batch\"\">DOI:10.1101/2021.08.15.456425<\/a>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1007/978-1-0716-1158-6_3\" ga-product=\"publication\" ga-id=\"cpat\"\">RNA Coding Potential Prediction Using Alignment-Free Logistic Regression Model<\/a>",null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1038/nbt.1621\" ga-product=\"publication\" ga-id=\"cufflinks\"\">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btr355\" ga-product=\"publication\" ga-id=\"cufflinks\"\">Identification of novel transcripts in annotated genomes using RNA-seq<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1038/nbt.2450\" ga-product=\"publication\" ga-id=\"cufflinks\"\">Differential analysis of gene regulation at transcript resolution with RNA-seq<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1186/gb-2011-12-3-r22\" ga-product=\"publication\" ga-id=\"cufflinks\"\">Improving RNA-Seq expression estimates by correcting for fragment bias<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.7490/f1000research.1114334.1\" ga-product=\"publication\" ga-id=\"cufflinks\"\">DOI:10.7490/f1000research.1114334.1<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/nprot.2012.016\" ga-product=\"publication\" ga-id=\"cummeRbund\"\">Erratum: Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks (Nature Protocols (2012) 7 (562-578))<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btt543\" ga-product=\"publication\" ga-id=\"customprodb\"\">CustomProDB: An R package to generate customized protein databases from RNA-Seq data for proteomics search<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.14806/ej.17.1.200\" ga-product=\"publication\" ga-id=\"cutadapt\"\">DOI:10.14806/ej.17.1.200<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.7490/f1000research.1114334.1\" ga-product=\"publication\" ga-id=\"cutadapt\"\">DOI:10.7490/f1000research.1114334.1<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1101/780700\" ga-product=\"publication\" ga-id=\"cutesv\"\">DOI:10.1101/780700<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1186/s13059-020-02107-y\" ga-product=\"publication\" ga-id=\"cutesv\"\">Long-read-based human genomic structural variation detection with cuteSV<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/nmeth.3869\" ga-product=\"publication\" ga-id=\"dada2\"\">DADA2: High-resolution sample inference from Illumina amplicon data<\/a>",null,null,null,null,null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/24799436\" ga-product=\"publication\" ga-id=\"deeptools\">DeepTools: A flexible platform for exploring deep-sequencing data<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btq228\" ga-product=\"publication\" ga-id=\"dendropy\"\">DendroPy: A Python library for phylogenetic computing<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1186/s13059-014-0550-8\" ga-product=\"publication\" ga-id=\"deseq2\"\">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1073/pnas.1307202110\" ga-product=\"publication\" ga-id=\"DEXSeq\"\">Drift and conservation of differential exon usage across tissues in primate species<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/S41592-019-0638-X\" ga-product=\"publication\" ga-id=\"diann\"\">DIA-NN: neural networks and interference correction enable deep proteome coverage in high throughput<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/25599550\" ga-product=\"publication\" ga-id=\"diaumpire\">DIA-Umpire: Comprehensive computational framework for data-independent acquisition proteomics<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/nmeth.3176\" ga-product=\"publication\" ga-id=\"diamond\"\">Fast and sensitive protein alignment using DIAMOND<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/S41592-020-00998-0\" ga-product=\"publication\" ga-id=\"diapysef\"\">diaPASEF: parallel accumulation–serial fragmentation combined with data-independent acquisition<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1186/bcr3664\" ga-product=\"publication\" ga-id=\"diffbind\"\">VAV3 mediates resistance to breast cancer endocrine therapy<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1101/234872\" ga-product=\"publication\" ga-id=\"dropletutils\"\">DOI:10.1101/234872<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1101/177048\" ga-product=\"publication\" ga-id=\"dropletutils\"\">DOI:10.1101/177048<\/a>",null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gks042\" ga-product=\"publication\" ga-id=\"edger\"\">Differential expression analysis of multifactor RNA-Seq experiments with respect to biological variation<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btp616\" ga-product=\"publication\" ga-id=\"edger\"\">edgeR: A Bioconductor package for differential expression analysis of digital gene expression data<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.12688/f1000research.8987.2\" ga-product=\"publication\" ga-id=\"edger\"\">From reads to genes to pathways: Differential expression analysis of RNA-Seq experiments using Rsubread and the edgeR quasi-likelihood pipeline<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gks042\" ga-product=\"publication\" ga-id=\"edger-repenrich\"\">Differential expression analysis of multifactor RNA-Seq experiments with respect to biological variation<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btp616\" ga-product=\"publication\" ga-id=\"edger-repenrich\"\">edgeR: A Bioconductor package for differential expression analysis of digital gene expression data<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.12688/f1000research.8987.2\" ga-product=\"publication\" ga-id=\"edger-repenrich\"\">From reads to genes to pathways: Differential expression analysis of RNA-Seq experiments using Rsubread and the edgeR quasi-likelihood pipeline<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkw989\" ga-product=\"publication\" ga-id=\"eggnog-mapper\"\">ProGenomes: A resource for consistent functional and taxonomic annotations of prokaryotic genomes<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/molbev/msx148\" ga-product=\"publication\" ga-id=\"eggnog-mapper\"\">Fast genome-wide functional annotation through orthology assignment by eggNOG-mapper<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/molbev/msab293\" ga-product=\"publication\" ga-id=\"eggnog-mapper\"\">eggNOG-mapper v2: Functional Annotation, Orthology Assignments, and Domain Prediction at the Metagenomic Scale<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btw623\" ga-product=\"publication\" ga-id=\"egsea\"\">Combining multiple tools outperforms individual methods in gene set enrichment analyses<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1109/TMI.2009.2035616\" ga-product=\"publication\" ga-id=\"elastix\"\">Elastix: A toolbox for intensity-based medical image registration<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.3389/fninf.2013.00050\" ga-product=\"publication\" ga-id=\"elastix\"\">Fast parallel image registration on CPU and GPU for diagnostic classification of Alzheimer's disease<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.3389/FONC.2019.01297\" ga-product=\"publication\" ga-id=\"elastix\"\">Evaluation of an Open Source Registration Package for Automatic Contour Propagation in Online Adaptive Intensity-Modulated Proton Therapy of Prostate Cancer<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/10827456\" ga-product=\"publication\" ga-id=\"emboss\">EMBOSS: The European Molecular Biology Open Software Suite<\/a>",null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/30510204\" ga-product=\"publication\" ga-id=\"encyclopedia\">Chromatogram libraries improve peptide detection and quantification by data independent acquisition mass spectrometry<\/a>",null,null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/19914931\" ga-product=\"publication\" ga-id=\"eupathdb\">EuPathDB: A portal to eukaryotic pathogen databases<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btv184\" ga-product=\"publication\" ga-id=\"examl\"\">ExaML version 3: A tool for phylogenomic analyses on supercomputers<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/15713233\" ga-product=\"publication\" ga-id=\"exonerate\">Automated generation of heuristics for biological sequence comparison<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1186/gb-2011-12-3-r22\" ga-product=\"publication\" ga-id=\"eXpress\"\">Improving RNA-Seq expression estimates by correcting for fragment bias<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.7490/f1000research.1114334.1\" ga-product=\"publication\" ga-id=\"eXpress\"\">DOI:10.7490/f1000research.1114334.1<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.5281/zenodo.35745\" ga-product=\"publication\" ga-id=\"pb-assembly\"\">DOI:10.5281/zenodo.35745<\/a>",null,null,null,null,null,null,null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/bty560\" ga-product=\"publication\" ga-id=\"fastp\"\">Fastp: An ultra-fast all-in-one FASTQ preprocessor<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.7490/f1000research.1114334.1\" ga-product=\"publication\" ga-id=\"fastqc\"\">DOI:10.7490/f1000research.1114334.1<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/molbev/msp077\" ga-product=\"publication\" ga-id=\"fasttree\"\">Fasttree: Computing large minimum evolution trees with profiles instead of a distance matrix<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1371/journal.pone.0009490\" ga-product=\"publication\" ga-id=\"fasttree\"\">FastTree 2 - Approximately maximum-likelihood trees for large alignments<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1006/geno.1997.4995\" ga-product=\"publication\" ga-id=\"fastx\"\">Comparison of DNA sequences with protein sequences<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1006/geno.1997.4995\" ga-product=\"publication\" ga-id=\"fastx_toolkit\"\">Comparison of DNA sequences with protein sequences<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1186/gb-2006-7-s1-s10\" ga-product=\"publication\" ga-id=\"fgenesh\"\">DOI:10.1186/gb-2006-7-s1-s10<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1101/060012\" ga-product=\"publication\" ga-id=\"fgsea\"\">DOI:10.1101/060012<\/a>",null,null,null,null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1099/MGEN.0.000294\" ga-product=\"publication\" ga-id=\"flye\"\">Comparison of long-read sequencing technologies in the hybrid assembly of complex bacterial genomes<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1038/s41587-019-0072-8\" ga-product=\"publication\" ga-id=\"flye\"\">Assembly of long, error-prone reads using repeat graphs<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1038/s41592-020-00971-x\" ga-product=\"publication\" ga-id=\"flye\"\">metaFlye: scalable long-read metagenome assembly using repeat graphs<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1038/s41587-023-01773-0\" ga-product=\"publication\" ga-id=\"foldseek\"\">Fast and accurate protein structure search with Foldseek<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1101/2022.02.07.479398\" ga-product=\"publication\" ga-id=\"foldseek\"\">DOI:10.1101/2022.02.07.479398<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/20478829\" ga-product=\"publication\" ga-id=\"fpocket\">fpocket: Online tools for protein ensemble pocket detection and tracking<\/a><br \\><a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/19486540\" ga-product=\"publication\" ga-id=\"fpocket\">Fpocket: An open source platform for ligand pocket detection<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/20805240\" ga-product=\"publication\" ga-id=\"fraggenescan\">FragGeneScan: Predicting genes in short and error-prone reads<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.7490/f1000research.1114334.1\" ga-product=\"publication\" ga-id=\"fraggenescan\"\">DOI:10.7490/f1000research.1114334.1<\/a>",null,null,null,null,null,null,null,null,null,null,null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/20644199\" ga-product=\"publication\" ga-id=\"gatk\">The genome analysis toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1080/10635150701472164\" ga-product=\"publication\" ga-id=\"gblock\"\">Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/oxfordjournals.molbev.a026334\" ga-product=\"publication\" ga-id=\"gblock\"\">Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1016/j.ajhg.2010.11.011\" ga-product=\"publication\" ga-id=\"gcta\"\">GCTA: A tool for genome-wide complex trait analysis<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1186/s12859-015-0679-9\" ga-product=\"publication\" ga-id=\"gecko\"\">Breaking the computational barriers of pairwise genome comparison<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1371/journal.pcbi.1003153\" ga-product=\"publication\" ga-id=\"gemini\"\">GEMINI: Integrative Exploration of Genetic Variation and Genome Annotations<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1186/s12859-018-2203-5\" ga-product=\"publication\" ga-id=\"gemoma\"\">Combining RNA-seq data and homology-based gene prediction for plants, animals and fungi<\/a>",null,null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1101/747568\" ga-product=\"publication\" ga-id=\"genomescope\"\">DOI:10.1101/747568<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1109/TCBB.2013.68\" ga-product=\"publication\" ga-id=\"genometools\"\">Genome tools: A comprehensive software library for efficient processing of structured genome annotations<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1186/S13059-020-02154-5\" ga-product=\"publication\" ga-id=\"getorganelle\"\">GetOrganelle: A fast and versatile toolkit for accurate de novo assembly of organelle genomes<\/a>",null,null,null,null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1007/978-3-319-24277-4\" ga-product=\"publication\" ga-id=\"ggplot2\"\">DOI:10.1007/978-3-319-24277-4<\/a>",null,null,null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1186/gb-2010-11-2-r14\" ga-product=\"publication\" ga-id=\"goseq\"\">Gene ontology analysis for RNA-seq: accounting for selection bias<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.7717/peerj.1029\" ga-product=\"publication\" ga-id=\"graphlan\"\">Compact graphical representation of phylogenetic data and metadata with GraPhlAn<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1016/j.softx.2015.06.001\" ga-product=\"publication\" ga-id=\"gromacs\"\">Gromacs: High performance molecular simulations through multi-level parallelism from laptops to supercomputers<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1002/jcc.20291\" ga-product=\"publication\" ga-id=\"gromacs\"\">GROMACS: Fast, flexible, and free<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1063/5.0018516\" ga-product=\"publication\" ga-id=\"gromacs\"\">Heterogeneous parallelization and acceleration of molecular dynamics simulations in GROMACS<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1007/978-3-319-15976-8_1\" ga-product=\"publication\" ga-id=\"gromacs\"\">Tackling exascale software challenges in molecular dynamics simulations with GROMACS<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btt055\" ga-product=\"publication\" ga-id=\"gromacs\"\">GROMACS 4.5: A high-throughput and highly parallel open source molecular simulation toolkit<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1021/ct700301q\" ga-product=\"publication\" ga-id=\"gromacs\"\">GRGMACS 4: Algorithms for highly efficient, load-balanced, and scalable molecular simulation<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1007/s008940100045\" ga-product=\"publication\" ga-id=\"gromacs\"\">GROMACS 3.0: A package for molecular simulation and trajectory analysis<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1016/0010-4655(95)00042-E\" ga-product=\"publication\" ga-id=\"gromacs\"\">GROMACS: A message-passing parallel molecular dynamics implementation<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1002/jcc.24030\" ga-product=\"publication\" ga-id=\"gromacs\"\">Best bang for your buck: GPU nodes for GROMACS biomolecular simulations<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1002/jcc.26011\" ga-product=\"publication\" ga-id=\"gromacs\"\">More bang for your buck: Improved use of GPU nodes for GROMACS 2018<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/BIOINFORMATICS/BTZ848\" ga-product=\"publication\" ga-id=\"gtdb-tk\"\">GTDB-Tk: A toolkit to classify genomes with the genome taxonomy database<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gku1196\" ga-product=\"publication\" ga-id=\"gubbins\"\">Rapid phylogenetic analysis of large samples of recombinant bacterial whole genome sequences using Gubbins<\/a>",null,null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/26023104\" ga-product=\"publication\" ga-id=\"heinz\">XHeinz: An algorithm for mining cross-species network modules under a flexible conservation model<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/nprot.2008.211\" ga-product=\"publication\" ga-id=\"hgv_david\"\">Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkw343\" ga-product=\"publication\" ga-id=\"hgv_david\"\">The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2016 update<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.7490/f1000research.1114334.1\" ga-product=\"publication\" ga-id=\"hgv_david\"\">DOI:10.7490/f1000research.1114334.1<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1086/381000\" ga-product=\"publication\" ga-id=\"hgv_ldtools\"\">Selecting a Maximally Informative Set of Single-Nucleotide Polymorphisms for Association Analyses Using Linkage Disequilibrium<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkw343\" ga-product=\"publication\" ga-id=\"hgv_ldtools\"\">The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2016 update<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.7490/f1000research.1114334.1\" ga-product=\"publication\" ga-id=\"hgv_ldtools\"\">DOI:10.7490/f1000research.1114334.1<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkm226\" ga-product=\"publication\" ga-id=\"hgv_linkToGProfile\"\">G:Profiler-a web-based toolset for functional profiling of gene lists from large-scale experiments<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkw343\" ga-product=\"publication\" ga-id=\"hgv_linkToGProfile\"\">The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2016 update<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.7490/f1000research.1114334.1\" ga-product=\"publication\" ga-id=\"hgv_linkToGProfile\"\">DOI:10.7490/f1000research.1114334.1<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/NAR/GKAA220\" ga-product=\"publication\" ga-id=\"hicexplorer\"\">Galaxy HiCExplorer 3: A web server for reproducible Hi-C, capture Hi-C and single-cell Hi-C data analysis, quality control and visualization<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1186/s12864-022-08375-1\" ga-product=\"publication\" ga-id=\"hifiadapterfilt\"\">HiFiAdapterFilt, a memory efficient read processing pipeline, prevents occurrence of adapter sequence in PacBio HiFi reads and their negative impacts on genome assembly<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/s41592-020-01056-5\" ga-product=\"publication\" ga-id=\"hifiasm\"\">Haplotype-resolved de novo assembly using phased assembly graphs with hifiasm<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1038/nmeth.3317\" ga-product=\"publication\" ga-id=\"hisat2\"\">HISAT: A fast spliced aligner with low memory requirements<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1038/nprot.2016.095\" ga-product=\"publication\" ga-id=\"hisat2\"\">Transcript-level expression analysis of RNA-seq experiments with HISAT, StringTie and Ballgown<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1038/nmeth.4106\" ga-product=\"publication\" ga-id=\"hisat2\"\">Simulation-based comprehensive benchmarking of RNA-seq aligners<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkr367\" ga-product=\"publication\" ga-id=\"hmmer\"\">HMMER web server: Interactive sequence similarity searching<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1371/journal.pcbi.1002195\" ga-product=\"publication\" ga-id=\"hmmer\"\">Accelerated profile HMM searches<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkt263\" ga-product=\"publication\" ga-id=\"hmmer\"\">Challenges in homology search: HMMER3 and convergent evolution of coiled-coil regions<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gky448\" ga-product=\"publication\" ga-id=\"hmmer\"\">HMMER web server: 2018 update<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btu638\" ga-product=\"publication\" ga-id=\"htseq\"\">HTSeq-A Python framework to work with high-throughput sequencing data<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/gigascience/giab007\" ga-product=\"publication\" ga-id=\"htslib\"\">HTSlib: C library for reading/writing high-Throughput sequencing data<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.7554/eLife.65088\" ga-product=\"publication\" ga-id=\"humann\"\">Integrating taxonomic, functional, and strain-level profiling of diverse microbial communities with biobakery 3<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/s41592-018-0176-y\" ga-product=\"publication\" ga-id=\"humann2\"\">Species-level functional profiling of metagenomes and metatranscriptomes<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1007/978-1-4939-8736-8_6\" ga-product=\"publication\" ga-id=\"HyPhy\"\">Detecting Amino Acid Coevolution with Bayesian Graphical Models<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/bti079\" ga-product=\"publication\" ga-id=\"HyPhy\"\">HyPhy: Hypothesis testing using phylogenies<\/a>",null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btt509\" ga-product=\"publication\" ga-id=\"infernal\"\">Infernal 1.1: 100-fold faster RNA homology searches<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/BIB/BBS017\" ga-product=\"publication\" ga-id=\"igv\"\">Integrative Genomics Viewer (IGV): High-performance genomics data visualization and exploration<\/a><br \\><a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/21221095\" ga-product=\"publication\" ga-id=\"igv\">Integrative genomics viewer<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/BIOINFORMATICS/BTAC830\" ga-product=\"publication\" ga-id=\"igv\"\">igv.js: an embeddable JavaScript implementation of the Integrative Genomics Viewer (IGV)<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkw319\" ga-product=\"publication\" ga-id=\"integron_finder\"\">Identification and analysis of integrons and cassette arrays in bacterial genomes<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.3390/microorganisms10020224\" ga-product=\"publication\" ga-id=\"integron_finder\"\">Integron Functionality and Genome Innovation: An Update on the Subtle and Smart Strategy of Integrase and Gene Cassette Expression Regulation<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btu031\" ga-product=\"publication\" ga-id=\"interproscan\"\">InterProScan 5: Genome-scale protein function classification<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkac240\" ga-product=\"publication\" ga-id=\"interproscan\"\">Search and sequence analysis tools services from EMBL-EBI in 2022<\/a>",null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/BIOINFORMATICS/BTZ966\" ga-product=\"publication\" ga-id=\"ipyrad\"\">Ipyrad: Interactive assembly and analysis of RADseq datasets<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/27084950\" ga-product=\"publication\" ga-id=\"iq-tree\">W-IQ-TREE: a fast online phylogenetic tool for maximum likelihood analysis<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.12688/f1000research.14966.1\" ga-product=\"publication\" ga-id=\"isee\"\">iSEE: Interactive SummarizedExperiment Explorer [version 1; referees: 2 approved]<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btx433\" ga-product=\"publication\" ga-id=\"isescan\"\">ISEScan: automated identification of insertion sequence elements in prokaryotic genomes<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1158/1541-7786.MCR-16-0459\" ga-product=\"publication\" ga-id=\"isoformswitchanalyzer\"\">The landscape of isoform switches in human cancers<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.22541/AU.160610419.99549785/V1\" ga-product=\"publication\" ga-id=\"ivar\"\">DOI:10.22541/AU.160610419.99549785/V1<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/bty090\" ga-product=\"publication\" ga-id=\"iwtomics\"\">IWTomics: Testing high-resolution sequence-based 'Omics' data at multiple locations and scales<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1101/2021.05.27.445886\" ga-product=\"publication\" ga-id=\"jasminesv\"\">DOI:10.1101/2021.05.27.445886<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1101/gr.094607.109\" ga-product=\"publication\" ga-id=\"jbrowse\"\">JBrowse: A next-generation genome browser<\/a>",null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/21217122\" ga-product=\"publication\" ga-id=\"jellyfish\">A fast, lock-free approach for efficient parallel counting of occurrences of k-mers<\/a>",null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1038/nbt.3519\" ga-product=\"publication\" ga-id=\"kallisto\"\">Near-optimal probabilistic RNA-seq quantification<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btw663\" ga-product=\"publication\" ga-id=\"kat\"\">KAT: A K-mer analysis toolkit to quality control NGS datasets and genome assemblies<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.12688/f1000research.6924.1\" ga-product=\"publication\" ga-id=\"khmer\"\">The khmer software package: Enabling efficient nucleotide sequence analysis<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1073/pnas.1121464109\" ga-product=\"publication\" ga-id=\"khmer\"\">Scaling metagenome sequence assembly with probabilistic de Bruijn graphs<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1371/journal.pone.0101271\" ga-product=\"publication\" ga-id=\"khmer\"\">These are not the K-mers you are looking for: Efficient online K-mer counting using a probabilistic data structure<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.7287/peerj.preprints.890v1\" ga-product=\"publication\" ga-id=\"khmer\"\">DOI:10.7287/peerj.preprints.890v1<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/25609798\" ga-product=\"publication\" ga-id=\"kmc\">KMC 2: Fast and resource-frugal k-mer counting<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1186/gb-2014-15-3-r46\" ga-product=\"publication\" ga-id=\"kraken\"\">Kraken: Ultrafast metagenomic sequence classification using exact alignments<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1101/762302\" ga-product=\"publication\" ga-id=\"kraken2\"\">DOI:10.1101/762302<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/21961884\" ga-product=\"publication\" ga-id=\"krona\">Interactive metagenomic visualization in a Web browser<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1101/2020.05.04.075853\" ga-product=\"publication\" ga-id=\"lama\"\">DOI:10.1101/2020.05.04.075853<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1021/ACS.JPROTEOME.9B00496\" ga-product=\"publication\" ga-id=\"LFQ-Analyst\"\">Lfq-Analyst: An easy-To-use interactive web platform to analyze and visualize label-free proteomics data preprocessed with maxquant<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkv007\" ga-product=\"publication\" ga-id=\"limma\"\">Limma powers differential expression analyses for RNA-sequencing and microarray studies<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1186/s13742-015-0076-3\" ga-product=\"publication\" ga-id=\"links\"\">LINKS: Scalable, alignment-free scaffolding of draft genomes with long reads<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gks918\" ga-product=\"publication\" ga-id=\"lofreq\"\">LoFreq: A sequence-quality aware, ultra-sensitive variant caller for uncovering cell-population heterogeneity from high-throughput sequencing datasets<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/s41467-019-12493-y\" ga-product=\"publication\" ga-id=\"longshot\"\">Longshot enables accurate variant calling in diploid genomes from single-molecule long read sequencing<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1104/pp.17.01310\" ga-product=\"publication\" ga-id=\"ltr_retriever\"\">LTR_retriever: A highly accurate and sensitive program for identification of long terminal repeat retrotransposons<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/18798982\" ga-product=\"publication\" ga-id=\"macs\">Model-based analysis of ChIP-Seq (MACS)<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/18798982\" ga-product=\"publication\" ga-id=\"macs2\">Model-based analysis of ChIP-Seq (MACS)<\/a>",null,null,null,null,null,null,null,null,null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1007/978-1-62703-646-7_8\" ga-product=\"publication\" ga-id=\"mafft\"\">MAFFT: Iterative refinement and additional methods<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.7490/f1000research.1114334.1\" ga-product=\"publication\" ga-id=\"mafft\"\">DOI:10.7490/f1000research.1114334.1<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/bty121\" ga-product=\"publication\" ga-id=\"mafft\"\">Parallelization of MAFFT for large-scale multiple sequence alignments<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bib/bbx108\" ga-product=\"publication\" ga-id=\"mafft\"\">MAFFT online service: Multiple sequence alignment, interactive sequence choice and visualization<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/molbev/mst010\" ga-product=\"publication\" ga-id=\"mafft\"\">MAFFT multiple sequence alignment software version 7: Improvements in performance and usability<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gki198\" ga-product=\"publication\" ga-id=\"mafft\"\">MAFFT version 5: Improvement in accuracy of multiple sequence alignment<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/26673418\" ga-product=\"publication\" ga-id=\"MAGeCK\">Quality control, modeling, and visualization of CRISPR screens with MAGeCK-VISPR<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1101/gr.6743907\" ga-product=\"publication\" ga-id=\"maker\"\">MAKER: An easy-to-use annotation pipeline designed for emerging model organism genomes<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1186/1471-2105-12-491\" ga-product=\"publication\" ga-id=\"maker\"\">MAKER2: An annotation pipeline and genome-database management tool for second-generation genome projects<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/bts447\" ga-product=\"publication\" ga-id=\"maldi_quant\"\">Maldiquant: A versatile R package for the analysis of mass spectrometry data<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1186/s13059-016-0997-x\" ga-product=\"publication\" ga-id=\"mash\"\">Mash: Fast genome and metagenome distance estimation using MinHash<\/a>",null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/23990416\" ga-product=\"publication\" ga-id=\"masurca\">The MaSuRCA genome assembler<\/a>",null,null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/26515820\" ga-product=\"publication\" ga-id=\"maxbin\">MaxBin 2.0: An automated binning algorithm to recover genomes from multiple metagenomic datasets<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1002/pmic.201400449\" ga-product=\"publication\" ga-id=\"maxquant\"\">Visualization of LC-MS/MS proteomics data in MaxQuant<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1007/978-1-4939-1142-4_24\" ga-product=\"publication\" ga-id=\"maxquant\"\">MaxQuant for in-depth analysis of large SILAC datasets<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1038/nbt.1511\" ga-product=\"publication\" ga-id=\"maxquant\"\">MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1021/ACS.JPROTEOME.0C00892\" ga-product=\"publication\" ga-id=\"maxquant\"\">MaxQuant.Live Enables Enhanced Selectivity and Identification of Peptides Modified by Endogenous SUMO and Ubiquitin<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1002/pmic.201400449\" ga-product=\"publication\" ga-id=\"biocontainers-maxquant\"\">Visualization of LC-MS/MS proteomics data in MaxQuant<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1007/978-1-4939-1142-4_24\" ga-product=\"publication\" ga-id=\"biocontainers-maxquant\"\">MaxQuant for in-depth analysis of large SILAC datasets<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1038/nbt.1511\" ga-product=\"publication\" ga-id=\"biocontainers-maxquant\"\">MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1021/ACS.JPROTEOME.0C00892\" ga-product=\"publication\" ga-id=\"biocontainers-maxquant\"\">MaxQuant.Live Enables Enhanced Selectivity and Identification of Peptides Modified by Endogenous SUMO and Ubiquitin<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1007/978-1-61779-361-5_15\" ga-product=\"publication\" ga-id=\"mcl\"\">Using MCL to extract clusters from networks<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1002/jcc.21787\" ga-product=\"publication\" ga-id=\"mdanalysis\"\">MDAnalysis: A toolkit for the analysis of molecular dynamics simulations<\/a>",null,null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/25609793\" ga-product=\"publication\" ga-id=\"megahit\">MEGAHIT: An ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1101/2020.03.15.992941\" ga-product=\"publication\" ga-id=\"merqury\"\">DOI:10.1101/2020.03.15.992941<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1186/s13059-020-02134-9\" ga-product=\"publication\" ga-id=\"meryl\"\">Merqury: Reference-free quality, completeness, and phasing assessment for genome assemblies<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.7717/PEERJ.7359\" ga-product=\"publication\" ga-id=\"metabat\"\">MetaBAT 2: An adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.7717/peerj.1165\" ga-product=\"publication\" ga-id=\"metabat\"\">MetaBAT, an efficient tool for accurately reconstructing single genomes from complex microbial communities<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.12688/F1000RESEARCH.18866.2\" ga-product=\"publication\" ga-id=\"MetaDEGalaxy\"\">MetaDEGalaxy: Galaxy workflow for differential abundance analysis of 16s metagenomic data<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1186/s40168-020-00808-x\" ga-product=\"publication\" ga-id=\"metaeuk\"\">MetaEuk-sensitive, high-throughput gene discovery, and annotation for large-scale eukaryotic metagenomics<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1101/851964\" ga-product=\"publication\" ga-id=\"metaeuk\"\">DOI:10.1101/851964<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/nmeth.2066\" ga-product=\"publication\" ga-id=\"metaphlan\"\">Metagenomic microbial community profiling using unique clade-specific marker genes<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1074/mcp.RA118.001240\" ga-product=\"publication\" ga-id=\"metaQuantome\"\">MetaQuantome: An integrated, quantitative metaproteomics approach reveals connections between taxonomy and protein function in complex microbiomes<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1021/ACS.JPROTEOME.0C00960\" ga-product=\"publication\" ga-id=\"metaQuantome\"\">Updates on metaQuantome Software for Quantitative Metaproteomics<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1089/cmb.2012.0021\" ga-product=\"publication\" ga-id=\"metaspades\"\">SPAdes: A new genome assembly algorithm and its applications to single-cell sequencing<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkw343\" ga-product=\"publication\" ga-id=\"metaspades\"\">The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2016 update<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.7490/f1000research.1114334.1\" ga-product=\"publication\" ga-id=\"metaspades\"\">DOI:10.7490/f1000research.1114334.1<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1186/s40168-018-0541-1\" ga-product=\"publication\" ga-id=\"metawrap\"\">MetaWRAP - A flexible pipeline for genome-resolved metagenomic data analysis 08 Information and Computing Sciences 0803 Computer Software 08 Information and Computing Sciences 0806 Information Systems<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1534/genetics.109.112532\" ga-product=\"publication\" ga-id=\"MIGRATE\"\">Unified framework to evaluate panmixia and migration direction among multiple sampling locations<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1073/pnas.1810239116\" ga-product=\"publication\" ga-id=\"MIGRATE\"\">Fractional coalescent<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1186/1748-7188-9-2\" ga-product=\"publication\" ga-id=\"minia\"\">Using cascading Bloom filters to improve the memory usage for de Brujin graphs<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btw152\" ga-product=\"publication\" ga-id=\"miniasm\"\">Minimap and miniasm: Fast mapping and de novo assembly for noisy long sequences<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/bty191\" ga-product=\"publication\" ga-id=\"minimap2\"\">Minimap2: Pairwise alignment for nucleotide sequences<\/a>",null,null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1016/j.ympev.2012.08.023\" ga-product=\"publication\" ga-id=\"mitos2\"\">MITOS: Improved de novo metazoan mitochondrial genome annotation<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1128/JCM.06094-11\" ga-product=\"publication\" ga-id=\"mlst\"\">Multilocus sequence typing of total-genome-sequenced bacteria<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/nbt.3988\" ga-product=\"publication\" ga-id=\"mmseqs2\"\">MMseqs2 enables sensitive protein sequence searching for the analysis of massive data sets<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1101/079681\" ga-product=\"publication\" ga-id=\"mmseqs2\"\">DOI:10.1101/079681<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1038/s41467-018-04964-5\" ga-product=\"publication\" ga-id=\"mmseqs2\"\">Clustering huge protein sequence sets in linear time<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/bty1057\" ga-product=\"publication\" ga-id=\"mmseqs2\"\">MMseqs2 desktop and local web server app for fast, interactive sequence searches<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1101/2020.11.27.401018\" ga-product=\"publication\" ga-id=\"mmseqs2\"\">DOI:10.1101/2020.11.27.401018<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btab184\" ga-product=\"publication\" ga-id=\"mmseqs2\"\">DOI:10.1093/bioinformatics/btab184<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1099/MGEN.0.000435\" ga-product=\"publication\" ga-id=\"mob-suite\"\">Universal whole-sequence-based plasmid typing and its utility to prediction of host range and epidemiological surveillance<\/a>",null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1038/nmeth.4075\" ga-product=\"publication\" ga-id=\"moff\"\">MoFF: A robust and automated approach to extract peptide ion intensities<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1021/pr301024c\" ga-product=\"publication\" ga-id=\"morpheus\"\">A proteomics search algorithm specifically designed for high-resolution tandem mass spectra<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1128/AEM.01541-09\" ga-product=\"publication\" ga-id=\"mothur\"\">DOI:10.1128/AEM.01541-09<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1007/s00335-015-9573-z\" ga-product=\"publication\" ga-id=\"mousemine\"\">MouseMine: a new data warehouse for MGI<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/17.8.754\" ga-product=\"publication\" ga-id=\"mrbayes\"\">MRBAYES: Bayesian inference of phylogenetic trees<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btg180\" ga-product=\"publication\" ga-id=\"mrbayes\"\">MrBayes 3: Bayesian phylogenetic inference under mixed models<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/sysbio/sys029\" ga-product=\"publication\" ga-id=\"mrbayes\"\">Mrbayes 3.2: Efficient bayesian phylogenetic inference and model choice across a large model space<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/nbt.2377\" ga-product=\"publication\" ga-id=\"msconvert\"\">A cross-platform toolkit for mass spectrometry and proteomics<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.21105/joss.04494\" ga-product=\"publication\" ga-id=\"msmetaenhancer\"\">DOI:10.21105/joss.04494<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/24794931\" ga-product=\"publication\" ga-id=\"msstats\">MSstats: An R package for statistical analysis of quantitative mass spectrometry-based proteomic experiments<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btw354\" ga-product=\"publication\" ga-id=\"multiqc\"\">MultiQC: Summarize analysis results for multiple tools and samples in a single report<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/10325427\" ga-product=\"publication\" ga-id=\"mummer\">Alignment of whole genomes<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1371/journal.pcbi.1005944\" ga-product=\"publication\" ga-id=\"mummer\"\">MUMmer4: A fast and versatile genome alignment system<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1186/gb-2004-5-2-r12\" ga-product=\"publication\" ga-id=\"mummer\"\">DOI:10.1186/gb-2004-5-2-r12<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/30.11.2478\" ga-product=\"publication\" ga-id=\"mummer\"\">Fast algorithms for large-scale genome alignment and comparison<\/a>",null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1101/843136\" ga-product=\"publication\" ga-id=\"nanocompore\"\">DOI:10.1101/843136<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/bty149\" ga-product=\"publication\" ga-id=\"nanoplot\"\">NanoPack: Visualizing and processing long-read sequencing data<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/nmeth.4184\" ga-product=\"publication\" ga-id=\"nanopolish\"\">Detecting DNA cytosine methylation using nanopore sequencing<\/a>",null,null,null,null,null,null,null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btq243\" ga-product=\"publication\" ga-id=\"newick_utils\"\">The Newick utilities: high-throughput phylogenetic tree processing in the UNIX shell<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1038/nbt.3820\" ga-product=\"publication\" ga-id=\"nextflow\"\">Nextflow enables reproducible computational workflows<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/23314324\" ga-product=\"publication\" ga-id=\"ngsutils\">NGSUtils: A software suite for analyzing and manipulating next-generation sequencing datasets<\/a>",null,null,null,null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/25959493\" ga-product=\"publication\" ga-id=\"obitools\">obitools: A unix-inspired software package for DNA metabarcoding<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1038/nmeth.3959\" ga-product=\"publication\" ga-id=\"openms\"\">OpenMS: A flexible open-source software platform for mass spectrometry data analysis<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1007/978-1-60761-987-1_23\" ga-product=\"publication\" ga-id=\"openms\"\">OpenMS and TOPP: open source software for LC-MS data analysis.<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1186/S13059-022-02664-4\" ga-product=\"publication\" ga-id=\"pangolin\"\">Predicting RNA splicing from DNA sequence using Pangolin<\/a>",null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btt285\" ga-product=\"publication\" ga-id=\"pathview\"\">Pathview: An R/Bioconductor package for pathway-based data integration and visualization<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1186/s12859-019-2597-8\" ga-product=\"publication\" ga-id=\"pblat\"\">pblat: A multithread blat algorithm speeding up aligning sequences to genomes<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/S41598-021-98889-7\" ga-product=\"publication\" ga-id=\"pe_histogram\"\">Productive visualization of high-throughput sequencing data using the SeqCode open portable platform<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btt593\" ga-product=\"publication\" ga-id=\"pear\"\">PEAR: A fast and accurate Illumina Paired-End reAd mergeR<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1038/nbt.3109\" ga-product=\"publication\" ga-id=\"peptideshaker\"\">PeptideShaker enables reanalysis of MS-derived proteomics data sets: To the editor<\/a>",null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/17952086\" ga-product=\"publication\" ga-id=\"percolator\">Semi-supervised learning for peptide identification from shotgun proteomics datasets<\/a>",null,null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1371/journal.pone.0061217\" ga-product=\"publication\" ga-id=\"phyloseq\"\">Phyloseq: An R Package for Reproducible Interactive Analysis and Graphics of Microbiome Census Data<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/sysbio/syq010\" ga-product=\"publication\" ga-id=\"phyml\"\">New algorithms and methods to estimate maximum-likelihood phylogenies: Assessing the performance of PhyML 3.0<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1080/10635150390235520\" ga-product=\"publication\" ga-id=\"phyml\"\">A Simple, Fast, and Accurate Algorithm to Estimate Large Phylogenies by Maximum Likelihood<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/bti713\" ga-product=\"publication\" ga-id=\"phyml\"\">Improving the efficiency of SPR moves in phylogenetic tree search methods based on maximum likelihood<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1080/10635150600755453\" ga-product=\"publication\" ga-id=\"phyml\"\">Approximate likelihood-ratio test for branches: A fast, accurate, and powerful alternative<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gki352\" ga-product=\"publication\" ga-id=\"phyml\"\">PHYML Online - A web server for fast maximum likelihood-based phylogenetic inference<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1038/nbt.2676\" ga-product=\"publication\" ga-id=\"picrust\"\">Predictive functional profiling of microbial communities using 16S rRNA marker gene sequences<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/s41587-020-0548-6\" ga-product=\"publication\" ga-id=\"picrust2\"\">PICRUSt2 for prediction of metagenome functions<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1371/journal.pone.0112963\" ga-product=\"publication\" ga-id=\"pilon\"\">Pilon: An integrated tool for comprehensive microbial variant detection and genome assembly improvement<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/S41598-019-53155-9\" ga-product=\"publication\" ga-id=\"pipe_t\"\">PIPE-T: a new Galaxy tool for the analysis of RT-qPCR expression data<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1086/519795\" ga-product=\"publication\" ga-id=\"plink\"\">PLINK: A tool set for whole-genome association and population-based linkage analyses<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1109/SC.2003.10053\" ga-product=\"publication\" ga-id=\"pnetcdf\"\">DOI:10.1109/SC.2003.10053<\/a>",null,null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/25143291\" ga-product=\"publication\" ga-id=\"poretools\">Poretools: A toolkit for analyzing nanopore sequence data<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/nmeth.3252\" ga-product=\"publication\" ga-id=\"pplacer\"\">Orchestrating high-throughput genomic analysis with Bioconductor<\/a>",null,null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/22131910\" ga-product=\"publication\" ga-id=\"prodigal\">Genome majority vote improves gene predictions<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btu153\" ga-product=\"publication\" ga-id=\"prokka\"\">Prokka: Rapid prokaryotic genome annotation<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1101/729962\" ga-product=\"publication\" ga-id=\"purge_dups\"\">DOI:10.1101/729962<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/BIOINFORMATICS/BTAA692\" ga-product=\"publication\" ga-id=\"pygenometracks\"\">pyGenomeTracks: reproducible plots for multivariate genomic datasets<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btp352\" ga-product=\"publication\" ga-id=\"pysam\"\">The Sequence Alignment/Map format and SAMtools<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1038/s41587-019-0209-9\" ga-product=\"publication\" ga-id=\"qiime2\"\">Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2<\/a>",null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/22914218\" ga-product=\"publication\" ga-id=\"qualimap\">Qualimap: Evaluating next-generation sequencing alignment data<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1101/733238\" ga-product=\"publication\" ga-id=\"quasitools\"\">DOI:10.1101/733238<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/23422339\" ga-product=\"publication\" ga-id=\"quast\">QUAST: Quality assessment tool for genome assemblies<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.12688/f1000research.16450.2\" ga-product=\"publication\" ga-id=\"query_tabular\"\">Improve your Galaxy text life: The Query Tabular Tool<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.11120/msor.2001.01010023\" ga-product=\"publication\" ga-id=\"r\"\">DOI:10.11120/msor.2001.01010023<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.3390/PLANTS8080270\" ga-product=\"publication\" ga-id=\"racon\"\">Constructing a reference genome in a single lab: The possibility to use oxford nanopore technology<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1021/ac501530d\" ga-product=\"publication\" ga-id=\"ramclustr\"\">RAMClust: A novel feature clustering method enables spectral-matching-based annotation for metabolomics data<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1101/2020.08.07.242461\" ga-product=\"publication\" ga-id=\"raven\"\">DOI:10.1101/2020.08.07.242461<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1021/acs.jproteome.8b00721\" ga-product=\"publication\" ga-id=\"rawtools\"\">RawTools: Rapid and Dynamic Interrogation of Orbitrap Data Files for Mass Spectrometer System Management<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btu033\" ga-product=\"publication\" ga-id=\"raxml\"\">RAxML version 8: A tool for phylogenetic analysis and post-analysis of large phylogenies<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btl446\" ga-product=\"publication\" ga-id=\"raxml\"\">RAxML-VI-HPC: Maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.26434/CHEMRXIV.7791947.V2\" ga-product=\"publication\" ga-id=\"rdkit\"\">DOI:10.26434/CHEMRXIV.7791947.V2<\/a>",null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/15215408\" ga-product=\"publication\" ga-id=\"recon\">RECON: A program for prediction of nucleosome formation potential<\/a>",null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/26930599\" ga-product=\"publication\" ga-id=\"red\">RED: A Java-MySQL software for identifying and visualizing RNA editing sites using rule- based and statistical filters<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/bti1018\" ga-product=\"publication\" ga-id=\"repeatscout\"\">DOI:10.1093/bioinformatics/bti1018<\/a>",null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1002/jms.3131\" ga-product=\"publication\" ga-id=\"rmassbank\"\">Automatic recalibration and processing of tandem mass spectra using formula annotation<\/a>",null,null,null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/26198102\" ga-product=\"publication\" ga-id=\"roary\">Roary: Rapid large-scale prokaryote pan genome analysis<\/a>",null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/20022975\" ga-product=\"publication\" ga-id=\"rsem\">RNA-Seq gene expression estimation with read mapping uncertainty<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/bts356\" ga-product=\"publication\" ga-id=\"rseqc\"\">RSeQC: Quality control of RNA-seq experiments<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1186/s12859-016-0922-z\" ga-product=\"publication\" ga-id=\"rseqc\"\">Measure transcript integrity using RNA-seq data<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1002/jae.1278\" ga-product=\"publication\" ga-id=\"rstudio\"\">RSTUDIO: A platform-independent IDE for R and sweave<\/a>",null,null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/24752080\" ga-product=\"publication\" ga-id=\"sailfish\">Sailfish enables alignment-free isoform quantification from RNA-seq reads using lightweight algorithms<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/nmeth.4197\" ga-product=\"publication\" ga-id=\"salmon\"\">Salmon provides fast and bias-aware quantification of transcript expression<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1371/JOURNAL.PCBI.1007273\" ga-product=\"publication\" ga-id=\"salsa\"\">Integrating Hi-C links with assembly graphs for chromosome-scale assembly<\/a>",null,null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/25697820\" ga-product=\"publication\" ga-id=\"sambamba\">Sambamba: Fast processing of NGS alignment formats<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/24812344\" ga-product=\"publication\" ga-id=\"samblaster\">SAMBLASTER: Fast duplicate marking and structural variant read extraction<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btp352\" ga-product=\"publication\" ga-id=\"samtools\"\">The Sequence Alignment/Map format and SAMtools<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/gigascience/giab008\" ga-product=\"publication\" ga-id=\"samtools\"\">Twelve years of SAMtools and BCFtools<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btr509\" ga-product=\"publication\" ga-id=\"samtools\"\">A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1186/s13059-017-1382-0\" ga-product=\"publication\" ga-id=\"scanpy\"\">SCANPY: Large-scale single-cell gene expression data analysis<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btw777\" ga-product=\"publication\" ga-id=\"scater\"\">Scater: Pre-processing, quality control, normalization and visualization of single-cell RNA-seq data in R<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.7287/PEERJ.PREPRINTS.336V2\" ga-product=\"publication\" ga-id=\"scikit-image\"\">DOI:10.7287/PEERJ.PREPRINTS.336V2<\/a>",null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1142/9789814366496_0024\" ga-product=\"publication\" ga-id=\"sepp\"\">DOI:10.1142/9789814366496_0024<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1371/journal.pone.0163962\" ga-product=\"publication\" ga-id=\"seqkit\"\">SeqKit: A cross-platform and ultrafast toolkit for FASTA/Q file manipulation<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1186/s13104-017-2616-7\" ga-product=\"publication\" ga-id=\"seqtk\"\">FastQ-brew: Module for analysis, preprocessing, and reformatting of FASTQ sequence data<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1016/j.cell.2021.04.048\" ga-product=\"publication\" ga-id=\"seurat\"\">Integrated analysis of multimodal single-cell data<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1038/s41587-020-0503-6\" ga-product=\"publication\" ga-id=\"shasta\"\">Nanopore sequencing and the Shasta toolkit enable efficient de novo assembly of eleven human genomes<\/a>",null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/19505939\" ga-product=\"publication\" ga-id=\"sicer\">A clustering approach for identification of enriched domains from histone modification ChIP-Seq data<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1101/2020.07.06.190629\" ga-product=\"publication\" ga-id=\"simtext\"\">DOI:10.1101/2020.07.06.190629<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1128/AEM.02265-19\" ga-product=\"publication\" ga-id=\"sistr\"\">Performance and accuracy of four open-source tools for in silico serotyping of salmonella spp. Based on whole-genome short-read sequencing data<\/a>",null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1101/747568\" ga-product=\"publication\" ga-id=\"Smudgeplots\"\">DOI:10.1101/747568<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/22908215\" ga-product=\"publication\" ga-id=\"snakemake\">Snakemake-a scalable bioinformatics workflow engine<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/15144565\" ga-product=\"publication\" ga-id=\"snap\">Gene finding in novel genomes<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/s41592-018-0001-7\" ga-product=\"publication\" ga-id=\"sniffles\"\">Accurate detection of complex structural variations using single-molecule sequencing<\/a>",null,null,null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/22728672\" ga-product=\"publication\" ga-id=\"snpeff\">PMID:22728672<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.3389/fgene.2012.00035\" ga-product=\"publication\" ga-id=\"snpsift\"\">Using Drosophila melanogaster as a model for genotoxic chemical mutational studies with a new program, SnpSift<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1186/2047-217X-1-18\" ga-product=\"publication\" ga-id=\"soapdenovo2\"\">SOAPdenovo2: An empirically improved memory-efficient short-read de novo assembler<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/bts611\" ga-product=\"publication\" ga-id=\"sortmerna\"\">SortMeRNA: Fast and accurate filtering of ribosomal RNAs in metatranscriptomic data<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1089/cmb.2012.0021\" ga-product=\"publication\" ga-id=\"spades\"\">SPAdes: A new genome assembly algorithm and its applications to single-cell sequencing<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.7490/f1000research.1114334.1\" ga-product=\"publication\" ga-id=\"spades\"\">DOI:10.7490/f1000research.1114334.1<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/bth103\" ga-product=\"publication\" ga-id=\"srf\"\">Spectral repeat finders (SRF): Identification of repetitive sequences using Fourier transformation<\/a>",null,null,null,null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/16381840\" ga-product=\"publication\" ga-id=\"sra-tools\">Database resources of the National Center for Biotechnology Information.<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/24324759\" ga-product=\"publication\" ga-id=\"ssw\">SSW library: An SIMD Smith-Waterman C/C++ library for use in genomic applications<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1111/mec.12354\" ga-product=\"publication\" ga-id=\"stacks\"\">Stacks: An analysis tool set for population genomics<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/bts635\" ga-product=\"publication\" ga-id=\"star\"\">STAR: Ultrafast universal RNA-seq aligner<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1038/nmeth.4106\" ga-product=\"publication\" ga-id=\"star\"\">Simulation-based comprehensive benchmarking of RNA-seq aligners<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btu146\" ga-product=\"publication\" ga-id=\"star\"\">Lacking alignments? The next-generation sequencing mapper segemehl revisited<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1038/nbt.3122\" ga-product=\"publication\" ga-id=\"stringtie\"\">StringTie enables improved reconstruction of a transcriptome from RNA-seq reads<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gkt214\" ga-product=\"publication\" ga-id=\"subread\"\">The Subread aligner: Fast, accurate and scalable read mapping by seed-and-vote<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1038/nmeth.4106\" ga-product=\"publication\" ga-id=\"subread\"\">Simulation-based comprehensive benchmarking of RNA-seq aligners<\/a>",null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/26454280\" ga-product=\"publication\" ga-id=\"superfocus\">SUPER-FOCUS: A tool for agile functional analysis of shotgun metagenomic data<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1101/2020.10.27.356907\" ga-product=\"publication\" ga-id=\"svim-asm\"\">DOI:10.1101/2020.10.27.356907<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1186/S13059-019-1911-0\" ga-product=\"publication\" ga-id=\"syri\"\">SyRI: finding genomic rearrangements and local sequence differences from whole-genome assemblies<\/a>",null,null,null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/20460454\" ga-product=\"publication\" ga-id=\"targetfinder\">Targetfinder.org: A resource for systematic discovery of transcription factor target genes<\/a>",null,null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1186/S12911-019-0974-X\" ga-product=\"publication\" ga-id=\"tensorflow\"\">Prediction of cognitive impairment via deep learning trained with multi-center neuropsychological test data<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1101/800177\" ga-product=\"publication\" ga-id=\"tesorter\"\">DOI:10.1101/800177<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1101/622852\" ga-product=\"publication\" ga-id=\"ThermoRawFileParser\"\">DOI:10.1101/622852<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btp120\" ga-product=\"publication\" ga-id=\"tophat\"\">TopHat: Discovering splice junctions with RNA-Seq<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.7490/f1000research.1114334.1\" ga-product=\"publication\" ga-id=\"tophat\"\">DOI:10.7490/f1000research.1114334.1<\/a>",null,null,null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/26447887\" ga-product=\"publication\" ga-id=\"transit\">TRANSIT - A Software Tool for Himar1 TnSeq Analysis<\/a>",null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/25631240\" ga-product=\"publication\" ga-id=\"tpp\">Trans-Proteomic Pipeline, a standardized data processing pipeline for large-scale reproducible proteomics informatics<\/a><br \\><a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/20013374\" ga-product=\"publication\" ga-id=\"tpp\">Trans-proteomic pipeline: a pipeline for proteomic analysis.<\/a><br \\><a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/21082435\" ga-product=\"publication\" ga-id=\"tpp\">Software pipeline and data analysis for MS/MS proteomics: the trans-proteomic pipeline.<\/a><br \\><a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/20101611\" ga-product=\"publication\" ga-id=\"tpp\">A guided tour of the Trans-Proteomic Pipeline<\/a>",null,"<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/9862982\" ga-product=\"publication\" ga-id=\"trf\">Tandem repeats finder: A program to analyze DNA sequences<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/19505945\" ga-product=\"publication\" ga-id=\"trimal\">trimAl: A tool for automated alignment trimming in large-scale phylogenetic analyses<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/gks540\" ga-product=\"publication\" ga-id=\"trimmomatic\"\">RobiNA: A user-friendly, integrated software solution for RNA-Seq-based transcriptomics<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1038/nbt.1883\" ga-product=\"publication\" ga-id=\"trinity\"\">Full-length transcriptome assembly from RNA-Seq data without a reference genome<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1038/nprot.2013.084\" ga-product=\"publication\" ga-id=\"trinity\"\">De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1016/j.celrep.2016.12.063\" ga-product=\"publication\" ga-id=\"trinotate\"\">A Tissue-Mapped Axolotl De Novo Transcriptome Enables Identification of Limb Regeneration Factors<\/a>","<a class=\"publication-title\" href=\"http://www.ncbi.nlm.nih.gov/pubmed/27174935\" ga-product=\"publication\" ga-id=\"trnascan-se\">tRNAscan-SE On-line: integrating search and context for analysis of transfer RNA genes<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/nar/25.5.955\" ga-product=\"publication\" ga-id=\"trnascan-se\"\">DOI:10.1093/nar/25.5.955<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1186/s13059-021-02483-z\" ga-product=\"publication\" ga-id=\"trycycler\"\">Trycycler: consensus long-read assemblies for bacterial genomes<\/a>",null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1101/gr.209601.116\" ga-product=\"publication\" ga-id=\"umi_tools\"\">UMI-tools: Modeling sequencing errors in Unique Molecular Identifiers to improve quantification accuracy<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1371/JOURNAL.PCBI.1005595\" ga-product=\"publication\" ga-id=\"unicycler\"\">Unicycler: Resolving bacterial genome assemblies from short and long sequencing reads<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btw039\" ga-product=\"publication\" ga-id=\"unipept\"\">Unipept web services for metaproteomics analysis<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1021/ACS.JPROTEOME.0C00855\" ga-product=\"publication\" ga-id=\"unipept\"\">Unipept Desktop: A Faster, More Powerful Metaproteomics Results Analysis Tool<\/a>",null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1186/s13059-016-0974-4\" ga-product=\"publication\" ga-id=\"ensembl-vep\"\">The Ensembl Variant Effect Predictor<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btp373\" ga-product=\"publication\" ga-id=\"varscan\"\">VarScan: Variant detection in massively parallel sequencing of individual and pooled samples<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1101/gr.129684.111\" ga-product=\"publication\" ga-id=\"varscan\"\">VarScan 2: Somatic mutation and copy number alteration discovery in cancer by exome sequencing<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1101/023754\" ga-product=\"publication\" ga-id=\"vcflib\"\">DOI:10.1101/023754<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btr330\" ga-product=\"publication\" ga-id=\"vcftools\"\">The variant call format and VCFtools<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1101/gr.074492.107\" ga-product=\"publication\" ga-id=\"velvet\"\">Velvet: Algorithms for de novo short read assembly using de Bruijn graphs<\/a>",null,null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.7717/peerj.2584\" ga-product=\"publication\" ga-id=\"vsearch\"\">VSEARCH: A versatile open source tool for metagenomics<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1089/cmb.2014.0157\" ga-product=\"publication\" ga-id=\"whatshap\"\">WhatsHap: Weighted Haplotype Assembly for Future-Generation Sequencing Reads<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1101/037101\" ga-product=\"publication\" ga-id=\"whatshap\"\">DOI:10.1101/037101<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1038/s41592-022-01457-8\" ga-product=\"publication\" ga-id=\"winnowmap\"\">Long-read mapping to repetitive reference sequences using Winnowmap2<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btaa435\" ga-product=\"publication\" ga-id=\"winnowmap\"\">Weighted minimizer sampling improves long read mapping<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btu813\" ga-product=\"publication\" ga-id=\"workflow4metabolomics\"\">Workflow4Metabolomics: A collaborative research infrastructure for computational metabolomics<\/a><br \\><a class=\"publication-title\" href=\"https://doi.org/10.1016/j.biocel.2017.07.002\" ga-product=\"publication\" ga-id=\"workflow4metabolomics\"\">Create, run, share, publish, and reference your LC–MS, FIA–MS, GC–MS, and NMR data analysis workflows with the Workflow4Metabolomics 3.0 Galaxy online infrastructure for metabolomics<\/a>",null,null,"<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/btq441\" ga-product=\"publication\" ga-id=\"xcms\"\">Correction of mass calibration gaps in liquid chromatography-mass spectrometry metabolomics data<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1101/2020.06.18.160010\" ga-product=\"publication\" ga-id=\"xpore\"\">DOI:10.1101/2020.06.18.160010<\/a>","<a class=\"publication-title\" href=\"https://doi.org/10.1093/bioinformatics/bth092\" ga-product=\"publication\" ga-id=\"xtandem\"\">TANDEM: Matching proteins with tandem mass spectra<\/a>",null,"<a class=\"publication-title\" href=\"https://doi.org/10.1101/2022.06.09.495093\" ga-product=\"publication\" ga-id=\"yahs\"\">DOI:10.1101/2022.06.09.495093<\/a>",null,null],["<a class=\"containers\" href=\"https://biocontainers.pro/tools/ABRicate\" ga-product=\"containers\" ga-id=\"abricate\">ABRicate<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/abyss\" ga-product=\"containers\" ga-id=\"abyss\">abyss<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/amber\" ga-product=\"containers\" ga-id=\"amber\">amber<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/anaconda\" ga-product=\"containers\" ga-id=\"anaconda\">anaconda<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/annotatemyids\" ga-product=\"containers\" ga-id=\"annotatemyids\">annotatemyids<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/antismash\" ga-product=\"containers\" ga-id=\"antismash\">antismash<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/any2fasta\" ga-product=\"containers\" ga-id=\"any2fasta\">any2fasta<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/apollo\" ga-product=\"containers\" ga-id=\"apollo\">apollo<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/aragorn\" ga-product=\"containers\" ga-id=\"aragorn\">aragorn<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/Arriba\" ga-product=\"containers\" ga-id=\"arriba\">Arriba<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/astral\" ga-product=\"containers\" ga-id=\"astral\">astral<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/augustus\" ga-product=\"containers\" ga-id=\"augustus\">augustus<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/autodock_vina\" ga-product=\"containers\" ga-id=\"autodock_vina\">autodock_vina<\/a>",null,null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/bakta\" ga-product=\"containers\" ga-id=\"bakta\">bakta<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/bamtools\" ga-product=\"containers\" ga-id=\"bamtools\">bamtools<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/bamutil\" ga-product=\"containers\" ga-id=\"bamutil_diff\">bamutil<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/bandage\" ga-product=\"containers\" ga-id=\"bandage\">bandage<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/barrnap\" ga-product=\"containers\" ga-id=\"barrnap\">barrnap<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/bbmap\" ga-product=\"containers\" ga-id=\"bbmap\">bbmap<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/gff3_rebase\" ga-product=\"containers\" ga-id=\"bcbiogff\">gff3_rebase<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/bcftools\" ga-product=\"containers\" ga-id=\"bcftools\">bcftools<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/beagle\" ga-product=\"containers\" ga-id=\"beagle\">beagle<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/beagle-lib\" ga-product=\"containers\" ga-id=\"beagle-lib\">beagle-lib<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/beast\" ga-product=\"containers\" ga-id=\"beast\">beast<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/beast2\" ga-product=\"containers\" ga-id=\"beast2\">beast2<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/bed_to_protein_map\" ga-product=\"containers\" ga-id=\"bed_to_protein_map\">bed_to_protein_map<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/bedtools\" ga-product=\"containers\" ga-id=\"bedtools\">bedtools<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/Bellerophon\" ga-product=\"containers\" ga-id=\"bellerophon\">Bellerophon<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/berokka\" ga-product=\"containers\" ga-id=\"berokka\">berokka<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/bio3d\" ga-product=\"containers\" ga-id=\"bio3d\">bio3d<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/biobambam\" ga-product=\"containers\" ga-id=\"biobambam\">biobambam<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/biomformat\" ga-product=\"containers\" ga-id=\"biom-format\">biomformat<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/bioperl\" ga-product=\"containers\" ga-id=\"bioperl\">bioperl<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/biopython\" ga-product=\"containers\" ga-id=\"biopython\">biopython<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/biotransformer\" ga-product=\"containers\" ga-id=\"biotransformer\">biotransformer<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/blast\" ga-product=\"containers\" ga-id=\"blast\">blast<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/blast\" ga-product=\"containers\" ga-id=\"blast+\">blast<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/blast\" ga-product=\"containers\" ga-id=\"blast2\">blast<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/blat\" ga-product=\"containers\" ga-id=\"blat\">blat<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/blockbuster\" ga-product=\"containers\" ga-id=\"Blockbuster\">blockbuster<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/boost\" ga-product=\"containers\" ga-id=\"boost\">boost<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/bowtie\" ga-product=\"containers\" ga-id=\"bowtie\">bowtie<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/bowtie2\" ga-product=\"containers\" ga-id=\"bowtie2\">bowtie2<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/breseq\" ga-product=\"containers\" ga-id=\"breseq\">breseq<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/busco\" ga-product=\"containers\" ga-id=\"busco\">busco<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/bwa\" ga-product=\"containers\" ga-id=\"bwa\">bwa<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/bwa-mem2\" ga-product=\"containers\" ga-id=\"bwa-mem2\">bwa-mem2<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/bx-python\" ga-product=\"containers\" ga-id=\"bx-python\">bx-python<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/C3\" ga-product=\"containers\" ga-id=\"c3s\">C3<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/cactus\" ga-product=\"containers\" ga-id=\"cactus\">cactus<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/camera\" ga-product=\"containers\" ga-id=\"camera\">camera<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/canu\" ga-product=\"containers\" ga-id=\"canu\">canu<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/cap3\" ga-product=\"containers\" ga-id=\"cap3\">cap3<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/cardinal\" ga-product=\"containers\" ga-id=\"cardinal\">cardinal<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/cat_bins\" ga-product=\"containers\" ga-id=\"cat_bins\">cat_bins<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/cd-hit\" ga-product=\"containers\" ga-id=\"cd-hit\">cd-hit<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/CellProfiler\" ga-product=\"containers\" ga-id=\"CellProfiler\">CellProfiler<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/cellxgene\" ga-product=\"containers\" ga-id=\"cellxgene\">cellxgene<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/checkm\" ga-product=\"containers\" ga-id=\"checkm\">checkm<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/chembl\" ga-product=\"containers\" ga-id=\"chembl\">chembl<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/chemfp\" ga-product=\"containers\" ga-id=\"chemfp\">chemfp<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/chipseeker\" ga-product=\"containers\" ga-id=\"chipseeker\">chipseeker<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/chromeister\" ga-product=\"containers\" ga-id=\"chromeister\">chromeister<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/circos\" ga-product=\"containers\" ga-id=\"circos\">circos<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/clinker\" ga-product=\"containers\" ga-id=\"clinker\">clinker<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/clustalo\" ga-product=\"containers\" ga-id=\"clustalo\">clustalo<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/clustalw\" ga-product=\"containers\" ga-id=\"clustalw\">clustalw<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/colabfold\" ga-product=\"containers\" ga-id=\"colabfold_batch\">colabfold<\/a>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/cpat\" ga-product=\"containers\" ga-id=\"cpat\">cpat<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/cuDNN\" ga-product=\"containers\" ga-id=\"cudnn\">cuDNN<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/cufflinks\" ga-product=\"containers\" ga-id=\"cufflinks\">cufflinks<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/cummerbund\" ga-product=\"containers\" ga-id=\"cummeRbund\">cummerbund<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/customprodb\" ga-product=\"containers\" ga-id=\"customprodb\">customprodb<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/cutadapt\" ga-product=\"containers\" ga-id=\"cutadapt\">cutadapt<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/cuteSV\" ga-product=\"containers\" ga-id=\"cutesv\">cuteSV<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/dada2\" ga-product=\"containers\" ga-id=\"dada2\">dada2<\/a>",null,null,null,null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/deeptools\" ga-product=\"containers\" ga-id=\"deeptools\">deeptools<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/dendropy\" ga-product=\"containers\" ga-id=\"dendropy\">dendropy<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/deseq2\" ga-product=\"containers\" ga-id=\"deseq2\">deseq2<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/dexseq\" ga-product=\"containers\" ga-id=\"DEXSeq\">dexseq<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/DIA-NN\" ga-product=\"containers\" ga-id=\"diann\">DIA-NN<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/DIA-Umpire\" ga-product=\"containers\" ga-id=\"diaumpire\">DIA-Umpire<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/diamond\" ga-product=\"containers\" ga-id=\"diamond\">diamond<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/diapasef\" ga-product=\"containers\" ga-id=\"diapysef\">diapasef<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/diffbind\" ga-product=\"containers\" ga-id=\"diffbind\">diffbind<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/dropletutils\" ga-product=\"containers\" ga-id=\"dropletutils\">dropletutils<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/easybuild\" ga-product=\"containers\" ga-id=\"easybuild\">easybuild<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/edger\" ga-product=\"containers\" ga-id=\"edger\">edger<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/edger\" ga-product=\"containers\" ga-id=\"edger-repenrich\">edger<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/eggnog-mapper\" ga-product=\"containers\" ga-id=\"eggnog-mapper\">eggnog-mapper<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/egsea\" ga-product=\"containers\" ga-id=\"egsea\">egsea<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/elastix\" ga-product=\"containers\" ga-id=\"elastix\">elastix<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/emboss\" ga-product=\"containers\" ga-id=\"emboss\">emboss<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/EncyclopeDIA\" ga-product=\"containers\" ga-id=\"encyclopedia\">EncyclopeDIA<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/eupathdb\" ga-product=\"containers\" ga-id=\"eupathdb\">eupathdb<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/exabayes\" ga-product=\"containers\" ga-id=\"exabayes\">exabayes<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/ExaML\" ga-product=\"containers\" ga-id=\"examl\">ExaML<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/exonerate\" ga-product=\"containers\" ga-id=\"exonerate\">exonerate<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/express\" ga-product=\"containers\" ga-id=\"eXpress\">express<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/falcon\" ga-product=\"containers\" ga-id=\"pb-assembly\">falcon<\/a>",null,null,null,null,null,null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/fastani\" ga-product=\"containers\" ga-id=\"fastani\">fastani<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/fastool\" ga-product=\"containers\" ga-id=\"fastool\">fastool<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/fastp\" ga-product=\"containers\" ga-id=\"fastp\">fastp<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/fastqc\" ga-product=\"containers\" ga-id=\"fastqc\">fastqc<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/fasttree\" ga-product=\"containers\" ga-id=\"fasttree\">fasttree<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/fastx-toolkit\" ga-product=\"containers\" ga-id=\"fastx\">fastx-toolkit<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/fastx-toolkit\" ga-product=\"containers\" ga-id=\"fastx_toolkit\">fastx-toolkit<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/fgenesh\" ga-product=\"containers\" ga-id=\"fgenesh\">fgenesh<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/fgsea\" ga-product=\"containers\" ga-id=\"fgsea\">fgsea<\/a>",null,null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/filtlong\" ga-product=\"containers\" ga-id=\"filtlong\">filtlong<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/flashlfq\" ga-product=\"containers\" ga-id=\"flashlfq\">flashlfq<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/Flye\" ga-product=\"containers\" ga-id=\"flye\">Flye<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/foldseek\" ga-product=\"containers\" ga-id=\"foldseek\">foldseek<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/fpocket\" ga-product=\"containers\" ga-id=\"fpocket\">fpocket<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/fraggenescan\" ga-product=\"containers\" ga-id=\"fraggenescan\">fraggenescan<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/freebayes\" ga-product=\"containers\" ga-id=\"freebayes\">freebayes<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/funannotate\" ga-product=\"containers\" ga-id=\"funannotate\">funannotate<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/gaeval\" ga-product=\"containers\" ga-id=\"gaeval\">gaeval<\/a>",null,null,null,null,null,null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/gatk\" ga-product=\"containers\" ga-id=\"gatk\">gatk<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/gblocks\" ga-product=\"containers\" ga-id=\"gblock\">gblocks<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/gcta\" ga-product=\"containers\" ga-id=\"gcta\">gcta<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/gecko\" ga-product=\"containers\" ga-id=\"gecko\">gecko<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/gemini\" ga-product=\"containers\" ga-id=\"gemini\">gemini<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/gemoma\" ga-product=\"containers\" ga-id=\"gemoma\">gemoma<\/a>",null,null,null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/GenomeScope_2.0\" ga-product=\"containers\" ga-id=\"genomescope\">GenomeScope_2.0<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/genometools\" ga-product=\"containers\" ga-id=\"genometools\">genometools<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/getorganelle\" ga-product=\"containers\" ga-id=\"getorganelle\">getorganelle<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/gfastats\" ga-product=\"containers\" ga-id=\"gfastats\">gfastats<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/gffcompare\" ga-product=\"containers\" ga-id=\"gffcompare\">gffcompare<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/gffread\" ga-product=\"containers\" ga-id=\"gffread\">gffread<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/ggplot2\" ga-product=\"containers\" ga-id=\"ggplot2\">ggplot2<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/parallel\" ga-product=\"containers\" ga-id=\"parallel\">parallel<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/gsl\" ga-product=\"containers\" ga-id=\"gsl\">gsl<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/goenrichment\" ga-product=\"containers\" ga-id=\"goenrichment\">goenrichment<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/goseq\" ga-product=\"containers\" ga-id=\"goseq\">goseq<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/graphlan\" ga-product=\"containers\" ga-id=\"graphlan\">graphlan<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/gromacs\" ga-product=\"containers\" ga-id=\"gromacs\">gromacs<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/GTDB-Tk\" ga-product=\"containers\" ga-id=\"gtdb-tk\">GTDB-Tk<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/gubbins\" ga-product=\"containers\" ga-id=\"gubbins\">gubbins<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/hapcut2\" ga-product=\"containers\" ga-id=\"hapcut2\">hapcut2<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/heinz\" ga-product=\"containers\" ga-id=\"heinz\">heinz<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/hgv_david\" ga-product=\"containers\" ga-id=\"hgv_david\">hgv_david<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/hgv_ldtools\" ga-product=\"containers\" ga-id=\"hgv_ldtools\">hgv_ldtools<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/hgv_linktogprofile\" ga-product=\"containers\" ga-id=\"hgv_linkToGProfile\">hgv_linktogprofile<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/hicexplorer\" ga-product=\"containers\" ga-id=\"hicexplorer\">hicexplorer<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/hifiadapterfilt\" ga-product=\"containers\" ga-id=\"hifiadapterfilt\">hifiadapterfilt<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/hifiasm\" ga-product=\"containers\" ga-id=\"hifiasm\">hifiasm<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/hisat2\" ga-product=\"containers\" ga-id=\"hisat2\">hisat2<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/hmmer3\" ga-product=\"containers\" ga-id=\"hmmer\">hmmer3<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/horovod\" ga-product=\"containers\" ga-id=\"horovod\">horovod<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/htseq\" ga-product=\"containers\" ga-id=\"htseq\">htseq<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/htslib\" ga-product=\"containers\" ga-id=\"htslib\">htslib<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/humann\" ga-product=\"containers\" ga-id=\"humann\">humann<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/humann2\" ga-product=\"containers\" ga-id=\"humann2\">humann2<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/HyPhy\" ga-product=\"containers\" ga-id=\"HyPhy\">HyPhy<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/ipa_hifi\" ga-product=\"containers\" ga-id=\"pbipa\">ipa_hifi<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/infernal\" ga-product=\"containers\" ga-id=\"infernal\">infernal<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/igv\" ga-product=\"containers\" ga-id=\"igv\">igv<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/integron_finder\" ga-product=\"containers\" ga-id=\"integron_finder\">integron_finder<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/interproscan_4\" ga-product=\"containers\" ga-id=\"interproscan\">interproscan_4<\/a>",null,null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/ipyrad\" ga-product=\"containers\" ga-id=\"ipyrad\">ipyrad<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/iq-tree\" ga-product=\"containers\" ga-id=\"iq-tree\">iq-tree<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/isee\" ga-product=\"containers\" ga-id=\"isee\">isee<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/isescan\" ga-product=\"containers\" ga-id=\"isescan\">isescan<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/isoformswitchanalyzer\" ga-product=\"containers\" ga-id=\"isoformswitchanalyzer\">isoformswitchanalyzer<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/ivar\" ga-product=\"containers\" ga-id=\"ivar\">ivar<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/iwtomics\" ga-product=\"containers\" ga-id=\"iwtomics\">iwtomics<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/jags\" ga-product=\"containers\" ga-id=\"jags\">jags<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/jasmine-sv\" ga-product=\"containers\" ga-id=\"jasminesv\">jasmine-sv<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/jbrowse\" ga-product=\"containers\" ga-id=\"jbrowse\">jbrowse<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/jellyfish\" ga-product=\"containers\" ga-id=\"jellyfish\">jellyfish<\/a>",null,null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/kallisto\" ga-product=\"containers\" ga-id=\"kallisto\">kallisto<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/kat\" ga-product=\"containers\" ga-id=\"kat\">kat<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/khmer\" ga-product=\"containers\" ga-id=\"khmer\">khmer<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/kmc\" ga-product=\"containers\" ga-id=\"kmc\">kmc<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/kraken\" ga-product=\"containers\" ga-id=\"kraken\">kraken<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/kraken2\" ga-product=\"containers\" ga-id=\"kraken2\">kraken2<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/krona\" ga-product=\"containers\" ga-id=\"krona\">krona<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/lama\" ga-product=\"containers\" ga-id=\"lama\">lama<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/lastz\" ga-product=\"containers\" ga-id=\"lastz\">lastz<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/LFQ-Analyst\" ga-product=\"containers\" ga-id=\"LFQ-Analyst\">LFQ-Analyst<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/limma\" ga-product=\"containers\" ga-id=\"limma\">limma<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/links\" ga-product=\"containers\" ga-id=\"links\">links<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/lofreq\" ga-product=\"containers\" ga-id=\"lofreq\">lofreq<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/longshot\" ga-product=\"containers\" ga-id=\"longshot\">longshot<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/ltr_retriever\" ga-product=\"containers\" ga-id=\"ltr_retriever\">ltr_retriever<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/macs\" ga-product=\"containers\" ga-id=\"macs\">macs<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/macs\" ga-product=\"containers\" ga-id=\"macs2\">macs<\/a>",null,null,null,null,null,null,null,null,null,null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/MAFFT\" ga-product=\"containers\" ga-id=\"mafft\">MAFFT<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/mageck\" ga-product=\"containers\" ga-id=\"MAGeCK\">mageck<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/maker\" ga-product=\"containers\" ga-id=\"maker\">maker<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/maldi_quant\" ga-product=\"containers\" ga-id=\"maldi_quant\">maldi_quant<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/mash\" ga-product=\"containers\" ga-id=\"mash\">mash<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/masurca\" ga-product=\"containers\" ga-id=\"masurca\">masurca<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/matchms\" ga-product=\"containers\" ga-id=\"matchms\">matchms<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/matplotlib\" ga-product=\"containers\" ga-id=\"matplotlib\">matplotlib<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/maxbin\" ga-product=\"containers\" ga-id=\"maxbin\">maxbin<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/maxquant\" ga-product=\"containers\" ga-id=\"maxquant\">maxquant<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/maxquant\" ga-product=\"containers\" ga-id=\"biocontainers-maxquant\">maxquant<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/mcl\" ga-product=\"containers\" ga-id=\"mcl\">mcl<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/mdanalysis\" ga-product=\"containers\" ga-id=\"mdanalysis\">mdanalysis<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/medaka\" ga-product=\"containers\" ga-id=\"medaka\">medaka<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/megahit\" ga-product=\"containers\" ga-id=\"megahit\">megahit<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/merqury\" ga-product=\"containers\" ga-id=\"merqury\">merqury<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/meryl\" ga-product=\"containers\" ga-id=\"meryl\">meryl<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/MetaBAT_2\" ga-product=\"containers\" ga-id=\"metabat\">MetaBAT_2<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/MetaDEGalaxy\" ga-product=\"containers\" ga-id=\"MetaDEGalaxy\">MetaDEGalaxy<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/MetaEuk\" ga-product=\"containers\" ga-id=\"metaeuk\">MetaEuk<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/metaphlan\" ga-product=\"containers\" ga-id=\"metaphlan\">metaphlan<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/metaQuantome\" ga-product=\"containers\" ga-id=\"metaQuantome\">metaQuantome<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/metaspades\" ga-product=\"containers\" ga-id=\"metaspades\">metaspades<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/metawrap\" ga-product=\"containers\" ga-id=\"metawrap\">metawrap<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/methyldackel\" ga-product=\"containers\" ga-id=\"MethylDackel\">methyldackel<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/MIGRATE\" ga-product=\"containers\" ga-id=\"MIGRATE\">MIGRATE<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/minia\" ga-product=\"containers\" ga-id=\"minia\">minia<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/miniasm\" ga-product=\"containers\" ga-id=\"miniasm\">miniasm<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/minimap2\" ga-product=\"containers\" ga-id=\"minimap2\">minimap2<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/miniprot\" ga-product=\"containers\" ga-id=\"miniprot\">miniprot<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/mirdeep2\" ga-product=\"containers\" ga-id=\"mirdeep2\">mirdeep2<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/mitos\" ga-product=\"containers\" ga-id=\"mitos2\">mitos<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/mlst\" ga-product=\"containers\" ga-id=\"mlst\">mlst<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/MMseqs2\" ga-product=\"containers\" ga-id=\"mmseqs2\">MMseqs2<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/mob-suite\" ga-product=\"containers\" ga-id=\"mob-suite\">mob-suite<\/a>",null,null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/moff\" ga-product=\"containers\" ga-id=\"moff\">moff<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/morpheus\" ga-product=\"containers\" ga-id=\"morpheus\">morpheus<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/mothur\" ga-product=\"containers\" ga-id=\"mothur\">mothur<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/mousemine\" ga-product=\"containers\" ga-id=\"mousemine\">mousemine<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/mrbayes\" ga-product=\"containers\" ga-id=\"mrbayes\">mrbayes<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/msconvert\" ga-product=\"containers\" ga-id=\"msconvert\">msconvert<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/msmetaenhancer\" ga-product=\"containers\" ga-id=\"msmetaenhancer\">msmetaenhancer<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/msstats\" ga-product=\"containers\" ga-id=\"msstats\">msstats<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/multiqc\" ga-product=\"containers\" ga-id=\"multiqc\">multiqc<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/mummer\" ga-product=\"containers\" ga-id=\"mummer\">mummer<\/a>",null,null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/Nanocompore\" ga-product=\"containers\" ga-id=\"nanocompore\">Nanocompore<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/nanoplot\" ga-product=\"containers\" ga-id=\"nanoplot\">nanoplot<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/nanopolish\" ga-product=\"containers\" ga-id=\"nanopolish\">nanopolish<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/ncbi_datasets\" ga-product=\"containers\" ga-id=\"ncbi-datasets-cli\">ncbi_datasets<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/nccmp\" ga-product=\"containers\" ga-id=\"nccmp\">nccmp<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/nco\" ga-product=\"containers\" ga-id=\"nco\">nco<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/ncview\" ga-product=\"containers\" ga-id=\"ncview\">ncview<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/netcdf\" ga-product=\"containers\" ga-id=\"netcdf\">netcdf<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/newick_utilities\" ga-product=\"containers\" ga-id=\"newick_utils\">newick_utilities<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/nextflow\" ga-product=\"containers\" ga-id=\"nextflow\">nextflow<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/ngsutils\" ga-product=\"containers\" ga-id=\"ngsutils\">ngsutils<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/ninja\" ga-product=\"containers\" ga-id=\"ninja\">ninja<\/a>",null,null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/obitools\" ga-product=\"containers\" ga-id=\"obitools\">obitools<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/openms\" ga-product=\"containers\" ga-id=\"openms\">openms<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/pangolin\" ga-product=\"containers\" ga-id=\"pangolin\">pangolin<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/paraview\" ga-product=\"containers\" ga-id=\"paraview\">paraview<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/pathview\" ga-product=\"containers\" ga-id=\"pathview\">pathview<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/pblat\" ga-product=\"containers\" ga-id=\"pblat\">pblat<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/seqcode\" ga-product=\"containers\" ga-id=\"pe_histogram\">seqcode<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/pear\" ga-product=\"containers\" ga-id=\"pear\">pear<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/peptideshaker\" ga-product=\"containers\" ga-id=\"peptideshaker\">peptideshaker<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/percolator\" ga-product=\"containers\" ga-id=\"percolator\">percolator<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/pfamscan\" ga-product=\"containers\" ga-id=\"pfamscan\">pfamscan<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/phyloseq\" ga-product=\"containers\" ga-id=\"phyloseq\">phyloseq<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/phyml\" ga-product=\"containers\" ga-id=\"phyml\">phyml<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/picard_tools\" ga-product=\"containers\" ga-id=\"picard\">picard_tools<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/picrust\" ga-product=\"containers\" ga-id=\"picrust\">picrust<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/picrust2\" ga-product=\"containers\" ga-id=\"picrust2\">picrust2<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/pilon\" ga-product=\"containers\" ga-id=\"pilon\">pilon<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/PIPE-T\" ga-product=\"containers\" ga-id=\"pipe_t\">PIPE-T<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/plink\" ga-product=\"containers\" ga-id=\"plink\">plink<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/pnetcdf\" ga-product=\"containers\" ga-id=\"pnetcdf\">pnetcdf<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/poretools\" ga-product=\"containers\" ga-id=\"poretools\">poretools<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/clstutils\" ga-product=\"containers\" ga-id=\"pplacer\">clstutils<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/pretextview\" ga-product=\"containers\" ga-id=\"pretextview\">pretextview<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/gmv\" ga-product=\"containers\" ga-id=\"prodigal\">gmv<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/prokka\" ga-product=\"containers\" ga-id=\"prokka\">prokka<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/purge_dups\" ga-product=\"containers\" ga-id=\"purge_dups\">purge_dups<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/pycoqc\" ga-product=\"containers\" ga-id=\"pycoqc\">pycoqc<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/pygenometracks\" ga-product=\"containers\" ga-id=\"pygenometracks\">pygenometracks<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/pysam\" ga-product=\"containers\" ga-id=\"pysam\">pysam<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/pytorch\" ga-product=\"containers\" ga-id=\"pytorch\">pytorch<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/qiime2\" ga-product=\"containers\" ga-id=\"qiime2\">qiime2<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/qualimap\" ga-product=\"containers\" ga-id=\"qualimap\">qualimap<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/quasitools\" ga-product=\"containers\" ga-id=\"quasitools\">quasitools<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/quast\" ga-product=\"containers\" ga-id=\"quast\">quast<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/Query_Tabular\" ga-product=\"containers\" ga-id=\"query_tabular\">Query_Tabular<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/r\" ga-product=\"containers\" ga-id=\"r\">r<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/Racon\" ga-product=\"containers\" ga-id=\"racon\">Racon<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/ramclustr\" ga-product=\"containers\" ga-id=\"ramclustr\">ramclustr<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/raven\" ga-product=\"containers\" ga-id=\"raven\">raven<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/rawtools\" ga-product=\"containers\" ga-id=\"rawtools\">rawtools<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/raxml\" ga-product=\"containers\" ga-id=\"raxml\">raxml<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/RDKit\" ga-product=\"containers\" ga-id=\"rdkit\">RDKit<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/recon\" ga-product=\"containers\" ga-id=\"recon\">recon<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/recetox-aplcms\" ga-product=\"containers\" ga-id=\"recetox-aplcms\">recetox-aplcms<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/red\" ga-product=\"containers\" ga-id=\"red\">red<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/repeatmasker\" ga-product=\"containers\" ga-id=\"repeatmasker\">repeatmasker<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/repeatmodeler\" ga-product=\"containers\" ga-id=\"repeatmodeler\">repeatmodeler<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/RepeatScout\" ga-product=\"containers\" ga-id=\"repeatscout\">RepeatScout<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/rjags\" ga-product=\"containers\" ga-id=\"rjags\">rjags<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/rmassbank\" ga-product=\"containers\" ga-id=\"rmassbank\">rmassbank<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/rmblast\" ga-product=\"containers\" ga-id=\"rmblast\">rmblast<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/roary\" ga-product=\"containers\" ga-id=\"roary\">roary<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/rsem\" ga-product=\"containers\" ga-id=\"rsem\">rsem<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/rseqc\" ga-product=\"containers\" ga-id=\"rseqc\">rseqc<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/rstudio\" ga-product=\"containers\" ga-id=\"rstudio\">rstudio<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/rtg_core\" ga-product=\"containers\" ga-id=\"rtg-tools\">rtg_core<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/sailfish\" ga-product=\"containers\" ga-id=\"sailfish\">sailfish<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/salmon\" ga-product=\"containers\" ga-id=\"salmon\">salmon<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/SALSA\" ga-product=\"containers\" ga-id=\"salsa\">SALSA<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/sambamba\" ga-product=\"containers\" ga-id=\"sambamba\">sambamba<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/samblaster\" ga-product=\"containers\" ga-id=\"samblaster\">samblaster<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/samtools\" ga-product=\"containers\" ga-id=\"samtools\">samtools<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/scanpy\" ga-product=\"containers\" ga-id=\"scanpy\">scanpy<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/scater\" ga-product=\"containers\" ga-id=\"scater\">scater<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/scikit-image\" ga-product=\"containers\" ga-id=\"scikit-image\">scikit-image<\/a>",null,null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/sepp\" ga-product=\"containers\" ga-id=\"sepp\">sepp<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/seqkit\" ga-product=\"containers\" ga-id=\"seqkit\">seqkit<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/seqtk\" ga-product=\"containers\" ga-id=\"seqtk\">seqtk<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/seurat\" ga-product=\"containers\" ga-id=\"seurat\">seurat<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/shasta\" ga-product=\"containers\" ga-id=\"shasta\">shasta<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/shovill\" ga-product=\"containers\" ga-id=\"shovill\">shovill<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/sicer\" ga-product=\"containers\" ga-id=\"sicer\">sicer<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/simtext\" ga-product=\"containers\" ga-id=\"simtext\">simtext<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/SISTR\" ga-product=\"containers\" ga-id=\"sistr\">SISTR<\/a>",null,null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/Smudgeplots\" ga-product=\"containers\" ga-id=\"Smudgeplots\">Smudgeplots<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/snakemake\" ga-product=\"containers\" ga-id=\"snakemake\">snakemake<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/snap\" ga-product=\"containers\" ga-id=\"snap\">snap<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/sniffles\" ga-product=\"containers\" ga-id=\"sniffles\">sniffles<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/snippy\" ga-product=\"containers\" ga-id=\"snippy\">snippy<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/snpeff\" ga-product=\"containers\" ga-id=\"snpeff\">snpeff<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/snpsift\" ga-product=\"containers\" ga-id=\"snpsift\">snpsift<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/soapdenovo2\" ga-product=\"containers\" ga-id=\"soapdenovo2\">soapdenovo2<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/sortmerna\" ga-product=\"containers\" ga-id=\"sortmerna\">sortmerna<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/spades\" ga-product=\"containers\" ga-id=\"spades\">spades<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/srf\" ga-product=\"containers\" ga-id=\"srf\">srf<\/a>",null,null,null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/sra-tools\" ga-product=\"containers\" ga-id=\"sra-tools\">sra-tools<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/ssw\" ga-product=\"containers\" ga-id=\"ssw\">ssw<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/stacks\" ga-product=\"containers\" ga-id=\"stacks\">stacks<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/star\" ga-product=\"containers\" ga-id=\"star\">star<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/star-fusion\" ga-product=\"containers\" ga-id=\"star_fusion\">star-fusion<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/stringtie\" ga-product=\"containers\" ga-id=\"stringtie\">stringtie<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/subread\" ga-product=\"containers\" ga-id=\"subread\">subread<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/super-focus\" ga-product=\"containers\" ga-id=\"superfocus\">super-focus<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/svim-asm\" ga-product=\"containers\" ga-id=\"svim-asm\">svim-asm<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/SyRI\" ga-product=\"containers\" ga-id=\"syri\">SyRI<\/a>",null,null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/targetfinder.org\" ga-product=\"containers\" ga-id=\"targetfinder\">targetfinder.org<\/a>",null,null,null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/tensorflow\" ga-product=\"containers\" ga-id=\"tensorflow\">tensorflow<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/TEsorter\" ga-product=\"containers\" ga-id=\"tesorter\">TEsorter<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/ThermoRawFileParser\" ga-product=\"containers\" ga-id=\"ThermoRawFileParser\">ThermoRawFileParser<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/tophat\" ga-product=\"containers\" ga-id=\"tophat\">tophat<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/totalview\" ga-product=\"containers\" ga-id=\"totalview\">totalview<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/TransDecoder\" ga-product=\"containers\" ga-id=\"transdecoder\">TransDecoder<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/transit\" ga-product=\"containers\" ga-id=\"transit\">transit<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/TPP_SPC\" ga-product=\"containers\" ga-id=\"tpp\">TPP_SPC<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/trf\" ga-product=\"containers\" ga-id=\"trf\">trf<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/trimal\" ga-product=\"containers\" ga-id=\"trimal\">trimal<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/trim_galore\" ga-product=\"containers\" ga-id=\"trim_galore\">trim_galore<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/trimmomatic\" ga-product=\"containers\" ga-id=\"trimmomatic\">trimmomatic<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/trinity\" ga-product=\"containers\" ga-id=\"trinity\">trinity<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/trinotate\" ga-product=\"containers\" ga-id=\"trinotate\">trinotate<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/trnascan-se\" ga-product=\"containers\" ga-id=\"trnascan-se\">trnascan-se<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/trycycler\" ga-product=\"containers\" ga-id=\"trycycler\">trycycler<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/UCSC_Genome_Browser_Utilities\" ga-product=\"containers\" ga-id=\"ucsc\">UCSC_Genome_Browser_Utilities<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/umi-tools\" ga-product=\"containers\" ga-id=\"umi_tools\">umi-tools<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/unicycler\" ga-product=\"containers\" ga-id=\"unicycler\">unicycler<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/unipept\" ga-product=\"containers\" ga-id=\"unipept\">unipept<\/a>",null,null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/vep\" ga-product=\"containers\" ga-id=\"ensembl-vep\">vep<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/varscan\" ga-product=\"containers\" ga-id=\"varscan\">varscan<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/vcflib\" ga-product=\"containers\" ga-id=\"vcflib\">vcflib<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/vcftools\" ga-product=\"containers\" ga-id=\"vcftools\">vcftools<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/velvet\" ga-product=\"containers\" ga-id=\"velvet\">velvet<\/a>",null,null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/vsearch\" ga-product=\"containers\" ga-id=\"vsearch\">vsearch<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/whatshap\" ga-product=\"containers\" ga-id=\"whatshap\">whatshap<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/winnowmap\" ga-product=\"containers\" ga-id=\"winnowmap\">winnowmap<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/workflow4metabolomics\" ga-product=\"containers\" ga-id=\"workflow4metabolomics\">workflow4metabolomics<\/a>",null,null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/xcms\" ga-product=\"containers\" ga-id=\"xcms\">xcms<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/xpore\" ga-product=\"containers\" ga-id=\"xpore\">xpore<\/a>","<a class=\"containers\" href=\"https://biocontainers.pro/tools/xtandem\" ga-product=\"containers\" ga-id=\"xtandem\">xtandem<\/a>",null,"<a class=\"containers\" href=\"https://biocontainers.pro/tools/yahs\" ga-product=\"containers\" ga-id=\"yahs\">yahs<\/a>",null,null],["GPL-2.0",null,null,null,null,null,"MIT",null,"GPL-3.0",null,"Not licensed","MIT",null,"Artistic-1.0",null,null,null,null,"GPL-3.0","MIT","GPL-3.0","GPL-3.0",null,"GPL-3.0","BSD-3-Clause",null,"MIT","GPL-3.0","GPL-3.0",null,null,null,"Not licensed",null,"GPL-2.0",null,"GPL-3.0",null,null,"GPL-3.0",null,"GPL-3.0","GPL-2.0",null,null,"MIT","LGPL-3.0",null,null,null,null,null,null,"Other",null,"GPL-3.0",null,null,null,null,"MIT","MIT",null,null,"MIT",null,null,"GPL-2.0",null,null,"Artistic-2.0","MIT",null,null,"BSD-3-Clause",null,"MIT","GPL-3.0",null,null,null,"Artistic-2.0","GPL-3.0",null,null,null,null,"MIT","GPL-2.0",null,"MIT",null,null,null,null,null,null,null,null,null,null,null,null,null,null,"GPL-3.0",null,null,null,"BSL-1.0","Artistic-2.0","Artistic-2.0",null,"MIT","MIT","GPL-3.0",null,null,null,null,null,"GPL-3.0","BSD-3-Clause","LGPL-2.1","GPL-3.0",null,"Apache-2.0","AGPL-3.0",null,"Artistic-2.0",null,null,"GPL-3.0",null,"GPL-2.0",null,"GPL-2.0","GPL-2.0","GPL-3.0",null,null,null,null,"Apache-2.0",null,null,null,"GPL-3.0","GPL-3.0","GPL-3.0",null,"Apache-2.0",null,null,null,null,null,null,null,null,null,"Apache-2.0","MIT","MIT","GPL-3.0",null,null,"AGPL-3.0","AGPL-3.0",null,null,"MIT",null,null,null,"GPL-3.0","LGPL-3.0",null,null,null,"GPL-3.0","Freeware","GPL-3.0","MIT",null,"BSD-2-Clause","Not licensed",null,null,null,null,null,null,null,null,null,"MIT","GPL-3.0","MIT","GPL-3.0",null,null,null,null,"Apache-2.0","BSD-3-Clause",null,null,"GPL-3.0",null,"MIT",null,null,"MIT","MIT",null,null,null,"GPL-3.0","GPL-3.0","Apache-2.0","GPL-2.0",null,"MIT","LGPL-2.1","GPL-3.0",null,null,"GPL-2.0",null,null,null,null,null,null,null,"GPL-3.0","MIT",null,"GPL-3.0",null,null,"Other","Apache-2.0","GPL-3.0","MIT","MIT",null,"Unlicense",null,null,null,"BSD-3-Clause",null,"LGPL-2.1","GPL-3.0",null,null,null,null,null,null,"GPL-3.0",null,"MIT",null,"GPL-2.0","AGPL-3.0","GPL-2.0","GPL-2.0","MIT",null,null,null,null,"GPL-3.0",null,null,null,"BSD-2-Clause","GPL-3.0",null,"BSD-3-Clause",null,null,"GFDL-1.3","MIT","Proprietary",null,null,null,null,"GPL-3.0",null,"GPL-2.0","GPL-3.0","MIT","MIT",null,"GPL-3.0","Artistic-2.0","Artistic-2.0",null,null,null,null,null,null,null,null,null,null,null,"BSD-Source-Code",null,"Artistic-2.0","GPL-3.0","CC-BY-4.0",null,null,"Apache-2.0","MIT",null,null,null,null,"GPL-2.0",null,"MPL-2.0",null,null,null,null,null,null,null,"GPL-3.0","MIT",null,null,"MIT","MIT",null,"MIT",null,"CECILL-2.0","MIT",null,"MIT","MIT",null,"GPL-3.0",null,null,"Other","GPL-3.0",null,null,null,null,null,null,"MIT","GPL-3.0",null,"GPL-3.0","Apache-2.0","MIT",null,"GPL-3.0","Artistic-2.0",null,null,null,"GPL-3.0",null,"GPL-3.0","MIT",null,null,null,null,"GPL-2.0",null,"BSD-3-Clause","GPL-1.0",null,null,null,"Apache-2.0","GPL-3.0","MIT",null,null,null,null,null,"BSD-3-Clause",null,null,"GPL-3.0",null,"BSD-3-Clause",null,"GPL-3.0","Unlicense","GPL-3.0","CC-BY-NC-1.0",null,"Apache-2.0",null,null,null,null,null,null,"GPL-3.0","GPL-2.0","MIT",null,"GPL-3.0",null,null,null,"MIT",null,"GPL-2.0",null,null,"Freeware",null,null,null,"GPL-3.0",null,"MIT",null,null,null,null,"MIT",null,"GPL-3.0","GPL-3.0",null,"MIT","BSD-3-Clause","BSD-3-Clause",null,null,"Apache-2.0","GPL-2.0","CC-BY-4.0",null,null,"MIT","GPL-2.0",null,"MIT","Apache-2.0",null,null,null,null,"GPL-2.0",null,"OSL-2.1","OSL-2.1",null,null,null,"GPL-2.0","Artistic-2.0",null,"OSL-2.1",null,null,null,null,null,null,"Other",null,null,"GPL-3.0",null,"MIT",null,null,null,null,null,"MIT","BSD-3-Clause",null,null,"BSD-3-Clause",null,null,null,"GPL-3.0",null,null,null,null,"MIT","MIT",null,"MIT","GPL-3.0",null,null,null,null,null,null,null,"Apache-2.0",null,null,"MIT","GPL-3.0",null,null,null,null,null,"LGPL-3.0","GPL-3.0",null,"GPL-2.0",null,null,null,null,null,null,null,null,"GPL-3.0","GPL-3.0",null,null,"Artistic-2.0","GPL-3.0",null,null,"GPL-3.0",null,null,null,null,null,null,null,null,null,null,null,null,"Apache-2.0",null,"Proprietary",null,null,null,null,null,null,"Other",null,null,null,null,null,null,"GPL-3.0","Other",null,null,"MIT","GPL-3.0","MIT",null,null,null,"Apache-2.0",null,null,null,"MIT","GPL-3.0","GPL-3.0",null,null,null,"GPL-3.0",null,"MIT",null,null,"Not licensed",null,null,null,"GPL-2.0",null,"MIT",null,null,"MIT",null,null],[null,null,null,null,null,null,null,null,null,"<a href=\"https://apollo-portal.genome.edu.au/\" ga-product=\"biocommons\" ga-id=\"apollo\">Apollo Portal<\/a>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<a href=\"https://github.com/AustralianBioCommons/Canu\" ga-product=\"biocommons\" ga-id=\"canu\">Testing & optimisation on Gadi (NCI)<\/a>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<a href=\"https://github.com/AustralianBioCommons/falcon\" ga-product=\"biocommons\" ga-id=\"pb-assembly\">Install, optimisation and testing at Pawsey<\/a>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<a href=\"https://support.pawsey.org.au/documentation/pages/viewpage.action?pageId=68818716\" ga-product=\"biocommons\" ga-id=\"fgenesh\">Fgenesh++<\/a>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<a href=\"https://github.com/AustralianBioCommons/ipa\" ga-product=\"biocommons\" ga-id=\"pbipa\">Install and testing at Pawsey<\/a>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<a href=\"https://github.com/AustralianBioCommons/migrate\" ga-product=\"biocommons\" ga-id=\"MIGRATE\">Install and testing on Gadi (NCI)<\/a>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<a href=\"https://github.com/AustralianBioCommons/xml4ena\" ga-product=\"biocommons\" ga-id=\"xml4ena\">Documentation link<\/a>",null,null,null,null,null,null],["<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fabricate%2Fabricate_list%2F1.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"abricate\">ABRicate List<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fabricate%2Fabricate%2F1.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"abricate\">ABRicate<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fabricate%2Fabricate_summary%2F1.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"abricate\">ABRicate Summary<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fabyss%2Fabyss-pe%2F2.3.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"abyss\">ABySS<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=aggregate_scores_in_intervals2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"aggregate_scores_in_intervals2\">Aggregate datapoints<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fchemteam%2Fambertools_acpype%2Fambertools_acpype%2F21.10%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"amber\">Generate MD topologies for small molecules<\/button><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fanndata_manipulate%2Fanndata_manipulate%2F0.7.5%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"anndata\">Manipulate AnnData<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fanndata_inspect%2Fanndata_inspect%2F0.7.5%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"anndata\">Inspect AnnData<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fanndata_import%2Fanndata_import%2F0.7.5%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"anndata\">Import Anndata and loom<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fannotatemyids%2Fannotatemyids%2F3.17.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"annotatemyids\">annotateMyIDs<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fantismash%2Fantismash%2F6.1.1%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"antismash\">Antismash<\/button><\/ul>",null,null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftrna_prediction%2Faragorn_trna%2F0.6','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"aragorn\">tRNA and tmRNA<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Farriba_get_filters%2Farriba_get_filters%2F2.4.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"arriba\">Arriba Get Filters<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Farriba_draw_fusions%2Farriba_draw_fusions%2F2.4.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"arriba\">Arriba Draw Fusions<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Farriba%2Farriba%2F2.4.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"arriba\">Arriba<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fpadge%2Fastral%2Fastral%2F0.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"astral\">ASTRAL-III. Tool for estimating an unrooted species tree given a set of unrooted gene trees.<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Faugustus_training%2Faugustus_training%2F3.4.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"augustus\">Train Augustus<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Faugustus%2Faugustus%2F3.4.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"augustus\">Augustus<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fautodock_vina%2Fdocking%2F1.2.3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"autodock_vina\">VINA Docking<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fautodock_vina_prepare_receptor%2Fprepare_receptor%2F1.5.7%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"autodock_vina\">Prepare receptor<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fautodock_vina_prepare_box%2Fprepare_box%2F2021.03.5%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"autodock_vina\">Calculate the box parameters using RDKit<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=axt_to_concat_fasta','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"axt_to_concat_fasta\">AXT to concatenated FASTA<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=axt_to_fasta','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"axt_to_fasta\">AXT to FASTA<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=axt_to_lav_1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"axt_to_lav_1\">AXT to LAV<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbakta%2Fbakta%2F1.5.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bakta\">Bakta<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fbamtools_filter%2FbamFilter%2F2.5.2%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bamtools\">Filter BAM<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbamutil_diff%2Fbamutil_diff%2F1.0.15%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bamutil_diff\">BamUtil diff<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbandage%2Fbandage_info%2F2022.09%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bandage\">Bandage Info<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbandage%2Fbandage_image%2F2022.09%2Bgalaxy4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bandage\">Bandage Image<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=barchart_gnuplot','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"barchart_gnuplot\">Bar chart<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbarrnap%2Fbarrnap%2F1.2.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"barrnap\">barrnap<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbbtools_bbduk%2Fbbtools_bbduk%2F39.01%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bbmap\">BBTools: BBduk<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgff3_rebase%2Fgff3.rebase%2F1.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcbiogff\">Rebase GFF3 features<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftbl2gff3%2Ftbl2gff3%2F1.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcbiogff\">Table to GFF3<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fblastxml_to_gapped_gff3%2Fblastxml_to_gapped_gff3%2F1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcbiogff\">BlastXML to gapped GFF3<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">30 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_stats%2Fbcftools_stats%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools stats<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_norm%2Fbcftools_norm%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools norm<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_mpileup%2Fbcftools_mpileup%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools mpileup<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_csq%2Fbcftools_csq%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools csq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_convert_to_vcf%2Fbcftools_convert_to_vcf%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools convert to vcf<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_consensus%2Fbcftools_consensus%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools consensus<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_view%2Fbcftools_view%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools view<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_roh%2Fbcftools_roh%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools roh<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_reheader%2Fbcftools_reheader%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools reheader<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_query_list_samples%2Fbcftools_query_list_samples%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools List Samples<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_query%2Fbcftools_query%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools query<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_plugin_tag2tag%2Fbcftools_plugin_tag2tag%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools tag2tag<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_plugin_setgt%2Fbcftools_plugin_setgt%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools setGT<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_plugin_missing2ref%2Fbcftools_plugin_missing2ref%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools missing2ref<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_plugin_mendelian%2Fbcftools_plugin_mendelian%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools mendelian<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_plugin_impute_info%2Fbcftools_plugin_impute_info%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools impute-info<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_plugin_fixploidy%2Fbcftools_plugin_fixploidy%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools fixploidy<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_plugin_fill_tags%2Fbcftools_plugin_fill_tags%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools fill-tags<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_plugin_fill_an_ac%2Fbcftools_plugin_fill_an_ac%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools fill-AN-AC<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_plugin_dosage%2Fbcftools_plugin_dosage%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools dosage<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_plugin_counts%2Fbcftools_plugin_counts%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools counts<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_merge%2Fbcftools_merge%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools merge<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_isec%2Fbcftools_isec%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools isec<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_gtcheck%2Fbcftools_gtcheck%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools gtcheck<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_filter%2Fbcftools_filter%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools filter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_convert_from_vcf%2Fbcftools_convert_from_vcf%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools convert from vcf<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_concat%2Fbcftools_concat%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools concat<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_cnv%2Fbcftools_cnv%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools cnv<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_call%2Fbcftools_call%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools call<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbcftools_annotate%2Fbcftools_annotate%2F1.15.1%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bcftools\">bcftools annotate<\/button><\/li><\/ul><\/ul>",null,null,null,null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=bed_to_bigBed','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bed_to_bigBed\">BED-to-bigBed<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fbed_to_protein_map%2Fbed_to_protein_map%2F0.2.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bed_to_protein_map\">bed to protein map<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=bed2gff1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bed2gff1\">BED-to-GFF<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">39 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_groupbybed%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools GroupByBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_complementbed%2F2.30.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools ComplementBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_mergebed%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools MergeBED<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_windowbed%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools WindowBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_spacingbed%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools SpacingBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_getfastabed%2F2.30.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools getfasta<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_randombed%2F2.30.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools RandomBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_multicovtbed%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools MultiCovBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_reldistbed%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools ReldistBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_slopbed%2F2.30.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools SlopBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_coveragebed%2F2.30.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools Compute both the depth and breadth of coverage<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_expandbed%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools ExpandBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_sortbed%2F2.30.0%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools SortBED<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_map%2F2.30.0.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools MapBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_shufflebed%2F2.30.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools ShuffleBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_annotatebed%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools AnnotateBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_nucbed%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools NucBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_bedtobam%2F2.30.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools BED to BAM<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_maskfastabed%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools MaskFastaBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_bedtoigv%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools BED to IGV<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_tagbed%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools TagBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_closestbed%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools ClosestBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_subtractbed%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools SubtractBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_multiintersectbed%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools Multiple Intersect<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_intersectbed%2F2.30.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools Intersect intervals<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_genomecoveragebed%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools Genome Coverage<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_overlapbed%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools OverlapBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_jaccard%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools JaccardBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_bedpetobam%2F2.30.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools BEDPE to BAM<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_makewindowsbed%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools MakeWindowsBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_fisher%2F2.30.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools FisherBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_links%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools LinksBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_bed12tobed6%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools BED12 to BED6<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_bamtobed%2F2.30.0%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools BAM to BED<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_unionbedgraph%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools Merge BedGraph files<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_clusterbed%2F2.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools ClusterBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_flankbed%2F2.30.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools FlankBed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbedtools%2Fbedtools_bamtofastq%2F2.27.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">bedtools Convert from BAM to FastQ<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fdeg_annotate%2Fdeg_annotate%2F1.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bedtools\">Annotate DESeq2/DEXSeq output tables<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbellerophon%2Fbellerophon%2F1.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bellerophon\">Filter and merge<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fberokka%2Fberokka%2F0.2.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"berokka\">Berokka<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fimgteam%2Fbfconvert%2Fip_convertimage%2F6.7.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bftools\">Convert image<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fimgteam%2Fimage_info%2Fip_imageinfo%2F0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bftools\">Image Info<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">4 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fchemteam%2Fbio3d_rmsf%2Fbio3d_rmsf%2F2.3.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bio3d\">RMSF Analysis<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fchemteam%2Fbio3d_rmsd%2Fbio3d_rmsd%2F2.3.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bio3d\">RMSD Analysis<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fchemteam%2Fbio3d_pca_visualize%2Fbio3d_pca_visualize%2F2.3.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bio3d\">PCA visualization<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fchemteam%2Fbio3d_pca%2Fbio3d_pca%2F2.3.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bio3d\">PCA<\/button><\/li><\/ul><\/ul>",null,null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbiom_convert%2Fbiom_convert%2F2.1.14%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"biom-format\">Convert<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbiom_add_metadata%2Fbiom_add_metadata%2F2.1.14%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"biom-format\">Add metadata<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fbionano_scaffold%2Fbionano_scaffold%2F3.7.0%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bionano_solve\">Bionano Hybrid Scaffold<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fpeterjc%2Fget_orfs_or_cdss%2Fget_orfs_or_cdss%2F0.2.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"biopython\">Get open reading frames (ORFs) or coding sequences (CDSs)<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Fbiotransformer%2Fbiotransformer%2F3.0_20230403%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"biotransformer\">BioTransformer<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">16 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_convert2blastmask_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast\">NCBI BLAST+ convert2blastmask<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_blastdbcmd_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast\">NCBI BLAST+ blastdbcmd entry(s)<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_makeblastdb%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast\">NCBI BLAST+ makeblastdb<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_blastdbcmd_info%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast\">NCBI BLAST+ database info<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_blastn_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast\">NCBI BLAST+ blastn<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_tblastx_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast\">NCBI BLAST+ tblastx<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fget_species_taxids%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast\">NCBI get species taxids<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_makeprofiledb%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast\">NCBI BLAST+ makeprofiledb<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_blastp_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast\">NCBI BLAST+ blastp<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_tblastn_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast\">NCBI BLAST+ tblastn<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_rpstblastn_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast\">NCBI BLAST+ rpstblastn<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_dustmasker_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast\">NCBI BLAST+ dustmasker<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fblastxml_to_tabular%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast\">BLAST XML to tabular<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_rpsblast_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast\">NCBI BLAST+ rpsblast<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_blastx_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast\">NCBI BLAST+ blastx<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_segmasker_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast\">NCBI BLAST+ segmasker<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">16 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_convert2blastmask_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast+\">NCBI BLAST+ convert2blastmask<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_blastdbcmd_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast+\">NCBI BLAST+ blastdbcmd entry(s)<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_makeblastdb%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast+\">NCBI BLAST+ makeblastdb<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_blastdbcmd_info%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast+\">NCBI BLAST+ database info<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_blastn_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast+\">NCBI BLAST+ blastn<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_tblastx_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast+\">NCBI BLAST+ tblastx<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fget_species_taxids%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast+\">NCBI get species taxids<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_makeprofiledb%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast+\">NCBI BLAST+ makeprofiledb<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_blastp_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast+\">NCBI BLAST+ blastp<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_tblastn_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast+\">NCBI BLAST+ tblastn<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_rpstblastn_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast+\">NCBI BLAST+ rpstblastn<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_dustmasker_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast+\">NCBI BLAST+ dustmasker<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fblastxml_to_tabular%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast+\">BLAST XML to tabular<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_rpsblast_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast+\">NCBI BLAST+ rpsblast<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_blastx_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast+\">NCBI BLAST+ blastx<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_segmasker_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast+\">NCBI BLAST+ segmasker<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">16 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_convert2blastmask_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast2\">NCBI BLAST+ convert2blastmask<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_blastdbcmd_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast2\">NCBI BLAST+ blastdbcmd entry(s)<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_makeblastdb%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast2\">NCBI BLAST+ makeblastdb<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_blastdbcmd_info%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast2\">NCBI BLAST+ database info<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_blastn_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast2\">NCBI BLAST+ blastn<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_tblastx_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast2\">NCBI BLAST+ tblastx<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fget_species_taxids%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast2\">NCBI get species taxids<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_makeprofiledb%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast2\">NCBI BLAST+ makeprofiledb<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_blastp_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast2\">NCBI BLAST+ blastp<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_tblastn_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast2\">NCBI BLAST+ tblastn<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_rpstblastn_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast2\">NCBI BLAST+ rpstblastn<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_dustmasker_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast2\">NCBI BLAST+ dustmasker<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fblastxml_to_tabular%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast2\">BLAST XML to tabular<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_rpsblast_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast2\">NCBI BLAST+ rpsblast<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_blastx_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast2\">NCBI BLAST+ blastx<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fncbi_blast_plus%2Fncbi_segmasker_wrapper%2F2.10.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"blast2\">NCBI BLAST+ segmasker<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frnateam%2Fblockbuster%2Fblockbuster%2F0.1.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Blockbuster\">blockbuster<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frnateam%2Fblockclust%2Fblockclust%2F1.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"BlockClust\">BlockClust<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fbowtie_wrappers%2Fbowtie_wrapper%2F1.2.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bowtie\">Map with Bowtie for Illumina<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fbowtie2%2Fbowtie2%2F2.5.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bowtie2\">Bowtie2<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbp_genbank2gff3%2Fbp_genbank2gff3%2F1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bp_genbank2gff3\">Genbank to GFF3<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbreseq%2Fbreseq%2F0.35.5%2B0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"breseq\">breseq<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnml%2Fbundle_collections%2Fbundle_collection%2F1.3.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bundle_collections\">Bundle Collection<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbusco%2Fbusco%2F5.4.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"busco\">Busco<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fbwa%2Fbwa%2F0.7.17.5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bwa\">Map with BWA<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fbwa%2Fbwa_mem%2F0.7.17.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bwa\">Map with BWA-MEM<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbwa_mem2%2Fbwa_mem2%2F2.2.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bwa-mem2\">BWA-MEM2<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fbwameth%2Fbwameth%2F0.2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"BWA-meth\">bwameth<\/button><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">12 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsubtract_query%2Fsubtract_query1%2F0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bx-python\">Subtract Whole Dataset<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fmerge%2Fgops_merge_1%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bx-python\">Merge<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fjoin%2Fgops_join_1%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bx-python\">Join<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fget_flanks%2Fget_flanks1%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bx-python\">Get flanks<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fflanking_features%2Fflanking_features_1%2F4.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bx-python\">Fetch closest non-overlapping feature<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcoverage%2Fgops_coverage_1%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bx-python\">Coverage<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fconcat%2Fgops_concat_1%2F1.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bx-python\">Concatenate<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fintersect%2Fgops_intersect_1%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bx-python\">Intersect<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcomplement%2Fgops_complement_1%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bx-python\">Complement<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcluster%2Fgops_cluster_1%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bx-python\">Cluster<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsubtract%2Fgops_subtract_1%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bx-python\">Subtract<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fbasecoverage%2Fgops_basecoverage_1%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"bx-python\">Base Coverage<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fclimate%2Fc3s%2Fc3s%2F0.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"c3s\">Copernicus Climate Data Store<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxy-australia%2Fcactus_export%2Fcactus_export%2F2.6.7%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cactus\">Cactus: export<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxy-australia%2Fcactus_cactus%2Fcactus_cactus%2F2.6.4%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cactus\">Cactus<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmmonsoor%2Fcamera_combinexsannos%2Fabims_CAMERA_combinexsAnnos%2F2.2.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"camera\">CAMERA.combinexsAnnos<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Flecorguille%2Fcamera_annotate%2Fabims_CAMERA_annotateDiffreport%2F2.2.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"camera\">CAMERA.annotate<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcanu%2Fcanu%2F2.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"canu\">Canu assembler<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fartbio%2Fcap3%2Fcap3%2F2.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cap3\">cap3<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">9 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fcardinal_preprocessing%2Fcardinal_preprocessing%2F2.10.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cardinal\">MSI preprocessing<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fcardinal_spectra_plots%2Fcardinal_spectra_plots%2F2.10.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cardinal\">MSI plot spectra<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fcardinal_segmentations%2Fcardinal_segmentations%2F2.10.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cardinal\">MSI segmentation<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fcardinal_quality_report%2Fcardinal_quality_report%2F2.10.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cardinal\">MSI Qualitycontrol<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fcardinal_mz_images%2Fcardinal_mz_images%2F2.10.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cardinal\">MSI mz images<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fcardinal_filtering%2Fcardinal_filtering%2F2.10.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cardinal\">MSI filtering<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fcardinal_data_exporter%2Fcardinal_data_exporter%2F2.10.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cardinal\">MSI data exporter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fcardinal_combine%2Fcardinal_combine%2F2.10.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cardinal\">MSI combine<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fcardinal_classification%2Fcardinal_classification%2F2.10.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cardinal\">MSI classification<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcat_summarise%2Fcat_summarise%2F5.2.3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cat_bins\">CAT summarise<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcat_bins%2Fcat_bins%2F5.0.3.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cat_bins\">CAT bins<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcategorize_elements_satisfying_criteria%2Fcategorize_elements_satisfying_criteria%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"categorize_elements_satisfying_criteria\">Categorize Elements<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbebatut%2Fcdhit%2Fcd_hit_est%2F1.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cd-hit\">CD-HIT-EST<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbebatut%2Fcdhit%2Fcd_hit_protein%2F1.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cd-hit\">CD-HIT PROTEIN<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">23 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_track_objects%2Fcp_track_objects%2F3.1.9%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">TrackObjects<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_tile%2Fcp_tile%2F3.1.9%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">Tile<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_save_images%2Fcp_save_images%2F3.1.9%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">SaveImages<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_relate_objects%2Fcp_relate_objects%2F3.1.9%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">RelateObjects<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_overlay_outlines%2Fcp_overlay_outlines%2F3.1.9%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">OverlayOutlines<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_measure_texture%2Fcp_measure_texture%2F3.1.9%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">MeasureTexture<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_measure_object_size_shape%2Fcp_measure_object_size_shape%2F3.1.9%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">MeasureObjectSizeShape<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_measure_object_intensity%2Fcp_measure_object_intensity%2F3.1.9%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">MeasureObjectIntensity<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_measure_image_quality%2Fcp_measure_image_quality%2F3.1.9%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">MeasureImageQuality<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_measure_image_intensity%2Fcp_measure_image_intensity%2F3.1.9%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">MeasureImageIntensity<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_measure_image_area_occupied%2Fcp_measure_image_area_occupied%2F3.1.9%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">MeasureImageAreaOccupied<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_measure_granularity%2Fcp_measure_granularity%2F3.1.9%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">MeasureGranularity<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_mask_image%2Fcp_mask_image%2F3.1.9%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">MaskImage<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_image_math%2Fcp_image_math%2F3.1.9%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">ImageMath<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_identify_primary_objects%2Fcp_identify_primary_objects%2F3.1.9%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">IdentifyPrimaryObjects<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_gray_to_color%2Fcp_gray_to_color%2F3.1.9%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">GrayToColor<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_export_to_spreadsheet%2Fcp_export_to_spreadsheet%2F3.1.9%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">ExportToSpreadsheet<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_enhance_or_suppress_features%2Fcp_enhance_or_suppress_features%2F3.1.9%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">EnhanceOrSuppressFeatures<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_display_data_on_image%2Fcp_display_data_on_image%2F3.1.9%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">DisplayDataOnImage<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_convert_objects_to_image%2Fcp_convert_objects_to_image%2F3.1.9%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">ConvertObjectsToImage<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_common%2Fcp_common%2F3.1.9%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">Starting Modules<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_color_to_gray%2Fcp_color_to_gray%2F3.1.9%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">ColorToGray<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcp_cellprofiler%2Fcp_cellprofiler%2F3.1.9%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CellProfiler\">CellProfiler<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner?tool_id=interactive_tool_cellxgene','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cellxgene\">Interactive CellXgene Environment<\/button><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fchembl%2Fchembl%2F0.10.1%2Bgalaxy4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"chembl\">Search ChEMBL database<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fchembl%2Fchembl_structure_pipeline%2F1.0.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"chembl\">ChEMBL structure pipeline<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">4 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fchemfp%2Fctb_chemfp_mol2fps%2F1.6.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"chemfp\">Molecule to fingerprint<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fchemfp%2Fctb_sdf2fps%2F1.6.1%2B0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"chemfp\">SDF to Fingerprint<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fchemfp%2Fctb_chemfp_butina_clustering%2F1.6.1%2B0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"chemfp\">Taylor-Butina clustering<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fchemfp%2Fctb_chemfp_nxn_clustering%2F1.6.1%2B0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"chemfp\">NxN clustering<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fchemical_data_sources%2Fctb_online_data_fetch%2F0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"chemical_data_sources\">Online data<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fchemical_data_sources%2Fctb_pubchem_download_as_smiles%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"chemical_data_sources\">PubChem Download<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fchemical_data_sources%2Fctb_pubchem_download_assays%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"chemical_data_sources\">PubChem Assay Downloader<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frnateam%2Fchipseeker%2Fchipseeker%2F1.18.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"chipseeker\">ChIPseeker<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fchromeister%2Fchromeister%2F1.5.a%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"chromeister\">Chromeister<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcircexplorer%2Fcircexplorer%2F1.1.9.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"circexplorer\">CIRCexplorer<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">12 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcircos%2Fcircos_interval_to_text%2F0.69.8%2Bgalaxy8','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"circos\">Circos: Interval to Circos Text Labels<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcircos%2Fcircos_aln_to_links%2F0.69.8%2Bgalaxy8','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"circos\">Circos: Alignments to links<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcircos%2Fcircos_wiggle_to_stacked%2F0.69.8%2Bgalaxy8','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"circos\">Circos: Stack bigWigs as Histogram<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcircos%2Fcircos_tableviewer%2F0.69.8%2Bgalaxy8','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"circos\">Circos: Table viewer<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcircos%2Fcircos_gc_skew%2F0.69.8%2Bgalaxy8','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"circos\">GC Skew<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcircos%2Fcircos_interval_to_tile%2F0.69.8%2Bgalaxy8','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"circos\">Circos: Interval to Tiles<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcircos%2Fcircos_wiggle_to_scatter%2F0.69.8%2Bgalaxy8','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"circos\">Circos: bigWig to Scatter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcircos%2Fcircos_resample%2F0.69.8%2Bgalaxy8','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"circos\">Circos: Resample 1/2D data<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcircos%2Fcircos_binlinks%2F0.69.8%2Bgalaxy8','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"circos\">Circos: Link Density Track<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcircos%2Fcircos%2F0.69.8%2Bgalaxy8','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"circos\">Circos<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcircos%2Fcircos_bundlelinks%2F0.69.8%2Bgalaxy8','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"circos\">Circos: Bundle Links<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcircos%2Fcircgraph%2F0.9-RC2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"circos\">Circos Builder<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fclifinder%2Fclifinder%2FCLIFinder%2F0.5.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"clifinder\">CLIFinder<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fclimate%2Fclimate_stripes%2Fclimate_stripes%2F1.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"climate_stripes\">climate stripes<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fthanhlv%2Fclinker%2Fclinker%2F0.0.23%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"clinker\">clinker<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fclustalw%2Fclustalw%2F2.1%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"clustalw\">ClustalW<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnml%2Fcollapse_collections%2Fcollapse_dataset%2F5.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"collapse_collections\">Collapse Collection<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcollection_column_join%2Fcollection_column_join%2F0.0.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"collection_column_join\">Column join<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcollection_element_identifiers%2Fcollection_element_identifiers%2F0.0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"collection_element_identifiers\">Extract element identifiers<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcolumn_arrange_by_header%2Fbg_column_arrange_by_header%2F0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"column_arrange_by_header\">Column arrange<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcolumn_maker%2FAdd_a_column1%2F2.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"column_maker\">Compute<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcompose_text_param%2Fcompose_text_param%2F0.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"compose_text_param\">Compose text parameter value<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcompress_file%2Fcompress_file%2F0.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"compress_file\">Compress file(s)<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcompute_motif_frequencies_for_all_motifs%2Fcompute_motif_frequencies_for_all_motifs%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"compute_motif_frequencies_for_all_motifs\">Compute Motif Frequencies For All Motifs<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcompute_motifs_frequency%2Fcompute_motifs_frequency%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"compute_motifs_frequency\">Compute Motif Frequencies<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fartbio%2Fconcatenate_multiple_datasets%2Fcat_multi_datasets%2F1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"concatenate_multiple_datasets\">Concatenate multiple datasets<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmvdbeek%2Fconcatenate_multiple_datasets%2Fcat_multiple%2F0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"concatenate_multiple_datasets\">Concatenate multiple datasets<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcondense_characters%2FCondense+characters1%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"condense_characters\">Condense<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_neostorezip_to_neostore','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"CONVERTER_neostorezip_to_neostore\">Convert neostore.zip files to neostore<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcorrelation%2Fcor2%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"correlation\">Correlation<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=Count1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Count1\">Count<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fcpat%2Fcpat%2F3.0.4%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cpat\">CPAT<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fclimate%2Fctsm_fates%2Fctsm_fates%2F2.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ctsm_fates\">CTSM/FATES-EMERALD<\/button><\/ul>",null,null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">5 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcuffquant%2Fcuffquant%2F2.2.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cufflinks\">Cuffquant<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcuffnorm%2Fcuffnorm%2F2.2.1.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cufflinks\">Cuffnorm<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcuffmerge%2Fcuffmerge%2F2.2.1.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cufflinks\">Cuffmerge<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcufflinks%2Fcufflinks%2F2.2.1.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cufflinks\">Cufflinks<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcuffcompare%2Fcuffcompare%2F2.2.1.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cufflinks\">Cuffcompare<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcummerbund%2FcummeRbund%2F2.16.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cummeRbund\">cummeRbund<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fcustom_pro_db%2Fcustom_pro_db%2F1.22.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"customprodb\">CustomProDB<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fr-lannes%2Fcut_include_exclude%2Fcut_fr%2F1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cut_include_exclude\">cut_francais<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Flparsons%2Fcutadapt%2Fcutadapt%2F4.4%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cutadapt\">Cutadapt<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcutesv%2Fcutesv%2F1.0.8%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"cutesv\">cuteSV<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">10 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fdada2_seqcounts%2Fdada2_seqCounts%2F1.28%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"dada2\">dada2: sequence counts<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fdada2_removebimeradenovo%2Fdada2_removeBimeraDenovo%2F1.28%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"dada2\">dada2: removeBimeraDenovo<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fdada2_plotqualityprofile%2Fdada2_plotQualityProfile%2F1.28%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"dada2\">dada2: plotQualityProfile<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fdada2_plotcomplexity%2Fdada2_plotComplexity%2F1.28%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"dada2\">dada2: plotComplexity<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fdada2_mergepairs%2Fdada2_mergePairs%2F1.28%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"dada2\">dada2: mergePairs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fdada2_makesequencetable%2Fdada2_makeSequenceTable%2F1.28%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"dada2\">dada2: makeSequenceTable<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fdada2_learnerrors%2Fdada2_learnErrors%2F1.28%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"dada2\">dada2: learnErrors<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fdada2_filterandtrim%2Fdada2_filterAndTrim%2F1.28%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"dada2\">dada2: filterAndTrim<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fdada2_dada%2Fdada2_dada%2F1.28%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"dada2\">dada2: dada<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fdada2_assigntaxonomyaddspecies%2Fdada2_assignTaxonomyAddspecies%2F1.20%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"dada2\">dada2: assignTaxonomy and addSpecies<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fdatamash_ops%2Fdatamash_ops%2F1.8%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"datamash\">Datamash<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fdbbuilder%2Fdbbuilder%2F0.3.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"dbbuilder\">Protein Database Downloader<\/button><\/ul>",null,null,null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">17 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fdeeptools_plot_heatmap%2Fdeeptools_plot_heatmap%2F3.5.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"deeptools\">plotHeatmap<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fdeeptools_multi_bigwig_summary%2Fdeeptools_multi_bigwig_summary%2F3.5.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"deeptools\">multiBigwigSummary<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fdeeptools_compute_matrix%2Fdeeptools_compute_matrix%2F3.5.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"deeptools\">computeMatrix<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fdeeptools_plot_profile%2Fdeeptools_plot_profile%2F3.5.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"deeptools\">plotProfile<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fdeeptools_plot_pca%2Fdeeptools_plot_pca%2F3.5.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"deeptools\">plotPCA<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fdeeptools_plot_fingerprint%2Fdeeptools_plot_fingerprint%2F3.5.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"deeptools\">plotFingerprint<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fdeeptools_plot_enrichment%2Fdeeptools_plot_enrichment%2F3.5.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"deeptools\">plotEnrichment<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fdeeptools_bigwig_compare%2Fdeeptools_bigwig_compare%2F3.5.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"deeptools\">bigwigCompare<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fdeeptools_bam_pe_fragmentsize%2Fdeeptools_bam_pe_fragmentsize%2F3.5.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"deeptools\">bamPEFragmentSize<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fdeeptools_plot_coverage%2Fdeeptools_plot_coverage%2F3.5.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"deeptools\">plotCoverage<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fdeeptools_plot_correlation%2Fdeeptools_plot_correlation%2F3.5.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"deeptools\">plotCorrelation<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fdeeptools_multi_bam_summary%2Fdeeptools_multi_bam_summary%2F3.5.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"deeptools\">multiBamSummary<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fdeeptools_correct_gc_bias%2Fdeeptools_correct_gc_bias%2F3.5.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"deeptools\">correctGCBias<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fdeeptools_compute_matrix_operations%2Fdeeptools_compute_matrix_operations%2F3.5.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"deeptools\">computeMatrixOperations<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fdeeptools_compute_gc_bias%2Fdeeptools_compute_gc_bias%2F3.5.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"deeptools\">computeGCBias<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fdeeptools_bam_coverage%2Fdeeptools_bam_coverage%2F3.5.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"deeptools\">bamCoverage<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fdeeptools_bam_compare%2Fdeeptools_bam_compare%2F3.5.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"deeptools\">bamCompare<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fdeseq2%2Fdeseq2%2F2.11.40.8%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"deseq2\">DESeq2<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fdexseq%2Fdexseq_count%2F1.44%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"DEXSeq\">DEXSeq-Count<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fdexseq%2Fdexseq%2F1.44%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"DEXSeq\">DEXSeq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fdexseq%2Fplotdexseq%2F1.44%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"DEXSeq\">plotDEXSeq<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fdiann%2Fdiann%2F1.8.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"diann\">DIA-NN<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fdia_umpire%2Fdia_umpire_se%2F2.1.3.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"diaumpire\">DIA_Umpire_SE<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fdiamond%2Fbg_diamond%2F2.0.15%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"diamond\">Diamond<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fdiamond%2Fbg_diamond_view%2F2.0.15%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"diamond\">Diamond view<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fdiamond%2Fbg_diamond_makedb%2F2.0.15%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"diamond\">Diamond makedb<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fdiapysef%2Fdiapysef%2F0.3.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"diapysef\">diapysef library generation<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fdiffbind%2Fdiffbind%2F2.10.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"diffbind\">DiffBind<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fdraw_stacked_barplots%2Fdraw_stacked_barplots%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"draw_stacked_barplots\">Draw Stacked Bar Plots<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fdropletutils%2Fdropletutils%2F1.10.0%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"dropletutils\">DropletUtils<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">5 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fdwt_var_perfeature%2Fdwt_var1%2F1.0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"dwt\">Wavelet variance<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fdwt_var_perclass%2Fcompute_p-values_max_variances_feature_occurrences_in_one_dataset_using_discrete_wavelet_transfom%2F1.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"dwt\">Compute P-values and Max Variances for Feature Occurrences<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fdwt_ivc_all%2Fcompute_p-values_second_moments_feature_occurrences_between_two_datasets_using_discrete_wavelet_transfom%2F1.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"dwt\">Compute P-values and Second Moments for Feature Occurrences<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fdwt_cor_avb_all%2Fcompute_p-values_correlation_coefficients_featureA_featureB_occurrences_between_two_datasets_using_discrete_wavelet_transfom%2F1.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"dwt\">Compute P-values and Correlation Coefficients for Occurrences of Two Set of Features<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fdwt_cor_ava_perclass%2Fcompute_p-values_correlation_coefficients_feature_occurrences_between_two_datasets_using_discrete_wavelet_transfom%2F1.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"dwt\">Compute P-values and Correlation Coefficients for Feature Occurrences<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner/data_source_redirect?tool_id=ebi_sra_main','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ebi_sra_main\">EBI SRA<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fedger%2Fedger%2F3.36.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"edger\">edgeR<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fartbio%2Frepenrich%2Fedger-repenrich%2F1.5.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"edger-repenrich\">edgeR-repenrich<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fedger%2Fedger%2F3.36.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"edger-repenrich\">edgeR<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Feggnog_mapper%2Feggnog_mapper%2F2.1.8%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"eggnog-mapper\">eggNOG Mapper<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fegsea%2Fegsea%2F1.20.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"egsea\">EGSEA<\/button><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">107 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+dreg27%2F5.0.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">dreg<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+octanol63%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">octanol<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+wordmatch110%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">wordmatch<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+sixpack90%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">sixpack<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+cpgreport16%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">cpgreport<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+plotorf76%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">plotorf<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+diffseq22%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">diffseq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+needle56%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">needle<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+pepcoil68%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">pepcoil<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+pepinfo69%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">pepinfo<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+revseq82%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">revseq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+sigcleave88%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">sigcleave<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+newcpgseek58%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">newcpgseek<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+pepwindow73%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">pepwindow<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+noreturn60%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">noreturn<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+textsearch98%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">textsearch<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+cusp17%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">cusp<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+showfeat85%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">showfeat<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+tranalign100%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">tranalign<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+geecee41%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">geecee<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+supermatcher95%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">supermatcher<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+codcmp12%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">codcmp<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+etandem33%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">etandem<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+biosed4%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">biosed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+coderet13%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">coderet<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+cai6%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">cai<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+est2genome32%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">est2genome<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+prettyplot79%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">prettyplot<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+nthseq62%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">nthseq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+lindna48%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">lindna<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+degapseq20%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">degapseq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+primersearch81%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">primersearch<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+tcode97%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">tcode<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+fuzznuc37%2F5.0.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">fuzznuc<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+polydot77%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">polydot<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+dotmatcher24%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">dotmatcher<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+garnier40%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">garnier<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+equicktandem31%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">equicktandem<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+fuzzpro38%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">fuzzpro<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+vectorstrip106%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">vectorstrip<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+merger54%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">merger<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+prettyseq80%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">prettyseq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+pepstats71%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">pepstats<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+plotcon75%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">plotcon<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+notseq61%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">notseq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+skipseq91%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">skipseq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+dotpath25%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">dotpath<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+shuffleseq87%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">shuffleseq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+patmatdb67%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">patmatdb<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+isochore47%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">isochore<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+dottup26%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">dottup<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+pepwindowall74%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">pepwindowall<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+matcher52%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">matcher<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+pepwheel72%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">pepwheel<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+maskseq51%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">maskseq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+trimseq103%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">trimseq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+infoseq46%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">infoseq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+backtranseq2%2F6.6.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">backtranseq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+freak36%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">freak<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+hmoment44%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">hmoment<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+sirna89%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">sirna<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+wobble108%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">wobble<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+chips10%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">chips<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+marscan49%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">marscan<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+transeq101%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">transeq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+twofeat104%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">twofeat<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+union105%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">union<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+chaos7%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">chaos<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+pasteseq66%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">pasteseq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+megamerger53%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">megamerger<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+iep45%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">iep<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+newcpgreport57%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">newcpgreport<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+wordcount109%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">wordcount<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+descseq21%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">descseq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+extractseq35%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">extractseq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+einverted28%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">einverted<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+fuzztran39%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">fuzztran<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+cutseq18%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">cutseq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+dan19%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">dan<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+banana3%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">banana<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+compseq14%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">compseq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+splitter92%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">splitter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+tmap99%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">tmap<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+msbar55%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">msbar<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+digest23%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">digest<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+trimest102%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">trimest<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+oddcomp64%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">oddcomp<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+maskfeat50%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">maskfeat<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+seqret84%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">seqret<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+cirdna11%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">cirdna<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+syco96%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">syco<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+btwisted5%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">btwisted<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+extractfeat34%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">extractfeat<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+getorf42%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">getorf<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+newseq59%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">newseq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+antigenic1%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">antigenic<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+charge8%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">charge<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+helixturnhelix43%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">helixturnhelix<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+checktrans9%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">checktrans<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+water107%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">water<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+pepnet70%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">pepnet<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+seqmatchall83%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">seqmatchall<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+preg78%2F5.0.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">preg<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+cai_custom6%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">cai custom<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+epestfind29%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">epestfind<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+cpgplot15%2F5.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">cpgplot<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Femboss_5%2FEMBOSS%3A+palindrome65%2F5.0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"emboss\">palindrome<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fena_upload%2Fena_upload%2F0.6.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ena_upload\">ENA Upload tool<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">4 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fencyclopedia_walnut%2Fencyclopedia_walnut%2F1.2.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"encyclopedia\">Walnut<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fencyclopedia_searchtolib%2Fencyclopedia_searchtolib%2F1.2.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"encyclopedia\">SearchToLib<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fencyclopedia_quantify%2Fencyclopedia_quantify%2F1.12.34%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"encyclopedia\">EncyclopeDIA Quantify<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fencyclopedia_encyclopedia%2Fencyclopedia_encyclopedia%2F1.2.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"encyclopedia\">EncyclopeDIA<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner?tool_id=interactive_tool_ethercalc','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ethercalc\">EtherCalc<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner/data_source_redirect?tool_id=eupathdb','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"eupathdb\">EuPathDB<\/button><\/ul>",null,null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fexonerate%2Fexonerate%2F2.4.0%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"exonerate\">Exonerate<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=export_remote','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"export_remote\">Export datasets<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fexpress%2Fexpress%2F1.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"eXpress\">eXpress<\/button><\/ul>",null,null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffasta_compute_length%2Ffasta_compute_length%2F1.0.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fasta_compute_length\">Compute sequence length<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffasta_concatenate_by_species%2Ffasta_concatenate0%2F0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fasta_concatenate_by_species\">Concatenate<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffasta_filter_by_length%2Ffasta_filter_by_length%2F1.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fasta_filter_by_length\">Filter sequences by length<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Ffasta_merge_files_and_filter_unique_sequences%2Ffasta_merge_files_and_filter_unique_sequences%2F1.2.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fasta_merge_files_and_filter_unique_sequences\">FASTA Merge Files and Filter Unique Sequences<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ffasta_stats%2Ffasta-stats%2F2.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fasta_stats\">Fasta Statistics<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffasta_to_tabular%2Ffasta2tab%2F1.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fasta_to_tabular\">FASTA-to-Tabular<\/button><\/ul>",null,null,null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ffastp%2Ffastp%2F0.23.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastp\">fastp<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastqc%2Ffastqc%2F0.74%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastqc\">FastQC<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ffastqe%2Ffastqe%2F0.2.6%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastqe\">FASTQE<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ffasttree%2Ffasttree%2F2.1.10%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fasttree\">FASTTREE<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">17 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastx_trimmer%2Fcshl_fastx_trimmer%2F1.0.2%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx\">Trim sequences<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastx_reverse_complement%2Fcshl_fastx_reverse_complement%2F1.0.2%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx\">Reverse-Complement<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastx_renamer%2Fcshl_fastx_renamer%2F0.0.14%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx\">Rename sequences<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastx_collapser%2Fcshl_fastx_collapser%2F1.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx\">Collapse<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastx_clipper%2Fcshl_fastx_clipper%2F1.0.3%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx\">Clip<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastx_barcode_splitter%2Fcshl_fastx_barcode_splitter%2F1.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx\">Barcode Splitter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastx_artifacts_filter%2Fcshl_fastx_artifacts_filter%2F1.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx\">Remove sequencing artifacts<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_quality_converter%2Fcshl_fastq_quality_converter%2F1.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx\">Quality format converter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffasta_nucleotide_changer%2Fcshl_fasta_nucleotides_changer%2F1.0.2%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx\">RNA/DNA<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffasta_formatter%2Fcshl_fasta_formatter%2F1.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx\">FASTA Width<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastqtofasta%2Ffastq_to_fasta_python%2F1.1.5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx\">FASTQ to FASTA<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastx_quality_statistics%2Fcshl_fastx_quality_statistics%2F1.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx\">Compute quality statistics<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastx_nucleotides_distribution%2Fcshl_fastx_nucleotides_distribution%2F1.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx\">Draw nucleotides distribution chart<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_quality_filter%2Fcshl_fastq_quality_filter%2F1.0.2%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx\">Filter by quality<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_trimmer_by_quality%2Ffastq_quality_trimmer%2F1.1.5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx\">FASTQ Quality Trimmer<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_to_fasta%2Fcshl_fastq_to_fasta%2F1.0.2%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx\">FASTQ to FASTA<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_quality_boxplot%2Fcshl_fastq_quality_boxplot%2F1.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx\">Draw quality score boxplot<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">9 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_quality_converter%2Fcshl_fastq_quality_converter%2F1.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx_toolkit\">Quality format converter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffasta_nucleotide_changer%2Fcshl_fasta_nucleotides_changer%2F1.0.2%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx_toolkit\">RNA/DNA<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffasta_formatter%2Fcshl_fasta_formatter%2F1.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx_toolkit\">FASTA Width<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffasta_clipping_histogram%2Fcshl_fasta_clipping_histogram%2F1.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx_toolkit\">Length Distribution<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastqtofasta%2Ffastq_to_fasta_python%2F1.1.5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx_toolkit\">FASTQ to FASTA<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_quality_filter%2Fcshl_fastq_quality_filter%2F1.0.2%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx_toolkit\">Filter by quality<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_trimmer_by_quality%2Ffastq_quality_trimmer%2F1.1.5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx_toolkit\">FASTQ Quality Trimmer<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_to_fasta%2Fcshl_fastq_to_fasta%2F1.0.2%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx_toolkit\">FASTQ to FASTA<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_quality_boxplot%2Fcshl_fastq_quality_boxplot%2F1.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fastx_toolkit\">Draw quality score boxplot<\/button><\/li><\/ul><\/ul>",null,null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ffgsea%2Ffgsea%2F1.8.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fgsea\">fgsea<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Ffilter_by_fasta_ids%2Ffilter_by_fasta_ids%2F2.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"filter_by_fasta_ids\">Filter FASTA<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ffilter_tabular%2Ffilter_tabular%2F3.3.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"filter_tabular\">Filter Tabular<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffilter_transcripts_via_tracking%2Ffilter_combined_via_tracking%2F0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"filter_transcripts_via_tracking\">Filter Combined Transcripts<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ffiltlong%2Ffiltlong%2F0.2.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"filtlong\">filtlong<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fflashlfq%2Fflashlfq%2F1.0.3.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"flashlfq\">FlashLFQ<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fflye%2Fflye%2F2.9.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"flye\">Flye<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner/data_source_redirect?tool_id=flymine','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"flymine\">Flymine<\/button><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ffpocket%2Fdpocket%2F4.0.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fpocket\">dpocket<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ffpocket%2Ffpocket%2F4.0.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fpocket\">fpocket<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ffraggenescan%2Ffraggenescan%2F1.30.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"fraggenescan\">FragGeneScan<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffreebayes%2Fbamleftalign%2F1.3.6','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"freebayes\">BamLeftAlign<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffreebayes%2Ffreebayes%2F1.3.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"freebayes\">FreeBayes<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">5 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ffunannotate_compare%2Ffunannotate_compare%2F1.8.15%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"funannotate\">Funannotate compare<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ffunannotate_predict%2Ffunannotate_predict%2F1.8.15%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"funannotate\">Funannotate predict annotation<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ffunannotate_annotate%2Ffunannotate_annotate%2F1.8.15%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"funannotate\">Funannotate functional<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ffunannotate_sort%2Ffunannotate_sort%2F1.8.15%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"funannotate\">Sort assembly<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ffunannotate_clean%2Ffunannotate_clean%2F1.8.15%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"funannotate\">Funannotate assembly clean<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">4 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Faegean_parseval%2Faegean_parseval%2F0.16.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gaeval\">AEGeAn ParsEval<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Faegean_locuspocus%2Faegean_locuspocus%2F0.16.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gaeval\">AEGeAn LocusPocus<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Faegean_gaeval%2Faegean_gaeval%2F0.16.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gaeval\">AEGeAn GAEVAL<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Faegean_canongff3%2Faegean_canongff3%2F0.16.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gaeval\"> AEGeAn CanonGFF3<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">13 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner?tool_id=__UNZIP_COLLECTION__','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_collection_operations\">Unzip collection<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner?tool_id=__ZIP_COLLECTION__','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_collection_operations\">Zip collections<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner?tool_id=__FILTER_FAILED_DATASETS__','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_collection_operations\">Filter failed datasets<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner?tool_id=__FILTER_EMPTY_DATASETS__','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_collection_operations\">Filter empty datasets<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner?tool_id=__FLATTEN__','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_collection_operations\">Flatten collection<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner?tool_id=__MERGE_COLLECTION__','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_collection_operations\">Merge collections<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner?tool_id=__RELABEL_FROM_FILE__','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_collection_operations\">Relabel identifiers<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner?tool_id=__FILTER_FROM_FILE__','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_collection_operations\">Filter collection<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner?tool_id=__SORTLIST__','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_collection_operations\">Sort collection<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner?tool_id=__TAG_FROM_FILE__','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_collection_operations\">Tag elements<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner?tool_id=__APPLY_RULES__','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_collection_operations\">Apply rules<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner?tool_id=__BUILD_LIST__','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_collection_operations\">Build list<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner?tool_id=__EXTRACT_DATASET__','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_collection_operations\">Extract dataset<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">60 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_bz2_to_uncompressed','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert compressed file to uncompressed.<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_uncompressed_to_gz','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert uncompressed file to compressed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_gz_to_uncompressed','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert compressed file to uncompressed.<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=csv_to_tabular','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert CSV to tabular<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_inchi_to_mol','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert InChI to MOL<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_fasta_to_2bit','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert FASTA to 2bit<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_bam_to_qname_sorted_bam','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert BAM to queryname-sorted BAM<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_bed_gff_or_vcf_to_bigwig_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert BED, GFF, or VCF to BigWig<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_interval_to_bgzip_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert Interval to BGZIP<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_molecules','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">OpenBabel converter for molecular formats<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_cram_to_bam_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert CRAM to BAM<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=pbed2ldindepconvert','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert plink pbed to ld reduced format<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_Gro_to_Pdb_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert GRO to PDB<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_interval_to_tabix_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert Interval to tabix<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_bcf_uncompressed_to_bcf','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert compressed and uncompressed BCF files<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_wig_to_bigwig','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert Wiggle to BigWig<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_gff_to_interval_index_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert GFF to Interval Index<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_fastq_to_fqtoc0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert FASTQ files to seek locations<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_biom','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert Biom datasets<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_interval_to_bed_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert Genomic Intervals To BED<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_mdconvert','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert XTC, DCD, and TRR<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_bam_to_bigwig_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert BAM to BigWig<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_SMILES_to_MOL','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert SMILES to MOL<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_Pdb_to_Gro_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert PDB to GRO<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_fasta_to_fai','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert FASTA to fai file<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=tabular_to_csv','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert tabular to CSV<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=tabular_to_dbnsfp','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert tabular to dbnsfp<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_bedgraph_to_bigwig','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert BedGraph to BigWig<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_interval_to_bedstrict_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert Genomic Intervals To Strict BED<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_ref_to_seq_taxomony','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert Ref taxonomy to Seq Taxonomy<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_bed_to_gff_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert BED to GFF<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_smiles_to_smiles','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">SMILES to SMILES<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_len_to_linecount','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert Len file to Linecount<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_fasta_to_tabular','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert FASTA to Tabular<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_wiggle_to_interval_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert Wiggle to Interval<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_sam_to_unsorted_bam','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert SAM to BAM without sorting<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_maf_to_fasta_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert MAF to Fasta<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_gff_to_bed_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert GFF to BED<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_maf_to_interval_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert MAF to Genomic Intervals<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_cml_to_smiles','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert CML to SMILES<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_interval_to_bed6_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert Genomic Intervals To Strict BED6<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_gff_to_fli_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert GFF to Feature Location Index<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_fasta_to_len','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert FASTA to len file<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_tar_to_directory','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert tar to directory<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=lped2pbedconvert','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert lped to plink pbed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_Bam_Bai_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert Bam to Bai<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_interval_to_bed12_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert Genomic Intervals To Strict BED12<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_parquet_to_csv','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert Parquet to csv<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_bam_to_coodinate_sorted_bam','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert BAM to coordinate-sorted BAM<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_fasta_to_bowtie_base_index','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert FASTA to Bowtie base space Index<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_mol2_to_mol','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert MOL2 to MOL<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_vcf_bgzip_to_tabix_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert BGZ VCF to tabix<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_interval_to_bigwig_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert Genomic Intervals To Coverage<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=lped2fpedconvert','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert lped to fped<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_bed_to_fli_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert BED to Feature Location Index<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=pbed2lpedconvert','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert plink pbed to linkage lped<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_fasta_to_bowtie_color_index','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert FASTA to Bowtie color space Index<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=bigwigtowig','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert BigWig to Wiggle<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_sam_to_bigwig_0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert SAM to BigWig<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=CONVERTER_picard_interval_list_to_bed6','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Galaxy CONVERTER\">Convert Picard Interval List to BED6<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">7 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=Filter1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_filter_and_sort\">Filter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=sort1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_filter_and_sort\">Sort<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=Grep1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_filter_and_sort\">Select<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=Extract_features1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_filter_and_sort\">Extract features<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=gff_filter_by_attribute','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_filter_and_sort\">Filter GFF data by attribute<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=gff_filter_by_feature_count','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_filter_and_sort\">Filter GFF data by feature count<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=gtf_filter_by_attribute_values_list','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_filter_and_sort\">Filter GTF data by attribute values_list<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">5 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fdatamash_transpose%2Fdatamash_transpose%2F1.8%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_join_subtract_and_group\">Transpose<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fdatamash_reverse%2Fdatamash_reverse%2F1.8%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_join_subtract_and_group\">Reverse<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=join1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_join_subtract_and_group\">Join two Datasets<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=comp1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_join_subtract_and_group\">Compare two Datasets<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=Grouping1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_join_subtract_and_group\">Group<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">18 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fregex_find_replace%2FregexColumn1%2F1.0.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_text_manipulation\">Column Regex Find And Replace<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fregex_find_replace%2Fregex1%2F1.0.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_text_manipulation\">Regex Find And Replace<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fadd_value%2FaddValue%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_text_manipulation\">Add column<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=addValue','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_text_manipulation\">Add column<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=cat1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_text_manipulation\">Concatenate datasets<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=Cut1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_text_manipulation\">Cut<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=mergeCols1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_text_manipulation\">Merge Columns<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=Convert+characters1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_text_manipulation\">Convert<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=createInterval','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_text_manipulation\">Create single interval<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=ChangeCase','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_text_manipulation\">Change Case<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=Paste1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_text_manipulation\">Paste<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=Remove+beginning1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_text_manipulation\">Remove beginning<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=random_lines1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_text_manipulation\">Select random lines<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=Show+beginning1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_text_manipulation\">Select first<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=Show+tail1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_text_manipulation\">Select last<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=trimmer','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_text_manipulation\">Trim<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=wc_gnu','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_text_manipulation\">Line/Word/Character count<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=secure_hash_message_digest','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_text_manipulation\">Secure Hash / Message Digest<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">13 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_trimmer%2Ffastq_trimmer%2F1.1.5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_sequence_utils\">FASTQ Trimmer<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_stats%2Ffastq_stats%2F1.1.5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_sequence_utils\">FASTQ Summary Statistics<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_paired_end_splitter%2Ffastq_paired_end_splitter%2F1.1.5%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_sequence_utils\">FASTQ splitter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_paired_end_joiner%2Ffastq_paired_end_joiner%2F2.0.1.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_sequence_utils\">FASTQ joiner<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_paired_end_interlacer%2Ffastq_paired_end_interlacer%2F1.2.0.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_sequence_utils\">FASTQ interlacer<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_paired_end_deinterlacer%2Ffastq_paired_end_deinterlacer%2F1.1.5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_sequence_utils\">FASTQ de-interlacer<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_masker_by_quality%2Ffastq_masker_by_quality%2F1.1.5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_sequence_utils\">FASTQ Masker<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_manipulation%2Ffastq_manipulation%2F1.1.5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_sequence_utils\">Manipulate FASTQ<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_groomer%2Ffastq_groomer%2F1.1.5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_sequence_utils\">FASTQ Groomer<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_filter%2Ffastq_filter%2F1.1.5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_sequence_utils\">Filter FASTQ<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_combiner%2Ffastq_combiner%2F1.1.5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_sequence_utils\">Combine FASTA and QUAL<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ftabular_to_fastq%2Ftabular_to_fastq%2F1.1.5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_sequence_utils\">Tabular to FASTQ<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_to_tabular%2Ffastq_to_tabular%2F1.1.5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy_sequence_utils\">FASTQ to Tabular<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fcount_gff_features%2Fcount_gff_features%2F0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"galaxy-ops\">Count GFF Features<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fearlhaminst%2Fgblocks%2Fgblocks%2F0.91b','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gblock\">Gblocks<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgecko%2Fgecko%2F1.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gecko\">Gecko<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">24 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_windower%2Fgemini_windower%2F0.20.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI windower<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_stats%2Fgemini_stats%2F0.20.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI stats<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_set_somatic%2Fgemini_set_somatic%2F0.20.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI set_somatic<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_roh%2Fgemini_roh%2F0.20.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI roh<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_region%2Fgemini_region%2F0.18.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI region<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_recessive_and_dominant%2Fgemini_recessive_and_dominant%2F0.18.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI autosomal recessive/dominant<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_query%2Fgemini_query%2F0.20.1%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI query<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_qc%2Fgemini_qc%2F0.20.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI qc<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_pathways%2Fgemini_pathways%2F0.20.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI pathways<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_mendel_errors%2Fgemini_mendel_errors%2F0.18.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI mendel_errors<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_lof_sieve%2Fgemini_lof_sieve%2F0.20.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI lof_sieve<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_load%2Fgemini_load%2F0.20.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI load<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_interactions%2Fgemini_interactions%2F0.20.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI interactions<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_inheritance%2Fgemini_inheritance%2F0.20.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI inheritance pattern<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_gene_wise%2Fgemini_gene_wise%2F0.20.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI gene_wise<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_fusions%2Fgemini_fusions%2F0.20.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI fusions<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_dump%2Fgemini_dump%2F0.18.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI dump<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_de_novo%2Fgemini_de_novo%2F0.18.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI de_novo<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_db_info%2Fgemini_db_info%2F0.20.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI database info<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_comp_hets%2Fgemini_comp_hets%2F0.18.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI comp_hets<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_burden%2Fgemini_burden%2F0.20.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI burden<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_annotate%2Fgemini_annotate%2F0.20.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI annotate<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_amend%2Fgemini_amend%2F0.20.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI amend<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgemini_actionable_mutations%2Fgemini_actionable_mutations%2F0.20.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gemini\">GEMINI actionable_mutations<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=gene2exon1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gene2exon1\">Gene BED To Exon/Intron/Codon BED<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=GeneBed_Maf_Fasta2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"GeneBed_Maf_Fasta2\">Stitch Gene blocks<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=testtoolshed.g2.bx.psu.edu%2Frepos%2Fjchung%2Fgenerate_count_matrix%2Fgenerate_count_matrix%2F1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"generate_count_matrix\">Generate count matrix<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fgenerate_pc_lda_matrix%2Fgenerate_matrix_for_pca_and_lda1%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"generate_pc_lda_matrix\">Generate A Matrix<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgenomescope%2Fgenomescope%2F2.0%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"genomescope\">GenomeScope<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgenrich%2Fgenrich%2F0.5%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"genrich\">Genrich<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fget_pdb%2Fget_pdb%2F0.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"get_pdb_file\">Get PDB file<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgetorganelle%2Fget_organelle_from_reads%2F1.7.7.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"getorganelle\">Get organelle from reads<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgfa_to_fa%2Fgfa_to_fa%2F0.1.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gfa_to_fa\">GFA to FASTA<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fgfastats%2Fgfastats%2F1.3.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gfastats\">gfastats<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=gff2bed1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gff2bed1\">GFF-to-BED<\/button><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fgffcompare_to_bed%2Fgffcompare_to_bed%2F0.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gffcompare\">Convert gffCompare annotated GTF to BED<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgffcompare%2Fgffcompare%2F0.12.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gffcompare\">GffCompare<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fgffread%2Fgffread%2F2.2.1.3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gffread\">gffread<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fggplot2_heatmap2%2Fggplot2_heatmap2%2F3.1.3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ggplot2\">heatmap2<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fggplot2_point%2Fggplot2_point%2F3.4.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ggplot2\">Scatterplot with ggplot2<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fggplot2_heatmap%2Fggplot2_heatmap%2F3.4.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ggplot2\">Heatmap w ggplot<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fgmaj%2Fgmaj_1%2F2.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"GMAJ\">GMAJ<\/button><\/ul>",null,null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgoslimmer%2Fgoslimmer%2F1.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"goenrichment\">GOSlimmer<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgoenrichment%2Fgoenrichment%2F2.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"goenrichment\">GOEnrichment<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgoseq%2Fgoseq%2F1.50.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"goseq\">goseq<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner/data_source_redirect?tool_id=gramenemart','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gramenemart\">GrameneMart<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgraphlan_annotate%2Fgraphlan_annotate%2F1.1.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"graphlan\">Generation, personalization and annotation of tree<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgraphlan%2Fgraphlan%2F1.1.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"graphlan\">GraPhlAn<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fexport2graphlan%2Fexport2graphlan%2F0.20%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"graphlan\">Export to GraPhlAn<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">7 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fchemteam%2Fgmx_merge_topology_files%2Fgmx_merge_topology_files%2F3.4.3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gromacs\">Merge GROMACS topologies<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fchemteam%2Fgmx_editconf%2Fgmx_editconf%2F2022%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gromacs\">GROMACS structure configuration<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fchemteam%2Fgmx_energy%2Fgmx_energy%2F2022%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gromacs\">Extract energy components with GROMACS<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fchemteam%2Fgmx_em%2Fgmx_em%2F2022%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gromacs\">GROMACS energy minimization<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fchemteam%2Fgmx_solvate%2Fgmx_solvate%2F2022%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gromacs\">GROMACS solvation and adding ions<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fchemteam%2Fgmx_sim%2Fgmx_sim%2F2022%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gromacs\">GROMACS simulation<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fchemteam%2Fgmx_setup%2Fgmx_setup%2F2022%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gromacs\">GROMACS initial setup<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgtdbtk_classify_wf%2Fgtdbtk_classify_wf%2F2.2.2%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gtdb-tk\">GTDB-Tk Classify genomes<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=gtf2bedgraph','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gtf2bedgraph\">GTF-to-BEDGraph<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fgtf2gene_list%2F_ensembl_gtf2gene_list%2F1.52.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gtf2gene_list\">GTF2GeneList<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgubbins%2Fgubbins%2F3.2.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"gubbins\">Gubbins<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxy-australia%2Fhapcut2%2Fhapcut2%2F1.3.3%2Bgalaxy0%2Bga1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hapcut2\">Hapcut2<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner/data_source_redirect?tool_id=hbvar','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hbvar\">HbVar<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">4 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fheinz%2Fheinz_scoring%2F1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"heinz\">Calculate a Heinz score<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fheinz%2Fheinz_visualization%2F0.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"heinz\">Visualize<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fheinz%2Fheinz_bum%2F1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"heinz\">Fit a BUM model<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fheinz%2Fheinz%2F1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"heinz\">Identify optimal scoring subnetwork<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=hgv_david','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hgv_david\">DAVID<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=hgv_ldtools','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hgv_ldtools\">LD<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=hgv_linkToGProfile','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hgv_linkToGProfile\">g:Profiler<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">5 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fhicexplorer_hicplotmatrix%2Fhicexplorer_hicplotmatrix%2F3.6%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hicexplorer\">hicPlotMatrix<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fhicexplorer_hicpca%2Fhicexplorer_hicpca%2F3.6%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hicexplorer\">hicPCA<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fhicexplorer_hicmergematrixbins%2Fhicexplorer_hicmergematrixbins%2F3.6%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hicexplorer\">hicMergeMatrixBins<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fhicexplorer_hicfindtads%2Fhicexplorer_hicfindtads%2F3.6%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hicexplorer\">hicFindTADs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fhicexplorer_hicbuildmatrix%2Fhicexplorer_hicbuildmatrix%2F3.6%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hicexplorer\">hicBuildMatrix<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxy-australia%2Fhifiadapterfilt%2Fhifiadapterfilt%2F2.0.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hifiadapterfilt\">HiFi Adapter Filter<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fhifiasm%2Fhifiasm%2F0.19.5%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hifiasm\">Hifiasm<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxy-australia%2Fhifiasm_meta%2Fhifiasm_meta%2F0.3.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hifiasm_meta\">Hifiasm_meta<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhisat2%2Fhisat2%2F2.2.1%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hisat2\">HISAT2<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fhistogram%2Fhistogram_rpy%2F1.0.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"histogram\">Histogram<\/button><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">12 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhmmer3%2Fhmmer_hmmconvert%2F0.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hmmer\">hmmconvert<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhmmer3%2Fhmmer_hmmalign%2F0.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hmmer\">hmmalign<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhmmer3%2Fhmmer_phmmer%2F0.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hmmer\">phmmer<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhmmer3%2Fhmmer_nhmmer%2F0.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hmmer\">nhmmer<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhmmer3%2Fhmmer_alimask%2F0.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hmmer\">alimask<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhmmer3%2Fhmmer_hmmscan%2F0.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hmmer\">hmmscan<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhmmer3%2Fhmmer_jackhmmer%2F0.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hmmer\">jackhmmer<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhmmer3%2Fhmmer_hmmsearch%2F0.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hmmer\">hmmsearch<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhmmer3%2Fhmmer_nhmmscan%2F0.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hmmer\">nhmmscan<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhmmer3%2Fhmmer_hmmbuild%2F0.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hmmer\">hmmbuild<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhmmer3%2Fhmmer_hmmfetch%2F0.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hmmer\">hmmfetch<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhmmer3%2Fhmmer_hmmemit%2F0.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"hmmer\">hmmemit<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Flparsons%2Fhtseq_count%2Fhtseq_count%2F0.9.1%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"htseq\">htseq-count<\/button><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">6 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhumann_unpack_pathways%2Fhumann_unpack_pathways%2F3.8%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"humann\">Unpack pathway abundances<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhumann_split_stratified_table%2Fhumann_split_stratified_table%2F3.8%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"humann\">Split a HUMAnN table<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhumann_renorm_table%2Fhumann_renorm_table%2F3.8%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"humann\">Renormalize<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhumann_rename_table%2Fhumann_rename_table%2F3.7%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"humann\">Rename features<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhumann_regroup_table%2Fhumann_regroup_table%2F3.7%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"humann\">Regroup<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhumann%2Fhumann%2F3.7%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"humann\">HUMAnN<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">7 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhumann2_join_tables%2Fhumann2_join_tables%2F0.11.1.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"humann2\">Join<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhumann2_unpack_pathways%2Fhumann2_unpack_pathways%2F0.11.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"humann2\">Unpack pathway abundances to show genes included<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhumann2_renorm_table%2Fhumann2_renorm_table%2F0.11.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"humann2\">Renormalize<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhumann2_regroup_table%2Fhumann2_regroup_table%2F0.11.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"humann2\">Regroup<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhumann2_genefamilies_genus_level%2Fhumann2_genefamilies_genus_level%2F0.11.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"humann2\">Create a genus level gene families file<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhumann2%2Fhumann2%2F0.11.1.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"humann2\">HUMAnN2<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbebatut%2Fcombine_metaphlan2_humann2%2Fcombine_metaphlan2_humann2%2F0.2.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"humann2\">Combine MetaPhlAn2 and HUMAnN2 outputs<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhyphy_gard%2Fhyphy_gard%2F2.5.47%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"HyPhy\">HyPhy-GARD<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fhyphy_absrel%2Fhyphy_absrel%2F2.5.47%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"HyPhy\">HyPhy-aBSREL<\/button><\/li><\/ul><\/ul>",null,null,null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">6 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Finfernal%2Finfernal_cmscan%2F1.1.4%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"infernal\">cmscan<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Finfernal%2Finfernal_cmalign%2F1.1.4%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"infernal\">cmalign<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Finfernal%2Finfernal_cmsearch%2F1.1.4%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"infernal\">cmsearch<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Finfernal%2Finfernal_cmpress%2F1.1.4%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"infernal\">cmpress<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Finfernal%2Finfernal_cmbuild%2F1.1.4%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"infernal\">cmbuild<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Finfernal%2Finfernal_cmstat%2F1.1.4%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"infernal\">cmstat<\/button><\/li><\/ul><\/ul>",null,null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fintegron_finder%2Fintegron_finder%2F2.0.2%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"integron_finder\">Integron Finder<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner/data_source_redirect?tool_id=intermine','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"intermine\">InterMine<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fintermine_galaxy_exchange%2Fgalaxy_intermine_exchange%2F0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"intermine_galaxy_exchange\">Create InterMine Interchange<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Finterproscan%2Finterproscan%2F5.59-91.0%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"interproscan\">InterProScan<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=Interval_Maf_Merged_Fasta2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Interval_Maf_Merged_Fasta2\">Stitch MAF blocks<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=Interval2Maf_pairwise1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Interval2Maf_pairwise1\">Extract Pairwise MAF blocks<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=Interval2Maf1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Interval2Maf1\">Extract MAF blocks<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fiqtree%2Fiqtree%2F2.1.2%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"iq-tree\">IQ-TREE<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner?tool_id=interactive_tool_isee','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"isee\">iSEE<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fisescan%2Fisescan%2F1.7.2.3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"isescan\">ISEScan<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fisoformswitchanalyzer%2Fisoformswitchanalyzer%2F1.20.0%2Bgalaxy5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"isoformswitchanalyzer\">IsoformSwitchAnalyzeR<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">6 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fivar_variants%2Fivar_variants%2F1.4.2%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ivar\">ivar variants<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fivar_trim%2Fivar_trim%2F1.4.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ivar\">ivar trim<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fivar_filtervariants%2Fivar_filtervariants%2F1.4.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ivar\">ivar filtervariants<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fivar_removereads%2Fivar_removereads%2F1.4.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ivar\">ivar removereads<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fivar_consensus%2Fivar_consensus%2F1.4.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ivar\">ivar consensus<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fivar_getmasked%2Fivar_getmasked%2F1.2.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ivar\">ivar getmasked<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fiwtomics_testandplot%2Fiwtomics_testandplot%2F1.0.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"iwtomics\">IWTomics Test<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fiwtomics_plotwithscale%2Fiwtomics_plotwithscale%2F1.0.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"iwtomics\">IWTomics Plot with Threshold<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fiwtomics_loadandplot%2Fiwtomics_loadandplot%2F1.0.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"iwtomics\">IWTomics Load<\/button><\/li><\/ul><\/ul>",null,null,null,null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fjbrowse%2Fjbrowse%2F1.16.11%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"jbrowse\">JBrowse<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fjbrowse%2Fjbrowse_to_standalone%2F1.16.11%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"jbrowse\">JBrowse - Data Directory to Standalone<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fjcvi_gff_stats%2Fjcvi_gff_stats%2F0.8.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"jcvi\">Genome annotation statistics<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fjellyfish%2Fjellyfish%2F2.3.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"jellyfish\">jellyfish<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fjoin_files_on_column_fuzzy%2Fjoin_files_on_column_fuzzy%2F1.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"join_files_on_column_fuzzy\">Join two files<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fjq%2Fjq%2F1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"jq\">JQ<\/button><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fkallisto_quant%2Fkallisto_quant%2F0.48.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"kallisto\">Kallisto quant<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fkallisto_pseudo%2Fkallisto_pseudo%2F0.48.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"kallisto\">Kallisto pseudo<\/button><\/li><\/ul><\/ul>",null,null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">8 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fkhmer_partition%2Fkhmer_partition%2F3.0.0a3%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"khmer\">khmer: Sequence partition all-in-one<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fkhmer_normalize_by_median%2Fkhmer_normalize_by_median%2F3.0.0a3%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"khmer\">khmer: Normalize By Median<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fkhmer_filter_below_abundance_cutoff%2Fkhmer_filter_below_abundance_cutoff%2F3.0.0a3%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"khmer\">khmer: Filter reads<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fkhmer_filter_abundance%2Fkhmer_filter_abundance%2F3.0.0a3%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"khmer\">khmer: Filter reads<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fkhmer_extract_partitions%2Fkhmer_extract_partitions%2F3.0.0a3%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"khmer\">khmer: Extract partitions<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fkhmer_count_median%2Fkhmer_count_median%2F3.0.0a3%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"khmer\">khmer: Count Median<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fkhmer_abundance_distribution_single%2Fkhmer_abundance_distribution_single%2F3.0.0a3%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"khmer\">khmer: Abundance Distribution (all-in-one)<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fkhmer_abundance_distribution%2Fkhmer_abundance_distribution%2F3.0.0a3%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"khmer\">khmer: Abundance Distribution<\/button><\/li><\/ul><\/ul>",null,null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">7 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fkrakentools_kreport2krona%2Fkrakentools_kreport2krona%2F1.2%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"kraken\">Krakentools: Convert kraken report file<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fkraken_translate%2Fkraken-translate%2F1.3.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"kraken\">Kraken-translate<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fkraken_report%2Fkraken-mpa-report%2F1.2.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"kraken\">Kraken-mpa-report<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fkraken_report%2Fkraken-report%2F1.3.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"kraken\">Kraken-report<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fkraken_filter%2Fkraken-filter%2F1.3.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"kraken\">Kraken-filter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fkraken%2Fkraken%2F1.3.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"kraken\">Kraken<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fkraken2tax%2FKraken2Tax%2F1.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"kraken\">Convert Kraken<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fkraken2%2Fkraken2%2F2.1.1%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"kraken2\">Kraken2<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fcrs4%2Ftaxonomy_krona_chart%2Ftaxonomy_krona_chart%2F2.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"krona\">Krona pie chart<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fsaskia-hiltemann%2Fkrona_text%2Fkrona-text%2F1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"krona\">Visualize  with Krona<\/button><\/li><\/ul><\/ul>",null,null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Flastz%2Flastz_d_wrapper%2F1.04.22%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"lastz\">LASTZ_D<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Flastz%2Flastz_wrapper_2%2F1.04.22%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"lastz\">LASTZ<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Flastz_paired_reads%2Flastz_paired_reads_wrapper%2F1.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"lastz\">Lastz paired reads<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=lav_to_bed1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"lav_to_bed1\">LAV to BED<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner?tool_id=interactive_tool_lfqanalyst','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"LFQ-Analyst\">LFQ Analyst<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=liftOver1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"liftOver1\">Convert genome coordinates<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Flimma_voom%2Flimma_voom%2F3.50.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"limma\">limma<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Flinks%2Flinks%2F2.0.1%2Bgalaxy%2B1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"links\">LINKS<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">5 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Flofreq_call%2Flofreq_call%2F2.1.5%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"lofreq\">Call variants<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Flofreq_alnqual%2Flofreq_alnqual%2F2.1.5%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"lofreq\">Add LoFreq alignment quality scores<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Flofreq_indelqual%2Flofreq_indelqual%2F2.1.5%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"lofreq\">Insert indel qualities<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Flofreq_viterbi%2Flofreq_viterbi%2F2.1.5%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"lofreq\">Realign reads<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Flofreq_filter%2Flofreq_filter%2F2.1.5%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"lofreq\">Lofreq filter<\/button><\/li><\/ul><\/ul>",null,null,null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">9 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmacs2%2Fmacs2_refinepeak%2F2.2.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"macs\">MACS2 refinepeak<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmacs2%2Fmacs2_bdgdiff%2F2.2.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"macs\">MACS2 bdgdiff<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmacs2%2Fmacs2_bdgpeakcall%2F2.2.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"macs\">MACS2 bdgpeakcall<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmacs2%2Fmacs2_filterdup%2F2.2.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"macs\">MACS2 filterdup<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmacs2%2Fmacs2_predictd%2F2.2.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"macs\">MACS2 predictd<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmacs2%2Fmacs2_callpeak%2F2.2.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"macs\">MACS2 callpeak<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmacs2%2Fmacs2_bdgcmp%2F2.2.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"macs\">MACS2 bdgcmp<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmacs2%2Fmacs2_bdgbroadcall%2F2.2.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"macs\">MACS2 bdgbroadcall<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmacs2%2Fmacs2_randsample%2F2.2.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"macs\">MACS2 randsample<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">10 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmacs2%2Fmacs2_refinepeak%2F2.2.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"macs2\">MACS2 refinepeak<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmacs2%2Fmacs2_bdgdiff%2F2.2.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"macs2\">MACS2 bdgdiff<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmacs2%2Fmacs2_bdgpeakcall%2F2.2.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"macs2\">MACS2 bdgpeakcall<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmacs2%2Fmacs2_filterdup%2F2.2.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"macs2\">MACS2 filterdup<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmacs2%2Fmacs2_predictd%2F2.2.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"macs2\">MACS2 predictd<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmacs2%2Fmacs2_callpeak%2F2.2.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"macs2\">MACS2 callpeak<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmacs2%2Fmacs2_bdgcmp%2F2.2.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"macs2\">MACS2 bdgcmp<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmacs2%2Fmacs2_bdgbroadcall%2F2.2.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"macs2\">MACS2 bdgbroadcall<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmacs2%2Fmacs2_randsample%2F2.2.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"macs2\">MACS2 randsample<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fpjbriggs%2Fmacs21%2Fmacs2_1_peakcalling%2F2.1.2.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"macs2\">MACS2.1.2<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=maf_by_block_number1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"maf_by_block_number1\">Extract MAF by block number<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=MAF_filter','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MAF_filter\">Filter MAF<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=maf_limit_size1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"maf_limit_size1\">Filter MAF blocks<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=MAF_Limit_To_Species1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MAF_Limit_To_Species1\">Filter MAF blocks<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=MAF_Reverse_Complement_1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MAF_Reverse_Complement_1\">Reverse Complement<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=MAF_split_blocks_by_species1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MAF_split_blocks_by_species1\">Split MAF blocks<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=maf_stats1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"maf_stats1\">MAF Coverage Stats<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=MAF_Thread_For_Species1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MAF_Thread_For_Species1\">Join MAF blocks<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=MAF_To_BED1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MAF_To_BED1\">MAF to BED<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=MAF_To_Fasta1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MAF_To_Fasta1\">MAF to FASTA<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=MAF_To_Interval1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MAF_To_Interval1\">MAF to Interval<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frnateam%2Fmafft%2Frbc_mafft%2F7.508%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mafft\">MAFFT<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frnateam%2Fmafft%2Frbc_mafft_add%2F7.508%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mafft\">MAFFT add<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">5 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmageck_test%2Fmageck_test%2F0.5.9.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MAGeCK\">MAGeCKs test<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmageck_pathway%2Fmageck_pathway%2F0.5.9.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MAGeCK\">MAGeCK pathway<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmageck_mle%2Fmageck_mle%2F0.5.9.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MAGeCK\">MAGeCK mle<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmageck_gsea%2Fmageck_gsea%2F0.5.9.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MAGeCK\">MAGeCK GSEA<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmageck_count%2Fmageck_count%2F0.5.9.2.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MAGeCK\">MAGeCK count<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmaker_map_ids%2Fmaker_map_ids%2F2.31.11','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"maker\">Map annotation ids<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmaker%2Fmaker%2F2.31.11%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"maker\">Maker<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fmaldi_quant_preprocessing%2Fmaldi_quant_preprocessing%2F1.22.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"maldi_quant\">MALDIquant preprocessing<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fmaldi_quant_peak_detection%2Fmaldi_quant_peak_detection%2F1.22.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"maldi_quant\">MALDIquant peak detection<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmash%2Fmash_screen%2F2.3%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mash\">mash screen<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmash_sketch%2Fmash_sketch%2F2.3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mash\">mash sketch<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=master2pgSnp','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"master2pgSnp\">MasterVar to pgSnp<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdnbenso%2Fmasurca_simple%2Fmasurca_simple%2F4.0.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"masurca\">MaSuRCA simple<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">11 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Frecetox_xmsannotator_advanced%2Frecetox_xmsannotator_advanced%2F0.10.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"matchms\">recetox-xMSannotator<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Fmatchms_split%2Fmatchms_split%2F0.20.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"matchms\">matchms split library<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Fmatchms_spectral_similarity%2Fmatchms_spectral_similarity%2F0.20.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"matchms\">matchms spectral similarity<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Fmatchms_similarity%2Fmatchms_similarity%2F0.20.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"matchms\">matchms similarity<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Fmatchms_networking%2Fmatchms_networking%2F0.20.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"matchms\">matchms networking<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Fmatchms_metadata_match%2Fmatchms_metadata_match%2F0.20.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"matchms\">matchms metadata match<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Fmatchms_metadata_export%2Fmatchms_metadata_export%2F0.20.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"matchms\">matchms metadata export <\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Fmatchms_formatter%2Fmatchms_formatter%2F0.20.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"matchms\">matchms scores formatter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Fmatchms_fingerprint_similarity%2Fmatchms_fingerprint_similarity%2F0.20.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"matchms\">matchms fingerprint similarity<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Fmatchms_filtering%2Fmatchms_filtering%2F0.20.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"matchms\">matchms filtering<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Fmatchms_convert%2Fmatchms_convert%2F0.20.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"matchms\">matchms convert<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmbernt%2Fmaxbin2%2Fmaxbin2%2F2.2.7%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"maxbin\">MaxBin2<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fmaxquant%2Fmaxquant_mqpar%2F2.0.3.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"maxquant\">MaxQuant (using mqpar.xml)<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fmaxquant%2Fmaxquant%2F2.0.3.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"maxquant\">MaxQuant<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fmaxquant_mqpar%2Fmaxquant_mqpar%2F1.6.10.43','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"maxquant\">MaxQuant (using mqpar.xml)<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fmaxquant%2Fmaxquant_mqpar%2F2.0.3.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"biocontainers-maxquant\">MaxQuant (using mqpar.xml)<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fmaxquant%2Fmaxquant%2F2.0.3.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"biocontainers-maxquant\">MaxQuant<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fmaxquant_mqpar%2Fmaxquant_mqpar%2F1.6.10.43','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"biocontainers-maxquant\">MaxQuant (using mqpar.xml)<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fchemteam%2Fmdanalysis_cosine_analysis%2Fmdanalysis_cosine_analysis%2F1.0.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mdanalysis\">Cosine Content<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fchemteam%2Fmd_converter%2Fmd_slicer%2F1.9.7%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MDTraj\">Slice MD trajectories<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fchemteam%2Fmd_converter%2Fmd_converter%2F1.9.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MDTraj\">MDTraj file converter<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmedaka_variant%2Fmedaka_variant%2F1.7.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"medaka\">medaka variant tool<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmedaka_consensus_pipeline%2Fmedaka_consensus_pipeline%2F1.7.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"medaka\">medaka consensus pipeline<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmedaka_consensus%2Fmedaka_consensus%2F1.7.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"medaka\">medaka consensus tool<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmegahit%2Fmegahit%2F1.2.9%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"megahit\">MEGAHIT<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fmerge_cols%2FmergeCols1%2F1.0.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"merge_cols\">Merge Columns<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmerqury%2Fmerqury%2F1.3%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"merqury\">Merqury<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmeryl%2Fmeryl%2F1.3%2Bgalaxy6','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"meryl\">Meryl<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmetabat2%2Fmetabat2%2F2.15%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"metabat\">MetaBAT2<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner/data_source_redirect?tool_id=metabolicmine','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"metabolicmine\">metabolicMine<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">9 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fqfabrepo%2Fmetadegalaxy_uc2otutable%2Fuclust2otutable%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MetaDEGalaxy\">OTUTable<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fqfabrepo%2Fmetadegalaxy_symmetric_plot%2FsymmetricPlot%2F1.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MetaDEGalaxy\">Symmetric Plot<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fqfabrepo%2Fmetadegalaxy_reheader%2Frename%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MetaDEGalaxy\">reheader<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fqfabrepo%2Fmetadegalaxy_phyloseq_richness%2Fphyloseq_richness%2F1.22.3.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MetaDEGalaxy\">Phyloseq Richness<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fqfabrepo%2Fmetadegalaxy_phyloseq_net%2Fphyloseq_net%2F1.24.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MetaDEGalaxy\">Phyloseq Network Plot<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fqfabrepo%2Fmetadegalaxy_phyloseq_deseq2%2Fphyloseq_DESeq2%2F1.22.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MetaDEGalaxy\">Phyloseq DESeq2<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fqfabrepo%2Fmetadegalaxy_phyloseq_abundance_taxonomy%2Fphyloseq_taxonomy%2F1.22.3.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MetaDEGalaxy\">Phyloseq Abundance Taxonomy<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fqfabrepo%2Fmetadegalaxy_phyloseq_abundance_factor%2Fphyloseq_abundance%2F1.22.3.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MetaDEGalaxy\">Phyloseq Abundance plot<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fqfabrepo%2Fmetadegalaxy_pear_stats%2FpearStat%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MetaDEGalaxy\">PEAR Statistics<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">4 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmetaphlan%2Fmetaphlan%2F4.0.6%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"metaphlan\">MetaPhlAn<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmerge_metaphlan_tables%2Fmerge_metaphlan_tables%2F4.0.6%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"metaphlan\">Merge<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmetaphlan2krona%2Fmetaphlan2krona%2F2.6.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"metaphlan\">Format MetaPhlAn2<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmetaphlan2%2Fmetaphlan2%2F2.6.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"metaphlan\">MetaPhlAn2<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">6 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fmetaquantome_viz%2Fmetaquantome_viz%2F2.0.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"metaQuantome\">metaQuantome: visualize<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fmetaquantome_stat%2Fmetaquantome_stat%2F2.0.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"metaQuantome\">metaQuantome: stat<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fmetaquantome_sample%2Fmetaquantome_sample%2F2.0.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"metaQuantome\">metaQuantome: create samples file<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fmetaquantome_filter%2Fmetaquantome_filter%2F2.0.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"metaQuantome\">metaQuantome: filter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fmetaquantome_expand%2Fmetaquantome_expand%2F2.0.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"metaQuantome\">metaQuantome: expand<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fmetaquantome_db%2Fmetaquantome_db%2F2.0.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"metaQuantome\">metaQuantome: database<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnml%2Fmetaspades%2Fmetaspades%2F3.15.4%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"metaspades\">metaSPAdes<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxy-australia%2Fmetawrapmg_binning%2Fmetawrapmg_binning%2F1.3.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"metawrap\">MetaWRAP<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fpileometh%2Fpileometh%2F0.5.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"MethylDackel\">MethylDackel<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frnateam%2Fmetilene%2Fmetilene%2F0.2.6.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"metilene\">metilene<\/button><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">14 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fwolma%2Fmimodd_main%2Fmimodd_varcall%2F0.1.9','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mimodd\">MiModD Variant Calling<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fwolma%2Fmimodd_main%2Fmimodd_covstats%2F0.1.9','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mimodd\">MiModD Coverage Statistics<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fwolma%2Fmimodd_main%2Fmimodd_header%2F0.1.9','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mimodd\">MiModD Run Annotation<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fwolma%2Fmimodd_main%2Fmimodd_info%2F0.1.9','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mimodd\">MiModD File Information<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fwolma%2Fmimodd_main%2Fmimodd_reheader%2F0.1.9','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mimodd\">MiModD Reheader<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fwolma%2Fmimodd_main%2Fmimodd_map%2F0.1.9','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mimodd\">MiModD NacreousMap<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fwolma%2Fmimodd_main%2Fmimodd_varreport%2F0.1.9','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mimodd\">MiModD Report Variants<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fwolma%2Fmimodd_main%2Fmimodd_delcall%2F0.1.9','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mimodd\">MiModD Deletion Calling (for PE data)<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fwolma%2Fmimodd_main%2Fmimodd_varextract%2F0.1.9','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mimodd\">MiModD Extract Variant Sites<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fwolma%2Fmimodd_main%2Fmimodd_rebase%2F0.1.9','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mimodd\">MiModD Rebase Sites<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fwolma%2Fmimodd_main%2Fmimodd_convert%2F0.1.9','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mimodd\">MiModD Convert<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fwolma%2Fmimodd_main%2Fmimodd_sort%2F0.1.9','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mimodd\">MiModD Sort<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fwolma%2Fmimodd_main%2Fmimodd_vcf_filter%2F0.1.9','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mimodd\">MiModD VCF Filter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fwolma%2Fmimodd_aln%2Fmimodd_align%2F0.1.9','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mimodd\">MiModD Read Alignment<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fminia%2Fminia%2F3.2.6','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"minia\">Minia<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fminiasm%2Fminiasm%2F0.3_r179%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"miniasm\">miniasm<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fminimap2%2Fminimap2%2F2.26%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"minimap2\">Map with minimap2<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fartbio%2Fmircounts%2Fmircounts%2F1.4.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mircounts\">miRcounts<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frnateam%2Fmirdeep2_quantifier%2Frbc_mirdeep2_quantifier%2F2.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mirdeep2\">MiRDeep2 Quantifier<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frnateam%2Fmirdeep2_mapper%2Frbc_mirdeep2_mapper%2F2.0.0.8.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mirdeep2\">MiRDeep2 Mapper<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frnateam%2Fmirdeep2%2Frbc_mirdeep2%2F2.0.1.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mirdeep2\">MiRDeep2<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmitobim%2Fmitobim%2F1.9.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mitobim\">MITObim<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmitos2%2Fmitos2%2F2.1.3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mitos2\">MITOS2<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmlst%2Fmlst%2F2.22.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mlst\">MLST<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmlst%2Fmlst_list%2F2.22.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mlst\">MLST List<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnml%2Fmob_suite%2Fmob_recon%2F3.0.3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mob-suite\">MOB-Recon<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnml%2Fmob_suite%2Fmob_typer%2F3.0.3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mob-suite\">MOB-Typer<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner/data_source_redirect?tool_id=modENCODEfly','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"modENCODEfly\">modENCODE fly<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner/data_source_redirect?tool_id=modENCODEworm','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"modENCODEworm\">modENCODE worm<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner/data_source_redirect?tool_id=modmine','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"modmine\">modENCODE modMine<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fproteomics_moff%2Fproteomics_moff%2F2.0.3.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"moff\">moFF<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fmorpheus%2Fmorpheus%2F2.255.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"morpheus\">Morpheus<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">131 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_venn%2Fmothur_venn%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Venn<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_unique_seqs%2Fmothur_unique_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Unique.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_unifrac_weighted%2Fmothur_unifrac_weighted%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">unifrac.weighted<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_unifrac_unweighted%2Fmothur_unifrac_unweighted%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">unifrac.unweighted<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_trim_seqs%2Fmothur_trim_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Trim.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_trim_flows%2Fmothur_trim_flows%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Trim.flows<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_tree_shared%2Fmothur_tree_shared%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Tree.shared<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_taxonomy_to_krona%2Fmothur_taxonomy_to_krona%2F1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Taxonomy-to-Krona<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_summary_tax%2Fmothur_summary_tax%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Summary.tax<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_summary_single%2Fmothur_summary_single%2F1.39.5.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Summary.single<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_summary_shared%2Fmothur_summary_shared%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Summary.shared<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_summary_seqs%2Fmothur_summary_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Summary.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_summary_qual%2Fmothur_summary_qual%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Summary.qual<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_sub_sample%2Fmothur_sub_sample%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Sub.sample<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_split_groups%2Fmothur_split_groups%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Split.groups<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_split_abund%2Fmothur_split_abund%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Split.abund<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_sort_seqs%2Fmothur_sort_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Sort.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_shhh_seqs%2Fmothur_shhh_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Shhh.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_shhh_flows%2Fmothur_shhh_flows%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Shhh.flows<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_sffinfo%2Fmothur_sffinfo%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Sffinfo<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_seq_error%2Fmothur_seq_error%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Seq.error<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_sens_spec%2Fmothur_sens_spec%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Sens.spec<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_screen_seqs%2Fmothur_screen_seqs%2F1.39.5.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Screen.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_reverse_seqs%2Fmothur_reverse_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Reverse.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_rename_seqs%2Fmothur_rename_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Rename.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_remove_seqs%2Fmothur_remove_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Remove.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_remove_rare%2Fmothur_remove_rare%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Remove.rare<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_remove_otus%2Fmothur_remove_otus%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Remove.otus<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_remove_otulabels%2Fmothur_remove_otulabels%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Remove.otulabels<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_remove_lineage%2Fmothur_remove_lineage%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Remove.lineage<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_remove_groups%2Fmothur_remove_groups%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Remove.groups<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_remove_dists%2Fmothur_remove_dists%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Remove.dists<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_rarefaction_single%2Fmothur_rarefaction_single%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Rarefaction.single<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_rarefaction_shared%2Fmothur_rarefaction_shared%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Rarefaction.shared<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_primer_design%2Fmothur_primer_design%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Primer.design<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_pre_cluster%2Fmothur_pre_cluster%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Pre.cluster<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_phylotype%2Fmothur_phylotype%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Phylotype<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_phylo_diversity%2Fmothur_phylo_diversity%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Phylo.diversity<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_pcr_seqs%2Fmothur_pcr_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Pcr.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_pcoa%2Fmothur_pcoa%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Pcoa<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_pca%2Fmothur_pca%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Pca<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_parsimony%2Fmothur_parsimony%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Parsimony<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_parse_list%2Fmothur_parse_list%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Parse.list<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_pairwise_seqs%2Fmothur_pairwise_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Pairwise.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_otu_hierarchy%2Fmothur_otu_hierarchy%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Otu.hierarchy<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_otu_association%2Fmothur_otu_association%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Otu.association<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_normalize_shared%2Fmothur_normalize_shared%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Normalize.shared<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_nmds%2Fmothur_nmds%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Nmds<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_mimarks_attributes%2Fmothur_mimarks_attributes%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Mimarks.attributes<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_metastats%2Fmothur_metastats%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Metastats<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_merge_taxsummary%2Fmothur_merge_taxsummary%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Merge.taxsummary<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_merge_sfffiles%2Fmothur_merge_sfffiles%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Merge.sfffiles<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_merge_groups%2Fmothur_merge_groups%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Merge.groups<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_merge_files%2Fmothur_merge_files%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Merge.files<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_merge_count%2Fmothur_merge_count%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Merge.count<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_mantel%2Fmothur_mantel%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Mantel<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_make_sra%2Fmothur_make_sra%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Make.sra<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_make_shared%2Fmothur_make_shared%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Make.shared<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_make_lookup%2Fmothur_make_lookup%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Make.lookup<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_make_lefse%2Fmothur_make_lefse%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Make.lefse<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_make_group%2Fmothur_make_group%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Make.group<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_make_fastq%2Fmothur_make_fastq%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Make.fastq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_make_design%2Fmothur_make_design%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Make Design<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_make_contigs%2Fmothur_make_contigs%2F1.39.5.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Make.contigs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_make_biom%2Fmothur_make_biom%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Make.biom<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_list_seqs%2Fmothur_list_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">List.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_list_otulabels%2Fmothur_list_otulabels%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">List.otulabels<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_libshuff%2Fmothur_libshuff%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Libshuff<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_lefse%2Fmothur_lefse%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Lefse<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_indicator%2Fmothur_indicator%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Indicator<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_homova%2Fmothur_homova%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Homova<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_heatmap_sim%2Fmothur_heatmap_sim%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Heatmap.sim<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_heatmap_bin%2Fmothur_heatmap_bin%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Heatmap.bin<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_hcluster%2Fmothur_hcluster%2F1.36.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Hcluster<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_get_sharedseqs%2Fmothur_get_sharedseqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Get.sharedseqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_get_seqs%2Fmothur_get_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Get.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_get_sabund%2Fmothur_get_sabund%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Get.sabund<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_get_relabund%2Fmothur_get_relabund%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Get.relabund<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_get_rabund%2Fmothur_get_rabund%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Get.rabund<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_get_otus%2Fmothur_get_otus%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Get.otus<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_get_oturep%2Fmothur_get_oturep%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Get.oturep<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_get_otulist%2Fmothur_get_otulist%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Get.otulist<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_get_otulabels%2Fmothur_get_otulabels%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Get.otulabels<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_get_mimarkspackage%2Fmothur_get_mimarkspackage%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Get.mimarkspackage<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_get_lineage%2Fmothur_get_lineage%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Get.lineage<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_get_label%2Fmothur_get_label%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Get.label<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_get_groups%2Fmothur_get_groups%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Get.groups<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_get_group%2Fmothur_get_group%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Get.group<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_get_dists%2Fmothur_get_dists%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Get.dists<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_get_coremicrobiome%2Fmothur_get_coremicrobiome%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Get.coremicrobiome<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_get_communitytype%2Fmothur_get_communitytype%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Get.communitytype<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_filter_shared%2Fmothur_filter_shared%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Filter.shared<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_filter_seqs%2Fmothur_filter_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Filter.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_fastq_info%2Fmothur_fastq_info%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Fastq.info<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_dist_shared%2Fmothur_dist_shared%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Dist.shared<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_dist_seqs%2Fmothur_dist_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Dist.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_deunique_tree%2Fmothur_deunique_tree%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Deunique.tree<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_deunique_seqs%2Fmothur_deunique_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Deunique.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_degap_seqs%2Fmothur_degap_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Degap.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_create_database%2Fmothur_create_database%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Create.database<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_count_seqs%2Fmothur_count_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Count.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_count_groups%2Fmothur_count_groups%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Count.groups<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_corr_axes%2Fmothur_corr_axes%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Corr.axes<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_cooccurrence%2Fmothur_cooccurrence%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Cooccurrence<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_consensus_seqs%2Fmothur_consensus_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Consensus.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_collect_single%2Fmothur_collect_single%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Collect.single<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_collect_shared%2Fmothur_collect_shared%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Collect.shared<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_cluster_split%2Fmothur_cluster_split%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Cluster.split<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_cluster_fragments%2Fmothur_cluster_fragments%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Cluster.fragments<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_cluster_classic%2Fmothur_cluster_classic%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Cluster.classic<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_cluster%2Fmothur_cluster%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Cluster<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_clearcut%2Fmothur_clearcut%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Clearcut<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_classify_tree%2Fmothur_classify_tree%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Classify.tree<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_classify_seqs%2Fmothur_classify_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Classify.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_classify_rf%2Fmothur_classify_rf%2F1.36.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Classify.rf<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_classify_otu%2Fmothur_classify_otu%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Classify.otu<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_chop_seqs%2Fmothur_chop_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Chop.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_chimera_vsearch%2Fmothur_chimera_vsearch%2F1.39.5.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Chimera.vsearch<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_chimera_uchime%2Fmothur_chimera_uchime%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Chimera.uchime<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_chimera_slayer%2Fmothur_chimera_slayer%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Chimera.slayer<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_chimera_pintail%2Fmothur_chimera_pintail%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Chimera.pintail<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_chimera_perseus%2Fmothur_chimera_perseus%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Chimera.perseus<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_chimera_check%2Fmothur_chimera_check%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Chimera.check<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_chimera_ccode%2Fmothur_chimera_ccode%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Chimera.ccode<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_chimera_bellerophon%2Fmothur_chimera_bellerophon%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Chimera.bellerophon<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_biom_info%2Fmothur_biom_info%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Biom.info<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_bin_seqs%2Fmothur_bin_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Bin.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_anosim%2Fmothur_anosim%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Anosim<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_amova%2Fmothur_amova%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Amova<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_align_seqs%2Fmothur_align_seqs%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Align.seqs<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmothur_align_check%2Fmothur_align_check%2F1.39.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mothur\">Align.check<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner/data_source_redirect?tool_id=mousemine','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mousemine\">MouseMine<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fmsconvert%2Fmsconvert%2F3.0.20287.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"msconvert\">msconvert<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Fmsmetaenhancer%2Fmsmetaenhancer%2F0.3.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"msmetaenhancer\">MSMetaEnhancer<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fmsstatstmt%2Fmsstatstmt%2F2.0.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"msstats\">MSstatsTMT<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fmsstats%2Fmsstats%2F4.0.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"msstats\">MSstats<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmultiqc%2Fmultiqc%2F1.11%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"multiqc\">MultiQC<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">6 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmummer_show_coords%2Fmummer_show_coords%2F4.0.0rc1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mummer\">Show-Coords<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmummer_nucmer%2Fmummer_nucmer%2F4.0.0rc1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mummer\">Nucmer<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmummer_mummerplot%2Fmummer_mummerplot%2F4.0.0rc1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mummer\">Mummerplot<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmummer_mummer%2Fmummer_mummer%2F4.0.0rc1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mummer\">Mummer<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmummer_dnadiff%2Fmummer_dnadiff%2F4.0.0rc1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mummer\">DNAdiff<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmummer_delta_filter%2Fmummer_delta_filter%2F4.0.0rc1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mummer\">Delta-Filter<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fmusic_deconvolution%2Fmusic_deconvolution%2F0.1.1%2Bgalaxy4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"music_deconvolution\">MuSiC Deconvolution<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fmusic_construct_eset%2Fmusic_construct_eset%2F0.1.1%2Bgalaxy4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"music_deconvolution\">Construct Expression Set Object<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fmusic_inspect_eset%2Fmusic_inspect_eset%2F0.1.1%2Bgalaxy4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"music_deconvolution\">Inspect Expression Set Object<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fmz_to_sqlite%2Fmz_to_sqlite%2F2.1.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"mztosqlite\">mz to sqlite<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fnanocompore_sampcomp%2Fnanocompore_sampcomp%2F1.0.0rc3.post2%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"nanocompore\">SampComp<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fleomrtns%2Fnanofilt%2Fnanofilt%2F0.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"nanofilt\">NanoFilt<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fnanoplot%2Fnanoplot%2F1.41.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"nanoplot\">NanoPlot<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">4 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fnanopolish_variants%2Fnanopolish_variants%2F0.13.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"nanopolish\">Nanopolish variants<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fnanopolish_polya%2Fnanopolish_polya%2F0.13.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"nanopolish\">Nanopolish polyA<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fnanopolish_methylation%2Fnanopolish_methylation%2F0.13.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"nanopolish\">Nanopolish methylation<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fnanopolish_eventalign%2Fnanopolish_eventalign%2F0.13.2%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"nanopolish\">Nanopolish eventalign<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fnatural_product_likeness%2Fctb_np-likeness-calculator%2F2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"natural_product_likeness_scorer\">Natural Product likeness calculator<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fncbi_acc_download%2Fncbi_acc_download%2F0.2.8%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ncbi_acc_download\">NCBI Accession Download<\/button><\/ul>",null,null,null,null,null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fnewick_utils%2Fnewick_display%2F1.6%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"newick_utils\">Newick Display<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fnextclade%2Fnextclade%2F2.7.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"nextclade\">Nextclade<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fngsutils_bam_filter%2Fngsutils_bam_filter%2F0.5.9','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ngsutils\">BAM filter<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fnovoplasty%2Fnovoplasty%2F4.3.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"NOVOPlasty\">NOVOplasty<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fblankenberg%2Fnaive_variant_caller%2Fnaive_variant_caller%2F0.0.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"nvc\">Naive Variant Caller (NVC)<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">10 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fobi_uniq%2Fobi_uniq%2F1.2.13','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"obitools\">obiuniq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fobi_tab%2Fobi_tab%2F1.2.13','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"obitools\">obitab<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fobi_stat%2Fobi_stat%2F1.2.13','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"obitools\">obistat<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fobi_sort%2Fobi_sort%2F1.2.13','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"obitools\">obisort<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fobi_ngsfilter%2Fobi_ngsfilter%2F1.2.13','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"obitools\">NGSfilter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fobi_illumina_pairend%2Fobi_illumina_pairend%2F1.2.13','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"obitools\">Illuminapairedend<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fobi_grep%2Fobi_grep%2F1.2.13','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"obitools\">obigrep<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fobi_convert%2Fobi_convert%2F1.2.13','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"obitools\">obiconvert<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fobi_clean%2Fobi_clean%2F1.2.13','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"obitools\">obiclean<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fobi_annotate%2Fobi_annotate%2F1.2.13','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"obitools\">obiannotate<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">36 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_msfraggeradapter%2FMSFraggerAdapter%2F2.8%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">MSFraggerAdapter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_xtandemadapter%2FXTandemAdapter%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">XTandemAdapter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_xmlvalidator%2FXMLValidator%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">XMLValidator<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_textexporter%2FTextExporter%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">TextExporter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_targetedfileconverter%2FTargetedFileConverter%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">TargetedFileConverter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_proteinquantifier%2FProteinQuantifier%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">ProteinQuantifier<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_peptideindexer%2FPeptideIndexer%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">PeptideIndexer<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_peakpickerhires%2FPeakPickerHiRes%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">PeakPickerHiRes<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_openswathworkflow%2FOpenSwathWorkflow%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">OpenSwathWorkflow<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_openswathrtnormalizer%2FOpenSwathRTNormalizer%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">OpenSwathRTNormalizer<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_openswathmzmlfilecacher%2FOpenSwathMzMLFileCacher%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">OpenSwathMzMLFileCacher<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_openswathfilesplitter%2FOpenSwathFileSplitter%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">OpenSwathFileSplitter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_openswathdiaprescoring%2FOpenSwathDIAPreScoring%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">OpenSwathDIAPreScoring<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_openswathdecoygenerator%2FOpenSwathDecoyGenerator%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">OpenSwathDecoyGenerator<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_openswathconfidencescoring%2FOpenSwathConfidenceScoring%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">OpenSwathConfidenceScoring<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_openswathassaygenerator%2FOpenSwathAssayGenerator%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">OpenSwathAssayGenerator<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_openswathanalyzer%2FOpenSwathAnalyzer%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">OpenSwathAnalyzer<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_mztabexporter%2FMzTabExporter%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">MzTabExporter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_multiplexresolver%2FMultiplexResolver%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">MultiplexResolver<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_msgfplusadapter%2FMSGFPlusAdapter%2F2.6%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">MSGFPlusAdapter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_idscoreswitcher%2FIDScoreSwitcher%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">IDScoreSwitcher<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_idposteriorerrorprobability%2FIDPosteriorErrorProbability%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">IDPosteriorErrorProbability<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_idmerger%2FIDMerger%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">IDMerger<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_idmapper%2FIDMapper%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">IDMapper<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_idfilter%2FIDFilter%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">IDFilter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_idconflictresolver%2FIDConflictResolver%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">IDConflictResolver<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_highresprecursormasscorrector%2FHighResPrecursorMassCorrector%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">HighResPrecursorMassCorrector<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_filemerger%2FFileMerger%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">FileMerger<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_fileinfo%2FFileInfo%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">FileInfo<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_filefilter%2FFileFilter%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">FileFilter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_fileconverter%2FFileConverter%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">FileConverter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_fidoadapter%2FFidoAdapter%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">FidoAdapter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_featurefindermultiplex%2FFeatureFinderMultiplex%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">FeatureFinderMultiplex<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_falsediscoveryrate%2FFalseDiscoveryRate%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">FalseDiscoveryRate<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_decoydatabase%2FDecoyDatabase%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">DecoyDatabase<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fopenms_consensusid%2FConsensusID%2F2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"openms\">ConsensusID<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">5 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fecology%2Fpampa_presabs%2Fpampa_presabs%2F0.0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pampa\">Calculate presence absence table<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fecology%2Fpampa_plotglm%2Fpampa_plotglm%2F0.0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pampa\">Create a plot from GLM data<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fecology%2Fpampa_glmsp%2Fpampa_glmsp%2F0.0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pampa\">Compute GLM on population data<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fecology%2Fpampa_glmcomm%2Fpampa_glmcomm%2F0.0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pampa\">Compute GLM on community data<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fecology%2Fpampa_communitymetrics%2Fpampa_communitymetrics%2F0.0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pampa\">Calculate community metrics<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpangolin%2Fpangolin%2F4.3%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pangolin\">Pangolin<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner?tool_id=param_value_from_file','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"param_value_from_file\">Parse parameter value<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fparse_mito_blast%2Fparse_mito_blast%2F1.0.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"parse_mito_blast\">Parse mitochondrial blast<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpathview%2Fpathview%2F1.34.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pathview\">Pathview<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpe_histogram%2Fpe_histogram%2F1.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pe_histogram\">Paired-end histogram<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpear%2Fiuc_pear%2F0.9.6.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pear\">Pear<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fpep_pointer%2Fpep_pointer%2F0.1.3%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"PepPointer\">PepPointer<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">4 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fpeptideshaker%2Ffasta_cli%2F4.0.41%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"peptideshaker\">FastaCLI<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fpeptideshaker%2Fsearch_gui%2F4.0.41%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"peptideshaker\">Search GUI<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fpeptideshaker%2Fpeptide_shaker%2F2.0.33%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"peptideshaker\">Peptide Shaker<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fpeptideshaker%2Fident_params%2F4.0.41%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"peptideshaker\">Identification Parameters<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fpeptide_genomic_coordinate%2Fpeptide_genomic_coordinate%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"peptide_genomic_coordinate\">Peptide Genomic Coordinate<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">4 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fpercolator%2Fpercolator_input_converters%2F3.5%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"percolator\">Search engine output to Pin converter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fpercolator%2Fpercolator%2F3.5%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"percolator\">Percolator<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fpercolator%2Fpout2mzid%2F0.3.03','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"percolator\">Pout2mzid<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fpercolator%2Fbatched_set_list_creator%2F3.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"percolator\">Create nested list<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fpfamscan%2Fpfamscan%2F1.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pfamscan\">PfamScan<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner?tool_id=interactive_tool_phinch','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"phinch\">Phinch Visualisation<\/button><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=testtoolshed.g2.bx.psu.edu%2Frepos%2Fsimon-gladman%2Fphyloseq_ordination_plot%2Fphyloseq_ordinate%2F1.22.3.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"phyloseq\">Phyloseq Ordination Plot<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=testtoolshed.g2.bx.psu.edu%2Frepos%2Fsimon-gladman%2Fphyloseq_filter%2Fbiom_filter%2F1.22.3.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"phyloseq\">Phyloseq Biom Filtering<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fphyml%2Fphyml%2F3.3.20190909','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"phyml\">PhyML<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">31 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_ReplaceSamHeader%2F2.18.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">ReplaceSamHeader<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_FilterSamReads%2F2.18.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">FilterSamReads<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_RevertOriginalBaseQualitiesAndAddMateCigar%2F2.18.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">RevertOriginalBaseQualitiesAndAddMateCigar<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_MergeBamAlignment%2F2.18.2.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">MergeBamAlignment<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_AddOrReplaceReadGroups%2F2.18.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">AddOrReplaceReadGroups<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_MergeSamFiles%2F2.18.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">MergeSamFiles<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_FixMateInformation%2F2.18.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">FixMateInformation<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_ReorderSam%2F2.18.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">ReorderSam<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_CollectGcBiasMetrics%2F2.18.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">CollectGcBiasMetrics<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_EstimateLibraryComplexity%2F2.18.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">EstimateLibraryComplexity<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_MarkDuplicatesWithMateCigar%2F2.18.2.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">MarkDuplicatesWithMateCigar<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_CollectBaseDistributionByCycle%2F2.18.2.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">CollectBaseDistributionByCycle<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_SamToFastq%2F2.18.2.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">SamToFastq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_BedToIntervalList%2F2.18.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">BedToIntervalList<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_MarkDuplicates%2F2.18.2.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">MarkDuplicates<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_CleanSam%2F2.18.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">CleanSam<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_FastqToSam%2F2.18.2.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">FastqToSam<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_CollectInsertSizeMetrics%2F2.18.2.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">CollectInsertSizeMetrics<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_MeanQualityByCycle%2F2.18.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">MeanQualityByCycle<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_NormalizeFasta%2F2.18.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">NormalizeFasta<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_AddCommentsToBam%2F2.18.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">AddCommentsToBam<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_QualityScoreDistribution%2F2.18.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">QualityScoreDistribution<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_SortSam%2F2.18.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">SortSam<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_RevertSam%2F2.18.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">RevertSam<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_CASM%2F2.18.2.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">Collect Alignment Summary Metrics<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_CollectWgsMetrics%2F2.18.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">CollectWgsMetrics<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_CollectRnaSeqMetrics%2F2.18.2.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">CollectRnaSeqMetrics<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_ValidateSamFile%2F2.18.2.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">ValidateSamFile<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_artifact_metrics%2F2.18.2.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">Picard Collect Sequencing Artifact Metrics<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_DownsampleSam%2F2.18.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">Downsample SAM/BAM<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpicard%2Fpicard_CollectHsMetrics%2F2.18.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picard\">CollectHsMetrics<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">6 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpicrust_predict_metagenomes%2Fpicrust_predict_metagenomes%2F1.1.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picrust\">Predict Metagenome<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpicrust_normalize_by_copy_number%2Fpicrust_normalize_by_copy_number%2F1.1.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picrust\">Normalize<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpicrust_metagenome_contributions%2Fpicrust_metagenome_contributions%2F1.1.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picrust\">Metagenome Contributions<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpicrust_format_tree_and_trait_table%2Fpicrust_format_tree_and_trait_table%2F1.1.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picrust\">Format<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpicrust_compare_biom%2Fpicrust_compare_biom%2F1.1.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picrust\">Compare BIOM tables<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpicrust_categorize%2Fpicrust_categorize%2F1.1.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picrust\">Categorize<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">7 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpicrust2_shuffle_predictions%2Fpicrust2_shuffle_predictions%2F2.5.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picrust2\">PICRUSt2 Generation of shuffled predictions<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpicrust2_place_seqs%2Fpicrust2_place_seqs%2F2.5.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picrust2\">PICRUSt2 Sequence placement<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpicrust2_pipeline%2Fpicrust2_pipeline%2F2.5.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picrust2\">PICRUSt2 Full pipeline<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpicrust2_pathway_pipeline%2Fpicrust2_pathway_pipeline%2F2.5.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picrust2\">PICRUSt2 Pathway abundance inference<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpicrust2_metagenome_pipeline%2Fpicrust2_metagenome_pipeline%2F2.5.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picrust2\">PICRUSt2 Metagenome prediction<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpicrust2_hsp%2Fpicrust2_hsp%2F2.5.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picrust2\">PICRUSt2 Hidden state prediction (HSP)<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpicrust2_add_descriptions%2Fpicrust2_add_descriptions%2F2.5.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"picrust2\">PICRUSt2 Add descriptions<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpileup_interval%2Fpileup_interval%2F1.0.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pileup_interval\">Pileup-to-Interval<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpileup_parser%2Fpileup_parser%2F1.0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pileup_parser\">Filter pileup<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpilon%2Fpilon%2F1.20.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pilon\">pilon<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdavidecangelosi%2Fpipe_t%2Fpipe-t%2F1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pipe_t\">PIPE-T<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fplasflow%2FPlasFlow%2F1.1.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"PlasFlow\">PlasFlow<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fplot_from_lda%2Fplot_for_lda_output1%2F1.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"plot_from_lda\">Draw ROC plot<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fplotly_regression_performance_plots%2Fplotly_regression_performance_plots%2F0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"plotly\">Plot actual vs predicted curves and residual plots<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fplotly_parallel_coordinates_plot%2Fplotly_parallel_coordinates_plot%2F0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"plotly\">Parallel Coordinates Plot<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fplotly_ml_performance_plots%2Fplotly_ml_performance_plots%2F0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"plotly\">Plot confusion matrix, precision, recall and ROC and AUC curves<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fpoisson2test%2Fpoisson2test%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"poisson2test\">Poisson two-sample test<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fporechop%2Fporechop%2F0.2.4%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"porechop\">Porechop<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">13 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fporetools_yield_plot%2Fporetools_yield_plot%2F0.6.1a1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"poretools\">Collector’s curve<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fporetools_winner%2Fporetools_winner%2F0.6.1a1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"poretools\">Get longest read<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fporetools_times%2Fporetools_times%2F0.6.1a1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"poretools\">Extract time<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fporetools_tabular%2Fporetools_tabular%2F0.6.1a1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"poretools\">Extract FASTQ<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fporetools_stats%2Fporetools_stats%2F0.6.1a1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"poretools\">Read length statistics<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fporetools_squiggle%2Fporetools_squiggle%2F0.6.1a1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"poretools\">Plot signals<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fporetools_qualpos%2Fporetools_qualpos%2F0.6.1a1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"poretools\">Generate box-whisker<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fporetools_qualdist%2Fporetools_qualdist%2F0.6.1a1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"poretools\">Show quality<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fporetools_occupancy%2Fporetools_occupancy%2F0.6.1a1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"poretools\">Plot performance<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fporetools_nucdist%2Fporetools_nucdist%2F0.6.1a1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"poretools\">Show nucleotide<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fporetools_hist%2Fporetools_hist%2F0.6.1a1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"poretools\">Generate histogram<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fporetools_extract%2Fporetools_extract%2F0.6.1a1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"poretools\">Extract reads<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fporetools_events%2Fporetools_events%2F0.6.1a1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"poretools\">Extract nanopore events<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpretext_map%2Fpretext_map%2F0.1.9%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pretext_map\">PretextMap<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpretext_snapshot%2Fpretext_snapshot%2F0.0.3%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pretextview\">Pretext Snapshot<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fcrs4%2Fprokka%2Fprokka%2F1.14.6%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"prokka\">Prokka<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fidot%2Fprop_venn%2Fprop_venn%2F0.5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"prop_venn\">proportional venn<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpurge_dups%2Fpurge_dups%2F1.2.6%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"purge_dups\">Purge overlaps<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpycoqc%2Fpycoqc%2F2.5.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pycoqc\">Pycoqc<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpygenometracks%2FpygenomeTracks%2F3.8%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pygenometracks\">pyGenomeTracks<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">4 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fpyprophet_score%2Fpyprophet_score%2F2.1.4.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pyprophet\">PyProphet score<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fpyprophet_protein%2Fpyprophet_protein%2F2.1.4.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pyprophet\">PyProphet protein<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fpyprophet_peptide%2Fpyprophet_peptide%2F2.1.4.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pyprophet\">PyProphet peptide<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fpyprophet_merge%2Fpyprophet_merge%2F2.1.4.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"pyprophet\">PyProphet merge<\/button><\/li><\/ul><\/ul>",null,null,null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=qual_stats_boxplot','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"qual_stats_boxplot\">Boxplot<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fqualimap_multi_bamqc%2Fqualimap_multi_bamqc%2F2.2.2d','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"qualimap\">QualiMap Multi-Sample BamQC<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fqualimap_bamqc%2Fqualimap_bamqc%2F2.2.2d%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"qualimap\">QualiMap BamQC<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fqualimap_rnaseq%2Fqualimap_rnaseq%2F2.2.2d%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"qualimap\">QualiMap RNA-Seq QC<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">12 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnml%2Fquasitools%2Fconsensus%2F0.7.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"quasitools\">Consensus Sequence<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnml%2Fquasitools%2Fcomplexity_bam%2F0.7.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"quasitools\">Complexity BAM<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnml%2Fquasitools%2Fdistance%2F0.7.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"quasitools\">Quasispecies Distance<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnml%2Fquasitools%2Fcomplexity_fasta%2F0.7.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"quasitools\">Complexity FASTA<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnml%2Fquasitools%2Fcallntvar%2F0.7.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"quasitools\">Nucleotide Variants<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnml%2Fquasitools%2Fquality%2F0.7.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"quasitools\">Quality control<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnml%2Fquasitools%2Faavariants%2F0.7.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"quasitools\">Amino Acid Variants<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnml%2Fquasitools%2Fhydra%2F0.7.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"quasitools\">Hydra pipeline<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnml%2Fquasitools%2Fcallcodonvar%2F0.7.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"quasitools\">Codon Variants<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnml%2Fquasitools%2Fdnds%2F0.7.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"quasitools\">dNdS Report<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnml%2Fquasitools%2Faacoverage%2F0.7.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"quasitools\">Amino Acid Coverage<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnml%2Fquasitools%2Fdrmutations%2F0.7.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"quasitools\">Drug Resistance Mutations<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fquast%2Fquast%2F5.2.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"quast\">Quast<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fquery_tabular%2Fquery_tabular%2F3.3.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"query_tabular\">Query Tabular<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Flength_and_gc_content%2Flength_and_gc_content%2F0.1.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"r\">Gene length and GC content<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">4 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fraceid_trajectory%2Fraceid_trajectory%2F0.2.3%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"r-raceid\">Lineage computation using StemID<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fraceid_inspecttrajectory%2Fraceid_inspecttrajectory%2F0.2.3%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"r-raceid\">Lineage Branch Analysis using StemID<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fraceid_inspectclusters%2Fraceid_inspectclusters%2F0.2.3%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"r-raceid\">Cluster Inspection using RaceID<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fraceid_clustering%2Fraceid_clustering%2F0.2.3%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"r-raceid\">Clustering using RaceID<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fracon%2Fracon%2F1.5.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"racon\">Racon<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Framclustr%2Framclustr%2F1.3.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ramclustr\">RAMClustR<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Framclustr_define_experiment%2Framclustr_define_experiment%2F1.0.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ramclustr\">RAMClustR define experiment<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner/data_source_redirect?tool_id=ratmine','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ratmine\">Ratmine<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fraven%2Fraven%2F1.8.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"raven\">Raven<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Frawtools%2Frawtools%2F1.4.2.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rawtools\">Raw Tools<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fraxml%2Fraxml%2F8.2.12%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"raxml\">Phyogenetic reconstruction with RAxML<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fsucos_max_score%2Fsucos_max_score%2F2020.03.4%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rdkit\">Max SuCOS score<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fsdf_to_tab%2Fsdf_to_tab%2F2020.03.4%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rdkit\">Extract values from an SD-file<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fenumerate_charges%2Fenumerate_charges%2F2020.03.4%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rdkit\">Enumerate changes<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fctb_frankenstein_ligand%2Fctb_frankenstein_ligand%2F2013.1-0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rdock\">Create Frankenstein ligand<\/button><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">8 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Frecetox_aplcms_remove_noise%2Frecetox_aplcms_remove_noise%2F0.12.0%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"recetox-aplcms\">recetox-aplcms - remove noise<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Frecetox_aplcms_recover_weaker_signals%2Frecetox_aplcms_recover_weaker_signals%2F0.12.0%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"recetox-aplcms\">recetox-aplcms - recover weaker signals<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Frecetox_aplcms_merge_known_table%2Frecetox_aplcms_merge_known_table%2F0.12.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"recetox-aplcms\">recetox-aplcms - merge known table<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Frecetox_aplcms_generate_feature_table%2Frecetox_aplcms_generate_feature_table%2F0.12.0%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"recetox-aplcms\">recetox-aplcms - generate feature table<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Frecetox_aplcms_correct_time%2Frecetox_aplcms_correct_time%2F0.12.0%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"recetox-aplcms\">recetox-aplcms - correct time<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Frecetox_aplcms_compute_template%2Frecetox_aplcms_compute_template%2F0.12.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"recetox-aplcms\">recetox-aplcms - compute template<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Frecetox_aplcms_compute_clusters%2Frecetox_aplcms_compute_clusters%2F0.12.0%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"recetox-aplcms\">recetox-aplcms - compute clusters<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Frecetox_aplcms_align_features%2Frecetox_aplcms_align_features%2F0.12.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"recetox-aplcms\">recetox-aplcms - align features<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fred%2Fred%2F2018.09.10%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"red\">Red<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Frepeat_masker%2Frepeatmasker_wrapper%2F4.1.5%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"repeatmasker\">RepeatMasker<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fcsbl%2Frepeatmodeler%2Frepeatmodeler%2F2.0.4%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"repeatmodeler\">RepeatModeler<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fartbio%2Frepenrich%2Frepenrich%2F1.6.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"repenrich\">RepEnrich<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Freplace_column_by_key_value_file%2Freplace_column_with_key_value_file%2F0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"replace_column_with_key_value_file\">Replace column<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frecetox%2Frmassbank%2Frmassbank%2F3.0.0%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rmassbank\">RMassBank<\/button><\/ul>",null,null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fpjbriggs%2Frnachipintegrator%2Frnachipintegrator_canonical_genes%2F1.1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rnachipintegrator\">Analyse canonical genes against 'peak' data<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fpjbriggs%2Frnachipintegrator%2Frnachipintegrator_wrapper%2F1.1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rnachipintegrator\">RnaChipIntegrator<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Froary%2Froary%2F3.13.0%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"roary\">Roary<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=testtoolshed.g2.bx.psu.edu%2Frepos%2Fsimon-gladman%2Frow_means%2Frow_means%2F0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"row_means\">Row Means<\/button><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">22 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_geneBody_coverage2%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">Gene Body Coverage (Bigwig)<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_read_GC%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">Read GC<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_read_quality%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">Read Quality<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_tin%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">Transcript Integrity Number<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_deletion_profile%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">Deletion Profile<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_read_distribution%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">Read Distribution<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_FPKM_count%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">FPKM Count<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_junction_annotation%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">Junction Annotation<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_RNA_fragment_size%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">RNA fragment size<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_insertion_profile%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">Insertion Profile<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_RPKM_saturation%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">RPKM Saturation<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_read_duplication%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">Read Duplication<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_geneBody_coverage%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">Gene Body Coverage (BAM)<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_bam_stat%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">BAM/SAM Mapping Stats<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_bam2wig%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">BAM to Wiggle<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_junction_saturation%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">Junction Saturation<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_infer_experiment%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">Infer Experiment<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_mismatch_profile%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">Mismatch Profile<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_read_hexamer%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">Hexamer frequency<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_clipping_profile%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">Clipping Profile<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_read_NVC%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">Read NVC<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnilesh%2Frseqc%2Frseqc_inner_distance%2F5.0.1%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rseqc\">Inner Distance<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner?tool_id=interactive_tool_rstudio','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rstudio\">RStudio<\/button><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Frxdock_rbdock%2Frxdock_rbdock%2F2013.1.1_148c5bd1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rxdock\">rxDock docking<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Frxdock_rbcavity%2Frxdock_rbcavity%2F2013.1.1_148c5bd1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"rxdock\">rxDock cavity definition<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fsailfish%2Fsailfish%2F0.10.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"sailfish\">Sailfish<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fsalmon%2Fsalmon%2F1.10.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"salmon\">Salmon quant<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fsalmon_kallisto_mtx_to_10x%2F_salmon_kallisto_mtx_to_10x%2F0.0.1%2Bgalaxy6','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"salmon_kallisto_mtx_to_10x\">salmonKallistoMtxTo10x<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsalsa%2Fsalsa%2F2.3%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"salsa\">SALSA<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsam_bitwise_flag_filter%2Fsam_bw_filter%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"sam_bitwise_flag_filter\">Filter SAM<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsam2interval%2Fsam2interval%2F1.0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"sam2interval\">Convert SAM<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fartbio%2Fsambamba%2Fsambamba_sample_or_filter%2F0.7.1%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"sambamba\">Sample, Slice or Filter BAM<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fpeterjc%2Fsample_seqs%2Fsample_seqs%2F0.2.5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"sample_seqs\">Sub-sample sequences files<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">19 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsamtools_view%2Fsamtools_view%2F1.15.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"samtools\">Samtools view<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_split%2Fsamtools_split%2F1.15.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"samtools\">Samtools split<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsamtools_merge%2Fsamtools_merge%2F1.15.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"samtools\">Samtools merge<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsamtools_fastx%2Fsamtools_fastx%2F1.15.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"samtools\">Samtools fastx<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_stats%2Fsamtools_stats%2F2.0.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"samtools\">Samtools stats<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_sort%2Fsamtools_sort%2F2.0.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"samtools\">Samtools sort<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_slice_bam%2Fsamtools_slice_bam%2F2.0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"samtools\">Slice<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_rmdup%2Fsamtools_rmdup%2F2.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"samtools\">RmDup<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_reheader%2Fsamtools_reheader%2F2.0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"samtools\">Samtools reheader<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_phase%2Fsamtools_phase%2F2.0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"samtools\">Samtools phase<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_mpileup%2Fsamtools_mpileup%2F2.1.6','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"samtools\">Samtools mpileup<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_idxstats%2Fsamtools_idxstats%2F2.0.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"samtools\">Samtools idxstats<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_flagstat%2Fsamtools_flagstat%2F2.0.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"samtools\">Samtools flagstat<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_calmd%2Fsamtools_calmd%2F2.0.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"samtools\">Samtools calmd<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtools_bedcov%2Fsamtools_bedcov%2F2.0.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"samtools\">Samtools bedcov<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsamtool_filter2%2Fsamtool_filter2%2F1.8%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"samtools\">Filter SAM or BAM, output SAM or BAM<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsam_to_bam%2Fsam_to_bam%2F2.1.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"samtools\">SAM-to-BAM<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsam_pileup%2Fsam_pileup%2F1.1.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"samtools\">Generate pileup<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fbam_to_sam%2Fbam_to_sam%2F2.0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"samtools\">BAM-to-SAM<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">33 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fscanpy_remove_confounders%2Fscanpy_remove_confounders%2F1.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Remove confounders<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fscanpy_plot%2Fscanpy_plot%2F1.7.1%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Plot<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fscanpy_normalize%2Fscanpy_normalize%2F1.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Normalize<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fscanpy_inspect%2Fscanpy_inspect%2F1.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Inspect and manipulate<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fscanpy_filter%2Fscanpy_filter%2F1.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Filter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fscanpy_cluster_reduce_dimension%2Fscanpy_cluster_reduce_dimension%2F1.7.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Cluster, infer trajectories and embed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_run_umap%2Fscanpy_run_umap%2F1.8.1%2B3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy RunUMAP<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_integrate_mnn%2Fscanpy_integrate_mnn%2F1.8.1%2B3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy MNN<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_scale_data%2Fscanpy_scale_data%2F1.8.1%2B1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy ScaleData<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_run_tsne%2Fscanpy_run_tsne%2F1.8.1%2B1%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy RunTSNE<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_run_pca%2Fscanpy_run_pca%2F1.8.1%2B1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy RunPCA<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_run_paga%2Fscanpy_run_paga%2F1.8.1%2B1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy PAGA<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_run_fdg%2Fscanpy_run_fdg%2F1.8.1%2B1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy RunFDG<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_run_dpt%2Fscanpy_run_dpt%2F1.8.1%2B1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy DPT<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_run_diffmap%2Fscanpy_run_diffmap%2F1.8.1%2B1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy DiffusionMap<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_regress_variable%2Fscanpy_regress_variable%2F1.8.1%2B1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy RegressOut<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_read_10x%2Fscanpy_read_10x%2F1.8.1%2B1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy Read10x<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_plot_trajectory%2Fscanpy_plot_trajectory%2F1.8.1%2B1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy PlotTrajectory<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_plot_scrublet%2Fscanpy_plot_scrublet%2F1.8.1%2B1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy Plot Scrublet<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_plot_embed%2Fscanpy_plot_embed%2F1.8.1%2B1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy PlotEmbed<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_parameter_iterator%2Fscanpy_parameter_iterator%2F0.0.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy ParameterIterator<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_normalise_data%2Fscanpy_normalise_data%2F1.8.1%2B1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy NormaliseData<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_multiplet_scrublet%2Fscanpy_multiplet_scrublet%2F1.8.1%2B1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy Scrublet<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_integrate_harmony%2Fscanpy_integrate_harmony%2F1.8.1%2B1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy Harmony<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_integrate_combat%2Fscanpy_integrate_combat%2F1.8.1%2B1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy ComBat<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_integrate_bbknn%2Fscanpy_integrate_bbknn%2F1.8.1%2B1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy BBKNN<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_find_variable_genes%2Fscanpy_find_variable_genes%2F1.8.1%2B1%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy FindVariableGenes<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_find_markers%2Fscanpy_find_markers%2F1.8.1%2B1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy FindMarkers<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_find_cluster%2Fscanpy_find_cluster%2F1.8.1%2B1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy FindCluster<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_filter_genes%2Fscanpy_filter_genes%2F1.8.1%2B1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy FilterGenes<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_filter_cells%2Fscanpy_filter_cells%2F1.8.1%2B1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy FilterCells<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fscanpy_compute_graph%2Fscanpy_compute_graph%2F1.8.1%2B1%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">Scanpy ComputeGraph<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2Fanndata_ops%2Fanndata_ops%2F1.8.1%2B1%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scanpy\">AnnData Operations<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">4 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fscater_plot_pca%2Fscater_plot_pca%2F1.22.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scater\">Scater: PCA plot<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fscater_plot_dist_scatter%2Fscater_plot_dist_scatter%2F1.22.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scater\">Scater: plot library QC<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fscater_filter%2Fscater_filter%2F1.22.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scater\">Scater: filter SCE<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fscater_create_qcmetric_ready_sce%2Fscater_create_qcmetric_ready_sce%2F1.22.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scater\">Scater: calculate QC metrics<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">7 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fimgteam%2Fcount_objects%2Fip_count_objects%2F0.0.5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scikit-image\">Count Objects<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fimgteam%2Fbinary2labelimage%2Fip_binary_to_labelimage%2F0.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scikit-image\">Binary 2 Label<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fimgteam%2F2d_simple_filter%2Fip_filter_standard%2F0.0.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scikit-image\">Filter Image<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fimgteam%2F2d_histogram_equalization%2Fip_histogram_equalization%2F0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scikit-image\">Histogram equalization<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fimgteam%2F2d_filter_segmentation_by_features%2Fip_2d_filter_segmentation_by_features%2F0.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scikit-image\">Filter segmentation<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fimgteam%2F2d_feature_extraction%2Fip_2d_feature_extraction%2F0.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scikit-image\">2D Feature Extraction<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fimgteam%2F2d_auto_threshold%2Fip_threshold%2F0.0.5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scikit-image\">Auto Threshold<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">14 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fsklearn_build_pipeline%2Fsklearn_build_pipeline%2F1.0.10.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scikit\">Pipeline Builder<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fsklearn_numeric_clustering%2Fsklearn_numeric_clustering%2F1.0.10.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scikit\">Numeric Clustering<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fsklearn_nn_classifier%2Fsklearn_nn_classifier%2F1.0.10.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scikit\">Nearest Neighbors Classification<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fsklearn_estimator_attributes%2Fsklearn_estimator_attributes%2F1.0.10.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scikit\">Estimator attributes<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fsklearn_to_categorical%2Fsklearn_to_categorical%2F1.0.10.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scikit\">To categorical<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fsklearn_svm_classifier%2Fsklearn_svm_classifier%2F1.0.10.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scikit\">Support vector machines (SVMs)<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fsklearn_searchcv%2Fsklearn_searchcv%2F1.0.10.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scikit\">Hyperparameter Search<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fsklearn_generalized_linear%2Fsklearn_generalized_linear%2F1.0.10.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scikit\">Generalized linear models<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fsklearn_ensemble%2Fsklearn_ensemble%2F1.0.10.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scikit\">Ensemble methods<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fmodel_prediction%2Fmodel_prediction%2F1.0.10.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scikit\">Model Prediction<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fml_visualization_ex%2Fml_visualization_ex%2F1.0.10.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scikit\">Machine Learning Visualization Extension<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fkeras_train_and_eval%2Fkeras_train_and_eval%2F1.0.10.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scikit\">Deep learning training and evaluation<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fkeras_model_config%2Fkeras_model_config%2F1.0.10.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scikit\">Create a deep learning model architecture<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fkeras_model_builder%2Fkeras_model_builder%2F1.0.10.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"scikit\">Create deep learning model<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=send_to_cloud','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"send_to_cloud\">Send to cloud<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fpeterjc%2Fseq_filter_by_id%2Fseq_filter_by_id%2F0.2.7','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"seq_filter_by_id\">Filter sequences by ID<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fpeterjc%2Fseq_select_by_id%2Fseq_select_by_id%2F0.0.14','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"seq_select_by_id\">Select sequences by ID<\/button><\/ul>",null,null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">14 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fseqtk%2Fseqtk_mergepe%2F1.3.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"seqtk\">seqtk_mergepe<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fseqtk%2Fseqtk_mergefa%2F1.3.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"seqtk\">seqtk_mergefa<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fseqtk%2Fseqtk_mutfa%2F1.3.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"seqtk\">seqtk_mutfa<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fseqtk%2Fseqtk_comp%2F1.3.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"seqtk\">seqtk_comp<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fseqtk%2Fseqtk_cutN%2F1.3.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"seqtk\">seqtk_cutN<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fseqtk%2Fseqtk_trimfq%2F1.3.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"seqtk\">seqtk_trimfq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fseqtk%2Fseqtk_fqchk%2F1.3.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"seqtk\">seqtk_fqchk<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fseqtk%2Fseqtk_seq%2F1.3.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"seqtk\">seqtk_seq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fseqtk%2Fseqtk_hety%2F1.3.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"seqtk\">seqtk_hety<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fseqtk%2Fseqtk_sample%2F1.3.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"seqtk\">seqtk_sample<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fseqtk%2Fseqtk_subseq%2F1.3.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"seqtk\">seqtk_subseq<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fseqtk%2Fseqtk_listhet%2F1.3.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"seqtk\">seqtk_listhet<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fseqtk%2Fseqtk_dropse%2F1.3.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"seqtk\">seqtk_dropse<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fseqtk%2Fseqtk_randbase%2F1.3.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"seqtk\">seqtk_randbase<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fseurat%2Fseurat%2F4.3.0.1%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"seurat\">Seurat<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=Sff_extractor','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Sff_extractor\">SFF converter<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fshasta%2Fshasta%2F0.6.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"shasta\">Shasta<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fshovill%2Fshovill%2F1.1.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"shovill\">Shovill<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsicer%2Fpeakcalling_sicer%2F1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"sicer\">SICER<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpubmed_by_queries%2Fpubmed_by_queries%2F0.0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"simtext\">PubMed query<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fpmids_to_pubtator_matrix%2Fpmids_to_pubtator_matrix%2F0.0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"simtext\">PMIDs to PubTator<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnml%2Fsistr_cmd%2Fsistr_cmd%2F1.1.1%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"sistr\">sistr_cmd<\/button><\/ul>",null,null,null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxy-australia%2Fsmudgeplot%2Fsmudgeplot%2F0.2.5%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Smudgeplots\">Smudgeplot<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsnap_training%2Fsnap_training%2F2013_11_29%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snap\">Train SNAP<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsniffles%2Fsniffles%2F1.0.12%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"sniffles\">sniffles<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsnippy%2Fsnippy_clean_full_aln%2F4.6.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snippy\">snippy-clean_full_aln<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsnippy%2Fsnippy_core%2F4.6.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snippy\">snippy-core<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsnippy%2Fsnippy%2F4.6.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snippy\">snippy<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsnp_sites%2Fsnp_sites%2F2.5.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snp_sites\">Finds SNP sites<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsnp_dists%2Fsnp_dists%2F0.8.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snp-dists\">SNP distance matrix<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">6 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsnpeff%2FsnpEff%2F4.3%2BT.galaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snpeff\">SnpEff eff:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsnpeff%2FsnpEff_build_gb%2F4.3%2BT.galaxy5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snpeff\">SnpEff build:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsnpeff%2FsnpEff_get_chr_names%2F4.3%2BT.galaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snpeff\">SnpEff chromosome-info:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsnpeff%2FsnpEff_databases%2F4.3%2BT.galaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snpeff\">SnpEff databases:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsnpeff%2FsnpEff_download%2F4.3%2BT.galaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snpeff\">SnpEff download:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsnpeff_sars_cov_2%2Fsnpeff_sars_cov_2%2F4.5covid19','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snpeff\">SnpEff eff:<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fsnpfreq%2Fhgv_snpFreq%2F1.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snpfreq\">snpFreq<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsnpfreqplot%2Fsnpfreqplot%2F1.0%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snpfreqplot\">Variant Frequency Plot<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">8 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsnpsift%2FsnpSift_caseControl%2F4.3%2Bt.galaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snpsift\">SnpSift CaseControl<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsnpsift%2FsnpSift_vcfCheck%2F4.3%2Bt.galaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snpsift\">SnpSift vcfCheck<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsnpsift%2Fsnpsift_vartype%2F4.3%2Bt.galaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snpsift\">SnpSift Variant Type<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsnpsift%2FsnpSift_extractFields%2F4.3%2Bt.galaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snpsift\">SnpSift Extract Fields<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsnpsift%2FsnpSift_annotate%2F4.3%2Bt.galaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snpsift\">SnpSift Annotate<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsnpsift%2FsnpSift_filter%2F4.3%2Bt.galaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snpsift\">SnpSift Filter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsnpsift%2FsnpSift_rmInfo%2F4.3%2Bt.galaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snpsift\">SnpSift rmInfo<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsnpsift%2FsnpSift_int%2F4.3%2Bt.galaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"snpsift\">SnpSift Intervals<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frnateam%2Fsortmerna%2Fbg_sortmerna%2F2.1b.6','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"sortmerna\">Filter with SortMeRNA<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">8 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fspades_rnaviralspades%2Fspades_rnaviralspades%2F3.15.4%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"spades\">rnaviralSPAdes<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fspades_plasmidspades%2Fspades_plasmidspades%2F3.15.4%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"spades\">plasmidSPAdes<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fspades_metaviralspades%2Fspades_metaviralspades%2F3.15.4%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"spades\">metaviralSPAdes<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fspades_metaplasmidspades%2Fspades_metaplasmidspades%2F3.15.4%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"spades\">metaplasmidSPAdes<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fspades_coronaspades%2Fspades_coronaspades%2F3.15.4%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"spades\">coronaSPAdes<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fspades_biosyntheticspades%2Fspades_biosyntheticspades%2F3.15.4%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"spades\">biosyntheticSPAdes<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Frnaspades%2Frnaspades%2F3.15.4%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"spades\">rnaSPAdes<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnml%2Fspades%2Fspades%2F3.15.4%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"spades\">SPAdes<\/button><\/li><\/ul><\/ul>",null,null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fsplit_file_to_collection%2Fsplit_file_to_collection%2F0.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"split_file_to_collection\">Split file<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsqlite_to_tabular%2Fsqlite_to_tabular%2F3.2.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"sqlite_to_tabular\">SQLite to tabular<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner/data_source_redirect?tool_id=sra_source','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"sra_source\">SRA<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsra_tools%2Fsam_dump%2F3.0.5%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"sra-tools\">Download and Extract Reads in BAM<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsra_tools%2Ffasterq_dump%2F3.0.5%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"sra-tools\">Faster Download and Extract Reads in FASTQ<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fsra_tools%2Ffastq_dump%2F3.0.5%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"sra-tools\">Download and Extract Reads in FASTQ<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">25 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks_populations%2Fstacks_populations%2F1.46.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks: populations<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks_sstacks%2Fstacks_sstacks%2F1.46.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks: sstacks<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks2_tsv2bam%2Fstacks2_tsv2bam%2F2.55%2Bgalaxy4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks2: tsv2bam<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks2_procrad%2Fstacks2_procrad%2F2.55%2Bgalaxy4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks2: process radtags<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks2_kmerfilter%2Fstacks2_kmerfilter%2F2.55%2Bgalaxy4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks2: kmer filter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks2_ustacks%2Fstacks2_ustacks%2F2.55%2Bgalaxy4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks2: ustacks<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks2_sstacks%2Fstacks2_sstacks%2F2.55%2Bgalaxy4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks2: sstacks<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks2_shortreads%2Fstacks2_shortreads%2F2.55%2Bgalaxy4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks2: process shortreads<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks2_refmap%2Fstacks2_refmap%2F2.55%2Bgalaxy4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks2: reference map<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks2_populations%2Fstacks2_populations%2F2.55%2Bgalaxy4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks2: populations<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks2_gstacks%2Fstacks2_gstacks%2F2.55%2Bgalaxy4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks2: gstacks<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks2_denovomap%2Fstacks2_denovomap%2F2.55%2Bgalaxy4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks2: de novo map<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks2_cstacks%2Fstacks2_cstacks%2F2.55%2Bgalaxy4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks2: cstacks<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks2_clonefilter%2Fstacks2_clonefilter%2F2.55%2Bgalaxy4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks2: clone filter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks_ustacks%2Fstacks_ustacks%2F1.46.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks: ustacks<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks_stats%2Fstacks_stats%2F1.46.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks: statistics<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks_rxstacks%2Fstacks_rxstacks%2F1.46.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks: rxstacks<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks_refmap%2Fstacks_refmap%2F1.46.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks: reference map<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks_pstacks%2Fstacks_pstacks%2F1.46.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks: pstacks<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks_procrad%2Fstacks_procrad%2F1.46.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks: process radtags<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks_genotypes%2Fstacks_genotypes%2F1.46.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks: genotypes<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks_denovomap%2Fstacks_denovomap%2F1.46.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks: de novo map<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks_cstacks%2Fstacks_cstacks%2F1.46.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks: cstacks<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks_clonefilter%2Fstacks_clonefilter%2F1.46.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks: clone filter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstacks_assembleperead%2Fstacks_assembleperead%2F1.46.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stacks\">Stacks: assemble read pairs by locus<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Frna_starsolo%2Frna_starsolo%2F2.7.10b%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"star\">RNA STARSolo<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Frgrnastar%2Frna_star%2F2.7.10b%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"star\">RNA STAR<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fnml%2Fstaramr%2Fstaramr_search%2F0.9.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"staramr\">staramr<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstar_fusion%2Fstar_fusion%2F0.5.4-3%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"star_fusion\">STAR-Fusion<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstringtie%2Fstringtie_merge%2F2.2.1%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stringtie\">StringTie merge<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fstringtie%2Fstringtie%2F2.2.1%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"stringtie\">StringTie<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=Summary_Statistics1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"Summary_Statistics1\">Summary Statistics<\/button><\/ul>",null,null,null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ft_test_two_samples%2Ft_test_two_samples%2F1.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"t_test_two_samples\">T Test for Two Samples<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftable_compute%2Ftable_compute%2F1.2.4%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"table_compute\">Table Compute<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ftabular_to_fasta%2Ftab2fasta%2F1.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"tabular_to_fasta\">Tabular-to-FASTA<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frnateam%2Ftargetfinder%2Ftargetfinder%2F1.7.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"targetfinder\">TargetFinder<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftb_variant_filter%2Ftb_variant_filter%2F0.4.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"tb_variant_filter\">TB Variant Filter<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftbprofiler%2Ftb_profiler_profile%2F4.4.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"tbprofiler\">TB-Profiler Profile<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftbvcfreport%2Ftbvcfreport%2F0.1.10%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"tbvcfreport\">TB Variant Report<\/button><\/ul>",null,null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">19 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftext_processing%2Ftp_sort_header_tool%2F1.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"text_processing\">Sort<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftext_processing%2Ftp_sorted_uniq%2F1.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"text_processing\">Unique<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftext_processing%2Ftp_easyjoin_tool%2F1.1.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"text_processing\">Join<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftext_processing%2Ftp_awk_tool%2F1.1.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"text_processing\">Text reformatting<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftext_processing%2Ftp_sort_rows%2F1.1.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"text_processing\">Sort a row<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftext_processing%2Ftp_head_tool%2F1.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"text_processing\">Select first<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftext_processing%2Ftp_unfold_column_tool%2F1.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"text_processing\">Unfold<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftext_processing%2Ftp_uniq_tool%2F1.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"text_processing\">Unique lines<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftext_processing%2Ftp_replace_in_column%2F1.1.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"text_processing\">Replace Text<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftext_processing%2Ftp_text_file_with_recurring_lines%2F1.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"text_processing\">Create text file<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftext_processing%2Ftp_tac%2F1.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"text_processing\">tac<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftext_processing%2Ftp_replace_in_line%2F1.1.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"text_processing\">Replace Text<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftext_processing%2Ftp_cut_tool%2F1.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"text_processing\">Cut<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftext_processing%2Ftp_find_and_replace%2F1.1.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"text_processing\">Replace<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftext_processing%2Ftp_cat%2F0.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"text_processing\">Concatenate datasets<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftext_processing%2Ftp_grep_tool%2F1.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"text_processing\">Search in textfiles<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftext_processing%2Ftp_tail_tool%2F1.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"text_processing\">Select last<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftext_processing%2Ftp_multijoin_tool%2F1.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"text_processing\">Multi-Join<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftext_processing%2Ftp_sed_tool%2F1.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"text_processing\">Text transformation<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Fthermo_raw_file_converter%2Fthermo_raw_file_converter%2F1.3.4%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ThermoRawFileParser\">Thermo<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ftophat_fusion_post%2Ftophat_fusion_post%2F0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"tophat\">Tophat Fusion Post<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ftophat2%2Ftophat2%2F2.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"tophat\">TopHat<\/button><\/li><\/ul><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">4 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frreumerman%2Fsnptools%2Ftrams%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trams\">TRAMS<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frreumerman%2Fsnptools%2Fsnpsplit%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trams\">SNP splitter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frreumerman%2Fsnptools%2Fgbk2fasta%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trams\">Genbank-to-Fasta<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Frreumerman%2Fsnptools%2Ftablemerger%2F1.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trams\">SNP table merger<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftransdecoder%2Ftransdecoder%2F5.5.0%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"transdecoder\">TransDecoder<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftransit_gumbel%2Ftransit_gumbel%2F3.0.2%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"transit\">TRANSIT Gumbel<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgff_to_prot%2Fgff_to_prot%2F3.0.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"transit\">Convert GFF3<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Ftranslate_bed%2Ftranslate_bed%2F0.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"translate_bed\">Translate BED transcripts<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">5 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Firacooke%2Ftpp_prophets%2Fproteomics_search_protein_prophet_1%2F1.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"tpp\">Protein Prophet<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Firacooke%2Ftpp_prophets%2Fpepxml_to_table_1%2F1.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"tpp\">PepXML to Table<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Firacooke%2Ftpp_prophets%2Fproteomics_search_interprophet_1%2F1.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"tpp\">InterProphet<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Firacooke%2Ftpp_prophets%2Fproteomics_search_peptide_prophet_1%2F1.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"tpp\">Peptide Prophet<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Firacooke%2Ftpp_prophets%2Fprotxml_to_table_1%2F1.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"tpp\">ProtXML to Table<\/button><\/li><\/ul><\/ul>",null,null,null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftrim_galore%2Ftrim_galore%2F0.6.7%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trim_galore\">Trim Galore!<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fpjbriggs%2Ftrimmomatic%2Ftrimmomatic%2F0.36.6','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trimmomatic\">Trimmomatic<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">11 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftrinity_samples_qccheck%2Ftrinity_samples_qccheck%2F2.15.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trinity\">RNASeq samples quality check<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftrinity_gene_to_trans_map%2Ftrinity_gene_to_trans_map%2F2.15.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trinity\">Generate gene to transcript map<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftrinity_filter_low_expr_transcripts%2Ftrinity_filter_low_expr_transcripts%2F2.15.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trinity\">Filter low expression transcripts<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftrinity_define_clusters_by_cutting_tree%2Ftrinity_define_clusters_by_cutting_tree%2F2.15.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trinity\">Partition genes into expression clusters<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftrinity_contig_exn50_statistic%2Ftrinity_contig_exn50_statistic%2F2.15.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trinity\">Compute contig Ex90N50 statistic and Ex90 transcript count<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftrinity_align_and_estimate_abundance%2Ftrinity_align_and_estimate_abundance%2F2.15.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trinity\">Align reads and estimate abundance<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftrinity_abundance_estimates_to_matrix%2Ftrinity_abundance_estimates_to_matrix%2F2.15.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trinity\">Build expression matrix<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fdescribe_samples%2Fdescribe_samples%2F2.15.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trinity\">Describe samples<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftrinity_analyze_diff_expr%2Ftrinity_analyze_diff_expr%2F2.15.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trinity\">Extract and cluster differentially expressed transcripts<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftrinity%2Ftrinity%2F2.15.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trinity\">Trinity<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftrinity_run_de_analysis%2Ftrinity_run_de_analysis%2F2.15.1%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trinity\">Differential expression analysis<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftrinotate%2Ftrinotate%2F3.2.2%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trinotate\">Trinotate<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftrna_prediction%2Ftrnascan%2F0.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trnascan-se\">tRNA prediction<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">5 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftrycycler_subsample%2Ftrycycler_subsample%2F0.5.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trycycler\">Trycycler subsample<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftrycycler_reconcile_msa%2Ftrycycler_reconcile_msa%2F0.5.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trycycler\">Trycycler reconcile/msa<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftrycycler_partition%2Ftrycycler_partition%2F0.5.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trycycler\">Trycycler partition<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftrycycler_consensus%2Ftrycycler_consensus%2F0.5.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trycycler\">Trycycler consensus<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Ftrycycler_cluster%2Ftrycycler_cluster%2F0.5.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"trycycler\">Trycycler cluster<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fucsc_fasplit%2Ffasplit%2F377','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ucsc\">faSplit<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fgtftobed12%2Fgtftobed12%2F357','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ucsc\">Convert GTF to BED12<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner/data_source_redirect?tool_id=ucsc_table_direct_archaea1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ucsc_table_direct_archaea1\">UCSC Archaea<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner/data_source_redirect?tool_id=ucsc_table_direct1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"ucsc_table_direct1\">UCSC Main<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fumi_tools_dedup%2Fumi_tools_dedup%2F1.1.2%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"umi_tools\">UMI-tools deduplicate<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fumi_tools_extract%2Fumi_tools_extract%2F1.1.2%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"umi_tools\">UMI-tools extract<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fumi_tools_count%2Fumi_tools_count%2F1.1.2%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"umi_tools\">UMI-tools count<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Funicycler%2Funicycler%2F0.5.0%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"unicycler\">Create assemblies with Unicycler<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Funipept%2Funipept%2F4.5.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"unipept\">Unipept<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fgalaxyp%2Funiprotxml_downloader%2Funiprotxml_downloader%2F2.4.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"UniProt_Downloader\">UniProt<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Funiprot_rest_interface%2Funiprot%2F0.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"uniprot_rest_interface\">UniProt<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fimgteam%2Funzip%2Funzip%2F6.0%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"unzip\">Unzip<\/button><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvarscan_version_2%2Fvarscan%2F2.4.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"varscan\">VarScan<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fvarscan_somatic%2Fvarscan_somatic%2F2.4.3.6','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"varscan\">VarScan somatic<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=vcf_to_maf_customtrack1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcf_to_maf_customtrack1\">VCF to MAF Custom Track<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fvcf2maf%2Fvcf2maf%2F1.6.21%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcf2maf\">Convert VCF to MAF<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">23 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfvcfintersect%2Fvcfvcfintersect%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCF-VCFintersect:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfsort%2Fvcfsort%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCFsort:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfselectsamples%2Fvcfselectsamples%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCFselectsamples:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfrandomsample%2Fvcfrandomsample%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCFrandomSample:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfprimers%2Fvcfprimers%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCFprimers:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfleftalign%2Fvcfleftalign%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCFleftAlign:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfhethom%2Fvcfhethom%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCFhetHomAlleles:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfgenotypes%2Fvcfgenotypes%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCFgenotypes:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfgeno2haplo%2Fvcfgeno2haplo%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCFgenotype-to-haplotype:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfflatten%2Fvcfflatten2%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCFflatten:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcffixup%2Fvcffixup%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCFfixup:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcffilter%2Fvcffilter2%2F1.0.0_rc3%2Bgalaxy3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCFfilter:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfdistance%2Fvcfdistance%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCFdistance:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfcommonsamples%2Fvcfcommonsamples%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCFcommonSamples:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfcombine%2Fvcfcombine%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCFcombine:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfcheck%2Fvcfcheck%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCFcheck:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfbreakcreatemulti%2Fvcfbreakcreatemulti%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCFbreakCreateMulti:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfbedintersect%2Fvcfbedintersect%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCF-BEDintersect:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfannotategenotypes%2Fvcfannotategenotypes%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCFannotateGenotypes:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfannotate%2Fvcfannotate%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCFannotate:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfallelicprimitives%2Fvcfallelicprimitives%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VcfAllelicPrimitives:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcfaddinfo%2Fvcfaddinfo%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCFaddinfo:<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcf2tsv%2Fvcf2tsv%2F1.0.0_rc3%2Bgalaxy0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcflib\">VCFtoTab-delimited:<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcftools_isec%2Fvcftools_isec%2F0.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcftools\">Intersect<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvcftools_merge%2Fvcftools_merge%2F0.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vcftools\">Merge<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">3 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fsimon-gladman%2Fvelvetoptimiser%2Fvelvetoptimiser%2F2.2.6%2Bgalaxy2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"velvet\">VelvetOptimiser<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvelvet%2Fvelvetg%2F1.2.10.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"velvet\">velvetg<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fvelvet%2Fvelveth%2F1.2.10.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"velvet\">velveth<\/button><\/li><\/ul><\/ul>",null,null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fvolcanoplot%2Fvolcanoplot%2F0.0.5','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"volcanoplot\">Volcano Plot<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">8 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fvsearch%2Fvsearch_clustering%2F2.8.3.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vsearch\">VSearch clustering<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fvsearch%2Fvsearch_masking%2F2.8.3.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vsearch\">VSearch masking<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fvsearch%2Fvsearch_dereplication%2F2.8.3.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vsearch\">VSearch dereplication<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fvsearch%2Fvsearch_chimera_detection%2F2.8.3.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vsearch\">VSearch chimera detection<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fvsearch%2Fvsearch_search%2F2.8.3.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vsearch\">VSearch search<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fvsearch%2Fvsearch_sorting%2F2.8.3.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vsearch\">VSearch sorting<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fvsearch%2Fvsearch_alignment%2F2.8.3.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vsearch\">VSearch alignment<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fvsearch%2Fvsearch_shuffling%2F2.8.3.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"vsearch\">VSearch shuffling<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fweblogo3%2Frgweblogo3%2F3.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"weblogo3\">Sequence Logo<\/button><\/ul>",null,"<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=wig_to_bigWig','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"wig_to_bigWig\">Wig/BedGraph-to-bigWig<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=wiggle2simple1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"wiggle2simple1\">Wiggle-to-Interval<\/button><\/ul>",null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">21 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Flecorguille%2Fxcms_summary%2Fabims_xcms_summary%2F3.4.4.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"workflow4metabolomics\">xcms process history<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fyguitton%2Fwithinvariation%2Fmixomics_multilevel%2F0.5.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"workflow4metabolomics\">Multilevel<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Feschen42%2Fw4mjoinpn%2Fw4mjoinpn%2F0.98.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"workflow4metabolomics\">Join +/- Ions<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Feschen42%2Fw4mcorcov%2Fw4mcorcov%2F0.98.17','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"workflow4metabolomics\">OPLS-DA_Contrasts<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Feschen42%2Fw4mclassfilter%2Fw4mclassfilter%2F0.98.11','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"workflow4metabolomics\">W4m Data Subset<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fethevenot%2Funivariate%2FUnivariate%2F2.2.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"workflow4metabolomics\">Univariate<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmelpetera%2Ftablemerge%2Ftablemerge%2F1.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"workflow4metabolomics\">Table Merge<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fethevenot%2Fqualitymetrics%2Fquality_metrics%2F2.2.8','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"workflow4metabolomics\">Quality Metrics<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fethevenot%2Fprofia%2Fprofia%2F3.1.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"workflow4metabolomics\">proFIA<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmarie-tremblay-metatoul%2Fnormalization%2Fnormalization%2F2.0.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"workflow4metabolomics\">Normalization<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmarie-tremblay-metatoul%2Fnmr_preprocessing%2FNMR_Read%2F3.3.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"workflow4metabolomics\">NMR_Read<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmarie-tremblay-metatoul%2Fnmr_preprocessing%2FNMR_Preprocessing%2F3.3.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"workflow4metabolomics\">NMR_Preprocessing<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmarie-tremblay-metatoul%2Fnmr_bucketing%2FNmrBucketing%2F2.0.3','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"workflow4metabolomics\">NMR_Bucketing<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmarie-tremblay-metatoul%2Fnmr_alignment%2FNmrAlignment%2F2.0.4','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"workflow4metabolomics\">NMR spectra alignment<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Flecorguille%2Fmsnbase_readmsdata%2Fmsnbase_readmsdata%2F2.8.2.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"workflow4metabolomics\">MSnbase readMSData<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fprog%2Flcmsmatching%2Flcmsmatching%2F4.0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"workflow4metabolomics\">LCMS matching<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Ffgiacomoni%2Fhmdb_ms_search%2Fwsdl_hmdb%2F1.6.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"workflow4metabolomics\">HMDB MS search<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmelpetera%2Fgeneric_filter%2Fgeneric_filter%2F2017.06','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"workflow4metabolomics\">Generic_Filter<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fethevenot%2Fcheckformat%2FcheckFormat%2F3.0.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"workflow4metabolomics\">Check Format<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmelpetera%2Fbatchcorrection%2FDetermine_BC%2F2.1.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"workflow4metabolomics\">Determine_batch_correction<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fmelpetera%2Fbatchcorrection%2FBatch_correction%2F2.1.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"workflow4metabolomics\">Batch_correction<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner/data_source_redirect?tool_id=wormbase','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"wormbase\">WormBase<\/button><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fecology%2Fxarray_select%2Fxarray_select%2F0.15.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"xarray\">NetCDF xarray Selection<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fecology%2Fxarray_metadata_info%2Fxarray_metadata_info%2F0.15.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"xarray\">NetCDF xarray Metadata Info<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">7 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Flecorguille%2Fxcms_xcmsset%2Fabims_xcms_xcmsSet%2F3.4.4.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"xcms\">xcms findChromPeaks (xcmsSet)<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Flecorguille%2Fxcms_retcor%2Fabims_xcms_retcor%2F3.4.4.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"xcms\">xcms adjustRtime (retcor)<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Flecorguille%2Fxcms_plot_chromatogram%2Fxcms_plot_chromatogram%2F3.4.4.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"xcms\">xcms plot chromatogram<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Flecorguille%2Fxcms_merge%2Fxcms_merge%2F3.4.4.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"xcms\">xcms findChromPeaks Merger<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Flecorguille%2Fxcms_group%2Fabims_xcms_group%2F3.4.4.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"xcms\">xcms groupChromPeaks (group)<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Flecorguille%2Fxcms_fillpeaks%2Fabims_xcms_fillPeaks%2F3.4.4.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"xcms\">xcms fillChromPeaks (fillPeaks)<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Flecorguille%2Fxcms_export_samplemetadata%2Fxcms_export_samplemetadata%2F3.4.4.0','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"xcms\">xcms get a sampleMetadata file<\/button><\/li><\/ul><\/ul>",null,null,"<ul class=\"galaxy-links-holder\"><li class=\"closed galaxy-collapsible\" onclick=\"$(this).parent('.galaxy-links-holder').find('.galaxy-links').toggle({duration:200,start:function(){$(this).parent('.galaxy-links-holder').find('.galaxy-collapsible').toggleClass('closed open')}})\" ><span class=\"galaxy-link-toggle\">2 tools<\/span><span class=\"button\"/><\/li><ul class=\"galaxy-links\" style=\"display: none;\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Firacooke%2Fxtandem%2Fproteomics_search_tandem_1%2F1.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"xtandem\">X!Tandem MSMS Search<\/button><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Firacooke%2Fxtandem%2Ftandem_to_pepxml_1%2F1.1.1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"xtandem\">Tandem to pepXML<\/button><\/li><\/ul><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Fxy_plot%2FXY_Plot_1%2F1.0.2','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"xy_plot\">Plotting tool<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fyahs%2Fyahs%2F1.2a.2%2Bgalaxy1','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"yahs\">YAHS<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner/data_source_redirect?tool_id=yeastmine','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"yeastmine\">YeastMine<\/button><\/ul>","<ul class=\"galaxy-links\"><button class=\"galaxy-link\" onclick=\"window.open('https://usegalaxy.org.au//tool_runner/data_source_redirect?tool_id=zebrafishmine','_blank').focus()\" ga-product=\"galaxy\" ga-id=\"zebrafishmine\">ZebrafishMine<\/button><\/ul>"],[null,"<p class=\"version\">2.2.3<\/p>",null,"<p class=\"version\">19.19.12<\/p><p class=\"version\">20<\/p><p class=\"version\">20-tools21<\/p><p class=\"version\">22<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">38.93<\/p>",null,"<p class=\"version\">1.9<\/p><p class=\"version\">1.12<\/p>",null,"<p class=\"version\">3.1.2<\/p>","<p class=\"version\">1.10.4<\/p>",null,null,null,null,"<p class=\"version\">2.28.0<\/p>",null,null,null,null,null,null,"<p class=\"version\">2.0.182<\/p>",null,null,null,null,null,"<p class=\"version\">2.10.1<\/p><p class=\"version\">2.11.0<\/p>",null,null,null,null,null,"<p class=\"version\">1.71.0<\/p><p class=\"version\">1.72.0<\/p><p class=\"version\">1.77.0<\/p><p class=\"version\">1.79.0<\/p><p class=\"version\">1.80.0<\/p>",null,"<p class=\"version\">2.3.5.1<\/p>",null,null,null,null,"<p class=\"version\">0.7.17<\/p>","<p class=\"version\">2.2.1<\/p>",null,null,null,null,"<p class=\"version\">2.0.3<\/p>",null,"<p class=\"version\">2.0<\/p><p class=\"version\">2.0t<\/p><p class=\"version\">1.9<\/p><p class=\"version\">2.1.1<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">10.1<\/p><p class=\"version\">11.0.3<\/p><p class=\"version\">11.2.2<\/p><p class=\"version\">11.4.1<\/p><p class=\"version\">11.6.1<\/p><p class=\"version\">11.7.0<\/p>","<p class=\"version\">7.6.5-cuda10.1<\/p><p class=\"version\">8.1.1-cuda11<\/p><p class=\"version\">8.2.2-cuda11.4<\/p><p class=\"version\">8.6.0-cuda11<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">3.0<\/p><p class=\"version\">3.0.1<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">0.20.0<\/p>","<p class=\"version\">0.11.7<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">4.1.4.0<\/p><p class=\"version\">4.2.1.0<\/p><p class=\"version\">4.1.8.1<\/p><p class=\"version\">4.2.5.0<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">20191022<\/p>","<p class=\"version\">2.6<\/p>",null,null,null,null,"<p class=\"version\">2019.3<\/p><p class=\"version\">2019.3-plumed<\/p><p class=\"version\">2019.3-gpuvolta<\/p><p class=\"version\">2018.3<\/p><p class=\"version\">2018.3-plumed<\/p><p class=\"version\">2020.1<\/p><p class=\"version\">2020.1-plumed<\/p><p class=\"version\">2020.1-gpuvolta<\/p><p class=\"version\">2020.3<\/p><p class=\"version\">2020.3-gpuvolta<\/p><p class=\"version\">2021-gpuvolta<\/p><p class=\"version\">2021<\/p><p class=\"version\">2021.2<\/p><p class=\"version\">2021.2-gpuvolta<\/p><p class=\"version\">2021.4<\/p><p class=\"version\">2021.4-gpuvolta<\/p><p class=\"version\">2022<\/p><p class=\"version\">2022-gpuvolta<\/p><p class=\"version\">2020.3-gpuampere<\/p><p class=\"version\">2019.3-gpuampere<\/p><p class=\"version\">2021-gpuampere<\/p><p class=\"version\">2020.1-gpuampere<\/p><p class=\"version\">2021.2-gpuampere<\/p><p class=\"version\">2021.4-gpuampere<\/p><p class=\"version\">2022-gpuampere<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">0.19.0<\/p><p class=\"version\">0.22.1<\/p>",null,"<p class=\"version\">1.9<\/p><p class=\"version\">1.12<\/p><p class=\"version\">1.16<\/p>",null,null,null,"<p class=\"version\">8.6<\/p><p class=\"version\">8.8<\/p>","<p class=\"version\">7.0.11<\/p>",null,null,"<p class=\"version\">5.2.1<\/p>","<p class=\"version\">2.13.1<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">2.3.0<\/p>",null,null,"<p class=\"version\">3.4.3-py3.9<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">2.17<\/p><p class=\"version\">2.22<\/p><p class=\"version\">2.24<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">1.9<\/p>",null,null,null,"<p class=\"version\">nll6i27dbl<\/p><p class=\"version\">nll6i27dbl-i8<\/p><p class=\"version\">nll6i27dbl-mkl<\/p><p class=\"version\">fll6i26dcl<\/p><p class=\"version\">fll6i26dcl-mkl<\/p>",null,null,null,null,null,null,null,null,"<p class=\"version\">1.8.5.0<\/p>","<p class=\"version\">6.6.2<\/p>","<p class=\"version\">4.7.7<\/p><p class=\"version\">4.9.2<\/p><p class=\"version\">5.0.5<\/p>","<p class=\"version\">2.1.7<\/p>","<p class=\"version\">4.7.1<\/p><p class=\"version\">4.7.1p<\/p><p class=\"version\">4.6.3<\/p><p class=\"version\">4.6.3p<\/p><p class=\"version\">4.7.3<\/p><p class=\"version\">4.7.3p<\/p><p class=\"version\">4.7.4<\/p><p class=\"version\">4.7.4p<\/p><p class=\"version\">4.6.3-i8r8<\/p><p class=\"version\">4.8.0<\/p><p class=\"version\">4.8.0p<\/p><p class=\"version\">4.9.0<\/p><p class=\"version\">4.9.0p<\/p>",null,null,"<p class=\"version\">21.04.3<\/p><p class=\"version\">22.04.3<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">5.8.0<\/p><p class=\"version\">5.8.0-mesa<\/p><p class=\"version\">5.8.0-gpu<\/p><p class=\"version\">5.9.1<\/p><p class=\"version\">5.9.1-mesa<\/p><p class=\"version\">5.9.1-gpu<\/p><p class=\"version\">5.10.1<\/p><p class=\"version\">5.9.1-mesa<\/p><p class=\"version\">5.10.1-mesa<\/p><p class=\"version\">5.9.1-gpu<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">5.6.40<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">1.11.2<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">1.4.0a0<\/p><p class=\"version\">1.5.1<\/p><p class=\"version\">1.9.0<\/p><p class=\"version\">1.10.0<\/p><p class=\"version\">1.12.1<\/p>",null,null,null,null,null,null,"<p class=\"version\">3.6.1<\/p><p class=\"version\">4.0.0<\/p><p class=\"version\">4.1.0<\/p><p class=\"version\">4.2.1<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">1.1.0<\/p>",null,null,null,null,"<p class=\"version\">0.8.1<\/p>","<p class=\"version\">0.1.24<\/p>",null,"<p class=\"version\">1.9<\/p><p class=\"version\">1.10<\/p><p class=\"version\">1.12<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">1.3<\/p>",null,null,null,null,null,null,null,null,null,"<p class=\"version\">0.8.0<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">2.41<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">2.0.0<\/p><p class=\"version\">2.1.0<\/p><p class=\"version\">2.3.0<\/p><p class=\"version\">2.4.1<\/p><p class=\"version\">2.6.0<\/p><p class=\"version\">2.8.0<\/p>",null,null,null,"<p class=\"version\">2.1.1<\/p>","<p class=\"version\">2019.3.14<\/p><p class=\"version\">2020.1.13<\/p>",null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">2.9.1<\/p><p class=\"version\">2.12.0<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">1.2.10<\/p>",null,"<p class=\"version\">3.1.2<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],[null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">3.4.0<\/p>",null,null,null,null,null,"<p class=\"version\">2.5.2<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">1.79<\/p>",null,"<p class=\"version\">2.13.0<\/p>",null,null,"<p class=\"version\">37<\/p>",null,null,null,null,null,null,null,null,"<p class=\"version\">5.2.1<\/p><p class=\"version\">5.4.0<\/p>",null,null,null,"<p class=\"version\">0.7.11<\/p>",null,null,null,null,null,null,null,null,null,"<p class=\"version\">4.8.1<\/p>",null,"<p class=\"version\">6.1.2<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">1.2.4<\/p>","<p class=\"version\">2.1<\/p>","<p class=\"version\">1.4.0<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">3.7<\/p>","<p class=\"version\">1.0.13<\/p>",null,null,null,"<p class=\"version\">1.0.20220720<\/p>","<p class=\"version\">1.0.0<\/p>","<p class=\"version\">1.2.0<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">4.9.0<\/p>",null,null,null,"<p class=\"version\">3.1.3<\/p>",null,null,null,null,"<p class=\"version\">2.2.0<\/p><p class=\"version\">2.4.0<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">2.1.11<\/p>",null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">2.9<\/p><p class=\"version\">2.9.1<\/p>",null,"<p class=\"version\">3-915ef7d<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">1.8<\/p><p class=\"version\">1.9<\/p>",null,null,null,null,"<p class=\"version\">1.0.0<\/p><p class=\"version\">1.0.0<\/p>","<p class=\"version\">1.6.2<\/p>",null,null,null,null,"<p class=\"version\">1.3.6<\/p>",null,null,null,"<p class=\"version\">0.12.7<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">2.0.0<\/p>","<p class=\"version\">0.16.1<\/p><p class=\"version\">0.18.9<\/p>",null,"<p class=\"version\">2.2.1<\/p>",null,"<p class=\"version\">0.180750<\/p>","<p class=\"version\">3.3.2<\/p>",null,null,null,null,null,null,null,null,"<p class=\"version\">1.5.0<\/p><p class=\"version\">1.8.0<\/p>",null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">0.9.84<\/p>","<p class=\"version\">2.1.2<\/p>",null,null,null,null,null,null,"<p class=\"version\">1.1.4<\/p>","<p class=\"version\">11.0.2<\/p>",null,null,null,null,null,null,null,null,"<p class=\"version\">2.4.2<\/p>","<p class=\"version\">0.0<\/p>",null,"<p class=\"version\">3.2.1<\/p>",null,null,null,null,null,"<p class=\"version\">0.9.100<\/p><p class=\"version\">1.0.0<\/p><p class=\"version\">1.0.1<\/p>","<p class=\"version\">1.04.15<\/p>",null,null,null,null,null,null,"<p class=\"version\">0.4.1<\/p>","<p class=\"version\">5.5.2.11<\/p>","<p class=\"version\">2.9.4<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">7.505<\/p>",null,"<p class=\"version\">3.01.04<\/p>",null,null,null,null,null,null,null,null,null,"<p class=\"version\">14-137<\/p>",null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">5-34c21f2<\/p>",null,null,null,null,null,null,null,null,null,"<p class=\"version\">0.3<\/p>","<p class=\"version\">4.12.0<\/p>",null,"<p class=\"version\">0.5<\/p>",null,null,null,null,null,"<p class=\"version\">13-45111<\/p>",null,null,null,null,null,"<p class=\"version\">0.6.0<\/p><p class=\"version\">0.7.0<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">1.2.4<\/p>",null,"<p class=\"version\">14.2.2<\/p><p class=\"version\">14.13.0<\/p><p class=\"version\">14.29.1<\/p>",null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">0.98-cluster_only<\/p>",null,"<p class=\"version\">1.0.1<\/p>",null,null,null,null,"<p class=\"version\">2.6.1<\/p>",null,null,null,null,null,null,"<p class=\"version\">2.5<\/p>",null,null,null,null,null,null,"<p class=\"version\">v5.26.3<\/p>",null,null,null,null,null,"<p class=\"version\">2.27.4<\/p>",null,null,null,null,null,null,null,"<p class=\"version\">v2.00a3.7<\/p>",null,null,null,null,null,null,null,null,null,"<p class=\"version\">2.6.3<\/p>",null,null,"<p class=\"version\">0<\/p>",null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">5.1.0rc1<\/p>",null,null,null,"<p class=\"version\">1.4.3<\/p>",null,null,null,null,null,null,null,"<p class=\"version\">1.08<\/p>",null,null,"<p class=\"version\">4.1.2-p1<\/p><p class=\"version\">4.1.5<\/p>","<p class=\"version\">2.0.3<\/p><p class=\"version\">2.0.4<\/p>","<p class=\"version\">1.0.6<\/p>",null,null,null,null,null,"<p class=\"version\">2.11.0<\/p><p class=\"version\">2.14.0<\/p>",null,null,null,null,null,null,"<p class=\"version\">3.12.1<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">1.4<\/p>",null,"<p class=\"version\">4.5.1<\/p>",null,null,"<p class=\"version\">2.2.0<\/p><p class=\"version\">2.3.1<\/p>",null,null,null,null,null,null,null,null,"<p class=\"version\">6.7.9<\/p>",null,"<p class=\"version\">6.0.1<\/p>","<p class=\"version\">0.3.0<\/p><p class=\"version\">1.0.0<\/p>",null,null,"<p class=\"version\">7.18.2<\/p>","<p class=\"version\">2006<\/p>","<p class=\"version\">2.0.2<\/p>",null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">2022.11.22<\/p>",null,"<p class=\"version\">3.36<\/p>",null,null,"<p class=\"version\">3.0.2<\/p>",null,null,"<p class=\"version\">2.7.10a<\/p>",null,null,null,"<p class=\"version\">2.0.3<\/p>",null,"<p class=\"version\">1.4.1<\/p>","<p class=\"version\">1.0.3<\/p>","<p class=\"version\">1.6<\/p>",null,null,null,null,null,null,null,null,null,"<p class=\"version\">1.4.6<\/p>",null,null,null,null,null,null,null,null,null,"<p class=\"version\">1.3.9<\/p>","<p class=\"version\">4.09.1<\/p>","<p class=\"version\">1.4.1<\/p>",null,"<p class=\"version\">0.39<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">106.1<\/p>",null,null,null,null,"<p class=\"version\">0.1.16<\/p>",null,"<p class=\"version\">1.1<\/p>",null,null,null,null,"<p class=\"version\">1.7<\/p>",null,null,"<p class=\"version\">2.03<\/p>",null,null,null,null,null,null,null,null,"<p class=\"version\">1.1<\/p>",null,null],[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">2.5.1--hd03093a_10<\/p>",null,null,null,null,"<p class=\"version\">38.96--h5c4e2a8_0<\/p>",null,"<p class=\"version\">1.15--haf5b3da_0<\/p>",null,null,"<p class=\"version\">1.10.4<\/p>","<p class=\"version\">2.6.7<\/p>",null,null,null,"<p class=\"version\">2.30.0--h468198e_3<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">2.12.0--pl5262h3289130_0<\/p>",null,null,null,null,null,null,null,"<p class=\"version\">2.4.5--py36hd4290be_0<\/p>",null,null,null,null,"<p class=\"version\">0.7.17--h7132678_9<\/p>","<p class=\"version\">2.2.1--hd03093a_2<\/p>",null,null,null,null,null,null,"<p class=\"version\">2.2--ha47f30e_0<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">1.2.4--h87f3376_5<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">3.7--py38hbff2b2d_0<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">2.0.14--hdcc8f71_0<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">1.5.1<\/p>","<p class=\"version\">3.0.22<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">0.11.9--hdfd78af_1<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">4.2.5.0--hdfd78af_0<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">3.01.03--pl526hb8757ab_0<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">3.2.7--h19cf415_2<\/p>",null,null,null,null,"<p class=\"version\">3.23--pl5321h1b792b2_13<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">0.8.1--h41abebc_0<\/p>",null,null,"<p class=\"version\">1.15--h3843a85_0<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">3.15.4--h95f258a_0<\/p>",null,null,null,null,null,null,null,null,null,"<p class=\"version\">2.7.10a--h9ee0642_0<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">0.39--hdfd78af_2<\/p>","<p class=\"version\">2.13.2--ha140323_0<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">0.1.16--pl5321h9a82719_6<\/p>","<p class=\"version\">1.2.10--h7132678_5<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],["<p class=\"version\">1.0.0-gompi-2021a<\/p>",null,null,null,"<p class=\"version\">2022.05<\/p>",null,null,null,"<p class=\"version\">0.4.2-gcccore-10.3.0<\/p>",null,null,null,null,"<p class=\"version\">3.4.0-foss-2021a<\/p>",null,null,null,null,null,"<p class=\"version\">2.5.2-gcc-10.3.0<\/p>",null,null,null,null,"<p class=\"version\">38.96-gcc-10.3.0<\/p>",null,"<p class=\"version\">1.12-gcc-10.3.0<\/p><p class=\"version\">1.15.1-gcc-11.3.0<\/p>","<p class=\"version\">5.4.22jul22.46e-java-11<\/p>",null,null,null,null,null,null,"<p class=\"version\">2.30.0-gcc-10.3.0<\/p>",null,null,null,"<p class=\"version\">1.7.3-gcc-10.3.0<\/p>",null,"<p class=\"version\">3.1<\/p>",null,null,null,"<p class=\"version\">1.7.8-gcccore-10.3.0<\/p>","<p class=\"version\">1.79-foss-2021a<\/p>",null,"<p class=\"version\">2.11.0-linux_x86_64<\/p><p class=\"version\">2.13.0--hf3cf87c_0<\/p>","<p class=\"version\">2.11.0-gompi-2021a<\/p>",null,null,null,null,"<p class=\"version\">1.76.0-gcc-10.3.0<\/p><p class=\"version\">1.79.0-gcc-11.3.0<\/p>",null,"<p class=\"version\">2.4.4-gcc-10.3.0<\/p><p class=\"version\">2.4.5-gcc-11.3.0<\/p>",null,null,null,"<p class=\"version\">5.4.2-foss-2021a<\/p>","<p class=\"version\">0.7.17-gcc-10.3.0<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">4.8.1-gcc-10.3.0<\/p>",null,"<p class=\"version\">7.1.0<\/p>",null,"<p class=\"version\">1.1.3-foss-2021a<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">11.3.1<\/p><p class=\"version\">11.7.0<\/p>","<p class=\"version\">8.2.1.32-cuda-11.3.1<\/p><p class=\"version\">8.4.1.50-cuda-11.7.0<\/p>",null,null,null,null,"<p class=\"version\">3.4-gcccore-10.3.0<\/p>",null,null,null,null,null,null,null,"<p class=\"version\">3.5.0-foss-2021a<\/p>","<p class=\"version\">4.5.2-gcccore-10.3.0<\/p>",null,null,null,null,"<p class=\"version\">2.0.13-gcc-10.3.0<\/p>",null,null,null,null,null,null,"<p class=\"version\">4.6.2<\/p><p class=\"version\">4.7.0<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">1.1--h0326b38_1<\/p>",null,null,null,null,null,null,null,"<p class=\"version\">1.0.0-gcccore-10.3.0<\/p>","<p class=\"version\">1.33-gcc-10.3.0<\/p>","<p class=\"version\">0.1.4--h7132678_6<\/p>","<p class=\"version\">0.23.2-gcc-11.3.0<\/p>","<p class=\"version\">0.11.9-java-11<\/p>",null,"<p class=\"version\">2.1.11-gcccore-10.3.0<\/p>",null,null,"<p class=\"version\">20190320-gcccore-10.3.0<\/p>",null,null,null,null,null,null,null,"<p class=\"version\">2.6.4-gcccore-10.3.0<\/p>","<p class=\"version\">2.9-gcc-10.3.0<\/p>",null,null,null,null,"<p class=\"version\">1.3.6-foss-2021a-r-4.1.0<\/p>","<p class=\"version\">20141119-gcccore-10.3.0<\/p>",null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">1.94.0beta-gfbf-2022a<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">0.1.a1a2bc9--hdfd78af_2<\/p>",null,"<p class=\"version\">0.12.7-gcccore-10.3.0<\/p>",null,"<p class=\"version\">9.54.0-gcccore-10.3.0<\/p>",null,"<p class=\"version\">20210622-gcccore-10.3.0<\/p>","<p class=\"version\">2.7-gcc-10.3.0<\/p><p class=\"version\">2.7-gcc-11.3.0<\/p>",null,null,null,null,"<p class=\"version\">2021.3-foss-2021a<\/p>","<p class=\"version\">1.7.0-foss-2021a<\/p><p class=\"version\">2.0.0-foss-2021a<\/p>",null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">0.16.1-gcccore-10.3.0<\/p>",null,"<p class=\"version\">2.2.1--h87f3376_4<\/p>",null,null,"<p class=\"version\">3.3.2-gompi-2021a<\/p>",null,null,"<p class=\"version\">1.12-gcc-10.3.0<\/p><p class=\"version\">1.15.1-gcc-11.3.0<\/p>","<p class=\"version\">3.6-foss-2022a<\/p>",null,null,null,"<p class=\"version\">7.0.11-14-gcccore-10.3.0<\/p>",null,null,null,null,null,null,null,"<p class=\"version\">5.55-88.0-foss-2021a<\/p>",null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">4.3.0-foss-2021a<\/p>",null,null,"<p class=\"version\">2.1-gcccore-10.3.0<\/p><p class=\"version\">2.1-gcccore-11.3.0<\/p>",null,null,"<p class=\"version\">2.3.0-gcc-10.3.0<\/p>",null,null,null,"<p class=\"version\">0.48.0-gompi-2021a<\/p><p class=\"version\">0.48.0--h15996b6_2<\/p>",null,null,null,null,"<p class=\"version\">1.3.0--hdfd78af_2<\/p>",null,"<p class=\"version\">2.1.2-gompi-2021a<\/p>",null,"<p class=\"version\">2.8.1-gcccore-11.3.0<\/p>",null,null,null,null,null,null,null,null,null,"<p class=\"version\">5.5.2.11-gcc-10.3.0<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">7.490-gcc-10.3.0-with-extensions<\/p>",null,null,null,"<p class=\"version\">2.3-gcc-10.3.0<\/p>",null,null,null,"<p class=\"version\">3.4.2-foss-2021a<\/p>",null,null,null,null,null,null,null,"<p class=\"version\">1.2.9-gcccore-10.3.0<\/p>",null,null,null,null,null,null,"<p class=\"version\">5-gcc-10.3.0<\/p>",null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">4.12.0<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">1.0.4--pyhdfd78af_0<\/p>",null,null,"<p class=\"version\">0.14.0--hb24e783_1<\/p>",null,null,null,null,null,null,null,null,"<p class=\"version\">4.8.0-gompi-2021a<\/p>",null,null,"<p class=\"version\">22.10.1<\/p>",null,"<p class=\"version\">1.10.2-gcccore-10.3.0<\/p><p class=\"version\">1.10.2-gcccore-11.3.0<\/p>",null,null,null,null,"<p class=\"version\">4.1.1--pyhdfd78af_0<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">2.25.1-java-11<\/p>",null,null,null,null,null,null,null,"<p class=\"version\">2.00a3.6-gcc-11.3.0<\/p>",null,null,null,null,null,null,"<p class=\"version\">1.1.alpha19<\/p>",null,null,"<p class=\"version\">2.6.3-gcccore-10.3.0<\/p>","<p class=\"version\">1.14.5-gompi-2021a<\/p>",null,null,null,"<p class=\"version\">0.3.18-foss-2021a<\/p>","<p class=\"version\">2.5.2-foss-2021a<\/p>",null,null,"<p class=\"version\">0.16.0.1-gcc-10.3.0<\/p>",null,"<p class=\"version\">2022.8<\/p>",null,null,null,"<p class=\"version\">5.0.2-foss-2021a<\/p>",null,"<p class=\"version\">4.1.0-foss-2021a<\/p><p class=\"version\">4.2.1-foss-2021a<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">4-10-foss-2021a-r-4.1.0<\/p>",null,"<p class=\"version\">4.1.2<\/p>",null,null,null,null,"<p class=\"version\">1.3.3--pl5321ha04fe3b_5<\/p>",null,null,null,null,null,"<p class=\"version\">1.4.0-gompi-2021a<\/p>",null,"<p class=\"version\">2.3--py27h16ec135_1<\/p>",null,null,null,null,null,"<p class=\"version\">1.13-gcc-10.3.0<\/p>",null,null,"<p class=\"version\">0.11.1-gcccore-10.3.0<\/p>",null,null,null,null,"<p class=\"version\">4.5.0-foss-2021a<\/p>",null,null,null,"<p class=\"version\">1.2.0-gcc-10.3.0<\/p>","<p class=\"version\">1.3-gcc-10.3.0<\/p>",null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">20160702-gcccore-10.3.0<\/p>",null,"<p class=\"version\">6.6.1-foss-2021a<\/p>",null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">3.15.3-gcc-10.3.0<\/p>","<p class=\"version\">1.0.1-gcccore-11.3.0<\/p>",null,null,null,null,null,"<p class=\"version\">3.0.3--h87f3376_0<\/p>","<p class=\"version\">1.1-gcccore-10.3.0<\/p>",null,"<p class=\"version\">2.7.10a--h9ee0642_0<\/p><p class=\"version\">2.7.9a-gcc-10.3.0<\/p>",null,null,"<p class=\"version\">2.1.7-gcc-10.3.0<\/p>",null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">20220427-linux64<\/p>",null,null,null,null,null,null,null,null,null,"<p class=\"version\">5.5.0--pl5321hdfd78af_5<\/p>",null,null,null,null,null,null,null,"<p class=\"version\">0.39-java-11<\/p>","<p class=\"version\">2.13.2--hea94271_3<\/p><p class=\"version\">2.9.1-foss-2021a<\/p><p class=\"version\">date.2011_11_26--ncurses5.9_8<\/p>","<p class=\"version\">3.2.2--pl5321hdfd78af_1<\/p>",null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">6.0-gcccore-10.3.0<\/p><p class=\"version\">6.0-gcccore-11.3.0<\/p>",null,null,null,null,"<p class=\"version\">1.0.3-foss-2021a-r-4.1.0<\/p>","<p class=\"version\">0.1.16-gcc-10.3.0<\/p>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<p class=\"version\">2.1--pyh5e36f6f_0<\/p>",null,null,null,null,null]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th class=\"dt-center\" style=\"font-size:11pt\" colspan=\"10\">Tool metadata<\/th>\n      <th class=\"dt-center\" style=\"font-size:11pt\" colspan=\"5\">Availability on Australian compute infrastructures<\/th>\n    <\/tr>\n    <tr>\n      <th>Name<\/th>\n      <th>Description<\/th>\n      <th>Homepage<\/th>\n      <th>Registry link<\/th>\n      <th>Tool identifier (e.g. module name)<\/th>\n      <th>Topic(s)<\/th>\n      <th>Publications<\/th>\n      <th>Containers available? (BioContainers)<\/th>\n      <th>License<\/th>\n      <th>Resources / documentation<\/th>\n      <th>Galaxy Australia<\/th>\n      <th>NCI (Gadi)<\/th>\n      <th>NCI (if89)<\/th>\n      <th>Pawsey (Setonix)<\/th>\n      <th>QRIScloud / UQ-RCC (Bunya)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"headerCallback":"function( thead, data, start, end, display ) {\n  $(thead).closest('thead').find('th').eq(0).css('background-color', '#205a86').css('color', '#ffffff');\n  $(thead).closest('thead').find('th').eq(1).css('background-color', '#2cb77c').css('color', '#ffffff');\n}","pageLength":1000,"dom":"Bfrtip","buttons":["copy","csv","colvis"],"columnDefs":[{"visible":false,"targets":[1,4,8,9]},{"className":"dt-center","targets":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14]}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[10,25,50,100,1000]},"callback":"function(table) {\ntable.on(\"search.dt\", function(evt) {gtag(\"event\", \"search\", {search_term: $(evt.target).parent().find(\"input\").val()})})\n}"},"evals":["options.headerCallback","callback"],"jsHooks":[]}</script>

<script>
  $(document).ready(function () {
    $('body').on('click', 'a', function (evt) { var l = $(evt.currentTarget); if (l.attr("ga-id") && l.attr("ga-product")) { gtag("event", "select_content", { content_type: l.attr("ga-product"), item_id: l.attr("ga-id") }); }; })
  });
</script>


<script>

  // add bootstrap table styles to pandoc tables
  function bootstrapStylePandocTables() {
    $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
  }
  $(document).ready(function () {
    bootstrapStylePandocTables();
  });


</script>


<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

